{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Note: include in the tests folder a .env file that contains the API keys for the services used in the tests\n",
    "import os\n",
    "if not os.environ.get('OPENAI_API_KEY'):\n",
    "    import context\n",
    "    \n",
    "from palimpzest.constants import PZ_DIR\n",
    "import palimpzest as pz\n",
    "\n",
    "import gradio as gr\n",
    "import numpy as np\n",
    "import pandas as pd\n",
    "\n",
    "import argparse\n",
    "import requests\n",
    "import json\n",
    "import time\n",
    "import os\n",
    "\n",
    "pz.DataDirectory().clearCache(keep_registry=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Schema definition\n",
    "In the following cell we target schema of the case study we are interested in. Note how we are not specifying target attributes with fine grained metadata but rather with natural language, possibly ambiguous, specifications."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "class CaseData(pz.Schema):\n",
    "    \"\"\"An individual row extracted from a table containing medical study data.\"\"\"\n",
    "    case_submitter_id = pz.Field(desc=\"The ID of the case\", required=True)\n",
    "    age_at_diagnosis = pz.Field(desc=\"The age of the patient at the time of diagnosis\", required=False)\n",
    "    race = pz.Field(desc=\"An arbitrary classification of a taxonomic group that is a division of a species.\", required=False)\n",
    "    ethnicity = pz.Field(desc=\"Whether an individual describes themselves as Hispanic or Latino or not.\", required=False)\n",
    "    gender = pz.Field(desc=\"Text designations that identify gender.\", required=False)\n",
    "    vital_status = pz.Field(desc=\"The vital status of the patient\", required=False)\n",
    "    ajcc_pathologic_t = pz.Field(desc=\"The AJCC pathologic T\", required=False)\n",
    "    ajcc_pathologic_n = pz.Field(desc=\"The AJCC pathologic N\", required=False)\n",
    "    ajcc_pathologic_stage = pz.Field(desc=\"The AJCC pathologic stage\", required=False)\n",
    "    tumor_grade = pz.Field(desc=\"The tumor grade\", required=False)\n",
    "    tumor_focality = pz.Field(desc=\"The tumor focality\", required=False)\n",
    "    tumor_largest_dimension_diameter = pz.Field(desc=\"The tumor largest dimension diameter\", required=False)\n",
    "    primary_diagnosis = pz.Field(desc=\"The primary diagnosis\", required=False)\n",
    "    morphology = pz.Field(desc=\"The morphology\", required=False)\n",
    "    tissue_or_organ_of_origin = pz.Field(desc=\"The tissue or organ of origin\", required=False)\n",
    "    # tumor_code = pz.Field(desc=\"The tumor code\", required=False)\n",
    "    study = pz.Field(desc=\"The last name of the author of the study, from the table name\", required=False)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Printing base dataset\n",
    "In the following cell we print the base dataset we are interested in. Note how we are not specifying target attributes with fine grained metadata but rather with natural language, possibly ambiguous, specifications."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "LOGICAL PLANS: 1\n",
      "INITIAL PLANS: 3\n",
      "DEDUP PLANS: 3\n",
      "PARETO PLANS: 3\n",
      "----------\n",
      "Policy is: Minimum Cost\n",
      "Chosen plan: Time est: 303.759 -- Cost est: 0.164 -- Quality est: 0.490\n",
      " 0. MarshalAndScanDataOp -> File \n",
      "\n",
      " 1. File -> InduceFromCandidateOp -> XLSFile \n",
      "    Using Model.GPT_3_5\n",
      "    (contents,filena...) -> (contents,filena...)\n",
      "\n",
      " 2. XLSFile -> InduceFromCandidateOp -> Table \n",
      "    Using Model.GPT_3_5\n",
      "    (contents,filena...) -> (filename,header...)\n",
      "\n",
      "Table name: dou_mmc1.xlsx_UCEC_CPTAC3_meta_table_V2.1\n",
      "idx | Proteomics_Participant_ID | Case_excluded | Proteomics_TMT_batch | Proteomics_TMT_plex | Prote ...\n",
      "S001 | C3L-00006 | No | 2 | 5 | 128N | C3L-00006-01 | CPT0001460012 | Tumor | No | United States | F ...\n",
      "S002 | C3L-00008 | No | 4 | 16 | 130N | C3L-00008-01 | CPT0001300009 | Tumor | No | United States |  ...\n",
      "S003 | C3L-00032 | No | 1 | 2 | 131 | C3L-00032-01 | CPT0001420009 | Tumor | No | United States | FI ...\n",
      "\n",
      "Table name: vasaikar_mmc1.xlsx_Description\n",
      "Sheet | Description ...\n",
      "A-Annotation | Column annotations ...\n",
      "B-ClinicalData | Clinical Information for 110 Colon cancer patients ...\n",
      "\n",
      "Table name: vasaikar_mmc1.xlsx_A-Annotation\n",
      "Column name | Column definition | Unnamed: 2 | Unnamed: 3 | Data type | Definition ...\n",
      "SampleID | ID of the prospective participant | nan | nan | Bin | Binary ...\n",
      "Clinical | Clinical data availability | nan | nan | Cat | Categorical ...\n",
      "Mutation | Mutation data availability | nan | nan | Con | Continuous ...\n",
      "\n",
      "Table name: vasaikar_mmc1.xlsx_B-ClinicalData\n",
      "SampleID | Clinical | Mutation | CNV | miRNA-seq | mRNA-seq | Label-free Proteome | TMT Proteome | T ...\n",
      "Type | BIN | BIN | BIN | BIN | BIN | BIN | BIN | BIN | BIN | BIN | CON | BIN | BIN | CAT | ORD | ORD ...\n",
      "01CO001 | 1 | 1 | 1 | 1 | 1 | 1 | nan | 1 | nan | 1 | 729 | Male | Mucinous | Sigmoid Colon | T4a |  ...\n",
      "01CO005 | 1 | 1 | 1 | 1 | 1 | 1 | 1 | 1 | 1 | 1 | 838 | Female | Not Mucinous | Sigmoid Colon | T3 | ...\n",
      "\n",
      "Table name: wang_mmc2.xlsx_README\n",
      "README for Table S1 | Unnamed: 1 | Unnamed: 2 ...\n",
      "nan | nan | nan ...\n",
      "Tab1 | clinical_data | Clinical data of the CPTAC GBM discovery cohort (99 tumors and 10 GTEx normal ...\n",
      "Tab2 | additional_annotations | Additional subtype and annotation for each case (see STAR Methods) ...\n",
      "\n",
      "Table name: wang_mmc2.xlsx_clinical_data\n",
      "case_id | age | gender | height | weight | bmi | country_of_origin | race | ethnicity | ethnicity_se ...\n",
      "C3L-00104 | 58 | Male | 188.0 | 115.0 | 32.54 | United States | White | Not-Hispanic or Latino | Cau ...\n",
      "C3L-00365 | 59 | Female | 162.0 | 54.0 | 20.61 | United States | White | Not-Hispanic or Latino | Ca ...\n",
      "C3L-00674 | 45 | Male | 193.0 | 102.0 | 27.44 | nan | White | Not-Hispanic or Latino | Caucasian | n ...\n",
      "\n",
      "Table name: wang_mmc2.xlsx_additional_annotations\n",
      "case | sample_type | multiomic | nmf_consensus | nmf_cluster_membership | rna_wang_cancer_cell_2017  ...\n",
      "C3L-00104 | tumor | nmf1 | nmf1 | 0.743 | Proneural | 0.678244020987608 | dm2 | 0.314080871350208 |  ...\n",
      "C3L-00365 | tumor | nmf3 | nmf3 | 0.614 | Classical | 0.681122006453892 | dm4 | 0.891148938751837 |  ...\n",
      "C3L-00674 | tumor | nmf1 | nmf1 | 0.507 | Mesenchymal | 0.744635304438083 | dm5 | 0.625633701322078  ...\n",
      "\n",
      "Table name: wang_mmc2.xlsx_cbttc_sample_info\n",
      "aliquot_id | sample_id | original_aliquot_id | batch | plex | tmt_channel | research_id | kids_first ...\n",
      "7316-288-TISS [894482] | 7316-288 | 7316-288-TISS [894482] | 1 | 1 | 127N | C54735 | PT_84Q11Z5X | A ...\n",
      "7316-89-TISS [894435] | 7316-89 | 7316-89-TISS [894435] | 1 | 1 | 128N | C25461 | PT_N32BYY8A | Aliv ...\n",
      "7316-2176-TISS [894450] | 7316-2176 | 7316-2176-TISS [894450] | 1 | 1 | 129N | C624225 | PT_VAJN5QP8 ...\n",
      "\n",
      "Table name: wang_mmc2.xlsx_survival_analysis\n",
      "Stratification group | Group value | n | events | median | 0.95LCL | 0.95UCL | log-rank p-value ...\n",
      "DNA methylation subtype\n",
      "(dna_methyl) | dm1 | 15 | 12 | 1.04 | 0.805 | nan | 0.55 ...\n",
      "nan | dm2 | 18 | 10 | 1.1 | 0.797 | nan | nan ...\n",
      "nan | dm3 | 21 | 14 | 1.07 | 0.931 | nan | nan ...\n",
      "\n"
     ]
    }
   ],
   "source": [
    "def print_tables(physicalTree):\n",
    "    for table in physicalTree:\n",
    "        header = table.header\n",
    "        subset_rows = table.rows[:3]\n",
    "\n",
    "        print(\"Table name:\", table.name)\n",
    "        print(\" | \".join(header)[:100], \"...\")\n",
    "        for row in subset_rows:\n",
    "            print(\" | \".join(row)[:100], \"...\")\n",
    "        print()\n",
    "\n",
    "policy = pz.MinCost()\n",
    "\n",
    "xls = pz.Dataset('biofabric-tiny', schema=pz.XLSFile)\n",
    "patient_tables = xls.convert(pz.Table, desc=\"All tables in the file\", cardinality=\"oneToMany\")\n",
    "output = patient_tables\n",
    "\n",
    "execution = pz.SimpleExecution(output, policy)\n",
    "physicalTree = execution.executeAndOptimize(verbose=True)\n",
    "print_tables(physicalTree)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Filtering stage\n",
    "In the following cell we define the first part of the workload, that comprises a filtering stage responsible for selecting the tables from all the spreadsheets that contain relevant biometric information about the patient.\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "LOGICAL PLANS: 1\n",
      "INITIAL PLANS: 9\n",
      "DEDUP PLANS: 9\n",
      "PARETO PLANS: 9\n",
      "----------\n",
      "Policy is: Minimum Cost\n",
      "Chosen plan: Time est: 303.889 -- Cost est: 0.173 -- Quality est: 0.412\n",
      " 0. MarshalAndScanDataOp -> File \n",
      "\n",
      " 1. File -> InduceFromCandidateOp -> XLSFile \n",
      "    Using Model.GPT_3_5\n",
      "    (contents,filena...) -> (contents,filena...)\n",
      "\n",
      " 2. XLSFile -> InduceFromCandidateOp -> Table \n",
      "    Using Model.GPT_3_5\n",
      "    (contents,filena...) -> (filename,header...)\n",
      "\n",
      " 3. Table -> FilterCandidateOp -> Table \n",
      "    Using Model.GPT_3_5\n",
      "    Filter: \"The table contains biometric information about the patient\"\n",
      "    (filename,header...) -> (filename,header...)\n",
      "\n",
      "Table name: dou_mmc1.xlsx_UCEC_CPTAC3_meta_table_V2.1\n",
      "idx | Proteomics_Participant_ID | Case_excluded | Proteomics_TMT_batch | Proteomics_TMT_plex | Prote ...\n",
      "S001 | C3L-00006 | No | 2 | 5 | 128N | C3L-00006-01 | CPT0001460012 | Tumor | No | United States | F ...\n",
      "S002 | C3L-00008 | No | 4 | 16 | 130N | C3L-00008-01 | CPT0001300009 | Tumor | No | United States |  ...\n",
      "S003 | C3L-00032 | No | 1 | 2 | 131 | C3L-00032-01 | CPT0001420009 | Tumor | No | United States | FI ...\n",
      "\n",
      "Table name: wang_mmc2.xlsx_clinical_data\n",
      "case_id | age | gender | height | weight | bmi | country_of_origin | race | ethnicity | ethnicity_se ...\n",
      "C3L-00104 | 58 | Male | 188.0 | 115.0 | 32.54 | United States | White | Not-Hispanic or Latino | Cau ...\n",
      "C3L-00365 | 59 | Female | 162.0 | 54.0 | 20.61 | United States | White | Not-Hispanic or Latino | Ca ...\n",
      "C3L-00674 | 45 | Male | 193.0 | 102.0 | 27.44 | nan | White | Not-Hispanic or Latino | Caucasian | n ...\n",
      "\n"
     ]
    }
   ],
   "source": [
    "# Make sure to run\n",
    "# pz reg --name biofabric-tiny --path testdata/biofabric-tiny\n",
    "pz.DataDirectory().clearCache(keep_registry=True)\n",
    "\n",
    "policy = pz.MinCost()\n",
    "\n",
    "xls = pz.Dataset('biofabric-tiny', schema=pz.XLSFile)\n",
    "patient_tables = xls.convert(pz.Table, desc=\"All tables in the file\", cardinality=\"oneToMany\")\n",
    "patient_tables = patient_tables.filterByStr(\"The table contains biometric information about the patient\")\n",
    "\n",
    "output = patient_tables\n",
    "\n",
    "execution = pz.SimpleExecution(output, policy)\n",
    "physicalTree = execution.executeAndOptimize(verbose=True)\n",
    "\n",
    "for table in physicalTree:\n",
    "    header = table.header\n",
    "    subset_rows = table.rows[:3]\n",
    "\n",
    "    print(\"Table name:\", table.name)\n",
    "    print(\" | \".join(header)[:100], \"...\")\n",
    "    for row in subset_rows:\n",
    "        print(\" | \".join(row)[:100], \"...\")\n",
    "    print()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Matching stage\n",
    "Once filtered, we can define the second part of the workload, that matches the patient records in the different tables and merging the information into a single table."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "LOGICAL PLANS: 1\n",
      "INITIAL PLANS: 1\n",
      "DEDUP PLANS: 1\n",
      "PARETO PLANS: 1\n",
      "----------\n",
      "Policy is: Minimum Cost\n",
      "Chosen plan: Time est: 0.000 -- Cost est: 0.000 -- Quality est: 1.000\n",
      " 0. CacheScanDataOp -> CaseData \n",
      "\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>age_at_diagnosis</th>\n",
       "      <th>ajcc_pathologic_n</th>\n",
       "      <th>ajcc_pathologic_stage</th>\n",
       "      <th>ajcc_pathologic_t</th>\n",
       "      <th>case_submitter_id</th>\n",
       "      <th>ethnicity</th>\n",
       "      <th>gender</th>\n",
       "      <th>morphology</th>\n",
       "      <th>primary_diagnosis</th>\n",
       "      <th>race</th>\n",
       "      <th>study</th>\n",
       "      <th>tissue_or_organ_of_origin</th>\n",
       "      <th>tumor_focality</th>\n",
       "      <th>tumor_grade</th>\n",
       "      <th>tumor_largest_dimension_diameter</th>\n",
       "      <th>vital_status</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>38.88</td>\n",
       "      <td>pN0</td>\n",
       "      <td>Stage I</td>\n",
       "      <td>pT1a (FIGO IA)</td>\n",
       "      <td>S001</td>\n",
       "      <td>Not-Hispanic or Latino</td>\n",
       "      <td>Female</td>\n",
       "      <td>Endometrioid</td>\n",
       "      <td>Carcinosarcoma</td>\n",
       "      <td>White</td>\n",
       "      <td>dou_mmc1.xlsx_UCEC_CPTAC3_meta_table_V2.1</td>\n",
       "      <td>Anterior endometrium</td>\n",
       "      <td>Unifocal</td>\n",
       "      <td>Cannot be determined</td>\n",
       "      <td>2.9</td>\n",
       "      <td>PASS</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>39.76</td>\n",
       "      <td>pNX</td>\n",
       "      <td>Stage IV</td>\n",
       "      <td>pT1a (FIGO IA)</td>\n",
       "      <td>S002</td>\n",
       "      <td>Not-Hispanic or Latino</td>\n",
       "      <td>Female</td>\n",
       "      <td>Endometrioid</td>\n",
       "      <td>Carcinosarcoma</td>\n",
       "      <td>White</td>\n",
       "      <td>dou_mmc1.xlsx_UCEC_CPTAC3_meta_table_V2.1</td>\n",
       "      <td>Posterior endometrium</td>\n",
       "      <td>Unifocal</td>\n",
       "      <td>Cannot be determined</td>\n",
       "      <td>3.5</td>\n",
       "      <td>PASS</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>51.19</td>\n",
       "      <td>pN0</td>\n",
       "      <td>Stage I</td>\n",
       "      <td>pT1a (FIGO IA)</td>\n",
       "      <td>S003</td>\n",
       "      <td>Not-Hispanic or Latino</td>\n",
       "      <td>Female</td>\n",
       "      <td>Endometrioid</td>\n",
       "      <td>Carcinosarcoma</td>\n",
       "      <td>White</td>\n",
       "      <td>dou_mmc1.xlsx_UCEC_CPTAC3_meta_table_V2.1</td>\n",
       "      <td>Anterior and Posterior endometrium</td>\n",
       "      <td>Unifocal</td>\n",
       "      <td>Cannot be determined</td>\n",
       "      <td>4.5</td>\n",
       "      <td>PASS</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>32.69</td>\n",
       "      <td>pNX</td>\n",
       "      <td>Stage I</td>\n",
       "      <td>pT1a (FIGO IA)</td>\n",
       "      <td>S005</td>\n",
       "      <td>Not-Hispanic or Latino</td>\n",
       "      <td>Female</td>\n",
       "      <td>Endometrioid</td>\n",
       "      <td>Carcinosarcoma</td>\n",
       "      <td>White</td>\n",
       "      <td>dou_mmc1.xlsx_UCEC_CPTAC3_meta_table_V2.1</td>\n",
       "      <td>Anterior and Posterior endometrium</td>\n",
       "      <td>Unifocal</td>\n",
       "      <td>Cannot be determined</td>\n",
       "      <td>3.5</td>\n",
       "      <td>PASS</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>729.00</td>\n",
       "      <td>N2b</td>\n",
       "      <td>Stage III</td>\n",
       "      <td>T4a</td>\n",
       "      <td>01CO001</td>\n",
       "      <td>Not Hispanic or Latino</td>\n",
       "      <td>Male</td>\n",
       "      <td>Mucinous</td>\n",
       "      <td>Sigmoid Colon</td>\n",
       "      <td>Unknown</td>\n",
       "      <td>vasaikar</td>\n",
       "      <td>Colon</td>\n",
       "      <td>Unspecified</td>\n",
       "      <td>4.9</td>\n",
       "      <td>3.333333</td>\n",
       "      <td>Living</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>838.00</td>\n",
       "      <td>N0</td>\n",
       "      <td>Stage II</td>\n",
       "      <td>T3</td>\n",
       "      <td>01CO005</td>\n",
       "      <td>Not Hispanic or Latino</td>\n",
       "      <td>Female</td>\n",
       "      <td>Not Mucinous</td>\n",
       "      <td>Sigmoid Colon</td>\n",
       "      <td>Unknown</td>\n",
       "      <td>vasaikar</td>\n",
       "      <td>Colon</td>\n",
       "      <td>Unspecified</td>\n",
       "      <td>1</td>\n",
       "      <td>5.9</td>\n",
       "      <td>Deceased</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>904.00</td>\n",
       "      <td>N2b</td>\n",
       "      <td>Stage III</td>\n",
       "      <td>T4a</td>\n",
       "      <td>01CO006</td>\n",
       "      <td>Not Hispanic or Latino</td>\n",
       "      <td>Female</td>\n",
       "      <td>Mucinous</td>\n",
       "      <td>Ascending Colon</td>\n",
       "      <td>Unknown</td>\n",
       "      <td>vasaikar</td>\n",
       "      <td>Colon</td>\n",
       "      <td>Unspecified</td>\n",
       "      <td>nan</td>\n",
       "      <td>3.166667</td>\n",
       "      <td>Living</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>652.00</td>\n",
       "      <td>N0</td>\n",
       "      <td>Stage II</td>\n",
       "      <td>T3</td>\n",
       "      <td>01CO008</td>\n",
       "      <td>Not Hispanic or Latino</td>\n",
       "      <td>Female</td>\n",
       "      <td>Mucinous</td>\n",
       "      <td>Descending Colon</td>\n",
       "      <td>Unknown</td>\n",
       "      <td>vasaikar</td>\n",
       "      <td>Colon</td>\n",
       "      <td>Unspecified</td>\n",
       "      <td>nan</td>\n",
       "      <td>6.7</td>\n",
       "      <td>Living</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>58.00</td>\n",
       "      <td>nan</td>\n",
       "      <td>PASS</td>\n",
       "      <td>1.0</td>\n",
       "      <td>C3L-00104</td>\n",
       "      <td>Not-Hispanic or Latino</td>\n",
       "      <td>Male</td>\n",
       "      <td>nan</td>\n",
       "      <td>Frontal Lobe</td>\n",
       "      <td>White</td>\n",
       "      <td>wang</td>\n",
       "      <td>Frontal Lobe</td>\n",
       "      <td>nan</td>\n",
       "      <td>nan</td>\n",
       "      <td>nan</td>\n",
       "      <td>Deceased</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>59.00</td>\n",
       "      <td>nan</td>\n",
       "      <td>FAIL</td>\n",
       "      <td>1.0</td>\n",
       "      <td>C3L-00365</td>\n",
       "      <td>Not-Hispanic or Latino</td>\n",
       "      <td>Female</td>\n",
       "      <td>nan</td>\n",
       "      <td>Parietal Lobe</td>\n",
       "      <td>White</td>\n",
       "      <td>wang</td>\n",
       "      <td>Parietal Lobe</td>\n",
       "      <td>nan</td>\n",
       "      <td>nan</td>\n",
       "      <td>nan</td>\n",
       "      <td>Deceased</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>45.00</td>\n",
       "      <td>nan</td>\n",
       "      <td>PASS</td>\n",
       "      <td>1.0</td>\n",
       "      <td>C3L-00674</td>\n",
       "      <td>Not-Hispanic or Latino</td>\n",
       "      <td>Male</td>\n",
       "      <td>nan</td>\n",
       "      <td>Frontal Lobe</td>\n",
       "      <td>White</td>\n",
       "      <td>wang</td>\n",
       "      <td>Frontal Lobe</td>\n",
       "      <td>nan</td>\n",
       "      <td>nan</td>\n",
       "      <td>nan</td>\n",
       "      <td>Deceased</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>69.00</td>\n",
       "      <td>nan</td>\n",
       "      <td>PASS</td>\n",
       "      <td>1.0</td>\n",
       "      <td>C3L-00677</td>\n",
       "      <td>Not-Hispanic or Latino</td>\n",
       "      <td>Female</td>\n",
       "      <td>nan</td>\n",
       "      <td>Frontal Lobe</td>\n",
       "      <td>White</td>\n",
       "      <td>wang</td>\n",
       "      <td>Frontal Lobe</td>\n",
       "      <td>nan</td>\n",
       "      <td>nan</td>\n",
       "      <td>nan</td>\n",
       "      <td>Deceased</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>77.00</td>\n",
       "      <td>nan</td>\n",
       "      <td>PASS</td>\n",
       "      <td>2.0</td>\n",
       "      <td>C3L-01040</td>\n",
       "      <td>nan</td>\n",
       "      <td>Male</td>\n",
       "      <td>nan</td>\n",
       "      <td>Frontal Lobe</td>\n",
       "      <td>nan</td>\n",
       "      <td>wang</td>\n",
       "      <td>Frontal Lobe</td>\n",
       "      <td>nan</td>\n",
       "      <td>nan</td>\n",
       "      <td>nan</td>\n",
       "      <td>Living</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    age_at_diagnosis ajcc_pathologic_n ajcc_pathologic_stage  \\\n",
       "0              38.88               pN0               Stage I   \n",
       "1              39.76               pNX              Stage IV   \n",
       "2              51.19               pN0               Stage I   \n",
       "3              32.69               pNX               Stage I   \n",
       "4             729.00               N2b             Stage III   \n",
       "5             838.00                N0              Stage II   \n",
       "6             904.00               N2b             Stage III   \n",
       "7             652.00                N0              Stage II   \n",
       "8              58.00               nan                  PASS   \n",
       "9              59.00               nan                  FAIL   \n",
       "10             45.00               nan                  PASS   \n",
       "11             69.00               nan                  PASS   \n",
       "12             77.00               nan                  PASS   \n",
       "\n",
       "   ajcc_pathologic_t case_submitter_id               ethnicity  gender  \\\n",
       "0     pT1a (FIGO IA)              S001  Not-Hispanic or Latino  Female   \n",
       "1     pT1a (FIGO IA)              S002  Not-Hispanic or Latino  Female   \n",
       "2     pT1a (FIGO IA)              S003  Not-Hispanic or Latino  Female   \n",
       "3     pT1a (FIGO IA)              S005  Not-Hispanic or Latino  Female   \n",
       "4                T4a           01CO001  Not Hispanic or Latino    Male   \n",
       "5                 T3           01CO005  Not Hispanic or Latino  Female   \n",
       "6                T4a           01CO006  Not Hispanic or Latino  Female   \n",
       "7                 T3           01CO008  Not Hispanic or Latino  Female   \n",
       "8                1.0         C3L-00104  Not-Hispanic or Latino    Male   \n",
       "9                1.0         C3L-00365  Not-Hispanic or Latino  Female   \n",
       "10               1.0         C3L-00674  Not-Hispanic or Latino    Male   \n",
       "11               1.0         C3L-00677  Not-Hispanic or Latino  Female   \n",
       "12               2.0         C3L-01040                     nan    Male   \n",
       "\n",
       "      morphology primary_diagnosis     race  \\\n",
       "0   Endometrioid    Carcinosarcoma    White   \n",
       "1   Endometrioid    Carcinosarcoma    White   \n",
       "2   Endometrioid    Carcinosarcoma    White   \n",
       "3   Endometrioid    Carcinosarcoma    White   \n",
       "4       Mucinous     Sigmoid Colon  Unknown   \n",
       "5   Not Mucinous     Sigmoid Colon  Unknown   \n",
       "6       Mucinous   Ascending Colon  Unknown   \n",
       "7       Mucinous  Descending Colon  Unknown   \n",
       "8            nan      Frontal Lobe    White   \n",
       "9            nan     Parietal Lobe    White   \n",
       "10           nan      Frontal Lobe    White   \n",
       "11           nan      Frontal Lobe    White   \n",
       "12           nan      Frontal Lobe      nan   \n",
       "\n",
       "                                        study  \\\n",
       "0   dou_mmc1.xlsx_UCEC_CPTAC3_meta_table_V2.1   \n",
       "1   dou_mmc1.xlsx_UCEC_CPTAC3_meta_table_V2.1   \n",
       "2   dou_mmc1.xlsx_UCEC_CPTAC3_meta_table_V2.1   \n",
       "3   dou_mmc1.xlsx_UCEC_CPTAC3_meta_table_V2.1   \n",
       "4                                    vasaikar   \n",
       "5                                    vasaikar   \n",
       "6                                    vasaikar   \n",
       "7                                    vasaikar   \n",
       "8                                        wang   \n",
       "9                                        wang   \n",
       "10                                       wang   \n",
       "11                                       wang   \n",
       "12                                       wang   \n",
       "\n",
       "             tissue_or_organ_of_origin tumor_focality           tumor_grade  \\\n",
       "0                 Anterior endometrium       Unifocal  Cannot be determined   \n",
       "1                Posterior endometrium       Unifocal  Cannot be determined   \n",
       "2   Anterior and Posterior endometrium       Unifocal  Cannot be determined   \n",
       "3   Anterior and Posterior endometrium       Unifocal  Cannot be determined   \n",
       "4                                Colon    Unspecified                   4.9   \n",
       "5                                Colon    Unspecified                     1   \n",
       "6                                Colon    Unspecified                   nan   \n",
       "7                                Colon    Unspecified                   nan   \n",
       "8                         Frontal Lobe            nan                   nan   \n",
       "9                        Parietal Lobe            nan                   nan   \n",
       "10                        Frontal Lobe            nan                   nan   \n",
       "11                        Frontal Lobe            nan                   nan   \n",
       "12                        Frontal Lobe            nan                   nan   \n",
       "\n",
       "   tumor_largest_dimension_diameter vital_status  \n",
       "0                               2.9         PASS  \n",
       "1                               3.5         PASS  \n",
       "2                               4.5         PASS  \n",
       "3                               3.5         PASS  \n",
       "4                          3.333333       Living  \n",
       "5                               5.9     Deceased  \n",
       "6                          3.166667       Living  \n",
       "7                               6.7       Living  \n",
       "8                               nan     Deceased  \n",
       "9                               nan     Deceased  \n",
       "10                              nan     Deceased  \n",
       "11                              nan     Deceased  \n",
       "12                              nan       Living  "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# Make sure to run\n",
    "# pz reg --name biofabric-tiny-filtered --path testdata/biofabric-tiny-filtered\n",
    "pz.DataDirectory().clearCache(keep_registry=True)\n",
    "\n",
    "input_dataset = pz.Dataset('biofabric-tiny-filtered', schema=pz.XLSFile)\n",
    "patient_tables = input_dataset.convert(pz.Table, desc=\"All tables in the file\", cardinality=\"oneToMany\")\n",
    "case_data = patient_tables.convert(CaseData, desc=\"The patient data in the table\",cardinality=\"oneToMany\")\n",
    "\n",
    "matched_tables = pz.SimpleExecution(case_data, policy)   \n",
    "matched_tables = matched_tables.executeAndOptimize(verbose=True)\n",
    "\n",
    "output_rows = []\n",
    "for output_table in matched_tables:\n",
    "    output_rows.append(output_table.asDict()) \n",
    "\n",
    "output_df = pd.DataFrame(output_rows)\n",
    "display(output_df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "LOGICAL PLANS: 1\n",
      "INITIAL PLANS: 9\n",
      "DEDUP PLANS: 9\n",
      "PARETO PLANS: 7\n",
      "----------\n",
      "Policy is: Maximum Quality\n",
      "Chosen plan: Time est: 425.756 -- Cost est: 0.956 -- Quality est: 0.523\n",
      " 0. MarshalAndScanDataOp -> File \n",
      "\n",
      " 1. File -> InduceFromCandidateOp -> XLSFile \n",
      "    Using Model.GPT_3_5\n",
      "    (contents,filena...) -> (contents,filena...)\n",
      "\n",
      " 2. XLSFile -> InduceFromCandidateOp -> Table \n",
      "    Using Model.GPT_4\n",
      "    (contents,filena...) -> (filename,header...)\n",
      "\n",
      " 3. Table -> InduceFromCandidateOp -> CaseData \n",
      "    Using Model.GPT_4\n",
      "    (filename,header...) -> (age_at_diagnosi...)\n",
      "\n",
      "Bonded query processing error: No output objects were generated with bonded query - trying with conventional query...\n",
      "BondedQuery Error: No output objects were generated with bonded query - trying with conventional query...\n",
      "Falling back to conventional query\n",
      "Could not find any items in the JSON response\n",
      "Could not find any items in the JSON response\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>age_at_diagnosis</th>\n",
       "      <th>ajcc_pathologic_n</th>\n",
       "      <th>ajcc_pathologic_stage</th>\n",
       "      <th>ajcc_pathologic_t</th>\n",
       "      <th>case_submitter_id</th>\n",
       "      <th>ethnicity</th>\n",
       "      <th>gender</th>\n",
       "      <th>morphology</th>\n",
       "      <th>primary_diagnosis</th>\n",
       "      <th>race</th>\n",
       "      <th>study</th>\n",
       "      <th>tissue_or_organ_of_origin</th>\n",
       "      <th>tumor_focality</th>\n",
       "      <th>tumor_grade</th>\n",
       "      <th>tumor_largest_dimension_diameter</th>\n",
       "      <th>vital_status</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>64.0</td>\n",
       "      <td>pN0</td>\n",
       "      <td>Stage I</td>\n",
       "      <td>pT1a (FIGO IA)</td>\n",
       "      <td>C3L-00006</td>\n",
       "      <td>Not-Hispanic or Latino</td>\n",
       "      <td>Female</td>\n",
       "      <td>Endometrioid</td>\n",
       "      <td>Endometrioid</td>\n",
       "      <td>White</td>\n",
       "      <td>UCEC_CPTAC3</td>\n",
       "      <td>Anterior endometrium</td>\n",
       "      <td>Unifocal</td>\n",
       "      <td>FIGO grade 1</td>\n",
       "      <td>2.9</td>\n",
       "      <td>Not provided</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>58.0</td>\n",
       "      <td>pNX</td>\n",
       "      <td>Stage IV</td>\n",
       "      <td>pT1a (FIGO IA)</td>\n",
       "      <td>C3L-00008</td>\n",
       "      <td>Not-Hispanic or Latino</td>\n",
       "      <td>Female</td>\n",
       "      <td>Endometrioid</td>\n",
       "      <td>Endometrioid</td>\n",
       "      <td>White</td>\n",
       "      <td>UCEC_CPTAC3</td>\n",
       "      <td>Posterior endometrium</td>\n",
       "      <td>Unifocal</td>\n",
       "      <td>FIGO grade 1</td>\n",
       "      <td>3.5</td>\n",
       "      <td>Not provided</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>50.0</td>\n",
       "      <td>pN0</td>\n",
       "      <td>Stage I</td>\n",
       "      <td>pT1a (FIGO IA)</td>\n",
       "      <td>C3L-00032</td>\n",
       "      <td>Not-Hispanic or Latino</td>\n",
       "      <td>Female</td>\n",
       "      <td>Endometrioid</td>\n",
       "      <td>Endometrioid</td>\n",
       "      <td>White</td>\n",
       "      <td>UCEC_CPTAC3</td>\n",
       "      <td>Anterior and Posterior endometrium</td>\n",
       "      <td>Unifocal</td>\n",
       "      <td>FIGO grade 2</td>\n",
       "      <td>4.5</td>\n",
       "      <td>Not provided</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Not provided</td>\n",
       "      <td>Not provided</td>\n",
       "      <td>Not provided</td>\n",
       "      <td>Not provided</td>\n",
       "      <td>C3L-00084</td>\n",
       "      <td>Not provided</td>\n",
       "      <td>Not provided</td>\n",
       "      <td>Carcinosarcoma</td>\n",
       "      <td>Carcinosarcoma</td>\n",
       "      <td>Not provided</td>\n",
       "      <td>UCEC_CPTAC3</td>\n",
       "      <td>Not provided</td>\n",
       "      <td>Not provided</td>\n",
       "      <td>Not provided</td>\n",
       "      <td>Not provided</td>\n",
       "      <td>Not provided</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>75.0</td>\n",
       "      <td>pNX</td>\n",
       "      <td>Stage I</td>\n",
       "      <td>pT1a (FIGO IA)</td>\n",
       "      <td>C3L-00090</td>\n",
       "      <td>Not-Hispanic or Latino</td>\n",
       "      <td>Female</td>\n",
       "      <td>Endometrioid</td>\n",
       "      <td>Endometrioid</td>\n",
       "      <td>White</td>\n",
       "      <td>UCEC_CPTAC3</td>\n",
       "      <td>Anterior and Posterior endometrium</td>\n",
       "      <td>Unifocal</td>\n",
       "      <td>FIGO grade 2</td>\n",
       "      <td>3.5</td>\n",
       "      <td>Not provided</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>729</td>\n",
       "      <td>N2b</td>\n",
       "      <td>Stage III</td>\n",
       "      <td>T4a</td>\n",
       "      <td>01CO001</td>\n",
       "      <td>None</td>\n",
       "      <td>Male</td>\n",
       "      <td>None</td>\n",
       "      <td>Mucinous</td>\n",
       "      <td>None</td>\n",
       "      <td>vasaikar</td>\n",
       "      <td>Sigmoid Colon</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>Living</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>838</td>\n",
       "      <td>N0</td>\n",
       "      <td>Stage II</td>\n",
       "      <td>T3</td>\n",
       "      <td>01CO005</td>\n",
       "      <td>None</td>\n",
       "      <td>Female</td>\n",
       "      <td>None</td>\n",
       "      <td>Not Mucinous</td>\n",
       "      <td>None</td>\n",
       "      <td>vasaikar</td>\n",
       "      <td>Sigmoid Colon</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>Deceased</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>904</td>\n",
       "      <td>N2b</td>\n",
       "      <td>Stage III</td>\n",
       "      <td>T4a</td>\n",
       "      <td>01CO006</td>\n",
       "      <td>None</td>\n",
       "      <td>Female</td>\n",
       "      <td>None</td>\n",
       "      <td>Mucinous</td>\n",
       "      <td>None</td>\n",
       "      <td>vasaikar</td>\n",
       "      <td>Ascending Colon</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>Living</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>652</td>\n",
       "      <td>N0</td>\n",
       "      <td>Stage II</td>\n",
       "      <td>T3</td>\n",
       "      <td>01CO008</td>\n",
       "      <td>None</td>\n",
       "      <td>Female</td>\n",
       "      <td>None</td>\n",
       "      <td>Mucinous</td>\n",
       "      <td>None</td>\n",
       "      <td>vasaikar</td>\n",
       "      <td>Descending Colon</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>Living</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>58</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>C3L-00104</td>\n",
       "      <td>Not-Hispanic or Latino</td>\n",
       "      <td>Male</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>White</td>\n",
       "      <td>wang</td>\n",
       "      <td>Frontal Lobe</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>Deceased</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>59</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>C3L-00365</td>\n",
       "      <td>Not-Hispanic or Latino</td>\n",
       "      <td>Female</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>White</td>\n",
       "      <td>wang</td>\n",
       "      <td>Parietal Lobe</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>Deceased</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>45</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>C3L-00674</td>\n",
       "      <td>Not-Hispanic or Latino</td>\n",
       "      <td>Male</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>White</td>\n",
       "      <td>wang</td>\n",
       "      <td>Frontal Lobe</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>Deceased</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>69</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>C3L-00677</td>\n",
       "      <td>Not-Hispanic or Latino</td>\n",
       "      <td>Female</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>White</td>\n",
       "      <td>wang</td>\n",
       "      <td>Frontal Lobe</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>Deceased</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>77</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>C3L-01040</td>\n",
       "      <td>None</td>\n",
       "      <td>Male</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>wang</td>\n",
       "      <td>Frontal Lobe</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>Living</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   age_at_diagnosis ajcc_pathologic_n ajcc_pathologic_stage ajcc_pathologic_t  \\\n",
       "0              64.0               pN0               Stage I    pT1a (FIGO IA)   \n",
       "1              58.0               pNX              Stage IV    pT1a (FIGO IA)   \n",
       "2              50.0               pN0               Stage I    pT1a (FIGO IA)   \n",
       "3      Not provided      Not provided          Not provided      Not provided   \n",
       "4              75.0               pNX               Stage I    pT1a (FIGO IA)   \n",
       "5               729               N2b             Stage III               T4a   \n",
       "6               838                N0              Stage II                T3   \n",
       "7               904               N2b             Stage III               T4a   \n",
       "8               652                N0              Stage II                T3   \n",
       "9                58              None                  None              None   \n",
       "10               59              None                  None              None   \n",
       "11               45              None                  None              None   \n",
       "12               69              None                  None              None   \n",
       "13               77              None                  None              None   \n",
       "\n",
       "   case_submitter_id               ethnicity        gender      morphology  \\\n",
       "0          C3L-00006  Not-Hispanic or Latino        Female    Endometrioid   \n",
       "1          C3L-00008  Not-Hispanic or Latino        Female    Endometrioid   \n",
       "2          C3L-00032  Not-Hispanic or Latino        Female    Endometrioid   \n",
       "3          C3L-00084            Not provided  Not provided  Carcinosarcoma   \n",
       "4          C3L-00090  Not-Hispanic or Latino        Female    Endometrioid   \n",
       "5            01CO001                    None          Male            None   \n",
       "6            01CO005                    None        Female            None   \n",
       "7            01CO006                    None        Female            None   \n",
       "8            01CO008                    None        Female            None   \n",
       "9          C3L-00104  Not-Hispanic or Latino          Male            None   \n",
       "10         C3L-00365  Not-Hispanic or Latino        Female            None   \n",
       "11         C3L-00674  Not-Hispanic or Latino          Male            None   \n",
       "12         C3L-00677  Not-Hispanic or Latino        Female            None   \n",
       "13         C3L-01040                    None          Male            None   \n",
       "\n",
       "   primary_diagnosis          race        study  \\\n",
       "0       Endometrioid         White  UCEC_CPTAC3   \n",
       "1       Endometrioid         White  UCEC_CPTAC3   \n",
       "2       Endometrioid         White  UCEC_CPTAC3   \n",
       "3     Carcinosarcoma  Not provided  UCEC_CPTAC3   \n",
       "4       Endometrioid         White  UCEC_CPTAC3   \n",
       "5           Mucinous          None     vasaikar   \n",
       "6       Not Mucinous          None     vasaikar   \n",
       "7           Mucinous          None     vasaikar   \n",
       "8           Mucinous          None     vasaikar   \n",
       "9               None         White         wang   \n",
       "10              None         White         wang   \n",
       "11              None         White         wang   \n",
       "12              None         White         wang   \n",
       "13              None          None         wang   \n",
       "\n",
       "             tissue_or_organ_of_origin tumor_focality   tumor_grade  \\\n",
       "0                 Anterior endometrium       Unifocal  FIGO grade 1   \n",
       "1                Posterior endometrium       Unifocal  FIGO grade 1   \n",
       "2   Anterior and Posterior endometrium       Unifocal  FIGO grade 2   \n",
       "3                         Not provided   Not provided  Not provided   \n",
       "4   Anterior and Posterior endometrium       Unifocal  FIGO grade 2   \n",
       "5                        Sigmoid Colon           None          None   \n",
       "6                        Sigmoid Colon           None          None   \n",
       "7                      Ascending Colon           None          None   \n",
       "8                     Descending Colon           None          None   \n",
       "9                         Frontal Lobe           None          None   \n",
       "10                       Parietal Lobe           None          None   \n",
       "11                        Frontal Lobe           None          None   \n",
       "12                        Frontal Lobe           None          None   \n",
       "13                        Frontal Lobe           None          None   \n",
       "\n",
       "   tumor_largest_dimension_diameter  vital_status  \n",
       "0                               2.9  Not provided  \n",
       "1                               3.5  Not provided  \n",
       "2                               4.5  Not provided  \n",
       "3                      Not provided  Not provided  \n",
       "4                               3.5  Not provided  \n",
       "5                              None        Living  \n",
       "6                              None      Deceased  \n",
       "7                              None        Living  \n",
       "8                              None        Living  \n",
       "9                              None      Deceased  \n",
       "10                             None      Deceased  \n",
       "11                             None      Deceased  \n",
       "12                             None      Deceased  \n",
       "13                             None        Living  "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "pz.DataDirectory().clearCache(keep_registry=True)\n",
    "\n",
    "policy = pz.MaxQuality()\n",
    "input_dataset = pz.Dataset('biofabric-tiny-filtered', schema=pz.XLSFile)\n",
    "patient_tables = input_dataset.convert(pz.Table, cardinality=\"oneToMany\")\n",
    "case_data = patient_tables.convert(CaseData, desc=\"The patient data in the table\",cardinality=\"oneToMany\")\n",
    "\n",
    "matched_tables = pz.SimpleExecution(case_data, policy)   \n",
    "matched_tables = matched_tables.executeAndOptimize(verbose=True)\n",
    "\n",
    "output_rows = []\n",
    "for output_table in matched_tables:\n",
    "    output_rows.append(output_table.asDict()) \n",
    "\n",
    "output_df = pd.DataFrame(output_rows)\n",
    "display(output_df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "palimpzest",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.14"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
