<!DOCTYPE html>
<html lang="en"     class="pb-page"  data-request-id="ad8b3f08-d5c1-41b3-b196-63a2ce63ca7b"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;issue:issue:pii\:S0092867419X00252;website:website:cell-site;ctype:string:Journal Content;article:article:pii\:S0092867420314008;journal:journal:cell;pageGroup:string:Article View;page:string:Show Full Text;requestedJournal:journal:cell;wgroup:string:Default Website Group;subPage:string:Full Text"/>




<!-- if there is any customization for Responsive Project widget  -->




























        
                <meta name="robots" content="noarchive"/>
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1"/>











        <meta property="og:title" content="Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy" />
    


        <meta property="og:type" content="Article" />
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(20)31400-8" />
    



        <meta property="og:site_name" content="Cell" />
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/356e593f-1a8e-43e4-ba5e-fd94d795b599/fx1.jpg" />
    


        <meta property="og:description" content="Breast cancer is a highly heterogeneous disease with variable outcomes and subtype-driven
treatment approaches, making precision medicine a considerable challenge. Proteogenomic
analyses of 122 primary breast cancers provide insights into clinically relevant biology,
including cell cycle dysregulation, tumor immunogenicity, aberrant metabolism, and
heterogeneity in therapeutic target expression." />
    











        <meta name="twitter:title" content="Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy" />
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/356e593f-1a8e-43e4-ba5e-fd94d795b599/fx1.jpg" />
    


        <meta name="twitter:description" content="Breast cancer is a highly heterogeneous disease with variable outcomes and subtype-driven
treatment approaches, making precision medicine a considerable challenge. Proteogenomic
analyses of 122 primary breast cancers provide insights into clinically relevant biology,
including cell cycle dysregulation, tumor immunogenicity, aberrant metabolism, and
heterogeneity in therapeutic target expression." />
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(20)31400-8?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867420314008%3Fshowall%3Dtrue">
    


<title>Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy: Cell</title><meta name="citation_journal_title" content="Cell"/> <meta name="citation_journal_abbrev" content="Cell"/> <meta name="citation_publisher" content="Elsevier"/> <meta name="citation_title" content="Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy"/> <meta name="citation_date" content="2020/11/25"/> <meta name="citation_online_date" content="2020/11/18"/> <meta name="citation_volume" content="183"/> <meta name="citation_issue" content="5"/> <meta name="citation_firstpage" content="1436"/> <meta name="citation_lastpage" content="1456.e31"/> <meta name="citation_pmid" content="33212010"/> <meta name="citation_abstract_html_url" content="http://www.cell.com/article/S0092867420314008/abstract"/> <meta name="citation_fulltext_html_url" content="http://www.cell.com/article/S0092867420314008/fulltext"/> <meta name="citation_pdf_url" content="http://www.cell.com/article/S0092867420314008/pdf"/> <meta name="citation_issn" content="0092-8674"/> <meta name="citation_issn" content="1097-4172"/> <meta name="citation_language" content="English"/> <meta name="citation_doi" content="10.1016/j.cell.2020.10.036"/> <meta name="citation_pmid" content="33212010"/> <meta name="citation_author" content="Karsten Krug"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="Eric J. Jaehnig"></meta><meta name="citation_author_orcid" content="0000-0001-9618-6011"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center and Dan L. Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Shankha Satpathy"></meta><meta name="citation_author_orcid" content="0000-0002-6040-4854"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="Lili Blumenberg"></meta><meta name="citation_author_institution" content="Institute for Systems Genetics and Department of Medicine, NYU Grossman School of Medicine, New York, NY 10016, USA"></meta><meta name="citation_author" content="Alla Karpova"></meta><meta name="citation_author_orcid" content="0000-0003-0467-0215"></meta><meta name="citation_author_institution" content="Department of Medicine and Genetics, Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA"></meta><meta name="citation_author" content="Meenakshi Anurag"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center and Dan L. Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="George Miles"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center and Dan L. Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Philipp Mertins"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author_institution" content="Max Delbrück Center for Molecular Medicine in the Helmholtz Society and Berlin Institute of Health, Berlin, Germany"></meta><meta name="citation_author" content="Yifat Geffen"></meta><meta name="citation_author_orcid" content="0000-0003-3692-4689"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="Lauren C. Tang"></meta><meta name="citation_author_orcid" content="0000-0001-8786-5418"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author_institution" content="Department of Biological Sciences, Columbia University, New York, NY 10027, USA"></meta><meta name="citation_author" content="David I. Heiman"></meta><meta name="citation_author_orcid" content="0000-0001-7895-4259"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="Song Cao"></meta><meta name="citation_author_institution" content="Department of Medicine and Genetics, Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA"></meta><meta name="citation_author" content="Yosef E. Maruvka"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="Jonathan T. Lei"></meta><meta name="citation_author_orcid" content="0000-0002-0209-8051"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center and Dan L. Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Chen Huang"></meta><meta name="citation_author_orcid" content="0000-0003-2777-7365"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center and Dan L. Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Ramani B. Kothadia"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="Antonio Colaprico"></meta><meta name="citation_author_orcid" content="0000-0003-1044-4361"></meta><meta name="citation_author_institution" content="Division of Biostatistics, Department of Public Health Science, University of Miami Miller School of Medicine, Miami, FL 33136, USA"></meta><meta name="citation_author" content="Chet Birger"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="Jarey Wang"></meta><meta name="citation_author_institution" content="Verna and Marrs McLean Department of Biochemistry and Molecular Biology, Department of Molecular and Human Genetics, and Therapeutic Innovation Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Yongchao Dou"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center and Dan L. Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Bo Wen"></meta><meta name="citation_author_orcid" content="0000-0003-2261-3150"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center and Dan L. Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Zhiao Shi"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center and Dan L. Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Yuxing Liao"></meta><meta name="citation_author_orcid" content="0000-0002-1238-2021"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center and Dan L. Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Maciej Wiznerowicz"></meta><meta name="citation_author_institution" content="Poznan University of Medical Sciences, Poznań 61-701, Poland"></meta><meta name="citation_author_institution" content="International Institute for Molecular Oncology, 60-203 Poznań, Poland"></meta><meta name="citation_author" content="Matthew A. Wyczalkowski"></meta><meta name="citation_author_institution" content="Department of Medicine and Genetics, Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA"></meta><meta name="citation_author" content="Xi Steven Chen"></meta><meta name="citation_author_institution" content="Division of Biostatistics, Department of Public Health Science, University of Miami Miller School of Medicine, Miami, FL 33136, USA"></meta><meta name="citation_author" content="Jacob J. Kennedy"></meta><meta name="citation_author_institution" content="Clinical Research Division, Fred Hutchinson Cancer Research Center, Seattle, WA 98109, USA"></meta><meta name="citation_author" content="Amanda G. Paulovich"></meta><meta name="citation_author_institution" content="Clinical Research Division, Fred Hutchinson Cancer Research Center, Seattle, WA 98109, USA"></meta><meta name="citation_author" content="Mathangi Thiagarajan"></meta><meta name="citation_author_orcid" content="0000-0003-3190-2691"></meta><meta name="citation_author_institution" content="Leidos Biomedical Research Inc., Frederick National Laboratory for Cancer Research, Frederick, MD 21702, USA"></meta><meta name="citation_author" content="Christopher R. Kinsinger"></meta><meta name="citation_author_orcid" content="0000-0001-5015-4428"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Tara Hiltke"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Emily S. Boja"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Mehdi Mesri"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Ana I. Robles"></meta><meta name="citation_author_orcid" content="0000-0001-5019-4374"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Henry Rodriguez"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Thomas F. Westbrook"></meta><meta name="citation_author_institution" content="Verna and Marrs McLean Department of Biochemistry and Molecular Biology, Department of Molecular and Human Genetics, and Therapeutic Innovation Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Li Ding"></meta><meta name="citation_author_institution" content="Department of Medicine and Genetics, Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA"></meta><meta name="citation_author" content="Gad Getz"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author_institution" content="Center for Cancer Research, Massachusetts General Hospital, Charlestown, MA 02114, USA"></meta><meta name="citation_author" content="Karl R. Clauser"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="David Fenyö"></meta><meta name="citation_author_institution" content="Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA"></meta><meta name="citation_author" content="Kelly V. Ruggles"></meta><meta name="citation_author_orcid" content="0000-0002-0152-0863"></meta><meta name="citation_author_institution" content="Institute for Systems Genetics and Department of Medicine, NYU Grossman School of Medicine, New York, NY 10016, USA"></meta><meta name="citation_author" content="Bing Zhang"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center and Dan L. Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="D.R. Mani"></meta><meta name="citation_author_email" content="manidr@broadinstitute.org"></meta><meta name="citation_author_orcid" content="0000-0003-3914-133X"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="Steven A. Carr"></meta><meta name="citation_author_email" content="scarr@broad.mit.edu"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="Matthew J. Ellis"></meta><meta name="citation_author_email" content="mjellis@bcm.edu"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center and Dan L. Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Michael A. Gillette"></meta><meta name="citation_author_email" content="gillette@broadinstitute.org"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author_institution" content="Division of Pulmonary and Critical Care Medicine, Massachusetts General Hospital, Boston, MA 02114, USA"></meta><meta name="citation_author" content="Shayan C. Avanessian"></meta><meta name="citation_author" content="Shuang Cai"></meta><meta name="citation_author" content="Daniel Chan"></meta><meta name="citation_author" content="Xian Chen"></meta><meta name="citation_author" content="Nathan J. Edwards"></meta><meta name="citation_author" content="Andrew N. Hoofnagle"></meta><meta name="citation_author" content="M. Harry Kane"></meta><meta name="citation_author" content="Karen A. Ketchum"></meta><meta name="citation_author" content="Eric Kuhn"></meta><meta name="citation_author" content="Douglas A. Levine"></meta><meta name="citation_author" content="Shunqiang Li"></meta><meta name="citation_author" content="Daniel C. Liebler"></meta><meta name="citation_author" content="Tao Liu"></meta><meta name="citation_author" content="Jingqin Luo"></meta><meta name="citation_author" content="Subha Madhavan"></meta><meta name="citation_author" content="Chris Maher"></meta><meta name="citation_author" content="Jason E. McDermott"></meta><meta name="citation_author" content="Peter B. McGarvey"></meta><meta name="citation_author" content="Mauricio Oberti"></meta><meta name="citation_author" content="Akhilesh Pandey"></meta><meta name="citation_author" content="Samuel H. Payne"></meta><meta name="citation_author" content="David F. Ransohoff"></meta><meta name="citation_author" content="Robert C. Rivers"></meta><meta name="citation_author" content="Karin D. Rodland"></meta><meta name="citation_author" content="Paul Rudnick"></meta><meta name="citation_author" content="Melinda E. Sanders"></meta><meta name="citation_author" content="Kenna M. Shaw"></meta><meta name="citation_author" content="Ie-Ming Shih"></meta><meta name="citation_author" content="Robbert J.C. Slebos"></meta><meta name="citation_author" content="Richard D. Smith"></meta><meta name="citation_author" content="Michael Snyder"></meta><meta name="citation_author" content="Stephen E. Stein"></meta><meta name="citation_author" content="David L. Tabb"></meta><meta name="citation_author" content="Ratna R. Thangudu"></meta><meta name="citation_author" content="Stefani Thomas"></meta><meta name="citation_author" content="Yue Wang"></meta><meta name="citation_author" content="Forest M. White"></meta><meta name="citation_author" content="Jeffrey R. Whiteaker"></meta><meta name="citation_author" content="Gordon A. Whiteley"></meta><meta name="citation_author" content="Hui Zhang"></meta><meta name="citation_author" content="Zhen Zhang"></meta><meta name="citation_author" content="Yingming Zhao"></meta><meta name="citation_author" content="Heng Zhu"></meta><meta name="citation_author" content="Lisa J. Zimmerman"></meta><meta name="citation_keywords" content="CPTAC; proteogenomics; breast cancer; genomics; proteomics; phosphoproteomics; acetylation; mass spectrometry; immune checkpoint therapy; CDK 4/6 inhibitors"></meta><meta name="citation_reference" content="citation_title=A Novel Sirtuin-3 Inhibitor, LC-0296, Inhibits Cell Survival and Proliferation, and Promotes Apoptosis of Head and Neck Cancer Cells; citation_author=T.Y. Alhazzazi; citation_author=P. Kamarajan; citation_author=Y. Xu; citation_author=T. Ai; citation_author=L. Chen; citation_author=E. Verdin; citation_author=Y.L. Kapila; citation_journal_title=Anticancer Res.; citation_volume=36; citation_firstpage=49; citation_lastpage=60; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Lysine Acetylation Goes Global: From Epigenetics to Metabolism and Therapeutics; citation_author=I. Ali; citation_author=R.J. Conrad; citation_author=E. Verdin; citation_author=M. Ott; citation_journal_title=Chem. Rev.; citation_volume=118; citation_firstpage=1216; citation_lastpage=1252; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Characterization of the immunophenotypes and antigenomes of colorectal cancers reveals distinct tumor escape mechanisms and novel targets for immunotherapy; citation_author=M. Angelova; citation_author=P. Charoentong; citation_author=H. Hackl; citation_author=M.L. Fischer; citation_author=R. Snajder; citation_author=A.M. Krogsdam; citation_author=M.J. Waldner; citation_author=G. Bindea; citation_author=B. Mlecnik; citation_author=J. Galon; citation_author=Z. Trajanoski; citation_journal_title=Genome Biol.; citation_volume=16; citation_firstpage=64; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Comprehensive Profiling of DNA Repair Defects in Breast Cancer Identifies a Novel Class of Endocrine Therapy Resistance Drivers; citation_author=M. Anurag; citation_author=N. Punturi; citation_author=J. Hoog; citation_author=M.N. Bainbridge; citation_author=M.J. Ellis; citation_author=S. Haricharan; citation_journal_title=Clin. Cancer Res.; citation_volume=24; citation_firstpage=4887; citation_lastpage=4899; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=DNA damage repair defects as a new class of endocrine treatment resistance driver; citation_author=M. Anurag; citation_author=M.J. Ellis; citation_author=S. Haricharan; citation_journal_title=Oncotarget; citation_volume=9; citation_firstpage=36252; citation_lastpage=36253; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Immune Checkpoint Profiles in Luminal B Breast Cancer (Alliance); citation_author=M. Anurag; citation_author=M. Zhu; citation_author=C. Huang; citation_author=S. Vasaikar; citation_author=J. Wang; citation_author=J. Hoog; citation_author=S. Burugu; citation_author=D. Gao; citation_author=V. Suman; citation_author=X.H. Zhang; citation_author=et al.; citation_journal_title=J. Natl. Cancer Inst; citation_volume=112; citation_firstpage=737; citation_lastpage=746; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=xCell: digitally portraying the tissue cellular heterogeneity landscape; citation_author=D. Aran; citation_author=Z. Hu; citation_author=A.J. Butte; citation_journal_title=Genome Biol.; citation_volume=18; citation_firstpage=220; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Single-Cell Dynamics Determines Response to CDK4/6 Inhibition in Triple-Negative Breast Cancer; citation_author=U.S. Asghar; citation_author=A.R. Barr; citation_author=R. Cutts; citation_author=M. Beaney; citation_author=I. Babina; citation_author=D. Sampath; citation_author=J. Giltnane; citation_author=J.A. Lacap; citation_author=L. Crocker; citation_author=A. Young; citation_author=et al.; citation_journal_title=Clin. Cancer Res.; citation_volume=23; citation_firstpage=5561; citation_lastpage=5572; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Gene ontology: tool for the unification of biology; citation_author=M. Ashburner; citation_author=C.A. Ball; citation_author=J.A. Blake; citation_author=D. Botstein; citation_author=H. Butler; citation_author=J.M. Cherry; citation_author=A.P. Davis; citation_author=K. Dolinski; citation_author=S.S. Dwight; citation_author=J.T. Eppig; citation_author=The Gene Ontology Consortium; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=25; citation_firstpage=25; citation_lastpage=29; citation_publication_date=2000; "></meta><meta name="citation_reference" content="citation_title=Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1; citation_author=D.A. Barbie; citation_author=P. Tamayo; citation_author=J.S. Boehm; citation_author=S.Y. Kim; citation_author=S.E. Moody; citation_author=I.F. Dunn; citation_author=A.C. Schinzel; citation_author=P. Sandy; citation_author=E. Meylan; citation_author=C. Scholl; citation_author=et al.; citation_journal_title=Nature; citation_volume=462; citation_firstpage=108; citation_lastpage=112; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=A comparison of nine confidence intervals for a Poisson parameter when the expected number of events is ≤ 5; citation_author=L. Barker; citation_journal_title=Am. Stat.; citation_volume=56; citation_firstpage=85; citation_lastpage=89; citation_publication_date=2002; "></meta><meta name="citation_reference" content="citation_title=Erratum to: Estimating the population abundance of tissue-infiltrating immune and stromal cell populations using gene expression; citation_author=E. Becht; citation_author=N.A. Giraldo; citation_author=L. Lacroix; citation_author=B. Buttard; citation_author=N. Elarouci; citation_author=F. Petitprez; citation_author=J. Selves; citation_author=P. Laurent-Puig; citation_author=C. Sautès-Fridman; citation_author=W.H. Fridman; citation_author=A. de Reyniès; citation_journal_title=Genome Biol.; citation_volume=17; citation_firstpage=249; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Estimating the population abundance of tissue-infiltrating immune and stromal cell populations using gene expression; citation_author=E. Becht; citation_author=N.A. Giraldo; citation_author=L. Lacroix; citation_author=B. Buttard; citation_author=N. Elarouci; citation_author=F. Petitprez; citation_author=J. Selves; citation_author=P. Laurent-Puig; citation_author=C. Sautès-Fridman; citation_author=W.H. Fridman; citation_author=A. de Reyniès; citation_journal_title=Genome Biol.; citation_volume=17; citation_firstpage=218; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing; citation_author=Y. Benjamini; citation_author=Y. Hochberg; citation_journal_title=J. R. Stat. Soc. Series B Methodol.; citation_volume=57; citation_firstpage=289; citation_lastpage=300; citation_publication_date=1995; "></meta><meta name="citation_reference" content="citation_title=COMPARTMENTS: unification and visualization of protein subcellular localization evidence; citation_author=J.X. Binder; citation_author=S. Pletscher-Frankild; citation_author=K. Tsafou; citation_author=C. Stolte; citation_author=S.I. O’Donoghue; citation_author=R. Schneider; citation_author=L.J. Jensen; citation_journal_title=Database (Oxford); citation_volume=2014; citation_firstpage=bau012; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=BlackSheep: A Bioconductor and Bioconda package for differential extreme value analysis; citation_author=L. Blumenberg; citation_author=E. Kawaler; citation_author=M. Cornwell; citation_author=S. Smith; citation_author=K. Ruggles; citation_author=D. Fenyö; citation_journal_title=bioRxiv; citation_doi=10.1101/825067; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Breast Cancer Classification Based on Proteotypes Obtained by SWATH Mass Spectrometry; citation_author=P. Bouchal; citation_author=O.T. Schubert; citation_author=J. Faktor; citation_author=L. Capkova; citation_author=H. Imrichova; citation_author=K. Zoufalova; citation_author=V. Paralova; citation_author=R. Hrstka; citation_author=Y. Liu; citation_author=H.A. Ebhardt; citation_author=et al.; citation_journal_title=Cell Rep.; citation_volume=28; citation_firstpage=832; citation_lastpage=843.e7; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Metagenes and molecular pattern discovery using matrix factorization; citation_author=J.-P. Brunet; citation_author=P. Tamayo; citation_author=T.R. Golub; citation_author=J.P. Mesirov; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=101; citation_firstpage=4164; citation_lastpage=4169; citation_publication_date=2004; "></meta><meta name="citation_reference" content="citation_title=Estrogen signaling and the DNA damage response in hormone dependent breast cancers; citation_author=C.E. Caldon; citation_journal_title=Front. Oncol.; citation_volume=4; citation_firstpage=106; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Comprehensive molecular portraits of human breast tumours; citation_author=Cancer Genome Atlas Network; citation_journal_title=Nature; citation_volume=490; citation_firstpage=61; citation_lastpage=70; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=Absolute quantification of somatic DNA alterations in human cancer; citation_author=S.L. Carter; citation_author=K. Cibulskis; citation_author=E. Helman; citation_author=A. McKenna; citation_author=H. Shen; citation_author=T. Zack; citation_author=P.W. Laird; citation_author=R.C. Onofrio; citation_author=W. Winckler; citation_author=B.A. Weir; citation_author=et al.; citation_journal_title=Nat. Biotechnol.; citation_volume=30; citation_firstpage=413; citation_lastpage=421; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=TNBCtype: A Subtyping Tool for Triple-Negative Breast Cancer; citation_author=X. Chen; citation_author=J. Li; citation_author=W.H. Gray; citation_author=B.D. Lehmann; citation_author=J.A. Bauer; citation_author=Y. Shyr; citation_author=J.A. Pietenpol; citation_journal_title=Cancer Inform.; citation_volume=11; citation_firstpage=147; citation_lastpage=156; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=Sirtuin-3 (SIRT3), a therapeutic target with oncogenic and tumor-suppressive function in cancer; citation_author=Y. Chen; citation_author=L.L. Fu; citation_author=X. Wen; citation_author=X.Y. Wang; citation_author=J. Liu; citation_author=Y. Cheng; citation_author=J. Huang; citation_journal_title=Cell Death Dis.; citation_volume=5; citation_firstpage=e1047; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Effects of mTOR Inhibitors on Components of the Salvador-Warts-Hippo Pathway; citation_author=J. Chiang; citation_author=J.A. Martinez-Agosto; citation_journal_title=Cells; citation_volume=1; citation_firstpage=886; citation_lastpage=904; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=Lysine acetylation targets protein complexes and co-regulates major cellular functions; citation_author=C. Choudhary; citation_author=C. Kumar; citation_author=F. Gnad; citation_author=M.L. Nielsen; citation_author=M. Rehman; citation_author=T.C. Walther; citation_author=J.V. Olsen; citation_author=M. Mann; citation_journal_title=Science; citation_volume=325; citation_firstpage=834; citation_lastpage=840; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=Domains A and B in the Rb pocket interact to form a transcriptional repressor motif; citation_author=K.N. Chow; citation_author=D.C. Dean; citation_journal_title=Mol. Cell. Biol.; citation_volume=16; citation_firstpage=4862; citation_lastpage=4868; citation_publication_date=1996; "></meta><meta name="citation_reference" content="citation_title=ContEst: estimating cross-contamination of human samples in next-generation sequencing data; citation_author=K. Cibulskis; citation_author=A. McKenna; citation_author=T. Fennell; citation_author=E. Banks; citation_author=M. DePristo; citation_author=G. Getz; citation_journal_title=Bioinformatics; citation_volume=27; citation_firstpage=2601; citation_lastpage=2602; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples; citation_author=K. Cibulskis; citation_author=M.S. Lawrence; citation_author=S.L. Carter; citation_author=A. Sivachenko; citation_author=D. Jaffe; citation_author=C. Sougnez; citation_author=S. Gabriel; citation_author=M. Meyerson; citation_author=E.S. Lander; citation_author=G. Getz; citation_journal_title=Nat. Biotechnol.; citation_volume=31; citation_firstpage=213; citation_lastpage=219; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Tailoring therapies--improving the management of early breast cancer: St Gallen International Expert Consensus on the Primary Therapy of Early Breast Cancer 2015; citation_author=A.S. Coates; citation_author=E.P. Winer; citation_author=A. Goldhirsch; citation_author=R.D. Gelber; citation_author=M. Gnant; citation_author=M. Piccart-Gebhart; citation_author=B. Thürlimann; citation_author=H.-J. Senn; citation_author=Panel Members; citation_journal_title=Ann. Oncol.; citation_volume=26; citation_firstpage=1533; citation_lastpage=1546; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data; citation_author=A. Colaprico; citation_author=T.C. Silva; citation_author=C. Olsen; citation_author=L. Garofano; citation_author=C. Cava; citation_author=D. Garolini; citation_author=T.S. Sabedot; citation_author=T.M. Malta; citation_author=S.M. Pagnotta; citation_author=I. Castiglioni; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=44; citation_firstpage=e71; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Interpreting pathways to discover cancer driver genes with Moonlight; citation_author=A. Colaprico; citation_author=C. Olsen; citation_author=M.H. Bailey; citation_author=G.J. Odom; citation_author=T. Terkelsen; citation_author=T.C. Silva; citation_author=A.V. Olsen; citation_author=L. Cantini; citation_author=A. Zinovyev; citation_author=E. Barillot; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=11; citation_firstpage=69; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=Genomic alterations in breast cancer: level of evidence for actionability according to ESMO Scale for Clinical Actionability of molecular Targets (ESCAT); citation_author=R. Condorelli; citation_author=F. Mosele; citation_author=B. Verret; citation_author=T. Bachelot; citation_author=P.L. Bedard; citation_author=J. Cortes; citation_author=D.M. Hyman; citation_author=D. Juric; citation_author=I. Krop; citation_author=I. Bieche; citation_author=et al.; citation_journal_title=Ann. Oncol.; citation_volume=30; citation_firstpage=365; citation_lastpage=373; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=DGIdb 3.0: a redesign and expansion of the drug-gene interaction database; citation_author=K.C. Cotto; citation_author=A.H. Wagner; citation_author=Y.-Y. Feng; citation_author=S. Kiwala; citation_author=A.C. Coffman; citation_author=G. Spies; citation_author=A. Wollam; citation_author=N.C. Spies; citation_author=O.L. Griffith; citation_author=M. Griffith; citation_journal_title=Nucleic Acids Res.; citation_volume=46; citation_number=D1; citation_firstpage=D1068; citation_lastpage=D1073; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=The genomic and transcriptomic architecture of 2,000 breast tumours reveals novel subgroups; citation_author=C. Curtis; citation_author=S.P. Shah; citation_author=S.-F. Chin; citation_author=G. Turashvili; citation_author=O.M. Rueda; citation_author=M.J. Dunning; citation_author=D. Speed; citation_author=A.G. Lynch; citation_author=S. Samarajiwa; citation_author=Y. Yuan; citation_author=METABRIC Group; citation_author=et al.; citation_journal_title=Nature; citation_volume=486; citation_firstpage=346; citation_lastpage=352; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=Molecular, phenotypic, and sample-associated data to describe pluripotent stem cell lines and derivatives; citation_author=K. Daily; citation_author=S.J. Ho Sui; citation_author=L.M. Schriml; citation_author=P.J. Dexheimer; citation_author=N. Salomonis; citation_author=R. Schroll; citation_author=S. Bush; citation_author=M. Keddache; citation_author=C. Mayhew; citation_author=S. Lotia; citation_author=et al.; citation_journal_title=Sci. Data; citation_volume=4; citation_firstpage=170030; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Triple-negative breast cancer in African-American women: disparities versus biology; citation_author=E.C. Dietze; citation_author=C. Sistrunk; citation_author=G. Miranda-Carboni; citation_author=R. O’Regan; citation_author=V.L. Seewaldt; citation_journal_title=Nat. Rev. Cancer; citation_volume=15; citation_firstpage=248; citation_lastpage=254; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=The IRE1 endoplasmic reticulum stress sensor activates natural killer cell immunity in part by regulating c-Myc; citation_author=H. Dong; citation_author=N.M. Adams; citation_author=Y. Xu; citation_author=J. Cao; citation_author=D.S.J. Allan; citation_author=J.R. Carlyle; citation_author=X. Chen; citation_author=J.C. Sun; citation_author=L.H. Glimcher; citation_journal_title=Nat. Immunol.; citation_volume=20; citation_firstpage=865; citation_lastpage=878; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Proteogenomic Characterization of Endometrial Carcinoma; citation_author=Y. Dou; citation_author=E.A. Kawaler; citation_author=D. Cui Zhou; citation_author=M.A. Gritsenko; citation_author=C. Huang; citation_author=L. Blumenberg; citation_author=A. Karpova; citation_author=V.A. Petyuk; citation_author=S.R. Savage; citation_author=S. Satpathy; citation_author=Clinical Proteomic Tumor Analysis Consortium; citation_author=et al.; citation_journal_title=Cell; citation_volume=180; citation_firstpage=729; citation_lastpage=748.e26; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=Ki67 Proliferation Index as a Tool for Chemotherapy Decisions During and After Neoadjuvant Aromatase Inhibitor Treatment of Breast Cancer: Results From the American College of Surgeons Oncology Group Z1031 Trial (Alliance); citation_author=M.J. Ellis; citation_author=V.J. Suman; citation_author=J. Hoog; citation_author=R. Goncalves; citation_author=S. Sanati; citation_author=C.J. Creighton; citation_author=K. DeSchryver; citation_author=E. Crouch; citation_author=A. Brink; citation_author=M. Watson; citation_author=et al.; citation_journal_title=J. Clin. Oncol.; citation_volume=35; citation_firstpage=1061; citation_lastpage=1069; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=The Reactome Pathway Knowledgebase; citation_author=A. Fabregat; citation_author=S. Jupe; citation_author=L. Matthews; citation_author=K. Sidiropoulos; citation_author=M. Gillespie; citation_author=P. Garapati; citation_author=R. Haw; citation_author=B. Jassal; citation_author=F. Korninger; citation_author=B. May; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=46; citation_number=D1; citation_firstpage=D649; citation_lastpage=D655; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Regulatory mechanisms in the control of protein kinases; citation_author=D.A. Flockhart; citation_author=J.D. Corbin; citation_journal_title=CRC Crit. Rev. Biochem.; citation_volume=12; citation_firstpage=133; citation_lastpage=186; citation_publication_date=1982; "></meta><meta name="citation_reference" content="citation_title=A flexible R package for nonnegative matrix factorization; citation_author=R. Gaujoux; citation_author=C. Seoighe; citation_journal_title=BMC Bioinformatics; citation_volume=11; citation_firstpage=367; citation_publication_date=2010; "></meta><meta name="citation_reference" content="citation_title=Next-generation characterization of the Cancer Cell Line Encyclopedia; citation_author=M. Ghandi; citation_author=F.W. Huang; citation_author=J. Jané-Valbuena; citation_author=G.V. Kryukov; citation_author=C.C. Lo; citation_author=E.R. McDonald; citation_author=J. Barretina; citation_author=E.T. Gelfand; citation_author=C.M. Bielski; citation_author=H. Li; citation_author=et al.; citation_journal_title=Nature; citation_volume=569; citation_firstpage=503; citation_lastpage=508; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Quantitative analysis of peptides and proteins in biomedicine by targeted mass spectrometry; citation_author=M.A. Gillette; citation_author=S.A. Carr; citation_journal_title=Nat. Methods; citation_volume=10; citation_firstpage=28; citation_lastpage=34; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Proteogenomic Characterization Reveals Therapeutic Vulnerabilities in Lung Adenocarcinoma; citation_author=M.A. Gillette; citation_author=S. Satpathy; citation_author=S. Cao; citation_author=S.M. Dhanasekaran; citation_author=S.V. Vasaikar; citation_author=K. Krug; citation_author=F. Petralia; citation_author=Y. Li; citation_author=W.-W. Liang; citation_author=B. Reva; citation_author=Clinical Proteomic Tumor Analysis Consortium; citation_author=et al.; citation_journal_title=Cell; citation_volume=182; citation_firstpage=200; citation_lastpage=225.e35; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=CDK4/6 Inhibition in Cancer: Beyond Cell Cycle Arrest; citation_author=S. Goel; citation_author=M.J. DeCristo; citation_author=S.S. McAllister; citation_author=J.J. Zhao; citation_journal_title=Trends Cell Biol.; citation_volume=28; citation_firstpage=911; citation_lastpage=925; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Molecular basis of low-penetrance retinoblastoma; citation_author=J.W. Harbour; citation_journal_title=Arch. Ophthalmol.; citation_volume=119; citation_firstpage=1699; citation_lastpage=1704; citation_publication_date=2001; "></meta><meta name="citation_reference" content="citation_title=Loss of MutL Disrupts CHK2-Dependent Cell-Cycle Control through CDK4/6 to Promote Intrinsic Endocrine Therapy Resistance in Primary Breast Cancer; citation_author=S. Haricharan; citation_author=N. Punturi; citation_author=P. Singh; citation_author=K.R. Holloway; citation_author=M. Anurag; citation_author=J. Schmelz; citation_author=C. Schmidt; citation_author=J.T. Lei; citation_author=V. Suman; citation_author=K. Hunt; citation_author=et al.; citation_journal_title=Cancer Discov.; citation_volume=7; citation_firstpage=1168; citation_lastpage=1183; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Topoisomerase IIalpha amplification does not predict benefit from dose-intense cyclophosphamide, doxorubicin, and fluorouracil therapy in HER2-amplified early breast cancer: results of CALGB 8541/150013; citation_author=L.N. Harris; citation_author=G. Broadwater; citation_author=M. Abu-Khalaf; citation_author=D. Cowan; citation_author=A.D. Thor; citation_author=D. Budman; citation_author=C.T. Cirrincione; citation_author=D.A. Berry; citation_author=E.P. Winer; citation_author=C.A. Hudis; citation_author=et al.; citation_journal_title=J. Clin. Oncol.; citation_volume=27; citation_firstpage=3430; citation_lastpage=3436; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=PhosphoSitePlus, 2014: mutations, PTMs and recalibrations; citation_author=P.V. Hornbeck; citation_author=B. Zhang; citation_author=B. Murray; citation_author=J.M. Kornhauser; citation_author=V. Latham; citation_author=E. Skrzypek; citation_journal_title=Nucleic Acids Res.; citation_volume=43; citation_firstpage=D512; citation_lastpage=D520; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Extensive Intratumor Proteogenomic Heterogeneity Revealed by Multiregion Sampling in a High-Grade Serous Ovarian Tumor Specimen; citation_author=A.L. Hunt; citation_author=N.W. Bateman; citation_author=B.L. Hood; citation_author=K.A. Conrads; citation_author=M. Zhou; citation_author=T.J. Litzi; citation_author=J. Oliver; citation_author=D. Mitchell; citation_author=G. Gist; citation_author=B. Blanton; citation_author=et al.; citation_journal_title=bioRxiv; citation_doi=10.1101/761155; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Implementing Genome-Driven Oncology; citation_author=D.M. Hyman; citation_author=B.S. Taylor; citation_author=J. Baselga; citation_journal_title=Cell; citation_volume=168; citation_firstpage=584; citation_lastpage=599; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=A Landscape of Pharmacogenomic Interactions in Cancer; citation_author=F. Iorio; citation_author=T.A. Knijnenburg; citation_author=D.J. Vis; citation_author=G.R. Bignell; citation_author=M.P. Menden; citation_author=M. Schubert; citation_author=N. Aben; citation_author=E. Gonçalves; citation_author=S. Barthorpe; citation_author=H. Lightfoot; citation_author=et al.; citation_journal_title=Cell; citation_volume=166; citation_firstpage=740; citation_lastpage=754; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=The single-cell pathology landscape of breast cancer; citation_author=H.W. Jackson; citation_author=J.R. Fischer; citation_author=V.R.T. Zanotelli; citation_author=H.R. Ali; citation_author=R. Mechera; citation_author=S.D. Soysal; citation_author=H. Moch; citation_author=S. Muenst; citation_author=Z. Varga; citation_author=W.P. Weber; citation_author=B. Bodenmiller; citation_journal_title=Nature; citation_volume=578; citation_firstpage=615; citation_lastpage=620; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=Breast cancer quantitative proteome and proteogenomic landscape; citation_author=H.J. Johansson; citation_author=F. Socciarelli; citation_author=N.M. Vacanti; citation_author=M.H. Haugen; citation_author=Y. Zhu; citation_author=I. Siavelis; citation_author=A. Fernandez-Woodbridge; citation_author=M.R. Aure; citation_author=B. Sennblad; citation_author=M. Vesterlund; citation_author=Consortia Oslo Breast Cancer Research Consortium (OSBREAC); citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=10; citation_firstpage=1600; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=KEGG: kyoto encyclopedia of genes and genomes; citation_author=M. Kanehisa; citation_author=S. Goto; citation_journal_title=Nucleic Acids Res.; citation_volume=28; citation_firstpage=27; citation_lastpage=30; citation_publication_date=2000; "></meta><meta name="citation_reference" content="citation_title=Whole-genome sequencing reveals activation-induced cytidine deaminase signatures during indolent chronic lymphocytic leukaemia evolution; citation_author=S. Kasar; citation_author=J. Kim; citation_author=R. Improgo; citation_author=G. Tiao; citation_author=P. Polak; citation_author=N. Haradhvala; citation_author=M.S. Lawrence; citation_author=A. Kiezun; citation_author=S.M. Fernandes; citation_author=S. Bahl; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=6; citation_firstpage=8866; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Enhancement of cytotoxic effects of gemcitabine by Dclk1 inhibition through suppression of Chk1 phosphorylation in human pancreatic cancer cells; citation_author=D. Kawamura; citation_author=Y. Takemoto; citation_author=A. Nishimoto; citation_author=K. Ueno; citation_author=T. Hosoyama; citation_author=B. Shirasawa; citation_author=T. Tanaka; citation_author=N. Kugimiya; citation_author=E. Harada; citation_author=K. Hamano; citation_journal_title=Oncol. Rep.; citation_volume=38; citation_firstpage=3238; citation_lastpage=3244; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Sparse non-negative matrix factorizations via alternating non-negativity-constrained least squares for microarray data analysis; citation_author=H. Kim; citation_author=H. Park; citation_journal_title=Bioinformatics; citation_volume=23; citation_firstpage=1495; citation_lastpage=1502; citation_publication_date=2007; "></meta><meta name="citation_reference" content="citation_title=Somatic ERCC2 mutations are associated with a distinct genomic signature in urothelial tumors; citation_author=J. Kim; citation_author=K.W. Mouw; citation_author=P. Polak; citation_author=L.Z. Braunstein; citation_author=A. Kamburov; citation_author=D.J. Kwiatkowski; citation_author=J.E. Rosenberg; citation_author=E.M. Van Allen; citation_author=A. D’Andrea; citation_author=G. Getz; citation_journal_title=Nat. Genet.; citation_volume=48; citation_firstpage=600; citation_lastpage=606; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Strelka2: fast and accurate calling of germline and somatic variants; citation_author=S. Kim; citation_author=K. Scheffler; citation_author=A.L. Halpern; citation_author=M.A. Bekritsky; citation_author=E. Noh; citation_author=M. Källberg; citation_author=X. Chen; citation_author=Y. Kim; citation_author=D. Beyter; citation_author=P. Krusche; citation_author=C.T. Saunders; citation_journal_title=Nat. Methods; citation_volume=15; citation_firstpage=591; citation_lastpage=594; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=A Curated Resource for Phosphosite-specific Signature Analysis; citation_author=K. Krug; citation_author=P. Mertins; citation_author=B. Zhang; citation_author=P. Hornbeck; citation_author=R. Raju; citation_author=R. Ahmad; citation_author=M. Szucs; citation_author=F. Mundt; citation_author=D. Forestier; citation_author=J. Jane-Valbuena; citation_author=et al.; citation_journal_title=Mol. Cell. Proteomics; citation_volume=18; citation_firstpage=576; citation_lastpage=593; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=The Connectivity Map: using gene-expression signatures to connect small molecules, genes, and disease; citation_author=J. Lamb; citation_author=E.D. Crawford; citation_author=D. Peck; citation_author=J.W. Modell; citation_author=I.C. Blat; citation_author=M.J. Wrobel; citation_author=J. Lerner; citation_author=J.-P. Brunet; citation_author=A. Subramanian; citation_author=K.N. Ross; citation_author=et al.; citation_journal_title=Science; citation_volume=313; citation_firstpage=1929; citation_lastpage=1935; citation_publication_date=2006; "></meta><meta name="citation_reference" content="citation_title=Discovery and saturation analysis of cancer genes across 21 tumour types; citation_author=M.S. Lawrence; citation_author=P. Stojanov; citation_author=C.H. Mermel; citation_author=J.T. Robinson; citation_author=L.A. Garraway; citation_author=T.R. Golub; citation_author=M. Meyerson; citation_author=S.B. Gabriel; citation_author=E.S. Lander; citation_author=G. Getz; citation_journal_title=Nature; citation_volume=505; citation_firstpage=495; citation_lastpage=501; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Learning the parts of objects by non-negative matrix factorization; citation_author=D.D. Lee; citation_author=H.S. Seung; citation_journal_title=Nature; citation_volume=401; citation_firstpage=788; citation_lastpage=791; citation_publication_date=1999; "></meta><meta name="citation_reference" content="citation_title=Algorithms for Non-negative Matrix Factorization; citation_inbook_title=Advances in Neural Information Processing Systems 13; citation_author=D.D. Lee; citation_author=H.S. Seung; citation_firstpage=556; citation_lastpage=562; citation_publication_date=2001; citation_publisher=MIT Press; "></meta><meta name="citation_reference" content="citation_title=Structure of the retinoblastoma tumour-suppressor pocket domain bound to a peptide from HPV E7; citation_author=J.O. Lee; citation_author=A.A. Russo; citation_author=N.P. Pavletich; citation_journal_title=Nature; citation_volume=391; citation_firstpage=859; citation_lastpage=865; citation_publication_date=1998; "></meta><meta name="citation_reference" content="citation_title=Identification of human triple-negative breast cancer subtypes and preclinical models for selection of targeted therapies; citation_author=B.D. Lehmann; citation_author=J.A. Bauer; citation_author=X. Chen; citation_author=M.E. Sanders; citation_author=A.B. Chakravarthy; citation_author=Y. Shyr; citation_author=J.A. Pietenpol; citation_journal_title=J. Clin. Invest.; citation_volume=121; citation_firstpage=2750; citation_lastpage=2767; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=PIK3CA mutations in androgen receptor-positive triple negative breast cancer confer sensitivity to the combination of PI3K and androgen receptor inhibitors; citation_author=B.D. Lehmann; citation_author=J.A. Bauer; citation_author=J.M. Schafer; citation_author=C.S. Pendleton; citation_author=L. Tang; citation_author=K.C. Johnson; citation_author=X. Chen; citation_author=J.M. Balko; citation_author=H. Gómez; citation_author=C.L. Arteaga; citation_author=et al.; citation_journal_title=Breast Cancer Res.; citation_volume=16; citation_firstpage=406; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Refinement of Triple-Negative Breast Cancer Molecular Subtypes: Implications for Neoadjuvant Chemotherapy Selection; citation_author=B.D. Lehmann; citation_author=B. Jovanović; citation_author=X. Chen; citation_author=M.V. Estrada; citation_author=K.N. Johnson; citation_author=Y. Shyr; citation_author=H.L. Moses; citation_author=M.E. Sanders; citation_author=J.A. Pietenpol; citation_journal_title=PLoS ONE; citation_volume=11; citation_firstpage=e0157368; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=WebGestalt 2019: gene set analysis toolkit with revamped UIs and APIs; citation_author=Y. Liao; citation_author=J. Wang; citation_author=E.J. Jaehnig; citation_author=Z. Shi; citation_journal_title=Nucleic Acids Res.; citation_volume=47; citation_firstpage=W199; citation_lastpage=W205; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=The Molecular Signatures Database (MSigDB) hallmark gene set collection; citation_author=A. Liberzon; citation_author=C. Birger; citation_author=H. Thorvaldsdóttir; citation_author=M. Ghandi; citation_author=J.P. Mesirov; citation_author=P. Tamayo; citation_journal_title=Cell Syst.; citation_volume=1; citation_firstpage=417; citation_lastpage=425; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=RIPK3 upregulation confers robust proliferation and collateral cystine-dependence on breast cancer recurrence; citation_author=C.-C. Lin; citation_author=N.W. Mabe; citation_author=Y.-T. Lin; citation_author=W.-H. Yang; citation_author=X. Tang; citation_author=L. Hong; citation_author=T. Sun; citation_author=J. Force; citation_author=J.R. Marks; citation_author=T.-P. Yao; citation_author=et al.; citation_journal_title=Cell Death Differ.; citation_volume=27; citation_firstpage=2234; citation_lastpage=2247; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=Doublecortin-like kinase 1 expression associates with breast cancer with neuroendocrine differentiation; citation_author=Y.-H. Liu; citation_author=J.Y.S. Tsang; citation_author=Y.-B. Ni; citation_author=T. Hlaing; citation_author=S.-K. Chan; citation_author=K.-F. Chan; citation_author=C.-W. Ko; citation_author=S.S. Mujtaba; citation_author=G.M. Tse; citation_journal_title=Oncotarget; citation_volume=7; citation_firstpage=1464; citation_lastpage=1476; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Combination of palbociclib with enzalutamide shows in vitro activity in RB proficient and androgen receptor positive triple negative breast cancer cells; citation_author=C.-Y. Liu; citation_author=K.-Y. Lau; citation_author=C.-C. Hsu; citation_author=J.-L. Chen; citation_author=C.-H. Lee; citation_author=T.-T. Huang; citation_author=Y.-T. Chen; citation_author=C.-T. Huang; citation_author=P.-H. Lin; citation_author=L.-M. Tseng; citation_journal_title=PLoS ONE; citation_volume=12; citation_firstpage=e0189007; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Targeting Metabolism for Cancer Therapy; citation_author=A. Luengo; citation_author=D.Y. Gui; citation_author=M.G. Vander Heiden; citation_journal_title=Cell Chem. Biol.; citation_volume=24; citation_firstpage=1161; citation_lastpage=1180; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Machine Learning Identifies Stemness Features Associated with Oncogenic Dedifferentiation; citation_author=T.M. Malta; citation_author=A. Sokolov; citation_author=A.J. Gentles; citation_author=T. Burzykowski; citation_author=L. Poisson; citation_author=J.N. Weinstein; citation_author=B. Kamińska; citation_author=J. Huelsken; citation_author=L. Omberg; citation_author=O. Gevaert; citation_author=Cancer Genome Atlas Research Network; citation_author=et al.; citation_journal_title=Cell; citation_volume=173; citation_firstpage=338; citation_lastpage=354.e15; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=The emergence of TNIK as a therapeutic target for colorectal cancer; citation_author=M. Masuda; citation_author=T. Yamada; citation_journal_title=Expert Opin. Ther. Targets; citation_volume=21; citation_firstpage=353; citation_lastpage=355; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=ATM and ATR substrate analysis reveals extensive protein networks responsive to DNA damage; citation_author=S. Matsuoka; citation_author=B.A. Ballif; citation_author=A. Smogorzewska; citation_author=E.R. McDonald; citation_author=K.E. Hurov; citation_author=J. Luo; citation_author=C.E. Bakalarski; citation_author=Z. Zhao; citation_author=N. Solimini; citation_author=Y. Lerenthal; citation_author=et al.; citation_journal_title=Science; citation_volume=316; citation_firstpage=1160; citation_lastpage=1166; citation_publication_date=2007; "></meta><meta name="citation_reference" content="citation_title=Maftools: efficient and comprehensive analysis of somatic variants in cancer; citation_author=A. Mayakonda; citation_author=D.-C. Lin; citation_author=Y. Assenov; citation_author=C. Plass; citation_author=H.P. Koeffler; citation_journal_title=Genome Res.; citation_volume=28; citation_firstpage=1747; citation_lastpage=1756; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data; citation_author=A. McKenna; citation_author=M. Hanna; citation_author=E. Banks; citation_author=A. Sivachenko; citation_author=K. Cibulskis; citation_author=A. Kernytsky; citation_author=K. Garimella; citation_author=D. Altshuler; citation_author=S. Gabriel; citation_author=M. Daly; citation_author=M.A. DePristo; citation_journal_title=Genome Res.; citation_volume=20; citation_firstpage=1297; citation_lastpage=1303; citation_publication_date=2010; "></meta><meta name="citation_reference" content="citation_title=The Ensembl Variant Effect Predictor; citation_author=W. McLaren; citation_author=L. Gil; citation_author=S.E. Hunt; citation_author=H.S. Riat; citation_author=G.R.S. Ritchie; citation_author=A. Thormann; citation_author=P. Flicek; citation_author=F. Cunningham; citation_journal_title=Genome Biol.; citation_volume=17; citation_firstpage=122; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers; citation_author=C.H. Mermel; citation_author=S.E. Schumacher; citation_author=B. Hill; citation_author=M.L. Meyerson; citation_author=R. Beroukhim; citation_author=G. Getz; citation_journal_title=Genome Biol.; citation_volume=12; citation_firstpage=R41; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=Ischemia in tumors induces early and sustained phosphorylation changes in stress kinase pathways but does not affect global protein levels; citation_author=P. Mertins; citation_author=F. Yang; citation_author=T. Liu; citation_author=D.R. Mani; citation_author=V.A. Petyuk; citation_author=M.A. Gillette; citation_author=K.R. Clauser; citation_author=J.W. Qiao; citation_author=M.A. Gritsenko; citation_author=R.J. Moore; citation_author=et al.; citation_journal_title=Mol. Cell. Proteomics; citation_volume=13; citation_firstpage=1690; citation_lastpage=1704; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Proteogenomics connects somatic mutations to signalling in breast cancer; citation_author=P. Mertins; citation_author=D.R. Mani; citation_author=K.V. Ruggles; citation_author=M.A. Gillette; citation_author=K.R. Clauser; citation_author=P. Wang; citation_author=X. Wang; citation_author=J.W. Qiao; citation_author=S. Cao; citation_author=F. Petralia; citation_author=NCI CPTAC; citation_author=et al.; citation_journal_title=Nature; citation_volume=534; citation_firstpage=55; citation_lastpage=62; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Reproducible workflow for multiplexed deep-scale proteome and phosphoproteome analysis of tumor tissues by liquid chromatography-mass spectrometry; citation_author=P. Mertins; citation_author=L.C. Tang; citation_author=K. Krug; citation_author=D.J. Clark; citation_author=M.A. Gritsenko; citation_author=L. Chen; citation_author=K.R. Clauser; citation_author=T.R. Clauss; citation_author=P. Shah; citation_author=M.A. Gillette; citation_author=et al.; citation_journal_title=Nat. Protoc.; citation_volume=13; citation_firstpage=1632; citation_lastpage=1661; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=HBO1 histone acetylase activity is essential for DNA replication licensing and inhibited by Geminin; citation_author=B. Miotto; citation_author=K. Struhl; citation_journal_title=Mol. Cell; citation_volume=37; citation_firstpage=57; citation_lastpage=66; citation_publication_date=2010; "></meta><meta name="citation_reference" content="citation_title=The Hbo1-Brd1/Brpf2 complex is responsible for global acetylation of H3K14 and required for fetal liver erythropoiesis; citation_author=Y. Mishima; citation_author=S. Miyagi; citation_author=A. Saraya; citation_author=M. Negishi; citation_author=M. Endoh; citation_author=T.A. Endo; citation_author=T. Toyoda; citation_author=J. Shinga; citation_author=T. Katsumoto; citation_author=T. Chiba; citation_author=et al.; citation_journal_title=Blood; citation_volume=118; citation_firstpage=2443; citation_lastpage=2453; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=New functionalities in the TCGAbiolinks package for the study and integration of cancer data from GDC and GTEx; citation_author=M. Mounir; citation_author=M. Lucchetta; citation_author=T.C. Silva; citation_author=C. Olsen; citation_author=G. Bontempi; citation_author=X. Chen; citation_author=H. Noushmehr; citation_author=A. Colaprico; citation_author=E. Papaleo; citation_journal_title=PLoS Comput. Biol.; citation_volume=15; citation_firstpage=e1006701; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Inhibition of 3-phosphoglycerate dehydrogenase (PHGDH) by indole amides abrogates de novo serine synthesis in cancer cells; citation_author=E. Mullarky; citation_author=J. Xu; citation_author=A.D. Robin; citation_author=D.J. Huggins; citation_author=A. Jennings; citation_author=N. Noguchi; citation_author=A. Olland; citation_author=D. Lakshminarasimhan; citation_author=M. Miller; citation_author=D. Tomita; citation_author=et al.; citation_journal_title=Bioorg. Med. Chem. Lett.; citation_volume=29; citation_firstpage=2503; citation_lastpage=2510; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=The NAD+ Salvage Pathway Supports PHGDH-Driven Serine Biosynthesis; citation_author=J.P. Murphy; citation_author=M.A. Giacomantonio; citation_author=J.A. Paulo; citation_author=R.A. Everley; citation_author=B.E. Kennedy; citation_author=G.P. Pathak; citation_author=D.R. Clements; citation_author=Y. Kim; citation_author=C. Dai; citation_author=T. Sharif; citation_author=et al.; citation_journal_title=Cell Rep.; citation_volume=24; citation_firstpage=2381; citation_lastpage=2391.e5; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Robust enumeration of cell subsets from tissue expression profiles; citation_author=A.M. Newman; citation_author=C.L. Liu; citation_author=M.R. Green; citation_author=A.J. Gentles; citation_author=W. Feng; citation_author=Y. Xu; citation_author=C.D. Hoang; citation_author=M. Diehn; citation_author=A.A. Alizadeh; citation_journal_title=Nat. Methods; citation_volume=12; citation_firstpage=453; citation_lastpage=457; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Landscape of somatic mutations in 560 breast cancer whole-genome sequences; citation_author=S. Nik-Zainal; citation_author=H. Davies; citation_author=J. Staaf; citation_author=M. Ramakrishna; citation_author=D. Glodzik; citation_author=X. Zou; citation_author=I. Martincorena; citation_author=L.B. Alexandrov; citation_author=S. Martin; citation_author=D.C. Wedge; citation_author=et al.; citation_journal_title=Nature; citation_volume=534; citation_firstpage=47; citation_lastpage=54; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Quantitative Proteomics of the Cancer Cell Line Encyclopedia; citation_author=D.P. Nusinow; citation_author=J. Szpyt; citation_author=M. Ghandi; citation_author=C.M. Rose; citation_author=E.R. McDonald; citation_author=M. Kalocsay; citation_author=J. Jané-Valbuena; citation_author=E. Gelfand; citation_author=D.K. Schweppe; citation_author=M. Jedrychowski; citation_author=et al.; citation_journal_title=Cell; citation_volume=180; citation_firstpage=387; citation_lastpage=402.e16; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=Treating cancer with selective CDK4/6 inhibitors; citation_author=B. O’Leary; citation_author=R.S. Finn; citation_author=N.C. Turner; citation_journal_title=Nat. Rev. Clin. Oncol.; citation_volume=13; citation_firstpage=417; citation_lastpage=430; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Supervised risk predictor of breast cancer based on intrinsic subtypes; citation_author=J.S. Parker; citation_author=M. Mullins; citation_author=M.C.U. Cheang; citation_author=S. Leung; citation_author=D. Voduc; citation_author=T. Vickery; citation_author=S. Davies; citation_author=C. Fauron; citation_author=X. He; citation_author=Z. Hu; citation_author=et al.; citation_journal_title=J. Clin. Oncol.; citation_volume=27; citation_firstpage=1160; citation_lastpage=1167; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=Activation of STING-Dependent Innate Immune Signaling By S-Phase-Specific DNA Damage in Breast Cancer; citation_author=E.E. Parkes; citation_author=S.M. Walker; citation_author=L.E. Taggart; citation_author=N. McCabe; citation_author=L.A. Knight; citation_author=R. Wilkinson; citation_author=K.D. McCloskey; citation_author=N.E. Buckley; citation_author=K.I. Savage; citation_author=M. Salto-Tellez; citation_author=et al.; citation_journal_title=J. Natl. Cancer Inst.; citation_volume=109; citation_firstpage=109; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Retinoblastoma protein expression and its predictors in triple-negative breast cancer; citation_author=J.M. Patel; citation_author=A. Goss; citation_author=J.E. Garber; citation_author=V. Torous; citation_author=E.T. Richardson; citation_author=M.J. Haviland; citation_author=M.R. Hacker; citation_author=G.J. Freeman; citation_author=T. Nalven; citation_author=B. Alexander; citation_author=et al.; citation_journal_title=NPJ Breast Cancer; citation_volume=6; citation_firstpage=19; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=The Emerging Hallmarks of Cancer Metabolism; citation_author=N.N. Pavlova; citation_author=C.B. Thompson; citation_journal_title=Cell Metab.; citation_volume=23; citation_firstpage=27; citation_lastpage=47; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=CDK4/6 inhibition in breast cancer: current practice and future directions; citation_author=S. Pernas; citation_author=S.M. Tolaney; citation_author=E.P. Winer; citation_author=S. Goel; citation_journal_title=Ther. Adv. Med. Oncol.; citation_volume=10; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Cancer metabolic reprogramming: importance, main features, and potentials for precise targeted anti-cancer therapies; citation_author=L.M. Phan; citation_author=S.-C.J. Yeung; citation_author=M.-H. Lee; citation_journal_title=Cancer Biol. Med.; citation_volume=11; citation_firstpage=1; citation_lastpage=19; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Scaling accurate genetic variant discovery to tens of thousands of samples; citation_author=R. Poplin; citation_author=V. Ruano-Rubio; citation_author=M.A. DePristo; citation_author=T.J. Fennell; citation_author=M.O. Carneiro; citation_author=G.A. Van der Auwera; citation_author=D.E. Kling; citation_author=L.D. Gauthier; citation_author=A. Levy-Moonshine; citation_author=D. Roazen; citation_author=et al.; citation_journal_title=bioRxiv; citation_doi=10.1101/201178; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Functional genomics reveal that the serine synthesis pathway is essential in breast cancer; citation_author=R. Possemato; citation_author=K.M. Marks; citation_author=Y.D. Shaul; citation_author=M.E. Pacold; citation_author=D. Kim; citation_author=K. Birsoy; citation_author=S. Sethumadhavan; citation_author=H.-K. Woo; citation_author=H.G. Jang; citation_author=A.K. Jha; citation_author=et al.; citation_journal_title=Nature; citation_volume=476; citation_firstpage=346; citation_lastpage=350; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=Oncotator: cancer variant annotation tool; citation_author=A.H. Ramos; citation_author=L. Lichtenstein; citation_author=M. Gupta; citation_author=M.S. Lawrence; citation_author=T.J. Pugh; citation_author=G. Saksena; citation_author=M. Meyerson; citation_author=G. Getz; citation_journal_title=Hum. Mutat.; citation_volume=36; citation_firstpage=E2423; citation_lastpage=E2429; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=g:Profiler-a web server for functional interpretation of gene lists (2016 update); citation_author=J. Reimand; citation_author=T. Arak; citation_author=P. Adler; citation_author=L. Kolberg; citation_author=S. Reisberg; citation_author=H. Peterson; citation_author=J. Vilo; citation_journal_title=Nucleic. Acids Res.; citation_volume=44; citation_firstpage=W83; citation_lastpage=W89; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=limma powers differential expression analyses for RNA-sequencing and microarray studies; citation_author=M.E. Ritchie; citation_author=B. Phipson; citation_author=D. Wu; citation_author=Y. Hu; citation_author=C.W. Law; citation_author=W. Shi; citation_author=G.K. Smyth; citation_journal_title=Nucleic Acids Res.; citation_volume=43; citation_firstpage=e47; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Hypermutation in human cancer genomes: footprints and mechanisms; citation_author=S.A. Roberts; citation_author=D.A. Gordenin; citation_journal_title=Nat. Rev. Cancer; citation_volume=14; citation_firstpage=786; citation_lastpage=800; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=DeconstructSigs: delineating mutational processes in single tumors distinguishes DNA repair deficiencies and patterns of carcinoma evolution; citation_author=R. Rosenthal; citation_author=N. McGranahan; citation_author=J. Herrero; citation_author=B.S. Taylor; citation_author=C. Swanton; citation_journal_title=Genome Biol.; citation_volume=17; citation_firstpage=31; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Standardizing slide-based assays in breast cancer: hormone receptors, HER2, and sentinel lymph nodes; citation_author=J.S. Ross; citation_author=W.F. Symmans; citation_author=L. Pusztai; citation_author=G.N. Hortobagyi; citation_journal_title=Clin. Cancer Res.; citation_volume=13; citation_firstpage=2831; citation_lastpage=2835; citation_publication_date=2007; "></meta><meta name="citation_reference" content="citation_title=An analysis of the sensitivity of proteogenomic mapping of somatic mutations and novel splicing events in cancer; citation_author=K.V. Ruggles; citation_author=Z. Tang; citation_author=X. Wang; citation_author=H. Grover; citation_author=M. Askenazi; citation_author=J. Teubl; citation_author=S. Cao; citation_author=M.D. McLellan; citation_author=K.R. Clauser; citation_author=D.L. Tabb; citation_author=et al.; citation_journal_title=Mol. Cell. Proteomics; citation_volume=15; citation_firstpage=1060; citation_lastpage=1071; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Methods, tools and current perspectives in proteogenomics; citation_author=K.V. Ruggles; citation_author=K. Krug; citation_author=X. Wang; citation_author=K.R. Clauser; citation_author=J. Wang; citation_author=S.H. Payne; citation_author=D. Fenyö; citation_author=B. Zhang; citation_author=D.R. Mani; citation_journal_title=Mol. Cell. Proteomics; citation_volume=16; citation_firstpage=959; citation_lastpage=981; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Integrated Genomic Analysis of Diverse Induced Pluripotent Stem Cells from the Progenitor Cell Biology Consortium; citation_author=N. Salomonis; citation_author=P.J. Dexheimer; citation_author=L. Omberg; citation_author=R. Schroll; citation_author=S. Bush; citation_author=J. Huo; citation_author=L. Schriml; citation_author=S. Ho Sui; citation_author=M. Keddache; citation_author=C. Mayhew; citation_author=et al.; citation_journal_title=Stem Cell Reports; citation_volume=7; citation_firstpage=110; citation_lastpage=125; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Microscaled proteogenomic methods for precision oncology; citation_author=S. Satpathy; citation_author=E.J. Jaehnig; citation_author=K. Krug; citation_author=B.-J. Kim; citation_author=A.B. Saltzman; citation_author=D.W. Chan; citation_author=K.R. Holloway; citation_author=M. Anurag; citation_author=C. Huang; citation_author=P. Singh; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=11; citation_firstpage=532; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=High-Throughput Assessment of Kinome-wide Activation States; citation_author=T. Schmidlin; citation_author=D.O. Debets; citation_author=C.A.G.H. van Gelder; citation_author=K.E. Stecker; citation_author=S. Rontogianni; citation_author=B.L. van den Eshof; citation_author=K. Kemper; citation_author=E.H. Lips; citation_author=M. van den Biggelaar; citation_author=D.S. Peeper; citation_author=et al.; citation_journal_title=Cell Syst.; citation_volume=9; citation_firstpage=366; citation_lastpage=374.e5; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=i-Tracker: for quantitative proteomics using iTRAQ; citation_author=I.P. Shadforth; citation_author=T.P.J. Dunkley; citation_author=K.S. Lilley; citation_author=C. Bessant; citation_journal_title=BMC Genomics; citation_volume=6; citation_firstpage=145; citation_publication_date=2005; "></meta><meta name="citation_reference" content="citation_title=Unveiling Hidden Dynamics of Hippo Signalling: A Systems Analysis; citation_author=S.-Y. Shin; citation_author=L.K. Nguyen; citation_journal_title=Genes (Basel); citation_volume=7; citation_firstpage=44; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Novel Biomarkers for Personalized Cancer Immunotherapy; citation_author=Y. Shindo; citation_author=S. Hazama; citation_author=R. Tsunedomi; citation_author=N. Suzuki; citation_author=H. Nagano; citation_journal_title=Cancers (Basel); citation_volume=11; citation_firstpage=1223; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Breast cancer genome and transcriptome integration implicates specific mutational signatures with immune cell infiltration; citation_author=M. Smid; citation_author=F.G. Rodríguez-González; citation_author=A.M. Sieuwerts; citation_author=R. Salgado; citation_author=W.J.C. Prager-Van der Smissen; citation_author=M.V. Vlugt-Daane; citation_author=A. van Galen; citation_author=S. Nik-Zainal; citation_author=J. Staaf; citation_author=A.B. Brinkman; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=7; citation_firstpage=12910; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Autophosphorylation: a salient feature of protein kinases; citation_author=J.A. Smith; citation_author=S.H. Francis; citation_author=J.D. Corbin; citation_journal_title=Mol. Cell. Biochem.; citation_volume=127-128; citation_firstpage=51; citation_lastpage=70; citation_publication_date=1993; "></meta><meta name="citation_reference" content="citation_title=ONE-CLASS DETECTION OF CELL STATES IN TUMOR SUBTYPES; citation_author=A. Sokolov; citation_author=E.O. Paull; citation_author=J.M. Stuart; citation_journal_title=Pac. Symp. Biocomput.; citation_volume=21; citation_firstpage=405; citation_lastpage=416; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles; citation_author=A. Subramanian; citation_author=P. Tamayo; citation_author=V.K. Mootha; citation_author=S. Mukherjee; citation_author=B.L. Ebert; citation_author=M.A. Gillette; citation_author=A. Paulovich; citation_author=S.L. Pomeroy; citation_author=T.R. Golub; citation_author=E.S. Lander; citation_author=J.P. Mesirov; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=102; citation_firstpage=15545; citation_lastpage=15550; citation_publication_date=2005; "></meta><meta name="citation_reference" content="citation_title=A Next Generation Connectivity Map: L1000 Platform and the First 1,000,000 Profiles; citation_author=A. Subramanian; citation_author=R. Narayan; citation_author=S.M. Corsello; citation_author=D.D. Peck; citation_author=T.E. Natoli; citation_author=X. Lu; citation_author=J. Gould; citation_author=J.F. Davis; citation_author=A.A. Tubelli; citation_author=J.K. Asiedu; citation_author=et al.; citation_journal_title=Cell; citation_volume=171; citation_firstpage=1437; citation_lastpage=1452.e17; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Dclk1 Inhibition Cancels 5-FU-induced Cell-cycle Arrest and Decreases Cell Survival in Colorectal Cancer; citation_author=Y. Suehiro; citation_author=Y. Takemoto; citation_author=A. Nishimoto; citation_author=K. Ueno; citation_author=B. Shirasawa; citation_author=T. Tanaka; citation_author=N. Kugimiya; citation_author=A. Suga; citation_author=E. Harada; citation_author=K. Hamano; citation_journal_title=Anticancer Res.; citation_volume=38; citation_firstpage=6225; citation_lastpage=6230; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Automatic relevance determination in nonnegative matrix factorization with the β-divergence; citation_author=V.Y.F. Tan; citation_author=C. Févotte; citation_journal_title=IEEE Trans. Pattern Anal. Mach. Intell.; citation_volume=35; citation_firstpage=1592; citation_lastpage=1605; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=COSMIC: the Catalogue Of Somatic Mutations In Cancer; citation_author=J.G. Tate; citation_author=S. Bamford; citation_author=H.C. Jubb; citation_author=Z. Sondka; citation_author=D.M. Beare; citation_author=N. Bindal; citation_author=H. Boutselakis; citation_author=C.G. Cole; citation_author=C. Creatore; citation_author=E. Dawson; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=47; citation_number=D1; citation_firstpage=D941; citation_lastpage=D947; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=DeTiN: overcoming tumor-in-normal contamination; citation_author=A. Taylor-Weiner; citation_author=C. Stewart; citation_author=T. Giordano; citation_author=M. Miller; citation_author=M. Rosenberg; citation_author=A. Macbeth; citation_author=N. Lennon; citation_author=E. Rheinbay; citation_author=D.-A. Landau; citation_author=C.J. Wu; citation_author=G. Getz; citation_journal_title=Nat. Methods; citation_volume=15; citation_firstpage=531; citation_lastpage=534; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=NCCN Guidelines Updates: Breast Cancer; citation_author=M.L. Telli; citation_author=W.J. Gradishar; citation_author=J.H. Ward; citation_journal_title=J. Natl. Compr. Canc. Netw.; citation_volume=17; citation_firstpage=552; citation_lastpage=555; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=The Immune Landscape of Cancer; citation_author=V. Thorsson; citation_author=D.L. Gibbs; citation_author=S.D. Brown; citation_author=D. Wolf; citation_author=D.S. Bortone; citation_author=T.-H. Ou Yang; citation_author=E. Porta-Pardo; citation_author=G.F. Gao; citation_author=C.L. Plaisier; citation_author=J.A. Eddy; citation_author=Cancer Genome Atlas Research Network; citation_author=et al.; citation_journal_title=Immunity; citation_volume=51; citation_firstpage=411; citation_lastpage=412; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation; citation_author=C. Trapnell; citation_author=B.A. Williams; citation_author=G. Pertea; citation_author=A. Mortazavi; citation_author=G. Kwan; citation_author=M.J. van Baren; citation_author=S.L. Salzberg; citation_author=B.J. Wold; citation_author=L. Pachter; citation_journal_title=Nat. Biotechnol.; citation_volume=28; citation_firstpage=511; citation_lastpage=515; citation_publication_date=2010; "></meta><meta name="citation_reference" content="citation_title=Proteomic maps of breast cancer subtypes; citation_author=S. Tyanova; citation_author=R. Albrechtsen; citation_author=P. Kronqvist; citation_author=J. Cox; citation_author=M. Mann; citation_author=T. Geiger; citation_journal_title=Nat. Commun.; citation_volume=7; citation_firstpage=10259; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Rapid and deep-scale ubiquitylation profiling for biology and translational research; citation_author=N.D. Udeshi; citation_author=D.C. Mani; citation_author=S. Satpathy; citation_author=S. Fereshetian; citation_author=J.A. Gasser; citation_author=T. Svinkina; citation_author=M.E. Olive; citation_author=B.L. Ebert; citation_author=P. Mertins; citation_author=S.A. Carr; citation_journal_title=Nat. Commun.; citation_volume=11; citation_firstpage=359; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=Understanding the Intersections between Metabolism and Cancer Biology; citation_author=M.G. Vander Heiden; citation_author=R.J. DeBerardinis; citation_journal_title=Cell; citation_volume=168; citation_firstpage=657; citation_lastpage=669; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Integrative analysis of breast cancer reveals prognostic haematopoietic activity and patient-specific immune response profiles; citation_author=F.S. Varn; citation_author=E.H. Andrews; citation_author=D.W. Mullins; citation_author=C. Cheng; citation_journal_title=Nat. Commun.; citation_volume=7; citation_firstpage=10248; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=LinkedOmics: analyzing multi-omics data within and across 32 cancer types; citation_author=S.V. Vasaikar; citation_author=P. Straub; citation_author=J. Wang; citation_author=B. Zhang; citation_journal_title=Nucleic Acids Res.; citation_volume=46; citation_number=D1; citation_firstpage=D956; citation_lastpage=D963; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities; citation_author=S. Vasaikar; citation_author=C. Huang; citation_author=X. Wang; citation_author=V.A. Petyuk; citation_author=S.R. Savage; citation_author=B. Wen; citation_author=Y. Dou; citation_author=Y. Zhang; citation_author=Z. Shi; citation_author=O.A. Arshad; citation_author=Clinical Proteomic Tumor Analysis Consortium; citation_author=et al.; citation_journal_title=Cell; citation_volume=177; citation_firstpage=1035; citation_lastpage=1049.e19; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=50 years of protein acetylation: from gene regulation to epigenetics, metabolism and beyond; citation_author=E. Verdin; citation_author=M. Ott; citation_journal_title=Nat. Rev. Mol. Cell Biol.; citation_volume=16; citation_firstpage=258; citation_lastpage=264; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Autophosphorylation kinetics of protein kinases; citation_author=Z.-X. Wang; citation_author=J.-W. Wu; citation_journal_title=Biochem. J.; citation_volume=368; citation_firstpage=947; citation_lastpage=952; citation_publication_date=2002; "></meta><meta name="citation_reference" content="citation_title=Analysis of acetylation stoichiometry suggests that SIRT3 repairs nonenzymatic acetylation lesions; citation_author=B.T. Weinert; citation_author=T. Moustafa; citation_author=V. Iesmantavicius; citation_author=R. Zechner; citation_author=C. Choudhary; citation_journal_title=EMBO J.; citation_volume=34; citation_firstpage=2620; citation_lastpage=2632; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Time-Resolved Analysis Reveals Rapid Dynamics and Broad Scope of the CBP/p300 Acetylome; citation_author=B.T. Weinert; citation_author=T. Narita; citation_author=S. Satpathy; citation_author=B. Srinivasan; citation_author=B.K. Hansen; citation_author=C. Schölz; citation_author=W.B. Hamilton; citation_author=B.E. Zucconi; citation_author=W.W. Wang; citation_author=W.R. Liu; citation_author=et al.; citation_journal_title=Cell; citation_volume=174; citation_firstpage=231; citation_lastpage=244.e12; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Intracellular Trapping of the Selective Phosphoglycerate Dehydrogenase (PHGDH) Inhibitor BI-4924 Disrupts Serine Biosynthesis; citation_author=H. Weinstabl; citation_author=M. Treu; citation_author=J. Rinnenthal; citation_author=S.K. Zahn; citation_author=P. Ettmayer; citation_author=G. Bader; citation_author=G. Dahmann; citation_author=D. Kessler; citation_author=K. Rumpel; citation_author=N. Mischerikow; citation_author=et al.; citation_journal_title=J. Med. Chem.; citation_volume=62; citation_firstpage=7976; citation_lastpage=7997; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Small molecule kinase inhibitor LRRK2-IN-1 demonstrates potent activity against colorectal and pancreatic cancer through inhibition of doublecortin-like kinase 1; citation_author=N. Weygant; citation_author=D. Qu; citation_author=W.L. Berry; citation_author=R. May; citation_author=P. Chandrakesan; citation_author=D.B. Owen; citation_author=S.M. Sureban; citation_author=N. Ali; citation_author=R. Janknecht; citation_author=C.W. Houchen; citation_journal_title=Mol. Cancer; citation_volume=13; citation_firstpage=103; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Identification of genes periodically expressed in the human cell cycle and their expression in tumors; citation_author=M.L. Whitfield; citation_author=G. Sherlock; citation_author=A.J. Saldanha; citation_author=J.I. Murray; citation_author=C.A. Ball; citation_author=K.E. Alexander; citation_author=J.C. Matese; citation_author=C.M. Perou; citation_author=M.M. Hurt; citation_author=P.O. Brown; citation_author=D. Botstein; citation_journal_title=Mol. Biol. Cell; citation_volume=13; citation_firstpage=1977; citation_lastpage=2000; citation_publication_date=2002; "></meta><meta name="citation_reference" content="citation_title=Sirtuin 3: A Janus face in cancer (Review); citation_author=Y. Xiong; citation_author=M. Wang; citation_author=J. Zhao; citation_author=Y. Han; citation_author=L. Jia; citation_journal_title=Int. J. Oncol.; citation_volume=49; citation_firstpage=2227; citation_lastpage=2235; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Synergistic anti-cancer activity of CDK4/6 inhibitor palbociclib and dual mTOR kinase inhibitor MLN0128 in pRb-expressing ER-negative breast cancer; citation_author=T. Yamamoto; citation_author=N. Kanaya; citation_author=G. Somlo; citation_author=S. Chen; citation_journal_title=Breast Cancer Res. Treat.; citation_volume=174; citation_firstpage=615; citation_lastpage=625; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Genomics of Drug Sensitivity in Cancer (GDSC): a resource for therapeutic biomarker discovery in cancer cells; citation_author=W. Yang; citation_author=J. Soares; citation_author=P. Greninger; citation_author=E.J. Edelman; citation_author=H. Lightfoot; citation_author=S. Forbes; citation_author=N. Bindal; citation_author=D. Beare; citation_author=J.A. Smith; citation_author=I.R. Thompson; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=41; citation_firstpage=D955; citation_lastpage=D961; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Inferring tumour purity and stromal and immune cell admixture from expression data; citation_author=K. Yoshihara; citation_author=M. Shahmoradgoli; citation_author=E. Martínez; citation_author=R. Vegesna; citation_author=H. Kim; citation_author=W. Torres-Garcia; citation_author=V. Treviño; citation_author=H. Shen; citation_author=P.W. Laird; citation_author=D.A. Levine; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=4; citation_firstpage=2612; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Proteogenomic characterization of human colon and rectal cancer; citation_author=B. Zhang; citation_author=J. Wang; citation_author=X. Wang; citation_author=J. Zhu; citation_author=Q. Liu; citation_author=Z. Shi; citation_author=M.C. Chambers; citation_author=L.J. Zimmerman; citation_author=K.F. Shaddox; citation_author=S. Kim; citation_author=NCI CPTAC; citation_author=et al.; citation_journal_title=Nature; citation_volume=513; citation_firstpage=382; citation_lastpage=387; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Clinical potential of mass spectrometry-based proteogenomics; citation_author=B. Zhang; citation_author=J.R. Whiteaker; citation_author=A.N. Hoofnagle; citation_author=G.S. Baird; citation_author=K.D. Rodland; citation_author=A.G. Paulovich; citation_journal_title=Nat. Rev. Clin. Oncol.; citation_volume=16; citation_firstpage=256; citation_lastpage=268; citation_publication_date=2019; "></meta><meta name="citation_abstract" content="&lt;h2&gt;Summary&lt;/h2&gt;&lt;p&gt;The integration of mass spectrometry-based proteomics with next-generation DNA and RNA sequencing profiles tumors more comprehensively. Here this &quot;proteogenomics&quot; approach was applied to 122 treatment-naive primary breast cancers accrued to preserve post-translational modifications, including protein phosphorylation and acetylation. Proteogenomics challenged standard breast cancer diagnoses, provided detailed analysis of the &lt;i&gt;ERBB2&lt;/i&gt; amplicon, defined tumor subsets that could benefit from immune checkpoint therapy, and allowed more accurate assessment of Rb status for prediction of CDK4/6 inhibitor responsiveness. Phosphoproteomics profiles uncovered novel associations between tumor suppressor loss and targetable kinases. Acetylproteome analysis highlighted acetylation on key nuclear proteins involved in the DNA damage response and revealed cross-talk between cytoplasmic and mitochondrial acetylation and metabolism. Our results underscore the potential of proteogenomics for clinical investigation of breast cancer through more accurate annotation of targetable pathways and biological features of this remarkably heterogeneous malignancy.&lt;/p&gt;"></meta><meta charset="UTF-8" /><!--

    Pagebuild admin UI
-->



<script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8"
            data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "true";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/nb6b~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/nb6b~selected-legacy.css"/>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans"/>
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-cba8bc975dae30d19357.css"/><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-cba8bc975dae30d19357.css" media="print"/><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css"/><script type="text/javascript">
    document.addEventListener('DOMContentLoaded', function(event) {
        var SETTINGS = {
            ENDPOINT: 'https://crossmark.crossref.org/dialog',
            SCRIPT_VERSION: 'v2.0',
            // LOGO_URL: 'http://crossmark-cdn.crossref.org/widget/v2.0/logos/crossmark-logo-square.svg',
            STYLESHEET_URL: 'https://crossmark-cdn.crossref.org/widget/v2.0/style.css',
        };

        var touchStarted = false;
        var touchArea;
        var tapEvent = function(element, callback) {
            element.addEventListener('click', function(event) {
                if (event.ctrlKey || event.shiftKey || event.metaKey || event.which !== 1) {
                    return;
                }
                return callback(event);
            }, false);

            element.addEventListener('touchstart', function(event) {
                if (event.touches.length > 1) return touchStarted = false;
                touchArea = { x: event.touches[0].screenX, y: event.touches[0].screenY };
                touchStarted = element;

                event.stopPropagation();
            }, false);

            window.addEventListener('touchstart', function(event) {
                touchStarted = false;
            });

            element.addEventListener('touchmove', function(event) {
                if (event.touches.length > 1) return touchStarted = false;
                var newTouchArea = { x: event.touches[0].screenX, y: event.touches[0].screenY };
                if (Math.pow(touchArea.x - newTouchArea.x, 2) + Math.pow(touchArea.y - newTouchArea.y, 2) > 500) {
                    touchStarted = false;
                }
            }, false);

            element.addEventListener('touchend', function(event) {
                if (touchStarted) {
                    var element = touchStarted;
                    touchStarted = false;

                    var x = event.changedTouches[0].pageX - window.pageXOffset;
                    var y = event.changedTouches[0].pageY - window.pageYOffset;
                    var target = document.elementFromPoint(x, y);

                    return callback(event);
                } else {
                    event.preventDefault();
                }
            }, false);
        };

        var buildQueryString = function(data) {
            var query = [];
            for (var key in data) {
                query.push(encodeURIComponent(key) + '=' + encodeURIComponent(data[key]));
            }
            return '?' + query.join('&');
        };

        var isIos = /iPad|iPhone|iPod/.test(navigator.userAgent) && !window.MSStream;

        // Scan for case-insensitive.
        var getDoiMeta = function() {
            var metaTags = document.querySelectorAll("meta");
            for (var i in metaTags) {
                if (metaTags.hasOwnProperty(i)) {
                    var tag = metaTags[i];
                    if ((tag.name || "").toLowerCase() == "dc.identifier" && tag.scheme == "doi") {
                        return tag;
                    }
                }
            }
        };

        var doiMeta = getDoiMeta();
        var doi = doiMeta ? doiMeta.getAttribute('content').replace(/^(info:doi\/|doi:)/, '') : null;
        if(doi == null) doi = $("#crossMark").data('doi');

        var queryData = {
            doi: doi,
            domain: window.location.hostname,
            uri_scheme: window.location.protocol,
            cm_version: SETTINGS.SCRIPT_VERSION
        };

        var css = document.createElement('link');
        css.setAttribute('href', SETTINGS.STYLESHEET_URL);
        css.setAttribute('type', 'text/css');
        css.setAttribute('rel', 'stylesheet');
        document.querySelector('head').appendChild(css);

        var widget = document.createElement('div');
        widget.setAttribute('id', 'crossmark-widget');

        // Initial display none to avoid a flash on load.
        widget.style.display = 'none';

        widget.innerHTML =
            '<div class="crossmark-reset crossmark-overlay"></div>' +
            '<div class="crossmark-reset crossmark-popup">' +
            '<div class="crossmark-reset crossmark-popup__offset">' +
            '<div class="crossmark-reset crossmark-popup__inner">' +
            '<div class="crossmark-reset crossmark-popup__header">' +
            '<a target="_blank" href="http://www.crossref.org/crossmark">' +
            '<img class="crossmark-reset crossmark-popup__logo">' +
            '</a>' +
            '<button class="crossmark-reset crossmark-popup__btn-close"></button>' +
            '</div>' +
            '<div class="crossmark-reset crossmark-popup__content-wrapper">' +
            '<iframe class="crossmark-reset crossmark-popup__content"></iframe>' +
            '</div>' +
            '</div>' +
            '</div>' +
            '</div>';

        var overlay = widget.querySelector('.crossmark-overlay');
        var popup = widget.querySelector('.crossmark-popup');
        var popupOffset = widget.querySelector('.crossmark-popup__offset');
        var popupInner = widget.querySelector('.crossmark-popup__inner');
        var logo = widget.querySelector('.crossmark-popup__logo');
        var content = widget.querySelector('.crossmark-popup__content');
        var closeButton = widget.querySelector('.crossmark-popup__btn-close');

        if (isIos) popupOffset.classList.add('is-ios');
        // logo.setAttribute('src', SETTINGS.LOGO_URL);

        document.body.appendChild(widget);

        [overlay, popup, closeButton].map(function(element) {
            tapEvent(element, function(event) {
                widget.style.display = 'none';

                event.preventDefault();
                event.stopPropagation();
            });
        });

        tapEvent(popupInner, function(event) {
            event.stopPropagation();
        });

        var initialised = false;

        // delete queryData.domain;

        var links = [].slice.call(document.querySelectorAll('[data-target=crossmark]'), 0);
        links.map(function(link) {
            link.style.cursor = 'pointer';
            link.setAttribute('href', SETTINGS.ENDPOINT + buildQueryString(queryData));

            tapEvent(link, function(event) {
                if (!initialised) {
                    content.setAttribute('src', SETTINGS.ENDPOINT + buildQueryString(queryData));
                    initialised = true;
                }
                widget.style.display = 'block';
                if (isIos) popupInner.style.top = window.scrollY + 'px';

                event.preventDefault();
                event.stopPropagation();
            });
        });
    });
</script><script type="text/javascript">
    var crossmark = {
        sTooltipID: 'crossmark-tooltip-130',
        sTooltipSelector: '#crossmark-tooltip-130',
        sTooltipCopy: 'Click to get updates and verify authenticity.',
        sOpenSelector: '#open-crossmark',
        activateTooltip: function() {
            var that = this;
            $('body').append('<div id="' + this.sTooltipID + '" class="crossmark-tooltip" style="display: none;"><div class="cmtttop"></div><div class="cmttmid"><p>' + this.sTooltipCopy + '</p></div><div class="cmttbot"></div></div>');
            var $icon = $('#crossmark-icon');
            $icon.attr({'title': '', 'alt': ''}).show();
            $icon.mouseover(function(){
                var o = $icon.offset();
                var x = o.left + $icon.width()/2 - $('#crossmark-icon').width()/2;
                var y = o.top - $(that.sTooltipSelector).height() + 10;
                $(that.sTooltipSelector).css({ 'left' : x, 'top' : y }).show();
            });
            $icon.mouseout(function(){
                $(that.sTooltipSelector).hide();
            });
        },
        addStylesheet: function() {
            $('#open-crossmark').parent().eq(0).prepend("<link media='screen' rel='preload' as='style' href='/templates/jsp/crossref.css'/>");
        }
    };
    $(document).ready(function() {
        crossmark.addStylesheet();
        crossmark.activateTooltip();
    });
</script>
        <link rel="stylesheet" type="text/css" href="/templates/jsp/crossref.css"/>
    







        
        <script class="optanon-category-4" type="text/plain" async src="https://www.youtube.com/iframe_api"></script>
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script class="optanon-category-4" async src="https://static.addtoany.com/menu/page.js"></script>
</head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>


<script type="text/javascript">

        if(true) {
            document.getElementById("skipNavigationLink").onclick =function skipElement () {
                document.getElementById('pageBody').setAttribute('tabindex','0');
                document.getElementById('pageBody').focus();
            }

        }

</script>




    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2931400-8%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS0092867420314008%253Fshowall%253Dtrue" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(20)31400-8?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867420314008%3Fshowall%3Dtrue"/><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(20)31400-8?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867420314008%3Fshowall%3Dtrue"/><input type="hidden" name="ecommRedirect"/><input type="hidden" name="code" value="cell-site"/><input type="hidden" name="UX3LoginWidget" value="true"/><input type="hidden" name="isPopup" value="true"/><div class="input-group"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"/></div><div class="input-group"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"/><a href="#" class="login__field__action js-show-password">Show</a><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"/><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"/></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2931400-8%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS0092867420314008%253Fshowall%253Dtrue" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"/><input type="hidden" name="requestResetPassword" value="true"/><input type="hidden" name="codeSite" value="cell-site"/><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"/></div><div class="submit"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"/></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br />
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br />
				<br />
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement"/>
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch">
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"/></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"/></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__loginHtml"><a href="https://www.editorialmanager.com/cell" data-aa-conversiondriver="jbs:global-header:submitpage">Submit</span></a></li><li class="profile-links__logIn"><a href="/action/showLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2931400-8%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS0092867420314008%253Fshowall%253Dtrue&amp;pii=S0092-8674%2820%2931400-8" title="Log in to Cell.com" class="show-login">Log in</a></li><li class="profile-links__register"><a href="/action/registration?journalCode=cell&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2931400-8%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS0092867420314008%253Fshowall%253Dtrue&amp;directReg=true" title="Register a username and password for free">Register</a></li><li class="hidden-lg"><div class="dropBlock profile-links__user-list__wrapper"><a href="#" data-db-target-for="loginMenu" aria-expanded="false" aria-controls="login-links" title="LogIn" class="profile-links__logIn__mobile">Log in</a><div data-db-target-of="loginMenu" id="login-links" class="dropBlock__holder profile-links__user-list__holder"><ul class="rlist article-tools__sub-list"><li><a href="https://www.editorialmanager.com/cell" data-aa-conversiondriver="jbs:global-header:submitpage">Submit</span></a></li><li><a href="/action/showLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2931400-8%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS0092867420314008%253Fshowall%253Dtrue&amp;pii=S0092-8674%2820%2931400-8" title="Log in to Cell.com" class="show-login">Log in</a></li></ul></div></div></li><li class="profile-links__subscribe"><a href="/cell/subscribe" title="Subscribe to a Cell journal">Subscribe</a></li><li class="profile-links__claim"><a href="/cell/activate" title="Claim your online subscription">Claim</a></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2020.10.036">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin widget-none  widget-compact-all" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d">
                
                
                    
                

                <div class="wrapped " id='CP TOP LEVEL NAV MENU'>
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-1" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About this journal</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            
                <div class="hidden-xs hidden-sm">
            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(23)X0017-8"><img src="/cms/asset/atypon:cms:attachment:img:d18e6:rev:1722883367683-3936:pii:S0092867423X00178/cover.tif.jpg" alt="Cover Image - Cell, Volume 187, Issue 16"/></a></div><div class="cover-image__meta"></div></div>

            
                </div>
            

        
</div>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life & medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host & Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods & Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology & Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology & Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences & engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/author-resources" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Author resources</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News & events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/lablinks" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>LabLinks</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help & support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://careers.cell.com/" data-index="1" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Scientific job board
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"/><div class="quick-search__toggle"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete"/><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"/><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"/><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"/><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    










        <main data-widget-def="UX3articleContent" data-widget-id="7f1f4a57-23b9-4395-ab00-2ba3b187e1d7" class="content">
        
    

        


    

            

            

            <article data-widget-id="7f1f4a57-23b9-4395-ab00-2ba3b187e1d7" data-widget-name="UX3articleContent" data-pii="S0092-8674(20)31400-8"><div id="articleHeader" class="article__header"><div class="article-header"><div class="article-header__overlay"></div><div class="what-image__wrapper"><img data-src="/cms/attachment/356e593f-1a8e-43e4-ba5e-fd94d795b599/fx1.jpg" aria-hidden="true" class="what-image b-lazy" loading="lazy"></img></div><div class="article-header__wrapper"><div class="article-header__top"><div class="row"><div class="col-md-6"><div class="article-header__meta"><span class="article-header__journal">Resource</span><span class="article-header__sep">|</span><a href="https://www.cell.com/cell/issue?pii=S0092-8674(19)X0025-2" class="article-header__vol faded">
                                                    Volume 183, ISSUE 5</a><span></span><span>, </span><span class="article-header__pages faded">P1436-1456.e31, </span><span class="article-header__date faded">November 25, 2020</span><div class="article-header__download-full-issue"><button title="Download Full Issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=183&amp;issue=5&amp;pii=S0092-8674%2819%29X0025-2" class="btn--cancel table-of-content__links--item download-full-issue"><span class="svg-icon"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 98 80"><path d="M77 30 71 23 54 40 54 0 44 0 44 40 26 23 19 30 48 59 77 30M88 48 88 68 10 68 10 48 0 48 0 78 98 78 98 48 88 48"></path></svg></span><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started.</p><a href="#" class="table-of-content__alert--close"> Ok</a></div></div></div></div><div class="col-md-6"><div class="article-header__dropZone pb-dropzone" data-pb-dropzone="articleTools-dropzone"><div data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><div class="article-tools__wrapper"><span class="article-tools__header"><!--.transplant(data-target="article h1" data-remove="false" data-target-class="hidden-xs hidden-sm" data-toggle="transplant" data-transplant="content")--></span><div class="article-tools__holder pull-right"><ul class="rlist rlist--inline article-tools__list"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2820%2931400-8" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><i aria-hidden="true" class="icon-gizmo-pdf-file"></i><span class="basic">PDF [13 MB]</span><span class="alternative">PDF [13 MB]</span></a></li><li class="article-tools__item article-tools__figures"><div aria-hidden="true" class="dropBlock article-tools__dropblock article-tools__figures"><a href="#" title="open Figures menu" data-db-target-for="article-tools-figures" aria-expanded="false" class="article-tools__ctrl article-tools__item__figures"><i aria-hidden="true" class="icon-gizmo-image_figure"></i><span>Figures</span></a><div data-db-target-of="article-tools-figures" class="dropBlock__holder article-tools__dropblock__holder"><ul class="rlist article-tools__sub-list"><li class="article-tools__sub-item"><a href="#" title="Figure Viewer (opens in a new tab)" target="_blank" class="article-tools__item__displayFigureViewer js--openFigureViewer ">Figure Viewer</a></li><li class="article-tools__sub-item"><a href="/cell/ppt/S0092-8674(20)31400-8.ppt" title="Download Figures (PPT) (opens in a new tab)" target="_blank" class="article-tools__item__displayDownloadFigures ">Download Figures (PPT)</a></li></ul></div></div></li><li class="article-tools__item article-tools__save"><div aria-hidden="false" class="dropBlock article-tools__dropblock article-tools__save"><a href="#" title="open Save menu" data-db-target-for="article-tools-save" aria-expanded="false" class="article-tools__ctrl article-tools__item__save"><i aria-hidden="true" class="icon-gizmo-folder-in"></i><span>Save</span></a><div data-db-target-of="article-tools-save" class="dropBlock__holder article-tools__dropblock__holder"><ul class="rlist article-tools__sub-list"><li class="article-tools__sub-item"><a href="#" title="Add to Online Library" class="article-tools__item__displayAddToOnlineLibrary mendeley ">Add to Online Library</a><span class="powered-by"><span>Powered By</span><i class="icon-mendeley"></i><span>Mendeley</span></span></li><li class="article-tools__sub-item"><a href="https://www.cell.com/personalize/addFavoritePublication?doi=10.1016%2Fj.cell.2020.10.036" title="Add to My Reading List (opens in a new tab)" target="_blank" class="article-tools__item__displayAddToMyReadingList ">Add to My Reading List</a></li><li class="article-tools__sub-item"><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2020.10.036&amp;pii=S0092-8674%2820%2931400-8" title="Export Citation (opens in a new tab)" target="_blank" class="article-tools__item__displayExportCitation ">Export Citation</a></li><li class="article-tools__sub-item"><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2020.10.036" title="Create Citation Alert (opens in a new tab)" target="_blank" class="article-tools__item__displayCreateCitationAlert ">Create Citation Alert</a></li></ul></div></div></li><li class="article-tools__item article-tools__coolBar">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><div data-widget-id="6f130e25-c0e4-42df-bb7d-35ca8d725c64" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><a href="#" data-db-target-for="6f130e25-c0e4-42df-bb7d-35ca8d725c64" aria-expanded="false" title="open Share menu" aria-labelledby="share-label-555" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-gizmo-share"></i><span id="share-label-555">Share</span></a><div data-db-target-of="6f130e25-c0e4-42df-bb7d-35ca8d725c64" id="6f130e25-c0e4-42df-bb7d-35ca8d725c64_Pop" class="share__block dropBlock__holder article-tools__dropblock__holder">
    

        


    

            

            

            <div data-widget-id="d7798b2a-1dc2-4a1b-bb92-9415c2383b15" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
    var res = window.location.href;
    res = res.replace(/\(/g, "%28");
    res = res.replace(/\)/g, "%29");    
    var addthis_share = {
       url: res
    }
    var addthis_config = {
       data_track_clickback: false 
    } 
</script> </div>

            

        
<span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2820%2931400-8"><li class="a2a_listitem_custom"><a role="link" title="Twitter" class="a2a_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter3"></i><span>Twitter</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook3"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin3"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-Email"></i><span>Email</span></a></li></ul></div></div>

            

        
</li><li><div class="dropBlock article-tools__dropblock article-tools__more"><a href="#" title="Show more tools menu" data-db-target-for="article-tools-more" aria-expanded="false" class="article-tools__ctrl"><i aria-hidden="true" class="icon-dots-three-horizontal"></i><span>more</span></a><div data-db-target-of="article-tools-more" class="dropBlock__holder article-tools__dropblock__holder"><ul class="rlist"><li><a href="http://www.medreprints.com/?pii=S0092867420314008&amp;issn=00928674&amp;source=jbs" title="Reprints" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-gizmo-reprints"></i><span>Reprints</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867420314008&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Request" class="article-tools__item__request"><i aria-hidden="true" class="icon-copyright"></i><span>Request</span></a></li></ul></div></div></li><li><a href="#" class="back-to-top"><i aria-hidden="true" class="icon-arrow_u_p"></i><span>Top</span></a></li></ul></div></div></div></div></div></div></div><div class="article-header__middle"><h1 class="article-header__title">Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy</h1><span class="article-header__sub-title"></span><ul class="rlist loa inline-bullet-list ellipsis-dot" data-number-of-author="6"><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au1-authorcard" aria-expanded="false" aria-controls="au1" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Karsten Krug">Karsten Krug</a> <a href="#" data-db-target-for="crosref0015" aria-expanded="false" aria-controls="crosref0015" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>17</sup></span></a><div data-db-target-of="crosref0015" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">17 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au1-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Karsten Krug</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">17 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Krug%2C+Karsten" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au2-authorcard" aria-expanded="false" aria-controls="au2" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Eric J. Jaehnig">Eric J. Jaehnig</a> <a href="#" data-db-target-for="crosref0025" aria-expanded="false" aria-controls="crosref0025" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>17</sup></span></a><div data-db-target-of="crosref0025" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">17 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au2-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Eric J. Jaehnig</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">17 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center and Dan L. Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Jaehnig%2C+Eric J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au3-authorcard" aria-expanded="false" aria-controls="au3" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Shankha Satpathy">Shankha Satpathy</a> <a href="#" data-db-target-for="crosref0035" aria-expanded="false" aria-controls="crosref0035" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>17</sup></span></a><div data-db-target-of="crosref0035" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">17 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au3-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Shankha Satpathy</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">17 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Satpathy%2C+Shankha" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au4-authorcard" aria-expanded="false" aria-controls="au4" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Lili Blumenberg">Lili Blumenberg</a> <a href="#" data-db-target-for="crosref0045" aria-expanded="false" aria-controls="crosref0045" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>17</sup></span></a><div data-db-target-of="crosref0045" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">17 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au4-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Lili Blumenberg</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">17 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Institute for Systems Genetics and Department of Medicine, NYU Grossman School of Medicine, New York, NY 10016, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Blumenberg%2C+Lili" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au5-authorcard" aria-expanded="false" aria-controls="au5" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Alla Karpova">Alla Karpova</a> <a href="#" data-db-target-for="crosref0055" aria-expanded="false" aria-controls="crosref0055" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>17</sup></span></a><div data-db-target-of="crosref0055" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">17 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au5-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Alla Karpova</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">17 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Medicine and Genetics, Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Karpova%2C+Alla" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au6-authorcard" aria-expanded="false" aria-controls="au6" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Meenakshi Anurag">Meenakshi Anurag</a> <a href="#" data-db-target-for="crosref0065" aria-expanded="false" aria-controls="crosref0065" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>17</sup></span></a><div data-db-target-of="crosref0065" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">17 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au6-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Meenakshi Anurag</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">17 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center and Dan L. Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Anurag%2C+Meenakshi" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au7-authorcard" aria-expanded="false" aria-controls="au7" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author George Miles">George Miles</a><div data-db-target-of="au7-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">George Miles</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center and Dan L. Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Miles%2C+George" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au8-authorcard" aria-expanded="false" aria-controls="au8" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Philipp Mertins">Philipp Mertins</a><div data-db-target-of="au8-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Philipp Mertins</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA<p></p>Max Delbrück Center for Molecular Medicine in the Helmholtz Society and Berlin Institute of Health, Berlin, Germany</div></div><a href="/action/doSearch?ContribAuthorRaw=Mertins%2C+Philipp" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au9-authorcard" aria-expanded="false" aria-controls="au9" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Yifat Geffen">Yifat Geffen</a><div data-db-target-of="au9-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Yifat Geffen</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Geffen%2C+Yifat" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au10-authorcard" aria-expanded="false" aria-controls="au10" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Lauren C. Tang">Lauren C. Tang</a><div data-db-target-of="au10-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Lauren C. Tang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA<p></p>Department of Biological Sciences, Columbia University, New York, NY 10027, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Tang%2C+Lauren C" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au11-authorcard" aria-expanded="false" aria-controls="au11" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author David I. Heiman">David I. Heiman</a><div data-db-target-of="au11-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">David I. Heiman</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Heiman%2C+David I" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au12-authorcard" aria-expanded="false" aria-controls="au12" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Song Cao">Song Cao</a><div data-db-target-of="au12-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Song Cao</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Medicine and Genetics, Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Cao%2C+Song" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au13-authorcard" aria-expanded="false" aria-controls="au13" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Yosef E. Maruvka">Yosef E. Maruvka</a><div data-db-target-of="au13-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Yosef E. Maruvka</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Maruvka%2C+Yosef E" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au14-authorcard" aria-expanded="false" aria-controls="au14" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Jonathan T. Lei">Jonathan T. Lei</a><div data-db-target-of="au14-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Jonathan T. Lei</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center and Dan L. Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Lei%2C+Jonathan T" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au15-authorcard" aria-expanded="false" aria-controls="au15" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Chen Huang">Chen Huang</a><div data-db-target-of="au15-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Chen Huang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center and Dan L. Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Huang%2C+Chen" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au16-authorcard" aria-expanded="false" aria-controls="au16" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Ramani B. Kothadia">Ramani B. Kothadia</a><div data-db-target-of="au16-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Ramani B. Kothadia</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Kothadia%2C+Ramani B" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au17-authorcard" aria-expanded="false" aria-controls="au17" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Antonio Colaprico">Antonio Colaprico</a><div data-db-target-of="au17-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Antonio Colaprico</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Division of Biostatistics, Department of Public Health Science, University of Miami Miller School of Medicine, Miami, FL 33136, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Colaprico%2C+Antonio" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au18-authorcard" aria-expanded="false" aria-controls="au18" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Chet Birger">Chet Birger</a><div data-db-target-of="au18-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Chet Birger</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Birger%2C+Chet" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au19-authorcard" aria-expanded="false" aria-controls="au19" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Jarey Wang">Jarey Wang</a><div data-db-target-of="au19-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Jarey Wang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Verna and Marrs McLean Department of Biochemistry and Molecular Biology, Department of Molecular and Human Genetics, and Therapeutic Innovation Center, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Wang%2C+Jarey" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au20-authorcard" aria-expanded="false" aria-controls="au20" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Yongchao Dou">Yongchao Dou</a><div data-db-target-of="au20-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Yongchao Dou</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center and Dan L. Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Dou%2C+Yongchao" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au21-authorcard" aria-expanded="false" aria-controls="au21" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Bo Wen">Bo Wen</a><div data-db-target-of="au21-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Bo Wen</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center and Dan L. Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Wen%2C+Bo" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au22-authorcard" aria-expanded="false" aria-controls="au22" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Zhiao Shi">Zhiao Shi</a><div data-db-target-of="au22-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Zhiao Shi</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center and Dan L. Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Shi%2C+Zhiao" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au23-authorcard" aria-expanded="false" aria-controls="au23" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Yuxing Liao">Yuxing Liao</a><div data-db-target-of="au23-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Yuxing Liao</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center and Dan L. Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Liao%2C+Yuxing" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au24-authorcard" aria-expanded="false" aria-controls="au24" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Maciej Wiznerowicz">Maciej Wiznerowicz</a><div data-db-target-of="au24-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Maciej Wiznerowicz</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Poznan University of Medical Sciences, Poznań 61-701, Poland<p></p>International Institute for Molecular Oncology, 60-203 Poznań, Poland</div></div><a href="/action/doSearch?ContribAuthorRaw=Wiznerowicz%2C+Maciej" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au25-authorcard" aria-expanded="false" aria-controls="au25" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Matthew A. Wyczalkowski">Matthew A. Wyczalkowski</a><div data-db-target-of="au25-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Matthew A. Wyczalkowski</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Medicine and Genetics, Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Wyczalkowski%2C+Matthew A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au26-authorcard" aria-expanded="false" aria-controls="au26" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Xi Steven Chen">Xi Steven Chen</a><div data-db-target-of="au26-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Xi Steven Chen</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Division of Biostatistics, Department of Public Health Science, University of Miami Miller School of Medicine, Miami, FL 33136, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Chen%2C+Xi Steven" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au27-authorcard" aria-expanded="false" aria-controls="au27" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Jacob J. Kennedy">Jacob J. Kennedy</a><div data-db-target-of="au27-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Jacob J. Kennedy</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Clinical Research Division, Fred Hutchinson Cancer Research Center, Seattle, WA 98109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Kennedy%2C+Jacob J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au28-authorcard" aria-expanded="false" aria-controls="au28" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Amanda G. Paulovich">Amanda G. Paulovich</a><div data-db-target-of="au28-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Amanda G. Paulovich</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Clinical Research Division, Fred Hutchinson Cancer Research Center, Seattle, WA 98109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Paulovich%2C+Amanda G" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au29-authorcard" aria-expanded="false" aria-controls="au29" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Mathangi Thiagarajan">Mathangi Thiagarajan</a><div data-db-target-of="au29-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Mathangi Thiagarajan</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Leidos Biomedical Research Inc., Frederick National Laboratory for Cancer Research, Frederick, MD 21702, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Thiagarajan%2C+Mathangi" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au30-authorcard" aria-expanded="false" aria-controls="au30" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Christopher R. Kinsinger">Christopher R. Kinsinger</a><div data-db-target-of="au30-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Christopher R. Kinsinger</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Kinsinger%2C+Christopher R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au31-authorcard" aria-expanded="false" aria-controls="au31" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Tara Hiltke">Tara Hiltke</a><div data-db-target-of="au31-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Tara Hiltke</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Hiltke%2C+Tara" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au32-authorcard" aria-expanded="false" aria-controls="au32" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Emily S. Boja">Emily S. Boja</a><div data-db-target-of="au32-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Emily S. Boja</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Boja%2C+Emily S" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au33-authorcard" aria-expanded="false" aria-controls="au33" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Mehdi Mesri">Mehdi Mesri</a><div data-db-target-of="au33-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Mehdi Mesri</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Mesri%2C+Mehdi" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au34-authorcard" aria-expanded="false" aria-controls="au34" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Ana I. Robles">Ana I. Robles</a><div data-db-target-of="au34-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Ana I. Robles</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Robles%2C+Ana I" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au35-authorcard" aria-expanded="false" aria-controls="au35" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Henry Rodriguez">Henry Rodriguez</a><div data-db-target-of="au35-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Henry Rodriguez</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Rodriguez%2C+Henry" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au36-authorcard" aria-expanded="false" aria-controls="au36" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Thomas F. Westbrook">Thomas F. Westbrook</a><div data-db-target-of="au36-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Thomas F. Westbrook</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Verna and Marrs McLean Department of Biochemistry and Molecular Biology, Department of Molecular and Human Genetics, and Therapeutic Innovation Center, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Westbrook%2C+Thomas F" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au37-authorcard" aria-expanded="false" aria-controls="au37" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Li Ding">Li Ding</a><div data-db-target-of="au37-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Li Ding</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Medicine and Genetics, Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Ding%2C+Li" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au38-authorcard" aria-expanded="false" aria-controls="au38" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Gad Getz">Gad Getz</a><div data-db-target-of="au38-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Gad Getz</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA<p></p>Center for Cancer Research, Massachusetts General Hospital, Charlestown, MA 02114, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Getz%2C+Gad" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au39-authorcard" aria-expanded="false" aria-controls="au39" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Karl R. Clauser">Karl R. Clauser</a><div data-db-target-of="au39-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Karl R. Clauser</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Clauser%2C+Karl R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au40-authorcard" aria-expanded="false" aria-controls="au40" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author David Fenyö">David Fenyö</a><div data-db-target-of="au40-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">David Fenyö</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Fenyö%2C+David" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au41-authorcard" aria-expanded="false" aria-controls="au41" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Kelly V. Ruggles">Kelly V. Ruggles</a><div data-db-target-of="au41-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Kelly V. Ruggles</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Institute for Systems Genetics and Department of Medicine, NYU Grossman School of Medicine, New York, NY 10016, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Ruggles%2C+Kelly V" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au42-authorcard" aria-expanded="false" aria-controls="au42" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Bing Zhang">Bing Zhang</a><div data-db-target-of="au42-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Bing Zhang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center and Dan L. Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Zhang%2C+Bing" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au43-authorcard" aria-expanded="false" aria-controls="au43" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author D.R. Mani">D.R. Mani<i aria-hidden="true" class="icon-gizmo-person faded"></i></a><div data-db-target-of="au43-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">D.R. Mani</div><span class="article-header__info__group__label">Correspondence</span><div class="article-header__info__sub-label">Corresponding author</div><div class="article-header__info__body"><div class="article-header__info__social"><a class="article-header__info__email default" href="/cdn-cgi/l/email-protection#bcd1ddd2d5d8cefcdeced3ddd8d5d2cfc8d5c8c9c8d992d3cedb" title="Send email to D.R. Mani"><i aria-hidden="true" class="icon-Email"></i><span>
                            Contact
                        </span></a></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Mani%2C+D.R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div class="dropBlock article-header__info"><a title="Send email to D.R. Mani" class="loa__item__name  loa__item__email" href="/cdn-cgi/l/email-protection#c6aba7a8afa2b486a4b4a9a7a2afa8b5b2afb2b3b2a3e8a9b4a1"><i aria-hidden="true" class="icon-Email faded"></i></a></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au44-authorcard" aria-expanded="false" aria-controls="au44" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Steven A. Carr">Steven A. Carr<i aria-hidden="true" class="icon-gizmo-person faded"></i></a><div data-db-target-of="au44-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Steven A. Carr</div><span class="article-header__info__group__label">Correspondence</span><div class="article-header__info__sub-label">Corresponding author</div><div class="article-header__info__body"><div class="article-header__info__social"><a class="article-header__info__email default" href="/cdn-cgi/l/email-protection#1261717360605270607d73763c7f7b663c777667" title="Send email to Steven A. Carr"><i aria-hidden="true" class="icon-Email"></i><span>
                            Contact
                        </span></a></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Carr%2C+Steven A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div class="dropBlock article-header__info"><a title="Send email to Steven A. Carr" class="loa__item__name  loa__item__email" href="/cdn-cgi/l/email-protection#ccbfafadbebe8caebea3ada8e2a1a5b8e2a9a8b9"><i aria-hidden="true" class="icon-Email faded"></i></a></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au45-authorcard" aria-expanded="false" aria-controls="au45" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Matthew J. Ellis">Matthew J. Ellis<i aria-hidden="true" class="icon-gizmo-person faded"></i></a><div data-db-target-of="au45-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Matthew J. Ellis</div><span class="article-header__info__group__label">Correspondence</span><div class="article-header__info__sub-label">Corresponding author</div><div class="article-header__info__body"><div class="article-header__info__social"><a class="article-header__info__email default" href="/cdn-cgi/l/email-protection#25484f4049494c56654746480b404150" title="Send email to Matthew J. Ellis"><i aria-hidden="true" class="icon-Email"></i><span>
                            Contact
                        </span></a></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center and Dan L. Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Ellis%2C+Matthew J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div class="dropBlock article-header__info"><a title="Send email to Matthew J. Ellis" class="loa__item__name  loa__item__email" href="/cdn-cgi/l/email-protection#55383f3039393c26153736387b303120"><i aria-hidden="true" class="icon-Email faded"></i></a></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au46-authorcard" aria-expanded="false" aria-controls="au46" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Michael A. Gillette">Michael A. Gillette<i aria-hidden="true" class="icon-gizmo-person faded"></i></a> <a href="#" data-db-target-for="crosref0310" aria-expanded="false" aria-controls="crosref0310" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>18</sup></span></a><div data-db-target-of="crosref0310" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">18 Lead Contact<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au46-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Michael A. Gillette</div><span class="article-header__info__group__label">Correspondence</span><div class="article-header__info__sub-label">Corresponding author</div><div class="article-header__info__body"><div class="article-header__info__social"><a class="article-header__info__email default" href="/cdn-cgi/l/email-protection#a9cec0c5c5ccddddcce9cbdbc6c8cdc0c7daddc0dddcddcc87c6dbce" title="Send email to Michael A. Gillette"><i aria-hidden="true" class="icon-Email"></i><span>
                            Contact
                        </span></a></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">18 Lead Contact<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA<p></p>Division of Pulmonary and Critical Care Medicine, Massachusetts General Hospital, Boston, MA 02114, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Gillette%2C+Michael A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div class="dropBlock article-header__info"><a title="Send email to Michael A. Gillette" class="loa__item__name  loa__item__email" href="/cdn-cgi/l/email-protection#47202e2b2b223333220725352826232e2934332e3332332269283520"><i aria-hidden="true" class="icon-Email faded"></i></a></div></li><li class="loa__item collaboration-author author"><span><div class="dropBlock article-header__info"><a href="#" data-db-target-for="collab0010" aria-expanded="false" aria-controls="collab0010" class="loa__item__name article-header__info__ctrl loa__item__email">Clinical Proteomic Tumor Analysis Consortium</a><div data-db-target-of="collab0010" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Clinical Proteomic Tumor Analysis Consortium</div><div class="article-header__info__body"><span class="article-header__info__group__label">Authors List</span><ul class="rlist dropBlock__slider"><li class="loa__item"><a href="#" title="Shayan C. Avanessian" class="dropBlock__slider__ctrl">Shayan C. Avanessian</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Shayan C. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Shayan C. Avanessian</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Avanessian%2C+Shayan C" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Shuang Cai" class="dropBlock__slider__ctrl">Shuang Cai</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Shuang back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Shuang Cai</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Cai%2C+Shuang" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Daniel Chan" class="dropBlock__slider__ctrl">Daniel Chan</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Daniel back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Daniel Chan</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Chan%2C+Daniel" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Xian Chen" class="dropBlock__slider__ctrl">Xian Chen</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Xian back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Xian Chen</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Chen%2C+Xian" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Nathan J. Edwards" class="dropBlock__slider__ctrl">Nathan J. Edwards</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Nathan J. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Nathan J. Edwards</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Edwards%2C+Nathan J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Andrew N. Hoofnagle" class="dropBlock__slider__ctrl">Andrew N. Hoofnagle</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Andrew N. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Andrew N. Hoofnagle</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hoofnagle%2C+Andrew N" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="M. Harry Kane" class="dropBlock__slider__ctrl">M. Harry Kane</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="M. Harry back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">M. Harry Kane</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Kane%2C+M. Harry" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Karen A. Ketchum" class="dropBlock__slider__ctrl">Karen A. Ketchum</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Karen A. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Karen A. Ketchum</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Ketchum%2C+Karen A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Eric Kuhn" class="dropBlock__slider__ctrl">Eric Kuhn</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Eric back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Eric Kuhn</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Kuhn%2C+Eric" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Douglas A. Levine" class="dropBlock__slider__ctrl">Douglas A. Levine</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Douglas A. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Douglas A. Levine</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Levine%2C+Douglas A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Shunqiang Li" class="dropBlock__slider__ctrl">Shunqiang Li</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Shunqiang back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Shunqiang Li</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Li%2C+Shunqiang" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Daniel C. Liebler" class="dropBlock__slider__ctrl">Daniel C. Liebler</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Daniel C. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Daniel C. Liebler</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Liebler%2C+Daniel C" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Tao Liu" class="dropBlock__slider__ctrl">Tao Liu</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Tao back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Tao Liu</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Liu%2C+Tao" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Jingqin Luo" class="dropBlock__slider__ctrl">Jingqin Luo</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Jingqin back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Jingqin Luo</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Luo%2C+Jingqin" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Subha Madhavan" class="dropBlock__slider__ctrl">Subha Madhavan</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Subha back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Subha Madhavan</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Madhavan%2C+Subha" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Chris Maher" class="dropBlock__slider__ctrl">Chris Maher</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Chris back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Chris Maher</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Maher%2C+Chris" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Jason E. McDermott" class="dropBlock__slider__ctrl">Jason E. McDermott</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Jason E. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Jason E. McDermott</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=McDermott%2C+Jason E" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Peter B. McGarvey" class="dropBlock__slider__ctrl">Peter B. McGarvey</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Peter B. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Peter B. McGarvey</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=McGarvey%2C+Peter B" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Mauricio Oberti" class="dropBlock__slider__ctrl">Mauricio Oberti</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Mauricio back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Mauricio Oberti</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Oberti%2C+Mauricio" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Akhilesh Pandey" class="dropBlock__slider__ctrl">Akhilesh Pandey</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Akhilesh back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Akhilesh Pandey</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Pandey%2C+Akhilesh" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Samuel H. Payne" class="dropBlock__slider__ctrl">Samuel H. Payne</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Samuel H. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Samuel H. Payne</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Payne%2C+Samuel H" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="David F. Ransohoff" class="dropBlock__slider__ctrl">David F. Ransohoff</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="David F. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">David F. Ransohoff</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Ransohoff%2C+David F" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Robert C. Rivers" class="dropBlock__slider__ctrl">Robert C. Rivers</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Robert C. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Robert C. Rivers</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Rivers%2C+Robert C" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Karin D. Rodland" class="dropBlock__slider__ctrl">Karin D. Rodland</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Karin D. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Karin D. Rodland</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Rodland%2C+Karin D" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Paul Rudnick" class="dropBlock__slider__ctrl">Paul Rudnick</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Paul back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Paul Rudnick</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Rudnick%2C+Paul" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Melinda E. Sanders" class="dropBlock__slider__ctrl">Melinda E. Sanders</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Melinda E. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Melinda E. Sanders</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Sanders%2C+Melinda E" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Kenna M. Shaw" class="dropBlock__slider__ctrl">Kenna M. Shaw</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Kenna M. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Kenna M. Shaw</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Shaw%2C+Kenna M" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Ie-Ming Shih" class="dropBlock__slider__ctrl">Ie-Ming Shih</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Ie-Ming back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Ie-Ming Shih</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Shih%2C+Ie-Ming" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Robbert J.C. Slebos" class="dropBlock__slider__ctrl">Robbert J.C. Slebos</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Robbert J.C. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Robbert J.C. Slebos</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Slebos%2C+Robbert J.C" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Richard D. Smith" class="dropBlock__slider__ctrl">Richard D. Smith</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Richard D. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Richard D. Smith</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Smith%2C+Richard D" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Michael Snyder" class="dropBlock__slider__ctrl">Michael Snyder</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Michael back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Michael Snyder</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Snyder%2C+Michael" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Stephen E. Stein" class="dropBlock__slider__ctrl">Stephen E. Stein</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Stephen E. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Stephen E. Stein</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Stein%2C+Stephen E" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="David L. Tabb" class="dropBlock__slider__ctrl">David L. Tabb</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="David L. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">David L. Tabb</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Tabb%2C+David L" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Ratna R. Thangudu" class="dropBlock__slider__ctrl">Ratna R. Thangudu</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Ratna R. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Ratna R. Thangudu</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Thangudu%2C+Ratna R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Stefani Thomas" class="dropBlock__slider__ctrl">Stefani Thomas</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Stefani back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Stefani Thomas</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Thomas%2C+Stefani" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Yue Wang" class="dropBlock__slider__ctrl">Yue Wang</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Yue back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Yue Wang</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Wang%2C+Yue" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Forest M. White" class="dropBlock__slider__ctrl">Forest M. White</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Forest M. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Forest M. White</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=White%2C+Forest M" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Jeffrey R. Whiteaker" class="dropBlock__slider__ctrl">Jeffrey R. Whiteaker</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Jeffrey R. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Jeffrey R. Whiteaker</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Whiteaker%2C+Jeffrey R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Gordon A. Whiteley" class="dropBlock__slider__ctrl">Gordon A. Whiteley</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Gordon A. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Gordon A. Whiteley</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Whiteley%2C+Gordon A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Hui Zhang" class="dropBlock__slider__ctrl">Hui Zhang</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Hui back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Hui Zhang</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Zhang%2C+Hui" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Zhen Zhang" class="dropBlock__slider__ctrl">Zhen Zhang</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Zhen back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Zhen Zhang</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Zhang%2C+Zhen" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Yingming Zhao" class="dropBlock__slider__ctrl">Yingming Zhao</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Yingming back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Yingming Zhao</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Zhao%2C+Yingming" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Heng Zhu" class="dropBlock__slider__ctrl">Heng Zhu</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Heng back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Heng Zhu</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Zhu%2C+Heng" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Lisa J. Zimmerman" class="dropBlock__slider__ctrl">Lisa J. Zimmerman</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Lisa J. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Lisa J. Zimmerman</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Zimmerman%2C+Lisa J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li></ul></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></span></li> <li class="loa__item"><div class="dropBlock article-header__info article-header__footnotes"><a href="#" data-db-target-for="foot-notes" aria-expanded="false" aria-controls="" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span class="show">Show footnotes</span><span class="hide">Hide footnotes</span></a><div data-db-target-of="foot-notes" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">17 These authors contributed equally<br/>18 Lead Contact<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div></li></ul><div class="inline-it"><span class="bulleted"><span><span class="article-header__access">Open Access</span></span></span><span class="article-header__publish-date bulleted"><span class="article-header__publish-date__label">Published:</span><span class="article-header__publish-date__value">November 18, 2020</span></span><span class="article-header__doi bulleted"><span class="article-header__doi__label">DOI:</span><a class="article-header__doi__value" href="https://doi.org/10.1016/j.cell.2020.10.036">https://doi.org/10.1016/j.cell.2020.10.036</a></span><span class="article-header__mark bulleted"><a id="open-crossmark" data-target="crossmark"><img id="crossmark-icon" width="150" src="//crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" loading="lazy"></img></a><meta name="dc.identifier" scheme="doi" content="doi:10.1016/j.cell.2020.10.036"></meta></span></div></div><div class="article-header__bottom clearfix row"><div class="col-md-6 static-md"></div><div class="col-md-6 static-md"><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" src="//cdn.plu.mx/widget-popup.js"></script><a href="https://plu.mx/plum/a/?doi=10.1016/j.cell.2020.10.036" class="plumx-plum-print-popup plum-jbs-theme article-header__metrics" data-popup="left" data-hide-when-empty="true" data-hide-usage="true" data-pass-hidden-categories="true" data-site="plum" data-badge="true" data-size="small">Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy</a></div></div></div><div class="article-header__dropZone pb-dropzone" data-pb-dropzone="articleNavigation-Dropzone"><div data-widget-id="54733c81-009b-4b22-b588-f003dedf997f" data-widget-name="UX3ArticleNavigation" class="content-navigation clearfix"><div class="content-navigation__btn__wrapper--prev"><a href="/cell/fulltext/S0092-8674(20)31387-8" title="Previous Article" class="content-navigation__btn--pre default"><span class="content-navigation__body pre"><span class="content-navigation__body__top">Previous Article</span><span class="content-navigation__body__title">An Organoid Biobank of Neuroendocrine Neoplasms Enables Genotype-Phenotype Mapping</span></span><i aria-hidden="true" class="icon-arrow_l"></i></a></div><div class="content-navigation__extra"></div></div>

            

        

    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div></div></div></div><div class="article__body"><button title="open table of sections" aria-label="open table of sections" type="button" class="left-side-nav__ctrl"><i aria-hidden="true" class="icon-sub-menu"></i></button><div class="left-side-nav col-md-2 col-lg-1"><ul role="menu" class="rlist left-side-nav__upper"></ul><ul role="menu" class="rlist left-side-nav__bottom"></ul></div><div class="adplaceholders hidden-xs hidden-sm not-sticky">
    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        

    

        


    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id='Marketplace-Recs'>
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement"/>
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async></script>
<div id="pbgrd-skyr-c"></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        


    

        


</div><div class="container"><div class="row"><div class="col-md-2 col-lg-1 hidden-sm"></div><div class="col-md-7 col-lg-9 article__sections"><section><h2 class="top" id="secsectitle0020" data-left-hand-nav="Highlights" tabindex="0"><span class="top__sub"></span><span class="top__text">Highlights</span></h2><div class="section-paragraph"><ul class="ce-list--remove-bullets" id="ulist0010"><li class="ce-list--remove-bullets__list-item" id="u0010"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Comprehensive proteogenomics resource from prospectively collected breast tumors</div></li><li class="ce-list--remove-bullets__list-item" id="u0015"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Proteogenomics defines ERBB2 and Rb status with clinical implications</div></li><li class="ce-list--remove-bullets__list-item" id="u0020"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Acetylproteome profiling yields insights into subtype-specific cancer metabolism</div></li><li class="ce-list--remove-bullets__list-item" id="u0025"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Immune profiling nominates subsets of luminal tumors for immune therapy</div></li></ul></div></section><section><h2 class="top" id="secsectitle0010" data-left-hand-nav="Summary" tabindex="0"><span class="top__sub"></span><span class="top__text">Summary</span></h2><div class="section-paragraph"><div class="section-paragraph">The integration of mass spectrometry-based proteomics with next-generation DNA and RNA sequencing profiles tumors more comprehensively. Here this “proteogenomics” approach was applied to 122 treatment-naive primary breast cancers accrued to preserve post-translational modifications, including protein phosphorylation and acetylation. Proteogenomics challenged standard breast cancer diagnoses, provided detailed analysis of the <em>ERBB2</em> amplicon, defined tumor subsets that could benefit from immune checkpoint therapy, and allowed more accurate assessment of Rb status for prediction of CDK4/6 inhibitor responsiveness. Phosphoproteomics profiles uncovered novel associations between tumor suppressor loss and targetable kinases. Acetylproteome analysis highlighted acetylation on key nuclear proteins involved in the DNA damage response and revealed cross-talk between cytoplasmic and mitochondrial acetylation and metabolism. Our results underscore the potential of proteogenomics for clinical investigation of breast cancer through more accurate annotation of targetable pathways and biological features of this remarkably heterogeneous malignancy.</div></div></section><section><h2 class="top" id="secsectitle0015" data-left-hand-nav="Graphical Abstract" tabindex="0"><span class="top__sub"></span><span class="top__text">Graphical Abstract</span></h2><div class="section-paragraph"><figure id="fx1" class="graphical-abstract" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/356e593f-1a8e-43e4-ba5e-fd94d795b599/fx1.jpg" alt="Figure thumbnail fx1" data-locator="undfig1" data-high-res="/cms/attachment/441742c1-952f-47e5-9333-55f25b935668/fx1_lrg.jpg" data-large-image="/cms/attachment/356e593f-1a8e-43e4-ba5e-fd94d795b599/fx1.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Graph. Abst." class="label figure__label">Graphical Abstract</span><span class="figure__title__text"></span></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/356e593f-1a8e-43e4-ba5e-fd94d795b599/fx1.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/441742c1-952f-47e5-9333-55f25b935668/fx1_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(20)31400-8&amp;id=fx1.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></section><section><h2 class="top" id="secsectitle0025" data-left-hand-nav="Keywords" tabindex="0"><span class="top__sub"></span><span class="top__text">Keywords</span></h2><ul class="rlist keywords-list inline-bullet-list"><li><a href="/action/doSearch?AllField=&#34;CPTAC&#34;&amp;ISSN=0092-8674">CPTAC</a></li><li><a href="/action/doSearch?AllField=&#34;proteogenomics&#34;&amp;ISSN=0092-8674">proteogenomics</a></li><li><a href="/action/doSearch?AllField=&#34;breast cancer&#34;&amp;ISSN=0092-8674">breast cancer</a></li><li><a href="/action/doSearch?AllField=&#34;genomics&#34;&amp;ISSN=0092-8674">genomics</a></li><li><a href="/action/doSearch?AllField=&#34;proteomics&#34;&amp;ISSN=0092-8674">proteomics</a></li><li><a href="/action/doSearch?AllField=&#34;phosphoproteomics&#34;&amp;ISSN=0092-8674">phosphoproteomics</a></li><li><a href="/action/doSearch?AllField=&#34;acetylation&#34;&amp;ISSN=0092-8674">acetylation</a></li><li><a href="/action/doSearch?AllField=&#34;mass spectrometry&#34;&amp;ISSN=0092-8674">mass spectrometry</a></li><li><a href="/action/doSearch?AllField=&#34;immune checkpoint therapy&#34;&amp;ISSN=0092-8674">immune checkpoint therapy</a></li><li><a href="/action/doSearch?AllField=&#34;CDK 4/6 inhibitors&#34;&amp;ISSN=0092-8674">CDK 4/6 inhibitors</a></li></ul></section><div data-pb-dropzone="relatedItems-dropzone" class="relatedItems-dropzone pb-dropzone"><!-- Empty dropzone --></div><section id="sec1" data-locator="sec1" tabindex="0"><h2 class="top" id="secsectitle0030" data-left-hand-nav="Introduction" tabindex="0"><span class="top__sub"></span><span class="top__text">Introduction</span></h2><div class="section-paragraph">The heterogeneity of breast cancer (BRCA) biology deeply challenges the drive for personalized treatment (<div class="dropBlock reference-citations"><a href="#bib52" id="back-bib52" data-db-target-for="bib52-d13420944e1641" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib52">Hyman et al., 2017</a><div data-db-target-of="bib52-d13420944e1641" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hyman D.M. </li><li class="loa__item">Taylor B.S. </li><li class="loa__item">Baselga J. </li></ul><div class="reference-citations__detials">Implementing Genome-Driven Oncology.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">168</span></span>: <span class="ref__seriesPages">584-599</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2016.12.015&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2016.12.015&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2016.12.015&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=28187282&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85012024906&amp;cf=" onclick="newWindow(this.href);return false">Scopus (244)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=168&publication_year=2017&pages=584-599&journal=Cell&author=Hyman+D.M.&author=Taylor+B.S.&author=Baselga+J.&title=Implementing+Genome-Driven+Oncology">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Contemporary precision therapies target defects in DNA repair, activated protein kinases, the estrogen receptor (ER), and the immune tumor microenvironment, often in combination (<div class="dropBlock reference-citations"><a href="#bib127" id="back-bib127" data-db-target-for="bib127-d13420944e1644" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib127">Telli et al., 2019</a><div data-db-target-of="bib127-d13420944e1644" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Telli M.L. </li><li class="loa__item">Gradishar W.J. </li><li class="loa__item">Ward J.H. </li></ul><div class="reference-citations__detials">NCCN Guidelines Updates: Breast Cancer.<div class="popup-ref"><span class="ref__series"><i>J. Natl. Compr. Canc. Netw.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">552-555</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31117035&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2019&pages=552-555&journal=J.%C2%A0Natl.+Compr.+Canc.+Netw.&author=Telli+M.L.&author=Gradishar+W.J.&author=Ward+J.H.&title=NCCN+Guidelines+Updates%3A+Breast+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Effective application of these approaches depends on our ability to accurately profile tumors to identify individual therapeutic vulnerabilities, but current methods in early-stage BRCA, including mRNA-based prognostic tests, are inadequate (<div class="dropBlock reference-citations"><a href="#bib29" id="back-bib29" data-db-target-for="bib29-d13420944e1647" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib29">Coates et al., 2015</a><div data-db-target-of="bib29-d13420944e1647" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Coates A.S. </li><li class="loa__item">Winer E.P. </li><li class="loa__item">Goldhirsch A. </li><li class="loa__item">Gelber R.D. </li><li class="loa__item">Gnant M. </li><li class="loa__item">Piccart-Gebhart M. </li><li class="loa__item">Thürlimann B. </li><li class="loa__item">Senn H.-J. </li></ul><div class="reference-citations__detials"><span>Panel Members</span><br />Tailoring therapies--improving the management of early breast cancer: St Gallen International Expert Consensus on the Primary Therapy of Early Breast Cancer 2015.<div class="popup-ref"><span class="ref__series"><i>Ann. Oncol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">1533-1546</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fannonc%2Fmdv221&amp;cf=abstract&amp;site=annonc-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fannonc%2Fmdv221&amp;cf=fulltext&amp;site=annonc-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fannonc%2Fmdv221&amp;cf=pdf&amp;site=annonc-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=25939896&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84939423024&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1118)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2015&pages=1533-1546&journal=Ann.+Oncol.&author=Panel+Members&title=Tailoring+therapies%2D%2Dimproving+the+management+of+early+breast+cancer%3A+St+Gallen+International+Expert+Consensus+on+the+Primary+Therapy+of+Early+Breast+Cancer+2015">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib109" id="back-bib109" data-db-target-for="bib109-d13420944e1650" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib109">Ross et al., 2007</a><div data-db-target-of="bib109-d13420944e1650" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ross J.S. </li><li class="loa__item">Symmans W.F. </li><li class="loa__item">Pusztai L. </li><li class="loa__item">Hortobagyi G.N. </li></ul><div class="reference-citations__detials">Standardizing slide-based assays in breast cancer: hormone receptors, HER2, and sentinel lymph nodes.<div class="popup-ref"><span class="ref__series"><i>Clin. Cancer Res.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">2831-2835</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1158%2F1078-0432.CCR-06-2522&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=17504980&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-34249822213&amp;cf=" onclick="newWindow(this.href);return false">Scopus (52)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2007&pages=2831-2835&journal=Clin.+Cancer+Res.&author=Ross+J.S.&author=Symmans+W.F.&author=Pusztai+L.&author=Hortobagyi+G.N.&title=Standardizing+slide-based+assays+in+breast+cancer%3A+hormone+receptors%2C+HER2%2C+and+sentinel+lymph+nodes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Although more comprehensive genomic techniques are used in the advanced disease setting, the interpretation and clinical implementation of the resulting data have proved to be challenging, with many recurrent mutations currently undruggable (<div class="dropBlock reference-citations"><a href="#bib32" id="back-bib32" data-db-target-for="bib32-d13420944e1653" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib32">Condorelli et al., 2019</a><div data-db-target-of="bib32-d13420944e1653" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Condorelli R. </li><li class="loa__item">Mosele F. </li><li class="loa__item">Verret B. </li><li class="loa__item">Bachelot T. </li><li class="loa__item">Bedard P.L. </li><li class="loa__item">Cortes J. </li><li class="loa__item">Hyman D.M. </li><li class="loa__item">Juric D. </li><li class="loa__item">Krop I. </li><li class="loa__item">Bieche I. </li><li>et al.</li></ul><div class="reference-citations__detials">Genomic alterations in breast cancer: level of evidence for actionability according to ESMO Scale for Clinical Actionability of molecular Targets (ESCAT).<div class="popup-ref"><span class="ref__series"><i>Ann. Oncol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">365-373</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fannonc%2Fmdz036&amp;cf=abstract&amp;site=annonc-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fannonc%2Fmdz036&amp;cf=fulltext&amp;site=annonc-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fannonc%2Fmdz036&amp;cf=pdf&amp;site=annonc-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30715161&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85064114526&amp;cf=" onclick="newWindow(this.href);return false">Scopus (40)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2019&pages=365-373&journal=Ann.+Oncol.&author=Condorelli+R.&author=Mosele+F.&author=Verret+B.&author=Bachelot+T.&author=Bedard+P.L.&author=Cortes+J.&author=Hyman+D.M.&author=Juric+D.&author=Krop+I.&author=Bieche+I.&title=Genomic+alterations+in+breast+cancer%3A+level+of+evidence+for+actionability+according+to+ESMO+Scale+for+Clinical+Actionability+of+molecular+Targets+%28ESCAT%29">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Furthermore, recently introduced treatments, such as CDK4/6 and immune checkpoint inhibitors, do not have robust predictive biomarkers, which can lead to missed therapeutic opportunities and overtreatment (<div class="dropBlock reference-citations"><a href="#bib95" id="back-bib95" data-db-target-for="bib95-d13420944e1657" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib95">O’Leary et al., 2016</a><div data-db-target-of="bib95-d13420944e1657" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">O’Leary B. </li><li class="loa__item">Finn R.S. </li><li class="loa__item">Turner N.C. </li></ul><div class="reference-citations__detials">Treating cancer with selective CDK4/6 inhibitors.<div class="popup-ref"><span class="ref__series"><i>Nat. Rev. Clin. Oncol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">417-430</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnrclinonc.2016.26&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27030077&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2016&pages=417-430&journal=Nat.+Rev.+Clin.+Oncol.&author=O%E2%80%99Leary+B.&author=Finn+R.S.&author=Turner+N.C.&title=Treating+cancer+with+selective+CDK4%2F6+inhibitors">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib117" id="back-bib117" data-db-target-for="bib117-d13420944e1660" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib117">Shindo et al., 2019</a><div data-db-target-of="bib117-d13420944e1660" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Shindo Y. </li><li class="loa__item">Hazama S. </li><li class="loa__item">Tsunedomi R. </li><li class="loa__item">Suzuki N. </li><li class="loa__item">Nagano H. </li></ul><div class="reference-citations__detials">Novel Biomarkers for Personalized Cancer Immunotherapy.<div class="popup-ref"><span class="ref__series"><i>Cancers (Basel).</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">1223</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.3390%2Fcancers11091223&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85071874767&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=1223&journal=Cancers+%28Basel%29&author=Shindo+Y.&author=Hazama+S.&author=Tsunedomi+R.&author=Suzuki+N.&author=Nagano+H.&title=Novel+Biomarkers+for+Personalized+Cancer+Immunotherapy">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div><div class="section-paragraph">Proteogenomics is an approach to tumor profiling that combines next-generation DNA and RNA sequencing with mass spectrometry-based proteomics to provide deep, unbiased quantification of proteins and post-translational modifications such as phosphorylation (<div class="dropBlock reference-citations"><a href="#bib111" id="back-bib111" data-db-target-for="bib111-d13420944e1665" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib111">Ruggles et al., 2017</a><div data-db-target-of="bib111-d13420944e1665" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Krug K. </li><li class="loa__item">Wang X. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Wang J. </li><li class="loa__item">Payne S.H. </li><li class="loa__item">Fenyö D. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Mani D.R. </li></ul><div class="reference-citations__detials">Methods, tools and current perspectives in proteogenomics.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">959-981</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.MR117.000024&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.MR117.000024&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.MR117.000024&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=28456751&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85020476458&amp;cf=" onclick="newWindow(this.href);return false">Scopus (68)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=959-981&journal=Mol.+Cell.+Proteomics&author=Ruggles+K.V.&author=Krug+K.&author=Wang+X.&author=Clauser+K.R.&author=Wang+J.&author=Payne+S.H.&author=Feny%C3%B6+D.&author=Zhang+B.&author=Mani+D.R.&title=Methods%2C+tools+and+current+perspectives+in+proteogenomics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The Clinical Proteomic Tumor Analysis Consortium (CPTAC) seeks to perform deep-scale proteogenomics profiling across multiple cancer types. Our initial proteogenomics analysis of BRCA using residual samples from The Cancer Genome Atlas (TCGA) provided proof of principle that proteogenomics represented an advance in BRCA profiling (<div class="dropBlock reference-citations"><a href="#bib85" id="back-bib85" data-db-target-for="bib85-d13420944e1668" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib85">Mertins et al., 2016</a><div data-db-target-of="bib85-d13420944e1668" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Wang P. </li><li class="loa__item">Wang X. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Cao S. </li><li class="loa__item">Petralia F. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomics connects somatic mutations to signalling in breast cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (780)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=NCI+CPTAC&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). However, the relatively small number of TCGA samples with sufficient material for deep proteomics represented different fragments from those used for genomics, did not uniformly pass proteomics quality assessment, and were not collected using protocols designed to preserve post-translational modifications (<div class="dropBlock reference-citations"><a href="#bib84" id="back-bib84" data-db-target-for="bib84-d13420944e1671" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib84">Mertins et al., 2014</a><div data-db-target-of="bib84-d13420944e1671" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Yang F. </li><li class="loa__item">Liu T. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Gritsenko M.A. </li><li class="loa__item">Moore R.J. </li><li>et al.</li></ul><div class="reference-citations__detials">Ischemia in tumors induces early and sustained phosphorylation changes in stress kinase pathways but does not affect global protein levels.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">1690-1704</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.M113.036392&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.M113.036392&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.M113.036392&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=24719451&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84904111890&amp;cf=" onclick="newWindow(this.href);return false">Scopus (222)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=1690-1704&journal=Mol.+Cell.+Proteomics&author=Mertins+P.&author=Yang+F.&author=Liu+T.&author=Mani+D.R.&author=Petyuk+V.A.&author=Gillette+M.A.&author=Clauser+K.R.&author=Qiao+J.W.&author=Gritsenko+M.A.&author=Moore+R.J.&title=Ischemia+in+tumors+induces+early+and+sustained+phosphorylation+changes+in+stress+kinase+pathways+but+does+not+affect+global+protein+levels">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Here we describe proteogenomics characterization of the largest cohort to date of BRCA samples that were acquired to minimize ischemic time, maximizing fidelity and reducing preanalytical variability. We offer the first comprehensive report of the BRCA acetylome; present testable hypotheses regarding therapeutic vulnerabilities, cancer biology, and advancement of diagnostic standards; and provide an extensive resource to stimulate further discovery.</div></section><section id="sec2" data-locator="sec2" tabindex="0"><h2 class="top" id="secsectitle0035" data-left-hand-nav="Results" tabindex="0"><span class="top__sub"></span><span class="top__text">Results</span></h2><section id="sec2.1" data-locator="sec2.1" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Proteogenomic (PG) Characterization of Prospectively Collected Breast Tumors</span></h3><div class="section-paragraph">We prospectively collected treatment-naive primary tumors under a stringent protocol that controlled tissue ischemia (<a href="/cms/10.1016/j.cell.2020.10.036/attachment/3dbb5a07-b92f-4ae5-9c88-f84c3dca95db/mmc1" class="scroll-into-link">Table S1</a>) to preserve post-translational modifications. Each tumor was cryopulverized, and DNA, RNA, and protein were extracted from the resulting single homogeneous sample (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">Figure S1</a>A). Tandem mass tag (TMT)-based isobaric labeling provided precise relative quantification of proteins and phosphorylation and acetylation sites following published CPTAC protocols (<div class="dropBlock reference-citations"><a href="#bib86" id="back-bib86" data-db-target-for="bib86-d13420944e1689" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib86">Mertins et al., 2018</a><div data-db-target-of="bib86-d13420944e1689" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Tang L.C. </li><li class="loa__item">Krug K. </li><li class="loa__item">Clark D.J. </li><li class="loa__item">Gritsenko M.A. </li><li class="loa__item">Chen L. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Clauss T.R. </li><li class="loa__item">Shah P. </li><li class="loa__item">Gillette M.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Reproducible workflow for multiplexed deep-scale proteome and phosphoproteome analysis of tumor tissues by liquid chromatography-mass spectrometry.<div class="popup-ref"><span class="ref__series"><i>Nat. Protoc.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">1632-1661</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41596-018-0006-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29988108&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85049660247&amp;cf=" onclick="newWindow(this.href);return false">Scopus (121)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=1632-1661&journal=Nat.+Protoc.&author=Mertins+P.&author=Tang+L.C.&author=Krug+K.&author=Clark+D.J.&author=Gritsenko+M.A.&author=Chen+L.&author=Clauser+K.R.&author=Clauss+T.R.&author=Shah+P.&author=Gillette+M.A.&title=Reproducible+workflow+for+multiplexed+deep-scale+proteome+and+phosphoproteome+analysis+of+tumor+tissues+by+liquid+chromatography-mass+spectrometry">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>A; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">Figures S1</a>A and S1B). Stringent criteria for protein identification and quantification resulted in high data quality across 15 tumor TMT-plexes (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">Figures S1</a>C–S1E) connected by a common reference sample (<a href="#sec4" class="scroll-into-link">STAR Methods</a>). Notably, all tumors passed post-data acquisition quality control (QC) metrics for proteomics analysis (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">Figures S1</a>F–S1H), an improvement compared with our previous study (<div class="dropBlock reference-citations"><a href="#bib85" id="back-bib85" data-db-target-for="bib85-d13420944e1709" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib85">Mertins et al., 2016</a><div data-db-target-of="bib85-d13420944e1709" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Wang P. </li><li class="loa__item">Wang X. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Cao S. </li><li class="loa__item">Petralia F. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomics connects somatic mutations to signalling in breast cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (780)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=NCI+CPTAC&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Longitudinal data quality and reproducibility were demonstrated across several months of mass spectrometry data acquisition by periodic analysis of full-process replicates of a proteomics comparative reference sample (CompRef; <div class="dropBlock reference-citations"><a href="#bib86" id="back-bib86" data-db-target-for="bib86-d13420944e1712" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib86">Mertins et al., 2018</a><div data-db-target-of="bib86-d13420944e1712" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Tang L.C. </li><li class="loa__item">Krug K. </li><li class="loa__item">Clark D.J. </li><li class="loa__item">Gritsenko M.A. </li><li class="loa__item">Chen L. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Clauss T.R. </li><li class="loa__item">Shah P. </li><li class="loa__item">Gillette M.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Reproducible workflow for multiplexed deep-scale proteome and phosphoproteome analysis of tumor tissues by liquid chromatography-mass spectrometry.<div class="popup-ref"><span class="ref__series"><i>Nat. Protoc.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">1632-1661</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41596-018-0006-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29988108&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85049660247&amp;cf=" onclick="newWindow(this.href);return false">Scopus (121)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=1632-1661&journal=Nat.+Protoc.&author=Mertins+P.&author=Tang+L.C.&author=Krug+K.&author=Clark+D.J.&author=Gritsenko+M.A.&author=Chen+L.&author=Clauser+K.R.&author=Clauss+T.R.&author=Shah+P.&author=Gillette+M.A.&title=Reproducible+workflow+for+multiplexed+deep-scale+proteome+and+phosphoproteome+analysis+of+tumor+tissues+by+liquid+chromatography-mass+spectrometry">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and by assessment of inter-plex common reference and replicate sample reproducibility (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">Figures S1</a>I–S1L). Across the dataset, appropriate filtering (<a href="#sec4" class="scroll-into-link">STAR Methods</a>) yielded identification of 29,647 somatic mutations, 23,692 gene-level copy number events, 23,121 gene transcripts, 10,107 proteins, 38,968 phosphorylation sites, and 9,869 acetylation sites (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>A; <a href="/cms/10.1016/j.cell.2020.10.036/attachment/a1bc3f3e-eeac-4605-a9ef-839abd334dcd/mmc2" class="scroll-into-link">Table S2</a>).<div class="floatDisplay"><figure id="figs1" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/12d0e07d-77c7-4c9b-a392-dbf61765d93c/figs1.jpg" alt="Figure thumbnail figs1" data-locator="figs1" data-high-res="/cms/attachment/1e647a06-1549-4892-a267-6f48d5af3d00/figs1_lrg.jpg" data-large-image="/cms/attachment/12d0e07d-77c7-4c9b-a392-dbf61765d93c/figs1.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. S1" class="label figure__label">Figure S1</span><span class="figure__title__text">Experimental Workflow and Data Quality Metrics, Related to <a data-locator="fig1" aria-expanded="false" aria-controls="crosref2570" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig1">Figure 1</a></span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">A) Schematic representation of sample processing steps. Fresh frozen tumors were cryopulverized, aliquoted and subjected to genomics and proteomics analyses. Cryopulverization promoted sample uniformity across multi-omics analyses.</div><div class="section-paragraph">B) Schematic representation of the workflow used for proteome, phosphoproteome and acetylproteome analyses. Tandem mass tags (TMT) were used to multiplex 9 tumors and the common reference in a single TMT10-plex. The common reference was constructed from 40 tumors based on hormone receptor status, including 9 triple negative, 12 HER2 positive, and 19 estrogen receptor positive specimens, and was used to connect the TMT-plexes (<a data-locator="sec4" aria-expanded="false" aria-controls="crosref2575" class=" caption__link scroll-into-link" href="&#xA;                                #sec4">STAR Methods</a>), enabling consistent identification and reproducible quantification of proteins, phosphorylation and acetylation sites.</div><div class="section-paragraph">C) Bar chart depicting the number of quantified proteins per channel per TMT plex. Alternating blocks represent all channels within the indicated plex. Red points and connecting lines show the median log2 ratio per channel (right axis). The blue boxes highlight plexes 14 and 15, which comprised normal adjacent tissue (NAT) and were excluded from further analysis chiefly for biological but also for QC reasons (<a data-locator="sec4" aria-expanded="false" aria-controls="crosref2580" class=" caption__link scroll-into-link" href="&#xA;                                #sec4">STAR Methods</a>).</div><div class="section-paragraph">D) Same layout as C) showing phosphosites.</div><div class="section-paragraph">E) Same layout as C) showing acetylation sites. Consistent with the low acetyl-enrichment yield derived from plex 11 (<a data-locator="sec4" aria-expanded="false" aria-controls="crosref2585" class=" caption__link scroll-into-link" href="&#xA;                                #sec4">STAR Methods</a>, “<a data-locator="sec4.5.2.2" aria-expanded="false" aria-controls="crosref2590" class=" caption__link scroll-into-link" href="&#xA;                                #sec4.5.2.2">PSM quality control</a> and <a data-locator="sectitle0265" aria-expanded="false" aria-controls="crosref2595" class=" caption__link scroll-into-link" href="&#xA;                                #sectitle0265">Quantification using TMT ratios</a>”), the number of quantified acetylation sites was significantly lower in this plex.</div><div class="section-paragraph">F) Profile plots of two-component normalized protein / common reference (CR) ratios.</div><div class="section-paragraph">G) Profile plots of phosphosite / CR ratios.</div><div class="section-paragraph">H) Profile plots of acetylsite / CR ratios.</div><div class="section-paragraph">I) Pairwise inter-plex correlations between phosphopeptide-level log reporter ion intensities of the common reference channel (131), measured in all 17 TMT-10 plexes. Lower triangle: pairwise scatterplots; diagonal: histograms depicting distribution of log intensities; upper triangle: Pearson correlation calculated on pairwise complete observations of log-transformed intensities. Red histograms depict TMT plexes 14 and 15, which contained normal adjacent tissue (NAT) and were excluded from the datasets used in the manuscript. Blue histograms correspond to 15 tumor plexes. Average inter-plex Pearson correlation on the phosphopeptide level across all tumor plexes was 0.69.</div><div class="section-paragraph">J) Boxplots summarizing pairwise inter-plex Pearson correlations between log reporter ion intensities of the common reference channel (131) across 15 tumor TMT-plexes on protein, phosphopeptide and acetylpeptide levels. Boxes depict the interquartile range (IQR) of the scores with horizontal lines depicting the median. Whiskers extend to 1.5xIQR from Q1 (25th percentile) and Q3 (75th percentile), respectively. Points depict individual data points.Isobaric-label LC-MS/MS experiments are purpose-built for a ratio-based quantification strategy. After samples are mixed to form a plex, the peptides from all samples within that plex should experience the same effects due to subsequent experimental sources of variation (offline basic-pH reversed-phase fractionation, phosphopeptide or acetylpeptide enrichment, on-line reversed-phase fractionation, selection of precursor ions for MS/MS, and mass spectrometer signal strength). While these collective effects may lead to substantial inter-plex variation in peptide-level reporter ion intensities, variation of the peptide-level ratios of sample to common reference can be expected to be consistent (though subject to peptide-level missing values between plexes due to individual peptides not always being selected for MS/MS).</div><div class="section-paragraph">K) Pairwise Spearman correlations between two-component normalized sample/CR ratios of replicates scattered across ten different TMT-plexes.</div><div class="section-paragraph">L) Dendrogram illustrating the clustering of two-component normalized sample/CR phosphopeptide-level ratios based on all quantified phosphosites without additional filtering (e.g., for missing values, most variable sites, etc.; n = 63,360). The annotation tracks below the leaves of the dendrogram depict the PAM50 subtype, the TMT10-plex and color-coordinated replicate measurements (gray if no replicate measurements). Samples did not cluster by TMT10 plex, but chiefly by PAM50 subtype. Replicates measured in different TMT-plexes acquired on the mass spectrometer months apart cluster together. Dendrogram was derived from complete-linkage hierarchical clustering using 1-Pearson correlation as the distance metric.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/12d0e07d-77c7-4c9b-a392-dbf61765d93c/figs1.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/1e647a06-1549-4892-a267-6f48d5af3d00/figs1_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(20)31400-8&amp;id=figs1.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div><div class="floatDisplay"><figure id="gr1" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/fabcaf28-1b5b-4272-8cdd-d5f935f768d6/gr1.jpg" alt="Figure thumbnail gr1" data-locator="fig1" data-high-res="/cms/attachment/69cdd50e-b54b-4135-9ad4-88e3a6f5c084/gr1_lrg.jpg" data-large-image="/cms/attachment/fabcaf28-1b5b-4272-8cdd-d5f935f768d6/gr1.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 1" class="label figure__label">Figure 1</span><span class="figure__title__text">Proteogenomics (PG) Landscape of BRCA</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Schematic overview of PG data acquired for this cohort.</div><div class="section-paragraph">(B) Unsupervised multi-omics identified four molecular subtypes. Samples are ordered by cluster and membership score in decreasing order.</div><div class="section-paragraph">(C) Kaplan-Meier curves showing survival outcome of PAM50 LumA samples in the METABRIC database that were assigned by a random forest mRNA-based classifier to the NMF LumA-I (red) or LumB-I subtypes (green) compared with PAM50 LumB samples (blue). The p values were derived from log rank tests.</div><div class="section-paragraph">(D) Heatmap showing the fraction of outlier values in each sample per protein. Proteins shown are kinases highly phosphorylated in each NMF cluster with an FDR of less than 0.01 using BlackSheep. Kinases shown in bold were detected as outliers in the prior study. The top panel shows PAM50 and NMF cluster membership as well as NMF membership score. The left panel indicates whether an inhibitor can be found for a given kinase using the DGIdb (Drug Gene Interaction Database). The right panels depict the abundance of the kinase activation loop and kinase substrate enrichment.</div><div class="section-paragraph">(E) Heatmap showing q values from BlackSheep for enrichment of phosphorylation outliers (y axis) in samples with the indicated mutated gene (x axis). Numbers in parentheses indicate the number of samples in each mutational subgroup. Kinases with an FDR of less than 0.01 are shown, and light gray cells indicate kinases that did not show enrichment (FDR ≥ 0.01).</div><div class="section-paragraph">See also <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">Figure S1</a>, <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figure S2</a>, <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">Figure S3</a> and <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref2425" href="/cms/10.1016/j.cell.2020.10.036/attachment/3dbb5a07-b92f-4ae5-9c88-f84c3dca95db/mmc1" target="_blank">Tables S1</a>, <a data-locator="mmc2" aria-expanded="false" aria-controls="crosref2430" href="/cms/10.1016/j.cell.2020.10.036/attachment/a1bc3f3e-eeac-4605-a9ef-839abd334dcd/mmc2" target="_blank">S2</a>, <a data-locator="mmc3" aria-expanded="false" aria-controls="crosref2435" href="/cms/10.1016/j.cell.2020.10.036/attachment/3870b80d-1b59-4995-a804-71c49e83c458/mmc3" target="_blank">S3</a>, <a data-locator="mmc4" aria-expanded="false" aria-controls="crosref2440" href="/cms/10.1016/j.cell.2020.10.036/attachment/d7a1a20c-1c01-4f40-98d9-81f918417a6d/mmc4" target="_blank">S4</a>, and <a data-locator="mmc5" aria-expanded="false" aria-controls="crosref2445" href="/cms/10.1016/j.cell.2020.10.036/attachment/48d23c21-8705-4760-9513-c5982cc559ee/mmc5" target="_blank">S5</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/fabcaf28-1b5b-4272-8cdd-d5f935f768d6/gr1.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/69cdd50e-b54b-4135-9ad4-88e3a6f5c084/gr1_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(20)31400-8&amp;id=gr1.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">The PAM50 model was applied to RNA sequencing (RNA-seq) data to determine representation of intrinsic subtypes (<div class="dropBlock reference-citations"><a href="#bib96" id="back-bib96" data-db-target-for="bib96-d13420944e1730" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib96">Parker et al., 2009</a><div data-db-target-of="bib96-d13420944e1730" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Parker J.S. </li><li class="loa__item">Mullins M. </li><li class="loa__item">Cheang M.C.U. </li><li class="loa__item">Leung S. </li><li class="loa__item">Voduc D. </li><li class="loa__item">Vickery T. </li><li class="loa__item">Davies S. </li><li class="loa__item">Fauron C. </li><li class="loa__item">He X. </li><li class="loa__item">Hu Z. </li><li>et al.</li></ul><div class="reference-citations__detials">Supervised risk predictor of breast cancer based on intrinsic subtypes.<div class="popup-ref"><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">27</span></span>: <span class="ref__seriesPages">1160-1167</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1200%2FJCO.2008.18.1370&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=19204204&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-62449131093&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2602)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2009&pages=1160-1167&journal=J.%C2%A0Clin.+Oncol.&author=Parker+J.S.&author=Mullins+M.&author=Cheang+M.C.U.&author=Leung+S.&author=Voduc+D.&author=Vickery+T.&author=Davies+S.&author=Fauron+C.&author=He+X.&author=Hu+Z.&title=Supervised+risk+predictor+of+breast+cancer+based+on+intrinsic+subtypes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <a href="#tbl1" id="back-tbl1" class="scroll-into-link">Table 1</a>). Somatic mutation profiles and subtype/somatic mutation associations were consistent with previous reports (<div class="dropBlock reference-citations"><a href="#bib20" id="back-bib20" data-db-target-for="bib20-d13420944e1737" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib20">Cancer Genome Atlas Network, 2012</a><div data-db-target-of="bib20-d13420944e1737" class="dropBlock__holder"><div class="dropBlock__body"><div class="reference-citations__detials"><span>Cancer Genome Atlas Network</span><br />Comprehensive molecular portraits of human breast tumours.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">490</span></span>: <span class="ref__seriesPages">61-70</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature11412&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=23000897&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84877028141&amp;cf=" onclick="newWindow(this.href);return false">Scopus (7217)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=490&publication_year=2012&pages=61-70&journal=Nature&author=Cancer+Genome+Atlas+Network&title=Comprehensive+molecular+portraits+of+human+breast+tumours">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). This BRCA cohort therefore represented a wide range of established genomic and transcriptomic features (<a href="#tbl1" id="back-tbl1" class="scroll-into-link">Table 1</a>; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figures S2</a>A and S2B). Somatic copy number alteration (SCNA) data were analyzed to detect focal and arm-level events (<div class="dropBlock reference-citations"><a href="#bib83" id="back-bib83" data-db-target-for="bib83-d13420944e1748" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib83">Mermel et al., 2011</a><div data-db-target-of="bib83-d13420944e1748" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mermel C.H. </li><li class="loa__item">Schumacher S.E. </li><li class="loa__item">Hill B. </li><li class="loa__item">Meyerson M.L. </li><li class="loa__item">Beroukhim R. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R41</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2Fgb-2011-12-4-r41&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=21527027&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-79955166265&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1334)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R41&journal=Genome+Biol.&author=Mermel+C.H.&author=Schumacher+S.E.&author=Hill+B.&author=Meyerson+M.L.&author=Beroukhim+R.&author=Getz+G.&title=GISTIC2.0+facilitates+sensitive+and+confident+localization+of+the+targets+of+focal+somatic+copy-number+alteration+in+human+cancers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figures S2</a>C and S2D) with confirmation of anticipated effects on mRNA and protein abundance (<div class="dropBlock reference-citations"><a href="#bib45" id="back-bib45" data-db-target-for="bib45-d13420944e1754" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib45">Gillette et al., 2020</a><div data-db-target-of="bib45-d13420944e1754" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gillette M.A. </li><li class="loa__item">Satpathy S. </li><li class="loa__item">Cao S. </li><li class="loa__item">Dhanasekaran S.M. </li><li class="loa__item">Vasaikar S.V. </li><li class="loa__item">Krug K. </li><li class="loa__item">Petralia F. </li><li class="loa__item">Li Y. </li><li class="loa__item">Liang W.-W. </li><li class="loa__item">Reva B. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Clinical Proteomic Tumor Analysis Consortium</span><br />Proteogenomic Characterization Reveals Therapeutic Vulnerabilities in Lung Adenocarcinoma.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">182</span></span>: <span class="ref__seriesPages">200-225.e35</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2020.06.013&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2020.06.013&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2020.06.013&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=32649874&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85087400742&amp;cf=" onclick="newWindow(this.href);return false">Scopus (76)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=182&publication_year=2020&pages=200-225.e35&journal=Cell&author=Clinical+Proteomic+Tumor+Analysis+Consortium&title=Proteogenomic+Characterization+Reveals+Therapeutic+Vulnerabilities+in+Lung+Adenocarcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib85" id="back-bib85" data-db-target-for="bib85-d13420944e1757" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib85">Mertins et al., 2016</a><div data-db-target-of="bib85-d13420944e1757" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Wang P. </li><li class="loa__item">Wang X. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Cao S. </li><li class="loa__item">Petralia F. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomics connects somatic mutations to signalling in breast cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (780)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=NCI+CPTAC&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib147" id="back-bib147" data-db-target-for="bib147-d13420944e1760" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib147">Zhang et al., 2014</a><div data-db-target-of="bib147-d13420944e1760" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang B. </li><li class="loa__item">Wang J. </li><li class="loa__item">Wang X. </li><li class="loa__item">Zhu J. </li><li class="loa__item">Liu Q. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Chambers M.C. </li><li class="loa__item">Zimmerman L.J. </li><li class="loa__item">Shaddox K.F. </li><li class="loa__item">Kim S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomic characterization of human colon and rectal cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">513</span></span>: <span class="ref__seriesPages">382-387</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature13438&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=25043054&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84907269780&amp;cf=" onclick="newWindow(this.href);return false">Scopus (797)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2014&pages=382-387&journal=Nature&author=NCI+CPTAC&title=Proteogenomic+characterization+of+human+colon+and+rectal+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Summaries of the results of these integrative multi-omics analytic approaches are provided (<a href="#tbl1" id="back-tbl1" class="scroll-into-link">Table 1</a>; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figures S2</a>E–S2G; <a href="/cms/10.1016/j.cell.2020.10.036/attachment/3870b80d-1b59-4995-a804-71c49e83c458/mmc3" class="scroll-into-link">Table S3</a>).<div class="floatDisplay"><div id="tbl1" class="inline-table" tabindex="0"><div class="inline-table__head"><span class="inline-table__label">Table 1</span><span class="inline-table__title">Summary and Assessment of the Sample Cohort.</span></div><table width="100%" cellpadding="0" cellspacing="0" border="0" class="scrollable bordered" tabindex="0"><tbody><tr><td>Tumor samples</td><td>134 prospectively collected tumors<br></br>125 tumors subjected to proteomic analysis<br></br>3 tumors excluded due to low quality RNA-seq data<br></br>122 tumors fully analyzed</td></tr><tr><td>PAM50 classification (<a href="/cms/10.1016/j.cell.2020.10.036/attachment/3dbb5a07-b92f-4ae5-9c88-f84c3dca95db/mmc1" class="scroll-into-link">Table S1</a>A)</td><td>HER2-enriched: 11.5%<br></br>Basal-like: 23.8%<br></br>LumA: 46.7%<br></br>LumB: 13.9% <br></br>Normal-like: 4.1%</td></tr><tr><td>SMGs landscape (MutSig2CV Q &lt; 0.1;<br></br><a href="/cms/10.1016/j.cell.2020.10.036/attachment/3dbb5a07-b92f-4ae5-9c88-f84c3dca95db/mmc1" class="scroll-into-link">Table S1</a>A; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figure S2</a>A</td><td><em>TP53</em> (43%), <em>PIK3CA</em> (33%), <em>MAP3K1</em> (9%), <em>GATA3</em> (7%), <em>PTEN</em> (7%), <em>AKT1</em> (4%)</td></tr><tr><td>Mutational signature analysis (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figure S2</a>B)</td><td>W1: chewing_tobacco (COSMIC 29)<br></br>W2: aging (COSMIC 1)<br></br>W3/W7: BRCA_Hrdefect (COSMIC 3)<br></br>W4: UV (COSMIC 7)<br></br>W5: CT_APOBEC (COSMIC 2)<br></br>W7: MSI (COSMIC 6)<br></br>Notes: One sample classified as basal was characterized by an extraordinarily high number of mutations comprising a dominant UV signature, raising the possibility that it might have been a metastatic melanoma, although the <em>BRAF</em> mutation present (F707I) was not pathognomonic.</td></tr><tr><td>SNCA landscape (GISTIC2 Q &lt; 0.25)<br></br>(<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figures S2</a>C and S2D;<br></br><a href="/cms/10.1016/j.cell.2020.10.036/attachment/3870b80d-1b59-4995-a804-71c49e83c458/mmc3" class="scroll-into-link">Table S3</a>A)</td><td>Arm-level amplifications 1q, 3q, 8p, 8q,16p, 20p, 20q<br></br>Arm-level deletions: 4q, 8p, 13q, 14q, 15q, 16p, 16q, 17p, 17q, 18p, 18q, 19p, 19q, 22q</td></tr><tr><td>SCNA to protein/mRNA correlation (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figure S2</a>E)</td><td>Pairwise correlations of SCNAs with mRNA and protein abundances in <em>cis</em> (within a locus) and in <em>trans</em> (across the genome) confirmed the characteristic <em>trans</em> effects of 5q and 16q reported previously (<div class="dropBlock reference-citations"><a href="#bib85" id="back-bib85" data-db-target-for="bib85-d13420944e823" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib85">Mertins et al., 2016</a><div data-db-target-of="bib85-d13420944e823" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Wang P. </li><li class="loa__item">Wang X. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Cao S. </li><li class="loa__item">Petralia F. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomics connects somatic mutations to signalling in breast cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (780)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=NCI+CPTAC&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</td></tr><tr><td>LINCS CMAP analysis (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figure S2</a>F; <a href="/cms/10.1016/j.cell.2020.10.036/attachment/3870b80d-1b59-4995-a804-71c49e83c458/mmc3" class="scroll-into-link">Table S3</a>B)</td><td>21 candidate driver genes, located in chromosomes 1, 5q, 6p, 7q, 8q, 10p, 13q and 16q.<br></br>Notes: Candidate driver genes differed from those reported previously (<div class="dropBlock reference-citations"><a href="#bib85" id="back-bib85" data-db-target-for="bib85-d13420944e839" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib85">Mertins et al., 2016</a><div data-db-target-of="bib85-d13420944e839" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Wang P. </li><li class="loa__item">Wang X. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Cao S. </li><li class="loa__item">Petralia F. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomics connects somatic mutations to signalling in breast cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (780)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=NCI+CPTAC&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), as expected, given a different technology platform for copy number data generation and significant changes in the underlying LINCS database and calculation of connectivity scores (<a href="#sec4" class="scroll-into-link">STAR Methods</a>).</td></tr><tr><td>mRNA-protein correlation (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figure S2</a>G;<br></br><a href="/cms/10.1016/j.cell.2020.10.036/attachment/3870b80d-1b59-4995-a804-71c49e83c458/mmc3" class="scroll-into-link">Table S3</a>C)</td><td>Number of RNA-protein pairs (gene level): 9,108; median r: 0.41<br></br>Number of significant correlations (FDR &lt; 0.01): 6,609; median r: 0.51</td></tr><tr><td>Proteogenomic events with MS/MS support (<a href="/cms/10.1016/j.cell.2020.10.036/attachment/3870b80d-1b59-4995-a804-71c49e83c458/mmc3" class="scroll-into-link">Tables S3</a>D and S3E)</td><td>3,444 single amino acid variants:<br></br>238 somatic<br></br>3,206 germline<br></br>891 alternative splice forms</td></tr></tbody></table><div class="inline-table__tail"></div><ul class="rlist--inline separator download-links"><li><a title="Open Full Table" class="table__open__ctrl" target="_blank" href="/action/showFullTableHTML?isHtml=true&amp;tableId=tbl1&amp;pii=S0092-8674%2820%2931400-8">
                            Open table in a new tab
                        </a></li></ul></div></div><div class="floatDisplay"><figure id="figs2" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/763a337a-0623-4d1a-9dcf-d8b8500f6835/figs2.jpg" alt="Figure thumbnail figs2" data-locator="figs2" data-high-res="/cms/attachment/63a3fab4-8e53-4a98-8db6-ebd41c82cae0/figs2_lrg.jpg" data-large-image="/cms/attachment/763a337a-0623-4d1a-9dcf-d8b8500f6835/figs2.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. S2" class="label figure__label">Figure S2</span><span class="figure__title__text">PG Landscape, Related to <a data-locator="tbl1" aria-expanded="false" aria-controls="crosref2600" class=" caption__link scroll-into-link" href="&#xA;                                #tbl1">Table 1</a></span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">A) CoMut (co-occurrence of mutations) plot summarizing the mutational landscape of this cohort. Each column represents a tumor sample. The top histogram shows the mutational burden of each tumor. The matrix shows mutation types of significantly (q &lt; 0.1) mutated genes (SMG) identified by MutSig2CV. Plots below the matrix visualize the allelic fraction and substitution frequencies of somatic mutations. The bar chart to the left indicates the percentage of samples with each SMG, mutation number and type (synonymous or non-synonymous); the one to the right shows the significance of that SMG in the cohort.</div><div class="section-paragraph">B) Mutational signatures detected in this cohort. Trinucleotide sequence motif: seven distinct mutational signatures were identified (<a data-locator="tbl1" aria-expanded="false" aria-controls="crosref2605" class=" caption__link scroll-into-link" href="&#xA;                                #tbl1">Table 1</a>).</div><div class="section-paragraph">C) Copy number amplifications and deletions analysis. The left panel depicts genomic positions of amplified regions, with x-axes representing the normalized amplification signals (top) and significance by Q value (bottom). The right panel shows genomic positions of deleted regions, with X-axes representing the normalized deletion signals (top) and significance by Q value (bottom). The green lines represent the significance cutoff at Q value = 0.25.</div><div class="section-paragraph">D) Heatmap depicting all detected somatic copy number aberrations detected in this cohort. Genes are ordered by chromosomal location (y axis) and samples are organized by PAM50 subtype and separately clustered for each subtype (x axis).</div><div class="section-paragraph">E) Correlations of SNCA (x axis) to mRNA (left) and protein (right) abundances (y axis).</div><div class="section-paragraph">F) Results of the CMAP analysis. Potential driver genes (permutation FDR &lt; 0.26) of observed CNA correlations identified using LINCS/CMAP connectivity analysis of proteome profiles from CNA amplified (AMP) or deleted (DEL) samples. The bar plots show, for each potential driver gene, the total number of statistically significant <em>trans</em> correlations (FDR &lt; 0.05), along with the number of <em>trans</em>-correlated genes (“cmap overlap”) that are also outliers (absolute value of z-score &gt; 1.96) in the LINCS/CMAP knockdown profiles. Sample annotations enriched in the sample subsets representing each potential driver gene are summarized in <a data-locator="mmc3" aria-expanded="false" aria-controls="crosref2610" href="/cms/10.1016/j.cell.2020.10.036/attachment/3870b80d-1b59-4995-a804-71c49e83c458/mmc3" target="_blank">Table S3</a>.</div><div class="section-paragraph">G) Histograms of gene-wise mRNA to protein Pearson correlations of all (gray) and significant correlations (FDR &lt; 0.01) (red).</div><div class="section-paragraph">H) Multi-omic landscape of key BRCA genes, TP53, PIK3CA, ERBB2, ESR1, PGR and GATA3. The heatmap has been generated with the CPTAC-BRCA2020 data viewer (see Data and Code Availability) and resembles <a data-locator="fig1" aria-expanded="false" aria-controls="crosref2615" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig1">Figure 1</a>c in the retrospective BRCA study (<div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib85" id="back-bib85" data-db-target-for="bib85-d13420944e332" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib85">Mertins et al., 2016</a><div data-db-target-of="bib85-d13420944e332" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Wang P. </li><li class="loa__item">Wang X. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Cao S. </li><li class="loa__item">Petralia F. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomics connects somatic mutations to signalling in breast cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (780)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=NCI+CPTAC&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div>).</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/763a337a-0623-4d1a-9dcf-d8b8500f6835/figs2.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/63a3fab4-8e53-4a98-8db6-ebd41c82cae0/figs2_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(20)31400-8&amp;id=figs2.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div></section><section id="sec2.2" data-locator="sec2.2" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Non-negative Matrix Factorization-Based Multi-omics Classification of BRCA</span></h3><div class="section-paragraph">To explore intrinsic cohort structure using the full complement of proteogenomics data, single-omic and multi-omics clustering were performed for SCNA, mRNA, protein, and individual phosphosite and acetylation site abundance using non-negative matrix factorization (NMF) (<div class="dropBlock reference-citations"><a href="#bib65" id="back-bib65" data-db-target-for="bib65-d13420944e1778" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib65">Lee and Seung, 1999</a><div data-db-target-of="bib65-d13420944e1778" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lee D.D. </li><li class="loa__item">Seung H.S. </li></ul><div class="reference-citations__detials">Learning the parts of objects by non-negative matrix factorization.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">1999; </span><span class="ref__seriesVolume">401</span></span>: <span class="ref__seriesPages">788-791</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2F44565&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10548103&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-0033592606&amp;cf=" onclick="newWindow(this.href);return false">Scopus (8387)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=401&publication_year=1999&pages=788-791&journal=Nature&author=Lee+D.D.&author=Seung+H.S.&title=Learning+the+parts+of+objects+by+non-negative+matrix+factorization">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#bib66" id="back-bib66" data-db-target-for="bib66-d13420944e1781" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib66">Lee and Seung, 2001</a><div data-db-target-of="bib66-d13420944e1781" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lee D.D. </li><li class="loa__item">Seung H.S. </li></ul><div class="reference-citations__detials">Algorithms for Non-negative Matrix Factorization.<div class="popup-ref">in: <span class="editors">Leen T.K. Dietterich T.G. Tresp V. </span>Advances in Neural Information Processing Systems 13.  <span class="refPublisherName">MIT Press</span>,
        <span class="refPublisherLoc"></span><span class="refDate">2001</span>: <span class="ref__seriesPages">556-562</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Lee+D.D.+Seung+H.S.+Algorithms+for+Non-negative+Matrix+Factorization.in%3A+Leen+T.K.+Dietterich+T.G.+Tresp+V.+Advances+in+Neural+Information+Processing+Systems+13.+MIT+Press%2C%0A+2001%3A+556-562">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Although NMF yielded between two and six clusters in single-omic analyses (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">Figure S3</a>A), integrative multi-omics analysis converged on four NMF clusters, with cluster membership scores indicating the strength of association of each sample with a given cluster (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>B; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">Figures S3</a>B and S3C; <a href="/cms/10.1016/j.cell.2020.10.036/attachment/d7a1a20c-1c01-4f40-98d9-81f918417a6d/mmc4" class="scroll-into-link">Table S4</a>). Clusters designated luminal A-inclusive (NMF LumA-I) and basal-inclusive (NMF Basal-I) were almost entirely composed of tumors with the corresponding PAM50 assignments. Thus, these samples represent the extremes of the BRCA intrinsic subtype classification (<div class="dropBlock reference-citations"><a href="#bib96" id="back-bib96" data-db-target-for="bib96-d13420944e1798" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib96">Parker et al., 2009</a><div data-db-target-of="bib96-d13420944e1798" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Parker J.S. </li><li class="loa__item">Mullins M. </li><li class="loa__item">Cheang M.C.U. </li><li class="loa__item">Leung S. </li><li class="loa__item">Voduc D. </li><li class="loa__item">Vickery T. </li><li class="loa__item">Davies S. </li><li class="loa__item">Fauron C. </li><li class="loa__item">He X. </li><li class="loa__item">Hu Z. </li><li>et al.</li></ul><div class="reference-citations__detials">Supervised risk predictor of breast cancer based on intrinsic subtypes.<div class="popup-ref"><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">27</span></span>: <span class="ref__seriesPages">1160-1167</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1200%2FJCO.2008.18.1370&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=19204204&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-62449131093&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2602)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2009&pages=1160-1167&journal=J.%C2%A0Clin.+Oncol.&author=Parker+J.S.&author=Mullins+M.&author=Cheang+M.C.U.&author=Leung+S.&author=Voduc+D.&author=Vickery+T.&author=Davies+S.&author=Fauron+C.&author=He+X.&author=Hu+Z.&title=Supervised+risk+predictor+of+breast+cancer+based+on+intrinsic+subtypes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>B; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">Figure S3</a>B). The NMF LumA-I cluster was enriched for hormone (estrogen and progesterone) receptor positivity and wild-type <em>TP53</em> and demonstrated high stromal infiltration scores (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">Figures S3</a>B and S3D). The NMF Basal-I cluster contained all but one PAM50 basal sample and was strongly enriched for <em>TP53</em> mutations and negative clinical hormone receptor status (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">Figure S3</a>B). Higher levels of immune, stemness, and chromosome instability (CIN) scores (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">Figures S3</a>D–S3G) as well as strong enrichment of proliferation-associated pathways such as E2F targets and the G2/M checkpoint were observed in NMF Basal-I tumors (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">Figure S3</a>H).<div class="floatDisplay"><figure id="figs3" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/640ef007-93ad-495c-929f-304ff4aea769/figs3.jpg" alt="Figure thumbnail figs3" data-locator="figs3" data-high-res="/cms/attachment/c72115b0-24fe-424c-9bf6-0e2acc67e3a0/figs3_lrg.jpg" data-large-image="/cms/attachment/640ef007-93ad-495c-929f-304ff4aea769/figs3.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. S3" class="label figure__label">Figure S3</span><span class="figure__title__text">Molecular Subtyping, Related to <a data-locator="fig1" aria-expanded="false" aria-controls="crosref2625" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig1">Figure 1</a></span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">A) Results of unsupervised non-negative matrix factorization (NMF) subtyping applied to individual data types (<a data-locator="sec4" aria-expanded="false" aria-controls="crosref2630" class=" caption__link scroll-into-link" href="&#xA;                                #sec4">STAR Methods</a>). The Sankey diagram depicts the flow of cluster assignments of all 122 tumors across data types.</div><div class="section-paragraph">B) Unsupervised multi-omics subtyping via NMF identified four molecular subtypes with distinct multi-omics expression patterns. The ability of NMF to assign weights to features that drive disease clustering (<a data-locator="sec4" aria-expanded="false" aria-controls="crosref2635" class=" caption__link scroll-into-link" href="&#xA;                                #sec4">STAR Methods</a>) allowed the identification of proteogenomic features (proteins, phosphorylation sites, acetylation sites, mRNA transcripts and SCNAs) that showed increased or decreased abundance levels specifically in each NMF cluster. The heatmap depicts the standardized abundances (z-scores) of 851 distinct features discriminating the molecular subtypes. acK = acetylated lysine sites; SCNA = somatic copy number alterations; RNA = gene-level transcripts, Prot = proteins; pSTY = phosphosites. Fisher’s exact test was used to test for enrichment of clinical variables in core sets of tumors in each cluster. The NMF LumA-I cluster was enriched (Fisher’s exact test) for hormone receptor positivity (estrogen receptor (ER), p = 2.66x10<sup>−5</sup> and progesterone receptor (PR), p = 3.55x10<sup>−6</sup>) and wild-type <em>TP53</em> (p = 8.51x10<sup>−5</sup>). The NMF Basal-I cluster was enriched for <em>TP53</em> mutations (p = 4.24x10<sup>−10</sup>) and negative clinical hormone receptor status (ER, p = 1.76x10<sup>−10</sup> and PR, p = 1.02x10<sup>−9</sup>). The NMF HER2-I was enriched for HER2-enriched PAM50 subtype samples (p = 1.38x10<sup>−8</sup>) and samples with centrally-confirmed, clinically positive ERBB2 status (p = 3.97x10<sup>−5</sup>). While NMF LumB-I and NMF LumA were only enriched for their respective PAM50 luminal subtype (p = 1.51x10<sup>−7</sup> and p = 3.95x10<sup>−11</sup>, respectively) both luminal clusters were enriched for positive receptor status (ER [p = 1.85x10<sup>−6</sup>, p = 1.85x10<sup>−5</sup>] and PR [p = 1.26x10<sup>−3</sup>, p = 3.55x10<sup>−6</sup>]) as well as wild-type <em>TP53</em> (p = 9.88x10<sup>−6</sup>, p = 8.51x10<sup>−5</sup>).</div><div class="section-paragraph">C) Graph showing cluster metrics (y axis) as a function of the number of clusters K (x axis). Cophenetic correlation (coph), dispersion of the consensus matrix (disp) and the product of both metrics are shown (product).</div><div class="section-paragraph">D) Boxplots depicting the distribution of stromal scores inferred by xCell between molecular subtypes. Boxes depict the interquartile range (IQR) of the scores with horizontal lines depicting the median. Whiskers extend to 1.5xIQR from Q1 (25th percentile) and Q3 (75th percentile), respectively. Points depict individual data points. P values were derived by pairwise Wilcoxon rank-sum tests.</div><div class="section-paragraph">E) Same layout as D), showing distribution of immune scores inferred by xCell.</div><div class="section-paragraph">F) Same layout as D), showing distribution of stemness scores.</div><div class="section-paragraph">G) Same layout as D), showing distribution of chromosomal instability index (CIN) scores.</div><div class="section-paragraph">H) Enrichment of Cancer Hallmark (<div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib72" id="back-bib72" data-db-target-for="bib72-d13420944e422" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib72">Liberzon et al., 2015</a><div data-db-target-of="bib72-d13420944e422" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liberzon A. </li><li class="loa__item">Birger C. </li><li class="loa__item">Thorvaldsdóttir H. </li><li class="loa__item">Ghandi M. </li><li class="loa__item">Mesirov J.P. </li><li class="loa__item">Tamayo P. </li></ul><div class="reference-citations__detials">The Molecular Signatures Database (MSigDB) hallmark gene set collection.<div class="popup-ref"><span class="ref__series"><i>Cell Syst.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">417-425</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26771021&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84955328286&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2256)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=417-425&journal=Cell+Syst.&author=Liberzon+A.&author=Birger+C.&author=Thorvaldsd%C3%B3ttir+H.&author=Ghandi+M.&author=Mesirov+J.P.&author=Tamayo+P.&title=The+Molecular+Signatures+Database+%28MSigDB%29+hallmark+gene+set+collection">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div>) gene sets in multi-omics subtypes detected by single sample gene set enrichment analysis (ssGSEA; <div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib10" id="back-bib10" data-db-target-for="bib10-d13420944e425" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib10">Barbie et al., 2009</a><div data-db-target-of="bib10-d13420944e425" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Barbie D.A. </li><li class="loa__item">Tamayo P. </li><li class="loa__item">Boehm J.S. </li><li class="loa__item">Kim S.Y. </li><li class="loa__item">Moody S.E. </li><li class="loa__item">Dunn I.F. </li><li class="loa__item">Schinzel A.C. </li><li class="loa__item">Sandy P. </li><li class="loa__item">Meylan E. </li><li class="loa__item">Scholl C. </li><li>et al.</li></ul><div class="reference-citations__detials">Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">462</span></span>: <span class="ref__seriesPages">108-112</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature08460&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=19847166&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-70449091786&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1353)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=462&publication_year=2009&pages=108-112&journal=Nature&author=Barbie+D.A.&author=Tamayo+P.&author=Boehm+J.S.&author=Kim+S.Y.&author=Moody+S.E.&author=Dunn+I.F.&author=Schinzel+A.C.&author=Sandy+P.&author=Meylan+E.&author=Scholl+C.&title=Systematic+RNA+interference+reveals+that+oncogenic+KRAS-driven+cancers+require+TBK1">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div>) applied to feature weights determined by NMF to identify the dominant pathway signatures in each cluster. Normalized enrichment scores (NES) are shown. Asterisks indicate gene sets detected at FDR &lt; 0.01.</div><div class="section-paragraph">I) Scatterplot comparing pathway (MSigDB Hallmark) enrichment scores between the LumA-I (x axis) and LumB-I (y axis) NMF subypes. Pathway scores were significantly anti-correlated (Pearson r = −0.58, p = 1.1x10<sup>−5</sup>).</div><div class="section-paragraph">J) NMF clustering applied to the RNA data in isolation. Heatmap depicting the relative contributions of each sample (x axis) to each cluster (y axis). This heatmap is a visualization of the (column-normalized) meta-feature matrix derived from decomposing the input matrix (<a data-locator="sec4" aria-expanded="false" aria-controls="crosref2650" class=" caption__link scroll-into-link" href="&#xA;                                #sec4">STAR Methods</a>), normalized per column by the maximum entry.</div><div class="section-paragraph">K) Kaplan-Meier plot showing survival outcome of PAM50 luminal A samples in the METABRIC database that were assigned by a Random Forest protein-based classifier to the NMF Lum-AI (red) or LumA/B subtypes (green) compared to PAM50 luminal B samples (blue) (similar to the Kaplan-Meier analysis in <a data-locator="fig1" aria-expanded="false" aria-controls="crosref2920" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig1">Figure 1</a>C, but using a protein- rather than RNA-based Random Forest classifier). The significance of comparisons is noted. P values were derived from log-rank tests.</div><div class="section-paragraph">L) Heatmap depicting standardized abundances of features defining the NMF HER2-I cluster. Features (proteins and phosphosites) are designated to the right of the heatmap in descending rank according to the number of HER2-I samples that have standardized abundances for that feature above the average for the entire tumor cohort (z-score &gt; 0). The horizontal line demarcates features up to the first occurence of an element from the <em>ERBB2</em> amplicon itself (GRB7), which are therefore labeled “ERBB2-independent.”</div><div class="section-paragraph">M) Overrepresentation analysis (hypergeometric test) of upregulated genes in the NMF HER2-I cluster that are independent of the <em>ERBB2</em> amplicon (box in D). The Manhattan-like plot depicts log-transformed FDR-corrected p values (y axis) for enriched gene ontologies (GO) terms (<div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib105" id="back-bib105" data-db-target-for="bib105-d13420944e452" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib105">Reimand et al., 2016</a><div data-db-target-of="bib105-d13420944e452" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Reimand J. </li><li class="loa__item">Arak T. </li><li class="loa__item">Adler P. </li><li class="loa__item">Kolberg L. </li><li class="loa__item">Reisberg S. </li><li class="loa__item">Peterson H. </li><li class="loa__item">Vilo J. </li></ul><div class="reference-citations__detials">g:Profiler-a web server for functional interpretation of gene lists (2016 update).<div class="popup-ref"><span class="ref__series"><i>Nucleic. Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">W83-W89</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgkw199&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27098042&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=W83-W89&journal=Nucleic.+Acids+Res.&author=Reimand+J.&author=Arak+T.&author=Adler+P.&author=Kolberg+L.&author=Reisberg+S.&author=Peterson+H.&author=Vilo+J.&title=g%3AProfiler-a+web+server+for+functional+interpretation+of+gene+lists+%282016+update%29">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div>). The dashed line indicates an FDR of 1x10<sup>−4</sup>, which was used to define statistical significance for this analysis.</div><div class="section-paragraph">N) Heatmap depicting multi-omics expression patterns of CD274, MKI67, PDCD1, CAV1 and COL1A1 (<a class="inter-ref-link" href="http://prot-shiny-vm.broadinstitute.org:3838/CPTAC-BRCA2020" target="_blank">http://prot-shiny-vm.broadinstitute.org:3838/CPTAC-BRCA2020</a>).</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/640ef007-93ad-495c-929f-304ff4aea769/figs3.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/c72115b0-24fe-424c-9bf6-0e2acc67e3a0/figs3_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(20)31400-8&amp;id=figs3.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">Two clusters showed sample compositions that were discordant with PAM50 subtypes. The luminal B-inclusive cluster (NMF LumB-I) comprised all but one LumB case but also included a subset of PAM50 LumA samples. Association analysis based on core membership (<a href="#sec4" class="scroll-into-link">STAR Methods</a>) showed that NMF LumB-I tumors had fewer <em>PIK3CA</em> mutations than NMF LumA-I (binomial p = 1.50 × 10<sup>−3</sup>) and lower stromal infiltration scores (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">Figures S3</a>B and S3D). The two luminal clusters also showed remarkable dichotomies in pathway space, supporting the concept that, although heterogeneous, these are biologically separate tumor types. For example, cancer hallmark gene set enrichment scores for LumA-I versus LumB-I were significantly anti-correlated even though estrogen response-related terms were positively enriched in both (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">Figures S3</a>H and S3I). Notably, a mixed PAM50 LumA/B cluster was also observed when clustering the global RNA data in isolation, indicating that PAM50 classification, a method simplified for clinical purposes, does not capture all biological distinctions between LumA and LumB (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">Figure S3</a>J).</div><div class="section-paragraph">To further probe NMF luminal cluster assignments, random forest classifiers were trained on protein or mRNA data to distinguish PAM50 LumA samples assigned to the NMF LumB-I cluster from PAM50 LumA samples assigned to the NMF LumA-I cluster. When these classifiers were applied to METABRIC data (<div class="dropBlock reference-citations"><a href="#bib34" id="back-bib34" data-db-target-for="bib34-d13420944e1846" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib34">Curtis et al., 2012</a><div data-db-target-of="bib34-d13420944e1846" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Curtis C. </li><li class="loa__item">Shah S.P. </li><li class="loa__item">Chin S.-F. </li><li class="loa__item">Turashvili G. </li><li class="loa__item">Rueda O.M. </li><li class="loa__item">Dunning M.J. </li><li class="loa__item">Speed D. </li><li class="loa__item">Lynch A.G. </li><li class="loa__item">Samarajiwa S. </li><li class="loa__item">Yuan Y. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>METABRIC Group</span><br />The genomic and transcriptomic architecture of 2,000 breast tumours reveals novel subgroups.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">486</span></span>: <span class="ref__seriesPages">346-352</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature10983&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=22522925&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84861527388&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3169)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=486&publication_year=2012&pages=346-352&journal=Nature&author=METABRIC+Group&title=The+genomic+and+transcriptomic+architecture+of+2%2C000+breast+tumours+reveals+novel+subgroups">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), samples from patients with NMF features that drove PAM50 LumA samples into the NMF LumB-I cluster had outcomes that were intermediate between the remaining PAM50 LumA samples and the PAM50 LumB samples (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>C; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">Figure S3</a>K). This finding supports the NMF assignment of some PAM50 LumA samples to the higher-risk LumB-I cluster.</div><div class="section-paragraph">The HER2-inclusive cluster (NMF HER2-I) was remarkably heterogeneous. Although predominantly composed of HER2-enriched PAM50 subtype samples and samples with centrally confirmed, clinically positive ERBB2 status, NMF HER2-I also included tumors from all four other PAM50 subtypes, suggesting the presence of unifying biological features in NMF informatic space that are absent in the PAM50-based classification (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">Figure S3</a>B). An in-depth analysis of HER2-unrelated proteomic and phosphoproteomic features that drove clustering in the NMF HER2-I group (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">Figure S3</a>L) revealed over-representation of Gene Ontology (GO) terms (<div class="dropBlock reference-citations"><a href="#bib9" id="back-bib9" data-db-target-for="bib9-d13420944e1863" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib9">Ashburner et al., 2000</a><div data-db-target-of="bib9-d13420944e1863" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ashburner M. </li><li class="loa__item">Ball C.A. </li><li class="loa__item">Blake J.A. </li><li class="loa__item">Botstein D. </li><li class="loa__item">Butler H. </li><li class="loa__item">Cherry J.M. </li><li class="loa__item">Davis A.P. </li><li class="loa__item">Dolinski K. </li><li class="loa__item">Dwight S.S. </li><li class="loa__item">Eppig J.T. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>The Gene Ontology Consortium</span><br />Gene ontology: tool for the unification of biology.<div class="popup-ref"><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2000; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">25-29</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2F75556&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10802651&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-0034069495&amp;cf=" onclick="newWindow(this.href);return false">Scopus (23857)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2000&pages=25-29&journal=Nat.+Genet.&author=The+Gene+Ontology+Consortium&title=Gene+ontology%3A+tool+for+the+unification+of+biology">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) for proteins serving functions in the endoplasmic reticulum (EnR) and for biosynthesis of sterols and cholesterol derivatives (produced in the EnR). These functional elements are targetable biological pathways (<div class="dropBlock reference-citations"><a href="#bib37" id="back-bib37" data-db-target-for="bib37-d13420944e1866" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib37">Dong et al., 2019</a><div data-db-target-of="bib37-d13420944e1866" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Dong H. </li><li class="loa__item">Adams N.M. </li><li class="loa__item">Xu Y. </li><li class="loa__item">Cao J. </li><li class="loa__item">Allan D.S.J. </li><li class="loa__item">Carlyle J.R. </li><li class="loa__item">Chen X. </li><li class="loa__item">Sun J.C. </li><li class="loa__item">Glimcher L.H. </li></ul><div class="reference-citations__detials">The IRE1 endoplasmic reticulum stress sensor activates natural killer cell immunity in part by regulating c-Myc.<div class="popup-ref"><span class="ref__series"><i>Nat. Immunol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">865-878</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41590-019-0388-z&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31086333&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85065791702&amp;cf=" onclick="newWindow(this.href);return false">Scopus (47)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=865-878&journal=Nat.+Immunol.&author=Dong+H.&author=Adams+N.M.&author=Xu+Y.&author=Cao+J.&author=Allan+D.S.J.&author=Carlyle+J.R.&author=Chen+X.&author=Sun+J.C.&author=Glimcher+L.H.&title=The+IRE1+endoplasmic+reticulum+stress+sensor+activates+natural+killer+cell+immunity+in+part+by+regulating+c-Myc">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">Figure S3</a>M; <a href="/cms/10.1016/j.cell.2020.10.036/attachment/48d23c21-8705-4760-9513-c5982cc559ee/mmc5" class="scroll-into-link">Table S5</a>). As expected, enrichment of immune signaling was seen in the NMF HER2-I and NMF Basal-I clusters (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">Figures S3</a>E and S3H), and mRNA and phosphoprotein expression of the key immune checkpoint targets PDCD1 (PD1) and CD274 (PD-L1) was also elevated relative to the two luminal NMF clusters (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">Figure S3</a>N).</div><div class="section-paragraph">Previous studies utilizing proteomics to profile and cluster breast tumors (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs4" href="#figs4" data-locator="figs4" id="back-figs4" class="figure-link scroll-into-link">Figure S4</a>A) have reported varying resemblance of proteomic subtypes to PAM50 subtypes (<div class="dropBlock reference-citations"><a href="#bib17" id="back-bib17" data-db-target-for="bib17-d13420944e1888" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib17">Bouchal et al., 2019</a><div data-db-target-of="bib17-d13420944e1888" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Bouchal P. </li><li class="loa__item">Schubert O.T. </li><li class="loa__item">Faktor J. </li><li class="loa__item">Capkova L. </li><li class="loa__item">Imrichova H. </li><li class="loa__item">Zoufalova K. </li><li class="loa__item">Paralova V. </li><li class="loa__item">Hrstka R. </li><li class="loa__item">Liu Y. </li><li class="loa__item">Ebhardt H.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Breast Cancer Classification Based on Proteotypes Obtained by SWATH Mass Spectrometry.<div class="popup-ref"><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">832-843.e7</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.celrep.2019.06.046&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.celrep.2019.06.046&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.celrep.2019.06.046&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31315058&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85068466554&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2019&pages=832-843.e7&journal=Cell+Rep.&author=Bouchal+P.&author=Schubert+O.T.&author=Faktor+J.&author=Capkova+L.&author=Imrichova+H.&author=Zoufalova+K.&author=Paralova+V.&author=Hrstka+R.&author=Liu+Y.&author=Ebhardt+H.A.&title=Breast+Cancer+Classification+Based+on+Proteotypes+Obtained+by+SWATH+Mass+Spectrometry">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib55" id="back-bib55" data-db-target-for="bib55-d13420944e1891" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib55">Johansson et al., 2019</a><div data-db-target-of="bib55-d13420944e1891" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Johansson H.J. </li><li class="loa__item">Socciarelli F. </li><li class="loa__item">Vacanti N.M. </li><li class="loa__item">Haugen M.H. </li><li class="loa__item">Zhu Y. </li><li class="loa__item">Siavelis I. </li><li class="loa__item">Fernandez-Woodbridge A. </li><li class="loa__item">Aure M.R. </li><li class="loa__item">Sennblad B. </li><li class="loa__item">Vesterlund M. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Consortia Oslo Breast Cancer Research Consortium (OSBREAC)</span><br />Breast cancer quantitative proteome and proteogenomic landscape.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">1600</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41467-019-09018-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30962452&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85064079271&amp;cf=" onclick="newWindow(this.href);return false">Scopus (66)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1600&journal=Nat.+Commun.&author=Consortia+Oslo+Breast+Cancer+Research+Consortium+%28OSBREAC%29&title=Breast+cancer+quantitative+proteome+and+proteogenomic+landscape">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib130" id="back-bib130" data-db-target-for="bib130-d13420944e1894" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib130">Tyanova et al., 2016</a><div data-db-target-of="bib130-d13420944e1894" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Tyanova S. </li><li class="loa__item">Albrechtsen R. </li><li class="loa__item">Kronqvist P. </li><li class="loa__item">Cox J. </li><li class="loa__item">Mann M. </li><li class="loa__item">Geiger T. </li></ul><div class="reference-citations__detials">Proteomic maps of breast cancer subtypes.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">10259</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fncomms10259&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26725330&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=10259&journal=Nat.+Commun.&author=Tyanova+S.&author=Albrechtsen+R.&author=Kronqvist+P.&author=Cox+J.&author=Mann+M.&author=Geiger+T.&title=Proteomic+maps+of+breast+cancer+subtypes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). We analyzed and compared the data in these studies with results of our NMF analyses (detailed in <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs4" href="#figs4" data-locator="figs4" id="back-figs4" class="figure-link scroll-into-link">Figures S4</a>B–S4H). Integration of the current dataset with that of <div class="dropBlock reference-citations"><a href="#bib55" id="back-bib55" data-db-target-for="bib55-d13420944e1901" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib55">Johansson et al., 2019</a><div data-db-target-of="bib55-d13420944e1901" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Johansson H.J. </li><li class="loa__item">Socciarelli F. </li><li class="loa__item">Vacanti N.M. </li><li class="loa__item">Haugen M.H. </li><li class="loa__item">Zhu Y. </li><li class="loa__item">Siavelis I. </li><li class="loa__item">Fernandez-Woodbridge A. </li><li class="loa__item">Aure M.R. </li><li class="loa__item">Sennblad B. </li><li class="loa__item">Vesterlund M. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Consortia Oslo Breast Cancer Research Consortium (OSBREAC)</span><br />Breast cancer quantitative proteome and proteogenomic landscape.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">1600</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41467-019-09018-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30962452&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85064079271&amp;cf=" onclick="newWindow(this.href);return false">Scopus (66)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1600&journal=Nat.+Commun.&author=Consortia+Oslo+Breast+Cancer+Research+Consortium+%28OSBREAC%29&title=Breast+cancer+quantitative+proteome+and+proteogenomic+landscape">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> supported NMF reassignment of some PAM50 LumA samples into the LumB-I group and suggested that their “basal immune” cluster was chiefly defined by an active immune microenvironment (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs4" href="#figs4" data-locator="figs4" id="back-figs4" class="figure-link scroll-into-link">Figures S4</a>B–S4E). The subtype and “proteotype” markers of <div class="dropBlock reference-citations"><a href="#bib130" id="back-bib130" data-db-target-for="bib130-d13420944e1907" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib130">Tyanova et al., 2016</a><div data-db-target-of="bib130-d13420944e1907" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Tyanova S. </li><li class="loa__item">Albrechtsen R. </li><li class="loa__item">Kronqvist P. </li><li class="loa__item">Cox J. </li><li class="loa__item">Mann M. </li><li class="loa__item">Geiger T. </li></ul><div class="reference-citations__detials">Proteomic maps of breast cancer subtypes.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">10259</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fncomms10259&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26725330&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=10259&journal=Nat.+Commun.&author=Tyanova+S.&author=Albrechtsen+R.&author=Kronqvist+P.&author=Cox+J.&author=Mann+M.&author=Geiger+T.&title=Proteomic+maps+of+breast+cancer+subtypes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> and <div class="dropBlock reference-citations"><a href="#bib17" id="back-bib17" data-db-target-for="bib17-d13420944e1910" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib17">Bouchal et al., 2019</a><div data-db-target-of="bib17-d13420944e1910" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Bouchal P. </li><li class="loa__item">Schubert O.T. </li><li class="loa__item">Faktor J. </li><li class="loa__item">Capkova L. </li><li class="loa__item">Imrichova H. </li><li class="loa__item">Zoufalova K. </li><li class="loa__item">Paralova V. </li><li class="loa__item">Hrstka R. </li><li class="loa__item">Liu Y. </li><li class="loa__item">Ebhardt H.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Breast Cancer Classification Based on Proteotypes Obtained by SWATH Mass Spectrometry.<div class="popup-ref"><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">832-843.e7</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.celrep.2019.06.046&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.celrep.2019.06.046&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.celrep.2019.06.046&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31315058&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85068466554&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2019&pages=832-843.e7&journal=Cell+Rep.&author=Bouchal+P.&author=Schubert+O.T.&author=Faktor+J.&author=Capkova+L.&author=Imrichova+H.&author=Zoufalova+K.&author=Paralova+V.&author=Hrstka+R.&author=Liu+Y.&author=Ebhardt+H.A.&title=Breast+Cancer+Classification+Based+on+Proteotypes+Obtained+by+SWATH+Mass+Spectrometry">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> were substantially reproduced in our dataset (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs4" href="#figs4" data-locator="figs4" id="back-figs4" class="figure-link scroll-into-link">Figures S4</a>F–S4H).<div class="floatDisplay"><figure id="figs4" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/91afd4ae-0d96-46a9-b062-89d695fa1be7/figs4.jpg" alt="Figure thumbnail figs4" data-locator="figs4" data-high-res="/cms/attachment/3112ee03-0e05-424a-b996-1223ccf5b6a8/figs4_lrg.jpg" data-large-image="/cms/attachment/91afd4ae-0d96-46a9-b062-89d695fa1be7/figs4.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. S4" class="label figure__label">Figure S4</span><span class="figure__title__text">Comparison with Previous BRCA Proteomics and PG Studies, Related to <a data-locator="fig1" aria-expanded="false" aria-controls="crosref2660" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig1">Figures 1</a> and <a data-locator="figs3" aria-expanded="false" aria-controls="crosref2665" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs3">S3</a></span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">A) Summary of this and prior studies utilizing proteomics or proteogenomics to profile and cluster breast tumors. Shown are sample size analyzed, mass spectrometry platform and proteomic workflow, proteomic depth achieved, subtype composition and unsupervised clustering results.</div><div class="section-paragraph">B) NMF clustering applied to the proteome dataset from <div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib55" id="back-bib55" data-db-target-for="bib55-d13420944e478" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib55">Johansson et al., 2019</a><div data-db-target-of="bib55-d13420944e478" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Johansson H.J. </li><li class="loa__item">Socciarelli F. </li><li class="loa__item">Vacanti N.M. </li><li class="loa__item">Haugen M.H. </li><li class="loa__item">Zhu Y. </li><li class="loa__item">Siavelis I. </li><li class="loa__item">Fernandez-Woodbridge A. </li><li class="loa__item">Aure M.R. </li><li class="loa__item">Sennblad B. </li><li class="loa__item">Vesterlund M. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Consortia Oslo Breast Cancer Research Consortium (OSBREAC)</span><br />Breast cancer quantitative proteome and proteogenomic landscape.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">1600</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41467-019-09018-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30962452&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85064079271&amp;cf=" onclick="newWindow(this.href);return false">Scopus (66)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1600&journal=Nat.+Commun.&author=Consortia+Oslo+Breast+Cancer+Research+Consortium+%28OSBREAC%29&title=Breast+cancer+quantitative+proteome+and+proteogenomic+landscape">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div>. Their study of 45 breast tumors with balanced representation of all five PAM50 subtypes described six stable proteomic clusters derived from the subset of 29 tumors that had Pearson correlation ≥ 0.5 relative to at least one other tumor (<div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib55" id="back-bib55" data-db-target-for="bib55-d13420944e481" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib55">Johansson et al., 2019</a><div data-db-target-of="bib55-d13420944e481" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Johansson H.J. </li><li class="loa__item">Socciarelli F. </li><li class="loa__item">Vacanti N.M. </li><li class="loa__item">Haugen M.H. </li><li class="loa__item">Zhu Y. </li><li class="loa__item">Siavelis I. </li><li class="loa__item">Fernandez-Woodbridge A. </li><li class="loa__item">Aure M.R. </li><li class="loa__item">Sennblad B. </li><li class="loa__item">Vesterlund M. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Consortia Oslo Breast Cancer Research Consortium (OSBREAC)</span><br />Breast cancer quantitative proteome and proteogenomic landscape.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">1600</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41467-019-09018-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30962452&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85064079271&amp;cf=" onclick="newWindow(this.href);return false">Scopus (66)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1600&journal=Nat.+Commun.&author=Consortia+Oslo+Breast+Cancer+Research+Consortium+%28OSBREAC%29&title=Breast+cancer+quantitative+proteome+and+proteogenomic+landscape">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div>). In addition to an alignment to the PAM50 subtypes, the authors reported a further subdivision of poor-prognosis basal-like and luminal B tumors driven by immune component infiltration. Application of the NMF pipeline to their complete proteomic data likewise identified six proteome clusters. Graph shows cluster metrics (y axis) as a function of the number of clusters (K, x axis). Cophenetic correlation (coph), dispersion of the consensus matrix (disp) and the product of both metrics are shown (product).</div><div class="section-paragraph">C) Heatmap depicting the relative contributions of each sample (x axis) to each cluster (y axis) resulting from application of the NMF pipeline to <div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib55" id="back-bib55" data-db-target-for="bib55-d13420944e486" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib55">Johansson et al., 2019</a><div data-db-target-of="bib55-d13420944e486" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Johansson H.J. </li><li class="loa__item">Socciarelli F. </li><li class="loa__item">Vacanti N.M. </li><li class="loa__item">Haugen M.H. </li><li class="loa__item">Zhu Y. </li><li class="loa__item">Siavelis I. </li><li class="loa__item">Fernandez-Woodbridge A. </li><li class="loa__item">Aure M.R. </li><li class="loa__item">Sennblad B. </li><li class="loa__item">Vesterlund M. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Consortia Oslo Breast Cancer Research Consortium (OSBREAC)</span><br />Breast cancer quantitative proteome and proteogenomic landscape.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">1600</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41467-019-09018-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30962452&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85064079271&amp;cf=" onclick="newWindow(this.href);return false">Scopus (66)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1600&journal=Nat.+Commun.&author=Consortia+Oslo+Breast+Cancer+Research+Consortium+%28OSBREAC%29&title=Breast+cancer+quantitative+proteome+and+proteogenomic+landscape">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div>. This heatmap is a visualization of the (column-normalized) meta-feature matrix derived from decomposing the input matrix (<a data-locator="sec4" aria-expanded="false" aria-controls="crosref2685" class=" caption__link scroll-into-link" href="&#xA;                                #sec4">STAR Methods</a>), normalized per column by the maximum-valued entry. High concordance was observed between consensus core tumor clusters (CoTC) reported in <div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib55" id="back-bib55" data-db-target-for="bib55-d13420944e492" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib55">Johansson et al., 2019</a><div data-db-target-of="bib55-d13420944e492" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Johansson H.J. </li><li class="loa__item">Socciarelli F. </li><li class="loa__item">Vacanti N.M. </li><li class="loa__item">Haugen M.H. </li><li class="loa__item">Zhu Y. </li><li class="loa__item">Siavelis I. </li><li class="loa__item">Fernandez-Woodbridge A. </li><li class="loa__item">Aure M.R. </li><li class="loa__item">Sennblad B. </li><li class="loa__item">Vesterlund M. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Consortia Oslo Breast Cancer Research Consortium (OSBREAC)</span><br />Breast cancer quantitative proteome and proteogenomic landscape.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">1600</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41467-019-09018-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30962452&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85064079271&amp;cf=" onclick="newWindow(this.href);return false">Scopus (66)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1600&journal=Nat.+Commun.&author=Consortia+Oslo+Breast+Cancer+Research+Consortium+%28OSBREAC%29&title=Breast+cancer+quantitative+proteome+and+proteogenomic+landscape">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div> and NMF-based subtyping.</div><div class="section-paragraph">D) NMF clustering applied to an integrated proteomic dataset derived from Johansson et al. and the present study (<a data-locator="sec4" aria-expanded="false" aria-controls="crosref2695" class=" caption__link scroll-into-link" href="&#xA;                                #sec4">STAR Methods</a>). The comparable MS workflows utilized in the studies together with the similar cluster structures revealed by the different analytical approaches enabled such integration. Only four PAM50 LumA samples from <div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib55" id="back-bib55" data-db-target-for="bib55-d13420944e500" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib55">Johansson et al., 2019</a><div data-db-target-of="bib55-d13420944e500" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Johansson H.J. </li><li class="loa__item">Socciarelli F. </li><li class="loa__item">Vacanti N.M. </li><li class="loa__item">Haugen M.H. </li><li class="loa__item">Zhu Y. </li><li class="loa__item">Siavelis I. </li><li class="loa__item">Fernandez-Woodbridge A. </li><li class="loa__item">Aure M.R. </li><li class="loa__item">Sennblad B. </li><li class="loa__item">Vesterlund M. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Consortia Oslo Breast Cancer Research Consortium (OSBREAC)</span><br />Breast cancer quantitative proteome and proteogenomic landscape.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">1600</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41467-019-09018-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30962452&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85064079271&amp;cf=" onclick="newWindow(this.href);return false">Scopus (66)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1600&journal=Nat.+Commun.&author=Consortia+Oslo+Breast+Cancer+Research+Consortium+%28OSBREAC%29&title=Breast+cancer+quantitative+proteome+and+proteogenomic+landscape">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div> were assigned to the NMF LumA-I cluster; of the remaining five assigned to the NMF LumB-I cluster, four had been excluded from their clustering exercise because of low correlation to other samples. These data support our contention that NMF clustering identifies a smaller, more homogeneous LumA-I group by reassigning some PAM50 LumA samples into the LumB-I group.</div><div class="section-paragraph">E) Normalized enrichment scores (NES) of immune-related Hallmark signatures based on the feature weights driving the clustering shown in D) (<a data-locator="sec4" aria-expanded="false" aria-controls="crosref2705" class=" caption__link scroll-into-link" href="&#xA;                                #sec4">STAR Methods</a>). The two samples that formed the Johansson et al. CoTC2 (“basal immune”) cluster joined a cluster in the combined dataset that was much more heterogeneous in PAM50 composition but had immune-related Hallmark gene signatures as a unifying characteristic. We therefore suggest that an active immune microenvironment rather than the basal subtype assignment was critical to the definition of the CoTC2 cluster.</div><div class="section-paragraph">F) <div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib130" id="back-bib130" data-db-target-for="bib130-d13420944e510" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib130">Tyanova et al., 2016</a><div data-db-target-of="bib130-d13420944e510" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Tyanova S. </li><li class="loa__item">Albrechtsen R. </li><li class="loa__item">Kronqvist P. </li><li class="loa__item">Cox J. </li><li class="loa__item">Mann M. </li><li class="loa__item">Geiger T. </li></ul><div class="reference-citations__detials">Proteomic maps of breast cancer subtypes.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">10259</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fncomms10259&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26725330&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=10259&journal=Nat.+Commun.&author=Tyanova+S.&author=Albrechtsen+R.&author=Kronqvist+P.&author=Cox+J.&author=Mann+M.&author=Geiger+T.&title=Proteomic+maps+of+breast+cancer+subtypes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div> identified a 19-protein predictive classifier for hormone receptor status-based subtypes (ERPR = ER+/PR+; HER2 = HER2+; TN = ER-/PR-/HER2-) by applying support vector machine (SVM)-based classification to super-SILAC proteomic profiles of 40 total ER/PR+, HER2 and triple negative breast tumors. The left box shows all 19 proteins grouped by subtype and annotated by their predictive abundance relative to the other subtypes: increased (+) or decreased (-) expression. The boxplots depict the abundance of 18 of these proteins quantified in our study across ERPR, HER2 and TN subtypes according to our clinical (ER, PR) and proteogenomic-based (HER2) assessment of receptor status (C9orf114 was not quantified in our analysis, which excluded 5 samples for which receptor status annotation was incomplete). A negative HER2 status was required for tumors classified as ERPR. Seventeen out of 18 measured markers maintained their subtype associations in our dataset (Kruskal-Wallis test). Only CAPN7 did not replicate, showing a trend as a negative marker of TNBC that did not reach statistical significance. Boxes depict the interquartile range (IQR) of the data with horizontal lines depicting the median. Whiskers extend to 1.5xIQR from Q1 (25th percentile) and Q3 (75th percentile), respectively. Points depict individual data points.</div><div class="section-paragraph">G) Heatmap depicting standardized protein abundances of MAPK3 and EEF1G across all tumors classified as ERPR in our dataset. These positive and negative ER/PR markers captured PAM50 LumA but interestingly not LumB samples (a distinction not made in the <div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib130" id="back-bib130" data-db-target-for="bib130-d13420944e515" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib130">Tyanova et al., 2016</a><div data-db-target-of="bib130-d13420944e515" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Tyanova S. </li><li class="loa__item">Albrechtsen R. </li><li class="loa__item">Kronqvist P. </li><li class="loa__item">Cox J. </li><li class="loa__item">Mann M. </li><li class="loa__item">Geiger T. </li></ul><div class="reference-citations__detials">Proteomic maps of breast cancer subtypes.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">10259</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fncomms10259&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26725330&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=10259&journal=Nat.+Commun.&author=Tyanova+S.&author=Albrechtsen+R.&author=Kronqvist+P.&author=Cox+J.&author=Mann+M.&author=Geiger+T.&title=Proteomic+maps+of+breast+cancer+subtypes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div> analysis).</div><div class="section-paragraph">H) <div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib17" id="back-bib17" data-db-target-for="bib17-d13420944e520" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib17">Bouchal et al., 2019</a><div data-db-target-of="bib17-d13420944e520" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Bouchal P. </li><li class="loa__item">Schubert O.T. </li><li class="loa__item">Faktor J. </li><li class="loa__item">Capkova L. </li><li class="loa__item">Imrichova H. </li><li class="loa__item">Zoufalova K. </li><li class="loa__item">Paralova V. </li><li class="loa__item">Hrstka R. </li><li class="loa__item">Liu Y. </li><li class="loa__item">Ebhardt H.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Breast Cancer Classification Based on Proteotypes Obtained by SWATH Mass Spectrometry.<div class="popup-ref"><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">832-843.e7</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.celrep.2019.06.046&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.celrep.2019.06.046&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.celrep.2019.06.046&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31315058&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85068466554&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2019&pages=832-843.e7&journal=Cell+Rep.&author=Bouchal+P.&author=Schubert+O.T.&author=Faktor+J.&author=Capkova+L.&author=Imrichova+H.&author=Zoufalova+K.&author=Paralova+V.&author=Hrstka+R.&author=Liu+Y.&author=Ebhardt+H.A.&title=Breast+Cancer+Classification+Based+on+Proteotypes+Obtained+by+SWATH+Mass+Spectrometry">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div> reported three “proteotypic” proteins associated with ER (INPP4B) and HER2 (ERBB2) receptor status and with tumor grade (CDK1) in a study using a label-free, data-independent MS acquisition method to examine 96 breast cancer specimens. The boxplots illustrate the protein abundance of INPP4B between ER+/− and ERBB2 between HER2+/− in our dataset, reproducing their results (Boxplots are defined in panel F). P values were derived from a Wilcoxon rank-sum test. Due to incomplete grade annotation in our dataset we did not test their association of CDK1 protein expression and grade; however, CDK1 was significantly correlated with MKi67, (Pearson R = 0.72), consistent with the correlation of R = 0.79 that they reported as a confirmatory analysis in the TCGA breast cancer dataset (scatterplot).</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/91afd4ae-0d96-46a9-b062-89d695fa1be7/figs4.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/3112ee03-0e05-424a-b996-1223ccf5b6a8/figs4_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(20)31400-8&amp;id=figs4.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div></section><section id="sec2.3" data-locator="sec2.3" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Subtype-Specific Expression of Targetable, Highly Phosphorylated Kinases</span></h3><div class="section-paragraph">To identify putative therapeutic targets specific for each NMF subtype, phosphoproteomic data were used as kinase activation surrogates (<div class="dropBlock reference-citations"><a href="#bib41" id="back-bib41" data-db-target-for="bib41-d13420944e1921" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib41">Flockhart and Corbin, 1982</a><div data-db-target-of="bib41-d13420944e1921" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Flockhart D.A. </li><li class="loa__item">Corbin J.D. </li></ul><div class="reference-citations__detials">Regulatory mechanisms in the control of protein kinases.<div class="popup-ref"><span class="ref__series"><i>CRC Crit. Rev. Biochem.</i> <span class="ref__seriesDate">1982; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">133-186</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.3109%2F10409238209108705&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=7039969&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1982&pages=133-186&journal=CRC+Crit.+Rev.+Biochem.&author=Flockhart+D.A.&author=Corbin+J.D.&title=Regulatory+mechanisms+in+the+control+of+protein+kinases">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib119" id="back-bib119" data-db-target-for="bib119-d13420944e1924" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib119">Smith et al., 1993</a><div data-db-target-of="bib119-d13420944e1924" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Smith J.A. </li><li class="loa__item">Francis S.H. </li><li class="loa__item">Corbin J.D. </li></ul><div class="reference-citations__detials">Autophosphorylation: a salient feature of protein kinases.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Biochem.</i> <span class="ref__seriesDate">1993; </span><span class="ref__seriesVolume">127-128</span></span>: <span class="ref__seriesPages">51-70</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1007%2FBF01076757&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=7935362&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127-128&publication_year=1993&pages=51-70&journal=Mol.+Cell.+Biochem.&author=Smith+J.A.&author=Francis+S.H.&author=Corbin+J.D.&title=Autophosphorylation%3A+a+salient+feature+of+protein+kinases">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib137" id="back-bib137" data-db-target-for="bib137-d13420944e1927" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib137">Wang and Wu, 2002</a><div data-db-target-of="bib137-d13420944e1927" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang Z.-X. </li><li class="loa__item">Wu J.-W. </li></ul><div class="reference-citations__detials">Autophosphorylation kinetics of protein kinases.<div class="popup-ref"><span class="ref__series"><i>Biochem. J.</i> <span class="ref__seriesDate">2002; </span><span class="ref__seriesVolume">368</span></span>: <span class="ref__seriesPages">947-952</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1042%2Fbj20020557&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=12190618&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2002&pages=947-952&journal=Biochem.+J.&author=Wang+Z.-X.&author=Wu+J.-W.&title=Autophosphorylation+kinetics+of+protein+kinases">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Phosphorylated kinases enriched in each NMF subtype were identified using outlier enrichment analysis (BlackSheep Python package) (<div class="dropBlock reference-citations"><a href="#bib16" id="back-bib16" data-db-target-for="bib16-d13420944e1930" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib16">Blumenberg et al., 2019</a><div data-db-target-of="bib16-d13420944e1930" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Blumenberg L. </li><li class="loa__item">Kawaler E. </li><li class="loa__item">Cornwell M. </li><li class="loa__item">Smith S. </li><li class="loa__item">Ruggles K. </li><li class="loa__item">Fenyö D. </li></ul><div class="reference-citations__detials">BlackSheep: A Bioconductor and Bioconda package for differential extreme value analysis.<div class="popup-ref"><span class="ref__series"><i>bioRxiv.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1101/825067">https://doi.org/10.1101/825067</a></span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1101%2F825067&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85095064917&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&journal=bioRxiv&author=Blumenberg+L.&author=Kawaler+E.&author=Cornwell+M.&author=Smith+S.&author=Ruggles+K.&author=Feny%C3%B6+D.&title=BlackSheep%3A+A+Bioconductor+and+Bioconda+package+for+differential+extreme+value+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>D; <a href="/cms/10.1016/j.cell.2020.10.036/attachment/d7a1a20c-1c01-4f40-98d9-81f918417a6d/mmc4" class="scroll-into-link">Table S4</a>). Many enriched kinases (false discovery rate [FDR] &lt; 0.01) observed in each PAM50 subtype in our initial study (<div class="dropBlock reference-citations"><a href="#bib85" id="back-bib85" data-db-target-for="bib85-d13420944e1940" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib85">Mertins et al., 2016</a><div data-db-target-of="bib85-d13420944e1940" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Wang P. </li><li class="loa__item">Wang X. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Cao S. </li><li class="loa__item">Petralia F. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomics connects somatic mutations to signalling in breast cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (780)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=NCI+CPTAC&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) were also enriched in this dataset using NMF subtypes, including PRKDC, MAP4K4 and SPEG in the NMF Basal-I subtype; ERBB2 and CDK12 in NMF HER2-I samples; and DCLK1 in NMF LumA-I samples (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>D). These putatively activated kinases are candidates for subtype-specific treatment (<div class="dropBlock reference-citations"><a href="#bib33" id="back-bib33" data-db-target-for="bib33-d13420944e1946" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib33">Cotto et al., 2018</a><div data-db-target-of="bib33-d13420944e1946" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Cotto K.C. </li><li class="loa__item">Wagner A.H. </li><li class="loa__item">Feng Y.-Y. </li><li class="loa__item">Kiwala S. </li><li class="loa__item">Coffman A.C. </li><li class="loa__item">Spies G. </li><li class="loa__item">Wollam A. </li><li class="loa__item">Spies N.C. </li><li class="loa__item">Griffith O.L. </li><li class="loa__item">Griffith M. </li></ul><div class="reference-citations__detials">DGIdb 3.0: a redesign and expansion of the drug-gene interaction database.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">46</span></span>: <span class="ref__seriesPages">D1068-D1073</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgkx1143&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29156001&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85070670869&amp;cf=" onclick="newWindow(this.href);return false">Scopus (303)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D1068-D1073&journal=Nucleic+Acids+Res.&author=Cotto+K.C.&author=Wagner+A.H.&author=Feng+Y.-Y.&author=Kiwala+S.&author=Coffman+A.C.&author=Spies+G.&author=Wollam+A.&author=Spies+N.C.&author=Griffith+O.L.&author=Griffith+M.&title=DGIdb+3.0%3A+a+redesign+and+expansion+of+the+drug-gene+interaction+database">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div><div class="section-paragraph">The BlackSheep approach also associated phosphorylated kinase outliers with recurrent somatic mutations (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>E). A noteworthy example was the increased phosphorylation levels of TRAF2- and NCK-interacting kinase (TNIK) in <em>ARID1A</em> mutant cases because TNIK is a therapeutic target due to its role in the WNT pathway (<div class="dropBlock reference-citations"><a href="#bib78" id="back-bib78" data-db-target-for="bib78-d13420944e1956" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib78">Masuda and Yamada, 2017</a><div data-db-target-of="bib78-d13420944e1956" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Masuda M. </li><li class="loa__item">Yamada T. </li></ul><div class="reference-citations__detials">The emergence of TNIK as a therapeutic target for colorectal cancer.<div class="popup-ref"><span class="ref__series"><i>Expert Opin. Ther. Targets.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">21</span></span>: <span class="ref__seriesPages">353-355</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1080%2F14728222.2017.1297799&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=28281900&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85014502350&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2017&pages=353-355&journal=Expert+Opin.+Ther.+Targets&author=Masuda+M.&author=Yamada+T.&title=The+emergence+of+TNIK+as+a+therapeutic+target+for+colorectal+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Upregulation of phosphorylated RIPK3 in tumors with <em>MAP3K1</em> mutation was also of interest because loss-of-function mutations in this stress kinase are a poorly understood but highly recurring event in luminal BRCA. Although RIPK3 has a role in triggering necroptosis, it may also have a tumor-promoting role under some circumstances (<div class="dropBlock reference-citations"><a href="#bib73" id="back-bib73" data-db-target-for="bib73-d13420944e1961" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib73">Lin et al., 2020</a><div data-db-target-of="bib73-d13420944e1961" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lin C.-C. </li><li class="loa__item">Mabe N.W. </li><li class="loa__item">Lin Y.-T. </li><li class="loa__item">Yang W.-H. </li><li class="loa__item">Tang X. </li><li class="loa__item">Hong L. </li><li class="loa__item">Sun T. </li><li class="loa__item">Force J. </li><li class="loa__item">Marks J.R. </li><li class="loa__item">Yao T.-P. </li><li>et al.</li></ul><div class="reference-citations__detials">RIPK3 upregulation confers robust proliferation and collateral cystine-dependence on breast cancer recurrence.<div class="popup-ref"><span class="ref__series"><i>Cell Death Differ.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">27</span></span>: <span class="ref__seriesPages">2234-2247</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41418-020-0499-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31988496&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85078494025&amp;cf=" onclick="newWindow(this.href);return false">Scopus (12)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2020&pages=2234-2247&journal=Cell+Death+Differ.&author=Lin+C.-C.&author=Mabe+N.W.&author=Lin+Y.-T.&author=Yang+W.-H.&author=Tang+X.&author=Hong+L.&author=Sun+T.&author=Force+J.&author=Marks+J.R.&author=Yao+T.-P.&title=RIPK3+upregulation+confers+robust+proliferation+and+collateral+cystine-dependence+on+breast+cancer+recurrence">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The high levels of phosphorylation of MAST4 and DCLK1, microtubule-associated kinases and neuroendocrine markers, in the context of <em>GATA3</em> mutation are newly described here and therefore require validation. A final example of these novel connections was increased phosphorylation of SLK/LATS1 in <em>AKT</em> mutated tumors, which may reflect cross-talk between the mTOR and HIPPO pathways (<div class="dropBlock reference-citations"><a href="#bib24" id="back-bib24" data-db-target-for="bib24-d13420944e1969" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib24">Chiang and Martinez-Agosto, 2012</a><div data-db-target-of="bib24-d13420944e1969" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Chiang J. </li><li class="loa__item">Martinez-Agosto J.A. </li></ul><div class="reference-citations__detials">Effects of mTOR Inhibitors on Components of the Salvador-Warts-Hippo Pathway.<div class="popup-ref"><span class="ref__series"><i>Cells.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">886-904</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.3390%2Fcells1040886&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=24710534&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2012&pages=886-904&journal=Cells&author=Chiang+J.&author=Martinez-Agosto+J.A.&title=Effects+of+mTOR+Inhibitors+on+Components+of+the+Salvador-Warts-Hippo+Pathway">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib116" id="back-bib116" data-db-target-for="bib116-d13420944e1972" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib116">Shin and Nguyen, 2016</a><div data-db-target-of="bib116-d13420944e1972" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Shin S.-Y. </li><li class="loa__item">Nguyen L.K. </li></ul><div class="reference-citations__detials">Unveiling Hidden Dynamics of Hippo Signalling: A Systems Analysis.<div class="popup-ref"><span class="ref__series"><i>Genes (Basel).</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">44</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.3390%2Fgenes7080044&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84982292513&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=44&journal=Genes+%28Basel%29&author=Shin+S.-Y.&author=Nguyen+L.K.&title=Unveiling+Hidden+Dynamics+of+Hippo+Signalling%3A+A+Systems+Analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div></section><section id="sec2.4" data-locator="sec2.4" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Proteogenomic Metabolic Profiling and Acetylproteomics Highlight Subtype-Specific Metabolism</span></h3><div class="section-paragraph">Therapeutic targeting of abnormal cancer metabolism is garnering increased attention (<div class="dropBlock reference-citations"><a href="#bib99" id="back-bib99" data-db-target-for="bib99-d13420944e1980" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib99">Pavlova and Thompson, 2016</a><div data-db-target-of="bib99-d13420944e1980" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Pavlova N.N. </li><li class="loa__item">Thompson C.B. </li></ul><div class="reference-citations__detials">The Emerging Hallmarks of Cancer Metabolism.<div class="popup-ref"><span class="ref__series"><i>Cell Metab.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">27-47</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cmet.2015.12.006&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cmet.2015.12.006&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cmet.2015.12.006&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26771115&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84955326448&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2067)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=27-47&journal=Cell+Metab.&author=Pavlova+N.N.&author=Thompson+C.B.&title=The+Emerging+Hallmarks+of+Cancer+Metabolism">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib101" id="back-bib101" data-db-target-for="bib101-d13420944e1983" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib101">Phan et al., 2014</a><div data-db-target-of="bib101-d13420944e1983" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Phan L.M. </li><li class="loa__item">Yeung S.-C.J. </li><li class="loa__item">Lee M.-H. </li></ul><div class="reference-citations__detials">Cancer metabolic reprogramming: importance, main features, and potentials for precise targeted anti-cancer therapies.<div class="popup-ref"><span class="ref__series"><i>Cancer Biol. Med.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">1-19</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=24738035&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2014&pages=1-19&journal=Cancer+Biol.+Med.&author=Phan+L.M.&author=Yeung+S.-C.J.&author=Lee+M.-H.&title=Cancer+metabolic+reprogramming%3A+importance%2C+main+features%2C+and+potentials+for+precise+targeted+anti-cancer+therapies">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Tumor metabolic characteristics were profiled at the level of the proteome, and unsupervised clustering of differentially expressed (DE) metabolism-related proteins (<a href="#sec4" class="scroll-into-link">STAR Methods</a>) grouped samples into 4 clusters that closely reflected the 4 NMF clusters described in <a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>A (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>A). Metabolism-driven cluster 1 almost exclusively represented NMF Basal-I tumors with upregulation of proteins involved in DNA elongation, translation, and metabolism of carbohydrates and downregulation of cholesterol biosynthesis, metabolism of amino acids, and vitamins and cofactors. Metabolism-driven clusters 2 and 3 largely coincided with NMF LumA-I and NMF LumB-I, respectively, with an inverse overall metabolic feature profile relative to NMF Basal-I. Only NMF LumA-I showed upregulated glycosaminoglycan metabolism, which may reflect the stroma-enriched features of these tumors (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">Figure S3</a>D). Metabolism-driven cluster 4, dominated by NMF HER2-I tumors, showed upregulation of cholesterol biosynthesis and lipid metabolism as a HER2-I feature that is independent of <em>ERBB2</em> amplification status (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">Figures S3</a>I and S3J).<div class="floatDisplay"><figure id="gr2" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/59dbaba2-2d90-4c64-bf23-28dd16af8c9c/gr2.jpg" alt="Figure thumbnail gr2" data-locator="fig2" data-high-res="/cms/attachment/aaaa7bdf-f186-4844-bd27-b5f612b74566/gr2_lrg.jpg" data-large-image="/cms/attachment/59dbaba2-2d90-4c64-bf23-28dd16af8c9c/gr2.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 2" class="label figure__label">Figure 2</span><span class="figure__title__text">Proteogenomics (PG) Metabolic Profiling</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Heatmap showing unsupervised clustering of DE metabolic proteins across NMF clusters (Kruskal-Wallis test, FDR p &lt; 5x10<sup>–</sup><sup>05</sup>). The bottom heatmap shows DE normalized Ac values (normalized to protein abundance; Kruskal-Wallis test, FDR p &lt; 0.005) with the same sample ordering as the top heatmap.</div><div class="section-paragraph">(B) Pathway schematic showing DE metabolic proteins and normalized Ac sites (Wilcoxon test, FDR p &lt; 0.05) mapped onto key metabolic pathways.</div><div class="section-paragraph">(C) Bubble chart showing breakdown of upregulated and downregulated proteins and normalized Ac sites in NMF Basal-I compared with any other subtype by cell compartment.</div><div class="section-paragraph">(D) Significant associations (linear model coefficient FDR p &lt; 0.1) between protein expression of mitochondrial HDACs (histone deacetylases) and HATs (histone acetyltransferases) (columns) and Ac of mitochondrial metabolic proteins (rows).</div><div class="section-paragraph">(E) Heatmap showing unsupervised clustering of nuclear protein acetylation, which was differentially expressed across NMF clusters (Kruskal-Wallis test, FDR p &lt; 0.05).</div><div class="section-paragraph">(F) Protein scores of DNA repair pathways across clusters defined in (E). Wilcoxon test p value significance is shown compared with cluster 1. <sup>∗</sup>p &lt; 0.05, <sup>∗∗</sup>p &lt; 0.01, <sup>∗∗∗</sup>p &lt; 0.001, <sup>∗∗∗∗</sup>p &lt; 0.0001. BER, base excision repair; NER, nucleotide excision repair; SSBR, single-strand break repair; DSBR, double-strand break repair; FA, Fanconi anemia; HR, homologous recombination. Boxplots show 1.5× the interquartile range for each group, centered on the median.</div><div class="section-paragraph">(G) Scatterplot showing global differential protein expression and Ac analysis results in cluster 1 versus cluster 3, representing the two subgroups of NMF Basal-I. The x axis shows the protein median fold change multiplied by -log10(FDR p value). The y axis shows the Ac site median fold change multiplied by -log10(FDR p value). Ac or protein changes were considered significantly different if FDR p value &lt; 0.05 and median fold change &gt; 0.5. The “Ac up in cluster 1” group is defined by significantly different Ac sites for which the Ac median fold change is positive and the protein change is not significant. The “protein up in cluster 1” group is defined by significantly different proteins for which the protein median fold change is positive and the Ac change is not significant.</div><div class="section-paragraph">(H) Significantly different Ac sites in cluster 1 versus cluster 3 are found in HATs, their complex partners, histone proteins, and the NHEJ pathway. Boxplots show 1.5× the interquartile range for each group, centered on the median.</div><div class="section-paragraph">See also <a data-locator="figs5" aria-expanded="false" aria-controls="crosref2450" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs5">Figure S5</a> and <a data-locator="mmc2" aria-expanded="false" aria-controls="crosref2455" href="/cms/10.1016/j.cell.2020.10.036/attachment/a1bc3f3e-eeac-4605-a9ef-839abd334dcd/mmc2" target="_blank">Tables S2</a> and <a data-locator="mmc6" aria-expanded="false" aria-controls="crosref2460" href="/cms/10.1016/j.cell.2020.10.036/attachment/eb237fc7-15fb-4b8d-bf02-cae5f285b239/mmc6" target="_blank">S6</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/59dbaba2-2d90-4c64-bf23-28dd16af8c9c/gr2.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/aaaa7bdf-f186-4844-bd27-b5f612b74566/gr2_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(20)31400-8&amp;id=gr2.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">Protein acetylation (Ac) has been implicated in cellular metabolism in addition to roles in epigenetic regulation (<div class="dropBlock reference-citations"><a href="#bib2" id="back-bib2" data-db-target-for="bib2-d13420944e2007" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib2">Ali et al., 2018</a><div data-db-target-of="bib2-d13420944e2007" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ali I. </li><li class="loa__item">Conrad R.J. </li><li class="loa__item">Verdin E. </li><li class="loa__item">Ott M. </li></ul><div class="reference-citations__detials">Lysine Acetylation Goes Global: From Epigenetics to Metabolism and Therapeutics.<div class="popup-ref"><span class="ref__series"><i>Chem. Rev.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">118</span></span>: <span class="ref__seriesPages">1216-1252</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1021%2Facs.chemrev.7b00181&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29405707&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85042009680&amp;cf=" onclick="newWindow(this.href);return false">Scopus (112)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=1216-1252&journal=Chem.+Rev.&author=Ali+I.&author=Conrad+R.J.&author=Verdin+E.&author=Ott+M.&title=Lysine+Acetylation+Goes+Global%3A+From+Epigenetics+to+Metabolism+and+Therapeutics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib25" id="back-bib25" data-db-target-for="bib25-d13420944e2010" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib25">Choudhary et al., 2009</a><div data-db-target-of="bib25-d13420944e2010" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Choudhary C. </li><li class="loa__item">Kumar C. </li><li class="loa__item">Gnad F. </li><li class="loa__item">Nielsen M.L. </li><li class="loa__item">Rehman M. </li><li class="loa__item">Walther T.C. </li><li class="loa__item">Olsen J.V. </li><li class="loa__item">Mann M. </li></ul><div class="reference-citations__detials">Lysine acetylation targets protein complexes and co-regulates major cellular functions.<div class="popup-ref"><span class="ref__series"><i>Science.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">325</span></span>: <span class="ref__seriesPages">834-840</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1126%2Fscience.1175371&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=19608861&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-68949212379&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2881)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=325&publication_year=2009&pages=834-840&journal=Science&author=Choudhary+C.&author=Kumar+C.&author=Gnad+F.&author=Nielsen+M.L.&author=Rehman+M.&author=Walther+T.C.&author=Olsen+J.V.&author=Mann+M.&title=Lysine+acetylation+targets+protein+complexes+and+co-regulates+major+cellular+functions">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib136" id="back-bib136" data-db-target-for="bib136-d13420944e2013" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib136">Verdin and Ott, 2015</a><div data-db-target-of="bib136-d13420944e2013" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Verdin E. </li><li class="loa__item">Ott M. </li></ul><div class="reference-citations__detials">50 years of protein acetylation: from gene regulation to epigenetics, metabolism and beyond.<div class="popup-ref"><span class="ref__series"><i>Nat. Rev. Mol. Cell Biol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">258-264</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnrm3931&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=25549891&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84926163974&amp;cf=" onclick="newWindow(this.href);return false">Scopus (398)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=258-264&journal=Nat.+Rev.+Mol.+Cell+Biol.&author=Verdin+E.&author=Ott+M.&title=50+years+of+protein+acetylation%3A+from+gene+regulation+to+epigenetics%2C+metabolism+and+beyond">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Here, Ac levels normalized to protein abundance were used to identify NMF cluster-specific protein Ac events (<a href="#sec4" class="scroll-into-link">STAR Methods</a>). Uniform upregulation of Ac for TCA cycle and β-oxidation proteins in the NMF Basal-I cluster and for glucose metabolism and interleukin-1 (IL-1) signaling-related proteins in the NMF LumB-I clusters was observed in these analyses (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>A). Ac levels were also differentially distributed across cellular compartments. Most of the DE mitochondrial Ac sites were upregulated in NMF Basal-I, whereas two thirds of DE cytoplasmic Ac sites were downregulated compared with LumB-I, implying compartment-specific regulation of Ac in the NMF Basal-I subtype (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figures 2</a>B and 2C). This suggests that major cytoplasmic and mitochondrial metabolic pathways are differentially regulated between NMF Basal-I and LumB-I subtypes. For example, for NMF Basal-I samples, the central metabolic pathway in the cytoplasm, glycolysis, was upregulated at the protein level (HK3, PFKP, GAPDH, ENO1, and LDHB) and hypoacetylated at the post-translational level (GPI, TPI1, GAPDH, PGK1, PGAM1, ENO1, PKM, and LDHA) (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>B; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs5" href="#figs5" data-locator="figs5" id="back-figs5" class="figure-link scroll-into-link">Figures S5</a>A and S5B). Serine synthesis proteins were also upregulated (PHGDH and PSAT1).<div class="floatDisplay"><figure id="figs5" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/7a6948ba-7b79-4f41-8826-0fb984ea1e57/figs5.jpg" alt="Figure thumbnail figs5" data-locator="figs5" data-high-res="/cms/attachment/a9e1cc96-0883-404b-97c0-3ca9ed9ac247/figs5_lrg.jpg" data-large-image="/cms/attachment/7a6948ba-7b79-4f41-8826-0fb984ea1e57/figs5.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. S5" class="label figure__label">Figure S5</span><span class="figure__title__text">PG Metabolic Profiling, Related to <a data-locator="fig2" aria-expanded="false" aria-controls="crosref2725" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig2">Figure 2</a></span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">A) Pathway schematic similar to <a data-locator="fig2" aria-expanded="false" aria-controls="crosref2730" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig2">Figure 2</a>B but showing differentially expressed metabolic proteins and normalized acetylation sites (Wilcoxon test FDR p value &lt; 0.05) mapped onto key metabolic pathways across all four NMF subtypes.</div><div class="section-paragraph">B) Scatterplots showing examples of proteins regulated at the protein or at the acetylation level across subtypes. Background color corresponds to pathway colors in <a data-locator="fig2" aria-expanded="false" aria-controls="crosref2735" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig2">Figure 2</a>C.</div><div class="section-paragraph">C) Scatterplots showing correlation between copy number variation and protein expression of genes involved in metabolism. Spearman’s Rho value is shown. Background color corresponds to pathway colors in <a data-locator="fig2" aria-expanded="false" aria-controls="crosref2740" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig2">Figure 2</a>C.</div><div class="section-paragraph">D) Boxplot showing SIRT3 protein expression in four NMF subtypes. Wilcoxon test FDR p values are shown: <sup>∗∗∗∗</sup> ≤ 0.0001, <sup>∗∗∗</sup> ≤ 0.001, <sup>∗∗</sup> ≤ 0.01. Boxplots show 1.5x the interquartile range for each group, centered on the median.</div><div class="section-paragraph">E) Further examples of nuclear proteins that were differentially acetylated between subtypes 1 and 3 (see <a data-locator="fig2" aria-expanded="false" aria-controls="crosref2745" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig2">Figure 2</a>E), but were not differential at the protein level. Boxplots show 1.5x the interquartile range for each group, centered on the median.</div><div class="section-paragraph">F) gProfiler pathway and GO overrepresentation analysis of proteins from the ‘Acetylation up in cluster 1’ group in <a data-locator="fig2" aria-expanded="false" aria-controls="crosref2750" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig2">Figure 2</a>G. Proteins were enriched for nucleoplasm, chromatin organization, HATs acetylate histones, and Metabolism of RNA pathways.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/7a6948ba-7b79-4f41-8826-0fb984ea1e57/figs5.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/a9e1cc96-0883-404b-97c0-3ca9ed9ac247/figs5_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(20)31400-8&amp;id=figs5.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">Copy number was correlated with metabolic enzyme expression in NMF Basal-I tumors but not in other subtypes, suggesting that activation of glycolysis and serine synthesis pathways might be uniquely driven by chromosomal aberrations in the NMF Basal-I subtype (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs5" href="#figs5" data-locator="figs5" id="back-figs5" class="figure-link scroll-into-link">Figure S5</a>C). As further examples of NMF Basal-I-specific metabolism, mitochondrial pyruvate dehydrogenase complex (PDC), TCA cycle, and β-oxidation enzyme proteins were specifically hyperacetylated (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figures 2</a>B; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs5" href="#figs5" data-locator="figs5" id="back-figs5" class="figure-link scroll-into-link">Figures S5</a>A and S5B). An unbiased search for potential regulators of metabolic protein Ac revealed significant negative associations between protein levels of the mitochondrial deacetylase SIRT3 and Ac of mitochondrial proteins (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>D), suggesting that deregulation of SIRT3 protein expression (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs5" href="#figs5" data-locator="figs5" id="back-figs5" class="figure-link scroll-into-link">Figure S5</a>D) could broadly affect mitochondrial Ac in BRCA. This is consistent with the role of SIRT3 in suppressing acetyl-coenzyme A (CoA)-mediated non-enzymatic mitochondrial Ac (<div class="dropBlock reference-citations"><a href="#bib138" id="back-bib138" data-db-target-for="bib138-d13420944e2051" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib138">Weinert et al., 2015</a><div data-db-target-of="bib138-d13420944e2051" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Weinert B.T. </li><li class="loa__item">Moustafa T. </li><li class="loa__item">Iesmantavicius V. </li><li class="loa__item">Zechner R. </li><li class="loa__item">Choudhary C. </li></ul><div class="reference-citations__detials">Analysis of acetylation stoichiometry suggests that SIRT3 repairs nonenzymatic acetylation lesions.<div class="popup-ref"><span class="ref__series"><i>EMBO J.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">34</span></span>: <span class="ref__seriesPages">2620-2632</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.15252%2Fembj.201591271&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26358839&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84946488466&amp;cf=" onclick="newWindow(this.href);return false">Scopus (80)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2015&pages=2620-2632&journal=EMBO+J.&author=Weinert+B.T.&author=Moustafa+T.&author=Iesmantavicius+V.&author=Zechner+R.&author=Choudhary+C.&title=Analysis+of+acetylation+stoichiometry+suggests+that+SIRT3+repairs+nonenzymatic+acetylation+lesions">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Although SIRT3 inhibition modulates cell survival and proliferation (<div class="dropBlock reference-citations"><a href="#bib1" id="back-bib1" data-db-target-for="bib1-d13420944e2054" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib1">Alhazzazi et al., 2016</a><div data-db-target-of="bib1-d13420944e2054" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Alhazzazi T.Y. </li><li class="loa__item">Kamarajan P. </li><li class="loa__item">Xu Y. </li><li class="loa__item">Ai T. </li><li class="loa__item">Chen L. </li><li class="loa__item">Verdin E. </li><li class="loa__item">Kapila Y.L. </li></ul><div class="reference-citations__detials">A Novel Sirtuin-3 Inhibitor, LC-0296, Inhibits Cell Survival and Proliferation, and Promotes Apoptosis of Head and Neck Cancer Cells.<div class="popup-ref"><span class="ref__series"><i>Anticancer Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">36</span></span>: <span class="ref__seriesPages">49-60</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26722027&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2016&pages=49-60&journal=Anticancer+Res.&author=Alhazzazi+T.Y.&author=Kamarajan+P.&author=Xu+Y.&author=Ai+T.&author=Chen+L.&author=Verdin+E.&author=Kapila+Y.L.&title=A+Novel+Sirtuin-3+Inhibitor%2C+LC-0296%2C+Inhibits+Cell+Survival+and+Proliferation%2C+and+Promotes+Apoptosis+of+Head+and+Neck+Cancer+Cells">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), SIRT3 has roles as a tumor suppressor and an oncogene (<div class="dropBlock reference-citations"><a href="#bib23" id="back-bib23" data-db-target-for="bib23-d13420944e2057" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib23">Chen et al., 2014</a><div data-db-target-of="bib23-d13420944e2057" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Chen Y. </li><li class="loa__item">Fu L.L. </li><li class="loa__item">Wen X. </li><li class="loa__item">Wang X.Y. </li><li class="loa__item">Liu J. </li><li class="loa__item">Cheng Y. </li><li class="loa__item">Huang J. </li></ul><div class="reference-citations__detials">Sirtuin-3 (SIRT3), a therapeutic target with oncogenic and tumor-suppressive function in cancer.<div class="popup-ref"><span class="ref__series"><i>Cell Death Dis.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">e1047</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fcddis.2014.14&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=24503539&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84896756082&amp;cf=" onclick="newWindow(this.href);return false">Scopus (119)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=e1047&journal=Cell+Death+Dis.&author=Chen+Y.&author=Fu+L.L.&author=Wen+X.&author=Wang+X.Y.&author=Liu+J.&author=Cheng+Y.&author=Huang+J.&title=Sirtuin-3+%28SIRT3%29%2C+a+therapeutic+target+with+oncogenic+and+tumor-suppressive+function+in+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib143" id="back-bib143" data-db-target-for="bib143-d13420944e2060" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib143">Xiong et al., 2016</a><div data-db-target-of="bib143-d13420944e2060" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Xiong Y. </li><li class="loa__item">Wang M. </li><li class="loa__item">Zhao J. </li><li class="loa__item">Han Y. </li><li class="loa__item">Jia L. </li></ul><div class="reference-citations__detials">Sirtuin 3: A Janus face in cancer (Review).<div class="popup-ref"><span class="ref__series"><i>Int. J. Oncol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">49</span></span>: <span class="ref__seriesPages">2227-2235</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.3892%2Fijo.2016.3767&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27840909&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84996557477&amp;cf=" onclick="newWindow(this.href);return false">Scopus (15)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2016&pages=2227-2235&journal=Int.+J.+Oncol.&author=Xiong+Y.&author=Wang+M.&author=Zhao+J.&author=Han+Y.&author=Jia+L.&title=Sirtuin+3%3A+A+Janus+face+in+cancer+%28Review%29">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), leaving open the question of whether SIRT3 is a viable therapeutic target.</div><div class="section-paragraph">Unsupervised clustering of nuclear protein Ac revealed two subgroups of NMF-Basal-I tumors (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>E; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">Figure S3</a>A). The nuclear Ac Basal-I cluster 1 (N-Ac Basal-I C1) showed significantly higher protein mean expression levels for multiple DNA repair pathways, such as the base excision repair (BER), nucleotide excision repair (NER), double-strand break repair (DSBR), single-strand break repair (SSBR), homologous recombination (HR), and Fanconi anemia pathways than the other N-Ac Basal-I cluster (C3) (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>F). <a href="/cms/10.1016/j.cell.2020.10.036/attachment/eb237fc7-15fb-4b8d-bf02-cae5f285b239/mmc6" class="scroll-into-link">Table S6</a> includes mean expression levels for unique proteins from specific repair pathways (<div class="dropBlock reference-citations"><a href="#bib4" id="back-bib4" data-db-target-for="bib4-d13420944e2077" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib4">Anurag et al., 2018a</a><div data-db-target-of="bib4-d13420944e2077" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Anurag M. </li><li class="loa__item">Punturi N. </li><li class="loa__item">Hoog J. </li><li class="loa__item">Bainbridge M.N. </li><li class="loa__item">Ellis M.J. </li><li class="loa__item">Haricharan S. </li></ul><div class="reference-citations__detials">Comprehensive Profiling of DNA Repair Defects in Breast Cancer Identifies a Novel Class of Endocrine Therapy Resistance Drivers.<div class="popup-ref"><span class="ref__series"><i>Clin. Cancer Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">4887-4899</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1158%2F1078-0432.CCR-17-3702&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29793947&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85054053323&amp;cf=" onclick="newWindow(this.href);return false">Scopus (33)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=4887-4899&journal=Clin.+Cancer+Res.&author=Anurag+M.&author=Punturi+N.&author=Hoog+J.&author=Bainbridge+M.N.&author=Ellis+M.J.&author=Haricharan+S.&title=Comprehensive+Profiling+of+DNA+Repair+Defects+in+Breast+Cancer+Identifies+a+Novel+Class+of+Endocrine+Therapy+Resistance+Drivers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) as well as for more inclusive SSBR and DSBR gene sets. The two N-Ac Basal-I clusters were distinguished by differential Ac of a number of specific Ac sites without change in the corresponding protein levels (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>G; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs5" href="#figs5" data-locator="figs5" id="back-figs5" class="figure-link scroll-into-link">Figure S5</a>E). These differentially acetylated proteins were enriched for nucleoplasmic proteins, RNA metabolism, chromatin-modifying enzymes, and histone Ac by the histone acetyltransferase (HAT) pathway (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs5" href="#figs5" data-locator="figs5" id="back-figs5" class="figure-link scroll-into-link">Figure S5</a>F). Interestingly, elevated Ac in the activation loop of CREBBP-K1591K1592 may explain the observed hyperacetylation of nuclear proteins in N-Ac C1 (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>H). The presence of active CREBBP was suggested by high Ac of multiple histone H2B N-terminal Ac sites (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>H), as observed previously (<div class="dropBlock reference-citations"><a href="#bib139" id="back-bib139" data-db-target-for="bib139-d13420944e2096" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib139">Weinert et al., 2018</a><div data-db-target-of="bib139-d13420944e2096" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Weinert B.T. </li><li class="loa__item">Narita T. </li><li class="loa__item">Satpathy S. </li><li class="loa__item">Srinivasan B. </li><li class="loa__item">Hansen B.K. </li><li class="loa__item">Schölz C. </li><li class="loa__item">Hamilton W.B. </li><li class="loa__item">Zucconi B.E. </li><li class="loa__item">Wang W.W. </li><li class="loa__item">Liu W.R. </li><li>et al.</li></ul><div class="reference-citations__detials">Time-Resolved Analysis Reveals Rapid Dynamics and Broad Scope of the CBP/p300 Acetylome.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">174</span></span>: <span class="ref__seriesPages">231-244.e12</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2018.04.033&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2018.04.033&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2018.04.033&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29804834&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85047227031&amp;cf=" onclick="newWindow(this.href);return false">Scopus (108)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=174&publication_year=2018&pages=231-244.e12&journal=Cell&author=Weinert+B.T.&author=Narita+T.&author=Satpathy+S.&author=Srinivasan+B.&author=Hansen+B.K.&author=Sch%C3%B6lz+C.&author=Hamilton+W.B.&author=Zucconi+B.E.&author=Wang+W.W.&author=Liu+W.R.&title=Time-Resolved+Analysis+Reveals+Rapid+Dynamics+and+Broad+Scope+of+the+CBP%2Fp300+Acetylome">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Other lysine acetyltransferases (KAT7 and KAT6A/B) and their complex partners (JADE3, BRPF3, BRD1, ING4, and MEAF6) were also hyperacetylated in N-Ac C1, although the effect of Ac on these proteins is largely unexplored. However, the increased Ac of histone H4 at site K13 and H3.3 at site K15 (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>H), known targets of KAT7 acetyltransferase (<div class="dropBlock reference-citations"><a href="#bib87" id="back-bib87" data-db-target-for="bib87-d13420944e2103" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib87">Miotto and Struhl, 2010</a><div data-db-target-of="bib87-d13420944e2103" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Miotto B. </li><li class="loa__item">Struhl K. </li></ul><div class="reference-citations__detials">HBO1 histone acetylase activity is essential for DNA replication licensing and inhibited by Geminin.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">37</span></span>: <span class="ref__seriesPages">57-66</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.molcel.2009.12.012&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.molcel.2009.12.012&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.molcel.2009.12.012&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=20129055&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-73649089696&amp;cf=" onclick="newWindow(this.href);return false">Scopus (174)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2010&pages=57-66&journal=Mol.+Cell&author=Miotto+B.&author=Struhl+K.&title=HBO1+histone+acetylase+activity+is+essential+for+DNA+replication+licensing+and+inhibited+by+Geminin">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib88" id="back-bib88" data-db-target-for="bib88-d13420944e2106" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib88">Mishima et al., 2011</a><div data-db-target-of="bib88-d13420944e2106" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mishima Y. </li><li class="loa__item">Miyagi S. </li><li class="loa__item">Saraya A. </li><li class="loa__item">Negishi M. </li><li class="loa__item">Endoh M. </li><li class="loa__item">Endo T.A. </li><li class="loa__item">Toyoda T. </li><li class="loa__item">Shinga J. </li><li class="loa__item">Katsumoto T. </li><li class="loa__item">Chiba T. </li><li>et al.</li></ul><div class="reference-citations__detials">The Hbo1-Brd1/Brpf2 complex is responsible for global acetylation of H3K14 and required for fetal liver erythropoiesis.<div class="popup-ref"><span class="ref__series"><i>Blood.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">118</span></span>: <span class="ref__seriesPages">2443-2453</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1182%2Fblood-2011-01-331892&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=21753189&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-80052408223&amp;cf=" onclick="newWindow(this.href);return false">Scopus (114)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2011&pages=2443-2453&journal=Blood&author=Mishima+Y.&author=Miyagi+S.&author=Saraya+A.&author=Negishi+M.&author=Endoh+M.&author=Endo+T.A.&author=Toyoda+T.&author=Shinga+J.&author=Katsumoto+T.&author=Chiba+T.&title=The+Hbo1-Brd1%2FBrpf2+complex+is+responsible+for+global+acetylation+of+H3K14+and+required+for+fetal+liver+erythropoiesis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), suggests higher activity in N-Ac Basal-I C1. Finally, both subunits of the Ku70/80 complex from the non-homologous end joining (NHEJ) pathway demonstrated elevated Ac of Ac sites located in the DNA-PK binding (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>H, XRCC5-K702) and C-terminal arm domains (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>H, XRCC6-K516).</div></section><section id="sec2.5" data-locator="sec2.5" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Proteogenomics Analysis of ERBB2+ BRCAs</span></h3><div class="section-paragraph">We recently explored ERBB2 status using microscaled proteogenomics analyses of core needle breast cancer biopsy specimens from ERBB2+ BRCA patients treated with neoadjuvant anti-ERBB2 antibody therapy (DP1; <div class="dropBlock reference-citations"><a href="#bib113" id="back-bib113" data-db-target-for="bib113-d13420944e2120" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib113">Satpathy et al., 2020</a><div data-db-target-of="bib113-d13420944e2120" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Satpathy S. </li><li class="loa__item">Jaehnig E.J. </li><li class="loa__item">Krug K. </li><li class="loa__item">Kim B.-J. </li><li class="loa__item">Saltzman A.B. </li><li class="loa__item">Chan D.W. </li><li class="loa__item">Holloway K.R. </li><li class="loa__item">Anurag M. </li><li class="loa__item">Huang C. </li><li class="loa__item">Singh P. </li><li>et al.</li></ul><div class="reference-citations__detials">Microscaled proteogenomic methods for precision oncology.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">532</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41467-020-14381-2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31988290&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85078365239&amp;cf=" onclick="newWindow(this.href);return false">Scopus (25)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=532&journal=Nat.+Commun.&author=Satpathy+S.&author=Jaehnig+E.J.&author=Krug+K.&author=Kim+B.-J.&author=Saltzman+A.B.&author=Chan+D.W.&author=Holloway+K.R.&author=Anurag+M.&author=Huang+C.&author=Singh+P.&title=Microscaled+proteogenomic+methods+for+precision+oncology">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). In addition to an unresponsive tumor lacking <em>ERBB2</em> amplification by exome sequencing and ERBB2 protein by mass spectrometry, these analyses determined that two treatment-resistant cases (of a total of 13 cases with <em>ERBB2</em> gene amplification) had “pseudo-ERBB2+” status, with low-level ERBB2 protein expression (more similar to non-amplified cases than amplified cases with pathologic complete response [pCR]) despite evidence of <em>ERBB2</em> amplification by exome sequencing (DP1 samples in <a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>A; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">Figure S6</a>A). Because these pseudo-ERBB2+ samples are examples where anti-ERBB2 treatment may not have been effective because of lack of drug target expression, proteogenomics approaches were used to assess ERBB2 driver status in the current dataset and our earlier cohort (<div class="dropBlock reference-citations"><a href="#bib85" id="back-bib85" data-db-target-for="bib85-d13420944e2138" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib85">Mertins et al., 2016</a><div data-db-target-of="bib85-d13420944e2138" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Wang P. </li><li class="loa__item">Wang X. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Cao S. </li><li class="loa__item">Petralia F. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomics connects somatic mutations to signalling in breast cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (780)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=NCI+CPTAC&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figures 3</a>A and 3B; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">Figures S6</a>A and S6B). Analysis of the current cohort classified 15 tumors proteogenomically as ERBB2+ (PG+) (“Prospective” samples in <a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figures 3</a>A and 3B; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">Figures S6</a>A and S6B). Central immunohistochemistry (IHC) testing was used to refine ERBB2 status where possible (68 tumors), and all of the ERBB2 PG+ samples were classified according to ASCO-CAP guidelines (<a class="inter-ref-link" href="https://www.cap.org/" target="_blank">https://www.cap.org/</a>) as ERBB2+ (IHC score of 3+ or IHC score of 2+ and amplified by fluorescence <em>in situ</em> hybridization [FISH]) or with equivocal status (IHC score of 2+ without FISH results or amplified by FISH without IHC results). Similar to the data in DP1, cases of pseudo-ERBB2 positivity were identified, with two of 17 instances of <em>ERBB2</em> gene amplification in the current cohort and one of 16 in the retrospective cohort being associated with protein expression levels that were within the distribution for ERBB2 PG− samples (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figures 3</a>A and 3B; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">Figures S6</a>A and S6B). The DP1 study also identified a pseudo-ERBB2+ case with amplification and overexpression of TOP2A, suggesting an alternative chromosome 17 amplicon driver in some cases (<div class="dropBlock reference-citations"><a href="#bib49" id="back-bib49" data-db-target-for="bib49-d13420944e2168" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib49">Harris et al., 2009</a><div data-db-target-of="bib49-d13420944e2168" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Harris L.N. </li><li class="loa__item">Broadwater G. </li><li class="loa__item">Abu-Khalaf M. </li><li class="loa__item">Cowan D. </li><li class="loa__item">Thor A.D. </li><li class="loa__item">Budman D. </li><li class="loa__item">Cirrincione C.T. </li><li class="loa__item">Berry D.A. </li><li class="loa__item">Winer E.P. </li><li class="loa__item">Hudis C.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Topoisomerase IIalpha amplification does not predict benefit from dose-intense cyclophosphamide, doxorubicin, and fluorouracil therapy in HER2-amplified early breast cancer: results of CALGB 8541/150013.<div class="popup-ref"><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">27</span></span>: <span class="ref__seriesPages">3430-3436</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1200%2FJCO.2008.18.4085&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=19470942&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-70249085601&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2009&pages=3430-3436&journal=J.%C2%A0Clin.+Oncol.&author=Harris+L.N.&author=Broadwater+G.&author=Abu-Khalaf+M.&author=Cowan+D.&author=Thor+A.D.&author=Budman+D.&author=Cirrincione+C.T.&author=Berry+D.A.&author=Winer+E.P.&author=Hudis+C.A.&title=Topoisomerase+IIalpha+amplification+does+not+predict+benefit+from+dose-intense+cyclophosphamide%2C+doxorubicin%2C+and+fluorouracil+therapy+in+HER2-amplified+early+breast+cancer%3A+results+of+CALGB+8541%2F150013">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Supporting this hypothesis, <em>TOP2A</em> amplification and protein overexpression in the absence of ERBB2 protein overexpression were observed in one pseudo-ERBB2+ case each in the present and retrospective cohorts (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>A; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">Figure S6</a>A).<div class="floatDisplay"><figure id="gr3" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/49f80b3a-0ff1-4741-b996-f2e67fa76794/gr3.jpg" alt="Figure thumbnail gr3" data-locator="fig3" data-high-res="/cms/attachment/4f38416b-e532-4972-bbd4-3fae9b9ba3be/gr3_lrg.jpg" data-large-image="/cms/attachment/49f80b3a-0ff1-4741-b996-f2e67fa76794/gr3.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 3" class="label figure__label">Figure 3</span><span class="figure__title__text">PG Classification of ERBB2 Tumors</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Proteogenomics analysis of the <em>ERBB2</em> locus in this study (“Prospective”), biopsies from ERBB2+ BRCA tumors (“DP1”; <div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib113" id="back-bib113" data-db-target-for="bib113-d13420944e106" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib113">Satpathy et al., 2020</a><div data-db-target-of="bib113-d13420944e106" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Satpathy S. </li><li class="loa__item">Jaehnig E.J. </li><li class="loa__item">Krug K. </li><li class="loa__item">Kim B.-J. </li><li class="loa__item">Saltzman A.B. </li><li class="loa__item">Chan D.W. </li><li class="loa__item">Holloway K.R. </li><li class="loa__item">Anurag M. </li><li class="loa__item">Huang C. </li><li class="loa__item">Singh P. </li><li>et al.</li></ul><div class="reference-citations__detials">Microscaled proteogenomic methods for precision oncology.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">532</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41467-020-14381-2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31988290&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85078365239&amp;cf=" onclick="newWindow(this.href);return false">Scopus (25)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=532&journal=Nat.+Commun.&author=Satpathy+S.&author=Jaehnig+E.J.&author=Krug+K.&author=Kim+B.-J.&author=Saltzman+A.B.&author=Chan+D.W.&author=Holloway+K.R.&author=Anurag+M.&author=Huang+C.&author=Singh+P.&title=Microscaled+proteogenomic+methods+for+precision+oncology">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div>), and TCGA tumors (“Retrospective”; <div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib85" id="back-bib85" data-db-target-for="bib85-d13420944e109" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib85">Mertins et al., 2016</a><div data-db-target-of="bib85-d13420944e109" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Wang P. </li><li class="loa__item">Wang X. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Cao S. </li><li class="loa__item">Petralia F. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomics connects somatic mutations to signalling in breast cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (780)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=NCI+CPTAC&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div>). The heatmap depicts clinical data (top panel), copy number alterations (center panel), and protein expression (bottom panel) of genes proximal to <em>ERBB2</em> on chromosome 17q for samples that were PAM50 HER2E, clinical ERBB2+/equivocal by immunohistochemistry (IHC) and/or <em>in situ</em> hybridization (ISH), or ERBB2 PG+. PG amplification of TOP2A, a potential alternative driver in the locus, is indicated by red arrowheads.</div><div class="section-paragraph">(B) Outlier analysis of ERBB2 and STARD3 or GRB7 confirms higher protein levels in most ERBB2-amplified samples (purple histogram) relative to the distribution of ERBB2 protein in non-amplified samples (blue histogram) in the prospective and retrospective datasets. Amplified samples with protein levels falling within the distribution of ERBB2 non-amplified samples are considered “pseudo-ERBB2+.”</div><div class="section-paragraph">(C) Phosphopeptide levels for components of the KEGG ErbB signaling pathway in HER2-associated tumors (PAM50 HER2E and ERBB2 PG+). The top panel of the heatmap shows subtype classifications and clinical marker status for each of these samples, and the bottom panel indicates somatic copy number aberrations (SCNAs) for genes in the amplicon closely linked to <em>ERBB2</em>, followed by the corresponding protein levels. The bottom panel depicts abundances of phosphopeptides from the ERBB2 pathway.</div><div class="section-paragraph">See also <a data-locator="figs6" aria-expanded="false" aria-controls="crosref2475" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs6">Figure S6</a> and <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref2480" href="/cms/10.1016/j.cell.2020.10.036/attachment/3dbb5a07-b92f-4ae5-9c88-f84c3dca95db/mmc1" target="_blank">Tables S1</a> and <a data-locator="mmc2" aria-expanded="false" aria-controls="crosref2485" href="/cms/10.1016/j.cell.2020.10.036/attachment/a1bc3f3e-eeac-4605-a9ef-839abd334dcd/mmc2" target="_blank">S2</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/49f80b3a-0ff1-4741-b996-f2e67fa76794/gr3.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/4f38416b-e532-4972-bbd4-3fae9b9ba3be/gr3_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(20)31400-8&amp;id=gr3.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div><div class="floatDisplay"><figure id="figs6" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/d73bc0cd-58af-4364-b0a2-7b30a9a2e050/figs6.jpg" alt="Figure thumbnail figs6" data-locator="figs6" data-high-res="/cms/attachment/8d855f7a-6059-422b-ba27-32c7f9e835e8/figs6_lrg.jpg" data-large-image="/cms/attachment/d73bc0cd-58af-4364-b0a2-7b30a9a2e050/figs6.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. S6" class="label figure__label">Figure S6</span><span class="figure__title__text">PG Classification of HER2, Related to <a data-locator="fig3" aria-expanded="false" aria-controls="crosref2755" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig3">Figure 3</a></span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">A) Proteogenomic analysis of the <em>ERBB2</em> amplicon for all samples from each dataset shown in <a data-locator="fig3" aria-expanded="false" aria-controls="crosref2760" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig3">Figure 3</a>A.</div><div class="section-paragraph">B) Assessment and frequencies of clinical ERBB2 positive/equivocal samples, <em>ERBB2</em> gene-amplified, and ERBB2 proteogenomically (PG) positive samples in each dataset shown in <a data-locator="fig3" aria-expanded="false" aria-controls="crosref2765" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig3">Figure 3</a>A. Clinical assessment was refined after centralized IHC (see <a data-locator="sec4" aria-expanded="false" aria-controls="crosref2770" class=" caption__link scroll-into-link" href="&#xA;                                #sec4">STAR Methods</a>), and discrepant cases represent those for which clinical status was ERBB2 positive or equivocal, but PG status was negative. “Pseudo-ERBB2 +” cases are those for which amplicon genes were amplified without a corresponding increase in protein expression.</div><div class="section-paragraph">C) Heatmap illustrating the RNA abundance levels of 49 PAM50 genes detected in this dataset. Rows and columns are hierarchically clustered.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/d73bc0cd-58af-4364-b0a2-7b30a9a2e050/figs6.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/8d855f7a-6059-422b-ba27-32c7f9e835e8/figs6_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(20)31400-8&amp;id=figs6.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">The lack of close alignment between ERBB2 positivity and intrinsic subtype was also investigated. Only seven of 15 ERBB2 PG+ samples were classified as HER2E by PAM50 subtyping, whereas an additional seven HER2E samples were not ERBB2 PG+ (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>A; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">Figure S6</a>C). To better understand biological characteristics that cause samples to cluster within the HER2E group despite inconsistent ERBB2 status, an analysis of phosphosites from the human Kyoto Encyclopedia of Genes and Genomes (KEGG) (<div class="dropBlock reference-citations"><a href="#bib56" id="back-bib56" data-db-target-for="bib56-d13420944e2187" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib56">Kanehisa and Goto, 2000</a><div data-db-target-of="bib56-d13420944e2187" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kanehisa M. </li><li class="loa__item">Goto S. </li></ul><div class="reference-citations__detials">KEGG: kyoto encyclopedia of genes and genomes.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2000; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">27-30</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2F28.1.27&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10592173&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=27-30&journal=Nucleic+Acids+Res.&author=Kanehisa+M.&author=Goto+S.&title=KEGG%3A+kyoto+encyclopedia+of+genes+and+genomes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) ERBB signaling pathway (hsa04012) was performed. As expected, all PAM50 HER2E/ERBB2 PG+ samples had high levels of ERBB2 phosphopeptides, whereas PAM50 HER2E/ERBB2 PG− samples had markedly lower levels but showed elevated levels of phosphorylation of other ERBB family members and of the mitogen-activated protein kinase (MAPK) signaling pathway compared with PG+ samples (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>C). This suggests that alternative drivers of ERBB signaling could be targeted in PAM50 HER2E tumors without <em>ERBB2</em> amplification.</div></section><section id="sec2.6" data-locator="sec2.6" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">PG Analysis of the Immune Tumor Microenvironment (I-TME) Suggests Broader Applicability of Immunotherapy in BRCA</span></h3><div class="section-paragraph">RNA-based immune cell deconvolution signatures and protein-level signatures for immune modulators (<div class="dropBlock reference-citations"><a href="#bib128" id="back-bib128" data-db-target-for="bib128-d13420944e2200" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib128">Thorsson et al., 2019</a><div data-db-target-of="bib128-d13420944e2200" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Thorsson V. </li><li class="loa__item">Gibbs D.L. </li><li class="loa__item">Brown S.D. </li><li class="loa__item">Wolf D. </li><li class="loa__item">Bortone D.S. </li><li class="loa__item">Ou Yang T.-H. </li><li class="loa__item">Porta-Pardo E. </li><li class="loa__item">Gao G.F. </li><li class="loa__item">Plaisier C.L. </li><li class="loa__item">Eddy J.A. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />The Immune Landscape of Cancer.<div class="popup-ref"><span class="ref__series"><i>Immunity.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">51</span></span>: <span class="ref__seriesPages">411-412</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.immuni.2019.08.004&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.immuni.2019.08.004&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.immuni.2019.08.004&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31433971&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85070213901&amp;cf=" onclick="newWindow(this.href);return false">Scopus (61)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2019&pages=411-412&journal=Immunity&author=Cancer+Genome+Atlas+Research+Network&title=The+Immune+Landscape+of+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) revealed a range of immune-related features across all four intrinsic subtypes (<a href="/cms/10.1016/j.cell.2020.10.036/attachment/eb237fc7-15fb-4b8d-bf02-cae5f285b239/mmc6" class="scroll-into-link">Table S6</a>), including the immune checkpoint proteins PD1 and PD-L1 at the RNA and phosphosite levels (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figures 4</a>A and 4B ; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">Figures S7</a>A and S7B). Anti-CD3 (pan-T cell) IHC validated evidence of an active I-TME (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figures 4</a>A, 4C, and 4D), with significant correlations between CD3+ T cell tumor infiltration and RNA-based estimates of overall I-TME provided by CIBERSORT absolute scores (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>E). A stimulatory immune modulator protein signature was even more highly correlated with the IHC I-TME data (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>F). RNA level profiles inferred for individual acquired and innate immune cell types (including B cells; cytotoxic, helper, and regulatory T cells; natural killer (NK) cells; dendritic cells (DCs); eosinophils; and macrophages) generally tracked with CIBERSORT absolute scores in each subtype (e.g., cluster 1; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">Figure S7</a>A). However, effector memory CD4+ and activated CD4+ and CD8+ signatures, which do not appear to be well correlated with the CIBERSORT score, were lower in the PAM50 LumA subtype than in all other subtypes (cluster 3; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">Figure S7</a>A). Furthermore, interferon gamma (IFNG) and antigen presentation machinery (APM1) protein signatures (<div class="dropBlock reference-citations"><a href="#bib128" id="back-bib128" data-db-target-for="bib128-d13420944e2231" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib128">Thorsson et al., 2019</a><div data-db-target-of="bib128-d13420944e2231" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Thorsson V. </li><li class="loa__item">Gibbs D.L. </li><li class="loa__item">Brown S.D. </li><li class="loa__item">Wolf D. </li><li class="loa__item">Bortone D.S. </li><li class="loa__item">Ou Yang T.-H. </li><li class="loa__item">Porta-Pardo E. </li><li class="loa__item">Gao G.F. </li><li class="loa__item">Plaisier C.L. </li><li class="loa__item">Eddy J.A. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />The Immune Landscape of Cancer.<div class="popup-ref"><span class="ref__series"><i>Immunity.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">51</span></span>: <span class="ref__seriesPages">411-412</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.immuni.2019.08.004&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.immuni.2019.08.004&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.immuni.2019.08.004&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31433971&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85070213901&amp;cf=" onclick="newWindow(this.href);return false">Scopus (61)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2019&pages=411-412&journal=Immunity&author=Cancer+Genome+Atlas+Research+Network&title=The+Immune+Landscape+of+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) appeared to track the immune score in all PAM50 subtypes except LumA, where they were lower than in Basal (cluster 4; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">Figure S7</a>A). Finally, stromal, fibroblast, mast cell, endothelial cell, and neutrophil signatures were elevated in PAM50 LumA tumors with higher CIBERSORT scores but lower overall in LumB and Basal tumors (cluster 2; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">Figure S7</a>A). Thus, acquired immune response hallmarks generally appear not to be activated in LumA, whereas other PAM50 subtypes, including LumB, exhibit features consistent with activation of acquired immunity, a finding consistent with previous comprehensive immune cell profiling of the METABRIC dataset (<div class="dropBlock reference-citations"><a href="#bib34" id="back-bib34" data-db-target-for="bib34-d13420944e2241" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib34">Curtis et al., 2012</a><div data-db-target-of="bib34-d13420944e2241" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Curtis C. </li><li class="loa__item">Shah S.P. </li><li class="loa__item">Chin S.-F. </li><li class="loa__item">Turashvili G. </li><li class="loa__item">Rueda O.M. </li><li class="loa__item">Dunning M.J. </li><li class="loa__item">Speed D. </li><li class="loa__item">Lynch A.G. </li><li class="loa__item">Samarajiwa S. </li><li class="loa__item">Yuan Y. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>METABRIC Group</span><br />The genomic and transcriptomic architecture of 2,000 breast tumours reveals novel subgroups.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">486</span></span>: <span class="ref__seriesPages">346-352</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature10983&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=22522925&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84861527388&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3169)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=486&publication_year=2012&pages=346-352&journal=Nature&author=METABRIC+Group&title=The+genomic+and+transcriptomic+architecture+of+2%2C000+breast+tumours+reveals+novel+subgroups">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib133" id="back-bib133" data-db-target-for="bib133-d13420944e2244" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib133">Varn et al., 2016</a><div data-db-target-of="bib133-d13420944e2244" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Varn F.S. </li><li class="loa__item">Andrews E.H. </li><li class="loa__item">Mullins D.W. </li><li class="loa__item">Cheng C. </li></ul><div class="reference-citations__detials">Integrative analysis of breast cancer reveals prognostic haematopoietic activity and patient-specific immune response profiles.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">10248</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fncomms10248&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26725977&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84953306200&amp;cf=" onclick="newWindow(this.href);return false">Scopus (29)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=10248&journal=Nat.+Commun.&author=Varn+F.S.&author=Andrews+E.H.&author=Mullins+D.W.&author=Cheng+C.&title=Integrative+analysis+of+breast+cancer+reveals+prognostic+haematopoietic+activity+and+patient-specific+immune+response+profiles">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). This analysis extends recent reports suggesting that a significant proportion of PAM50 LumB tumors have an active I-TME and overexpress immune checkpoint and STAT1/IFNG genes (<div class="dropBlock reference-citations"><a href="#bib6" id="back-bib6" data-db-target-for="bib6-d13420944e2247" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib6">Anurag et al., 2020</a><div data-db-target-of="bib6-d13420944e2247" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Anurag M. </li><li class="loa__item">Zhu M. </li><li class="loa__item">Huang C. </li><li class="loa__item">Vasaikar S. </li><li class="loa__item">Wang J. </li><li class="loa__item">Hoog J. </li><li class="loa__item">Burugu S. </li><li class="loa__item">Gao D. </li><li class="loa__item">Suman V. </li><li class="loa__item">Zhang X.H. </li><li>et al.</li></ul><div class="reference-citations__detials">Immune Checkpoint Profiles in Luminal B Breast Cancer (Alliance).<div class="popup-ref"><span class="ref__series"><i>J. Natl. Cancer Inst.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">112</span></span>: <span class="ref__seriesPages">737-746</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fjnci%2Fdjz213&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31665365&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85079588411&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2020&pages=737-746&journal=J.%C2%A0Natl.+Cancer+Inst&author=Anurag+M.&author=Zhu+M.&author=Huang+C.&author=Vasaikar+S.&author=Wang+J.&author=Hoog+J.&author=Burugu+S.&author=Gao+D.&author=Suman+V.&author=Zhang+X.H.&title=Immune+Checkpoint+Profiles+in+Luminal+B+Breast+Cancer+%28Alliance%29">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). This suggests that immunotherapy should be considered for subsets of luminal tumors with an active I-TME.<div class="floatDisplay"><figure id="gr4" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/8613c5b0-c639-426b-a03e-21405ff3f4e2/gr4.jpg" alt="Figure thumbnail gr4" data-locator="fig4" data-high-res="/cms/attachment/648f379f-c96f-4d9d-991d-4b227c721905/gr4_lrg.jpg" data-large-image="/cms/attachment/8613c5b0-c639-426b-a03e-21405ff3f4e2/gr4.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 4" class="label figure__label">Figure 4</span><span class="figure__title__text">Immunological Landscape of BRCA</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Heatmap showing the wide range of expression levels for immune-related features in each PAM50 subtype. <em>Z</em> scores of RNA-based immune signatures from CIBERSORT, ESTIMATE, and xCell and for protein-derived signatures for immune modulator gene sets from <div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib128" id="back-bib128" data-db-target-for="bib128-d13420944e143" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib128">Thorsson et al., 2019</a><div data-db-target-of="bib128-d13420944e143" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Thorsson V. </li><li class="loa__item">Gibbs D.L. </li><li class="loa__item">Brown S.D. </li><li class="loa__item">Wolf D. </li><li class="loa__item">Bortone D.S. </li><li class="loa__item">Ou Yang T.-H. </li><li class="loa__item">Porta-Pardo E. </li><li class="loa__item">Gao G.F. </li><li class="loa__item">Plaisier C.L. </li><li class="loa__item">Eddy J.A. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />The Immune Landscape of Cancer.<div class="popup-ref"><span class="ref__series"><i>Immunity.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">51</span></span>: <span class="ref__seriesPages">411-412</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.immuni.2019.08.004&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.immuni.2019.08.004&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.immuni.2019.08.004&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31433971&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85070213901&amp;cf=" onclick="newWindow(this.href);return false">Scopus (61)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2019&pages=411-412&journal=Immunity&author=Cancer+Genome+Atlas+Research+Network&title=The+Immune+Landscape+of+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div> are shown in the top two data panels. The third data panel shows log2 ratios for normalized RNA-seq and proteomics data (phosphoprotein is the median for all sites on a given protein) for FDA-approved immune checkpoint targets PD-L1, PD1, and CTLA4. The bottom panel shows CD3 IHC results for samples available for centralized IHC. Within each subtype, samples are ordered by increasing CIBERSORT immune score.</div><div class="section-paragraph">(B) Distribution of CIBERSORT immune scores in each PAM50 subtype. Boxplots show 1.5× the interquartile range for each group, centered on the median.</div><div class="section-paragraph">(C) Representative images for CD3 IHC for samples classified as CD3− (top) and CD3-excluded (bottom).</div><div class="section-paragraph">(D) Images showing examples of CD3+ samples with elevated CIBERSORT scores in each PAM50 subtype.</div><div class="section-paragraph">(E) Spearman-rank correlation of CD3+ cell counts with CIBERSORT score.</div><div class="section-paragraph">(F) Spearman-rank correlation of CD3+ cell counts with stimulatory immune modulator protein scores.</div><div class="section-paragraph">See also <a data-locator="figs7" aria-expanded="false" aria-controls="crosref2495" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs7">Figure S7</a> and <a data-locator="mmc2" aria-expanded="false" aria-controls="crosref2500" href="/cms/10.1016/j.cell.2020.10.036/attachment/a1bc3f3e-eeac-4605-a9ef-839abd334dcd/mmc2" target="_blank">Tables S2</a> and <a data-locator="mmc6" aria-expanded="false" aria-controls="crosref2505" href="/cms/10.1016/j.cell.2020.10.036/attachment/eb237fc7-15fb-4b8d-bf02-cae5f285b239/mmc6" target="_blank">S6</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/8613c5b0-c639-426b-a03e-21405ff3f4e2/gr4.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/648f379f-c96f-4d9d-991d-4b227c721905/gr4_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(20)31400-8&amp;id=gr4.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div><div class="floatDisplay"><figure id="figs7" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/4ff1008b-c691-4e9e-ae2a-6c118dd7d5b4/figs7.jpg" alt="Figure thumbnail figs7" data-locator="figs7" data-high-res="/cms/attachment/2af00499-a5e7-4f0b-af1a-41d44e03688a/figs7_lrg.jpg" data-large-image="/cms/attachment/4ff1008b-c691-4e9e-ae2a-6c118dd7d5b4/figs7.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. S7" class="label figure__label">Figure S7</span><span class="figure__title__text">Immune Landscape of BRCA and Phospho-Rb-Dependent Kinase Activity in TNBC, Related to <a data-locator="fig4" aria-expanded="false" aria-controls="crosref2775" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig4">Figures 4</a>, <a data-locator="fig5" aria-expanded="false" aria-controls="crosref2780" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig5">5</a>, and <a data-locator="fig6" aria-expanded="false" aria-controls="crosref2785" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig6">6</a></span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">A) Heatmap of scores for immune and stromal cell types inferred from RNA-seq data by MCP-Counter (labeled MCPcounter; <div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib12" id="back-bib12" data-db-target-for="bib12-d13420944e611" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib12">Becht et al., 2016a</a><div data-db-target-of="bib12-d13420944e611" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Becht E. </li><li class="loa__item">Giraldo N.A. </li><li class="loa__item">Lacroix L. </li><li class="loa__item">Buttard B. </li><li class="loa__item">Elarouci N. </li><li class="loa__item">Petitprez F. </li><li class="loa__item">Selves J. </li><li class="loa__item">Laurent-Puig P. </li><li class="loa__item">Sautès-Fridman C. </li><li class="loa__item">Fridman W.H. </li><li class="loa__item">de Reyniès A. </li></ul><div class="reference-citations__detials">Erratum to: Estimating the population abundance of tissue-infiltrating immune and stromal cell populations using gene expression.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">249</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2Fs13059-016-1113-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27908289&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85002746810&amp;cf=" onclick="newWindow(this.href);return false">Scopus (13)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=249&journal=Genome+Biol.&author=Becht+E.&author=Giraldo+N.A.&author=Lacroix+L.&author=Buttard+B.&author=Elarouci+N.&author=Petitprez+F.&author=Selves+J.&author=Laurent-Puig+P.&author=Saut%C3%A8s-Fridman+C.&author=Fridman+W.H.&author=de+Reyni%C3%A8s+A.&title=Erratum+to%3A+Estimating+the+population+abundance+of+tissue-infiltrating+immune+and+stromal+cell+populations+using+gene+expression">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div>, <div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib13" id="back-bib13" data-db-target-for="bib13-d13420944e614" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib13">Becht et al., 2016b</a><div data-db-target-of="bib13-d13420944e614" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Becht E. </li><li class="loa__item">Giraldo N.A. </li><li class="loa__item">Lacroix L. </li><li class="loa__item">Buttard B. </li><li class="loa__item">Elarouci N. </li><li class="loa__item">Petitprez F. </li><li class="loa__item">Selves J. </li><li class="loa__item">Laurent-Puig P. </li><li class="loa__item">Sautès-Fridman C. </li><li class="loa__item">Fridman W.H. </li><li class="loa__item">de Reyniès A. </li></ul><div class="reference-citations__detials">Estimating the population abundance of tissue-infiltrating immune and stromal cell populations using gene expression.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">218</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2Fs13059-016-1070-5&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27765066&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84992053166&amp;cf=" onclick="newWindow(this.href);return false">Scopus (497)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=218&journal=Genome+Biol.&author=Becht+E.&author=Giraldo+N.A.&author=Lacroix+L.&author=Buttard+B.&author=Elarouci+N.&author=Petitprez+F.&author=Selves+J.&author=Laurent-Puig+P.&author=Saut%C3%A8s-Fridman+C.&author=Fridman+W.H.&author=de+Reyni%C3%A8s+A.&title=Estimating+the+population+abundance+of+tissue-infiltrating+immune+and+stromal+cell+populations+using+gene+expression">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div>) and by the immunophenotyping approach described by <div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib3" id="back-bib3" data-db-target-for="bib3-d13420944e617" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib3">Angelova et al., 2015</a><div data-db-target-of="bib3-d13420944e617" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Angelova M. </li><li class="loa__item">Charoentong P. </li><li class="loa__item">Hackl H. </li><li class="loa__item">Fischer M.L. </li><li class="loa__item">Snajder R. </li><li class="loa__item">Krogsdam A.M. </li><li class="loa__item">Waldner M.J. </li><li class="loa__item">Bindea G. </li><li class="loa__item">Mlecnik B. </li><li class="loa__item">Galon J. </li><li class="loa__item">Trajanoski Z. </li></ul><div class="reference-citations__detials">Characterization of the immunophenotypes and antigenomes of colorectal cancers reveals distinct tumor escape mechanisms and novel targets for immunotherapy.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">64</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2Fs13059-015-0620-6&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=25853550&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84931426001&amp;cf=" onclick="newWindow(this.href);return false">Scopus (283)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=64&journal=Genome+Biol.&author=Angelova+M.&author=Charoentong+P.&author=Hackl+H.&author=Fischer+M.L.&author=Snajder+R.&author=Krogsdam+A.M.&author=Waldner+M.J.&author=Bindea+G.&author=Mlecnik+B.&author=Galon+J.&author=Trajanoski+Z.&title=Characterization+of+the+immunophenotypes+and+antigenomes+of+colorectal+cancers+reveals+distinct+tumor+escape+mechanisms+and+novel+targets+for+immunotherapy">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div> (labeled gb), as well as for protein-based Eigen signatures from <div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib128" id="back-bib128" data-db-target-for="bib128-d13420944e620" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib128">Thorsson et al., 2019</a><div data-db-target-of="bib128-d13420944e620" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Thorsson V. </li><li class="loa__item">Gibbs D.L. </li><li class="loa__item">Brown S.D. </li><li class="loa__item">Wolf D. </li><li class="loa__item">Bortone D.S. </li><li class="loa__item">Ou Yang T.-H. </li><li class="loa__item">Porta-Pardo E. </li><li class="loa__item">Gao G.F. </li><li class="loa__item">Plaisier C.L. </li><li class="loa__item">Eddy J.A. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />The Immune Landscape of Cancer.<div class="popup-ref"><span class="ref__series"><i>Immunity.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">51</span></span>: <span class="ref__seriesPages">411-412</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.immuni.2019.08.004&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.immuni.2019.08.004&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.immuni.2019.08.004&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31433971&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85070213901&amp;cf=" onclick="newWindow(this.href);return false">Scopus (61)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2019&pages=411-412&journal=Immunity&author=Cancer+Genome+Atlas+Research+Network&title=The+Immune+Landscape+of+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div> (labeled Eigen_sig). <a data-locator="mmc6" aria-expanded="false" aria-controls="crosref2810" href="/cms/10.1016/j.cell.2020.10.036/attachment/eb237fc7-15fb-4b8d-bf02-cae5f285b239/mmc6" target="_blank">Table S6</a> includes these profile scores, as well as scores for cell types inferred by CIBERSORT and xCell, which showed similar patterns but were excluded from this figure for the sake of clarity and because they had a high prevalence of 0 values. Hierarchical clustering by Euclidean distance across all samples separated these signatures into four clusters: Cluster 1: Immune signatures that track with Cibersort absolute score in all PAM50 subtypes. Cluster 2: Immune signatures that track with Cibersort in PAM50 LumA tumors but show an overall reduction in levels in Basal and LumB tumors, including stromal (ESTIMATE), fibroblast (MCPcounter), mast cell (<div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib3" id="back-bib3" data-db-target-for="bib3-d13420944e627" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib3">Angelova et al., 2015</a><div data-db-target-of="bib3-d13420944e627" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Angelova M. </li><li class="loa__item">Charoentong P. </li><li class="loa__item">Hackl H. </li><li class="loa__item">Fischer M.L. </li><li class="loa__item">Snajder R. </li><li class="loa__item">Krogsdam A.M. </li><li class="loa__item">Waldner M.J. </li><li class="loa__item">Bindea G. </li><li class="loa__item">Mlecnik B. </li><li class="loa__item">Galon J. </li><li class="loa__item">Trajanoski Z. </li></ul><div class="reference-citations__detials">Characterization of the immunophenotypes and antigenomes of colorectal cancers reveals distinct tumor escape mechanisms and novel targets for immunotherapy.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">64</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2Fs13059-015-0620-6&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=25853550&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84931426001&amp;cf=" onclick="newWindow(this.href);return false">Scopus (283)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=64&journal=Genome+Biol.&author=Angelova+M.&author=Charoentong+P.&author=Hackl+H.&author=Fischer+M.L.&author=Snajder+R.&author=Krogsdam+A.M.&author=Waldner+M.J.&author=Bindea+G.&author=Mlecnik+B.&author=Galon+J.&author=Trajanoski+Z.&title=Characterization+of+the+immunophenotypes+and+antigenomes+of+colorectal+cancers+reveals+distinct+tumor+escape+mechanisms+and+novel+targets+for+immunotherapy">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div>), endothelial cell (MCPcounter), and neutrophil (MCPcounter) signatures (p = 2.17x10<sup>−4</sup>, 1.31x10<sup>−4</sup>, 6.35x10<sup>−6</sup>, 9.63x10<sup>−7</sup>, 2.51x10<sup>−3</sup>, respectively for cell type, Wilcoxon rank sum tests comparing LumA to Basal + LumB). Cluster 3: Effector memory and activated CD4+ and activated CD8+ signatures (<div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib3" id="back-bib3" data-db-target-for="bib3-d13420944e646" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib3">Angelova et al., 2015</a><div data-db-target-of="bib3-d13420944e646" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Angelova M. </li><li class="loa__item">Charoentong P. </li><li class="loa__item">Hackl H. </li><li class="loa__item">Fischer M.L. </li><li class="loa__item">Snajder R. </li><li class="loa__item">Krogsdam A.M. </li><li class="loa__item">Waldner M.J. </li><li class="loa__item">Bindea G. </li><li class="loa__item">Mlecnik B. </li><li class="loa__item">Galon J. </li><li class="loa__item">Trajanoski Z. </li></ul><div class="reference-citations__detials">Characterization of the immunophenotypes and antigenomes of colorectal cancers reveals distinct tumor escape mechanisms and novel targets for immunotherapy.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">64</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2Fs13059-015-0620-6&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=25853550&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84931426001&amp;cf=" onclick="newWindow(this.href);return false">Scopus (283)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=64&journal=Genome+Biol.&author=Angelova+M.&author=Charoentong+P.&author=Hackl+H.&author=Fischer+M.L.&author=Snajder+R.&author=Krogsdam+A.M.&author=Waldner+M.J.&author=Bindea+G.&author=Mlecnik+B.&author=Galon+J.&author=Trajanoski+Z.&title=Characterization+of+the+immunophenotypes+and+antigenomes+of+colorectal+cancers+reveals+distinct+tumor+escape+mechanisms+and+novel+targets+for+immunotherapy">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div>) that were lower in the PAM50 LumA subtype than in all other subtypes (p = 5.79x10<sup>−7</sup>, 9.29x10<sup>−15</sup>, and 4.86x10<sup>−14</sup>, respectively, Wilcoxon rank sum tests comparing LumA to all other tumors). Cluster 4: Interferon gamma (IFNG), STAT1, MHC1, and antigen presentation machinery (APM1) protein signatures (<div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib128" id="back-bib128" data-db-target-for="bib128-d13420944e658" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib128">Thorsson et al., 2019</a><div data-db-target-of="bib128-d13420944e658" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Thorsson V. </li><li class="loa__item">Gibbs D.L. </li><li class="loa__item">Brown S.D. </li><li class="loa__item">Wolf D. </li><li class="loa__item">Bortone D.S. </li><li class="loa__item">Ou Yang T.-H. </li><li class="loa__item">Porta-Pardo E. </li><li class="loa__item">Gao G.F. </li><li class="loa__item">Plaisier C.L. </li><li class="loa__item">Eddy J.A. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />The Immune Landscape of Cancer.<div class="popup-ref"><span class="ref__series"><i>Immunity.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">51</span></span>: <span class="ref__seriesPages">411-412</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.immuni.2019.08.004&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.immuni.2019.08.004&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.immuni.2019.08.004&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31433971&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85070213901&amp;cf=" onclick="newWindow(this.href);return false">Scopus (61)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2019&pages=411-412&journal=Immunity&author=Cancer+Genome+Atlas+Research+Network&title=The+Immune+Landscape+of+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div>) that tracked with Cibersort immune score in all PAM50 subtypes except LumA, where they were lower than in Basal (p = 2.08x10<sup>−3</sup>, 6.67x10<sup>−3</sup>, 6.81x10<sup>−5</sup>, and 2.75x10<sup>−5</sup>, respectively, Wilcoxon rank sum tests comparing LumA and Basal).</div><div class="section-paragraph">B) Boxplot of PD-L1 mRNA expression in each PAM50 subtype. Boxplots show 1.5x the interquartile range for each group, centered on the median.</div><div class="section-paragraph">C) Scatterplot showing that PD-L1 mRNA was well correlated (Spearman) with PD-L1 protein in the DP1 study (<div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib113" id="back-bib113" data-db-target-for="bib113-d13420944e678" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib113">Satpathy et al., 2020</a><div data-db-target-of="bib113-d13420944e678" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Satpathy S. </li><li class="loa__item">Jaehnig E.J. </li><li class="loa__item">Krug K. </li><li class="loa__item">Kim B.-J. </li><li class="loa__item">Saltzman A.B. </li><li class="loa__item">Chan D.W. </li><li class="loa__item">Holloway K.R. </li><li class="loa__item">Anurag M. </li><li class="loa__item">Huang C. </li><li class="loa__item">Singh P. </li><li>et al.</li></ul><div class="reference-citations__detials">Microscaled proteogenomic methods for precision oncology.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">532</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41467-020-14381-2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31988290&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85078365239&amp;cf=" onclick="newWindow(this.href);return false">Scopus (25)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=532&journal=Nat.+Commun.&author=Satpathy+S.&author=Jaehnig+E.J.&author=Krug+K.&author=Kim+B.-J.&author=Saltzman+A.B.&author=Chan+D.W.&author=Holloway+K.R.&author=Anurag+M.&author=Huang+C.&author=Singh+P.&title=Microscaled+proteogenomic+methods+for+precision+oncology">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div>).</div><div class="section-paragraph">D) Boxplot comparing mutational load between APOBEC-enriched and -unenriched samples. The boxplot shows 1.5x the interquartile range for each group, centered on the median. P value is from the Wilcoxon rank-sum test.</div><div class="section-paragraph">E) Scatterplots showing that the GO BP nucleotide excision repair pathway (GO:0006289) is negatively correlated with number of non-synonymous mutations (top), which in turn is positively correlated with PD-L1 mRNA (bottom), in luminal but not basal samples. For clarity, one hyper-mutated basal sample was excluded from these plots. Spearman’s rank correlation coefficients (rho) and p values are shown.</div><div class="section-paragraph">F) Heatmaps for pairwise Spearman rank correlations between the features shown in <a data-locator="fig5" aria-expanded="false" aria-controls="crosref2835" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig5">Figure 5</a>E within samples from the PAM50 basal subtype in the prospective (left heatmap) and retrospective (right heatmap) datasets. NER: nucleotide excision repair, BER: base excision repair, MMR: mismatch repair, DR: direct repair, TLS: translesion synthesis, NHEJ: non-homologous end joining, FA: Fanconi anemia, HR: homologous recombination, DDR: dna damage response (primarily checkpoint proteins), SSBR: single strand break repair, DSBR: double strand break repair.</div><div class="section-paragraph">G) Multi-omics comparisons of the ATM DNA damage checkpoint kinase in HR+/ERBB2- samples and TNBC samples. ATM mRNA (1st panel) and protein (2nd panel) levels were unchanged (Wilcoxon rank sum tests: p = 0.052 and p = 0.354, respectively) while the levels of a known auto-phosphorylation site (3rd panel) and the ATM/ATR target site outlier scores (4th panel) were significantly lower in HR+/ERBB2- than in TNBC tumors (Wilcoxon rank sum tests: p = 0.010 and p = 6.5x10<sup>−5</sup>, respectively). Boxplots show 1.5x the interquartile range for each group, centered on the median.</div><div class="section-paragraph">H) Chk2 (CHEK2) protein levels were significantly lower in HR+/ERBB2-luminal than in TNBC-basal tumors (Wilcoxon rank sum tests: p = 8.2x10<sup>−7</sup>). The boxplot shows 1.5x the interquartile range for each group, centered on the median.</div><div class="section-paragraph">I) Histogram showing distribution of Rb phosphoprotein (mean of add Rb sites) levels in TNBC samples. Samples were classified as pRb low (yellow) and pRb (purple) high for the analysis in J).</div><div class="section-paragraph">J) CDK4 and CDK6 kinase activity scores inferred from phosphosite data by PTM-SEA were significantly lower in TNBC samples with low Rb phosphorylation than in TNBC samples with high Rb phosphorylation (p = 3.5x10<sup>−5</sup> for CDK4 activity; p = 7.3x10<sup>−7</sup> for CDK6 activity). p values are from Wilcoxon rank sum tests comparing the two groups shown in I). Volcano plot shows the -log 10 p values (y axis) versus differences for kinase activity scores inferred by PTM-SEA (kinase target phosphosite sets; x axis). Kinases with p &lt; 0.05 are labeled and colored by higher activity in Rb high (purple) or in Rb low (yellow) samples. delta NES: difference in median activity score for pRb low and pRb high samples.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/4ff1008b-c691-4e9e-ae2a-6c118dd7d5b4/figs7.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/2af00499-a5e7-4f0b-af1a-41d44e03688a/figs7_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(20)31400-8&amp;id=figs7.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div></section><section id="sec2.7" data-locator="sec2.7" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">APOBEC-Mediated Mutagenesis Correlates with an Active I-TME in Luminal BRCA</span></h3><div class="section-paragraph">To identify potential drivers of immunogenicity across common BRCA subtypes, PD-L1 mRNA levels were correlated separately with proteomics data from PAM50 luminal and basal cases (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>A; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">Figure S7</a>C). Several mostly immune-related proteins, including APOBEC3G, showed strong positive correlation with PD-L1 in both subtypes. APOBEC3G protein levels were also correlated with the CIBERSORT immune score and were associated with APOBEC mutagenesis signatures (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>B). Applying a stringent filter for APOBEC enrichment to whole-exome data (<a href="#sec4" class="scroll-into-link">STAR Methods</a>), six cases were diagnosed as APOBEC-enriched (one PAM50 HER2E, three PAM50 LumA, and two PAM50 LumB), and two of the luminal samples were also positive for the SBS13 APOBEC Catalogue Of Somatic Mutations In Cancer (COSMIC) signature, which has been associated previously with lymphocyte infiltration in ER+ BRCA (<div class="dropBlock reference-citations"><a href="#bib118" id="back-bib118" data-db-target-for="bib118-d13420944e2268" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib118">Smid et al., 2016</a><div data-db-target-of="bib118-d13420944e2268" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Smid M. </li><li class="loa__item">Rodríguez-González F.G. </li><li class="loa__item">Sieuwerts A.M. </li><li class="loa__item">Salgado R. </li><li class="loa__item">Prager-Van der Smissen W.J.C. </li><li class="loa__item">Vlugt-Daane M.V. </li><li class="loa__item">van Galen A. </li><li class="loa__item">Nik-Zainal S. </li><li class="loa__item">Staaf J. </li><li class="loa__item">Brinkman A.B. </li><li>et al.</li></ul><div class="reference-citations__detials">Breast cancer genome and transcriptome integration implicates specific mutational signatures with immune cell infiltration.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">12910</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fncomms12910&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27666519&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=12910&journal=Nat.+Commun.&author=Smid+M.&author=Rodr%C3%ADguez-Gonz%C3%A1lez+F.G.&author=Sieuwerts+A.M.&author=Salgado+R.&author=Prager-Van+der+Smissen+W.J.C.&author=Vlugt-Daane+M.V.&author=van+Galen+A.&author=Nik-Zainal+S.&author=Staaf+J.&author=Brinkman+A.B.&title=Breast+cancer+genome+and+transcriptome+integration+implicates+specific+mutational+signatures+with+immune+cell+infiltration">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). High activity of APOBEC enzymes, particularly APOBEC3B, has been associated with a hypermutation phenotype (<div class="dropBlock reference-citations"><a href="#bib107" id="back-bib107" data-db-target-for="bib107-d13420944e2272" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib107">Roberts and Gordenin, 2014</a><div data-db-target-of="bib107-d13420944e2272" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Roberts S.A. </li><li class="loa__item">Gordenin D.A. </li></ul><div class="reference-citations__detials">Hypermutation in human cancer genomes: footprints and mechanisms.<div class="popup-ref"><span class="ref__series"><i>Nat. Rev. Cancer.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">786-800</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnrc3816&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=25568919&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84925497848&amp;cf=" onclick="newWindow(this.href);return false">Scopus (243)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=786-800&journal=Nat.+Rev.+Cancer&author=Roberts+S.A.&author=Gordenin+D.A.&title=Hypermutation+in+human+cancer+genomes%3A+footprints+and+mechanisms">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Consistent with this association, APOBEC-enriched cases had higher mutation loads and higher APOBEC3B protein levels compared with the rest of the tumors (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>B; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">Figure S7</a>D; p = 0.008 and p = 0.1 respectively, Wilcoxon test). Most of the samples with high APOBEC-associated SBS2 and SBS13 signature scores (from COSMIC v.3; <div class="dropBlock reference-citations"><a href="#bib125" id="back-bib125" data-db-target-for="bib125-d13420944e2281" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib125">Tate et al., 2019</a><div data-db-target-of="bib125-d13420944e2281" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Tate J.G. </li><li class="loa__item">Bamford S. </li><li class="loa__item">Jubb H.C. </li><li class="loa__item">Sondka Z. </li><li class="loa__item">Beare D.M. </li><li class="loa__item">Bindal N. </li><li class="loa__item">Boutselakis H. </li><li class="loa__item">Cole C.G. </li><li class="loa__item">Creatore C. </li><li class="loa__item">Dawson E. </li><li>et al.</li></ul><div class="reference-citations__detials">COSMIC: the Catalogue Of Somatic Mutations In Cancer.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">47</span></span>: <span class="ref__seriesPages">D941-D947</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgky1015&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30371878&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85057093998&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1040)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D941-D947&journal=Nucleic+Acids+Res.&author=Tate+J.G.&author=Bamford+S.&author=Jubb+H.C.&author=Sondka+Z.&author=Beare+D.M.&author=Bindal+N.&author=Boutselakis+H.&author=Cole+C.G.&author=Creatore+C.&author=Dawson+E.&title=COSMIC%3A+the+Catalogue+Of+Somatic+Mutations+In+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) also had high levels of APOBEC3G. These observations suggest that APOBEC-driven mutagenesis can contribute to an active immune microenvironment in ER+ BRCA with links to PD-L1 mRNA expression.<div class="floatDisplay"><figure id="gr5" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/26c73d76-b2f3-4909-8f32-3cfc55a981fd/gr5.jpg" alt="Figure thumbnail gr5" data-locator="fig5" data-high-res="/cms/attachment/3282019b-5417-467f-9f1d-abefc92a1d10/gr5_lrg.jpg" data-large-image="/cms/attachment/26c73d76-b2f3-4909-8f32-3cfc55a981fd/gr5.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 5" class="label figure__label">Figure 5</span><span class="figure__title__text">Association of APOBEC Mutations and DNA Damage Repair Pathway Levels with the Immune Microenvironment in Luminal Tumors</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Correlation of protein levels with PD-L1 mRNA in PAM50 basal (x axis) and luminal (LumA and LumB, y axis) samples. Signed log10 FDR-corrected p values of Spearman-rank correlations are plotted. Protein data for PD-L1 was sparse in this study, but we observed high correlation between PD-L1 RNA and protein in the DP1 study, indicating that the RNA is a suitable surrogate for protein (<a data-locator="figs7" aria-expanded="false" aria-controls="crosref2510" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs7">Figure S7</a>C).</div><div class="section-paragraph">(B) Although mutation load is correlated with the immune microenvironment in PAM50 luminal and basal BRCA, luminal samples with a high mutation load specifically show enrichment for APOBEC mutations. Luminal samples without APOBEC enrichment, luminal samples with APOBEC enrichment, basal samples (no APOBEC enrichment), PAM50 HER2E samples without APOBEC enrichment, and HER2E samples with APOBEC enrichment are ordered by increasing CIBERSORT scores. SBS13 and SBS2 are similarity scores for the whole-exome sequencing (WES)-derived mutation profile of a given sample with the corresponding COSMIC signature. APOBEC mutation fraction indicates the fraction of mutations that are APOBEC-associated mutations. APOBEC3G and APOBEC3B protein levels are also shown.</div><div class="section-paragraph">(C) Nucleotide excision repair (NER), mRNA processing, and RNA splicing are negatively correlated with PD-L1 in PAM50 luminal but not basal BRCA. The bar graph shows normalized enrichment scores (NESs) for the top GO biological process gene sets correlated with PD-L1 mRNA in luminal samples (blue bars) together with the corresponding NES for basal samples (red bars) from the gene set enrichment analysis (GSEA) of signed log10 p values from (A).</div><div class="section-paragraph">(D) The mean log2 TMT ratio for proteins from the GO BP NER pathway is negatively correlated (Spearman) with PD-L1 RNA expression in PAM50 luminal but not basal samples in the prospective (top) and retrospective (bottom) datasets. Scatterplots show the mean log2 TMT ratios on the y axis and log2 mRNA ratios (median-MAD-normalized data) on the x axis. Blue points show PAM50 luminal (LumA and LumB) samples, red points show PAM50 basal samples, and lines show the linear fit for each group.</div><div class="section-paragraph">(E) Heatmaps showing pairwise Spearman-rank correlations within the PAM50 luminal (combined A and B) samples from the prospective (left) and retrospective (right) datasets for immune microenvironment features (CTLA4, PD1, and PD-L1 RNA and CIBERSORT and protein-based signatures from A), GO BP scores anti-correlated with PD-L1 in luminal tumors (C), specific DNA repair pathway scores, single- and double-strand break repair (SSBR and DSBR) scores, mutation load (not included for retrospective), APOBEC mutation signatures (SBS2 and SBS13), chromosomal instability (CIN, also not included for retrospective), and RNA processing/splicing. MMR, mismatch repair; BER, base excision repair; NER, nucleotide excison repair; TLS, translesion synthesis; HR, homologous recombination; FA, Fanconi anemia; DR, direct repair; NHEJ, non-homologous end joining; DDR, DNA damage response (primarily checkpoint proteins). Gene set-based scores are the mean protein levels of all genes in the set.</div><div class="section-paragraph">See also <a data-locator="figs7" aria-expanded="false" aria-controls="crosref2515" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs7">Figure S7</a> and <a data-locator="mmc2" aria-expanded="false" aria-controls="crosref2520" href="/cms/10.1016/j.cell.2020.10.036/attachment/a1bc3f3e-eeac-4605-a9ef-839abd334dcd/mmc2" target="_blank">Tables S2</a>, <a data-locator="mmc6" aria-expanded="false" aria-controls="crosref2525" href="/cms/10.1016/j.cell.2020.10.036/attachment/eb237fc7-15fb-4b8d-bf02-cae5f285b239/mmc6" target="_blank">S6</a>, and <a data-locator="mmc7" aria-expanded="false" aria-controls="crosref2530" href="/cms/10.1016/j.cell.2020.10.036/attachment/ce88682c-a2ae-458a-a550-73ad6faa7636/mmc7" target="_blank">S7</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/26c73d76-b2f3-4909-8f32-3cfc55a981fd/gr5.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/3282019b-5417-467f-9f1d-abefc92a1d10/gr5_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(20)31400-8&amp;id=gr5.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div></section><section id="sec2.8" data-locator="sec2.8" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Loss of SSBR Proteins Promotes Immunity in Luminal BRCA</span></h3><div class="section-paragraph">Gene set enrichment analysis (GSEA) (<div class="dropBlock reference-citations"><a href="#bib121" id="back-bib121" data-db-target-for="bib121-d13420944e2289" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib121">Subramanian et al., 2005</a><div data-db-target-of="bib121-d13420944e2289" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Subramanian A. </li><li class="loa__item">Tamayo P. </li><li class="loa__item">Mootha V.K. </li><li class="loa__item">Mukherjee S. </li><li class="loa__item">Ebert B.L. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Paulovich A. </li><li class="loa__item">Pomeroy S.L. </li><li class="loa__item">Golub T.R. </li><li class="loa__item">Lander E.S. </li><li class="loa__item">Mesirov J.P. </li></ul><div class="reference-citations__detials">Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles.<div class="popup-ref"><span class="ref__series"><i>Proc. Natl. Acad. Sci. USA.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">102</span></span>: <span class="ref__seriesPages">15545-15550</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1073%2Fpnas.0506580102&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=16199517&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-27344435774&amp;cf=" onclick="newWindow(this.href);return false">Scopus (20710)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=15545-15550&journal=Proc.+Natl.+Acad.+Sci.+USA&author=Subramanian+A.&author=Tamayo+P.&author=Mootha+V.K.&author=Mukherjee+S.&author=Ebert+B.L.&author=Gillette+M.A.&author=Paulovich+A.&author=Pomeroy+S.L.&author=Golub+T.R.&author=Lander+E.S.&author=Mesirov+J.P.&title=Gene+set+enrichment+analysis%3A+a+knowledge-based+approach+for+interpreting+genome-wide+expression+profiles">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) for GO biological processes (GO BP) using the correlation analysis results from <a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>A confirmed that multiple immune-related processes were positively correlated with PD-L1 expression in PAM50 basal and luminal samples (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>C). In contrast, NER, RNA splicing, and mRNA processing were negatively correlated with PD-L1 in PAM50 luminal samples only, suggesting loss of these processes in luminal tumors with active immunity. The mean expression level of GO NER proteins was also negatively correlated with PD-L1 RNA specifically in the luminal but not the basal PAM50 subtypes in the TCGA retrospective study, providing supportive evidence of these associations (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>D). Because increased PD-L1 expression has been associated previously with DNA repair deficiency in breast tumors (<div class="dropBlock reference-citations"><a href="#bib97" id="back-bib97" data-db-target-for="bib97-d13420944e2301" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib97">Parkes et al., 2016</a><div data-db-target-of="bib97-d13420944e2301" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Parkes E.E. </li><li class="loa__item">Walker S.M. </li><li class="loa__item">Taggart L.E. </li><li class="loa__item">McCabe N. </li><li class="loa__item">Knight L.A. </li><li class="loa__item">Wilkinson R. </li><li class="loa__item">McCloskey K.D. </li><li class="loa__item">Buckley N.E. </li><li class="loa__item">Savage K.I. </li><li class="loa__item">Salto-Tellez M. </li><li>et al.</li></ul><div class="reference-citations__detials">Activation of STING-Dependent Innate Immune Signaling By S-Phase-Specific DNA Damage in Breast Cancer.<div class="popup-ref"><span class="ref__series"><i>J. Natl. Cancer Inst.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">109</span></span>: <span class="ref__seriesPages">109</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2016&pages=109&journal=J.%C2%A0Natl.+Cancer+Inst.&author=Parkes+E.E.&author=Walker+S.M.&author=Taggart+L.E.&author=McCabe+N.&author=Knight+L.A.&author=Wilkinson+R.&author=McCloskey+K.D.&author=Buckley+N.E.&author=Savage+K.I.&author=Salto-Tellez+M.&title=Activation+of+STING-Dependent+Innate+Immune+Signaling+By+S-Phase-Specific+DNA+Damage+in+Breast+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), and loss of expression of NER and BER genes was associated with resistance to endocrine therapy in ER+ BRCA (<div class="dropBlock reference-citations"><a href="#bib4" id="back-bib4" data-db-target-for="bib4-d13420944e2305" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib4">Anurag et al., 2018a</a><div data-db-target-of="bib4-d13420944e2305" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Anurag M. </li><li class="loa__item">Punturi N. </li><li class="loa__item">Hoog J. </li><li class="loa__item">Bainbridge M.N. </li><li class="loa__item">Ellis M.J. </li><li class="loa__item">Haricharan S. </li></ul><div class="reference-citations__detials">Comprehensive Profiling of DNA Repair Defects in Breast Cancer Identifies a Novel Class of Endocrine Therapy Resistance Drivers.<div class="popup-ref"><span class="ref__series"><i>Clin. Cancer Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">4887-4899</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1158%2F1078-0432.CCR-17-3702&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29793947&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85054053323&amp;cf=" onclick="newWindow(this.href);return false">Scopus (33)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=4887-4899&journal=Clin.+Cancer+Res.&author=Anurag+M.&author=Punturi+N.&author=Hoog+J.&author=Bainbridge+M.N.&author=Ellis+M.J.&author=Haricharan+S.&title=Comprehensive+Profiling+of+DNA+Repair+Defects+in+Breast+Cancer+Identifies+a+Novel+Class+of+Endocrine+Therapy+Resistance+Drivers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib48" id="back-bib48" data-db-target-for="bib48-d13420944e2308" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib48">Haricharan et al., 2017</a><div data-db-target-of="bib48-d13420944e2308" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Haricharan S. </li><li class="loa__item">Punturi N. </li><li class="loa__item">Singh P. </li><li class="loa__item">Holloway K.R. </li><li class="loa__item">Anurag M. </li><li class="loa__item">Schmelz J. </li><li class="loa__item">Schmidt C. </li><li class="loa__item">Lei J.T. </li><li class="loa__item">Suman V. </li><li class="loa__item">Hunt K. </li><li>et al.</li></ul><div class="reference-citations__detials">Loss of MutL Disrupts CHK2-Dependent Cell-Cycle Control through CDK4/6 to Promote Intrinsic Endocrine Therapy Resistance in Primary Breast Cancer.<div class="popup-ref"><span class="ref__series"><i>Cancer Discov.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">1168-1183</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1158%2F2159-8290.CD-16-1179&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=28801307&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85030681935&amp;cf=" onclick="newWindow(this.href);return false">Scopus (12)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=1168-1183&journal=Cancer+Discov.&author=Haricharan+S.&author=Punturi+N.&author=Singh+P.&author=Holloway+K.R.&author=Anurag+M.&author=Schmelz+J.&author=Schmidt+C.&author=Lei+J.T.&author=Suman+V.&author=Hunt+K.&title=Loss+of+MutL+Disrupts+CHK2-Dependent+Cell-Cycle+Control+through+CDK4%2F6+to+Promote+Intrinsic+Endocrine+Therapy+Resistance+in+Primary+Breast+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), the consistent anti-correlation of NER proteins with PD-L1 expression suggests therapeutic potential for immunotherapy in endocrine-resistant ER+ BRCA. Importantly, low protein levels for the GO NER pathway were associated with high mutation load in luminal but not basal samples (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">Figure S7</a>E). Upregulated immune checkpoint components in endocrine therapy-resistant LumB BRCA have been noted previously (<div class="dropBlock reference-citations"><a href="#bib6" id="back-bib6" data-db-target-for="bib6-d13420944e2314" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib6">Anurag et al., 2020</a><div data-db-target-of="bib6-d13420944e2314" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Anurag M. </li><li class="loa__item">Zhu M. </li><li class="loa__item">Huang C. </li><li class="loa__item">Vasaikar S. </li><li class="loa__item">Wang J. </li><li class="loa__item">Hoog J. </li><li class="loa__item">Burugu S. </li><li class="loa__item">Gao D. </li><li class="loa__item">Suman V. </li><li class="loa__item">Zhang X.H. </li><li>et al.</li></ul><div class="reference-citations__detials">Immune Checkpoint Profiles in Luminal B Breast Cancer (Alliance).<div class="popup-ref"><span class="ref__series"><i>J. Natl. Cancer Inst.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">112</span></span>: <span class="ref__seriesPages">737-746</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fjnci%2Fdjz213&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31665365&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85079588411&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2020&pages=737-746&journal=J.%C2%A0Natl.+Cancer+Inst&author=Anurag+M.&author=Zhu+M.&author=Huang+C.&author=Vasaikar+S.&author=Wang+J.&author=Hoog+J.&author=Burugu+S.&author=Gao+D.&author=Suman+V.&author=Zhang+X.H.&title=Immune+Checkpoint+Profiles+in+Luminal+B+Breast+Cancer+%28Alliance%29">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), suggesting that NER deficiency is a potential link between upregulated immune checkpoints and endocrine therapy resistance. Because the GO BP NER pathway contains genes that overlap with other DNA repair pathways, pairwise Spearman correlations of immune features with scores from <a href="/cms/10.1016/j.cell.2020.10.036/attachment/eb237fc7-15fb-4b8d-bf02-cae5f285b239/mmc6" class="scroll-into-link">Table S6</a> for specific DNA repair pathways were examined together with the negatively correlated GO BP pathways. Although the unique NER protein score was not significantly anti-correlated with PD-L1, it was negatively correlated with the CIBERSORT immune score (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>E; <a href="/cms/10.1016/j.cell.2020.10.036/attachment/ce88682c-a2ae-458a-a550-73ad6faa7636/mmc7" class="scroll-into-link">Table S7</a>). Indeed, nearly all SSBR pathways, which include BER, mismatch repair (MMR), and NER, were anti-correlated with the CIBERSORT as well as the protein-derived stimulatory and inhibitory immune modulator scores, associations that were confirmed in the retrospective dataset. As with NER, the associations between low levels of other SSBR protein levels and an active tumor microenvironment appeared to be specific for PAM50 luminal tumors and were not observed in basal-like tumors (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>E; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">Figure S7</a>F; <a href="/cms/10.1016/j.cell.2020.10.036/attachment/ce88682c-a2ae-458a-a550-73ad6faa7636/mmc7" class="scroll-into-link">Table S7</a>).</div><div class="section-paragraph">ER signaling modulates the DNA damage response (DDR) (<div class="dropBlock reference-citations"><a href="#bib19" id="back-bib19" data-db-target-for="bib19-d13420944e2338" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib19">Caldon, 2014</a><div data-db-target-of="bib19-d13420944e2338" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Caldon C.E. </li></ul><div class="reference-citations__detials">Estrogen signaling and the DNA damage response in hormone dependent breast cancers.<div class="popup-ref"><span class="ref__series"><i>Front. Oncol.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">106</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.3389%2Ffonc.2014.00106&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=24860786&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84904599047&amp;cf=" onclick="newWindow(this.href);return false">Scopus (91)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2014&pages=106&journal=Front.+Oncol.&author=Caldon+C.E.&title=Estrogen+signaling+and+the+DNA+damage+response+in+hormone+dependent+breast+cancers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Therefore, outlier phosphopeptide abundance DDR scores for a set of ataxia telangiectasia mutated (ATM)/ATR/DNAPK target SQ/TQ peptides, shown previously to be induced in response to DNA damage, were examined (<div class="dropBlock reference-citations"><a href="#bib79" id="back-bib79" data-db-target-for="bib79-d13420944e2341" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib79">Matsuoka et al., 2007</a><div data-db-target-of="bib79-d13420944e2341" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Matsuoka S. </li><li class="loa__item">Ballif B.A. </li><li class="loa__item">Smogorzewska A. </li><li class="loa__item">McDonald 3rd, E.R. </li><li class="loa__item">Hurov K.E. </li><li class="loa__item">Luo J. </li><li class="loa__item">Bakalarski C.E. </li><li class="loa__item">Zhao Z. </li><li class="loa__item">Solimini N. </li><li class="loa__item">Lerenthal Y. </li><li>et al.</li></ul><div class="reference-citations__detials">ATM and ATR substrate analysis reveals extensive protein networks responsive to DNA damage.<div class="popup-ref"><span class="ref__series"><i>Science.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">316</span></span>: <span class="ref__seriesPages">1160-1166</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1126%2Fscience.1140321&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=17525332&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-34249947699&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2181)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=316&publication_year=2007&pages=1160-1166&journal=Science&author=Matsuoka+S.&author=Ballif+B.A.&author=Smogorzewska+A.&author=McDonald+E.R.&author=Hurov+K.E.&author=Luo+J.&author=Bakalarski+C.E.&author=Zhao+Z.&author=Solimini+N.&author=Lerenthal+Y.&title=ATM+and+ATR+substrate+analysis+reveals+extensive+protein+networks+responsive+to+DNA+damage">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). DDR scores, an ATM auto-phosphorylation site, and the downstream Chk2 protein were lower in PAM50 luminal samples than in basal samples (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">Figures S7</a>G and S7H). Notably, differences in ATM activity were present only in phosphoproteomic data, with the RNA and protein levels showing no significant differences between PAM50 luminal and basal subtypes (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">Figure S7</a>G). This suggests that luminal samples may have relatively suppressed DNA damage checkpoint activity, possibly because of inhibition of ATM by ESR1 (<div class="dropBlock reference-citations"><a href="#bib4" id="back-bib4" data-db-target-for="bib4-d13420944e2350" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib4">Anurag et al., 2018a</a><div data-db-target-of="bib4-d13420944e2350" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Anurag M. </li><li class="loa__item">Punturi N. </li><li class="loa__item">Hoog J. </li><li class="loa__item">Bainbridge M.N. </li><li class="loa__item">Ellis M.J. </li><li class="loa__item">Haricharan S. </li></ul><div class="reference-citations__detials">Comprehensive Profiling of DNA Repair Defects in Breast Cancer Identifies a Novel Class of Endocrine Therapy Resistance Drivers.<div class="popup-ref"><span class="ref__series"><i>Clin. Cancer Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">4887-4899</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1158%2F1078-0432.CCR-17-3702&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29793947&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85054053323&amp;cf=" onclick="newWindow(this.href);return false">Scopus (33)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=4887-4899&journal=Clin.+Cancer+Res.&author=Anurag+M.&author=Punturi+N.&author=Hoog+J.&author=Bainbridge+M.N.&author=Ellis+M.J.&author=Haricharan+S.&title=Comprehensive+Profiling+of+DNA+Repair+Defects+in+Breast+Cancer+Identifies+a+Novel+Class+of+Endocrine+Therapy+Resistance+Drivers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib48" id="back-bib48" data-db-target-for="bib48-d13420944e2354" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib48">Haricharan et al., 2017</a><div data-db-target-of="bib48-d13420944e2354" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Haricharan S. </li><li class="loa__item">Punturi N. </li><li class="loa__item">Singh P. </li><li class="loa__item">Holloway K.R. </li><li class="loa__item">Anurag M. </li><li class="loa__item">Schmelz J. </li><li class="loa__item">Schmidt C. </li><li class="loa__item">Lei J.T. </li><li class="loa__item">Suman V. </li><li class="loa__item">Hunt K. </li><li>et al.</li></ul><div class="reference-citations__detials">Loss of MutL Disrupts CHK2-Dependent Cell-Cycle Control through CDK4/6 to Promote Intrinsic Endocrine Therapy Resistance in Primary Breast Cancer.<div class="popup-ref"><span class="ref__series"><i>Cancer Discov.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">1168-1183</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1158%2F2159-8290.CD-16-1179&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=28801307&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85030681935&amp;cf=" onclick="newWindow(this.href);return false">Scopus (12)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=1168-1183&journal=Cancer+Discov.&author=Haricharan+S.&author=Punturi+N.&author=Singh+P.&author=Holloway+K.R.&author=Anurag+M.&author=Schmelz+J.&author=Schmidt+C.&author=Lei+J.T.&author=Suman+V.&author=Hunt+K.&title=Loss+of+MutL+Disrupts+CHK2-Dependent+Cell-Cycle+Control+through+CDK4%2F6+to+Promote+Intrinsic+Endocrine+Therapy+Resistance+in+Primary+Breast+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). This could facilitate tolerance for single-strand break repair defects (NER, BER, and MMR) in luminal BRCA and also de-repress CDK4/6, consistent with recent postulates on the efficacy of CDK4/6i in ER+ BRCA (<div class="dropBlock reference-citations"><a href="#bib48" id="back-bib48" data-db-target-for="bib48-d13420944e2357" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib48">Haricharan et al., 2017</a><div data-db-target-of="bib48-d13420944e2357" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Haricharan S. </li><li class="loa__item">Punturi N. </li><li class="loa__item">Singh P. </li><li class="loa__item">Holloway K.R. </li><li class="loa__item">Anurag M. </li><li class="loa__item">Schmelz J. </li><li class="loa__item">Schmidt C. </li><li class="loa__item">Lei J.T. </li><li class="loa__item">Suman V. </li><li class="loa__item">Hunt K. </li><li>et al.</li></ul><div class="reference-citations__detials">Loss of MutL Disrupts CHK2-Dependent Cell-Cycle Control through CDK4/6 to Promote Intrinsic Endocrine Therapy Resistance in Primary Breast Cancer.<div class="popup-ref"><span class="ref__series"><i>Cancer Discov.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">1168-1183</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1158%2F2159-8290.CD-16-1179&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=28801307&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85030681935&amp;cf=" onclick="newWindow(this.href);return false">Scopus (12)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=1168-1183&journal=Cancer+Discov.&author=Haricharan+S.&author=Punturi+N.&author=Singh+P.&author=Holloway+K.R.&author=Anurag+M.&author=Schmelz+J.&author=Schmidt+C.&author=Lei+J.T.&author=Suman+V.&author=Hunt+K.&title=Loss+of+MutL+Disrupts+CHK2-Dependent+Cell-Cycle+Control+through+CDK4%2F6+to+Promote+Intrinsic+Endocrine+Therapy+Resistance+in+Primary+Breast+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib100" id="back-bib100" data-db-target-for="bib100-d13420944e2360" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib100">Pernas et al., 2018</a><div data-db-target-of="bib100-d13420944e2360" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Pernas S. </li><li class="loa__item">Tolaney S.M. </li><li class="loa__item">Winer E.P. </li><li class="loa__item">Goel S. </li></ul><div class="reference-citations__detials">CDK4/6 inhibition in breast cancer: current practice and future directions.<div class="popup-ref"><span class="ref__series"><i>Ther. Adv. Med. Oncol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">10</span> (<span class="refComment">1758835918786451</span>)</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1177%2F1758835918786451&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30038670&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85054681352&amp;cf=" onclick="newWindow(this.href);return false">Scopus (129)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&journal=Ther.+Adv.+Med.+Oncol.&author=Pernas+S.&author=Tolaney+S.M.&author=Winer+E.P.&author=Goel+S.&title=CDK4%2F6+inhibition+in+breast+cancer%3A+current+practice+and+future+directions">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div></section><section id="sec2.9" data-locator="sec2.9" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">PG Analysis of Rb Status May Inform the Response to CDK4/6 Inhibitor Therapy</span></h3><div class="section-paragraph">Proliferation rate is a critical prognostic feature in BRCA, and the cell cycle is a target for endocrine therapy (<div class="dropBlock reference-citations"><a href="#bib39" id="back-bib39" data-db-target-for="bib39-d13420944e2368" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib39">Ellis et al., 2017</a><div data-db-target-of="bib39-d13420944e2368" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ellis M.J. </li><li class="loa__item">Suman V.J. </li><li class="loa__item">Hoog J. </li><li class="loa__item">Goncalves R. </li><li class="loa__item">Sanati S. </li><li class="loa__item">Creighton C.J. </li><li class="loa__item">DeSchryver K. </li><li class="loa__item">Crouch E. </li><li class="loa__item">Brink A. </li><li class="loa__item">Watson M. </li><li>et al.</li></ul><div class="reference-citations__detials">Ki67 Proliferation Index as a Tool for Chemotherapy Decisions During and After Neoadjuvant Aromatase Inhibitor Treatment of Breast Cancer: Results From the American College of Surgeons Oncology Group Z1031 Trial (Alliance).<div class="popup-ref"><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">1061-1069</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1200%2FJCO.2016.69.4406&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=28045625&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85016622416&amp;cf=" onclick="newWindow(this.href);return false">Scopus (127)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2017&pages=1061-1069&journal=J.%C2%A0Clin.+Oncol.&author=Ellis+M.J.&author=Suman+V.J.&author=Hoog+J.&author=Goncalves+R.&author=Sanati+S.&author=Creighton+C.J.&author=DeSchryver+K.&author=Crouch+E.&author=Brink+A.&author=Watson+M.&title=Ki67+Proliferation+Index+as+a+Tool+for+Chemotherapy+Decisions+During+and+After+Neoadjuvant+Aromatase+Inhibitor+Treatment+of+Breast+Cancer%3A+Results+From+the+American+College+of+Surgeons+Oncology+Group+Z1031+Trial+%28Alliance%29">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and CDK4/6 inhibition in ER+, ERBB2− advanced BRCA (<div class="dropBlock reference-citations"><a href="#bib100" id="back-bib100" data-db-target-for="bib100-d13420944e2371" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib100">Pernas et al., 2018</a><div data-db-target-of="bib100-d13420944e2371" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Pernas S. </li><li class="loa__item">Tolaney S.M. </li><li class="loa__item">Winer E.P. </li><li class="loa__item">Goel S. </li></ul><div class="reference-citations__detials">CDK4/6 inhibition in breast cancer: current practice and future directions.<div class="popup-ref"><span class="ref__series"><i>Ther. Adv. Med. Oncol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">10</span> (<span class="refComment">1758835918786451</span>)</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1177%2F1758835918786451&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30038670&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85054681352&amp;cf=" onclick="newWindow(this.href);return false">Scopus (129)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&journal=Ther.+Adv.+Med.+Oncol.&author=Pernas+S.&author=Tolaney+S.M.&author=Winer+E.P.&author=Goel+S.&title=CDK4%2F6+inhibition+in+breast+cancer%3A+current+practice+and+future+directions">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). CDK4 and CDK6, in complex with Cyclin D, promote cell cycle progression by phosphorylating and inactivating the Retinoblastoma transcriptional inhibitor (Rb) during G1, whereas the Cyclin E/CDK2 complex further contributes to inhibition of Rb during G1-S transition (<div class="dropBlock reference-citations"><a href="#bib46" id="back-bib46" data-db-target-for="bib46-d13420944e2374" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib46">Goel et al., 2018</a><div data-db-target-of="bib46-d13420944e2374" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Goel S. </li><li class="loa__item">DeCristo M.J. </li><li class="loa__item">McAllister S.S. </li><li class="loa__item">Zhao J.J. </li></ul><div class="reference-citations__detials">CDK4/6 Inhibition in Cancer: Beyond Cell Cycle Arrest.<div class="popup-ref"><span class="ref__series"><i>Trends Cell Biol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">911-925</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.tcb.2018.07.002&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.tcb.2018.07.002&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.tcb.2018.07.002&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30061045&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85050508022&amp;cf=" onclick="newWindow(this.href);return false">Scopus (100)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=911-925&journal=Trends+Cell+Biol.&author=Goel+S.&author=DeCristo+M.J.&author=McAllister+S.S.&author=Zhao+J.J.&title=CDK4%2F6+Inhibition+in+Cancer%3A+Beyond+Cell+Cycle+Arrest">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). To compare PG features with cell cycle control in hormone receptor (HR)+/ERBB2 PG− and triple-negative BRCA (TNBC) tumors, the multi-gene proliferation score (MGPS; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>A; <a href="/cms/10.1016/j.cell.2020.10.036/attachment/eb237fc7-15fb-4b8d-bf02-cae5f285b239/mmc6" class="scroll-into-link">Table S6</a>) was generated for each sample (<div class="dropBlock reference-citations"><a href="#bib39" id="back-bib39" data-db-target-for="bib39-d13420944e2385" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib39">Ellis et al., 2017</a><div data-db-target-of="bib39-d13420944e2385" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ellis M.J. </li><li class="loa__item">Suman V.J. </li><li class="loa__item">Hoog J. </li><li class="loa__item">Goncalves R. </li><li class="loa__item">Sanati S. </li><li class="loa__item">Creighton C.J. </li><li class="loa__item">DeSchryver K. </li><li class="loa__item">Crouch E. </li><li class="loa__item">Brink A. </li><li class="loa__item">Watson M. </li><li>et al.</li></ul><div class="reference-citations__detials">Ki67 Proliferation Index as a Tool for Chemotherapy Decisions During and After Neoadjuvant Aromatase Inhibitor Treatment of Breast Cancer: Results From the American College of Surgeons Oncology Group Z1031 Trial (Alliance).<div class="popup-ref"><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">1061-1069</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1200%2FJCO.2016.69.4406&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=28045625&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85016622416&amp;cf=" onclick="newWindow(this.href);return false">Scopus (127)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2017&pages=1061-1069&journal=J.%C2%A0Clin.+Oncol.&author=Ellis+M.J.&author=Suman+V.J.&author=Hoog+J.&author=Goncalves+R.&author=Sanati+S.&author=Creighton+C.J.&author=DeSchryver+K.&author=Crouch+E.&author=Brink+A.&author=Watson+M.&title=Ki67+Proliferation+Index+as+a+Tool+for+Chemotherapy+Decisions+During+and+After+Neoadjuvant+Aromatase+Inhibitor+Treatment+of+Breast+Cancer%3A+Results+From+the+American+College+of+Surgeons+Oncology+Group+Z1031+Trial+%28Alliance%29">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib142" id="back-bib142" data-db-target-for="bib142-d13420944e2388" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib142">Whitfield et al., 2002</a><div data-db-target-of="bib142-d13420944e2388" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Whitfield M.L. </li><li class="loa__item">Sherlock G. </li><li class="loa__item">Saldanha A.J. </li><li class="loa__item">Murray J.I. </li><li class="loa__item">Ball C.A. </li><li class="loa__item">Alexander K.E. </li><li class="loa__item">Matese J.C. </li><li class="loa__item">Perou C.M. </li><li class="loa__item">Hurt M.M. </li><li class="loa__item">Brown P.O. </li><li class="loa__item">Botstein D. </li></ul><div class="reference-citations__detials">Identification of genes periodically expressed in the human cell cycle and their expression in tumors.<div class="popup-ref"><span class="ref__series"><i>Mol. Biol. Cell.</i> <span class="ref__seriesDate">2002; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">1977-2000</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1091%2Fmbc.02-02-0030.&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=12058064&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-0035985177&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1127)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2002&pages=1977-2000&journal=Mol.+Biol.+Cell&author=Whitfield+M.L.&author=Sherlock+G.&author=Saldanha+A.J.&author=Murray+J.I.&author=Ball+C.A.&author=Alexander+K.E.&author=Matese+J.C.&author=Perou+C.M.&author=Hurt+M.M.&author=Brown+P.O.&author=Botstein+D.&title=Identification+of+genes+periodically+expressed+in+the+human+cell+cycle+and+their+expression+in+tumors">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Multi-omics analyses of Cyclin E (<em>CCNE1</em>) and CDK2, stemness scores, E2F activity scores (derived from their target genes in the Molecular Signatures Database (MSigDB; <div class="dropBlock reference-citations"><a href="#bib72" id="back-bib72" data-db-target-for="bib72-d13420944e2393" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib72">Liberzon et al., 2015</a><div data-db-target-of="bib72-d13420944e2393" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liberzon A. </li><li class="loa__item">Birger C. </li><li class="loa__item">Thorvaldsdóttir H. </li><li class="loa__item">Ghandi M. </li><li class="loa__item">Mesirov J.P. </li><li class="loa__item">Tamayo P. </li></ul><div class="reference-citations__detials">The Molecular Signatures Database (MSigDB) hallmark gene set collection.<div class="popup-ref"><span class="ref__series"><i>Cell Syst.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">417-425</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26771021&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84955328286&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2256)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=417-425&journal=Cell+Syst.&author=Liberzon+A.&author=Birger+C.&author=Thorvaldsd%C3%B3ttir+H.&author=Ghandi+M.&author=Mesirov+J.P.&author=Tamayo+P.&title=The+Molecular+Signatures+Database+%28MSigDB%29+hallmark+gene+set+collection">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), and CDK2 activity scores (derived from kinase target sites; <div class="dropBlock reference-citations"><a href="#bib50" id="back-bib50" data-db-target-for="bib50-d13420944e2396" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib50">Hornbeck et al., 2015</a><div data-db-target-of="bib50-d13420944e2396" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hornbeck P.V. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Murray B. </li><li class="loa__item">Kornhauser J.M. </li><li class="loa__item">Latham V. </li><li class="loa__item">Skrzypek E. </li></ul><div class="reference-citations__detials">PhosphoSitePlus, 2014: mutations, PTMs and recalibrations.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">43</span></span>: <span class="ref__seriesPages">D512-D520</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgku1267&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=25514926&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84941120469&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1289)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=D512-D520&journal=Nucleic+Acids+Res.&author=Hornbeck+P.V.&author=Zhang+B.&author=Murray+B.&author=Kornhauser+J.M.&author=Latham+V.&author=Skrzypek+E.&title=PhosphoSitePlus%2C+2014%3A+mutations%2C+PTMs+and+recalibrations">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib62" id="back-bib62" data-db-target-for="bib62-d13420944e2399" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib62">Krug et al., 2019</a><div data-db-target-of="bib62-d13420944e2399" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Krug K. </li><li class="loa__item">Mertins P. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Hornbeck P. </li><li class="loa__item">Raju R. </li><li class="loa__item">Ahmad R. </li><li class="loa__item">Szucs M. </li><li class="loa__item">Mundt F. </li><li class="loa__item">Forestier D. </li><li class="loa__item">Jane-Valbuena J. </li><li>et al.</li></ul><div class="reference-citations__detials">A Curated Resource for Phosphosite-specific Signature Analysis.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">576-593</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30563849&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85062992827&amp;cf=" onclick="newWindow(this.href);return false">Scopus (42)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=576-593&journal=Mol.+Cell.+Proteomics&author=Krug+K.&author=Mertins+P.&author=Zhang+B.&author=Hornbeck+P.&author=Raju+R.&author=Ahmad+R.&author=Szucs+M.&author=Mundt+F.&author=Forestier+D.&author=Jane-Valbuena+J.&title=A+Curated+Resource+for+Phosphosite-specific+Signature+Analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) revealed positive correlations with MGPSs in both groups (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figures 6</a>A and 6B; <a href="/cms/10.1016/j.cell.2020.10.036/attachment/ce88682c-a2ae-458a-a550-73ad6faa7636/mmc7" class="scroll-into-link">Table S7</a>), although the MGPS was higher in TNBC than in HR+/ERBB2− samples (p = 3.1 × 10<sup>−5</sup>, Wilcoxon rank-sum test; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>A). Although Cyclin D1 (<em>CCND1</em>) RNA, protein, and phosphorylation levels showed weak or no correlation with MGPS in both groups, these features were significantly higher in HR+/ERBB2− samples than in TNBC samples (p = 1.0 × 10<sup>−7</sup>, 1.7 × 10<sup>−6</sup>, and 0.023, respectively; Wilcoxon rank-sum test). Despite lack of correlation between Cyclin D1 and MGPS, CDK4 and CDK6 activity levels were positively correlated with MGPS in HR+/ERBB2− samples but had weakly negative or no correlation in TNBC samples (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figures 6</a>A and 6B; <a href="/cms/10.1016/j.cell.2020.10.036/attachment/ce88682c-a2ae-458a-a550-73ad6faa7636/mmc7" class="scroll-into-link">Table S7</a>), suggesting that variability in CDK4/6 activity controls the variability in proliferation rates in HR+ BRCA but not in highly proliferative TNBC tumors. More distinctly, although Rb RNA levels did not show significant correlation with MGPS (Spearman rho = −0.069, p = 0.55 for HR+/ERBB2−, rho = −0.36, p = 0.060 for TNBC), Rb protein and phosphoprotein levels were significantly positively correlated in HR+/ERBB2− samples (Spearman rho = 0.24, p = 0.035 for protein and rho = 0.53, p = 1.06 × 10<sup>−6</sup> for median of all Rb phosphosites) but significantly negatively correlated in TNBC samples (Spearman rho = −0.54, p = 0.003 for protein and rho = −0.46, p = 0.015 for phosphorylation; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figures 6</a>A and 6C). Loss of Rb in TNBC leading to higher proliferation is consistent with the inhibitory role of Rb in cell cycle progression, but the association of higher Rb levels with greater proliferation in HR+ samples is contrary to its role as negative regulator of proliferation (<div class="dropBlock reference-citations"><a href="#bib46" id="back-bib46" data-db-target-for="bib46-d13420944e2436" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib46">Goel et al., 2018</a><div data-db-target-of="bib46-d13420944e2436" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Goel S. </li><li class="loa__item">DeCristo M.J. </li><li class="loa__item">McAllister S.S. </li><li class="loa__item">Zhao J.J. </li></ul><div class="reference-citations__detials">CDK4/6 Inhibition in Cancer: Beyond Cell Cycle Arrest.<div class="popup-ref"><span class="ref__series"><i>Trends Cell Biol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">911-925</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.tcb.2018.07.002&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.tcb.2018.07.002&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.tcb.2018.07.002&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30061045&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85050508022&amp;cf=" onclick="newWindow(this.href);return false">Scopus (100)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=911-925&journal=Trends+Cell+Biol.&author=Goel+S.&author=DeCristo+M.J.&author=McAllister+S.S.&author=Zhao+J.J.&title=CDK4%2F6+Inhibition+in+Cancer%3A+Beyond+Cell+Cycle+Arrest">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). However, phosphorylation of Rb by cyclin-dependent kinases relieves this inhibition, and, consistent with these regulatory events, phosphorylation levels of Rb showed a stronger correlation with MGPS in HR+/ERBB2− cases than Rb protein levels (Spearman rho = 0.53, p = 1.1 × 10<sup>−6</sup> for mean of Rb phosphosites and rho = 0.24, p = 0.035 for protein; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>A; <a href="/cms/10.1016/j.cell.2020.10.036/attachment/ce88682c-a2ae-458a-a550-73ad6faa7636/mmc7" class="scroll-into-link">Table S7</a>).<div class="floatDisplay"><figure id="gr6" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/08d8a986-1678-4762-8ba4-94ef563d7e00/gr6.jpg" alt="Figure thumbnail gr6" data-locator="fig6" data-high-res="/cms/attachment/e7c53f6d-6c55-4342-bae8-5188501ff70a/gr6_lrg.jpg" data-large-image="/cms/attachment/08d8a986-1678-4762-8ba4-94ef563d7e00/gr6.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 6" class="label figure__label">Figure 6</span><span class="figure__title__text">Rb Phosphorylation Status Indicates Potential Candidates for CDK4/6 Inhibitor Therapy in TNBC</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Heatmap of PG features related to regulation of cell cycle by the Rb protein. Samples are ordered by RNA-based multi-gene proliferation score (MGPS; <div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib39" id="back-bib39" data-db-target-for="bib39-d13420944e208" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib39">Ellis et al., 2017</a><div data-db-target-of="bib39-d13420944e208" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ellis M.J. </li><li class="loa__item">Suman V.J. </li><li class="loa__item">Hoog J. </li><li class="loa__item">Goncalves R. </li><li class="loa__item">Sanati S. </li><li class="loa__item">Creighton C.J. </li><li class="loa__item">DeSchryver K. </li><li class="loa__item">Crouch E. </li><li class="loa__item">Brink A. </li><li class="loa__item">Watson M. </li><li>et al.</li></ul><div class="reference-citations__detials">Ki67 Proliferation Index as a Tool for Chemotherapy Decisions During and After Neoadjuvant Aromatase Inhibitor Treatment of Breast Cancer: Results From the American College of Surgeons Oncology Group Z1031 Trial (Alliance).<div class="popup-ref"><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">1061-1069</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1200%2FJCO.2016.69.4406&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=28045625&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85016622416&amp;cf=" onclick="newWindow(this.href);return false">Scopus (127)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2017&pages=1061-1069&journal=J.%C2%A0Clin.+Oncol.&author=Ellis+M.J.&author=Suman+V.J.&author=Hoog+J.&author=Goncalves+R.&author=Sanati+S.&author=Creighton+C.J.&author=DeSchryver+K.&author=Crouch+E.&author=Brink+A.&author=Watson+M.&title=Ki67+Proliferation+Index+as+a+Tool+for+Chemotherapy+Decisions+During+and+After+Neoadjuvant+Aromatase+Inhibitor+Treatment+of+Breast+Cancer%3A+Results+From+the+American+College+of+Surgeons+Oncology+Group+Z1031+Trial+%28Alliance%29">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div>) within HR+ (ER+ or PR+) / ERBB2 PG− and TNBC subtypes. Correlation of each feature with the MGPS in each subtype is indicated by the bar plots along the side. The pathway diagram on the left depicts how the features included in the heatmap regulate G1-S progression to promote E2F transcription. Red boxes for SCNAs indicate gene amplification, whereas blue boxes indicate gene deletions. Phosphoprotein levels are represented by the median log2 TMT ratio of all phosphosites for a given gene. <em>Z</em> scores of kinase target NESs from single sample post-translational modification-signature enrichment analysis (PTM-SEA), of single sample GSEA NES values using MSigDb Hallmark sets, and of the stemness and CIBERSORT (CS) immune scores are also shown.</div><div class="section-paragraph">(B) Plot of Spearman correlations of kinase activity scores (kinase target PTM-SEA NES) for each Cyclin-dependent kinase (CDK) with MGPS, showing strong positive correlations between CDK4 and CDK6 with MGPS in hormone receptor+ (HR+) / ERBB2 PG− but not TNBC samples. Density plots of the distributions of the activity scores in each of the groups are shown below the corresponding point for each kinase. P values were derived from Wilcoxon rank-sum tests.</div><div class="section-paragraph">(C) Loss of Rb drives proliferation in TNBC samples, whereas phosphorylation of Rb is strongly associated with proliferation in HR+/ERBB2− samples. A scatterplot of Rb phosphoprotein (median of all phosphosites) log2 TMT ratios versus MGPS shows strong negative correlation between phospho-Rb and proliferation in TNBC samples, whereas phospho-Rb is positively correlated in HR+ / ERBB2 PG− samples. Points are colored by subtype. Red, TNBC; blue, HR+ / ERBB2 PG−.</div><div class="section-paragraph">(D) Response to palbociclib (AUC, area under the dose-response curve) in ER+ / HER2− (circles) and ER− / HER2− (triangles) BRCA cell lines from the Genomics of Drug Sensitivity to Cancer (GDSC) database (<div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib53" id="back-bib53" data-db-target-for="bib53-d13420944e219" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib53">Iorio et al., 2016</a><div data-db-target-of="bib53-d13420944e219" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Iorio F. </li><li class="loa__item">Knijnenburg T.A. </li><li class="loa__item">Vis D.J. </li><li class="loa__item">Bignell G.R. </li><li class="loa__item">Menden M.P. </li><li class="loa__item">Schubert M. </li><li class="loa__item">Aben N. </li><li class="loa__item">Gonçalves E. </li><li class="loa__item">Barthorpe S. </li><li class="loa__item">Lightfoot H. </li><li>et al.</li></ul><div class="reference-citations__detials">A Landscape of Pharmacogenomic Interactions in Cancer.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">166</span></span>: <span class="ref__seriesPages">740-754</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2016.06.017&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2016.06.017&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2016.06.017&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27397505&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84979649916&amp;cf=" onclick="newWindow(this.href);return false">Scopus (702)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2016&pages=740-754&journal=Cell&author=Iorio+F.&author=Knijnenburg+T.A.&author=Vis+D.J.&author=Bignell+G.R.&author=Menden+M.P.&author=Schubert+M.&author=Aben+N.&author=Gon%C3%A7alves+E.&author=Barthorpe+S.&author=Lightfoot+H.&title=A+Landscape+of+Pharmacogenomic+Interactions+in+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div>; <div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib145" id="back-bib145" data-db-target-for="bib145-d13420944e222" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib145">Yang et al., 2013</a><div data-db-target-of="bib145-d13420944e222" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Yang W. </li><li class="loa__item">Soares J. </li><li class="loa__item">Greninger P. </li><li class="loa__item">Edelman E.J. </li><li class="loa__item">Lightfoot H. </li><li class="loa__item">Forbes S. </li><li class="loa__item">Bindal N. </li><li class="loa__item">Beare D. </li><li class="loa__item">Smith J.A. </li><li class="loa__item">Thompson I.R. </li><li>et al.</li></ul><div class="reference-citations__detials">Genomics of Drug Sensitivity in Cancer (GDSC): a resource for therapeutic biomarker discovery in cancer cells.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">41</span></span>: <span class="ref__seriesPages">D955-D961</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgks1111&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=23180760&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84876563391&amp;cf=" onclick="newWindow(this.href);return false">Scopus (893)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=D955-D961&journal=Nucleic+Acids+Res.&author=Yang+W.&author=Soares+J.&author=Greninger+P.&author=Edelman+E.J.&author=Lightfoot+H.&author=Forbes+S.&author=Bindal+N.&author=Beare+D.&author=Smith+J.A.&author=Thompson+I.R.&title=Genomics+of+Drug+Sensitivity+in+Cancer+%28GDSC%29%3A+a+resource+for+therapeutic+biomarker+discovery+in+cancer+cells">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div>). ER− / HER2− cell lines with <em>RB1</em> mutations/deletions are refractory to treatment (AUC), whereas ER− / HER2− cell lines with wild-type <em>RB1</em> show similar sensitivity as ER+ / HER2− cell lines. Boxplots show 1.5× the interquartile range for each group, centered on the median. P value is from the Kruskal-Wallis test.</div><div class="section-paragraph">(E) Rb protein levels are negatively correlated with response to palbociclib across all HER2− BRCA cell lines from the GDSC. A scatterplot shows log2 TMT ratios for Rb protein on the y axis and AUC on the x axis. Shown are cell lines from (D) with Rb protein data. Gray triangles, wild-type (WT) ER+ / HER2− cells; gray circles, WT ER− / HER2− cells; green circles, <em>RB1</em> deletion or frameshift mutant ER− / HER2− cells; yellow circles, RB1 missense ER− / HER2− cells. A line shows the linear regression fit for Rb protein versus AUC. Spearman correlation rho and p values are also shown.</div><div class="section-paragraph">See also <a data-locator="figs7" aria-expanded="false" aria-controls="crosref2550" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs7">Figure S7</a> and <a data-locator="mmc2" aria-expanded="false" aria-controls="crosref2555" href="/cms/10.1016/j.cell.2020.10.036/attachment/a1bc3f3e-eeac-4605-a9ef-839abd334dcd/mmc2" target="_blank">Tables S2</a>, <a data-locator="mmc6" aria-expanded="false" aria-controls="crosref2560" href="/cms/10.1016/j.cell.2020.10.036/attachment/eb237fc7-15fb-4b8d-bf02-cae5f285b239/mmc6" target="_blank">S6</a>, and <a data-locator="mmc7" aria-expanded="false" aria-controls="crosref2565" href="/cms/10.1016/j.cell.2020.10.036/attachment/ce88682c-a2ae-458a-a550-73ad6faa7636/mmc7" target="_blank">S7</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/08d8a986-1678-4762-8ba4-94ef563d7e00/gr6.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/e7c53f6d-6c55-4342-bae8-5188501ff70a/gr6_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(20)31400-8&amp;id=gr6.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">Consistent with expectations (<div class="dropBlock reference-citations"><a href="#bib20" id="back-bib20" data-db-target-for="bib20-d13420944e2451" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib20">Cancer Genome Atlas Network, 2012</a><div data-db-target-of="bib20-d13420944e2451" class="dropBlock__holder"><div class="dropBlock__body"><div class="reference-citations__detials"><span>Cancer Genome Atlas Network</span><br />Comprehensive molecular portraits of human breast tumours.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">490</span></span>: <span class="ref__seriesPages">61-70</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature11412&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=23000897&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84877028141&amp;cf=" onclick="newWindow(this.href);return false">Scopus (7217)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=490&publication_year=2012&pages=61-70&journal=Nature&author=Cancer+Genome+Atlas+Network&title=Comprehensive+molecular+portraits+of+human+breast+tumours">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), the TNBC cases here were often <em>TP53</em> mutant, with active CDK2 and high levels of Cyclin E mRNA and phosphoprotein (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>A; p = 1.04 × 10<sup>−7</sup> for CDK2 activity, p = 6.2 × 10<sup>−12</sup> for CCNE1 RNA, and p = 7.3 × 10<sup>−10</sup> for CCNE1 phosphoprotein, Wilcoxon rank-sum tests comparing TNBC with HR+/ERBB2−). However, there was a clear separation of TNBC samples into Rb-low and Rb-high phosphoprotein groups (n = 16 for Rb low and n = 12 for Rb high; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">Figure S7</a>I). This difference reflects Rb protein levels and CDK4/6 activity. For example, the inferred activities for CDK4 and CDK6 were higher in TNBC tumors with high levels of Rb phosphorylation compared with TNBC tumors with low Rb phosphorylation (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">Figure S7</a>J). Predictably, three of the four TNBC tumors with <em>RB1</em> mutations/deletions had low levels of Rb phosphorylation. The role of Rb in CDK4/6 regulation in TNBC was further probed by examining published cell line perturbation experiments from the Genomics of Drug Sensitivity in Cancer (GDSC) resource (<div class="dropBlock reference-citations"><a href="#bib53" id="back-bib53" data-db-target-for="bib53-d13420944e2477" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib53">Iorio et al., 2016</a><div data-db-target-of="bib53-d13420944e2477" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Iorio F. </li><li class="loa__item">Knijnenburg T.A. </li><li class="loa__item">Vis D.J. </li><li class="loa__item">Bignell G.R. </li><li class="loa__item">Menden M.P. </li><li class="loa__item">Schubert M. </li><li class="loa__item">Aben N. </li><li class="loa__item">Gonçalves E. </li><li class="loa__item">Barthorpe S. </li><li class="loa__item">Lightfoot H. </li><li>et al.</li></ul><div class="reference-citations__detials">A Landscape of Pharmacogenomic Interactions in Cancer.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">166</span></span>: <span class="ref__seriesPages">740-754</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2016.06.017&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2016.06.017&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2016.06.017&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27397505&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84979649916&amp;cf=" onclick="newWindow(this.href);return false">Scopus (702)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2016&pages=740-754&journal=Cell&author=Iorio+F.&author=Knijnenburg+T.A.&author=Vis+D.J.&author=Bignell+G.R.&author=Menden+M.P.&author=Schubert+M.&author=Aben+N.&author=Gon%C3%A7alves+E.&author=Barthorpe+S.&author=Lightfoot+H.&title=A+Landscape+of+Pharmacogenomic+Interactions+in+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib145" id="back-bib145" data-db-target-for="bib145-d13420944e2480" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib145">Yang et al., 2013</a><div data-db-target-of="bib145-d13420944e2480" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Yang W. </li><li class="loa__item">Soares J. </li><li class="loa__item">Greninger P. </li><li class="loa__item">Edelman E.J. </li><li class="loa__item">Lightfoot H. </li><li class="loa__item">Forbes S. </li><li class="loa__item">Bindal N. </li><li class="loa__item">Beare D. </li><li class="loa__item">Smith J.A. </li><li class="loa__item">Thompson I.R. </li><li>et al.</li></ul><div class="reference-citations__detials">Genomics of Drug Sensitivity in Cancer (GDSC): a resource for therapeutic biomarker discovery in cancer cells.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">41</span></span>: <span class="ref__seriesPages">D955-D961</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgks1111&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=23180760&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84876563391&amp;cf=" onclick="newWindow(this.href);return false">Scopus (893)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=D955-D961&journal=Nucleic+Acids+Res.&author=Yang+W.&author=Soares+J.&author=Greninger+P.&author=Edelman+E.J.&author=Lightfoot+H.&author=Forbes+S.&author=Bindal+N.&author=Beare+D.&author=Smith+J.A.&author=Thompson+I.R.&title=Genomics+of+Drug+Sensitivity+in+Cancer+%28GDSC%29%3A+a+resource+for+therapeutic+biomarker+discovery+in+cancer+cells">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). In this dataset, TNBC cells with <em>RB1</em> mutations or deletions did not respond to the CDK4/6 inhibitor palbociclib, whereas some wild-type TNBC lines were responsive (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figures 6</a>A and 6D). This suggests that knowledge of genomic Rb status could prove useful for repurposing CDK4/6 inhibitors for TNBC. However, TNBC samples often showed loss of Rb protein without a detectable genomic aberration in the <em>RB1</em> gene (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>A). This raised the question of whether Rb protein estimates could contribute to prediction of CDK4/6 inhibitor activity when the <em>RB1</em> status is wild type according to genomic analysis. Consistent with this hypothesis, further analysis of the GDSC data revealed examples of <em>RB1</em> wild-type cell lines with low levels of Rb protein that were indeed less responsive to CDK4/6 inhibitor treatment (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>E). In general, Rb protein levels were correlated with response to palbociclib regardless of <em>RB1</em> genotype (Spearman rho = −0.61, p = 0.022; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>E). An exception was a cell line that had high levels of Rb protein but showed a poor response; however, this example harbored two hotspot <em>RB1</em> missense mutations in the pocket domain that is required for transcriptional repression (I388S and P515L) (<div class="dropBlock reference-citations"><a href="#bib26" id="back-bib26" data-db-target-for="bib26-d13420944e2509" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib26">Chow and Dean, 1996</a><div data-db-target-of="bib26-d13420944e2509" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Chow K.N. </li><li class="loa__item">Dean D.C. </li></ul><div class="reference-citations__detials">Domains A and B in the Rb pocket interact to form a transcriptional repressor motif.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Biol.</i> <span class="ref__seriesDate">1996; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">4862-4868</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1128%2FMCB.16.9.4862&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=8756645&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1996&pages=4862-4868&journal=Mol.+Cell.+Biol.&author=Chow+K.N.&author=Dean+D.C.&title=Domains+A+and+B+in+the+Rb+pocket+interact+to+form+a+transcriptional+repressor+motif">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). A second line with an in-frame deletion of N480 in <em>RB1</em> was resistant and had low Rb levels (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>E), consistent with reports that the N480 mutation may destabilize the Rb protein (<div class="dropBlock reference-citations"><a href="#bib47" id="back-bib47" data-db-target-for="bib47-d13420944e2518" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib47">Harbour, 2001</a><div data-db-target-of="bib47-d13420944e2518" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Harbour J.W. </li></ul><div class="reference-citations__detials">Molecular basis of low-penetrance retinoblastoma.<div class="popup-ref"><span class="ref__series"><i>Arch. Ophthalmol.</i> <span class="ref__seriesDate">2001; </span><span class="ref__seriesVolume">119</span></span>: <span class="ref__seriesPages">1699-1704</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1001%2Farchopht.119.11.1699&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=11709023&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2001&pages=1699-1704&journal=Arch.+Ophthalmol.&author=Harbour+J.W.&title=Molecular+basis+of+low-penetrance+retinoblastoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib67" id="back-bib67" data-db-target-for="bib67-d13420944e2521" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib67">Lee et al., 1998</a><div data-db-target-of="bib67-d13420944e2521" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lee J.O. </li><li class="loa__item">Russo A.A. </li><li class="loa__item">Pavletich N.P. </li></ul><div class="reference-citations__detials">Structure of the retinoblastoma tumour-suppressor pocket domain bound to a peptide from HPV E7.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">1998; </span><span class="ref__seriesVolume">391</span></span>: <span class="ref__seriesPages">859-865</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2F36038&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=9495340&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-2642601106&amp;cf=" onclick="newWindow(this.href);return false">Scopus (353)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=391&publication_year=1998&pages=859-865&journal=Nature&author=Lee+J.O.&author=Russo+A.A.&author=Pavletich+N.P.&title=Structure+of+the+retinoblastoma+tumour-suppressor+pocket+domain+bound+to+a+peptide+from+HPV+E7">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Thus, analysis of Rb provides a good example of how PG data integration could enhance prediction of drug efficacy.</div><div class="section-paragraph">To further investigate Rb-associated heterogeneity of proliferation within TNBC samples, TNBCtype was deployed (<div class="dropBlock reference-citations"><a href="#bib22" id="back-bib22" data-db-target-for="bib22-d13420944e2526" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib22">Chen et al., 2012</a><div data-db-target-of="bib22-d13420944e2526" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Chen X. </li><li class="loa__item">Li J. </li><li class="loa__item">Gray W.H. </li><li class="loa__item">Lehmann B.D. </li><li class="loa__item">Bauer J.A. </li><li class="loa__item">Shyr Y. </li><li class="loa__item">Pietenpol J.A. </li></ul><div class="reference-citations__detials">TNBCtype: A Subtyping Tool for Triple-Negative Breast Cancer.<div class="popup-ref"><span class="ref__series"><i>Cancer Inform.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">147-156</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.4137%2FCIN.S9983&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=22872785&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84864579879&amp;cf=" onclick="newWindow(this.href);return false">Scopus (144)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=147-156&journal=Cancer+Inform.&author=Chen+X.&author=Li+J.&author=Gray+W.H.&author=Lehmann+B.D.&author=Bauer+J.A.&author=Shyr+Y.&author=Pietenpol+J.A.&title=TNBCtype%3A+A+Subtyping+Tool+for+Triple-Negative+Breast+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Tumors classified as basal-like 1 (BL1) had higher proliferation scores, and most showed loss of Rb (seven of 10 BL1 TNBC samples were Rb low) as well as <em>TP53</em> mutations (nine of 10) (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>A). The few <em>TP53</em> wild-type TNBC tumors were predominantly classified as luminal androgen receptor (LAR) tumors, with lower proliferation scores than BL1 tumors (p = 0.014, Wilcoxon rank-sum test), the highest protein levels of androgen receptor (AR) within TNBC, and the presence of <em>PIK3CA</em> mutations (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>A). However, only two of the four LAR tumors were classified as Rb high, and AR protein did not show strong correlation with Rb protein (Spearman Rho = 0.27, p = 0.17) or phosphoprotein (Spearman Rho = 0.12, p = 0.54) levels in TNBC samples, in contrast to a previous study showing that 83% of AR+ samples were also Rb+ (by IHC) (<div class="dropBlock reference-citations"><a href="#bib98" id="back-bib98" data-db-target-for="bib98-d13420944e2542" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib98">Patel et al., 2020</a><div data-db-target-of="bib98-d13420944e2542" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Patel J.M. </li><li class="loa__item">Goss A. </li><li class="loa__item">Garber J.E. </li><li class="loa__item">Torous V. </li><li class="loa__item">Richardson E.T. </li><li class="loa__item">Haviland M.J. </li><li class="loa__item">Hacker M.R. </li><li class="loa__item">Freeman G.J. </li><li class="loa__item">Nalven T. </li><li class="loa__item">Alexander B. </li><li>et al.</li></ul><div class="reference-citations__detials">Retinoblastoma protein expression and its predictors in triple-negative breast cancer.<div class="popup-ref"><span class="ref__series"><i>NPJ Breast Cancer.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">19</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41523-020-0160-4&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=32550264&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85085969298&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2020&pages=19&journal=NPJ+Breast+Cancer&author=Patel+J.M.&author=Goss+A.&author=Garber+J.E.&author=Torous+V.&author=Richardson+E.T.&author=Haviland+M.J.&author=Hacker+M.R.&author=Freeman+G.J.&author=Nalven+T.&author=Alexander+B.&title=Retinoblastoma+protein+expression+and+its+predictors+in+triple-negative+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Of note, inferred mTOR kinase activity was also higher in TNBC tumors with elevated Rb phosphoprotein levels compared with Rb-low tumors, suggesting activation of the PIK3-AKT-mTOR pathway (p = 0.037, Wilcoxon rank-sum test; <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">Figure S7</a>J) despite similar frequencies of PIK3CA mutations in both groups (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>A). Thus, TNBC tumors with features demonstrating intact Rb and/or LAR represent a complex setting where PIK3CA, CDK4/6, and AR inhibition are therapeutic options to consider depending on the specific molecular characteristics of a particular tumor (<div class="dropBlock reference-citations"><a href="#bib8" id="back-bib8" data-db-target-for="bib8-d13420944e2551" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib8">Asghar et al., 2017</a><div data-db-target-of="bib8-d13420944e2551" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Asghar U.S. </li><li class="loa__item">Barr A.R. </li><li class="loa__item">Cutts R. </li><li class="loa__item">Beaney M. </li><li class="loa__item">Babina I. </li><li class="loa__item">Sampath D. </li><li class="loa__item">Giltnane J. </li><li class="loa__item">Lacap J.A. </li><li class="loa__item">Crocker L. </li><li class="loa__item">Young A. </li><li>et al.</li></ul><div class="reference-citations__detials">Single-Cell Dynamics Determines Response to CDK4/6 Inhibition in Triple-Negative Breast Cancer.<div class="popup-ref"><span class="ref__series"><i>Clin. Cancer Res.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">5561-5572</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1158%2F1078-0432.CCR-17-0369&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=28606920&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85029515087&amp;cf=" onclick="newWindow(this.href);return false">Scopus (111)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=5561-5572&journal=Clin.+Cancer+Res.&author=Asghar+U.S.&author=Barr+A.R.&author=Cutts+R.&author=Beaney+M.&author=Babina+I.&author=Sampath+D.&author=Giltnane+J.&author=Lacap+J.A.&author=Crocker+L.&author=Young+A.&title=Single-Cell+Dynamics+Determines+Response+to+CDK4%2F6+Inhibition+in+Triple-Negative+Breast+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib69" id="back-bib69" data-db-target-for="bib69-d13420944e2554" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib69">Lehmann et al., 2014</a><div data-db-target-of="bib69-d13420944e2554" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lehmann B.D. </li><li class="loa__item">Bauer J.A. </li><li class="loa__item">Schafer J.M. </li><li class="loa__item">Pendleton C.S. </li><li class="loa__item">Tang L. </li><li class="loa__item">Johnson K.C. </li><li class="loa__item">Chen X. </li><li class="loa__item">Balko J.M. </li><li class="loa__item">Gómez H. </li><li class="loa__item">Arteaga C.L. </li><li>et al.</li></ul><div class="reference-citations__detials">PIK3CA mutations in androgen receptor-positive triple negative breast cancer confer sensitivity to the combination of PI3K and androgen receptor inhibitors.<div class="popup-ref"><span class="ref__series"><i>Breast Cancer Res.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">406</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2Fs13058-014-0406-x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=25103565&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84919875495&amp;cf=" onclick="newWindow(this.href);return false">Scopus (181)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2014&pages=406&journal=Breast+Cancer+Res.&author=Lehmann+B.D.&author=Bauer+J.A.&author=Schafer+J.M.&author=Pendleton+C.S.&author=Tang+L.&author=Johnson+K.C.&author=Chen+X.&author=Balko+J.M.&author=G%C3%B3mez+H.&author=Arteaga+C.L.&title=PIK3CA+mutations+in+androgen+receptor-positive+triple+negative+breast+cancer+confer+sensitivity+to+the+combination+of+PI3K+and+androgen+receptor+inhibitors">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib75" id="back-bib75" data-db-target-for="bib75-d13420944e2558" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib75">Liu et al., 2017</a><div data-db-target-of="bib75-d13420944e2558" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liu C.-Y. </li><li class="loa__item">Lau K.-Y. </li><li class="loa__item">Hsu C.-C. </li><li class="loa__item">Chen J.-L. </li><li class="loa__item">Lee C.-H. </li><li class="loa__item">Huang T.-T. </li><li class="loa__item">Chen Y.-T. </li><li class="loa__item">Huang C.-T. </li><li class="loa__item">Lin P.-H. </li><li class="loa__item">Tseng L.-M. </li></ul><div class="reference-citations__detials">Combination of palbociclib with enzalutamide shows in vitro activity in RB proficient and androgen receptor positive triple negative breast cancer cells.<div class="popup-ref"><span class="ref__series"><i>PLoS ONE.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">e0189007</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1371%2Fjournal.pone.0189007&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29261702&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85038890108&amp;cf=" onclick="newWindow(this.href);return false">Scopus (9)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=e0189007&journal=PLoS+ONE&author=Liu+C.-Y.&author=Lau+K.-Y.&author=Hsu+C.-C.&author=Chen+J.-L.&author=Lee+C.-H.&author=Huang+T.-T.&author=Chen+Y.-T.&author=Huang+C.-T.&author=Lin+P.-H.&author=Tseng+L.-M.&title=Combination+of+palbociclib+with+enzalutamide+shows+in%C2%A0vitro+activity+in+RB+proficient+and+androgen+receptor+positive+triple+negative+breast+cancer+cells">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib144" id="back-bib144" data-db-target-for="bib144-d13420944e2561" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib144">Yamamoto et al., 2019</a><div data-db-target-of="bib144-d13420944e2561" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Yamamoto T. </li><li class="loa__item">Kanaya N. </li><li class="loa__item">Somlo G. </li><li class="loa__item">Chen S. </li></ul><div class="reference-citations__detials">Synergistic anti-cancer activity of CDK4/6 inhibitor palbociclib and dual mTOR kinase inhibitor MLN0128 in pRb-expressing ER-negative breast cancer.<div class="popup-ref"><span class="ref__series"><i>Breast Cancer Res. Treat.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">174</span></span>: <span class="ref__seriesPages">615-625</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1007%2Fs10549-018-05104-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30607633&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85059592784&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=174&publication_year=2019&pages=615-625&journal=Breast+Cancer+Res.+Treat.&author=Yamamoto+T.&author=Kanaya+N.&author=Somlo+G.&author=Chen+S.&title=Synergistic+anti-cancer+activity+of+CDK4%2F6+inhibitor+palbociclib+and+dual+mTOR+kinase+inhibitor+MLN0128+in+pRb-expressing+ER-negative+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div></section></section><section id="sec3" data-locator="sec3" tabindex="0"><h2 class="top" id="secsectitle0085" data-left-hand-nav="Discussion" tabindex="0"><span class="top__sub"></span><span class="top__text">Discussion</span></h2><div class="section-paragraph">The high-quality, multi-omics resource we created allows investigators to explore correlations between the genomic landscape and the downstream effects in the BRCA proteome, phosphoproteome, and acetylproteome, extending and refining analytical opportunities provided by prior studies (<div class="dropBlock reference-citations"><a href="#bib17" id="back-bib17" data-db-target-for="bib17-d13420944e2569" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib17">Bouchal et al., 2019</a><div data-db-target-of="bib17-d13420944e2569" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Bouchal P. </li><li class="loa__item">Schubert O.T. </li><li class="loa__item">Faktor J. </li><li class="loa__item">Capkova L. </li><li class="loa__item">Imrichova H. </li><li class="loa__item">Zoufalova K. </li><li class="loa__item">Paralova V. </li><li class="loa__item">Hrstka R. </li><li class="loa__item">Liu Y. </li><li class="loa__item">Ebhardt H.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Breast Cancer Classification Based on Proteotypes Obtained by SWATH Mass Spectrometry.<div class="popup-ref"><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">832-843.e7</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.celrep.2019.06.046&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.celrep.2019.06.046&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.celrep.2019.06.046&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31315058&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85068466554&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2019&pages=832-843.e7&journal=Cell+Rep.&author=Bouchal+P.&author=Schubert+O.T.&author=Faktor+J.&author=Capkova+L.&author=Imrichova+H.&author=Zoufalova+K.&author=Paralova+V.&author=Hrstka+R.&author=Liu+Y.&author=Ebhardt+H.A.&title=Breast+Cancer+Classification+Based+on+Proteotypes+Obtained+by+SWATH+Mass+Spectrometry">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib55" id="back-bib55" data-db-target-for="bib55-d13420944e2572" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib55">Johansson et al., 2019</a><div data-db-target-of="bib55-d13420944e2572" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Johansson H.J. </li><li class="loa__item">Socciarelli F. </li><li class="loa__item">Vacanti N.M. </li><li class="loa__item">Haugen M.H. </li><li class="loa__item">Zhu Y. </li><li class="loa__item">Siavelis I. </li><li class="loa__item">Fernandez-Woodbridge A. </li><li class="loa__item">Aure M.R. </li><li class="loa__item">Sennblad B. </li><li class="loa__item">Vesterlund M. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Consortia Oslo Breast Cancer Research Consortium (OSBREAC)</span><br />Breast cancer quantitative proteome and proteogenomic landscape.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">1600</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41467-019-09018-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30962452&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85064079271&amp;cf=" onclick="newWindow(this.href);return false">Scopus (66)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1600&journal=Nat.+Commun.&author=Consortia+Oslo+Breast+Cancer+Research+Consortium+%28OSBREAC%29&title=Breast+cancer+quantitative+proteome+and+proteogenomic+landscape">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib85" id="back-bib85" data-db-target-for="bib85-d13420944e2575" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib85">Mertins et al., 2016</a><div data-db-target-of="bib85-d13420944e2575" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Wang P. </li><li class="loa__item">Wang X. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Cao S. </li><li class="loa__item">Petralia F. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomics connects somatic mutations to signalling in breast cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (780)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=NCI+CPTAC&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib130" id="back-bib130" data-db-target-for="bib130-d13420944e2578" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib130">Tyanova et al., 2016</a><div data-db-target-of="bib130-d13420944e2578" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Tyanova S. </li><li class="loa__item">Albrechtsen R. </li><li class="loa__item">Kronqvist P. </li><li class="loa__item">Cox J. </li><li class="loa__item">Mann M. </li><li class="loa__item">Geiger T. </li></ul><div class="reference-citations__detials">Proteomic maps of breast cancer subtypes.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">10259</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fncomms10259&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26725330&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=10259&journal=Nat.+Commun.&author=Tyanova+S.&author=Albrechtsen+R.&author=Kronqvist+P.&author=Cox+J.&author=Mann+M.&author=Geiger+T.&title=Proteomic+maps+of+breast+cancer+subtypes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Numerous observations with diagnostic or therapeutic potential emerged from our analyses. In the case of ERBB2+ BRCA, we suggest that integrated DNA and protein level analysis of the long arm of chromosome 17 could be a more quantitative approach than FISH/IHC. Integrated analysis of mutational signatures and DNA repair processes, I-TME profiles, and expression of targets for immune checkpoint (IC)-directed therapies defined subsets of LumA and LumB tumors with APOBEC-mediated mutagenesis or single-strand break repair defects that could benefit from IC treatment. Our data also hint that accurate PG assessment of Rb could prove useful as a predictive marker that could enable the use of CDK4/6 inhibitors in a subset of TNBC.</div><div class="section-paragraph">Deep, quantitative analyses of phosphorylation and acetylation by proteomics provided unique observations with potential clinical effects. For example, phosphoproteomics identified new connections between tumor suppressor loss and signaling, including upregulation of RIPK3 in <em>MAP3K1</em> mutant tumors, the WNT pathway mediator TNIK1 in <em>ARID1A</em> mutant tumors, and the microtubule-associated kinase and neuroendocrine differentiation markers MAST4 and DCLK1 (<div class="dropBlock reference-citations"><a href="#bib74" id="back-bib74" data-db-target-for="bib74-d13420944e2587" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib74">Liu et al., 2016</a><div data-db-target-of="bib74-d13420944e2587" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liu Y.-H. </li><li class="loa__item">Tsang J.Y.S. </li><li class="loa__item">Ni Y.-B. </li><li class="loa__item">Hlaing T. </li><li class="loa__item">Chan S.-K. </li><li class="loa__item">Chan K.-F. </li><li class="loa__item">Ko C.-W. </li><li class="loa__item">Mujtaba S.S. </li><li class="loa__item">Tse G.M. </li></ul><div class="reference-citations__detials">Doublecortin-like kinase 1 expression associates with breast cancer with neuroendocrine differentiation.<div class="popup-ref"><span class="ref__series"><i>Oncotarget.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">1464-1476</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.18632%2Foncotarget.6386&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26621833&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84957692009&amp;cf=" onclick="newWindow(this.href);return false">Scopus (25)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=1464-1476&journal=Oncotarget&author=Liu+Y.-H.&author=Tsang+J.Y.S.&author=Ni+Y.-B.&author=Hlaing+T.&author=Chan+S.-K.&author=Chan+K.-F.&author=Ko+C.-W.&author=Mujtaba+S.S.&author=Tse+G.M.&title=Doublecortin-like+kinase+1+expression+associates+with+breast+cancer+with+neuroendocrine+differentiation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) in <em>GATA3</em> mutant tumors. The first two findings suggest potential therapeutic directions in the difficult arena of targeting tumor suppressor loss, whereas DCLK1 inhibition via the small-molecule kinase inhibitor LRRK2-IN-1 has shown preclinical efficacy in some cancers (<div class="dropBlock reference-citations"><a href="#bib58" id="back-bib58" data-db-target-for="bib58-d13420944e2592" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib58">Kawamura et al., 2017</a><div data-db-target-of="bib58-d13420944e2592" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kawamura D. </li><li class="loa__item">Takemoto Y. </li><li class="loa__item">Nishimoto A. </li><li class="loa__item">Ueno K. </li><li class="loa__item">Hosoyama T. </li><li class="loa__item">Shirasawa B. </li><li class="loa__item">Tanaka T. </li><li class="loa__item">Kugimiya N. </li><li class="loa__item">Harada E. </li><li class="loa__item">Hamano K. </li></ul><div class="reference-citations__detials">Enhancement of cytotoxic effects of gemcitabine by Dclk1 inhibition through suppression of Chk1 phosphorylation in human pancreatic cancer cells.<div class="popup-ref"><span class="ref__series"><i>Oncol. Rep.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">38</span></span>: <span class="ref__seriesPages">3238-3244</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.3892%2For.2017.5974&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29048622&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85031027382&amp;cf=" onclick="newWindow(this.href);return false">Scopus (9)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=3238-3244&journal=Oncol.+Rep.&author=Kawamura+D.&author=Takemoto+Y.&author=Nishimoto+A.&author=Ueno+K.&author=Hosoyama+T.&author=Shirasawa+B.&author=Tanaka+T.&author=Kugimiya+N.&author=Harada+E.&author=Hamano+K.&title=Enhancement+of+cytotoxic+effects+of+gemcitabine+by+Dclk1+inhibition+through+suppression+of+Chk1+phosphorylation+in+human+pancreatic+cancer+cells">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib123" id="back-bib123" data-db-target-for="bib123-d13420944e2596" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib123">Suehiro et al., 2018</a><div data-db-target-of="bib123-d13420944e2596" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Suehiro Y. </li><li class="loa__item">Takemoto Y. </li><li class="loa__item">Nishimoto A. </li><li class="loa__item">Ueno K. </li><li class="loa__item">Shirasawa B. </li><li class="loa__item">Tanaka T. </li><li class="loa__item">Kugimiya N. </li><li class="loa__item">Suga A. </li><li class="loa__item">Harada E. </li><li class="loa__item">Hamano K. </li></ul><div class="reference-citations__detials">Dclk1 Inhibition Cancels 5-FU-induced Cell-cycle Arrest and Decreases Cell Survival in Colorectal Cancer.<div class="popup-ref"><span class="ref__series"><i>Anticancer Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">38</span></span>: <span class="ref__seriesPages">6225-6230</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.21873%2Fanticanres.12977&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30396941&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85056107950&amp;cf=" onclick="newWindow(this.href);return false">Scopus (12)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2018&pages=6225-6230&journal=Anticancer+Res.&author=Suehiro+Y.&author=Takemoto+Y.&author=Nishimoto+A.&author=Ueno+K.&author=Shirasawa+B.&author=Tanaka+T.&author=Kugimiya+N.&author=Suga+A.&author=Harada+E.&author=Hamano+K.&title=Dclk1+Inhibition+Cancels+5-FU-induced+Cell-cycle+Arrest+and+Decreases+Cell+Survival+in+Colorectal+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib141" id="back-bib141" data-db-target-for="bib141-d13420944e2599" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib141">Weygant et al., 2014</a><div data-db-target-of="bib141-d13420944e2599" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Weygant N. </li><li class="loa__item">Qu D. </li><li class="loa__item">Berry W.L. </li><li class="loa__item">May R. </li><li class="loa__item">Chandrakesan P. </li><li class="loa__item">Owen D.B. </li><li class="loa__item">Sureban S.M. </li><li class="loa__item">Ali N. </li><li class="loa__item">Janknecht R. </li><li class="loa__item">Houchen C.W. </li></ul><div class="reference-citations__detials">Small molecule kinase inhibitor LRRK2-IN-1 demonstrates potent activity against colorectal and pancreatic cancer through inhibition of doublecortin-like kinase 1.<div class="popup-ref"><span class="ref__series"><i>Mol. Cancer.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">103</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2F1476-4598-13-103&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=24885928&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84901279099&amp;cf=" onclick="newWindow(this.href);return false">Scopus (62)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=103&journal=Mol.+Cancer&author=Weygant+N.&author=Qu+D.&author=Berry+W.L.&author=May+R.&author=Chandrakesan+P.&author=Owen+D.B.&author=Sureban+S.M.&author=Ali+N.&author=Janknecht+R.&author=Houchen+C.W.&title=Small+molecule+kinase+inhibitor+LRRK2-IN-1+demonstrates+potent+activity+against+colorectal+and+pancreatic+cancer+through+inhibition+of+doublecortin-like+kinase+1">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Proteomics and acetylproteomics profiling in the context of metabolism also revealed, for the first time in a large BRCA cohort, marked differences in metabolic enzyme expression and acetylation between luminally- and basally-enriched subtypes, which may translate to a better understanding of metabolic vulnerabilities. Suppression of serine metabolic enzymes such as PHGDH selectively decreases proliferation in cells with elevated serine flux (<div class="dropBlock reference-citations"><a href="#bib103" id="back-bib103" data-db-target-for="bib103-d13420944e2602" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib103">Possemato et al., 2011</a><div data-db-target-of="bib103-d13420944e2602" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Possemato R. </li><li class="loa__item">Marks K.M. </li><li class="loa__item">Shaul Y.D. </li><li class="loa__item">Pacold M.E. </li><li class="loa__item">Kim D. </li><li class="loa__item">Birsoy K. </li><li class="loa__item">Sethumadhavan S. </li><li class="loa__item">Woo H.-K. </li><li class="loa__item">Jang H.G. </li><li class="loa__item">Jha A.K. </li><li>et al.</li></ul><div class="reference-citations__detials">Functional genomics reveal that the serine synthesis pathway is essential in breast cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">476</span></span>: <span class="ref__seriesPages">346-350</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature10350&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=21760589&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-80051923932&amp;cf=" onclick="newWindow(this.href);return false">Scopus (953)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=476&publication_year=2011&pages=346-350&journal=Nature&author=Possemato+R.&author=Marks+K.M.&author=Shaul+Y.D.&author=Pacold+M.E.&author=Kim+D.&author=Birsoy+K.&author=Sethumadhavan+S.&author=Woo+H.-K.&author=Jang+H.G.&author=Jha+A.K.&title=Functional+genomics+reveal+that+the+serine+synthesis+pathway+is+essential+in+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), opening a potential therapeutic alternative for difficult-to-treat basal tumors (<div class="dropBlock reference-citations"><a href="#bib90" id="back-bib90" data-db-target-for="bib90-d13420944e2605" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib90">Mullarky et al., 2019</a><div data-db-target-of="bib90-d13420944e2605" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mullarky E. </li><li class="loa__item">Xu J. </li><li class="loa__item">Robin A.D. </li><li class="loa__item">Huggins D.J. </li><li class="loa__item">Jennings A. </li><li class="loa__item">Noguchi N. </li><li class="loa__item">Olland A. </li><li class="loa__item">Lakshminarasimhan D. </li><li class="loa__item">Miller M. </li><li class="loa__item">Tomita D. </li><li>et al.</li></ul><div class="reference-citations__detials">Inhibition of 3-phosphoglycerate dehydrogenase (PHGDH) by indole amides abrogates de novo serine synthesis in cancer cells.<div class="popup-ref"><span class="ref__series"><i>Bioorg. Med. Chem. Lett.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">2503-2510</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.bmcl.2019.07.011&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31327531&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85069538867&amp;cf=" onclick="newWindow(this.href);return false">Scopus (21)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=2503-2510&journal=Bioorg.+Med.+Chem.+Lett.&author=Mullarky+E.&author=Xu+J.&author=Robin+A.D.&author=Huggins+D.J.&author=Jennings+A.&author=Noguchi+N.&author=Olland+A.&author=Lakshminarasimhan+D.&author=Miller+M.&author=Tomita+D.&title=Inhibition+of+3-phosphoglycerate+dehydrogenase+%28PHGDH%29+by+indole+amides+abrogates+de+novo+serine+synthesis+in+cancer+cells">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib91" id="back-bib91" data-db-target-for="bib91-d13420944e2608" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib91">Murphy et al., 2018</a><div data-db-target-of="bib91-d13420944e2608" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Murphy J.P. </li><li class="loa__item">Giacomantonio M.A. </li><li class="loa__item">Paulo J.A. </li><li class="loa__item">Everley R.A. </li><li class="loa__item">Kennedy B.E. </li><li class="loa__item">Pathak G.P. </li><li class="loa__item">Clements D.R. </li><li class="loa__item">Kim Y. </li><li class="loa__item">Dai C. </li><li class="loa__item">Sharif T. </li><li>et al.</li></ul><div class="reference-citations__detials">The NAD<sup>+</sup> Salvage Pathway Supports PHGDH-Driven Serine Biosynthesis.<div class="popup-ref"><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">2381-2391.e5</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.celrep.2018.07.086&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.celrep.2018.07.086&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.celrep.2018.07.086&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30157431&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85051748917&amp;cf=" onclick="newWindow(this.href);return false">Scopus (23)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=2381-2391.e5&journal=Cell+Rep.&author=Murphy+J.P.&author=Giacomantonio+M.A.&author=Paulo+J.A.&author=Everley+R.A.&author=Kennedy+B.E.&author=Pathak+G.P.&author=Clements+D.R.&author=Kim+Y.&author=Dai+C.&author=Sharif+T.&title=The+NAD%2B+Salvage+Pathway+Supports+PHGDH-Driven+Serine+Biosynthesis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib140" id="back-bib140" data-db-target-for="bib140-d13420944e2611" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib140">Weinstabl et al., 2019</a><div data-db-target-of="bib140-d13420944e2611" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Weinstabl H. </li><li class="loa__item">Treu M. </li><li class="loa__item">Rinnenthal J. </li><li class="loa__item">Zahn S.K. </li><li class="loa__item">Ettmayer P. </li><li class="loa__item">Bader G. </li><li class="loa__item">Dahmann G. </li><li class="loa__item">Kessler D. </li><li class="loa__item">Rumpel K. </li><li class="loa__item">Mischerikow N. </li><li>et al.</li></ul><div class="reference-citations__detials">Intracellular Trapping of the Selective Phosphoglycerate Dehydrogenase (PHGDH) Inhibitor BI-4924 Disrupts Serine Biosynthesis.<div class="popup-ref"><span class="ref__series"><i>J. Med. Chem.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">62</span></span>: <span class="ref__seriesPages">7976-7997</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1021%2Facs.jmedchem.9b00718&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31365252&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85071640342&amp;cf=" onclick="newWindow(this.href);return false">Scopus (23)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=7976-7997&journal=J.%C2%A0Med.+Chem.&author=Weinstabl+H.&author=Treu+M.&author=Rinnenthal+J.&author=Zahn+S.K.&author=Ettmayer+P.&author=Bader+G.&author=Dahmann+G.&author=Kessler+D.&author=Rumpel+K.&author=Mischerikow+N.&title=Intracellular+Trapping+of+the+Selective+Phosphoglycerate+Dehydrogenase+%28PHGDH%29+Inhibitor+BI-4924+Disrupts+Serine+Biosynthesis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Our results suggest a synergistic interaction between hypoacetylation and elevated protein expression leading to increased activity of the glycolysis pathway in the NMF Basal-I subtype; in contrast, mitochondrial function appeared to be suppressed by hyperacetylation mediated by depleted SIRT3. Broad dependence of tissues on glucose and products of respiration suggests that the therapeutic window for targeting increased aerobic glycolysis or compromised TCA cycle enzymes is narrow (<div class="dropBlock reference-citations"><a href="#bib76" id="back-bib76" data-db-target-for="bib76-d13420944e2615" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib76">Luengo et al., 2017</a><div data-db-target-of="bib76-d13420944e2615" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Luengo A. </li><li class="loa__item">Gui D.Y. </li><li class="loa__item">Vander Heiden M.G. </li></ul><div class="reference-citations__detials">Targeting Metabolism for Cancer Therapy.<div class="popup-ref"><span class="ref__series"><i>Cell Chem. Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">1161-1180</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.chembiol.2017.08.028&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.chembiol.2017.08.028&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.chembiol.2017.08.028&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=28938091&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=1161-1180&journal=Cell+Chem.+Biol.&author=Luengo+A.&author=Gui+D.Y.&author=Vander+Heiden+M.G.&title=Targeting+Metabolism+for+Cancer+Therapy">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Nevertheless, the prospect of effective therapeutic targeting of metabolism is predicated on such nuanced insights into the metabolic phenotypes of specific disease states (<div class="dropBlock reference-citations"><a href="#bib132" id="back-bib132" data-db-target-for="bib132-d13420944e2618" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib132">Vander Heiden and DeBerardinis, 2017</a><div data-db-target-of="bib132-d13420944e2618" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Vander Heiden M.G. </li><li class="loa__item">DeBerardinis R.J. </li></ul><div class="reference-citations__detials">Understanding the Intersections between Metabolism and Cancer Biology.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">168</span></span>: <span class="ref__seriesPages">657-669</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2016.12.039&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2016.12.039&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2016.12.039&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=28187287&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85012011397&amp;cf=" onclick="newWindow(this.href);return false">Scopus (791)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=168&publication_year=2017&pages=657-669&journal=Cell&author=Vander+Heiden+M.G.&author=DeBerardinis+R.J.&title=Understanding+the+Intersections+between+Metabolism+and+Cancer+Biology">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div><div class="section-paragraph">There are limitations to this study and to multi-omics resource studies in general. Investment in prospective sample collection promoted data quality but meant that the sample population might not be optimized for subgroup or demographic representation. Use of cryopulverized bulk tumor material improved the depth and internal concordance of molecular analysis but sacrificed architectural information and the cellular resolution afforded by methods such as imaging mass cytometry (<div class="dropBlock reference-citations"><a href="#bib54" id="back-bib54" data-db-target-for="bib54-d13420944e2623" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib54">Jackson et al., 2020</a><div data-db-target-of="bib54-d13420944e2623" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Jackson H.W. </li><li class="loa__item">Fischer J.R. </li><li class="loa__item">Zanotelli V.R.T. </li><li class="loa__item">Ali H.R. </li><li class="loa__item">Mechera R. </li><li class="loa__item">Soysal S.D. </li><li class="loa__item">Moch H. </li><li class="loa__item">Muenst S. </li><li class="loa__item">Varga Z. </li><li class="loa__item">Weber W.P. </li><li class="loa__item">Bodenmiller B. </li></ul><div class="reference-citations__detials">The single-cell pathology landscape of breast cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">578</span></span>: <span class="ref__seriesPages">615-620</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41586-019-1876-x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31959985&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85078202432&amp;cf=" onclick="newWindow(this.href);return false">Scopus (155)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=578&publication_year=2020&pages=615-620&journal=Nature&author=Jackson+H.W.&author=Fischer+J.R.&author=Zanotelli+V.R.T.&author=Ali+H.R.&author=Mechera+R.&author=Soysal+S.D.&author=Moch+H.&author=Muenst+S.&author=Varga+Z.&author=Weber+W.P.&author=Bodenmiller+B.&title=The+single-cell+pathology+landscape+of+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Higher spatial resolution could be achieved by approaches optimized for smaller amounts of input material (<div class="dropBlock reference-citations"><a href="#bib51" id="back-bib51" data-db-target-for="bib51-d13420944e2626" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib51">Hunt et al., 2019</a><div data-db-target-of="bib51-d13420944e2626" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hunt A.L. </li><li class="loa__item">Bateman N.W. </li><li class="loa__item">Hood B.L. </li><li class="loa__item">Conrads K.A. </li><li class="loa__item">Zhou M. </li><li class="loa__item">Litzi T.J. </li><li class="loa__item">Oliver J. </li><li class="loa__item">Mitchell D. </li><li class="loa__item">Gist G. </li><li class="loa__item">Blanton B. </li><li>et al.</li></ul><div class="reference-citations__detials">Extensive Intratumor Proteogenomic Heterogeneity Revealed by Multiregion Sampling in a High-Grade Serous Ovarian Tumor Specimen.<div class="popup-ref"><span class="ref__series"><i>bioRxiv.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1101/761155">https://doi.org/10.1101/761155</a></span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1101%2F761155&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85095660210&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&journal=bioRxiv&author=Hunt+A.L.&author=Bateman+N.W.&author=Hood+B.L.&author=Conrads+K.A.&author=Zhou+M.&author=Litzi+T.J.&author=Oliver+J.&author=Mitchell+D.&author=Gist+G.&author=Blanton+B.&title=Extensive+Intratumor+Proteogenomic+Heterogeneity+Revealed+by+Multiregion+Sampling+in+a+High-Grade+Serous+Ovarian+Tumor+Specimen">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib113" id="back-bib113" data-db-target-for="bib113-d13420944e2629" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib113">Satpathy et al., 2020</a><div data-db-target-of="bib113-d13420944e2629" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Satpathy S. </li><li class="loa__item">Jaehnig E.J. </li><li class="loa__item">Krug K. </li><li class="loa__item">Kim B.-J. </li><li class="loa__item">Saltzman A.B. </li><li class="loa__item">Chan D.W. </li><li class="loa__item">Holloway K.R. </li><li class="loa__item">Anurag M. </li><li class="loa__item">Huang C. </li><li class="loa__item">Singh P. </li><li>et al.</li></ul><div class="reference-citations__detials">Microscaled proteogenomic methods for precision oncology.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">532</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41467-020-14381-2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31988290&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85078365239&amp;cf=" onclick="newWindow(this.href);return false">Scopus (25)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=532&journal=Nat.+Commun.&author=Satpathy+S.&author=Jaehnig+E.J.&author=Krug+K.&author=Kim+B.-J.&author=Saltzman+A.B.&author=Chan+D.W.&author=Holloway+K.R.&author=Anurag+M.&author=Huang+C.&author=Singh+P.&title=Microscaled+proteogenomic+methods+for+precision+oncology">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) or thoughtful integration of single-cell genomics and proteomics. The type of associations described throughout this manuscript are hypothesis generating and therefore cannot be understood in terms of firm biological conclusions or direct evidence of specific therapeutic interventions. Nevertheless, successful integration of deep-scale proteomics and post-translational modification (PTM) data from a large, prospectively collected BRCA sample set represents a substantial advance over prior genomics studies and an important complement to other PG efforts.</div><div class="section-paragraph">Deep PG analyses of high-quality tissues from well-annotated cancer patient cohorts are an important resource for the clinical and research communities. The future direction of PG requires full integration of these analytical approaches into therapeutic trials and, ultimately, clinical care. Most clinical decision-making is based on core needle biopsies, hence our emphasis on microscaled workflows that reduce sample requirements in comparison with the surgical specimen-scale analyses described here (<div class="dropBlock reference-citations"><a href="#bib113" id="back-bib113" data-db-target-for="bib113-d13420944e2634" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib113">Satpathy et al., 2020</a><div data-db-target-of="bib113-d13420944e2634" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Satpathy S. </li><li class="loa__item">Jaehnig E.J. </li><li class="loa__item">Krug K. </li><li class="loa__item">Kim B.-J. </li><li class="loa__item">Saltzman A.B. </li><li class="loa__item">Chan D.W. </li><li class="loa__item">Holloway K.R. </li><li class="loa__item">Anurag M. </li><li class="loa__item">Huang C. </li><li class="loa__item">Singh P. </li><li>et al.</li></ul><div class="reference-citations__detials">Microscaled proteogenomic methods for precision oncology.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">532</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41467-020-14381-2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31988290&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85078365239&amp;cf=" onclick="newWindow(this.href);return false">Scopus (25)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=532&journal=Nat.+Commun.&author=Satpathy+S.&author=Jaehnig+E.J.&author=Krug+K.&author=Kim+B.-J.&author=Saltzman+A.B.&author=Chan+D.W.&author=Holloway+K.R.&author=Anurag+M.&author=Huang+C.&author=Singh+P.&title=Microscaled+proteogenomic+methods+for+precision+oncology">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Microscaled PG will also facilitate detection of treatment perturbations that shed light on mechanisms of response and resistance to therapy. The results of such studies could then be used to develop candidate lists of peptides and their modifications for targeted, rapid, mass spectrometry-based assays that could be implemented in the clinic (<div class="dropBlock reference-citations"><a href="#bib44" id="back-bib44" data-db-target-for="bib44-d13420944e2637" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib44">Gillette and Carr, 2013</a><div data-db-target-of="bib44-d13420944e2637" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gillette M.A. </li><li class="loa__item">Carr S.A. </li></ul><div class="reference-citations__detials">Quantitative analysis of peptides and proteins in biomedicine by targeted mass spectrometry.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">28-34</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnmeth.2309&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=23269374&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84871983086&amp;cf=" onclick="newWindow(this.href);return false">Scopus (326)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=28-34&journal=Nat.+Methods&author=Gillette+M.A.&author=Carr+S.A.&title=Quantitative+analysis+of+peptides+and+proteins+in+biomedicine+by+targeted+mass+spectrometry">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib148" id="back-bib148" data-db-target-for="bib148-d13420944e2640" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib148">Zhang et al., 2019</a><div data-db-target-of="bib148-d13420944e2640" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang B. </li><li class="loa__item">Whiteaker J.R. </li><li class="loa__item">Hoofnagle A.N. </li><li class="loa__item">Baird G.S. </li><li class="loa__item">Rodland K.D. </li><li class="loa__item">Paulovich A.G. </li></ul><div class="reference-citations__detials">Clinical potential of mass spectrometry-based proteogenomics.<div class="popup-ref"><span class="ref__series"><i>Nat. Rev. Clin. Oncol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">256-268</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41571-018-0135-7&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30487530&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85057891078&amp;cf=" onclick="newWindow(this.href);return false">Scopus (66)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2019&pages=256-268&journal=Nat.+Rev.+Clin.+Oncol.&author=Zhang+B.&author=Whiteaker+J.R.&author=Hoofnagle+A.N.&author=Baird+G.S.&author=Rodland+K.D.&author=Paulovich+A.G.&title=Clinical+potential+of+mass+spectrometry-based+proteogenomics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Thus, we propose that strategic introduction of PG into clinical workflows will enable more rapid progress of precision diagnostics and therapeutics.</div></section><section id="sec4" data-locator="sec4" tabindex="0"><h2 class="top" id="secsectitle0090" data-left-hand-nav="STAR★Methods" tabindex="0"><span class="top__sub"></span><span class="top__text">STAR★Methods</span></h2><section id="sec4.1" data-locator="sec4.1" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Key Resources Table</span></h3><div class="section-paragraph"><div id="undtbl1" class="inline-table" tabindex="0"><div class="inline-table__head no-label"><span class="inline-table__label no-caption">Tabled
                            1</span><span class="inline-table__title"></span></div><table width="100%" cellpadding="0" cellspacing="0" border="0" class="scrollable bordered" tabindex="0"><thead><tr><th class="rowsep">REAGENT or RESOURCE</th><th class="rowsep">SOURCE</th><th class="rowsep">IDENTIFIER</th></tr></thead><tbody><tr><td class="rowsep" colspan="3"><strong>Antibodies</strong></td></tr><tr><td>Mouse monoclonal anti-CD8 (C8/144B)</td><td>Cellmarque</td><td>Catalog #108M; RRID: <a class="inter-ref-link" href="http://antibodyregistry.org/AB_1158205-1158210" target="_blank">AB_1158205-1158210</a></td></tr><tr><td>Rabbit monoclonal anti-CD4 (SP35)</td><td>Roche</td><td>Catalog #790-4423; RRID: <a class="inter-ref-link" href="http://antibodyregistry.org/AB_2335982" target="_blank">AB_2335982</a></td></tr><tr><td>Liquid Concentrated Monoclonal Antibody anti-CD163</td><td>Leica Biosystems</td><td>Catalog #NCL-L-163; RRID: <a class="inter-ref-link" href="http://antibodyregistry.org/AB_2756375" target="_blank">AB_2756375</a></td></tr><tr><td class="rowsep">PTMScan Acetyl-lysine Kit</td><td class="rowsep">Cell Signaling Technology</td><td class="rowsep">Catalog: 13416</td></tr><tr><td class="rowsep" colspan="3"><strong>Biological Samples</strong></td></tr><tr><td class="rowsep">Primary tumor samples</td><td class="rowsep">See <a href="#sec4.3" class="scroll-into-link">Experimental Model and Subject Details</a></td><td class="rowsep">N/A</td></tr><tr><td class="rowsep" colspan="3"><strong>Chemicals and Reagents</strong></td></tr><tr><td>HPLC-grade water</td><td>J.T. Baker</td><td>Catalog: 4218-03</td></tr><tr><td>Urea</td><td>Sigma</td><td>Catalog: U0631</td></tr><tr><td>Sodium chloride</td><td>Sigma</td><td>Catalog: 71376</td></tr><tr><td>1M Tris, pH 8.0</td><td>Invitrogen</td><td>Catalog: AM9855G</td></tr><tr><td>Ethylenediaminetetraacetic acid</td><td>Sigma</td><td>Catalog: E7889</td></tr><tr><td>Aprotinin</td><td>Sigma</td><td>Catalog: A6103</td></tr><tr><td>Leupeptin</td><td>Roche</td><td>Catalog: 11017101001</td></tr><tr><td>Phenylmethylsulfonyl fluoride</td><td>Sigma</td><td>Catalog: 78830</td></tr><tr><td>Sodium fluoride</td><td>Sigma</td><td>Catalog: S7920</td></tr><tr><td>Phosphatase inhibitor cocktail 2</td><td>Sigma</td><td>Catalog: P5726</td></tr><tr><td>Phosphatase inhibitor cocktail 3</td><td>Sigma</td><td>Catalog: P0044</td></tr><tr><td>Dithiothretiol, No-Weigh Format</td><td>Fisher Scientific</td><td>Catalog: 20291</td></tr><tr><td>Iodoacetamide</td><td>Sigma</td><td>Catalog: A3221</td></tr><tr><td>Lysyl endopeptidase</td><td>Wako Chemicals</td><td>Catalog: 129-02541</td></tr><tr><td>Sequencing-grade modified trypsin</td><td>Promega</td><td>Catalog: V511X</td></tr><tr><td>Formic acid</td><td>Sigma</td><td>Catalog: F0507</td></tr><tr><td>Acetonitrile</td><td>Honeywell</td><td>Catalog: 34967</td></tr><tr><td>Trifluoroacetic acid</td><td>Sigma</td><td>Catalog: 302031</td></tr><tr><td>Tandem Mass Tag reagent kit – 10plex</td><td>ThermoFisher</td><td>Catalog: 90406</td></tr><tr><td>0.5M HEPES, pH 8.5</td><td>Alfa Aesar</td><td>Catalog: J63218</td></tr><tr><td>Hydroxylamine solution, 50% (vol/vol) in H<sub>2</sub>O</td><td>Aldrich</td><td>Catalog: 467804</td></tr><tr><td>Methanol</td><td>Honeywell</td><td>Catalog: 34966</td></tr><tr><td>Ammonium hydroxide solution, 28% (wt/vol) in H<sub>2</sub>O</td><td>Sigma</td><td>Catalog: 338818</td></tr><tr><td>Ni-NTA agarose beads</td><td>QIAGEN</td><td>Catalog: 30410</td></tr><tr><td>Iron (III) chloride</td><td>Sigma</td><td>Catalog: 451649</td></tr><tr><td>Acetic acid, glacial</td><td>Sigma</td><td>Catalog: AX0073</td></tr><tr><td>Potassium phosphate, monobasic</td><td>Sigma</td><td>Catalog: P0662</td></tr><tr><td>Potassium phosphate, dibasic</td><td>Sigma</td><td>Catalog: P3786</td></tr><tr><td>MOPS</td><td>Sigma</td><td>Catalog: M5162</td></tr><tr><td>Sodium hydroxide</td><td>VWR</td><td>Catalog: BDH7225</td></tr><tr><td>Sodium phosphate, dibasic</td><td>Sigma</td><td>Catalog: S9763</td></tr><tr><td>Phosphate-buffered saline</td><td>Fisher Scientific</td><td>Catalog: 10010023</td></tr><tr><td class="rowsep">iVIEW DAB Detection Kit</td><td class="rowsep">Roche</td><td class="rowsep">Catalog: 760-091</td></tr><tr><td class="rowsep" colspan="3"><strong>Equipment</strong></td></tr><tr><td>Reversed-phase tC18 SepPak, 1cc 100mg</td><td>Waters</td><td>Catalog: WAT∖036820</td></tr><tr><td>Solid-phase C18 disk, for Stage-tips</td><td>Empore</td><td>Catalog: 66883-U</td></tr><tr><td>Stage-tip needle</td><td>Cadence</td><td>Catalog: 7928</td></tr><tr><td>Stage-tip puncher, PEEK tubing</td><td>Idex Health &amp; Science</td><td>Catalog: 1581</td></tr><tr><td>PicoFrit LC-MS column</td><td>New Objective</td><td>Catalog: PF360-75-10-N-5</td></tr><tr><td>ReproSil-Pur, 120 Å, C18-AQ, 1.9-μm resin</td><td>Dr. Maisch</td><td>Catalog: r119.aq</td></tr><tr><td>Nanospray column heater</td><td>Phoenix S&amp;T</td><td>Catalog: PST-CH-20U</td></tr><tr><td>Column heater controller</td><td>Phoenix S&amp;T</td><td>Catalog: PST-CHC</td></tr><tr><td>300 μL LC-MS autosampler vial and cap</td><td>Waters</td><td>Catalog: 186002639</td></tr><tr><td>Offline HPLC column, 3.5-μm particle size, 4.6 um × 250 mm</td><td>Agilent</td><td>Catalog: Custom order</td></tr><tr><td>Offline 96-well fractionation plate</td><td>Whatman</td><td>Catalog: 77015200</td></tr><tr><td>700 μL bRP fractionation autosampler vial</td><td>ThermoFisher</td><td>Catalog: C4010-14</td></tr><tr><td>700 μL bRP fractionation autosampler cap</td><td>ThermoFisher</td><td>Catalog: C4010-55A</td></tr><tr><td>96-well microplate for BCA</td><td>Greiner</td><td>Catalog: 655101</td></tr><tr><td>Microplate foil cover</td><td>Corning</td><td>Catalog: PCR-AS-200</td></tr><tr><td>Vacuum centrifuge</td><td>ThermoFisher</td><td>Catalog: SPD121P-115</td></tr><tr><td>Centrifuge</td><td>Eppendorf</td><td>Catalog: 5427 R</td></tr><tr><td>Benchtop mini centrifuge</td><td>Corning</td><td>Catalog: 6765</td></tr><tr><td>Benchtop vortex</td><td>Scientific Industries</td><td>Catalog: SI-0236</td></tr><tr><td>Incubating shaker</td><td>VWR</td><td>Catalog: 12620-942</td></tr><tr><td>15 mL centrifuge tube</td><td>Corning</td><td>Catalog: 352097</td></tr><tr><td>50 mL centrifuge tube</td><td>Corning</td><td>Catalog: 352070</td></tr><tr><td>1.5 mL microtube w/o cap</td><td>Sarstedt</td><td>Catalog: 72.607</td></tr><tr><td>2.0 mL microtube w/o cap</td><td>Sarstedt</td><td>Catalog: 72.608</td></tr><tr><td>Microtube caps</td><td>Sarstedt</td><td>Catalog: 72.692</td></tr><tr><td>1.5 mL snapcap tube</td><td>ThermoFisher</td><td>Catalog: AM12450</td></tr><tr><td class="rowsep">2.0 mL snapcap tube</td><td class="rowsep">ThermoFisher</td><td class="rowsep">Catalog: AM12475</td></tr><tr><td class="rowsep" colspan="3"><strong>Instrumentation</strong></td></tr><tr><td>Microplate Reader</td><td>Molecular Devices</td><td>Catalog: M2</td></tr><tr><td>Offline HPLC System for bRP fractionation</td><td>Agilent</td><td>Catalog: G1380-90000</td></tr><tr><td>Online LC for LC-MS</td><td>ThermoFisher</td><td>Catalog: LC140</td></tr><tr><td>Q Exactive Plus Mass Spectrometer</td><td>ThermoFisher</td><td>Catalog: IQLAAEGA APFALGMBDK</td></tr><tr><td class="rowsep">Orbitrap Fusion Lumos Tribrid Mass Spectrometer</td><td class="rowsep">ThermoFisher</td><td class="rowsep">Catalog: IQLAAEGA APFADBMBHQ</td></tr><tr><td class="rowsep" colspan="3"><strong>Critical Commercial Assays</strong></td></tr><tr><td>TruSeq Stranded Total RNA Library Prep Kit with Ribo-Zero Gold</td><td>Illumina</td><td>Catalog: RS-122-2301</td></tr><tr><td>Infinium MethylationEPIC Kit</td><td>Illumina</td><td>Catalog: WG-317-1003</td></tr><tr><td>Nextera DNA Exosome Kit</td><td>Illumina</td><td>Catalog: 20020617</td></tr><tr><td>KAPA Hyper Prep Kit, PCR-free</td><td>Roche</td><td>Catalog: 07962371001</td></tr><tr><td class="rowsep">BCA Protein Assay Kit</td><td class="rowsep">ThermoFisher</td><td class="rowsep">Catalog: 23225</td></tr><tr><td class="rowsep" colspan="3"><strong>Deposited Data</strong></td></tr><tr><td>Proteomics data</td><td>CPTAC Data Portal (<a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu" target="_blank">https://cptac-data-portal.georgetown.edu</a>)</td><td><a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/study-summary/S060" target="_blank">https://cptac-data-portal.georgetown.edu/study-summary/S060</a></td></tr><tr><td>Proteomics data</td><td>Proteomic Data Commons (<a class="inter-ref-link" href="https://pdc.cancer.gov" target="_blank">https://pdc.cancer.gov</a>)</td><td>PDC000120</td></tr><tr><td class="rowsep">Genomics data</td><td class="rowsep"><a class="inter-ref-link" href="http://dbGaP" target="_blank">dbGaP</a></td><td class="rowsep">phs000892</td></tr><tr><td class="rowsep" colspan="3"><strong>Software and Algorithms</strong></td></tr><tr><td>Terra</td><td>Broad Institute data science platform.</td><td><a class="inter-ref-link" href="https://terra.bio/" target="_blank">https://terra.bio/</a></td></tr><tr><td>ContEst</td><td><div class="dropBlock reference-citations"><a href="#bib27" id="back-bib27" data-db-target-for="bib27-d13420944e3276" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib27">Cibulskis et al., 2011</a><div data-db-target-of="bib27-d13420944e3276" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Cibulskis K. </li><li class="loa__item">McKenna A. </li><li class="loa__item">Fennell T. </li><li class="loa__item">Banks E. </li><li class="loa__item">DePristo M. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">ContEst: estimating cross-contamination of human samples in next-generation sequencing data.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">27</span></span>: <span class="ref__seriesPages">2601-2602</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fbioinformatics%2Fbtr446&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=21803805&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-80052707308&amp;cf=" onclick="newWindow(this.href);return false">Scopus (152)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2011&pages=2601-2602&journal=Bioinformatics&author=Cibulskis+K.&author=McKenna+A.&author=Fennell+T.&author=Banks+E.&author=DePristo+M.&author=Getz+G.&title=ContEst%3A+estimating+cross-contamination+of+human+samples+in+next-generation+sequencing+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://software.broadinstitute.org/cancer/cga/contest" target="_blank">https://software.broadinstitute.org/cancer/cga/contest</a></td></tr><tr><td>MuTect</td><td><div class="dropBlock reference-citations"><a href="#bib28" id="back-bib28" data-db-target-for="bib28-d13420944e3286" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib28">Cibulskis et al., 2013</a><div data-db-target-of="bib28-d13420944e3286" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Cibulskis K. </li><li class="loa__item">Lawrence M.S. </li><li class="loa__item">Carter S.L. </li><li class="loa__item">Sivachenko A. </li><li class="loa__item">Jaffe D. </li><li class="loa__item">Sougnez C. </li><li class="loa__item">Gabriel S. </li><li class="loa__item">Meyerson M. </li><li class="loa__item">Lander E.S. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples.<div class="popup-ref"><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">213-219</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnbt.2514&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=23396013&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84874025843&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2431)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2013&pages=213-219&journal=Nat.+Biotechnol.&author=Cibulskis+K.&author=Lawrence+M.S.&author=Carter+S.L.&author=Sivachenko+A.&author=Jaffe+D.&author=Sougnez+C.&author=Gabriel+S.&author=Meyerson+M.&author=Lander+E.S.&author=Getz+G.&title=Sensitive+detection+of+somatic+point+mutations+in+impure+and+heterogeneous+cancer+samples">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://software.broadinstitute.org/cancer/cga/mutect" target="_blank">https://software.broadinstitute.org/cancer/cga/mutect</a></td></tr><tr><td>Strelka</td><td><div class="dropBlock reference-citations"><a href="#bib61" id="back-bib61" data-db-target-for="bib61-d13420944e3295" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib61">Kim et al., 2018</a><div data-db-target-of="bib61-d13420944e3295" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kim S. </li><li class="loa__item">Scheffler K. </li><li class="loa__item">Halpern A.L. </li><li class="loa__item">Bekritsky M.A. </li><li class="loa__item">Noh E. </li><li class="loa__item">Källberg M. </li><li class="loa__item">Chen X. </li><li class="loa__item">Kim Y. </li><li class="loa__item">Beyter D. </li><li class="loa__item">Krusche P. </li><li class="loa__item">Saunders C.T. </li></ul><div class="reference-citations__detials">Strelka2: fast and accurate calling of germline and somatic variants.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">591-594</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41592-018-0051-x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30013048&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85049953163&amp;cf=" onclick="newWindow(this.href);return false">Scopus (263)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=591-594&journal=Nat.+Methods&author=Kim+S.&author=Scheffler+K.&author=Halpern+A.L.&author=Bekritsky+M.A.&author=Noh+E.&author=K%C3%A4llberg+M.&author=Chen+X.&author=Kim+Y.&author=Beyter+D.&author=Krusche+P.&author=Saunders+C.T.&title=Strelka2%3A+fast+and+accurate+calling+of+germline+and+somatic+variants">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://github.com/Illumina/strelka" target="_blank">https://github.com/Illumina/strelka</a></td></tr><tr><td>AllelicCapSeg</td><td></td><td><a class="inter-ref-link" href="https://github.com/aaronmck/CapSeg" target="_blank">https://github.com/aaronmck/CapSeg</a></td></tr><tr><td>ABSOLUTE</td><td><div class="dropBlock reference-citations"><a href="#bib21" id="back-bib21" data-db-target-for="bib21-d13420944e3311" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib21">Carter et al., 2012</a><div data-db-target-of="bib21-d13420944e3311" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Carter S.L. </li><li class="loa__item">Cibulskis K. </li><li class="loa__item">Helman E. </li><li class="loa__item">McKenna A. </li><li class="loa__item">Shen H. </li><li class="loa__item">Zack T. </li><li class="loa__item">Laird P.W. </li><li class="loa__item">Onofrio R.C. </li><li class="loa__item">Winckler W. </li><li class="loa__item">Weir B.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Absolute quantification of somatic DNA alterations in human cancer.<div class="popup-ref"><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">413-421</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnbt.2203&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=22544022&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84860782006&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1036)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2012&pages=413-421&journal=Nat.+Biotechnol.&author=Carter+S.L.&author=Cibulskis+K.&author=Helman+E.&author=McKenna+A.&author=Shen+H.&author=Zack+T.&author=Laird+P.W.&author=Onofrio+R.C.&author=Winckler+W.&author=Weir+B.A.&title=Absolute+quantification+of+somatic+DNA+alterations+in+human+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://software.broadinstitute.org/cancer/cga/absolute" target="_blank">https://software.broadinstitute.org/cancer/cga/absolute</a></td></tr><tr><td>deTiN</td><td><div class="dropBlock reference-citations"><a href="#bib126" id="back-bib126" data-db-target-for="bib126-d13420944e3320" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib126">Taylor-Weiner et al., 2018</a><div data-db-target-of="bib126-d13420944e3320" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Taylor-Weiner A. </li><li class="loa__item">Stewart C. </li><li class="loa__item">Giordano T. </li><li class="loa__item">Miller M. </li><li class="loa__item">Rosenberg M. </li><li class="loa__item">Macbeth A. </li><li class="loa__item">Lennon N. </li><li class="loa__item">Rheinbay E. </li><li class="loa__item">Landau D.-A. </li><li class="loa__item">Wu C.J. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">DeTiN: overcoming tumor-in-normal contamination.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">531-534</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41592-018-0036-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29941871&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85048968644&amp;cf=" onclick="newWindow(this.href);return false">Scopus (20)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=531-534&journal=Nat.+Methods&author=Taylor-Weiner+A.&author=Stewart+C.&author=Giordano+T.&author=Miller+M.&author=Rosenberg+M.&author=Macbeth+A.&author=Lennon+N.&author=Rheinbay+E.&author=Landau+D.-A.&author=Wu+C.J.&author=Getz+G.&title=DeTiN%3A+overcoming+tumor-in-normal+contamination">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://github.com/getzlab/deTiN" target="_blank">https://github.com/getzlab/deTiN</a></td></tr><tr><td>GATK4</td><td><div class="dropBlock reference-citations"><a href="#bib81" id="back-bib81" data-db-target-for="bib81-d13420944e3329" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib81">McKenna et al., 2010</a><div data-db-target-of="bib81-d13420944e3329" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">McKenna A. </li><li class="loa__item">Hanna M. </li><li class="loa__item">Banks E. </li><li class="loa__item">Sivachenko A. </li><li class="loa__item">Cibulskis K. </li><li class="loa__item">Kernytsky A. </li><li class="loa__item">Garimella K. </li><li class="loa__item">Altshuler D. </li><li class="loa__item">Gabriel S. </li><li class="loa__item">Daly M. </li><li class="loa__item">DePristo M.A. </li></ul><div class="reference-citations__detials">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">1297-1303</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1101%2Fgr.107524.110&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=20644199&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-77956295988&amp;cf=" onclick="newWindow(this.href);return false">Scopus (12104)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=1297-1303&journal=Genome+Res.&author=McKenna+A.&author=Hanna+M.&author=Banks+E.&author=Sivachenko+A.&author=Cibulskis+K.&author=Kernytsky+A.&author=Garimella+K.&author=Altshuler+D.&author=Gabriel+S.&author=Daly+M.&author=DePristo+M.A.&title=The+Genome+Analysis+Toolkit%3A+a+MapReduce+framework+for+analyzing+next-generation+DNA+sequencing+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://gatk.broadinstitute.org/hc/en-us" target="_blank">https://gatk.broadinstitute.org/hc/en-us</a></td></tr><tr><td>Oncotator</td><td><div class="dropBlock reference-citations"><a href="#bib104" id="back-bib104" data-db-target-for="bib104-d13420944e3338" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib104">Ramos et al., 2015</a><div data-db-target-of="bib104-d13420944e3338" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ramos A.H. </li><li class="loa__item">Lichtenstein L. </li><li class="loa__item">Gupta M. </li><li class="loa__item">Lawrence M.S. </li><li class="loa__item">Pugh T.J. </li><li class="loa__item">Saksena G. </li><li class="loa__item">Meyerson M. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">Oncotator: cancer variant annotation tool.<div class="popup-ref"><span class="ref__series"><i>Hum. Mutat.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">36</span></span>: <span class="ref__seriesPages">E2423-E2429</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1002%2Fhumu.22771&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=25703262&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84925965197&amp;cf=" onclick="newWindow(this.href);return false">Scopus (283)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=E2423-E2429&journal=Hum.+Mutat.&author=Ramos+A.H.&author=Lichtenstein+L.&author=Gupta+M.&author=Lawrence+M.S.&author=Pugh+T.J.&author=Saksena+G.&author=Meyerson+M.&author=Getz+G.&title=Oncotator%3A+cancer+variant+annotation+tool">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://software.broadinstitute.org/cancer/cga/oncotator" target="_blank">https://software.broadinstitute.org/cancer/cga/oncotator</a></td></tr><tr><td>The Ensembl Variant Effect Predictor</td><td><div class="dropBlock reference-citations"><a href="#bib82" id="back-bib82" data-db-target-for="bib82-d13420944e3347" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib82">McLaren et al., 2016</a><div data-db-target-of="bib82-d13420944e3347" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">McLaren W. </li><li class="loa__item">Gil L. </li><li class="loa__item">Hunt S.E. </li><li class="loa__item">Riat H.S. </li><li class="loa__item">Ritchie G.R.S. </li><li class="loa__item">Thormann A. </li><li class="loa__item">Flicek P. </li><li class="loa__item">Cunningham F. </li></ul><div class="reference-citations__detials">The Ensembl Variant Effect Predictor.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">122</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2Fs13059-016-0974-4&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27268795&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84973338712&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1927)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=122&journal=Genome+Biol.&author=McLaren+W.&author=Gil+L.&author=Hunt+S.E.&author=Riat+H.S.&author=Ritchie+G.R.S.&author=Thormann+A.&author=Flicek+P.&author=Cunningham+F.&title=The+Ensembl+Variant+Effect+Predictor">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://useast.ensembl.org//uswest.ensembl.org/info/docs/tools/vep/index.html?redirectsrc=//useast.ensembl.org%2Finfo%2Fdocs%2Ftools%2Fvep%2Findex.html" target="_blank">http://useast.ensembl.org//uswest.ensembl.org/info/docs/tools/vep/index.html?redirectsrc=//useast.ensembl.org%2Finfo%2Fdocs%2Ftools%2Fvep%2Findex.html</a></td></tr><tr><td>HaplotypeCaller</td><td><div class="dropBlock reference-citations"><a href="#bib102" id="back-bib102" data-db-target-for="bib102-d13420944e3356" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib102">Poplin et al., 2017</a><div data-db-target-of="bib102-d13420944e3356" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Poplin R. </li><li class="loa__item">Ruano-Rubio V. </li><li class="loa__item">DePristo M.A. </li><li class="loa__item">Fennell T.J. </li><li class="loa__item">Carneiro M.O. </li><li class="loa__item">Van der Auwera G.A. </li><li class="loa__item">Kling D.E. </li><li class="loa__item">Gauthier L.D. </li><li class="loa__item">Levy-Moonshine A. </li><li class="loa__item">Roazen D. </li><li>et al.</li></ul><div class="reference-citations__detials">Scaling accurate genetic variant discovery to tens of thousands of samples.<div class="popup-ref"><span class="ref__series"><i>bioRxiv.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1101/201178">https://doi.org/10.1101/201178</a></span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1101%2F201178&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85094275866&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&journal=bioRxiv&author=Poplin+R.&author=Ruano-Rubio+V.&author=DePristo+M.A.&author=Fennell+T.J.&author=Carneiro+M.O.&author=Van+der+Auwera+G.A.&author=Kling+D.E.&author=Gauthier+L.D.&author=Levy-Moonshine+A.&author=Roazen+D.&title=Scaling+accurate+genetic+variant+discovery+to+tens+of+thousands+of+samples">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://gatk.broadinstitute.org/hc/en-us/articles/360037225632-HaplotypeCaller" target="_blank">https://gatk.broadinstitute.org/hc/en-us/articles/360037225632-HaplotypeCaller</a></td></tr><tr><td>Cufflinks</td><td><div class="dropBlock reference-citations"><a href="#bib129" id="back-bib129" data-db-target-for="bib129-d13420944e3365" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib129">Trapnell et al., 2010</a><div data-db-target-of="bib129-d13420944e3365" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Trapnell C. </li><li class="loa__item">Williams B.A. </li><li class="loa__item">Pertea G. </li><li class="loa__item">Mortazavi A. </li><li class="loa__item">Kwan G. </li><li class="loa__item">van Baren M.J. </li><li class="loa__item">Salzberg S.L. </li><li class="loa__item">Wold B.J. </li><li class="loa__item">Pachter L. </li></ul><div class="reference-citations__detials">Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation.<div class="popup-ref"><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">511-515</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnbt.1621&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=20436464&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-77952123055&amp;cf=" onclick="newWindow(this.href);return false">Scopus (9185)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2010&pages=511-515&journal=Nat.+Biotechnol.&author=Trapnell+C.&author=Williams+B.A.&author=Pertea+G.&author=Mortazavi+A.&author=Kwan+G.&author=van+Baren+M.J.&author=Salzberg+S.L.&author=Wold+B.J.&author=Pachter+L.&title=Transcript+assembly+and+quantification+by+RNA-Seq+reveals+unannotated+transcripts+and+isoform+switching+during+cell+differentiation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://cole-trapnell-lab.github.io/cufflinks/" target="_blank">http://cole-trapnell-lab.github.io/cufflinks/</a></td></tr><tr><td>GISTIC2.0</td><td><div class="dropBlock reference-citations"><a href="#bib83" id="back-bib83" data-db-target-for="bib83-d13420944e3374" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib83">Mermel et al., 2011</a><div data-db-target-of="bib83-d13420944e3374" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mermel C.H. </li><li class="loa__item">Schumacher S.E. </li><li class="loa__item">Hill B. </li><li class="loa__item">Meyerson M.L. </li><li class="loa__item">Beroukhim R. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R41</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2Fgb-2011-12-4-r41&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=21527027&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-79955166265&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1334)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R41&journal=Genome+Biol.&author=Mermel+C.H.&author=Schumacher+S.E.&author=Hill+B.&author=Meyerson+M.L.&author=Beroukhim+R.&author=Getz+G.&title=GISTIC2.0+facilitates+sensitive+and+confident+localization+of+the+targets+of+focal+somatic+copy-number+alteration+in+human+cancers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://portals.broadinstitute.org/cgi-bin/cancer/publications/pub_paper.cgi?mode=view&amp;paper_id=216&amp;p=t" target="_blank">http://portals.broadinstitute.org/cgi-bin/cancer/publications/pub_paper.cgi?mode=view&amp;paper_id=216&amp;p=t</a></td></tr><tr><td>MutSig2CV</td><td><div class="dropBlock reference-citations"><a href="#bib64" id="back-bib64" data-db-target-for="bib64-d13420944e3384" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib64">Lawrence et al., 2014</a><div data-db-target-of="bib64-d13420944e3384" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lawrence M.S. </li><li class="loa__item">Stojanov P. </li><li class="loa__item">Mermel C.H. </li><li class="loa__item">Robinson J.T. </li><li class="loa__item">Garraway L.A. </li><li class="loa__item">Golub T.R. </li><li class="loa__item">Meyerson M. </li><li class="loa__item">Gabriel S.B. </li><li class="loa__item">Lander E.S. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">Discovery and saturation analysis of cancer genes across 21 tumour types.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">505</span></span>: <span class="ref__seriesPages">495-501</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature12912&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=24390350&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84892833777&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1826)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=505&publication_year=2014&pages=495-501&journal=Nature&author=Lawrence+M.S.&author=Stojanov+P.&author=Mermel+C.H.&author=Robinson+J.T.&author=Garraway+L.A.&author=Golub+T.R.&author=Meyerson+M.&author=Gabriel+S.B.&author=Lander+E.S.&author=Getz+G.&title=Discovery+and+saturation+analysis+of+cancer+genes+across+21+tumour+types">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://software.broadinstitute.org/cancer/cga/msp" target="_blank">https://software.broadinstitute.org/cancer/cga/msp</a></td></tr><tr><td>SignatureAnalyzer</td><td><div class="dropBlock reference-citations"><a href="#bib60" id="back-bib60" data-db-target-for="bib60-d13420944e3393" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib60">Kim et al., 2016</a><div data-db-target-of="bib60-d13420944e3393" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kim J. </li><li class="loa__item">Mouw K.W. </li><li class="loa__item">Polak P. </li><li class="loa__item">Braunstein L.Z. </li><li class="loa__item">Kamburov A. </li><li class="loa__item">Kwiatkowski D.J. </li><li class="loa__item">Rosenberg J.E. </li><li class="loa__item">Van Allen E.M. </li><li class="loa__item">D’Andrea A. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">Somatic ERCC2 mutations are associated with a distinct genomic signature in urothelial tumors.<div class="popup-ref"><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">48</span></span>: <span class="ref__seriesPages">600-606</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fng.3557&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27111033&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84964308124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (189)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2016&pages=600-606&journal=Nat.+Genet.&author=Kim+J.&author=Mouw+K.W.&author=Polak+P.&author=Braunstein+L.Z.&author=Kamburov+A.&author=Kwiatkowski+D.J.&author=Rosenberg+J.E.&author=Van+Allen+E.M.&author=D%E2%80%99Andrea+A.&author=Getz+G.&title=Somatic+ERCC2+mutations+are+associated+with+a+distinct+genomic+signature+in+urothelial+tumors">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://software.broadinstitute.org/cancer/cga/" target="_blank">https://software.broadinstitute.org/cancer/cga/</a></td></tr><tr><td>COSMIC</td><td><div class="dropBlock reference-citations"><a href="#bib125" id="back-bib125" data-db-target-for="bib125-d13420944e3402" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib125">Tate et al., 2019</a><div data-db-target-of="bib125-d13420944e3402" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Tate J.G. </li><li class="loa__item">Bamford S. </li><li class="loa__item">Jubb H.C. </li><li class="loa__item">Sondka Z. </li><li class="loa__item">Beare D.M. </li><li class="loa__item">Bindal N. </li><li class="loa__item">Boutselakis H. </li><li class="loa__item">Cole C.G. </li><li class="loa__item">Creatore C. </li><li class="loa__item">Dawson E. </li><li>et al.</li></ul><div class="reference-citations__detials">COSMIC: the Catalogue Of Somatic Mutations In Cancer.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">47</span></span>: <span class="ref__seriesPages">D941-D947</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgky1015&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30371878&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85057093998&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1040)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D941-D947&journal=Nucleic+Acids+Res.&author=Tate+J.G.&author=Bamford+S.&author=Jubb+H.C.&author=Sondka+Z.&author=Beare+D.M.&author=Bindal+N.&author=Boutselakis+H.&author=Cole+C.G.&author=Creatore+C.&author=Dawson+E.&title=COSMIC%3A+the+Catalogue+Of+Somatic+Mutations+In+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://cancer.sanger.ac.uk/cosmic" target="_blank">https://cancer.sanger.ac.uk/cosmic</a></td></tr><tr><td>deconstructSigs (R-package)</td><td><div class="dropBlock reference-citations"><a href="#bib108" id="back-bib108" data-db-target-for="bib108-d13420944e3411" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib108">Rosenthal et al., 2016</a><div data-db-target-of="bib108-d13420944e3411" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Rosenthal R. </li><li class="loa__item">McGranahan N. </li><li class="loa__item">Herrero J. </li><li class="loa__item">Taylor B.S. </li><li class="loa__item">Swanton C. </li></ul><div class="reference-citations__detials">DeconstructSigs: delineating mutational processes in single tumors distinguishes DNA repair deficiencies and patterns of carcinoma evolution.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">31</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2Fs13059-016-0893-4&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26899170&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84959467752&amp;cf=" onclick="newWindow(this.href);return false">Scopus (462)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=31&journal=Genome+Biol.&author=Rosenthal+R.&author=McGranahan+N.&author=Herrero+J.&author=Taylor+B.S.&author=Swanton+C.&title=DeconstructSigs%3A+delineating+mutational+processes+in+single+tumors+distinguishes+DNA+repair+deficiencies+and+patterns+of+carcinoma+evolution">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://cran.r-project.org/web/packages/deconstructSigs/" target="_blank">https://cran.r-project.org/web/packages/deconstructSigs/</a></td></tr><tr><td>Maftool (R-package)</td><td><div class="dropBlock reference-citations"><a href="#bib80" id="back-bib80" data-db-target-for="bib80-d13420944e3420" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib80">Mayakonda et al., 2018</a><div data-db-target-of="bib80-d13420944e3420" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mayakonda A. </li><li class="loa__item">Lin D.-C. </li><li class="loa__item">Assenov Y. </li><li class="loa__item">Plass C. </li><li class="loa__item">Koeffler H.P. </li></ul><div class="reference-citations__detials">Maftools: efficient and comprehensive analysis of somatic variants in cancer.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">1747-1756</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1101%2Fgr.239244.118&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30341162&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85055908064&amp;cf=" onclick="newWindow(this.href);return false">Scopus (519)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=1747-1756&journal=Genome+Res.&author=Mayakonda+A.&author=Lin+D.-C.&author=Assenov+Y.&author=Plass+C.&author=Koeffler+H.P.&title=Maftools%3A+efficient+and+comprehensive+analysis+of+somatic+variants+in+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://bioconductor.org/packages/release/bioc/html/maftools.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/maftools.html</a></td></tr><tr><td>Spectrum Mill software package v7.0</td><td>Agilent Technologies, Santa Clara, CA</td><td><a class="inter-ref-link" href="https://proteomics.broadinstitute.org/" target="_blank">https://proteomics.broadinstitute.org/</a></td></tr><tr><td>CMap</td><td><div class="dropBlock reference-citations"><a href="#bib63" id="back-bib63" data-db-target-for="bib63-d13420944e3437" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib63">Lamb et al., 2006</a><div data-db-target-of="bib63-d13420944e3437" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lamb J. </li><li class="loa__item">Crawford E.D. </li><li class="loa__item">Peck D. </li><li class="loa__item">Modell J.W. </li><li class="loa__item">Blat I.C. </li><li class="loa__item">Wrobel M.J. </li><li class="loa__item">Lerner J. </li><li class="loa__item">Brunet J.-P. </li><li class="loa__item">Subramanian A. </li><li class="loa__item">Ross K.N. </li><li>et al.</li></ul><div class="reference-citations__detials">The Connectivity Map: using gene-expression signatures to connect small molecules, genes, and disease.<div class="popup-ref"><span class="ref__series"><i>Science.</i> <span class="ref__seriesDate">2006; </span><span class="ref__seriesVolume">313</span></span>: <span class="ref__seriesPages">1929-1935</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1126%2Fscience.1132939&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=17008526&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-33749335282&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3071)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=313&publication_year=2006&pages=1929-1935&journal=Science&author=Lamb+J.&author=Crawford+E.D.&author=Peck+D.&author=Modell+J.W.&author=Blat+I.C.&author=Wrobel+M.J.&author=Lerner+J.&author=Brunet+J.-P.&author=Subramanian+A.&author=Ross+K.N.&title=The+Connectivity+Map%3A+using+gene-expression+signatures+to+connect+small+molecules%2C+genes%2C+and+disease">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib122" id="back-bib122" data-db-target-for="bib122-d13420944e3440" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib122">Subramanian et al., 2017</a><div data-db-target-of="bib122-d13420944e3440" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Subramanian A. </li><li class="loa__item">Narayan R. </li><li class="loa__item">Corsello S.M. </li><li class="loa__item">Peck D.D. </li><li class="loa__item">Natoli T.E. </li><li class="loa__item">Lu X. </li><li class="loa__item">Gould J. </li><li class="loa__item">Davis J.F. </li><li class="loa__item">Tubelli A.A. </li><li class="loa__item">Asiedu J.K. </li><li>et al.</li></ul><div class="reference-citations__detials">A Next Generation Connectivity Map: L1000 Platform and the First 1,000,000 Profiles.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">171</span></span>: <span class="ref__seriesPages">1437-1452.e17</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29195078&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85036572148&amp;cf=" onclick="newWindow(this.href);return false">Scopus (772)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2017&pages=1437-1452.e17&journal=Cell&author=Subramanian+A.&author=Narayan+R.&author=Corsello+S.M.&author=Peck+D.D.&author=Natoli+T.E.&author=Lu+X.&author=Gould+J.&author=Davis+J.F.&author=Tubelli+A.A.&author=Asiedu+J.K.&title=A+Next+Generation+Connectivity+Map%3A+L1000+Platform+and+the+First+1%2C000%2C000+Profiles">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://clue.io/cmap" target="_blank">https://clue.io/cmap</a></td></tr><tr><td>QUILTS v3.0</td><td><div class="dropBlock reference-citations"><a href="#bib110" id="back-bib110" data-db-target-for="bib110-d13420944e3449" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib110">Ruggles et al., 2015</a><div data-db-target-of="bib110-d13420944e3449" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Tang Z. </li><li class="loa__item">Wang X. </li><li class="loa__item">Grover H. </li><li class="loa__item">Askenazi M. </li><li class="loa__item">Teubl J. </li><li class="loa__item">Cao S. </li><li class="loa__item">McLellan M.D. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Tabb D.L. </li><li>et al.</li></ul><div class="reference-citations__detials">An analysis of the sensitivity of proteogenomic mapping of somatic mutations and novel splicing events in cancer.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">1060-1071</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.M115.056226&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.M115.056226&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.M115.056226&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26631509&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84962585103&amp;cf=" onclick="newWindow(this.href);return false">Scopus (56)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=1060-1071&journal=Mol.+Cell.+Proteomics&author=Ruggles+K.V.&author=Tang+Z.&author=Wang+X.&author=Grover+H.&author=Askenazi+M.&author=Teubl+J.&author=Cao+S.&author=McLellan+M.D.&author=Clauser+K.R.&author=Tabb+D.L.&title=An+analysis+of+the+sensitivity+of+proteogenomic+mapping+of+somatic+mutations+and+novel+splicing+events+in+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://openslice.fenyolab.org/cgi-bin/pyquilts_cgi.pl" target="_blank">http://openslice.fenyolab.org/cgi-bin/pyquilts_cgi.pl</a></td></tr><tr><td>PTM-SEA</td><td><div class="dropBlock reference-citations"><a href="#bib62" id="back-bib62" data-db-target-for="bib62-d13420944e3458" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib62">Krug et al., 2019</a><div data-db-target-of="bib62-d13420944e3458" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Krug K. </li><li class="loa__item">Mertins P. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Hornbeck P. </li><li class="loa__item">Raju R. </li><li class="loa__item">Ahmad R. </li><li class="loa__item">Szucs M. </li><li class="loa__item">Mundt F. </li><li class="loa__item">Forestier D. </li><li class="loa__item">Jane-Valbuena J. </li><li>et al.</li></ul><div class="reference-citations__detials">A Curated Resource for Phosphosite-specific Signature Analysis.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">576-593</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30563849&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85062992827&amp;cf=" onclick="newWindow(this.href);return false">Scopus (42)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=576-593&journal=Mol.+Cell.+Proteomics&author=Krug+K.&author=Mertins+P.&author=Zhang+B.&author=Hornbeck+P.&author=Raju+R.&author=Ahmad+R.&author=Szucs+M.&author=Mundt+F.&author=Forestier+D.&author=Jane-Valbuena+J.&title=A+Curated+Resource+for+Phosphosite-specific+Signature+Analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://github.com/broadinstitute/ssGSEA2.0" target="_blank">https://github.com/broadinstitute/ssGSEA2.0</a></td></tr><tr><td>Protigy</td><td>Broad Institute, Proteomics Platform</td><td><a class="inter-ref-link" href="https://github.com/broadinstitute/protigy" target="_blank">https://github.com/broadinstitute/protigy</a></td></tr><tr><td>Reactome</td><td><div class="dropBlock reference-citations"><a href="#bib40" id="back-bib40" data-db-target-for="bib40-d13420944e3475" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib40">Fabregat et al., 2018</a><div data-db-target-of="bib40-d13420944e3475" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Fabregat A. </li><li class="loa__item">Jupe S. </li><li class="loa__item">Matthews L. </li><li class="loa__item">Sidiropoulos K. </li><li class="loa__item">Gillespie M. </li><li class="loa__item">Garapati P. </li><li class="loa__item">Haw R. </li><li class="loa__item">Jassal B. </li><li class="loa__item">Korninger F. </li><li class="loa__item">May B. </li><li>et al.</li></ul><div class="reference-citations__detials">The Reactome Pathway Knowledgebase.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">46</span></span>: <span class="ref__seriesPages">D649-D655</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgkx1132&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29145629&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85040929878&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1166)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D649-D655&journal=Nucleic+Acids+Res.&author=Fabregat+A.&author=Jupe+S.&author=Matthews+L.&author=Sidiropoulos+K.&author=Gillespie+M.&author=Garapati+P.&author=Haw+R.&author=Jassal+B.&author=Korninger+F.&author=May+B.&title=The+Reactome+Pathway+Knowledgebase">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://reactome.org/" target="_blank">https://reactome.org/</a></td></tr><tr><td>COMPARTMENTS</td><td><div class="dropBlock reference-citations"><a href="#bib15" id="back-bib15" data-db-target-for="bib15-d13420944e3485" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib15">Binder et al., 2014</a><div data-db-target-of="bib15-d13420944e3485" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Binder J.X. </li><li class="loa__item">Pletscher-Frankild S. </li><li class="loa__item">Tsafou K. </li><li class="loa__item">Stolte C. </li><li class="loa__item">O’Donoghue S.I. </li><li class="loa__item">Schneider R. </li><li class="loa__item">Jensen L.J. </li></ul><div class="reference-citations__detials">COMPARTMENTS: unification and visualization of protein subcellular localization evidence.<div class="popup-ref"><span class="ref__series"><i>Database (Oxford).</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">2014</span></span>: <span class="ref__seriesPages">bau012</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fdatabase%2Fbau012&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=24573882&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84901788541&amp;cf=" onclick="newWindow(this.href);return false">Scopus (250)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2014&publication_year=2014&pages=bau012&journal=Database+%28Oxford%29&author=Binder+J.X.&author=Pletscher-Frankild+S.&author=Tsafou+K.&author=Stolte+C.&author=O%E2%80%99Donoghue+S.I.&author=Schneider+R.&author=Jensen+L.J.&title=COMPARTMENTS%3A+unification+and+visualization+of+protein+subcellular+localization+evidence">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://compartments.jensenlab.org/Search" target="_blank">https://compartments.jensenlab.org/Search</a></td></tr><tr><td>Blacksheep</td><td>Ruggles/Fenyo lab</td><td><a class="inter-ref-link" href="https://www.biorxiv.org/content/10.1101/825067v2" target="_blank">https://www.biorxiv.org/content/10.1101/825067v2</a>, <a class="inter-ref-link" href="https://github.com/ruggleslab/blackSheep" target="_blank">https://github.com/ruggleslab/blackSheep</a>, <a class="inter-ref-link" href="https://github.com/ruggleslab/blackSheepr" target="_blank">https://github.com/ruggleslab/blackSheepr</a></td></tr><tr><td>NMF (R-package)</td><td><div class="dropBlock reference-citations"><a href="#bib42" id="back-bib42" data-db-target-for="bib42-d13420944e3508" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib42">Gaujoux and Seoighe, 2010</a><div data-db-target-of="bib42-d13420944e3508" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gaujoux R. </li><li class="loa__item">Seoighe C. </li></ul><div class="reference-citations__detials">A flexible R package for nonnegative matrix factorization.<div class="popup-ref"><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">367</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2F1471-2105-11-367&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=20598126&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-77954069195&amp;cf=" onclick="newWindow(this.href);return false">Scopus (528)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=367&journal=BMC+Bioinformatics&author=Gaujoux+R.&author=Seoighe+C.&title=A+flexible+R+package+for+nonnegative+matrix+factorization">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://cran.r-project.org/web/packages/NMF/index.html" target="_blank">https://cran.r-project.org/web/packages/NMF/index.html</a></td></tr><tr><td>TCGAbiolinks</td><td><div class="dropBlock reference-citations"><a href="#bib30" id="back-bib30" data-db-target-for="bib30-d13420944e3517" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib30">Colaprico et al., 2016</a><div data-db-target-of="bib30-d13420944e3517" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Colaprico A. </li><li class="loa__item">Silva T.C. </li><li class="loa__item">Olsen C. </li><li class="loa__item">Garofano L. </li><li class="loa__item">Cava C. </li><li class="loa__item">Garolini D. </li><li class="loa__item">Sabedot T.S. </li><li class="loa__item">Malta T.M. </li><li class="loa__item">Pagnotta S.M. </li><li class="loa__item">Castiglioni I. </li><li>et al.</li></ul><div class="reference-citations__detials">TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">e71</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgkv1507&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26704973&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84966269471&amp;cf=" onclick="newWindow(this.href);return false">Scopus (839)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=e71&journal=Nucleic+Acids+Res.&author=Colaprico+A.&author=Silva+T.C.&author=Olsen+C.&author=Garofano+L.&author=Cava+C.&author=Garolini+D.&author=Sabedot+T.S.&author=Malta+T.M.&author=Pagnotta+S.M.&author=Castiglioni+I.&title=TCGAbiolinks%3A+an+R%2FBioconductor+package+for+integrative+analysis+of+TCGA+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://bioconductor.org/packages/release/bioc/html/TCGAbiolinks.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/TCGAbiolinks.html</a></td></tr><tr><td>LinkedOmics</td><td><div class="dropBlock reference-citations"><a href="#bib134" id="back-bib134" data-db-target-for="bib134-d13420944e3526" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib134">Vasaikar et al., 2018</a><div data-db-target-of="bib134-d13420944e3526" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Vasaikar S.V. </li><li class="loa__item">Straub P. </li><li class="loa__item">Wang J. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">LinkedOmics: analyzing multi-omics data within and across 32 cancer types.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">46</span></span>: <span class="ref__seriesPages">D956-D963</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgkx1090&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29136207&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85040936922&amp;cf=" onclick="newWindow(this.href);return false">Scopus (521)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D956-D963&journal=Nucleic+Acids+Res.&author=Vasaikar+S.V.&author=Straub+P.&author=Wang+J.&author=Zhang+B.&title=LinkedOmics%3A+analyzing+multi-omics+data+within+and+across+32+cancer+types">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://www.linkedomics.orglogin.php" target="_blank">http://www.linkedomics.orglogin.php</a></td></tr><tr><td>WebGestalt</td><td><div class="dropBlock reference-citations"><a href="#bib71" id="back-bib71" data-db-target-for="bib71-d13420944e3535" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib71">Liao et al., 2019</a><div data-db-target-of="bib71-d13420944e3535" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liao Y. </li><li class="loa__item">Wang J. </li><li class="loa__item">Jaehnig E.J. </li><li class="loa__item">Shi Z. </li></ul><div class="reference-citations__detials">WebGestalt 2019: gene set analysis toolkit with revamped UIs and APIs.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">47</span></span>: <span class="ref__seriesPages">W199-W205</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgkz401&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31114916&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85069234777&amp;cf=" onclick="newWindow(this.href);return false">Scopus (567)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=W199-W205&journal=Nucleic+Acids+Res.&author=Liao+Y.&author=Wang+J.&author=Jaehnig+E.J.&author=Shi+Z.&title=WebGestalt+2019%3A+gene+set+analysis+toolkit+with+revamped+UIs+and+APIs">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://www.webgestalt.org/" target="_blank">http://www.webgestalt.org/</a></td></tr><tr><td>MoonlightR</td><td><div class="dropBlock reference-citations"><a href="#bib31" id="back-bib31" data-db-target-for="bib31-d13420944e3544" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib31">Colaprico et al., 2020</a><div data-db-target-of="bib31-d13420944e3544" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Colaprico A. </li><li class="loa__item">Olsen C. </li><li class="loa__item">Bailey M.H. </li><li class="loa__item">Odom G.J. </li><li class="loa__item">Terkelsen T. </li><li class="loa__item">Silva T.C. </li><li class="loa__item">Olsen A.V. </li><li class="loa__item">Cantini L. </li><li class="loa__item">Zinovyev A. </li><li class="loa__item">Barillot E. </li><li>et al.</li></ul><div class="reference-citations__detials">Interpreting pathways to discover cancer driver genes with Moonlight.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">69</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41467-019-13803-0&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31900418&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85077479923&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=69&journal=Nat.+Commun.&author=Colaprico+A.&author=Olsen+C.&author=Bailey+M.H.&author=Odom+G.J.&author=Terkelsen+T.&author=Silva+T.C.&author=Olsen+A.V.&author=Cantini+L.&author=Zinovyev+A.&author=Barillot+E.&title=Interpreting+pathways+to+discover+cancer+driver+genes+with+Moonlight">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://bioconductor.org/packages/release/bioc/html/MoonlightR.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/MoonlightR.html</a></td></tr><tr><td>Cibersort</td><td><div class="dropBlock reference-citations"><a href="#bib92" id="back-bib92" data-db-target-for="bib92-d13420944e3553" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib92">Newman et al., 2015</a><div data-db-target-of="bib92-d13420944e3553" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Newman A.M. </li><li class="loa__item">Liu C.L. </li><li class="loa__item">Green M.R. </li><li class="loa__item">Gentles A.J. </li><li class="loa__item">Feng W. </li><li class="loa__item">Xu Y. </li><li class="loa__item">Hoang C.D. </li><li class="loa__item">Diehn M. </li><li class="loa__item">Alizadeh A.A. </li></ul><div class="reference-citations__detials">Robust enumeration of cell subsets from tissue expression profiles.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">453-457</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnmeth.3337&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=25822800&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84928927858&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2633)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=453-457&journal=Nat.+Methods&author=Newman+A.M.&author=Liu+C.L.&author=Green+M.R.&author=Gentles+A.J.&author=Feng+W.&author=Xu+Y.&author=Hoang+C.D.&author=Diehn+M.&author=Alizadeh+A.A.&title=Robust+enumeration+of+cell+subsets+from+tissue+expression+profiles">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://cibersort.stanford.edu/" target="_blank">https://cibersort.stanford.edu/</a></td></tr><tr><td>xCell</td><td><div class="dropBlock reference-citations"><a href="#bib7" id="back-bib7" data-db-target-for="bib7-d13420944e3562" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib7">Aran et al., 2017</a><div data-db-target-of="bib7-d13420944e3562" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Aran D. </li><li class="loa__item">Hu Z. </li><li class="loa__item">Butte A.J. </li></ul><div class="reference-citations__detials">xCell: digitally portraying the tissue cellular heterogeneity landscape.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">220</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2Fs13059-017-1349-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29141660&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85034614124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (627)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=220&journal=Genome+Biol.&author=Aran+D.&author=Hu+Z.&author=Butte+A.J.&title=xCell%3A+digitally+portraying+the+tissue+cellular+heterogeneity+landscape">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://xcell.ucsf.edu/" target="_blank">https://xcell.ucsf.edu/</a></td></tr><tr><td>ESTIMATE (R-package)</td><td><div class="dropBlock reference-citations"><a href="#bib146" id="back-bib146" data-db-target-for="bib146-d13420944e3571" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib146">Yoshihara et al., 2013</a><div data-db-target-of="bib146-d13420944e3571" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Yoshihara K. </li><li class="loa__item">Shahmoradgoli M. </li><li class="loa__item">Martínez E. </li><li class="loa__item">Vegesna R. </li><li class="loa__item">Kim H. </li><li class="loa__item">Torres-Garcia W. </li><li class="loa__item">Treviño V. </li><li class="loa__item">Shen H. </li><li class="loa__item">Laird P.W. </li><li class="loa__item">Levine D.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Inferring tumour purity and stromal and immune cell admixture from expression data.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">2612</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fncomms3612&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=24113773&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84885673911&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1675)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2612&journal=Nat.+Commun.&author=Yoshihara+K.&author=Shahmoradgoli+M.&author=Mart%C3%ADnez+E.&author=Vegesna+R.&author=Kim+H.&author=Torres-Garcia+W.&author=Trevi%C3%B1o+V.&author=Shen+H.&author=Laird+P.W.&author=Levine+D.A.&title=Inferring+tumour+purity+and+stromal+and+immune+cell+admixture+from+expression+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://bioinformatics.mdanderson.org/estimate/rpackage.html" target="_blank">https://bioinformatics.mdanderson.org/estimate/rpackage.html</a></td></tr></tbody></table><div class="inline-table__tail"></div><ul class="rlist--inline separator download-links"><li><a title="Open Full Table" class="table__open__ctrl" target="_blank" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2820%2931400-8">
                            Open table in a new tab
                        </a></li></ul></div></div></section><section id="sec4.2" data-locator="sec4.2" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Resource Availability</span></h3><section id="sec4.2.1" data-locator="sec4.2.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Lead Contact</span></h4><div class="section-paragraph">This study did not generate new unique reagents. Further information and requests should be directed to and will be fulfilled by the lead author, Michael A. Gillette (<a class="inter-ref-link" href="/cdn-cgi/l/email-protection#d9beb0b5b5bcadadbc99bbabb6b8bdb0b7aaadb0adacadbcf7b6abbe" target="_blank"><span class="__cf_email__" data-cfemail="096e6065656c7d7d6c496b7b66686d60677a7d607d7c7d6c27667b6e">[email&#160;protected]</span></a>).</div></section><section id="sec4.2.2" data-locator="sec4.2.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Materials Availability</span></h4><div class="section-paragraph">This study did not generate new unique reagents.</div></section><section id="sec4.2.3" data-locator="sec4.2.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Data and Code Availability</span></h4><div class="section-paragraph">Proteomics raw and characterized datasets are publicly available though the CPTAC data portal <a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/study-summary/S060" target="_blank">https://cptac-data-portal.georgetown.edu/study-summary/S060</a> and at the Proteomic Data Commons (<a class="inter-ref-link" href="https://pdc.cancer.gov/pdc/" target="_blank">https://pdc.cancer.gov/pdc/</a>). The accession number for the proteomic data at the CPTAC data portal is S060. The accession number for the proteomic data characterized by the Proteomic Data Commons is PDC: PDC000120. The proteomics raw data consists of 17 plexes. Plexes 1-13 and 16-17 are tumor-only plexes and 14-15 are normal adjacent tissue (NAT)-only plexes. Results reported in this study are solely based on tumor-only plexes.</div><div class="section-paragraph">Raw genomic data (WES, RNA-seq, miRNA-seq,) associated with this study (harmonized with the GRCh38 reference genome) has been released at the Genomic Data Commons (<a class="inter-ref-link" href="https://gdc.cancer.gov" target="_blank">https://gdc.cancer.gov</a>) and is accessible via the database of Genotypes and Phenotypes (dbGaP). The accession number for the raw genomics data (WES, RNA-seq, miRNA-seq,) reported in this paper is dbGaP: phs000892.</div><div class="section-paragraph">Sample annotation, processed and normalized data files are provided in <a href="/cms/10.1016/j.cell.2020.10.036/attachment/3dbb5a07-b92f-4ae5-9c88-f84c3dca95db/mmc1" class="scroll-into-link">Tables S1</a> and <a href="/cms/10.1016/j.cell.2020.10.036/attachment/a1bc3f3e-eeac-4605-a9ef-839abd334dcd/mmc2" class="scroll-into-link">S2</a>. In addition, all processed data matrices will be available at LinkedOmics (<div class="dropBlock reference-citations"><a href="#bib134" id="back-bib134" data-db-target-for="bib134-d13420944e3616" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib134">Vasaikar et al., 2018</a><div data-db-target-of="bib134-d13420944e3616" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Vasaikar S.V. </li><li class="loa__item">Straub P. </li><li class="loa__item">Wang J. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">LinkedOmics: analyzing multi-omics data within and across 32 cancer types.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">46</span></span>: <span class="ref__seriesPages">D956-D963</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgkx1090&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29136207&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85040936922&amp;cf=" onclick="newWindow(this.href);return false">Scopus (521)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D956-D963&journal=Nucleic+Acids+Res.&author=Vasaikar+S.V.&author=Straub+P.&author=Wang+J.&author=Zhang+B.&title=LinkedOmics%3A+analyzing+multi-omics+data+within+and+across+32+cancer+types">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) (<a class="inter-ref-link" href="http://www.linkedomics.orglogin.php" target="_blank">http://www.linkedomics.orglogin.php</a>) upon publication, where computational tools are available for further exploration of this dataset.</div><div class="section-paragraph">A website for interactive visualization of the multi-omics dataset is available at: <a class="inter-ref-link" href="http://prot-shiny-vm.broadinstitute.org:3838/CPTAC-BRCA2020" target="_blank">http://prot-shiny-vm.broadinstitute.org:3838/CPTAC-BRCA2020</a>. The heatmap depicts somatic copy number aberrations, mRNA, protein, phosphosite and acetylsite abundances across 122 tumors. Copy number alterations are relative to matched normal blood samples and are on log2(CNA)-1 scale. For other data types the heatmap depicts abundances relative to the common reference (proteomics) or the median abundance across all tumors (RNA-seq).</div><div class="section-paragraph">The entire workflow described under ‘<a href="#sec4.5.3.10" class="scroll-into-link">Multi-omics clustering</a>’ has been implemented as a module for Broad’s cloud platform Terra (<a class="inter-ref-link" href="https://app.terra.bio/" target="_blank">https://app.terra.bio/</a>). The docker containers encapsulating the source code and required R-packages for NMF clustering and ssGSEA have been submitted to Dockerhub (broadcptac/pgdac_mo_nmf:9, broadcptac/pgdac_ssgsea:5). The source code for ssGSEA and PTM-SEA is available on GitHub: <a class="inter-ref-link" href="https://github.com/broadinstitute/ssGSEA2.0" target="_blank">https://github.com/broadinstitute/ssGSEA2.0</a>.</div></section></section><section id="sec4.3" data-locator="sec4.3" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Experimental Model and Subject Details</span></h3><section id="sec4.3.1" data-locator="sec4.3.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Human subjects</span></h4><div class="section-paragraph">A total of 134 participants were included in the study. Histopathologically-defined adult breast tumors from newly diagnosed patients were considered for analysis. The cohort ranged in age from 30-95. Institutional review boards at tissue source sites reviewed protocols and consent documentation adhering to the Clinical Proteomic Tumor Analysis Consortium (CPTAC) guidelines.</div></section></section><section id="sec4.4" data-locator="sec4.4" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Method Details</span></h3><section id="sec4.4.1" data-locator="sec4.4.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Specimens and clinical data</span></h4><div class="section-paragraph">Tumor, adjacent normal, and blood samples were collected by several tissue source sites in strict accordance with the CPTAC2 breast procurement protocol (<a class="inter-ref-link" href="https://brd.nci.nih.gov/brd/sop/download-pdf/301" target="_blank">https://brd.nci.nih.gov/brd/sop/download-pdf/301</a>). All patients provided written informed consent. Inclusion criteria included newly diagnosed, untreated patients undergoing definitive surgery for breast cancer (stage IIA-IIIC) or undergoing core needle biopsy at the time of placement of a vascular access device prior to neoadjuvant therapy for breast cancer. Patients with more than one newly detected and independent breast masses were allowed. Cases with prior history of other malignancies within the past 12 months were excluded. Cases with any prior systemic chemotherapy, endocrine therapy or biological therapy for any cancer, or prior history of radiation therapy involving the breast such as mantle field radiation for Hodgkins Disease or radiotherapy for lung cancer, were excluded. Patients who were found to have a diagnosis other than invasive breast cancer as a result of the surgery were also excluded. Peripheral venous blood samples from each patient were collected prior to administration of anesthesia. Samples were qualified for the study if two or more tumor tissue core biopsies or surgical resection segments had a minimum mass of 200 mg and demonstrated greater than 60% tumor cell nuclei and less than 20% tumor necrosis on frozen tissue section review.</div><div class="section-paragraph">To ensure tissue suitability for phosphoprotein analysis, the tumor and normal adjacent tissue specimens were collected in less than 30 minutes total ischemic time from interruption of the vascular supply and embedded in optimal cutting temperature (OCT) compound for processing at a common CPTAC-2 specimen core resource center. Pathologically qualified cases underwent further molecular qualification for extraction and co-isolation of nucleic acids. Tissue segments that were qualified both for pathology and for molecular integrity were shipped to the proteomic characterization centers. DNA and RNA from the same tumor segment and DNA from germline blood were further aliquoted and quantified per protocol. DNA quality was confirmed using gel electrophoresis and Nanodrop methods. RNA quality was confirmed using Nanodrop and Agilent bioanalyzer. Sufficient yield, a good gel score and passing value of 7 or greater RNA Integrity Number (RIN) qualified the DNA and RNA, respectively, for sequencing. The analytes were then shipped to the sequencing center.</div><div class="section-paragraph">Patient history, procedural details, and other relevant clinical and diagnostic information were collected using case report forms. The corresponding clinical data were formatted and distributed through the CPTAC data coordinating center (<a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/study-summary/S039" target="_blank">https://cptac-data-portal.georgetown.edu/study-summary/S039</a>). One year follow-up forms captured updated histories after completion of the initial treatment regimen. Deidentified Pathology Reports (AJCC 7th edition 2013) including ER (estrogen receptor), PGR (progesterone receptor), and HER2 (ERBB2) status and representative diagnostic slide images were utilized to review and qualify cases for this study. Final clinical assessment of hormone receptor status by IHC and FISH classified tumors as follows: ER and/or PR positive: 83; HER2 positive: 13; ER/PR/HER2 negative: 16; ER/PR negative with equivocal or unknown HER2 status: 12. PAM50-based classification of tumors was also performed (<div class="dropBlock reference-citations"><a href="#bib96" id="back-bib96" data-db-target-for="bib96-d13420944e3664" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib96">Parker et al., 2009</a><div data-db-target-of="bib96-d13420944e3664" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Parker J.S. </li><li class="loa__item">Mullins M. </li><li class="loa__item">Cheang M.C.U. </li><li class="loa__item">Leung S. </li><li class="loa__item">Voduc D. </li><li class="loa__item">Vickery T. </li><li class="loa__item">Davies S. </li><li class="loa__item">Fauron C. </li><li class="loa__item">He X. </li><li class="loa__item">Hu Z. </li><li>et al.</li></ul><div class="reference-citations__detials">Supervised risk predictor of breast cancer based on intrinsic subtypes.<div class="popup-ref"><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">27</span></span>: <span class="ref__seriesPages">1160-1167</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1200%2FJCO.2008.18.1370&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=19204204&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-62449131093&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2602)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2009&pages=1160-1167&journal=J.%C2%A0Clin.+Oncol.&author=Parker+J.S.&author=Mullins+M.&author=Cheang+M.C.U.&author=Leung+S.&author=Voduc+D.&author=Vickery+T.&author=Davies+S.&author=Fauron+C.&author=He+X.&author=Hu+Z.&title=Supervised+risk+predictor+of+breast+cancer+based+on+intrinsic+subtypes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), confirming that available samples represented all major subtypes, including 14 Her2 enriched (HER2-E), 29 Basal, 57 Luminal A (LumA), 17 Luminal B (LumB) and 5 Normal-like tumors (<a href="/cms/10.1016/j.cell.2020.10.036/attachment/3dbb5a07-b92f-4ae5-9c88-f84c3dca95db/mmc1" class="scroll-into-link">Table S1</a>). To support ERBB2-focused analyses, additional, centralized HER2 IHC was performed on sections from 68 tumors for which remaining tissue was available. ERBB2 clinical status was defined using a combination of the updated ERBB2 IHC scores where available (original IHC scores from the pathology reports were used for samples for which additional tissue was not available) and ERBB2 fluorescence <em>in situ</em> hybridization (FISH) results from the pathology reports. Samples were classified in a manner consistent with ASCO-CAP guidelines (<a class="inter-ref-link" href="https://www.cap.org/" target="_blank">https://www.cap.org/</a>); specifically, clinical ERBB2 negative cases were those where the IHC score was 0 or 1+ or FISH was negative, clinical ERBB2 positive cases were those where the IHC score was 3+ and FISH was positive or not available or where the IHC score was 2+ and FISH was positive, and equivocal cases were those with IHC score of 2+ that lacked FISH confirmation or had a positive FISH result without IHC confirmation (ASCO guidelines require further testing for these equivocal cases, but this was not possible here). Central staining for ER was consistent with ER status from the pathology reports. Triple negative breast cancer (TNBC) status was classified using the clinical status for ER and PR from IHC and the ERBB2 proteogenomic (PG) status applied to all samples as described below. Samples that were positive for any of these markers were classified as TNBC negative whereas samples that were negative for all three were classified as TNBC positive (samples that were missing classification for ER or PR but negative for all other markers were classified as NA because clinical status of all 3 markers could not be assessed). The known propensity of TNBC to affect patients with African ancestry (<div class="dropBlock reference-citations"><a href="#bib36" id="back-bib36" data-db-target-for="bib36-d13420944e3676" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib36">Dietze et al., 2015</a><div data-db-target-of="bib36-d13420944e3676" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Dietze E.C. </li><li class="loa__item">Sistrunk C. </li><li class="loa__item">Miranda-Carboni G. </li><li class="loa__item">O’Regan R. </li><li class="loa__item">Seewaldt V.L. </li></ul><div class="reference-citations__detials">Triple-negative breast cancer in African-American women: disparities versus biology.<div class="popup-ref"><span class="ref__series"><i>Nat. Rev. Cancer.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">248-254</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnrc3896&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=25673085&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84925819377&amp;cf=" onclick="newWindow(this.href);return false">Scopus (194)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=248-254&journal=Nat.+Rev.+Cancer&author=Dietze+E.C.&author=Sistrunk+C.&author=Miranda-Carboni+G.&author=O%E2%80%99Regan+R.&author=Seewaldt+V.L.&title=Triple-negative+breast+cancer+in+African-American+women%3A+disparities+versus+biology">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) was observed in the present dataset (p = 0.0009 versus Caucasian, Fisher’s exact test).</div></section><section id="sec4.4.2" data-locator="sec4.4.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Centralized Immunohistochemistry</span></h4><div class="section-paragraph">For immunohistochemistry (IHC) cut tissue sections (5μm) on charged glass slides were baked for 10-12 hours at 58°C in a dry slide incubator, deparaffinized in xylene and rehydrated via an ethanol step gradient. Heat-induced antigen retrieval steps were performed at pH 9.0 for all targets. All primary antibodies were incubated at room temperature for 1 hour [clone, manufacturer, dilution: Her2 (SP3, Neomarkers, 1:100); ER (6F11, Leica, 1:200); CD3 (polyclonal, Dako, 1:100)] followed by a standard chromogenic staining protocol with the Envision Polymer-HRP anti-mouse/3,3′diaminobenzidine (DAB, Dako) process. Slides were counterstained in Harris hematoxylin. Immunohistochemistry scoring was performed using established guidelines, when appropriate. All IHC results were evaluated against positive and negative tissue controls.</div></section><section id="sec4.4.3" data-locator="sec4.4.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Sequencing Sample preparation</span></h4><section id="sec4.4.3.1" data-locator="sec4.4.3.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Whole exome sequencing (WES)</span></h4><div class="section-paragraph">Genomic DNA samples were used to prepare indexed libraries using the Nextera Rapid Capture Exome kit from Illumina. Library preparation was performed using a semi-automated 96-well plate method, with washing and clean-up/concentration steps performed on the Beckman Coulter Biomek NXP platform and with ZR-96 DNA Clean &amp; Concentrator-5 plates, respectively. Libraries were quantified using the Agilent 2100 Bioanalyzer. Pooled libraries were run on HiSeq4000 (2x150 paired end runs) to achieve a minimum of 150x on-target coverage per sample library. The raw Illumina sequence data was demultiplexed and converted to fastq files, and adaptor and low-quality sequences were trimmed. WES data was used for somatic mutation detection, microsatellite instability prediction, and somatic copy number alteration (SCNA) analysis as described below.</div></section><section id="sec4.4.3.2" data-locator="sec4.4.3.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">mRNA sequencing</span></h4><div class="section-paragraph">Indexed cDNA sequencing libraries were prepared from the RNA samples using the TruSeq Stranded RNA Sample Preparation Kit and bar-coded with individual tags. Library preparation was performed similarly to the WES. Quality control was performed at every step, and the libraries were quantified using the Agilent 2100 Bioanalyzer. Indexed libraries were prepared as equimolar pools and run on HiSeq4000 (2x150 paired end runs) to generate a minimum of 30 million paired-end reads per sample library. The raw Illumina sequence data was demultiplexed and converted to fastq files, and adaptor and low-quality sequences were trimmed.</div></section></section><section id="sec4.4.4" data-locator="sec4.4.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Proteomic analysis</span></h4><div class="section-paragraph">The proteomic, phosphoproteomic, and acetylproteomic analyses of breast cancer samples were structured as TMT-10-plex experiments. To facilitate quantitative comparison between all samples across experiments, a tumor-only common reference sample was included in each 10-plex. A common physical, rather than <em>in silico</em> reference was used for this purpose for optimal quantitative precision between TMT-10 experiments. 125 unique samples representing 122 tumors and three process replicates were distributed among 15 10-plex experiments. Eighteen normal adjacent samples were also included in two additional 10-plex experiments, for a total of 17 10-plex experiments. For each experiment, nine individual samples occupied the first nine channels and the 10th channel was reserved for the tumor-only reference sample (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">Figure S1</a>B). To avoid systematic bias in sample processing or missing values in detection across the experiments, samples underwent stratified randomization before processing, with each intrinsic subtype proportionally represented in each processing tranche and subsequent incorporation to each 10-plex (<a href="/cms/10.1016/j.cell.2020.10.036/attachment/3dbb5a07-b92f-4ae5-9c88-f84c3dca95db/mmc1" class="scroll-into-link">Table S1</a>). Longitudinal quality control of the process was tested by periodic analysis of full process replicates of a comparative reference (CompRef; <div class="dropBlock reference-citations"><a href="#bib86" id="back-bib86" data-db-target-for="bib86-d13420944e3710" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib86">Mertins et al., 2018</a><div data-db-target-of="bib86-d13420944e3710" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Tang L.C. </li><li class="loa__item">Krug K. </li><li class="loa__item">Clark D.J. </li><li class="loa__item">Gritsenko M.A. </li><li class="loa__item">Chen L. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Clauss T.R. </li><li class="loa__item">Shah P. </li><li class="loa__item">Gillette M.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Reproducible workflow for multiplexed deep-scale proteome and phosphoproteome analysis of tumor tissues by liquid chromatography-mass spectrometry.<div class="popup-ref"><span class="ref__series"><i>Nat. Protoc.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">1632-1661</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41596-018-0006-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29988108&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85049660247&amp;cf=" onclick="newWindow(this.href);return false">Scopus (121)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=1632-1661&journal=Nat.+Protoc.&author=Mertins+P.&author=Tang+L.C.&author=Krug+K.&author=Clark+D.J.&author=Gritsenko+M.A.&author=Chen+L.&author=Clauser+K.R.&author=Clauss+T.R.&author=Shah+P.&author=Gillette+M.A.&title=Reproducible+workflow+for+multiplexed+deep-scale+proteome+and+phosphoproteome+analysis+of+tumor+tissues+by+liquid+chromatography-mass+spectrometry">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) sample composed of a basal and a luminal patient-derived xenograft tumor. Four interstitial CompRef experiments were performed, before plex one and after plexes five, 10, and 17. The protocols below for protein extraction, tryptic digestion, TMT-10 labeling of peptides, peptide fractionation by basic reversed-phase liquid chromatography, phosphopeptide enrichment using immobilized metal affinity chromatography, and LC-MS/MS were performed as previously described in depth (<div class="dropBlock reference-citations"><a href="#bib86" id="back-bib86" data-db-target-for="bib86-d13420944e3713" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib86">Mertins et al., 2018</a><div data-db-target-of="bib86-d13420944e3713" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Tang L.C. </li><li class="loa__item">Krug K. </li><li class="loa__item">Clark D.J. </li><li class="loa__item">Gritsenko M.A. </li><li class="loa__item">Chen L. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Clauss T.R. </li><li class="loa__item">Shah P. </li><li class="loa__item">Gillette M.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Reproducible workflow for multiplexed deep-scale proteome and phosphoproteome analysis of tumor tissues by liquid chromatography-mass spectrometry.<div class="popup-ref"><span class="ref__series"><i>Nat. Protoc.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">1632-1661</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41596-018-0006-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29988108&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85049660247&amp;cf=" onclick="newWindow(this.href);return false">Scopus (121)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=1632-1661&journal=Nat.+Protoc.&author=Mertins+P.&author=Tang+L.C.&author=Krug+K.&author=Clark+D.J.&author=Gritsenko+M.A.&author=Chen+L.&author=Clauser+K.R.&author=Clauss+T.R.&author=Shah+P.&author=Gillette+M.A.&title=Reproducible+workflow+for+multiplexed+deep-scale+proteome+and+phosphoproteome+analysis+of+tumor+tissues+by+liquid+chromatography-mass+spectrometry">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div><section id="sec4.4.4.1" data-locator="sec4.4.4.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Common reference pool construction</span></h4><div class="section-paragraph">Considerations informing generation of the common reference sample were that it needed to be available at the onset of discovery work, of adequate quantity to cover all planned experiments with overhead for additional possible experiments, and broadly representative of the population of breast cancer samples in the overall sample cohort. To ensure capacity for additional samples or experiments given a target input of 400 ug protein per channel per experiment, 12 mg total was targeted for reference material. To meet these collective requirements, 40 samples with an average of 2.7 mg total protein yield were selected based on hormone receptor status, including 9 triple negative, 12 HER2 positive, and 19 estrogen receptor positive specimens. After reserving 400 ug protein / sample for individual sample analysis, an additional amount of 300 ug for each of the 40 samples was pooled. The resulting 12 mg of pooled reference material was divided into 400 ug aliquots and frozen at −80°C until use.</div><div class="section-paragraph">Making the internal reference representative of the study as a whole was particularly important since by definition only analytes represented in the reference sample would be included in the final ratio-based data analyses. To accomplish this goal, similar percentages found in the total sample population of specific subtypes were implemented in the internal reference. As noted, samples were selected on this basis of hormone receptor status, as PAM50 status was not available at the time of reference preparation; however, of the 40 samples included in the internal reference, 11 were subsequently shown to be basal, 7 were HER2+, 12 were Luminal A, 8 were Luminal B, 1 was normal-like, and 1 was not determined.</div></section><section id="sec4.4.4.2" data-locator="sec4.4.4.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Protein extraction and digestion</span></h4><div class="section-paragraph">Cryopulverized human breast cancer patient tumor samples were homogenized in lysis buffer at a ratio of 750 uL lysis buffer for every 100-125 mg wet weight tissue. The lysis buffer consisted of 8 M urea, 75 mM NaCl, 1mM EDTA, 50 mM Tris HCl (pH 8), 10 mM NaF, phosphatase inhibitor cocktail 2 (1:100; Sigma, P5726) and cocktail 3 (1:100; Sigma, P0044), 2 μg/mL aprotinin (Sigma, A6103), 10 μg/mL Leupeptin (Roche, 11017101001), and 1 mM PMSF (Sigma, 78830). Lysates were centrifuged at 20,000 g for 10 minutes and protein concentrations of the clarified lysates were measured by BCA assay (Pierce). Protein lysates were subsequently reduced with 5 mM dithiothreitol (Thermo Scientific, 20291) for 45 minutes at room temperature and alkylated with 10 mM iodoacetamide (Sigma, A3221) for 45 minutes in the dark. Prior to digestion, samples were diluted 4-fold to achieve 2 M urea with 50mM Tris HCl (pH 8). Digestion was performed with LysC (Wako, 100369-826) for 2 hours and with trypsin (Promega, V511X) overnight, both at a 1:50 enzyme-to-protein ratio and at room temperature. Digested samples were acidified with formic acid (FA; Fluka, 56302) to achieve a final volumetric concentration of 1% (final pH of ∼3), and centrifuged at 1,500 g for 15 minutes to clear precipitated urea from peptide lysates. Samples were desalted on C18 SepPak columns (Waters, 100mg, WAT036820) and dried down using a SpeedVac apparatus.</div></section><section id="sec4.4.4.3" data-locator="sec4.4.4.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">TMT-10 labeling of peptides</span></h4><div class="section-paragraph">400 ug of desalted peptides per sample (based on protein-level BCA prior to digestion) were labeled with 10-plex TMT reagents according to the manufacturer’s instructions (Thermo Scientific; Pierce Biotechnology, Germany). For each 400 ug peptide aliquot of an individual breast tumor sample, 3.2 mg of labeling reagent was used. Peptides were dissolved in 400 μL of 50 mM HEPES (pH 8.5) solution and labeling reagent was added in 164 μL of acetonitrile. After 1 h incubation with shaking and after confirming good label incorporation, 32 uL of 5% hydroxylamine was added to quench the unreacted TMT reagents. Good label incorporation was defined as having a minimum of 95% fully labeled MS/MS spectra in each sample, as measured by LC-MS/MS after taking out a 2 μg aliquot from each sample and analyzing 1μg. If a sample did not have sufficient label incorporation, additional TMT was added to the sample and another 1 h incubation was performed with shaking. At the time that the labeling efficiency quality control samples were taken out, an additional 2 μg of material from each sample was taken out and combined as a mixing control. After analyzing the mixing control sample by LC-MS/MS, intensity values of the individual TMT reporter ions were summed across all peptide spectrum matches and compared to ensure that the total reporter ion intensity of each sample met a threshold of ± 25% of the internal reference. If necessary, adjustments were made by either labeling additional material or reducing an individual sample’s contribution to the mixture, and analyzing a subsequent mixing control, until all samples met the threshold and were thus approximately 1:1:1. Differentially labeled peptides were then mixed (10 × 400 ug) and dried down via vacuum centrifuge, and the quenched, combined sample was subsequently desalted on a 500 mg C18 SepPak column.</div></section><section id="sec4.4.4.4" data-locator="sec4.4.4.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Peptide fractionation</span></h4><div class="section-paragraph">To reduce sample complexity, peptide samples were separated by high pH reversed-phase (RP) chromatography as described previously. A desalted 4 mg, 10-plex TMT-labeled experiment (based on protein-level BCA prior to digestion) was reconstituted in 900 μL 0.0455% ammonium formate (pH 10) and 2% acetonitrile, loaded on a 4.6 mm x 250 mm column RP Zorbax 300 A Extend-C18 column (Agilent, 3.5 μm bead size), and separated on an Agilent 1100 Series HPLC instrument using basic pH reversed-phase chromatography. Solvent A (2% acetonitrile, 4.4 mM ammonium formate, pH 10) and a nonlinear increasing concentration of solvent B (90% acetonitrile, 4.5 mM ammonium formate, pH 10) were used to separate peptides. The 4.5 mM ammonium formate solvents were made by 40–fold dilution of a stock solution of 180 mM ammonium formate, pH 10. To make 200 mL of stock solution, slowly add 4.6 mL of 30% (wt/vol) ammonium hydroxide (Ammonia solution 28.0%–30.0% (NH<sub>3</sub> basis) ACS, 0.9 g/ml, Fluka) to ∼150mL of HPLC grade water, then titrate to pH 10.0 with ∼9.0 mL of concentrated formic acid (&gt; 95% Sigma-Aldrich); bring to final volume of 200 mL with HPLC grade water. The 96 minute separation LC gradient followed this profile: (min: %B) 0:0; 7:0; 13:16; 73:40; 77:44; 82:60; 96:60. The flow rate was 1 mL/min. For each 4 mg separation, 77 fractions were collected into a 96 deep-well 2mL plate (Whatman, #7701- 5200), with fractions combined in a stepwise concatenation strategy and acidified to a final concentration of 0.1% FA as reported previously. An additional 12 fractions were collected from the 96 deep-well plate for fraction A, representing the early-eluting fractions that tend to contain multi-phosphorylated peptides. 5% of the volume of each of the 24+A proteome fractions was allocated for proteome analysis, dried down, and re-suspended in 3% MeCN/0.1% FA (MeCN; acetonitrile) to a peptide concentration of 0.5 μg/uL for LC-MS/MS analysis. The remaining 95% of 24 concatenated fractions were further combined into 12 fractions, with fraction A as a separate fraction. These 13 fractions were then enriched for phosphopeptides as described below.</div></section><section id="sec4.4.4.5" data-locator="sec4.4.4.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Phosphopeptide enrichment</span></h4><div class="section-paragraph">Ni-NTA agarose beads were used to prepare Fe<sup>3+</sup>-NTA agarose beads. In each phosphoproteome fraction, ∼317 μg peptides (based on protein-level BCA prior to digestion, with uniform distribution across fractions presumed) was reconstituted in 633 μL 80% MeCN/0.1% TFA (trifluoroacetic acid) solvent and incubated with 10 μL of the IMAC beads for 30 minutes on a shaker at RT. After incubation, samples were briefly spun down on a tabletop centrifuge; clarified peptide flow-throughs were separated from the beads; and the beads were reconstituted in 200 μL IMAC binding/wash buffer (80 MeCN/0.1% TFA) and loaded onto equilibrated Empore C18 silica-packed stage tips (3M, 2315). Samples were then washed twice with 50 μL of IMAC binding/wash buffer and once with 50 uL 1% FA, and were eluted from the IMAC beads to the stage tips with 3 × 70 uL washes of 500 mM dibasic sodium phosphate (pH 7.0, Sigma S9763). Stage tips were then washed once with 100 μL 1% FA and phosphopeptides were eluted from the stage tips with 60 uL 50% MeCN/0.1% FA. Phosphopeptides were dried down and resuspended in 9 μL 50% MeCN/0.1%FA for LC-MS/MS analysis, with 4 μL injected per run.</div></section><section id="sec4.4.4.6" data-locator="sec4.4.4.6" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Acetylpeptide enrichment</span></h4><div class="section-paragraph">Acetylated lysine peptides were enriched using an antibody against the Acetyl-Lysine motif (CST PTM-SCAN Catalogue No. 13416) as described before (<div class="dropBlock reference-citations"><a href="#bib45" id="back-bib45" data-db-target-for="bib45-d13420944e3754" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib45">Gillette et al., 2020</a><div data-db-target-of="bib45-d13420944e3754" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gillette M.A. </li><li class="loa__item">Satpathy S. </li><li class="loa__item">Cao S. </li><li class="loa__item">Dhanasekaran S.M. </li><li class="loa__item">Vasaikar S.V. </li><li class="loa__item">Krug K. </li><li class="loa__item">Petralia F. </li><li class="loa__item">Li Y. </li><li class="loa__item">Liang W.-W. </li><li class="loa__item">Reva B. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Clinical Proteomic Tumor Analysis Consortium</span><br />Proteogenomic Characterization Reveals Therapeutic Vulnerabilities in Lung Adenocarcinoma.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">182</span></span>: <span class="ref__seriesPages">200-225.e35</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2020.06.013&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2020.06.013&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2020.06.013&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=32649874&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85087400742&amp;cf=" onclick="newWindow(this.href);return false">Scopus (76)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=182&publication_year=2020&pages=200-225.e35&journal=Cell&author=Clinical+Proteomic+Tumor+Analysis+Consortium&title=Proteogenomic+Characterization+Reveals+Therapeutic+Vulnerabilities+in+Lung+Adenocarcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib131" id="back-bib131" data-db-target-for="bib131-d13420944e3757" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib131">Udeshi et al., 2020</a><div data-db-target-of="bib131-d13420944e3757" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Udeshi N.D. </li><li class="loa__item">Mani D.C. </li><li class="loa__item">Satpathy S. </li><li class="loa__item">Fereshetian S. </li><li class="loa__item">Gasser J.A. </li><li class="loa__item">Svinkina T. </li><li class="loa__item">Olive M.E. </li><li class="loa__item">Ebert B.L. </li><li class="loa__item">Mertins P. </li><li class="loa__item">Carr S.A. </li></ul><div class="reference-citations__detials">Rapid and deep-scale ubiquitylation profiling for biology and translational research.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">359</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41467-019-14175-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31953384&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85078023220&amp;cf=" onclick="newWindow(this.href);return false">Scopus (26)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=359&journal=Nat.+Commun.&author=Udeshi+N.D.&author=Mani+D.C.&author=Satpathy+S.&author=Fereshetian+S.&author=Gasser+J.A.&author=Svinkina+T.&author=Olive+M.E.&author=Ebert+B.L.&author=Mertins+P.&author=Carr+S.A.&title=Rapid+and+deep-scale+ubiquitylation+profiling+for+biology+and+translational+research">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). IMAC eluents were concatenated into 6 fractions (∼330 μg peptides per fraction) and dried down using a SpeedVac apparatus. Peptides were reconstituted with 1.4ml of IAP buffer (5 mM MOPS pH 7.2, 1 mM Sodium Phosphate (dibasic), 5 mM NaCl) per fraction and incubated for 2 hours at 4°C with pre-washed (4 times with IAP buffer) agarose beads bound to acetyl-lysine motif antibody. Peptide-bound beads were washed 4 times with ice-cold PBS followed by elution with 100ul of 0.15% TFA. Eluents were desalted using C18 stage-tips, eluted with 50% ACN and dried down. Acetylpeptides were suspended in 7ul of 0.1% FA and 3% ACN, with 4ul injected per run.</div></section></section><section id="sec4.4.5" data-locator="sec4.4.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">LC-MS/MS for proteomic analysis</span></h4><section id="sec4.4.5.1" data-locator="sec4.4.5.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Liquid chromatography</span></h4><div class="section-paragraph">Online separation was done with a nanoflow Proxeon EASY-nLC 1200 UHPLC system (Thermo Fisher Scientific). In this set up, the LC system, column, and platinum wire used to deliver electrospray source voltage were connected via a stainless-steel cross (360 μm, IDEX Health &amp; Science, UH-906x). The column was heated to 50°C using a column heater sleeve (Phoenix-ST) to prevent over-pressuring of columns during UHPLC separation. Each peptide fraction containing ∼1ug (based on protein-level BCA prior to digestion, with uniform distribution of fraction content presumed), the equivalent of 12% of each global proteome sample in a 2 ul injection volume or 50% of each phosphoproteome sample in a 4 ul injection volume, was injected onto an in-house packed 20cm x 75um diameter C18 silica picofrit capillary column (1.9 μm ReproSil-Pur C18-AQ beads, Dr. Maisch GmbH, r119.aq; Picofrit 10um tip opening, New Objective, PF360-75-10-N-5). Mobile phase flow rate was 200 nL/min, comprising 3% acetonitrile/0.1% formic acid (Solvent A) and 90% acetonitrile/0.1% formic acid (Solvent B). The 110-minute LC-MS/MS method consisted of a 10-min column-equilibration procedure, a 20-min sample-loading procedure, and the following gradient profile: (min:%B) 0:2; 1:6; 85:30; 94:60; 95;90; 100:90; 101:50; 110:50 (the last two steps at 500 nL/min flow rate).</div></section><section id="sec4.4.5.2" data-locator="sec4.4.5.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Mass spectrometry</span></h4><div class="section-paragraph">Samples were analyzed with a benchtop Orbitrap Fusion Lumos mass spectrometer (Thermo Fisher Scientific) equipped with a NanoSpray Flex NG ion source. Data-dependent acquisition was performed using Xcalibur QExactive v2.1 software in positive ion mode at a spray voltage of 1.8 kV. MS1 spectra were measured with a resolution of 60,000, an AGC target of 4e5 and a mass range from 350 to 1800 m/z. The data-dependent mode cycle time was set at 2 s with an MS2 resolution of 50,000, an AGC target of 1e5, an isolation window of 0.7 m/z, a maximum injection time of 105 msec, and an HCD collision energy of 38%. Peptide mode was selected for monoisotopic peak determination, and charge state screening was enabled to only include precursor charge states 2-6, with an intensity threshold of 1e4. Peptides that triggered MS/MS scans were dynamically excluded from further MS/MS scans for 45 s, with a ± 10 ppm mass tolerance. “Perform dependent scan on single charge state per precursor only” was enabled.</div></section></section></section><section id="sec4.5" data-locator="sec4.5" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Quantification and Statistical Analysis</span></h3><section id="sec4.5.1" data-locator="sec4.5.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Genomic data analysis</span></h4><section id="sec4.5.1.1" data-locator="sec4.5.1.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Somatic mutation and copy number detection</span></h4><div class="section-paragraph">WES data were analyzed on the Terra cloud-based analysis platform (<a class="inter-ref-link" href="https://terra.bio/" target="_blank">https://terra.bio/</a>).</div><div class="section-paragraph">Somatic mutations were detected using the Cancer Genome Analysis WES Characterization Pipeline (available on Terra - <a class="inter-ref-link" href="https://portal.firecloud.org/?return=terra#methods/getzlab/CGA_WES_Characterization_Pipeline_v0.1_Dec2018/2" target="_blank">https://portal.firecloud.org/?return=terra#methods/getzlab/CGA_WES_Characterization_Pipeline_v0.1_Dec2018/2</a>). This pipeline is the Getz Lab’s standard computational workflow for characterizing a tumor sample’s somatic variants through contrastive computational analysis of matched tumor-normal WES BAMs. The pipeline includes state-of-the-art tools for quality control (QC) and characterization of paired (tumor/normal) whole exome sequencing data. The pipeline is organized into five modules: (1) DNA Sequence Data Quality Control such as ContEst for detecting cross-patient contamination (<div class="dropBlock reference-citations"><a href="#bib27" id="back-bib27" data-db-target-for="bib27-d13420944e3792" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib27">Cibulskis et al., 2011</a><div data-db-target-of="bib27-d13420944e3792" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Cibulskis K. </li><li class="loa__item">McKenna A. </li><li class="loa__item">Fennell T. </li><li class="loa__item">Banks E. </li><li class="loa__item">DePristo M. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">ContEst: estimating cross-contamination of human samples in next-generation sequencing data.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">27</span></span>: <span class="ref__seriesPages">2601-2602</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fbioinformatics%2Fbtr446&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=21803805&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-80052707308&amp;cf=" onclick="newWindow(this.href);return false">Scopus (152)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2011&pages=2601-2602&journal=Bioinformatics&author=Cibulskis+K.&author=McKenna+A.&author=Fennell+T.&author=Banks+E.&author=DePristo+M.&author=Getz+G.&title=ContEst%3A+estimating+cross-contamination+of+human+samples+in+next-generation+sequencing+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), (2) Variant Discovery including MuTect for detection of somatic single nucleotide variants (<div class="dropBlock reference-citations"><a href="#bib28" id="back-bib28" data-db-target-for="bib28-d13420944e3795" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib28">Cibulskis et al., 2013</a><div data-db-target-of="bib28-d13420944e3795" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Cibulskis K. </li><li class="loa__item">Lawrence M.S. </li><li class="loa__item">Carter S.L. </li><li class="loa__item">Sivachenko A. </li><li class="loa__item">Jaffe D. </li><li class="loa__item">Sougnez C. </li><li class="loa__item">Gabriel S. </li><li class="loa__item">Meyerson M. </li><li class="loa__item">Lander E.S. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples.<div class="popup-ref"><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">213-219</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnbt.2514&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=23396013&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84874025843&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2431)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2013&pages=213-219&journal=Nat.+Biotechnol.&author=Cibulskis+K.&author=Lawrence+M.S.&author=Carter+S.L.&author=Sivachenko+A.&author=Jaffe+D.&author=Sougnez+C.&author=Gabriel+S.&author=Meyerson+M.&author=Lander+E.S.&author=Getz+G.&title=Sensitive+detection+of+somatic+point+mutations+in+impure+and+heterogeneous+cancer+samples">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and Strelka for detecting small insertions and deletions (<div class="dropBlock reference-citations"><a href="#bib61" id="back-bib61" data-db-target-for="bib61-d13420944e3798" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib61">Kim et al., 2018</a><div data-db-target-of="bib61-d13420944e3798" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kim S. </li><li class="loa__item">Scheffler K. </li><li class="loa__item">Halpern A.L. </li><li class="loa__item">Bekritsky M.A. </li><li class="loa__item">Noh E. </li><li class="loa__item">Källberg M. </li><li class="loa__item">Chen X. </li><li class="loa__item">Kim Y. </li><li class="loa__item">Beyter D. </li><li class="loa__item">Krusche P. </li><li class="loa__item">Saunders C.T. </li></ul><div class="reference-citations__detials">Strelka2: fast and accurate calling of germline and somatic variants.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">591-594</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41592-018-0051-x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30013048&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85049953163&amp;cf=" onclick="newWindow(this.href);return false">Scopus (263)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=591-594&journal=Nat.+Methods&author=Kim+S.&author=Scheffler+K.&author=Halpern+A.L.&author=Bekritsky+M.A.&author=Noh+E.&author=K%C3%A4llberg+M.&author=Chen+X.&author=Kim+Y.&author=Beyter+D.&author=Krusche+P.&author=Saunders+C.T.&title=Strelka2%3A+fast+and+accurate+calling+of+germline+and+somatic+variants">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), (3) Copy Number Characterization employing AllelicCapSeg for assessing allele-specific copy-number alterations and ABSOLUTE for estimating tumor purity, ploidy, absolute allelic copy number and Cancer Cell Fraction (CCFs) (<div class="dropBlock reference-citations"><a href="#bib21" id="back-bib21" data-db-target-for="bib21-d13420944e3801" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib21">Carter et al., 2012</a><div data-db-target-of="bib21-d13420944e3801" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Carter S.L. </li><li class="loa__item">Cibulskis K. </li><li class="loa__item">Helman E. </li><li class="loa__item">McKenna A. </li><li class="loa__item">Shen H. </li><li class="loa__item">Zack T. </li><li class="loa__item">Laird P.W. </li><li class="loa__item">Onofrio R.C. </li><li class="loa__item">Winckler W. </li><li class="loa__item">Weir B.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Absolute quantification of somatic DNA alterations in human cancer.<div class="popup-ref"><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">413-421</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnbt.2203&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=22544022&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84860782006&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1036)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2012&pages=413-421&journal=Nat.+Biotechnol.&author=Carter+S.L.&author=Cibulskis+K.&author=Helman+E.&author=McKenna+A.&author=Shen+H.&author=Zack+T.&author=Laird+P.W.&author=Onofrio+R.C.&author=Winckler+W.&author=Weir+B.A.&title=Absolute+quantification+of+somatic+DNA+alterations+in+human+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), (4) Variant rescue, Annotation and Filtering including deTiN, which estimates potential tumor-in-normal contamination (<div class="dropBlock reference-citations"><a href="#bib126" id="back-bib126" data-db-target-for="bib126-d13420944e3805" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib126">Taylor-Weiner et al., 2018</a><div data-db-target-of="bib126-d13420944e3805" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Taylor-Weiner A. </li><li class="loa__item">Stewart C. </li><li class="loa__item">Giordano T. </li><li class="loa__item">Miller M. </li><li class="loa__item">Rosenberg M. </li><li class="loa__item">Macbeth A. </li><li class="loa__item">Lennon N. </li><li class="loa__item">Rheinbay E. </li><li class="loa__item">Landau D.-A. </li><li class="loa__item">Wu C.J. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">DeTiN: overcoming tumor-in-normal contamination.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">531-534</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41592-018-0036-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29941871&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85048968644&amp;cf=" onclick="newWindow(this.href);return false">Scopus (20)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=531-534&journal=Nat.+Methods&author=Taylor-Weiner+A.&author=Stewart+C.&author=Giordano+T.&author=Miller+M.&author=Rosenberg+M.&author=Macbeth+A.&author=Lennon+N.&author=Rheinbay+E.&author=Landau+D.-A.&author=Wu+C.J.&author=Getz+G.&title=DeTiN%3A+overcoming+tumor-in-normal+contamination">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), and (5) Visualization. Each of these modules consists of multiple additional tools (<div class="dropBlock reference-citations"><a href="#bib82" id="back-bib82" data-db-target-for="bib82-d13420944e3808" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib82">McLaren et al., 2016</a><div data-db-target-of="bib82-d13420944e3808" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">McLaren W. </li><li class="loa__item">Gil L. </li><li class="loa__item">Hunt S.E. </li><li class="loa__item">Riat H.S. </li><li class="loa__item">Ritchie G.R.S. </li><li class="loa__item">Thormann A. </li><li class="loa__item">Flicek P. </li><li class="loa__item">Cunningham F. </li></ul><div class="reference-citations__detials">The Ensembl Variant Effect Predictor.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">122</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2Fs13059-016-0974-4&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27268795&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84973338712&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1927)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=122&journal=Genome+Biol.&author=McLaren+W.&author=Gil+L.&author=Hunt+S.E.&author=Riat+H.S.&author=Ritchie+G.R.S.&author=Thormann+A.&author=Flicek+P.&author=Cunningham+F.&title=The+Ensembl+Variant+Effect+Predictor">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib104" id="back-bib104" data-db-target-for="bib104-d13420944e3811" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib104">Ramos et al., 2015</a><div data-db-target-of="bib104-d13420944e3811" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ramos A.H. </li><li class="loa__item">Lichtenstein L. </li><li class="loa__item">Gupta M. </li><li class="loa__item">Lawrence M.S. </li><li class="loa__item">Pugh T.J. </li><li class="loa__item">Saksena G. </li><li class="loa__item">Meyerson M. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">Oncotator: cancer variant annotation tool.<div class="popup-ref"><span class="ref__series"><i>Hum. Mutat.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">36</span></span>: <span class="ref__seriesPages">E2423-E2429</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1002%2Fhumu.22771&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=25703262&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84925965197&amp;cf=" onclick="newWindow(this.href);return false">Scopus (283)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=E2423-E2429&journal=Hum.+Mutat.&author=Ramos+A.H.&author=Lichtenstein+L.&author=Gupta+M.&author=Lawrence+M.S.&author=Pugh+T.J.&author=Saksena+G.&author=Meyerson+M.&author=Getz+G.&title=Oncotator%3A+cancer+variant+annotation+tool">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Note that we conducted our analysis on hg19, requiring replacing the hg38 reference file inputs with their hg19 analogs.</div><div class="section-paragraph">The MAF Panel of Normals (PoN) Filter is a method within the Variant rescue, Annotation and Filtering module. It is a highly effective tool for filtering false-positive germline variants and common artifacts from somatic mutation calls. This tool requires as input a Panel of Normals (PoN) constructed from a collection derived from BAMs of normal samples. To be effective at filtering artifacts, the library preparation and sequencing technology used for the PoN’s normal samples should mirror that used in the processing of the matched tumor/normal pairs within the study cohort. For this analysis, we used a TCGA-based PoN and an ICE PoN.</div></section><section id="sec4.5.1.2" data-locator="sec4.5.1.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Germline short variant discovery from WES</span></h4><div class="section-paragraph">Germline mutations were also analyzed on the Terra cloud-based analysis platform utilizing the GATK4 SNPS + Indels best practice workflow (<a class="inter-ref-link" href="https://gatk.broadinstitute.org?id=11145" target="_blank">https://gatk.broadinstitute.org?id=11145</a>) (<div class="dropBlock reference-citations"><a href="#bib81" id="back-bib81" data-db-target-for="bib81-d13420944e3824" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib81">McKenna et al., 2010</a><div data-db-target-of="bib81-d13420944e3824" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">McKenna A. </li><li class="loa__item">Hanna M. </li><li class="loa__item">Banks E. </li><li class="loa__item">Sivachenko A. </li><li class="loa__item">Cibulskis K. </li><li class="loa__item">Kernytsky A. </li><li class="loa__item">Garimella K. </li><li class="loa__item">Altshuler D. </li><li class="loa__item">Gabriel S. </li><li class="loa__item">Daly M. </li><li class="loa__item">DePristo M.A. </li></ul><div class="reference-citations__detials">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">1297-1303</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1101%2Fgr.107524.110&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=20644199&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-77956295988&amp;cf=" onclick="newWindow(this.href);return false">Scopus (12104)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=1297-1303&journal=Genome+Res.&author=McKenna+A.&author=Hanna+M.&author=Banks+E.&author=Sivachenko+A.&author=Cibulskis+K.&author=Kernytsky+A.&author=Garimella+K.&author=Altshuler+D.&author=Gabriel+S.&author=Daly+M.&author=DePristo+M.A.&title=The+Genome+Analysis+Toolkit%3A+a+MapReduce+framework+for+analyzing+next-generation+DNA+sequencing+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). This workflow consists of three sub-workflows: (i) Processing-For-Variant-Discovery workflow, which takes a single sample’s sequencing data in unmapped BAM (uBAM) format and outputs a clean BAM file and its index, suitable for variant discovery analysis, (ii) Haplotypecaller-GVCF workflow, which runs the HaplotypeCaller tool (<div class="dropBlock reference-citations"><a href="#bib102" id="back-bib102" data-db-target-for="bib102-d13420944e3827" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib102">Poplin et al., 2017</a><div data-db-target-of="bib102-d13420944e3827" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Poplin R. </li><li class="loa__item">Ruano-Rubio V. </li><li class="loa__item">DePristo M.A. </li><li class="loa__item">Fennell T.J. </li><li class="loa__item">Carneiro M.O. </li><li class="loa__item">Van der Auwera G.A. </li><li class="loa__item">Kling D.E. </li><li class="loa__item">Gauthier L.D. </li><li class="loa__item">Levy-Moonshine A. </li><li class="loa__item">Roazen D. </li><li>et al.</li></ul><div class="reference-citations__detials">Scaling accurate genetic variant discovery to tens of thousands of samples.<div class="popup-ref"><span class="ref__series"><i>bioRxiv.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1101/201178">https://doi.org/10.1101/201178</a></span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1101%2F201178&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85094275866&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&journal=bioRxiv&author=Poplin+R.&author=Ruano-Rubio+V.&author=DePristo+M.A.&author=Fennell+T.J.&author=Carneiro+M.O.&author=Van+der+Auwera+G.A.&author=Kling+D.E.&author=Gauthier+L.D.&author=Levy-Moonshine+A.&author=Roazen+D.&title=Scaling+accurate+genetic+variant+discovery+to+tens+of+thousands+of+samples">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) from GATK4 in GVCF mode on the BAM and BAI created in the previous step, and (iii) Joint-Discovery workflow, which conducts the joint-calling and VQSR-filtering portions of the GATK Best Practices for germline SNP and Indel discovery. In our analysis, Processing-For-Variant-Discovery was skipped, as our WES pipeline produced BAM files compatible for the next step of analysis and an Illumina-compatible interval list was used. Details regarding the specific Terra workflows used to conduct this analysis can be found in the public workspace (<a class="inter-ref-link" href="https://app.terra.bio/#workspaces/help-gatk/Germline-SNPs-Indels-GATK4-hg38" target="_blank">https://app.terra.bio/#workspaces/help-gatk/Germline-SNPs-Indels-GATK4-hg38</a>). Note that we conducted our analysis on hg19, requiring replacing the hg38 reference file inputs with their hg19 analogs.</div></section><section id="sec4.5.1.3" data-locator="sec4.5.1.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">RNA quantification</span></h4><div class="section-paragraph">The raw Illumina sequence data from HiSeq4000 was demultiplexed and converted to .fastq files. Read quality was examined using FastQC (version 0.10.1) and adaptor and low quality sequences were trimmed using Trim Galore (version 0.3.3) using a quality score cutoff of Q &lt; 30 and Length &lt; 50 bp.. Trimmed reads were mapped to the hg19 reference genome using MapSplice (version 2.1.8). Transcripts were assembled and RNA expressions were quantified in Fragments Per Kilobase of transcript per Million mapped reads (FPKM) using Cufflinks (v2.1.1) (<div class="dropBlock reference-citations"><a href="#bib129" id="back-bib129" data-db-target-for="bib129-d13420944e3838" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib129">Trapnell et al., 2010</a><div data-db-target-of="bib129-d13420944e3838" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Trapnell C. </li><li class="loa__item">Williams B.A. </li><li class="loa__item">Pertea G. </li><li class="loa__item">Mortazavi A. </li><li class="loa__item">Kwan G. </li><li class="loa__item">van Baren M.J. </li><li class="loa__item">Salzberg S.L. </li><li class="loa__item">Wold B.J. </li><li class="loa__item">Pachter L. </li></ul><div class="reference-citations__detials">Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation.<div class="popup-ref"><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">511-515</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnbt.1621&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=20436464&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-77952123055&amp;cf=" onclick="newWindow(this.href);return false">Scopus (9185)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2010&pages=511-515&journal=Nat.+Biotechnol.&author=Trapnell+C.&author=Williams+B.A.&author=Pertea+G.&author=Mortazavi+A.&author=Kwan+G.&author=van+Baren+M.J.&author=Salzberg+S.L.&author=Wold+B.J.&author=Pachter+L.&title=Transcript+assembly+and+quantification+by+RNA-Seq+reveals+unannotated+transcripts+and+isoform+switching+during+cell+differentiation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and transcript coverage was calculated using (bedtools version 2.20.1). Relevant QC metrics and statistics can be found in <a href="/cms/10.1016/j.cell.2020.10.036/attachment/3dbb5a07-b92f-4ae5-9c88-f84c3dca95db/mmc1" class="scroll-into-link">Table S1</a>B. Derived data matrix of FPKM values was further processed in R. FPKM values of transcripts mapping to the same HGNC symbol were averaged within a sample to create a gene-centric data matrix. FPKM values of 0 were considered as missing values and replaced by NA before applying log2 transformation. For integrative multi-omics subtyping, we first normalized each gene by the median log2(FPKM) across all tumors (gene-centering) before applying a robustified z-score transformation (median-centered, MAD-scaled) per sample.</div></section><section id="sec4.5.1.4" data-locator="sec4.5.1.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">GISTIC and MutSig analysis</span></h4><div class="section-paragraph">Genomic Identification of Significant Targets in Cancer (GISTIC2.0) algorithm (<div class="dropBlock reference-citations"><a href="#bib83" id="back-bib83" data-db-target-for="bib83-d13420944e3849" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib83">Mermel et al., 2011</a><div data-db-target-of="bib83-d13420944e3849" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mermel C.H. </li><li class="loa__item">Schumacher S.E. </li><li class="loa__item">Hill B. </li><li class="loa__item">Meyerson M.L. </li><li class="loa__item">Beroukhim R. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R41</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2Fgb-2011-12-4-r41&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=21527027&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-79955166265&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1334)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R41&journal=Genome+Biol.&author=Mermel+C.H.&author=Schumacher+S.E.&author=Hill+B.&author=Meyerson+M.L.&author=Beroukhim+R.&author=Getz+G.&title=GISTIC2.0+facilitates+sensitive+and+confident+localization+of+the+targets+of+focal+somatic+copy-number+alteration+in+human+cancers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) was used to identify significantly amplified or deleted focal-level and arm-level events, with q values smaller than 0.25 considered significant. The following parameters were used:<ul class="ce-list--remove-bullets" id="olist0010"><li class="ce-list--remove-bullets__list-item" id="o0010"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">Amplification Threshold = 0.1</div></li><li class="ce-list--remove-bullets__list-item" id="o0015"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">Deletion Threshold = - 0.1</div></li><li class="ce-list--remove-bullets__list-item" id="o0020"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">Cap Values = 1.5</div></li><li class="ce-list--remove-bullets__list-item" id="o0025"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">Broad Length Cutoff = 0.98</div></li><li class="ce-list--remove-bullets__list-item" id="o0030"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">Remove X-Chromosome = 0</div></li><li class="ce-list--remove-bullets__list-item" id="o0035"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">Confidence Level = 0.99</div></li><li class="ce-list--remove-bullets__list-item" id="o0040"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">Join Segment Size = 4</div></li><li class="ce-list--remove-bullets__list-item" id="o0045"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">Arm Level Peel-Off = 1</div></li><li class="ce-list--remove-bullets__list-item" id="o0050"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">Maximum Sample Segments = 2000</div></li><li class="ce-list--remove-bullets__list-item" id="o0055"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">Gene GISTIC = 1</div></li></ul></div><div class="section-paragraph">Each gene of every sample is assigned a thresholded copy number level that reflects the magnitude of its deletion or amplification. These are integer values ranging from −2 to 2, where 0 means no amplification or deletion of magnitude greater than the threshold parameters described above. Amplifications are represented by positive numbers: 1 means amplification above the amplification threshold; 2 means amplification larger than the arm level amplifications observed in the sample. Deletions are represented by negative numbers: −1 means deletion beyond the threshold; −2 means deletions greater than the minimum arm-level copy number observed in the sample.</div><div class="section-paragraph">The somatic variants were filtered through a panel of normals to remove potential sequencing artifacts and undetected germline variants (see “<a href="#sec4.5.1.1" class="scroll-into-link">Somatic Mutation and Copy Number Detection</a>”). MutSig2CV (<div class="dropBlock reference-citations"><a href="#bib64" id="back-bib64" data-db-target-for="bib64-d13420944e3900" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib64">Lawrence et al., 2014</a><div data-db-target-of="bib64-d13420944e3900" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lawrence M.S. </li><li class="loa__item">Stojanov P. </li><li class="loa__item">Mermel C.H. </li><li class="loa__item">Robinson J.T. </li><li class="loa__item">Garraway L.A. </li><li class="loa__item">Golub T.R. </li><li class="loa__item">Meyerson M. </li><li class="loa__item">Gabriel S.B. </li><li class="loa__item">Lander E.S. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">Discovery and saturation analysis of cancer genes across 21 tumour types.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">505</span></span>: <span class="ref__seriesPages">495-501</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature12912&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=24390350&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84892833777&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1826)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=505&publication_year=2014&pages=495-501&journal=Nature&author=Lawrence+M.S.&author=Stojanov+P.&author=Mermel+C.H.&author=Robinson+J.T.&author=Garraway+L.A.&author=Golub+T.R.&author=Meyerson+M.&author=Gabriel+S.B.&author=Lander+E.S.&author=Getz+G.&title=Discovery+and+saturation+analysis+of+cancer+genes+across+21+tumour+types">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) was run on these filtered results to evaluate the significance of mutated genes and estimate mutation densities of samples. These results were constrained to genes given in (<div class="dropBlock reference-citations"><a href="#bib93" id="back-bib93" data-db-target-for="bib93-d13420944e3903" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib93">Nik-Zainal et al., 2016</a><div data-db-target-of="bib93-d13420944e3903" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Nik-Zainal S. </li><li class="loa__item">Davies H. </li><li class="loa__item">Staaf J. </li><li class="loa__item">Ramakrishna M. </li><li class="loa__item">Glodzik D. </li><li class="loa__item">Zou X. </li><li class="loa__item">Martincorena I. </li><li class="loa__item">Alexandrov L.B. </li><li class="loa__item">Martin S. </li><li class="loa__item">Wedge D.C. </li><li>et al.</li></ul><div class="reference-citations__detials">Landscape of somatic mutations in 560 breast cancer whole-genome sequences.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">47-54</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature17676&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27135926&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84973594792&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1028)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=47-54&journal=Nature&author=Nik-Zainal+S.&author=Davies+H.&author=Staaf+J.&author=Ramakrishna+M.&author=Glodzik+D.&author=Zou+X.&author=Martincorena+I.&author=Alexandrov+L.B.&author=Martin+S.&author=Wedge+D.C.&title=Landscape+of+somatic+mutations+in+560+breast+cancer+whole-genome+sequences">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), with false discovery rates (q values) recalculated. Genes of q value &lt; 0.1 were declared significant.</div></section><section id="sec4.5.1.5" data-locator="sec4.5.1.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">De novo mutational signature extraction</span></h4><div class="section-paragraph">For results reported in <a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>A, non-negative matrix factorization algorithm (NMF) was used to decipher <em>de novo</em> mutation signatures in cancer somatic mutations stratified by 96 base substitutions in tri-nucleotide sequence contexts. To obtain a reliable signature profile, we used somaticwrapper to call mutations from WGS data (<a class="inter-ref-link" href="https://github.com/ding-lab/somaticwrapper" target="_blank">https://github.com/ding-lab/somaticwrapper</a>). SignatureAnalyzer exploited the Bayesian variant of the NMF algorithm and enabled an inference for the optimal number of signatures from the data itself at a balance between data fidelity (likelihood) and model complexity (regularization) (<div class="dropBlock reference-citations"><a href="#bib57" id="back-bib57" data-db-target-for="bib57-d13420944e3919" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib57">Kasar et al., 2015</a><div data-db-target-of="bib57-d13420944e3919" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kasar S. </li><li class="loa__item">Kim J. </li><li class="loa__item">Improgo R. </li><li class="loa__item">Tiao G. </li><li class="loa__item">Polak P. </li><li class="loa__item">Haradhvala N. </li><li class="loa__item">Lawrence M.S. </li><li class="loa__item">Kiezun A. </li><li class="loa__item">Fernandes S.M. </li><li class="loa__item">Bahl S. </li><li>et al.</li></ul><div class="reference-citations__detials">Whole-genome sequencing reveals activation-induced cytidine deaminase signatures during indolent chronic lymphocytic leukaemia evolution.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">8866</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fncomms9866&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26638776&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=8866&journal=Nat.+Commun.&author=Kasar+S.&author=Kim+J.&author=Improgo+R.&author=Tiao+G.&author=Polak+P.&author=Haradhvala+N.&author=Lawrence+M.S.&author=Kiezun+A.&author=Fernandes+S.M.&author=Bahl+S.&title=Whole-genome+sequencing+reveals+activation-induced+cytidine+deaminase+signatures+during+indolent+chronic+lymphocytic+leukaemia+evolution">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib60" id="back-bib60" data-db-target-for="bib60-d13420944e3922" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib60">Kim et al., 2016</a><div data-db-target-of="bib60-d13420944e3922" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kim J. </li><li class="loa__item">Mouw K.W. </li><li class="loa__item">Polak P. </li><li class="loa__item">Braunstein L.Z. </li><li class="loa__item">Kamburov A. </li><li class="loa__item">Kwiatkowski D.J. </li><li class="loa__item">Rosenberg J.E. </li><li class="loa__item">Van Allen E.M. </li><li class="loa__item">D’Andrea A. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">Somatic ERCC2 mutations are associated with a distinct genomic signature in urothelial tumors.<div class="popup-ref"><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">48</span></span>: <span class="ref__seriesPages">600-606</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fng.3557&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27111033&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84964308124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (189)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2016&pages=600-606&journal=Nat.+Genet.&author=Kim+J.&author=Mouw+K.W.&author=Polak+P.&author=Braunstein+L.Z.&author=Kamburov+A.&author=Kwiatkowski+D.J.&author=Rosenberg+J.E.&author=Van+Allen+E.M.&author=D%E2%80%99Andrea+A.&author=Getz+G.&title=Somatic+ERCC2+mutations+are+associated+with+a+distinct+genomic+signature+in+urothelial+tumors">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib124" id="back-bib124" data-db-target-for="bib124-d13420944e3926" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib124">Tan and Févotte, 2013</a><div data-db-target-of="bib124-d13420944e3926" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Tan V.Y.F. </li><li class="loa__item">Févotte C. </li></ul><div class="reference-citations__detials">Automatic relevance determination in nonnegative matrix factorization with the β-divergence.<div class="popup-ref"><span class="ref__series"><i>IEEE Trans. Pattern Anal. Mach. Intell.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">1592-1605</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1109%2FTPAMI.2012.240&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=23681989&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84878158173&amp;cf=" onclick="newWindow(this.href);return false">Scopus (150)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2013&pages=1592-1605&journal=IEEE+Trans.+Pattern+Anal.+Mach.+Intell.&author=Tan+V.Y.F.&author=F%C3%A9votte+C.&title=Automatic+relevance+determination+in+nonnegative+matrix+factorization+with+the+%CE%B2-divergence">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). After decomposing into signatures, the inferred signatures were compared against known signatures derived from COSMIC (<div class="dropBlock reference-citations"><a href="#bib125" id="back-bib125" data-db-target-for="bib125-d13420944e3929" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib125">Tate et al., 2019</a><div data-db-target-of="bib125-d13420944e3929" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Tate J.G. </li><li class="loa__item">Bamford S. </li><li class="loa__item">Jubb H.C. </li><li class="loa__item">Sondka Z. </li><li class="loa__item">Beare D.M. </li><li class="loa__item">Bindal N. </li><li class="loa__item">Boutselakis H. </li><li class="loa__item">Cole C.G. </li><li class="loa__item">Creatore C. </li><li class="loa__item">Dawson E. </li><li>et al.</li></ul><div class="reference-citations__detials">COSMIC: the Catalogue Of Somatic Mutations In Cancer.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">47</span></span>: <span class="ref__seriesPages">D941-D947</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgky1015&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30371878&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85057093998&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1040)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D941-D947&journal=Nucleic+Acids+Res.&author=Tate+J.G.&author=Bamford+S.&author=Jubb+H.C.&author=Sondka+Z.&author=Beare+D.M.&author=Bindal+N.&author=Boutselakis+H.&author=Cole+C.G.&author=Creatore+C.&author=Dawson+E.&title=COSMIC%3A+the+Catalogue+Of+Somatic+Mutations+In+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and cosine similarity was calculated to identify the best match.</div></section><section id="sec4.5.1.6" data-locator="sec4.5.1.6" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Mutational signature projection (used in <a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>B and <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">Figure S7</a>D)</span></h4><div class="section-paragraph">For results reported in <a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>B and <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">Figure S7</a>D, parallel approach based COSMIC signature scores for every sample were estimated using deconstructSigs (<div class="dropBlock reference-citations"><a href="#bib108" id="back-bib108" data-db-target-for="bib108-d13420944e3949" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib108">Rosenthal et al., 2016</a><div data-db-target-of="bib108-d13420944e3949" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Rosenthal R. </li><li class="loa__item">McGranahan N. </li><li class="loa__item">Herrero J. </li><li class="loa__item">Taylor B.S. </li><li class="loa__item">Swanton C. </li></ul><div class="reference-citations__detials">DeconstructSigs: delineating mutational processes in single tumors distinguishes DNA repair deficiencies and patterns of carcinoma evolution.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">31</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2Fs13059-016-0893-4&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26899170&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84959467752&amp;cf=" onclick="newWindow(this.href);return false">Scopus (462)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=31&journal=Genome+Biol.&author=Rosenthal+R.&author=McGranahan+N.&author=Herrero+J.&author=Taylor+B.S.&author=Swanton+C.&title=DeconstructSigs%3A+delineating+mutational+processes+in+single+tumors+distinguishes+DNA+repair+deficiencies+and+patterns+of+carcinoma+evolution">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) package in R. In addition to COSMIC signatures SBS 2 and 13, APOBEC enrichment was also assessed using TrinucleotideMatrix and plotApobecDiff functions of the maftool package (<div class="dropBlock reference-citations"><a href="#bib80" id="back-bib80" data-db-target-for="bib80-d13420944e3952" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib80">Mayakonda et al., 2018</a><div data-db-target-of="bib80-d13420944e3952" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mayakonda A. </li><li class="loa__item">Lin D.-C. </li><li class="loa__item">Assenov Y. </li><li class="loa__item">Plass C. </li><li class="loa__item">Koeffler H.P. </li></ul><div class="reference-citations__detials">Maftools: efficient and comprehensive analysis of somatic variants in cancer.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">1747-1756</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1101%2Fgr.239244.118&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30341162&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85055908064&amp;cf=" onclick="newWindow(this.href);return false">Scopus (519)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=1747-1756&journal=Genome+Res.&author=Mayakonda+A.&author=Lin+D.-C.&author=Assenov+Y.&author=Plass+C.&author=Koeffler+H.P.&title=Maftools%3A+efficient+and+comprehensive+analysis+of+somatic+variants+in+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). APOBEC enrichment scores greater than four were used to identify high confidence APOBEC-enriched cases.</div></section></section><section id="sec4.5.2" data-locator="sec4.5.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Proteomics data analysis</span></h4><section id="sec4.5.2.1" data-locator="sec4.5.2.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Spectrum quality filtering and database searching</span></h4><div class="section-paragraph">All MS data were interpreted using the Spectrum Mill software package v7.0 pre-release (Agilent Technologies, Santa Clara, CA) co-developed by Karl Clauser of the Carr laboratory (<a class="inter-ref-link" href="https://www.broadinstitute.org/proteomics" target="_blank">https://www.broadinstitute.org/proteomics</a>). Similar MS/MS spectra acquired on the same precursor m/z within ± 40 s were merged. MS/MS spectra were excluded from searching if they failed the quality filter by not having a sequence tag length &gt; 0 (i.e., minimum of two masses separated by the in-chain mass of an amino acid) or did not have a precursor MH+ in the range of 800-6000. MS/MS spectra were searched against a RefSeq-based sequence database containing 37,579 proteins mapped to the human reference genome (hg19) obtained via the UCSC Table Browser (<a class="inter-ref-link" href="https://genome.ucsc.edu/cgi-bin/hgTables" target="_blank">https://genome.ucsc.edu/cgi-bin/hgTables</a>) on September 14, 2016, with the addition of 13 proteins encoded in the human mitochondrial genome, 150 common laboratory contaminant proteins, and 553 non-canonical small open reading frames (38,295 total sequences). Scoring parameters were ESI-QEXACTIVE-HCD-v2, for whole proteome datasets, and ESI-QEXACTIVE-HCD-v3, for phosphoproteome datasets. All spectra were allowed ± 20 ppm mass tolerance for precursor and product ions, 30% minimum matched peak intensity, and “trypsin allow P” enzyme specificity with up to 4 missed cleavages. Allowed fixed modifications included carbamidomethylation of cysteine and selenocysteine. TMT labeling was required at lysine, but peptide N-termini were allowed to be either labeled or unlabeled. Allowed variable modifications for whole proteome datasets were acetylation of protein N-termini, oxidized methionine, deamidation of asparagine, hydroxylation of proline in PG motifs, pyro-glutamic acid at peptide N-terminal glutamine, and pyro-carbamidomethylation at peptide N-terminal cysteine with a precursor MH+ shift range of −18 to 97 Da. For the phosphoproteome dataset the allowed variable modifications were revised to allow phosphorylation of serine, threonine, and tyrosine, allow deamidation only in NG motifs, and disallow hydroxylation of proline with a precursor MH+ shift range of −18 to 272 Da. For the acetylproteome dataset the allowed variable modifications were revised to allow acetylation of lysine, allow deamidation only in NG motifs, and disallow hydroxylation of proline with a precursor MH+ shift range of −400 to 70 Da.</div></section><section id="sec4.5.2.2" data-locator="sec4.5.2.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">PSM quality control</span></h4><div class="section-paragraph">Identities interpreted for individual spectra were automatically designated as confidently assigned using the Spectrum Mill autovalidation module to use target-decoy-based false discovery rate (FDR) estimates to apply score threshold criteria. For the whole proteome dataset, thresholding was done in 3 steps: at the peptide spectrum match (PSM) level, the protein level for each TMT-plex, and the protein level for all 17 TMT-plexes. For the phosphoproteome and acetylproteome datasets, thresholding was done in two steps: at the PSM and variable modification (VM) site levels.</div><div class="section-paragraph">In step 1 for all datasets, PSM-level autovalidation was done first and separately for each TMT-plex experiment consisting of either 25 LC-MS/MS runs (whole proteome), 13 LC-MS/MS runs (phosphoproteome), or 6 LC-MS/MS runs (acetylproteome), using an auto-thresholds strategy with a minimum sequence length of 7; automatic variable range precursor mass filtering; and score and delta Rank1 – Rank2 score thresholds optimized to yield a PSM-level FDR estimate for precursor charges 2 through 4 of &lt; 0.6% for each precursor charge state in each LC-MS/MS run. To achieve reasonable statistics for precursor charges 5-6, thresholds were optimized to yield a PSM-level FDR estimate of &lt; 0.3% across all runs per TMT-plex experiment (instead of per each run), since many fewer spectra are generated for the higher charge states.</div><div class="section-paragraph">In step 2 for the whole proteome dataset, protein-polishing autovalidation was applied separately to each TMTplex experiment to further filter the PSMs using a target protein-level FDR threshold of zero. The primary goal of this step was to eliminate peptides identified with low scoring PSMs that represent proteins identified by a single peptide, so-called “one-hit wonders.” After assembling protein groups from the autovalidated PSMs, protein polishing determined the maximum protein level score of a protein group that consisted entirely of distinct peptides estimated to be false-positive identifications (PSMs with negative delta forward-reverse scores). PSMs were removed from the set obtained in the initial peptide-level autovalidation step if they contributed to protein groups that had protein scores below the maximum false-positive protein score. Step 3 was then applied, consisting of protein-polishing autovalidation across all TMT plexes together using the protein grouping method “expand subgroups, top uses shared” to retain protein subgroups with either a minimum protein score of 25 or observation in at least 2 TMT plexes. The primary goal of this step was to eliminate low-scoring proteins that were infrequently detected in the sample cohort. As a consequence of these two protein-polishing steps, each identified protein reported in the study comprised multiple peptides, unless a single excellent scoring peptide was the sole match and that peptide was observed in at least 2 TMT-plexes. In calculating scores at the protein level and reporting the identified proteins, peptide redundancy was addressed in Spectrum Mill as follows: The protein score was the sum of the scores of distinct peptides. A distinct peptide was the single highest scoring instance of a peptide detected through an MS/MS spectrum. MS/MS spectra for a particular peptide may have been recorded multiple times (e.g., as different precursor charge states, in adjacent bRP fractions, modified by deamidation at asparagine or oxidation of methionine, or with different phosphosite localization), but were still counted as a single distinct peptide. When a peptide sequence of &gt; 8 residues was contained in multiple protein entries in the sequence database, the proteins were grouped together and the highest scoring one and its accession number were reported. In some cases when the protein sequences were grouped in this manner, there were distinct peptides that uniquely represent a lower scoring member of the group (isoforms, family members, and different species). Each of these instances spawned a subgroup. Multiple subgroups were reported, counted toward the total number of proteins, and were given related protein subgroup numbers (e.g., 3.1 and 3.2 for group 3, subgroups 1 and 2). For the whole proteome datasets the above criteria yielded false discovery rates (FDR) for each TMT-plex experiment of &lt; 0.5% at the peptide-spectrum match level and &lt; 0.6% at the distinct peptide level. After assembling proteins with all the PSMs from all the TMT-plex experiments together, the aggregate FDR estimates were 0.41% at the peptide-spectrum match level, 1.6% at the distinct peptide level, and &lt; 0.01% (1/10,633) at the protein group level. Since the protein-level FDR estimate neither explicitly required a minimum number of distinct peptides per protein nor adjusted for the number of possible tryptic peptides per protein, it may underestimate false positive protein identifications for large proteins observed only on the basis of multiple low scoring PSMs.</div><div class="section-paragraph">In step 2 for the phosphoproteome and acetylproteome datasets, variable modification (VM) site polishing autovalidation was applied across all 17 TMT plexes to retain all VM-site identifications with either a minimum id score of 8.0 or observation in at least 3 TMT plexes. The intention of the VM-site polishing step is to control FDR by eliminating unreliable VM site-level identifications, particularly low-scoring VM sites that are only detected as low-scoring peptides that are also infrequently detected across all of the TMT plexes in the study. In calculating scores at the VM site level and reporting the identified VM sites, redundancy was addressed in Spectrum Mill as follows: A VM site table was assembled with columns for individual TMT-plex experiments and rows for individual VM sites. PSMs were combined into a single row for all non-conflicting observations of a particular VM site (e.g., different missed cleavage forms, different precursor charges, confident and ambiguous localizations, and different sample-handling modifications). For related peptides, neither observations with a different number of VM sites nor different confident localizations were allowed to be combined. Selecting the representative peptide from the combined observations was done such that once confident VM site localization was established, higher identification scores and longer peptide lengths were preferred. While a Spectrum Mill identification score was based on the number of matching peaks, their ion type assignment, and the relative height of unmatched peaks, the VM site localization score was the difference in identification score between the top two localizations. The score threshold for confident localization, &gt; 1.1, essentially corresponded to at least 1 b or y ion located between two candidate sites that had a peak height &gt; 10% of the tallest fragment ion (neutral losses of phosphate from the precursor and related ions as well as immonium and TMT reporter ions were excluded from the relative height calculation). The ion type scores for b-H<sub>3</sub>PO<sub>4</sub>, y-H<sub>3</sub>PO<sub>4</sub>, b-H<sub>2</sub>O, and y-H<sub>2</sub>O ion types were all set to 0.5. This prevented inappropriate confident localization assignment when a spectrum lacked primary b or y ions between two possible sites but contained ions that could be assigned as either phosphate-loss ions for one localization or water-loss ions for another localization. VM site polishing yielded 63,416 phosphosites with an aggregate FDR of 0.44% at the phosphosite level. In aggregate, 70% of the reported phosphosites in this study were fully localized to a particular serine, threonine, or tyrosine residue. VM site polishing yielded 18,392 acetylsites with an aggregate FDR of 0.57% at the acetylsite level. In aggregate, 99% of the reported acetylsites in this study were fully localized to a particular lysine residue. The overall peptide identifications enabled calculation of enrichment rates (modified peptides/all peptides) for phosphopeptides (by IMAC) and acetylpeptides (by anti-acetyl-Lysine antibodies). Phospho-STY enrichment rates for each plex ranged from 88%–97% (plex 2 was an outlier at 71%). Acetyl-K enrichment rates for each plex ranged from 45%–69% (plex 11 was an outlier at 24%).</div></section><section id="sec4.5.2.3" data-locator="sec4.5.2.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Quantification using TMT ratios</span></h4><div class="section-paragraph">Using the Spectrum Mill Protein/Peptide Summary module, a protein comparison report was generated for the proteome dataset using the protein grouping method “expand subgroups, top uses shared” (SGT). For the phosphoproteome and acetylproteome datasets, a Variable Modification site comparison report limited to either phospho or acetyl sites, respectively, was generated using the protein grouping method “unexpand subgroups.” Relative abundances of proteins and VM sites were determined in Spectrum Mill using TMT reporter ion intensity ratios from each PSM. TMT reporter ion intensities were corrected for isotopic impurities in the Spectrum Mill Protein/Peptide summary module using the afRICA correction method, which implements determinant calculations according to Cramer’s Rule (<div class="dropBlock reference-citations"><a href="#bib115" id="back-bib115" data-db-target-for="bib115-d13420944e4004" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib115">Shadforth et al., 2005</a><div data-db-target-of="bib115-d13420944e4004" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Shadforth I.P. </li><li class="loa__item">Dunkley T.P.J. </li><li class="loa__item">Lilley K.S. </li><li class="loa__item">Bessant C. </li></ul><div class="reference-citations__detials">i-Tracker: for quantitative proteomics using iTRAQ.<div class="popup-ref"><span class="ref__series"><i>BMC Genomics.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">145</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2F1471-2164-6-145&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=16242023&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-27744450286&amp;cf=" onclick="newWindow(this.href);return false">Scopus (220)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=145&journal=BMC+Genomics&author=Shadforth+I.P.&author=Dunkley+T.P.J.&author=Lilley+K.S.&author=Bessant+C.&title=i-Tracker%3A+for+quantitative+proteomics+using+iTRAQ">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and correction factors obtained from the reagent manufacturer’s certificate of analysis (<a class="inter-ref-link" href="https://www.thermofisher.com/order/catalog/product/90406" target="_blank">https://www.thermofisher.com/order/catalog/product/90406</a>) for TMT10 lot number QK226692A. A protein-level, phosphosite-level, or acetylsite-level TMT ratio was calculated as the median of all PSM-level ratios contributing to a protein subgroup, phosphosite, or acetylsite. PSMs were excluded from the calculation if they lacked a TMT label, had a precursor ion purity &lt; 50% (MS/MS has significant precursor isolation contamination from co-eluting peptides), or had a negative delta forward-reverse identification score (half of all false-positive identifications). Lack of TMT label led to exclusion of PSMs per TMT plex with a range of 1.8 to 3.1% for the proteome, 1.2 to 3.8% for the phosphoproteome, and 1.5 to 5.2% (outliers were plex 11 at 37% and plex 14 at 8.6%) for the acetylproteome datasets. Low precursor ion purity led to exclusion of PSMs per TMT plex with a range of 3.7 to 6.4% for the proteome, 2.1 to 2.9% for the phosphoproteome, and 3.0 to 6.5% for the acetylproteome datasets.</div></section><section id="sec4.5.2.4" data-locator="sec4.5.2.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Two-component normalization of TMT ratio</span></h4><div class="section-paragraph">It was assumed that for every sample there would be a set of unregulated proteins or phosphosites that had abundance comparable to the common reference (CR) sample. In the normalized sample, these proteins, phosphosites, or acetylsites should have a log TMT ratio centered at zero. In addition, there were proteins, phosphosites, and acetylsites that were either up- or downregulated compared to the CR. A normalization scheme was employed that attempted to identify the unregulated proteins and phosphosites, and centered the distribution of these log-ratios around zero in order to nullify the effect of differential protein loading and/or systematic MS variation. A 2-component Gaussian mixture model-based normalization algorithm was used to achieve this effect. The two Gaussians (<em>μi</em>1,1) and <em>N</em>(<em>μi</em>2,<em>σi</em>2) for a sample <em>i</em> were fitted and used in the normalization process as follows: the mode <em>mi</em> of the log-ratio distribution was determined for each sample using kernel density estimation with a Gaussian kernel and Shafer-Jones bandwidth. A two-component Gaussian mixture model was then fit with the mean of <em>both</em> Gaussians constrained to be <em>mi</em>, i.e., <em>μi</em>1 = <em>μi</em>2 = <em>mi</em>. The Gaussian with the smaller estimated standard deviation <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si1.gif"><mrow><mi>σ</mi><mi>i</mi><mo linebreak="goodbreak" linebreakstyle="after">=</mo><mi>min</mi><mrow><mo>(</mo><mrow><mover accent="true"><mi>σ</mi><mo>ˆ</mo></mover><mn>1</mn><mi>i</mi><mo>,</mo><mover accent="true"><mi>σ</mi><mo>ˆ</mo></mover><mn>2</mn><mi>i</mi></mrow><mo>)</mo></mrow></mrow></math> was assumed to represent the unregulated component of proteins/phosphosites/acetylsites, and was used to normalize the sample. The sample was standardized using (<em>mi</em>,) by subtracting the mean <em>mi</em> from each protein/phosphosite/acetylsite and dividing by the standard deviation <em>σi</em>.</div></section><section id="sec4.5.2.5" data-locator="sec4.5.2.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Identification of patient-specific single amino acid variants, indels, and spliceforms</span></h4><div class="section-paragraph">For each of the 122 patients’ tumors analyzed in this study, whole exome DNA sequencing and Illumina RNA-seq data generated from aliquots of the cryopulverized tumors and accompanying germline DNA samples were obtained under controlled access. Tumor-specific somatic DNA-variant calls and germline DNA-variant calls from the same individual, and splice junctions predicted from RNA-seq assemblies were generated as described above (Genomic Data Analysis). The proteogenomic database tool QUILTS v3.0 (<a class="inter-ref-link" href="http://openslice.fenyolab.org/cgi-bin/pyquilts_cgi.pl" target="_blank">http://openslice.fenyolab.org/cgi-bin/pyquilts_cgi.pl</a>) (<div class="dropBlock reference-citations"><a href="#bib110" id="back-bib110" data-db-target-for="bib110-d13420944e4090" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib110">Ruggles et al., 2015</a><div data-db-target-of="bib110-d13420944e4090" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Tang Z. </li><li class="loa__item">Wang X. </li><li class="loa__item">Grover H. </li><li class="loa__item">Askenazi M. </li><li class="loa__item">Teubl J. </li><li class="loa__item">Cao S. </li><li class="loa__item">McLellan M.D. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Tabb D.L. </li><li>et al.</li></ul><div class="reference-citations__detials">An analysis of the sensitivity of proteogenomic mapping of somatic mutations and novel splicing events in cancer.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">1060-1071</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.M115.056226&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.M115.056226&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.M115.056226&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26631509&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84962585103&amp;cf=" onclick="newWindow(this.href);return false">Scopus (56)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=1060-1071&journal=Mol.+Cell.+Proteomics&author=Ruggles+K.V.&author=Tang+Z.&author=Wang+X.&author=Grover+H.&author=Askenazi+M.&author=Teubl+J.&author=Cao+S.&author=McLellan+M.D.&author=Clauser+K.R.&author=Tabb+D.L.&title=An+analysis+of+the+sensitivity+of+proteogenomic+mapping+of+somatic+mutations+and+novel+splicing+events+in+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) was used to incorporate the germline and somatic non-synonymous single nucleotide variant calls (SNVs), indels, RNA-seq predicted splice junctions and gene fusions into a protein sequence database for each patient. The human RefSeq protein database (version 20160914) was used as a reference for the hg19 proteome and genome. QUILTS was run with the following thresholds for number of RNA-seq reads supporting splice junctions: both exon boundaries annotated (2), left boundary annotated (3), and no boundaries annotated (3).</div><div class="section-paragraph">The QUILTS personalized databases for each patient were merged for searching the MS/MS spectra to accommodate the multiplexed samples used in LC-MS/MS data generation. Since each of the 15 plexes of TMT10 labeled tumor samples was prepared by combining 9 individual tumor samples plus an aliquot of common reference (which was a mixture of 40 tumors), each MS/MS spectrum could be derived from a peptide sequence shared by up to 49 individual tumors. One combined sequence database was made by concatenating the QUILTS-generated 122 individual FASTA files. When concatenating, variant and spliceform summary files for the whole exome and RNA-seg derived information, respectively, were generated with the Spectrum Mill Protein Databases utilities to enable subsequent matching of individual tumors to sequence identifiers and positions of genomic features. Completely novel junctions, with both boundaries matching no known exons, were omitted. The concatenated file was made non-redundant by removing repeat entries with identical full-length sequences. Protein sequences with length &lt; 7 amino acids were also removed. The resulting non-redundant, patient-specific protein sequence database containing somatic and germline single amino acid variants (179,768 sequences), spliceforms (283,149 sequences), indels (11,586 sequences), and gene fusions (1601 sequences), was concatenated together with the human reference database, RefSeq version 20160914 (38,281 sequences), to yield the database (514,385 total sequences) used for searches with MS/MS spectra.</div><div class="section-paragraph">MS/MS spectra from the whole proteome datasets were searched in two stages: 1) all spectra against the RefSeq reference database, as described above, then 2) the remaining unidentified spectra against the patient-specific sequence database as described here. This was done to control the false-discovery rate since there are several orders of magnitude fewer high confidence PSM’s expected to the patient-specific sequences not present in the reference database. Search parameters other than the database were the same as the stage 1 searches.</div><div class="section-paragraph">Separate PG event tables for the two primary PG event types, variants and spliceforms (including indels and frameshifts), were assembled with columns for individual iTRAQ 4-plex experiments and rows for individual PG events. PSM’s with a minimum identification score of 8.0 were combined into a single row for all non-conflicting observations of a particular PG event (i.e., multiple peptides containing altered coding sequence due to a frameshift, different trypsin missed cleavage forms of peptides that span a splice junction or contain an SAAV or new protein C terminus resulting from introduction of a novel stop codon, different precursor charges, different sample handling modifications of the same peptide, and repeat observations in adjacent bRP fractions). The representative peptide reported from the combined observations is the one with the highest identification score. A polishing step was manually applied to each table to further filter the PG events to reach a suitable PG-event level identification FDR. The following thresholds were applied to the representative peptide of each PG event: delta Rank1 – Rank2 score &gt; 1.0, minimum sequence length &gt; 7 (variants), &gt; 8 (spliceforms). Lower-scoring, infrequently observed spliceforms were further filtered to exclude those with both a score &lt; 9.3 and detection in &lt; 3 TMT10 plexes. Consequently, the final PG event-level cumulative FDR estimates were variants (1.0%), and spliceforms (1.1%).</div><div class="section-paragraph">Relative abundances of each PG event in a patient sample were determined in Spectrum Mill using TMT reporter ion intensity ratios from each PSM. A PG event-level TMT ratio was calculated as the median of all PSM level ratios contributing to each event remaining after excluding those PSM’s lacking a TMT label, having a negative delta forward-reverse score (half of all false-positive identifications), or having a precursor ion purity &lt; 50% (MS/MS has significant precursor isolation contamination from co-eluting peptides). The ratios for all PG events for a patient were then standardized by subtracting the centering factor and dividing by the scaling factor of the protein-level TMT ratios for that patient derived from the results of the stage 1 search (reference database only). Since each MS/MS spectrum has 10 TMT reporter ions for 9 patients and the common control (40 patients), the detection of a rare PG event can typically be attributed to a specific patient when 1 ratio is significantly higher than the other 9.</div></section></section><section id="sec4.5.3" data-locator="sec4.5.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Systems biology analysis</span></h4><section id="sec4.5.3.1" data-locator="sec4.5.3.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Sample exclusion</span></h4><div class="section-paragraph">Typical TMT LC-MS/MS experiments exhibit TMT log<sub>2</sub> ratios (individual sample / common reference) with the median ratio value of all peptides from a sample being approximately constant across the LC retention time range of each LC-MS/MS run in an experiment. However, for all 18 normal adjacent samples (constituting plexes 14 and 15), the median ratios steadily rose by 1 to 2 log<sub>2</sub> units from the beginning to the end of the LC gradient in all fractions of all data types (proteome, phosphoproteome, and acetylome). This behavior would be consistent with elution failure from a desalting step prior to fractionation. Consequently, the resulting quantitative data for 18 normal adjacent samples were considered to not meet quality standards for inclusion in subsequent analyses. Notably, all QC-fail samples in our prior study (<div class="dropBlock reference-citations"><a href="#bib85" id="back-bib85" data-db-target-for="bib85-d13420944e4115" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib85">Mertins et al., 2016</a><div data-db-target-of="bib85-d13420944e4115" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Wang P. </li><li class="loa__item">Wang X. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Cao S. </li><li class="loa__item">Petralia F. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomics connects somatic mutations to signalling in breast cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (780)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=NCI+CPTAC&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) exhibited this chromatographic behavior, though it was not appreciated at the time.</div></section><section id="sec4.5.3.2" data-locator="sec4.5.3.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Dataset filtering</span></h4><div class="section-paragraph">Proteins (global proteome), phosphosites and acetylsites present in fewer than 30% of samples (i.e., missing in &gt; 70% of samples) were removed from the respective datasets. Furthermore:<ul class="ce-list--remove-bullets" id="olist0015"><li class="ce-list--remove-bullets__list-item" id="o0060"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">Proteins were required to have at least two observed TMT ratios in &gt; 25% of samples in order to be included in the proteome dataset. Phosphosites and acetylsites were required to have at least one observed TMT ratio in &gt; 25% of samples.</div></li><li class="ce-list--remove-bullets__list-item" id="o0065"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">Proteins, phosphosites and acetylsites were required to have TMT ratios with an overall standard deviation &gt; 0.5 across all the samples where they were observed. This ensured that a small number of proteins, phosphosites and acetylsites that did not vary much over the set of samples were excluded to minimize noise.</div></li><li class="ce-list--remove-bullets__list-item" id="o0070"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">Replicate samples in the dataset were merged by taking the mean of the respective expression values or ratios.</div></li></ul></div><div class="section-paragraph">Some of the filtering steps were modified for specific analyses in the study. For many of the marker selection and gene set enrichment analyses, at least 50% of samples were required to have non-missing values for proteins/phosphosites/acetylsites, since missing values were imputed, and excessive missing values can result in poor imputation. Alternate filtering has been noted in descriptions of the relevant methods and are summarized <a href="/cms/10.1016/j.cell.2020.10.036/attachment/a1bc3f3e-eeac-4605-a9ef-839abd334dcd/mmc2" class="scroll-into-link">Table S2</a>A.</div></section><section id="sec4.5.3.3" data-locator="sec4.5.3.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">CNA-driven cis and trans effects</span></h4><div class="section-paragraph">Correlations between copy number alterations (CNA) and RNA, proteome, phosphoproteome and acetylproteome (with proteome and PTM data mapped to genes, by choosing the most variable protein isoform/PTM site as the gene-level representative) were determined using Pearson correlation of common genes present in CNA-RNA-proteome (8,668 genes), CNA-RNA-phosphoproteome (4,534 genes) and CNA-RNA-acetylproteome (1,604 genes). In addition, <em>p</em>-values (corrected for multiple testing using Benjamini-Hochberg FDR; <div class="dropBlock reference-citations"><a href="#bib14" id="back-bib14" data-db-target-for="bib14-d13420944e4148" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib14">Benjamini and Hochberg, 1995</a><div data-db-target-of="bib14-d13420944e4148" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Benjamini Y. </li><li class="loa__item">Hochberg Y. </li></ul><div class="reference-citations__detials">Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing.<div class="popup-ref"><span class="ref__series"><i>J. R. Stat. Soc. Series B Methodol.</i> <span class="ref__seriesDate">1995; </span><span class="ref__seriesVolume">57</span></span>: <span class="ref__seriesPages">289-300</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1111%2Fj.2517-6161.1995.tb02031.x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1995&pages=289-300&journal=J.%C2%A0R.+Stat.+Soc.+Series+B+Methodol.&author=Benjamini+Y.&author=Hochberg+Y.&title=Controlling+the+False+Discovery+Rate%3A+A+Practical+and+Powerful+Approach+to+Multiple+Testing">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) for assessing the statistical significance of the correlation values were also calculated. CNA <em>trans</em>-effects for a given gene were determined by identifying genes with statistically significant (FDR &lt; 0.05) positive or negative correlations.</div></section><section id="sec4.5.3.4" data-locator="sec4.5.3.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">CMAP analysis</span></h4><div class="section-paragraph">Candidate genes driving response to copy number alterations were identified using large-scale Connectivity Map (CMAP) queries. The CMAP (<div class="dropBlock reference-citations"><a href="#bib63" id="back-bib63" data-db-target-for="bib63-d13420944e4158" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib63">Lamb et al., 2006</a><div data-db-target-of="bib63-d13420944e4158" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lamb J. </li><li class="loa__item">Crawford E.D. </li><li class="loa__item">Peck D. </li><li class="loa__item">Modell J.W. </li><li class="loa__item">Blat I.C. </li><li class="loa__item">Wrobel M.J. </li><li class="loa__item">Lerner J. </li><li class="loa__item">Brunet J.-P. </li><li class="loa__item">Subramanian A. </li><li class="loa__item">Ross K.N. </li><li>et al.</li></ul><div class="reference-citations__detials">The Connectivity Map: using gene-expression signatures to connect small molecules, genes, and disease.<div class="popup-ref"><span class="ref__series"><i>Science.</i> <span class="ref__seriesDate">2006; </span><span class="ref__seriesVolume">313</span></span>: <span class="ref__seriesPages">1929-1935</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1126%2Fscience.1132939&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=17008526&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-33749335282&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3071)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=313&publication_year=2006&pages=1929-1935&journal=Science&author=Lamb+J.&author=Crawford+E.D.&author=Peck+D.&author=Modell+J.W.&author=Blat+I.C.&author=Wrobel+M.J.&author=Lerner+J.&author=Brunet+J.-P.&author=Subramanian+A.&author=Ross+K.N.&title=The+Connectivity+Map%3A+using+gene-expression+signatures+to+connect+small+molecules%2C+genes%2C+and+disease">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib122" id="back-bib122" data-db-target-for="bib122-d13420944e4161" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib122">Subramanian et al., 2017</a><div data-db-target-of="bib122-d13420944e4161" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Subramanian A. </li><li class="loa__item">Narayan R. </li><li class="loa__item">Corsello S.M. </li><li class="loa__item">Peck D.D. </li><li class="loa__item">Natoli T.E. </li><li class="loa__item">Lu X. </li><li class="loa__item">Gould J. </li><li class="loa__item">Davis J.F. </li><li class="loa__item">Tubelli A.A. </li><li class="loa__item">Asiedu J.K. </li><li>et al.</li></ul><div class="reference-citations__detials">A Next Generation Connectivity Map: L1000 Platform and the First 1,000,000 Profiles.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">171</span></span>: <span class="ref__seriesPages">1437-1452.e17</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29195078&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85036572148&amp;cf=" onclick="newWindow(this.href);return false">Scopus (772)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2017&pages=1437-1452.e17&journal=Cell&author=Subramanian+A.&author=Narayan+R.&author=Corsello+S.M.&author=Peck+D.D.&author=Natoli+T.E.&author=Lu+X.&author=Gould+J.&author=Davis+J.F.&author=Tubelli+A.A.&author=Asiedu+J.K.&title=A+Next+Generation+Connectivity+Map%3A+L1000+Platform+and+the+First+1%2C000%2C000+Profiles">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) is a collection of about 1.3 million gene expression profiles from cell lines treated with bioactive small molecules (∼20,000 drug perturbagens), shRNA gene knockdowns (∼4,300) and ectopic expression of genes. The CMAP dataset is available on GEO (Series GSE92742). For this analysis, we use the Level 5 (signatures from aggregating replicates) TouchStone dataset with 473,647 total profiles, containing 36,720 gene knock-down profiles, with measurements for 12,328 genes. See <a class="inter-ref-link" href="https://clue.io/GEO-guide" target="_blank">https://clue.io/GEO-guide</a> for more information.</div><div class="section-paragraph">To identify candidate driver genes, proteome profiles of copy number-altered samples were correlated with gene knockdown mRNA profiles in the above CMAP dataset, and enrichment of up/downregulated genes was evaluated. Normalized log2 copy number values less than −0.3 defined deletion (loss), and values greater than +0.3 defined copy number amplifications (gains). In the copy number-altered samples (separately for CNA amplification and CNA deletion), the <em>trans</em>-genes (identified by significant correlation in “CNA driven <em>cis</em> and <em>trans</em> effects” above) were grouped into UP and DOWN categories by comparing the protein ratios of these genes to their ratios in the copy number neutral samples (normalized log2 copy number between −0.3 and +0.3). The lists of UP and DOWN <em>trans</em>-genes were then used as queries to interrogate CMAP signatures and calculate weighted connectivity scores (WTCS) using the single-sample GSEA algorithm (<div class="dropBlock reference-citations"><a href="#bib62" id="back-bib62" data-db-target-for="bib62-d13420944e4177" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib62">Krug et al., 2019</a><div data-db-target-of="bib62-d13420944e4177" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Krug K. </li><li class="loa__item">Mertins P. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Hornbeck P. </li><li class="loa__item">Raju R. </li><li class="loa__item">Ahmad R. </li><li class="loa__item">Szucs M. </li><li class="loa__item">Mundt F. </li><li class="loa__item">Forestier D. </li><li class="loa__item">Jane-Valbuena J. </li><li>et al.</li></ul><div class="reference-citations__detials">A Curated Resource for Phosphosite-specific Signature Analysis.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">576-593</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30563849&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85062992827&amp;cf=" onclick="newWindow(this.href);return false">Scopus (42)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=576-593&journal=Mol.+Cell.+Proteomics&author=Krug+K.&author=Mertins+P.&author=Zhang+B.&author=Hornbeck+P.&author=Raju+R.&author=Ahmad+R.&author=Szucs+M.&author=Mundt+F.&author=Forestier+D.&author=Jane-Valbuena+J.&title=A+Curated+Resource+for+Phosphosite-specific+Signature+Analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The weighted connectivity scores were then normalized for each perturbation type and cell line to obtain normalized connectivity scores (NCS). See (<div class="dropBlock reference-citations"><a href="#bib122" id="back-bib122" data-db-target-for="bib122-d13420944e4181" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib122">Subramanian et al., 2017</a><div data-db-target-of="bib122-d13420944e4181" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Subramanian A. </li><li class="loa__item">Narayan R. </li><li class="loa__item">Corsello S.M. </li><li class="loa__item">Peck D.D. </li><li class="loa__item">Natoli T.E. </li><li class="loa__item">Lu X. </li><li class="loa__item">Gould J. </li><li class="loa__item">Davis J.F. </li><li class="loa__item">Tubelli A.A. </li><li class="loa__item">Asiedu J.K. </li><li>et al.</li></ul><div class="reference-citations__detials">A Next Generation Connectivity Map: L1000 Platform and the First 1,000,000 Profiles.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">171</span></span>: <span class="ref__seriesPages">1437-1452.e17</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29195078&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85036572148&amp;cf=" onclick="newWindow(this.href);return false">Scopus (772)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2017&pages=1437-1452.e17&journal=Cell&author=Subramanian+A.&author=Narayan+R.&author=Corsello+S.M.&author=Peck+D.D.&author=Natoli+T.E.&author=Lu+X.&author=Gould+J.&author=Davis+J.F.&author=Tubelli+A.A.&author=Asiedu+J.K.&title=A+Next+Generation+Connectivity+Map%3A+L1000+Platform+and+the+First+1%2C000%2C000+Profiles">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) for details on WTCS and NCS. For each query we then identified outlier NCS scores, where a score was considered an outlier if it fell beyond 1.5 times the interquartile range of score distribution for the query. The query gene was a candidate driver if (i) the score outliers were statistically <em>cis</em>-enriched (Fisher test with BH-FDR multiple testing correction) and (ii) the gene had statistically significant and positive <em>cis</em>-correlation.</div><div class="section-paragraph">For a gene to be considered for inclusion in a CMAP query it needed to i) have a copy number change (amplification or deletion) in at least 15 samples; ii) have at least 20 significant <em>trans</em> genes; and iii) be on the list of shRNA knockdowns in the CMAP. Of the genes satisfying these conditions, the top 501 genes (sorted based on the number of <em>trans</em>-events) were used for the analysis, and resulted in 910 queries (CNA amplification and deletion combined) that were tested for enrichment. 21 candidate driver genes were identified with Fisher test FDR &lt; 0.26 using this process.</div><div class="section-paragraph">In order to ensure that the identified candidate driver genes were not a random occurrence, we performed a permutation test to determine how many candidate driver genes would be identified with random input (<div class="dropBlock reference-citations"><a href="#bib85" id="back-bib85" data-db-target-for="bib85-d13420944e4196" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib85">Mertins et al., 2016</a><div data-db-target-of="bib85-d13420944e4196" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Wang P. </li><li class="loa__item">Wang X. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Cao S. </li><li class="loa__item">Petralia F. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomics connects somatic mutations to signalling in breast cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (780)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=NCI+CPTAC&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). For the 910 queries used, we substituted the bona-fide trans-genes with randomly chosen genes, and repeated the CMAP enrichment process. To determine FDR, each permutation run was treated as a Poisson sample with rate λ, counting the number of identified candidate driver genes. Given the small <em>n</em> ( = 10) and λ, a Score confidence interval was calculated (<div class="dropBlock reference-citations"><a href="#bib11" id="back-bib11" data-db-target-for="bib11-d13420944e4201" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib11">Barker, 2002</a><div data-db-target-of="bib11-d13420944e4201" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Barker L. </li></ul><div class="reference-citations__detials">A comparison of nine confidence intervals for a Poisson parameter when the expected number of events is ≤ 5.<div class="popup-ref"><span class="ref__series"><i>Am. Stat.</i> <span class="ref__seriesDate">2002; </span><span class="ref__seriesVolume">56</span></span>: <span class="ref__seriesPages">85-89</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1198%2F000313002317572736&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-0035995688&amp;cf=" onclick="newWindow(this.href);return false">Scopus (35)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2002&pages=85-89&journal=Am.+Stat.&author=Barker+L.&title=A+comparison+of+nine+confidence+intervals+for+a+Poisson+parameter+when+the+expected+number+of+events+is+%E2%89%A4+5">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and the midpoint of the confidence interval used to estimate the expected number of false positives. Using 10 random permutations, we determined the overall false discovery rate to be FDR = 0.26, with a 95% CI of (0.19, 0.32).</div><div class="section-paragraph">To identify how many trans-correlated genes for all candidate regulatory genes could be directly explained by gene expression changes measured in the CMAP shRNA perturbation experiments, knockdown gene expression consensus signature z-scores (knockdown/control) were used to identify regulated genes with α = 0.05, followed by counting the number of trans-genes in this list of regulated genes.</div><div class="section-paragraph">To obtain biological insight into the list of candidate driver genes, we performed (i) enrichment analysis on samples with extreme CNA values (amplification or deletion) to identify statistically enriched sample annotation subgroups; and (ii) GSEA on <em>cis</em>/trans-correlation values to find enriched pathways.</div><div class="section-paragraph">Note that the connectivity score calculation described above, and the underlying CMAP data, was based on a recent publication (<div class="dropBlock reference-citations"><a href="#bib63" id="back-bib63" data-db-target-for="bib63-d13420944e4212" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib63">Lamb et al., 2006</a><div data-db-target-of="bib63-d13420944e4212" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lamb J. </li><li class="loa__item">Crawford E.D. </li><li class="loa__item">Peck D. </li><li class="loa__item">Modell J.W. </li><li class="loa__item">Blat I.C. </li><li class="loa__item">Wrobel M.J. </li><li class="loa__item">Lerner J. </li><li class="loa__item">Brunet J.-P. </li><li class="loa__item">Subramanian A. </li><li class="loa__item">Ross K.N. </li><li>et al.</li></ul><div class="reference-citations__detials">The Connectivity Map: using gene-expression signatures to connect small molecules, genes, and disease.<div class="popup-ref"><span class="ref__series"><i>Science.</i> <span class="ref__seriesDate">2006; </span><span class="ref__seriesVolume">313</span></span>: <span class="ref__seriesPages">1929-1935</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1126%2Fscience.1132939&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=17008526&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-33749335282&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3071)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=313&publication_year=2006&pages=1929-1935&journal=Science&author=Lamb+J.&author=Crawford+E.D.&author=Peck+D.&author=Modell+J.W.&author=Blat+I.C.&author=Wrobel+M.J.&author=Lerner+J.&author=Brunet+J.-P.&author=Subramanian+A.&author=Ross+K.N.&title=The+Connectivity+Map%3A+using+gene-expression+signatures+to+connect+small+molecules%2C+genes%2C+and+disease">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib122" id="back-bib122" data-db-target-for="bib122-d13420944e4215" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib122">Subramanian et al., 2017</a><div data-db-target-of="bib122-d13420944e4215" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Subramanian A. </li><li class="loa__item">Narayan R. </li><li class="loa__item">Corsello S.M. </li><li class="loa__item">Peck D.D. </li><li class="loa__item">Natoli T.E. </li><li class="loa__item">Lu X. </li><li class="loa__item">Gould J. </li><li class="loa__item">Davis J.F. </li><li class="loa__item">Tubelli A.A. </li><li class="loa__item">Asiedu J.K. </li><li>et al.</li></ul><div class="reference-citations__detials">A Next Generation Connectivity Map: L1000 Platform and the First 1,000,000 Profiles.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">171</span></span>: <span class="ref__seriesPages">1437-1452.e17</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29195078&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85036572148&amp;cf=" onclick="newWindow(this.href);return false">Scopus (772)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2017&pages=1437-1452.e17&journal=Cell&author=Subramanian+A.&author=Narayan+R.&author=Corsello+S.M.&author=Peck+D.D.&author=Natoli+T.E.&author=Lu+X.&author=Gould+J.&author=Davis+J.F.&author=Tubelli+A.A.&author=Asiedu+J.K.&title=A+Next+Generation+Connectivity+Map%3A+L1000+Platform+and+the+First+1%2C000%2C000+Profiles">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and was different from that used in <div class="dropBlock reference-citations"><a href="#bib85" id="back-bib85" data-db-target-for="bib85-d13420944e4218" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib85">Mertins et al., 2016</a><div data-db-target-of="bib85-d13420944e4218" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Wang P. </li><li class="loa__item">Wang X. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Cao S. </li><li class="loa__item">Petralia F. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomics connects somatic mutations to signalling in breast cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (780)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=NCI+CPTAC&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>. Furthermore, the CNA data in the current publication was derived from WES sequencing data, in contrast to SNP array-based CNA data used in <div class="dropBlock reference-citations"><a href="#bib85" id="back-bib85" data-db-target-for="bib85-d13420944e4221" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib85">Mertins et al., 2016</a><div data-db-target-of="bib85-d13420944e4221" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Wang P. </li><li class="loa__item">Wang X. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Cao S. </li><li class="loa__item">Petralia F. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomics connects somatic mutations to signalling in breast cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (780)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=NCI+CPTAC&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>. Thus, given a different technology platform for copy number data generation and significant changes in both the underlying CMAP database and the calculation of connectivity scores, the candidate driver genes identified here do not overlap with those reported in <div class="dropBlock reference-citations"><a href="#bib85" id="back-bib85" data-db-target-for="bib85-d13420944e4224" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib85">Mertins et al., 2016</a><div data-db-target-of="bib85-d13420944e4224" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Wang P. </li><li class="loa__item">Wang X. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Cao S. </li><li class="loa__item">Petralia F. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomics connects somatic mutations to signalling in breast cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (780)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=NCI+CPTAC&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>.</div></section><section id="sec4.5.3.5" data-locator="sec4.5.3.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">RNA-protein correlation</span></h4><div class="section-paragraph">Correlations between mRNA expression and protein abundance for each gene-protein pair were measured using Pearson correlation. To assess the statistical significance of the correlation, a p value (adjusted for multiple testing using FDR) was also calculated. RefSeq protein IDs in the protein data were mapped to HUGO gene symbols. In total, 8,362 genes were quantified in both mRNA and protein data and subsequently used for RNA-protein correlation calculations.</div></section><section id="sec4.5.3.6" data-locator="sec4.5.3.6" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Kinase activity prediction via PTM-SEA</span></h4><div class="section-paragraph">Kinase activity scores were inferred from phosphorylation sites by employing PTM signature enrichment analysis (PTM-SEA) using the PTM signatures database (PTMsigDB) v1.9.0 (<a class="inter-ref-link" href="https://github.com/broadinstitute/ssGSEA2.0" target="_blank">https://github.com/broadinstitute/ssGSEA2.0</a>). Sequence windows flanking the phosphorylation site by 7 amino acids in both directions were used as unique site identifiers. Only fully localized phosphorylation sites as determined by Spectrum Mill software were taken into consideration. Phosphorylation sites on multiply phosphorylated peptides were resolved using the approach described in <div class="dropBlock reference-citations"><a href="#bib62" id="back-bib62" data-db-target-for="bib62-d13420944e4240" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib62">Krug et al., 2019</a><div data-db-target-of="bib62-d13420944e4240" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Krug K. </li><li class="loa__item">Mertins P. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Hornbeck P. </li><li class="loa__item">Raju R. </li><li class="loa__item">Ahmad R. </li><li class="loa__item">Szucs M. </li><li class="loa__item">Mundt F. </li><li class="loa__item">Forestier D. </li><li class="loa__item">Jane-Valbuena J. </li><li>et al.</li></ul><div class="reference-citations__detials">A Curated Resource for Phosphosite-specific Signature Analysis.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">576-593</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30563849&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85062992827&amp;cf=" onclick="newWindow(this.href);return false">Scopus (42)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=576-593&journal=Mol.+Cell.+Proteomics&author=Krug+K.&author=Mertins+P.&author=Zhang+B.&author=Hornbeck+P.&author=Raju+R.&author=Ahmad+R.&author=Szucs+M.&author=Mundt+F.&author=Forestier+D.&author=Jane-Valbuena+J.&title=A+Curated+Resource+for+Phosphosite-specific+Signature+Analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> resulting in a total of 29,406 phosphorylation sites that were subjected to PTM-SEA analysis using the following parameters:<ul class="ce-list--remove-bullets" id="olist0020"><li class="ce-list--remove-bullets__list-item" id="o0075"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">gene.set.database = “ptm.sig.db.all.flanking.human.v1.9.0.gmt”</div></li><li class="ce-list--remove-bullets__list-item" id="o0080"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">sample.norm.type = “rank”</div></li><li class="ce-list--remove-bullets__list-item" id="o0085"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">weight = 0.75</div></li><li class="ce-list--remove-bullets__list-item" id="o0090"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">statistic = “area.under.RES”</div></li><li class="ce-list--remove-bullets__list-item" id="o0095"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">output.score.type = ”NES”</div></li><li class="ce-list--remove-bullets__list-item" id="o0100"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">nperm = 1000</div></li><li class="ce-list--remove-bullets__list-item" id="o0105"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">global.fdr = TRUE</div></li><li class="ce-list--remove-bullets__list-item" id="o0110"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">min.overlap = 5</div></li><li class="ce-list--remove-bullets__list-item" id="o0115"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">correl.type = “z.score”</div></li></ul></div><div class="section-paragraph">NMF subtype-specific PTM-SEA was based on signed log-transformed p values derived from a two-sample moderated t test (<div class="dropBlock reference-citations"><a href="#bib106" id="back-bib106" data-db-target-for="bib106-d13420944e4282" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib106">Ritchie et al., 2015</a><div data-db-target-of="bib106-d13420944e4282" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ritchie M.E. </li><li class="loa__item">Phipson B. </li><li class="loa__item">Wu D. </li><li class="loa__item">Hu Y. </li><li class="loa__item">Law C.W. </li><li class="loa__item">Shi W. </li><li class="loa__item">Smyth G.K. </li></ul><div class="reference-citations__detials">limma powers differential expression analyses for RNA-sequencing and microarray studies.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">43</span></span>: <span class="ref__seriesPages">e47</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgkv007&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=25605792&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84926507971&amp;cf=" onclick="newWindow(this.href);return false">Scopus (10188)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=e47&journal=Nucleic+Acids+Res.&author=Ritchie+M.E.&author=Phipson+B.&author=Wu+D.&author=Hu+Y.&author=Law+C.W.&author=Shi+W.&author=Smyth+G.K.&title=limma+powers+differential+expression+analyses+for+RNA-sequencing+and+microarray+studies">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) comparing each cluster to all other clusters. The same parameters as described above were used with the exception of “weight = 1.”</div></section><section id="sec4.5.3.7" data-locator="sec4.5.3.7" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Pathway projection using ssGSEA</span></h4><div class="section-paragraph">The Gene Set Enrichment Analysis (ssGSEA) implementation available on <a class="inter-ref-link" href="https://github.com/broadinstitute/ssGSEA2.0" target="_blank">https://github.com/broadinstitute/ssGSEA2.0</a> was used to separately project mRNA abundances to signaling pathways. The gene-centric and row-normalized (gene-centered) RNA data matrix derived as described in “RNA Quantitation” was then subjected to ssGSEA using the following parameters:<ul class="ce-list--remove-bullets" id="olist0025"><li class="ce-list--remove-bullets__list-item" id="o0120"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">gene.set.database = “h.all.v6.2.symbols.gmt”</div></li><li class="ce-list--remove-bullets__list-item" id="o0125"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">sample.norm.type = “rank”</div></li><li class="ce-list--remove-bullets__list-item" id="o0130"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">weight = 0.75</div></li><li class="ce-list--remove-bullets__list-item" id="o0135"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">statistic = ”area.under.RES”</div></li><li class="ce-list--remove-bullets__list-item" id="o0140"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">output.score.type = “NES”</div></li><li class="ce-list--remove-bullets__list-item" id="o0145"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">nperm = 1000</div></li><li class="ce-list--remove-bullets__list-item" id="o0150"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">global.fdr = TRUE</div></li><li class="ce-list--remove-bullets__list-item" id="o0155"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">min.overlap = 10</div></li><li class="ce-list--remove-bullets__list-item" id="o0160"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">correl.type = ”z.score”</div></li></ul></div></section><section id="sec4.5.3.8" data-locator="sec4.5.3.8" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Analysis of acetylation data</span></h4><div class="section-paragraph">We used the Reactome (<div class="dropBlock reference-citations"><a href="#bib40" id="back-bib40" data-db-target-for="bib40-d13420944e4335" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib40">Fabregat et al., 2018</a><div data-db-target-of="bib40-d13420944e4335" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Fabregat A. </li><li class="loa__item">Jupe S. </li><li class="loa__item">Matthews L. </li><li class="loa__item">Sidiropoulos K. </li><li class="loa__item">Gillespie M. </li><li class="loa__item">Garapati P. </li><li class="loa__item">Haw R. </li><li class="loa__item">Jassal B. </li><li class="loa__item">Korninger F. </li><li class="loa__item">May B. </li><li>et al.</li></ul><div class="reference-citations__detials">The Reactome Pathway Knowledgebase.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">46</span></span>: <span class="ref__seriesPages">D649-D655</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgkx1132&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29145629&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85040929878&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1166)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D649-D655&journal=Nucleic+Acids+Res.&author=Fabregat+A.&author=Jupe+S.&author=Matthews+L.&author=Sidiropoulos+K.&author=Gillespie+M.&author=Garapati+P.&author=Haw+R.&author=Jassal+B.&author=Korninger+F.&author=May+B.&title=The+Reactome+Pathway+Knowledgebase">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) Metabolism gene set containing 2,212 genes to define proteins involved in metabolism. Unsupervised clustering was performed on metabolic proteins differentially expressed between NMF clusters (Kruskal–Wallis test FDR p value &lt; 5e-05). Differentially acetylated normalized (see below) metabolic Ac sites were selected using a similar procedure with FDR p value &lt; 0.005. All p values were adjusted to FDR using the Benjamini-Hochberg procedure. Normalization of acetylation abundance was performed globally using a linear regression model <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si3.gif"><mrow><mi>A</mi><msub><mi>c</mi><mrow><mi>s</mi><mi>i</mi><mi>t</mi><mi>e</mi></mrow></msub><mo linebreak="goodbreak" linebreakstyle="after">≈</mo><msub><mi>β</mi><mn>0</mn></msub><mo linebreak="goodbreak" linebreakstyle="after">+</mo><msub><mi>β</mi><mn>1</mn></msub><mo>∗</mo><mi>P</mi><mi>r</mi><mo linebreak="goodbreak" linebreakstyle="after">+</mo><mi>ε</mi></mrow></math> where <em>Ac</em> = acetylation abundance of a given protein Ac site; <em>Pr</em> = protein abundance of a given protein; β<sub>1</sub> = predicted coefficient between <em>Pr</em> and <em>Ac;</em> β<sub>0</sub> = constant, and ε = residual values. The residual value ε of every fitted model was used as a new normalized acetylation value not explained by protein abundance.</div><div class="section-paragraph">Subcellular location of metabolic proteins was identified using the COMPARTMENTS database (<div class="dropBlock reference-citations"><a href="#bib15" id="back-bib15" data-db-target-for="bib15-d13420944e4396" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib15">Binder et al., 2014</a><div data-db-target-of="bib15-d13420944e4396" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Binder J.X. </li><li class="loa__item">Pletscher-Frankild S. </li><li class="loa__item">Tsafou K. </li><li class="loa__item">Stolte C. </li><li class="loa__item">O’Donoghue S.I. </li><li class="loa__item">Schneider R. </li><li class="loa__item">Jensen L.J. </li></ul><div class="reference-citations__detials">COMPARTMENTS: unification and visualization of protein subcellular localization evidence.<div class="popup-ref"><span class="ref__series"><i>Database (Oxford).</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">2014</span></span>: <span class="ref__seriesPages">bau012</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fdatabase%2Fbau012&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=24573882&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84901788541&amp;cf=" onclick="newWindow(this.href);return false">Scopus (250)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2014&publication_year=2014&pages=bau012&journal=Database+%28Oxford%29&author=Binder+J.X.&author=Pletscher-Frankild+S.&author=Tsafou+K.&author=Stolte+C.&author=O%E2%80%99Donoghue+S.I.&author=Schneider+R.&author=Jensen+L.J.&title=COMPARTMENTS%3A+unification+and+visualization+of+protein+subcellular+localization+evidence">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), filtered by evidence score &gt; 4. An unpaired two-sample Wilcoxon test was used to find proteins and normalized Ac sites differentially expressed between pairs of NMF clusters as shown in <a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figures 2</a>B and <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">S2</a>A (FDR p value &lt; 0.05).</div><div class="section-paragraph">Association between histone acetyltransferases and histone deacetylases was tested using a linear regression model:<!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si2.gif"><mrow><mi>A</mi><msub><mi>c</mi><mrow><mi>s</mi><mi>u</mi><mi>b</mi><mi>s</mi><mi>t</mi><mi>r</mi><mi>a</mi><mi>t</mi><mi>e</mi><mspace width="0.25em"></mspace><mi>s</mi><mi>i</mi><mi>t</mi><mi>e</mi></mrow></msub><mo linebreak="goodbreak" linebreakstyle="after">≈</mo><msub><mi>β</mi><mn>0</mn></msub><mo linebreak="goodbreak" linebreakstyle="after">+</mo><msub><mi>β</mi><mn>1</mn></msub><mo>∗</mo><mi>P</mi><msub><mi>r</mi><mrow><mi>s</mi><mi>u</mi><mi>b</mi><mi>s</mi><mi>t</mi><mi>r</mi><mi>a</mi><mi>t</mi><mi>e</mi></mrow></msub><mo linebreak="goodbreak" linebreakstyle="after">+</mo><msub><mi>β</mi><mn>2</mn></msub><mo>∗</mo><mi>P</mi><msub><mi>r</mi><mrow><mrow><mi>H</mi><mi>A</mi><mi>T</mi></mrow><mo>/</mo><mrow><mi>H</mi><mi>D</mi><mi>A</mi><mi>C</mi></mrow></mrow></msub><mo linebreak="goodbreak" linebreakstyle="after">+</mo><mi>ε</mi></mrow></math>. P values of β<sub>2</sub> coefficients were adjusted to FDR using Benjamini-Hochberg procedure. The following HATs and HDACs were used to test association with all possible metabolic Ac sites: <em>CREBBP, EP300, HAT1, KAT2A, KAT2B, TAF1, KAT5, KAT6A, KAT6B, KAT7, KAT8, CLOCK, NCOA1, NCOA3, MCM3AP, ATF2, ELP3, HDAC1, HDAC2, HDAC3, HDAC4, HDAC5, HDAC6, HDAC7, HDAC8, HDAC9, HDAC10, HDAC11, SIRT1, SIRT2, SIRT3, SIRT5, SIRT6, SIRT7</em>. Significant (FDR p value &lt; 0.1) associations between mitochondrial HATs and HDACs and mitochondrial substrates Ac sites were defined by FDR p value &lt; 0.1 and are shown in <a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>D.</div><div class="section-paragraph">Nuclear proteins were determined by the COMPARTMENTS database with the nucleus score = 5. Samples with the highest acetylation coverage (N = 92, &gt; 80% of nuclear Ac sites detected) were used for clustering. Acetylation sites differentially abundant between NMF clusters (not normalized to the protein abundance) were selected for analysis (Kruskal–Wallis test FDR p value &lt; 0.05). Global comparison of acetylation sites abundance and protein abundance between clusters 1 and 3 (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>E) was performed using a nonparametric Wilcoxon test. Acetylation or protein changes were considered significantly different if they had FDR p value &lt; 0.05 and median fold change &gt; 0.5. The ‘Acetylation up in cluster 1’ group is defined by significantly different Ac sites in which the acetylation median fold change is positive, while the protein change is not significant; the ‘Protein up in cluster 1’ group is defined by significantly different proteins in which the protein median fold change is positive, while the acetylation change is not significant. Pathway and GO terms overrepresentation testing was performed using gProfiler (<div class="dropBlock reference-citations"><a href="#bib105" id="back-bib105" data-db-target-for="bib105-d13420944e4535" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib105">Reimand et al., 2016</a><div data-db-target-of="bib105-d13420944e4535" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Reimand J. </li><li class="loa__item">Arak T. </li><li class="loa__item">Adler P. </li><li class="loa__item">Kolberg L. </li><li class="loa__item">Reisberg S. </li><li class="loa__item">Peterson H. </li><li class="loa__item">Vilo J. </li></ul><div class="reference-citations__detials">g:Profiler-a web server for functional interpretation of gene lists (2016 update).<div class="popup-ref"><span class="ref__series"><i>Nucleic. Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">W83-W89</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgkw199&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27098042&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=W83-W89&journal=Nucleic.+Acids+Res.&author=Reimand+J.&author=Arak+T.&author=Adler+P.&author=Kolberg+L.&author=Reisberg+S.&author=Peterson+H.&author=Vilo+J.&title=g%3AProfiler-a+web+server+for+functional+interpretation+of+gene+lists+%282016+update%29">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div></section><section id="sec4.5.3.9" data-locator="sec4.5.3.9" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Kinase phosphorylation outliers</span></h4><div class="section-paragraph">To nominate kinase activity characteristic to each PAM50 and NMF cluster, as in previous studies (<div class="dropBlock reference-citations"><a href="#bib38" id="back-bib38" data-db-target-for="bib38-d13420944e4543" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib38">Dou et al., 2020</a><div data-db-target-of="bib38-d13420944e4543" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Dou Y. </li><li class="loa__item">Kawaler E.A. </li><li class="loa__item">Cui Zhou D. </li><li class="loa__item">Gritsenko M.A. </li><li class="loa__item">Huang C. </li><li class="loa__item">Blumenberg L. </li><li class="loa__item">Karpova A. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Savage S.R. </li><li class="loa__item">Satpathy S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Clinical Proteomic Tumor Analysis Consortium</span><br />Proteogenomic Characterization of Endometrial Carcinoma.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">180</span></span>: <span class="ref__seriesPages">729-748.e26</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2020.01.026&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2020.01.026&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2020.01.026&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=32059776&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85079388976&amp;cf=" onclick="newWindow(this.href);return false">Scopus (80)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=180&publication_year=2020&pages=729-748.e26&journal=Cell&author=Clinical+Proteomic+Tumor+Analysis+Consortium&title=Proteogenomic+Characterization+of+Endometrial+Carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib85" id="back-bib85" data-db-target-for="bib85-d13420944e4546" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib85">Mertins et al., 2016</a><div data-db-target-of="bib85-d13420944e4546" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Wang P. </li><li class="loa__item">Wang X. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Cao S. </li><li class="loa__item">Petralia F. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomics connects somatic mutations to signalling in breast cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (780)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=NCI+CPTAC&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), we used BlackSheep’s differential extreme value analysis module (<div class="dropBlock reference-citations"><a href="#bib16" id="back-bib16" data-db-target-for="bib16-d13420944e4549" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib16">Blumenberg et al., 2019</a><div data-db-target-of="bib16-d13420944e4549" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Blumenberg L. </li><li class="loa__item">Kawaler E. </li><li class="loa__item">Cornwell M. </li><li class="loa__item">Smith S. </li><li class="loa__item">Ruggles K. </li><li class="loa__item">Fenyö D. </li></ul><div class="reference-citations__detials">BlackSheep: A Bioconductor and Bioconda package for differential extreme value analysis.<div class="popup-ref"><span class="ref__series"><i>bioRxiv.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1101/825067">https://doi.org/10.1101/825067</a></span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1101%2F825067&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85095064917&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&journal=bioRxiv&author=Blumenberg+L.&author=Kawaler+E.&author=Cornwell+M.&author=Smith+S.&author=Ruggles+K.&author=Feny%C3%B6+D.&title=BlackSheep%3A+A+Bioconductor+and+Bioconda+package+for+differential+extreme+value+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). For each phosphosite, the median and interquartile range (IQR) were calculated across all tumors. A site was defined as an outlier if it was more than 1.5 times the IQR above the median. Phosphosites were then collapsed into proteins by counting outlier and non-outlier values per sample. For each group of interest (e.g., NMF clusters), proteins not enriched in outliers in that group and proteins without at least 30% of samples with an outlier were removed. Following filtering, outlier and non-outlier sites per gene were counted for each group of interest and a Fisher’s exact test was used to calculate a p value. P values were corrected for multiple hypothesis testing using the Benjamini-Hochberg procedure. For additional insight into kinase activity, we visualized enrichment of kinase activation loops phosphorylation, calculated by a rank sum test, of loops taken from a curated list from <div class="dropBlock reference-citations"><a href="#bib114" id="back-bib114" data-db-target-for="bib114-d13420944e4552" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib114">Schmidlin et al., 2019</a><div data-db-target-of="bib114-d13420944e4552" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Schmidlin T. </li><li class="loa__item">Debets D.O. </li><li class="loa__item">van Gelder C.A.G.H. </li><li class="loa__item">Stecker K.E. </li><li class="loa__item">Rontogianni S. </li><li class="loa__item">van den Eshof B.L. </li><li class="loa__item">Kemper K. </li><li class="loa__item">Lips E.H. </li><li class="loa__item">van den Biggelaar M. </li><li class="loa__item">Peeper D.S. </li><li>et al.</li></ul><div class="reference-citations__detials">High-Throughput Assessment of Kinome-wide Activation States.<div class="popup-ref"><span class="ref__series"><i>Cell Syst.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">366-374.e5</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cels.2019.08.005&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cels.2019.08.005&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cels.2019.08.005&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31521607&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85073437904&amp;cf=" onclick="newWindow(this.href);return false">Scopus (12)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=366-374.e5&journal=Cell+Syst.&author=Schmidlin+T.&author=Debets+D.O.&author=van+Gelder+C.A.G.H.&author=Stecker+K.E.&author=Rontogianni+S.&author=van+den+Eshof+B.L.&author=Kemper+K.&author=Lips+E.H.&author=van+den+Biggelaar+M.&author=Peeper+D.S.&title=High-Throughput+Assessment+of+Kinome-wide+Activation+States">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; in addition we visualized enrichment of phosphorylation of kinase substrate sets from the PTM-SEA analysis (<div class="dropBlock reference-citations"><a href="#bib62" id="back-bib62" data-db-target-for="bib62-d13420944e4555" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib62">Krug et al., 2019</a><div data-db-target-of="bib62-d13420944e4555" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Krug K. </li><li class="loa__item">Mertins P. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Hornbeck P. </li><li class="loa__item">Raju R. </li><li class="loa__item">Ahmad R. </li><li class="loa__item">Szucs M. </li><li class="loa__item">Mundt F. </li><li class="loa__item">Forestier D. </li><li class="loa__item">Jane-Valbuena J. </li><li>et al.</li></ul><div class="reference-citations__detials">A Curated Resource for Phosphosite-specific Signature Analysis.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">576-593</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30563849&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85062992827&amp;cf=" onclick="newWindow(this.href);return false">Scopus (42)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=576-593&journal=Mol.+Cell.+Proteomics&author=Krug+K.&author=Mertins+P.&author=Zhang+B.&author=Hornbeck+P.&author=Raju+R.&author=Ahmad+R.&author=Szucs+M.&author=Mundt+F.&author=Forestier+D.&author=Jane-Valbuena+J.&title=A+Curated+Resource+for+Phosphosite-specific+Signature+Analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div></section><section id="sec4.5.3.10" data-locator="sec4.5.3.10" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Multi-omics clustering</span></h4><div class="section-paragraph">Non-negative matrix factorization (NMF) implemented in the NMF R-package (<div class="dropBlock reference-citations"><a href="#bib42" id="back-bib42" data-db-target-for="bib42-d13420944e4563" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib42">Gaujoux and Seoighe, 2010</a><div data-db-target-of="bib42-d13420944e4563" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gaujoux R. </li><li class="loa__item">Seoighe C. </li></ul><div class="reference-citations__detials">A flexible R package for nonnegative matrix factorization.<div class="popup-ref"><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">367</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2F1471-2105-11-367&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=20598126&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-77954069195&amp;cf=" onclick="newWindow(this.href);return false">Scopus (528)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=367&journal=BMC+Bioinformatics&author=Gaujoux+R.&author=Seoighe+C.&title=A+flexible+R+package+for+nonnegative+matrix+factorization">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) was used to perform unsupervised clustering of tumor samples and to identify proteogenomic features (proteins, phosphosites, acetylsites, RNA transcripts and somatic copy number alterations) that showed characteristic abundance patterns for each cluster. Briefly, given a factorization rank <em>k</em> (where <em>k</em> is the number of clusters), NMF decomposes a <em>p x n</em> data matrix <em>V</em> into two matrices <em>W</em> and <em>H</em> such that multiplication of <em>W</em> and <em>H</em> approximates <em>V</em>. Matrix <em>H</em> is a <em>k x n</em> matrix whose entries represent weights for each sample (1 to <em>N</em>) to contribute to each cluster (1 to <em>k</em>), whereas matrix <em>W</em> is a <em>p x k</em> matrix representing weights for each feature (1 to <em>p</em>) to contribute to each cluster (1 to <em>k</em>). Matrix <em>H</em> was used to assign samples to clusters by choosing the <em>k</em> with maximum score in each column of <em>H</em>. For each sample, we calculated a cluster membership score as the maximal fractional score of the corresponding column in matrix <em>H.</em> We defined a ”cluster core” as the set of samples with cluster membership score &gt; 0.5. Matrix <em>W</em> containing the weights of each feature in a certain cluster was used to derive a list of representative features separating the clusters using the method proposed in <div class="dropBlock reference-citations"><a href="#bib59" id="back-bib59" data-db-target-for="bib59-d13420944e4614" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib59">Kim and Park, 2007</a><div data-db-target-of="bib59-d13420944e4614" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kim H. </li><li class="loa__item">Park H. </li></ul><div class="reference-citations__detials">Sparse non-negative matrix factorizations via alternating non-negativity-constrained least squares for microarray data analysis.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">1495-1502</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fbioinformatics%2Fbtm134&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=17483501&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-34547844077&amp;cf=" onclick="newWindow(this.href);return false">Scopus (531)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2007&pages=1495-1502&journal=Bioinformatics&author=Kim+H.&author=Park+H.&title=Sparse+non-negative+matrix+factorizations+via+alternating+non-negativity-constrained+least+squares+for+microarray+data+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>. Cluster-specific features were further subjected to a 2-sample moderated t test (<div class="dropBlock reference-citations"><a href="#bib106" id="back-bib106" data-db-target-for="bib106-d13420944e4617" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib106">Ritchie et al., 2015</a><div data-db-target-of="bib106-d13420944e4617" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ritchie M.E. </li><li class="loa__item">Phipson B. </li><li class="loa__item">Wu D. </li><li class="loa__item">Hu Y. </li><li class="loa__item">Law C.W. </li><li class="loa__item">Shi W. </li><li class="loa__item">Smyth G.K. </li></ul><div class="reference-citations__detials">limma powers differential expression analyses for RNA-sequencing and microarray studies.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">43</span></span>: <span class="ref__seriesPages">e47</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgkv007&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=25605792&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84926507971&amp;cf=" onclick="newWindow(this.href);return false">Scopus (10188)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=e47&journal=Nucleic+Acids+Res.&author=Ritchie+M.E.&author=Phipson+B.&author=Wu+D.&author=Hu+Y.&author=Law+C.W.&author=Shi+W.&author=Smyth+G.K.&title=limma+powers+differential+expression+analyses+for+RNA-sequencing+and+microarray+studies">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) comparing the feature abundance between the respective cluster and all other clusters. Derived p values were adjusted for multiple hypothesis testing using the methods proposed in <div class="dropBlock reference-citations"><a href="#bib14" id="back-bib14" data-db-target-for="bib14-d13420944e4620" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib14">Benjamini and Hochberg, 1995</a><div data-db-target-of="bib14-d13420944e4620" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Benjamini Y. </li><li class="loa__item">Hochberg Y. </li></ul><div class="reference-citations__detials">Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing.<div class="popup-ref"><span class="ref__series"><i>J. R. Stat. Soc. Series B Methodol.</i> <span class="ref__seriesDate">1995; </span><span class="ref__seriesVolume">57</span></span>: <span class="ref__seriesPages">289-300</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1111%2Fj.2517-6161.1995.tb02031.x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1995&pages=289-300&journal=J.%C2%A0R.+Stat.+Soc.+Series+B+Methodol.&author=Benjamini+Y.&author=Hochberg+Y.&title=Controlling+the+False+Discovery+Rate%3A+A+Practical+and+Powerful+Approach+to+Multiple+Testing">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>.</div><div class="section-paragraph">To enable integrative multi-omics clustering, we required all data types (and converted if necessary) to represent ratios to either a common reference measured in each TMT plex (proteome, phosphoproteome, acetylproteome) or an <em>in-silico</em> common reference calculated as the median abundance across all samples (mRNA, see “<a href="#sec4.5.1.3" class="scroll-into-link">RNA quantification</a>”). All data tables were then concatenated and only features quantified in all tumors were used for subsequent analysis. Features with the lowest standard deviation (bottom 5<sup>th</sup> percentile) across all samples were deemed uninformative and were removed from the dataset. Each row in the data matrix was further scaled and standardized such that all features from different data types were represented as z-scores.</div><div class="section-paragraph">Since NMF requires a non-negative input matrix, the data matrix of z-scores was further converted into a non-negative matrix as follows:<ul class="ce-list--remove-bullets" id="olist0030"><li class="ce-list--remove-bullets__list-item" id="o0165"><span class="label">1)</span><div class="ce-list--remove-bullets__list-item__text">Create one data matrix with all negative numbers zeroed.</div></li><li class="ce-list--remove-bullets__list-item" id="o0170"><span class="label">2)</span><div class="ce-list--remove-bullets__list-item__text">Create another data matrix with all positive numbers zeroed and the signs of all negative numbers removed.</div></li><li class="ce-list--remove-bullets__list-item" id="o0175"><span class="label">3)</span><div class="ce-list--remove-bullets__list-item__text">Concatenate both matrices resulting in a data matrix twice as large as the original, but with positive values only and zeros and hence appropriate for NMF.</div></li></ul></div><div class="section-paragraph">The resulting matrix was then subjected to NMF analysis leveraging the NMF R-package (<div class="dropBlock reference-citations"><a href="#bib42" id="back-bib42" data-db-target-for="bib42-d13420944e4650" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib42">Gaujoux and Seoighe, 2010</a><div data-db-target-of="bib42-d13420944e4650" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gaujoux R. </li><li class="loa__item">Seoighe C. </li></ul><div class="reference-citations__detials">A flexible R package for nonnegative matrix factorization.<div class="popup-ref"><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">367</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2F1471-2105-11-367&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=20598126&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-77954069195&amp;cf=" onclick="newWindow(this.href);return false">Scopus (528)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=367&journal=BMC+Bioinformatics&author=Gaujoux+R.&author=Seoighe+C.&title=A+flexible+R+package+for+nonnegative+matrix+factorization">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and using the factorization method described in <div class="dropBlock reference-citations"><a href="#bib18" id="back-bib18" data-db-target-for="bib18-d13420944e4653" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib18">Brunet et al., 2004</a><div data-db-target-of="bib18-d13420944e4653" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Brunet J.-P. </li><li class="loa__item">Tamayo P. </li><li class="loa__item">Golub T.R. </li><li class="loa__item">Mesirov J.P. </li></ul><div class="reference-citations__detials">Metagenes and molecular pattern discovery using matrix factorization.<div class="popup-ref"><span class="ref__series"><i>Proc. Natl. Acad. Sci. USA.</i> <span class="ref__seriesDate">2004; </span><span class="ref__seriesVolume">101</span></span>: <span class="ref__seriesPages">4164-4169</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1073%2Fpnas.0308531101&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=15016911&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-1642529511&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1117)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=4164-4169&journal=Proc.+Natl.+Acad.+Sci.+USA&author=Brunet+J.-P.&author=Tamayo+P.&author=Golub+T.R.&author=Mesirov+J.P.&title=Metagenes+and+molecular+pattern+discovery+using+matrix+factorization">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>. To determine the optimal factorization rank <em>k</em> (number of clusters) for the multi-omic data matrix, a range of clusters between <em>k</em> = 2 and 8 was tested. For each <em>k</em> we factorized matrix <em>V</em> using 50 iterations with random initializations of <em>W</em> and <em>H</em>. To determine the optimal factorization rank we calculated two metrics for each <em>k</em>: 1) cophenetic correlation coefficient measuring how well the intrinsic structure of the data was recapitulated after clustering and 2) the dispersion coefficient of the consensus matrix as defined in <div class="dropBlock reference-citations"><a href="#bib59" id="back-bib59" data-db-target-for="bib59-d13420944e4671" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib59">Kim and Park, 2007</a><div data-db-target-of="bib59-d13420944e4671" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kim H. </li><li class="loa__item">Park H. </li></ul><div class="reference-citations__detials">Sparse non-negative matrix factorizations via alternating non-negativity-constrained least squares for microarray data analysis.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">1495-1502</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fbioinformatics%2Fbtm134&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=17483501&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-34547844077&amp;cf=" onclick="newWindow(this.href);return false">Scopus (531)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2007&pages=1495-1502&journal=Bioinformatics&author=Kim+H.&author=Park+H.&title=Sparse+non-negative+matrix+factorizations+via+alternating+non-negativity-constrained+least+squares+for+microarray+data+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> measuring the reproducibility of the clustering across 50 iterations. The optimal <em>k</em> was defined as the maximum of the product of both metrics for cluster numbers between k = 3 and 8 (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">Figures S3</a>C and <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs4" href="#figs4" data-locator="figs4" id="back-figs4" class="figure-link scroll-into-link">S4</a>B).</div><div class="section-paragraph">Having determined the optimal factorization rank <em>k</em>, and in order to achieve robust factorization of the multi-omics data matrix <em>V,</em> the NMF analysis was repeated using 1000 iterations with random initializations of <em>W</em> and <em>H</em> and partitioning of samples into clusters as described above. Due to the non-negative transformation applied to the z-scored data matrix as described above, matrix <em>W</em> of feature weights contained two separate weights for positive and negative z-scores of each feature, respectively. In order to reverse the non-negative transformation and to derive a single signed weight for each feature, each row in matrix <em>W</em> was first normalized by dividing by the sum of feature weights in each row. Weights per feature and cluster were then aggregated by keeping the maximal normalized weight and multiplying with the sign of the z-score from the initial data matrix. Thus, the resulting transformed version of matrix <em>W</em><sub>signed</sub> contained signed cluster weights for each feature present in the input matrix.</div><div class="section-paragraph">In order to functionally characterize the clustering results, normalized enrichment scores (NES) of cancer-relevant gene sets were calculated by projecting the matrix of signed multi-omic feature weights (<em>W</em><sub>signed</sub>) onto Hallmark pathway gene sets (<div class="dropBlock reference-citations"><a href="#bib72" id="back-bib72" data-db-target-for="bib72-d13420944e4708" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib72">Liberzon et al., 2015</a><div data-db-target-of="bib72-d13420944e4708" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liberzon A. </li><li class="loa__item">Birger C. </li><li class="loa__item">Thorvaldsdóttir H. </li><li class="loa__item">Ghandi M. </li><li class="loa__item">Mesirov J.P. </li><li class="loa__item">Tamayo P. </li></ul><div class="reference-citations__detials">The Molecular Signatures Database (MSigDB) hallmark gene set collection.<div class="popup-ref"><span class="ref__series"><i>Cell Syst.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">417-425</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26771021&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84955328286&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2256)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=417-425&journal=Cell+Syst.&author=Liberzon+A.&author=Birger+C.&author=Thorvaldsd%C3%B3ttir+H.&author=Ghandi+M.&author=Mesirov+J.P.&author=Tamayo+P.&title=The+Molecular+Signatures+Database+%28MSigDB%29+hallmark+gene+set+collection">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) using ssGSEA (<div class="dropBlock reference-citations"><a href="#bib10" id="back-bib10" data-db-target-for="bib10-d13420944e4711" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib10">Barbie et al., 2009</a><div data-db-target-of="bib10-d13420944e4711" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Barbie D.A. </li><li class="loa__item">Tamayo P. </li><li class="loa__item">Boehm J.S. </li><li class="loa__item">Kim S.Y. </li><li class="loa__item">Moody S.E. </li><li class="loa__item">Dunn I.F. </li><li class="loa__item">Schinzel A.C. </li><li class="loa__item">Sandy P. </li><li class="loa__item">Meylan E. </li><li class="loa__item">Scholl C. </li><li>et al.</li></ul><div class="reference-citations__detials">Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">462</span></span>: <span class="ref__seriesPages">108-112</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature08460&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=19847166&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-70449091786&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1353)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=462&publication_year=2009&pages=108-112&journal=Nature&author=Barbie+D.A.&author=Tamayo+P.&author=Boehm+J.S.&author=Kim+S.Y.&author=Moody+S.E.&author=Dunn+I.F.&author=Schinzel+A.C.&author=Sandy+P.&author=Meylan+E.&author=Scholl+C.&title=Systematic+RNA+interference+reveals+that+oncogenic+KRAS-driven+cancers+require+TBK1">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). To derive a single weight for each gene measured across multiple omics data types (protein, RNA, phosphorylation site, acetylation site) we retained the weight with maximal absolute amplitude. We used the ssGSEA implementation available on <a class="inter-ref-link" href="https://github.com/broadinstitute/ssGSEA2.0" target="_blank">https://github.com/broadinstitute/ssGSEA2.0</a> using the following parameters:<ul class="ce-list--remove-bullets" id="olist0035"><li class="ce-list--remove-bullets__list-item" id="o0180"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">gene.set.database = “h.all.v6.2.symbols.gmt”</div></li><li class="ce-list--remove-bullets__list-item" id="o0185"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">sample.norm.type = “rank”</div></li><li class="ce-list--remove-bullets__list-item" id="o0190"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">weight = 1</div></li><li class="ce-list--remove-bullets__list-item" id="o0195"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">statistic = ”area.under.RES”</div></li><li class="ce-list--remove-bullets__list-item" id="o0200"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">output.score.type = “NES”</div></li><li class="ce-list--remove-bullets__list-item" id="o0205"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">nperm = 1000</div></li><li class="ce-list--remove-bullets__list-item" id="o0210"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">global.fdr = TRUE</div></li><li class="ce-list--remove-bullets__list-item" id="o0215"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">min.overlap = 5</div></li><li class="ce-list--remove-bullets__list-item" id="o0220"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">correl.type = ”z.score”</div></li></ul></div><div class="section-paragraph">To test the association between the resulting clusters and clinical variables, either a Fisher’s exact test (R function <em>fisher.test</em>) for discrete variables or a Wilcoxon rank-sum test (<em>ggpubr</em> R-package) in case of continuous variables was used to assess overrepresentation in the set of samples defining the cluster core as described above.</div></section><section id="sec4.5.3.11" data-locator="sec4.5.3.11" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Survival analysis</span></h4><div class="section-paragraph">To explore differences in prognosis for PAM50 LumA samples in the NMF LumA-I cluster compared to those in the NMF LumB-I cluster, we leveraged outcome data from the METABRIC study [METABRIC data was downloaded from cBioPortal (<a class="inter-ref-link" href="https://www.cbioportal.org/study/summary?id=brca_metabric" target="_blank">https://www.cbioportal.org/study/summary?id=brca_metabric</a>) on Jun 2, 2020]. We trained random forest classifiers to discriminate these two groups—(A) PAM50 Luminal A in NMF LumA-I versus (B) PAM50 Luminal A in mixed Luminal A/B NMF LumB-I—using RNA-seq expression data from this study. The classifier was trained on genes common to our dataset and METABRIC, using the caret package in R. Cross validation (10-fold) over the training data was used to optimize model parameters. The final model was trained on the entire training dataset using the optimal parameters, and then used to predict NMF cluster assignment for all PAM50 LumA samples in METABRIC. Kaplan-Meier plots and log-rank tests for statistical significance were executed using the survival and survminer packages in R. For comparison, survival information for PAM50 LumB samples in METABRIC were included. Similar results were obtained when the classifier was trained using gene-level global proteome data (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">Figure S3</a>K).</div></section><section id="sec4.5.3.12" data-locator="sec4.5.3.12" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Single-omics clustering and application to the Johansson et al. breast proteogenomic dataset</span></h4><div class="section-paragraph">The NMF pipeline described above was applied to each data type individually using the same parameters as for multi-omics analysis except for 500 random restarts of the factorization. An identical NMF clustering approach was applied to the <div class="dropBlock reference-citations"><a href="#bib55" id="back-bib55" data-db-target-for="bib55-d13420944e4777" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib55">Johansson et al., 2019</a><div data-db-target-of="bib55-d13420944e4777" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Johansson H.J. </li><li class="loa__item">Socciarelli F. </li><li class="loa__item">Vacanti N.M. </li><li class="loa__item">Haugen M.H. </li><li class="loa__item">Zhu Y. </li><li class="loa__item">Siavelis I. </li><li class="loa__item">Fernandez-Woodbridge A. </li><li class="loa__item">Aure M.R. </li><li class="loa__item">Sennblad B. </li><li class="loa__item">Vesterlund M. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Consortia Oslo Breast Cancer Research Consortium (OSBREAC)</span><br />Breast cancer quantitative proteome and proteogenomic landscape.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">1600</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41467-019-09018-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30962452&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85064079271&amp;cf=" onclick="newWindow(this.href);return false">Scopus (66)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1600&journal=Nat.+Commun.&author=Consortia+Oslo+Breast+Cancer+Research+Consortium+%28OSBREAC%29&title=Breast+cancer+quantitative+proteome+and+proteogenomic+landscape">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> dataset (their <a href="/cms/10.1016/j.cell.2020.10.036/attachment/3dbb5a07-b92f-4ae5-9c88-f84c3dca95db/mmc1" class="scroll-into-link">Supplementary Data 1</a>). To allow integrative analysis of our protein data with the protein data in <div class="dropBlock reference-citations"><a href="#bib55" id="back-bib55" data-db-target-for="bib55-d13420944e4783" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib55">Johansson et al., 2019</a><div data-db-target-of="bib55-d13420944e4783" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Johansson H.J. </li><li class="loa__item">Socciarelli F. </li><li class="loa__item">Vacanti N.M. </li><li class="loa__item">Haugen M.H. </li><li class="loa__item">Zhu Y. </li><li class="loa__item">Siavelis I. </li><li class="loa__item">Fernandez-Woodbridge A. </li><li class="loa__item">Aure M.R. </li><li class="loa__item">Sennblad B. </li><li class="loa__item">Vesterlund M. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Consortia Oslo Breast Cancer Research Consortium (OSBREAC)</span><br />Breast cancer quantitative proteome and proteogenomic landscape.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">1600</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41467-019-09018-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30962452&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85064079271&amp;cf=" onclick="newWindow(this.href);return false">Scopus (66)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1600&journal=Nat.+Commun.&author=Consortia+Oslo+Breast+Cancer+Research+Consortium+%28OSBREAC%29&title=Breast+cancer+quantitative+proteome+and+proteogenomic+landscape">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> (<a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs4" href="#figs4" data-locator="figs4" id="back-figs4" class="figure-link scroll-into-link">Figure S4</a>D), we first aggregated the protein-level data from our study to generate a gene-level data matrix by retaining the dominant isoform (identified by the lowest protein subgroup number) associated with each gene symbol. Both gene-level protein data matrices were then separately subjected to gene-level z-score transformation before joining the matrices using the unique gene symbols as a key. The NMF pipeline was applied to the integrated protein data matrix using 500 random restarts to cluster all 167 tumors into six clusters, a number pre-specified to correspond to the number of clusters identified in the <div class="dropBlock reference-citations"><a href="#bib55" id="back-bib55" data-db-target-for="bib55-d13420944e4789" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib55">Johansson et al., 2019</a><div data-db-target-of="bib55-d13420944e4789" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Johansson H.J. </li><li class="loa__item">Socciarelli F. </li><li class="loa__item">Vacanti N.M. </li><li class="loa__item">Haugen M.H. </li><li class="loa__item">Zhu Y. </li><li class="loa__item">Siavelis I. </li><li class="loa__item">Fernandez-Woodbridge A. </li><li class="loa__item">Aure M.R. </li><li class="loa__item">Sennblad B. </li><li class="loa__item">Vesterlund M. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Consortia Oslo Breast Cancer Research Consortium (OSBREAC)</span><br />Breast cancer quantitative proteome and proteogenomic landscape.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">1600</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41467-019-09018-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30962452&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85064079271&amp;cf=" onclick="newWindow(this.href);return false">Scopus (66)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1600&journal=Nat.+Commun.&author=Consortia+Oslo+Breast+Cancer+Research+Consortium+%28OSBREAC%29&title=Breast+cancer+quantitative+proteome+and+proteogenomic+landscape">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> analysis.</div></section><section id="sec4.5.3.13" data-locator="sec4.5.3.13" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">LinkedOmics data preparation</span></h4><div class="section-paragraph">Sample metadata, gene-centric GISTIC copy number log ratios, median-MAD normalized RNA expression levels, and 2-component normalized TMT log ratios for proteome, phosphoproteome, and acetylproteome datasets were deposited in LinkedOmics. Since LinkedOmics is a gene-centric database, proteome data was aggregated to the gene level according to the following process: for each subgroup, the HGNC symbol for the dominant protein in the subgroup, which was aggregated from common PSMs for the subgroup as well as unique PSMs for that protein by Spectrum Mill, was retained. If other proteins in the subgroup were reported (aggregated from unique PSMs by Spectrum Mill), the median of all entries from the subgroup for each unique HGNC symbol (other than the dominant protein gene) was retained. The median of each entry for each retained gene was uploaded into LinkedOmics. Data processed in this manner was also used for the ERBB2 proteogenomic analysis reported in <a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figures 3</a>A and 3B, the immune analysis in <a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figures 4</a> and <a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">5</a>, and the cell cycle analysis in <a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>. Gene level data for the phospho- and acetylproteomes was aggregated by the median of all sites assigned to each HGNC symbol, and site level data was aggregated by taking the median of all PSMs with high confidence localization (best score VML &gt; = 1.1) for each phospho/acetyl site position in each protein.</div></section><section id="sec4.5.3.14" data-locator="sec4.5.3.14" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Proteogenomic status of ERBB2 and TOP2A</span></h4><div class="section-paragraph">Samples were classified as proteogenomic (PG) positive for a given gene amplification when that amplification led to high levels of protein relative to the population of samples without the gene amplification. Gene-amplified samples were defined by a GISTIC threshold score of 2. All other samples were considered non-amplified. Protein Z-scores were calculated for each amplified sample relative to the distribution of log2 TMT ratios for the non-amplified samples using an outlier approach described previously (<div class="dropBlock reference-citations"><a href="#bib113" id="back-bib113" data-db-target-for="bib113-d13420944e4814" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib113">Satpathy et al., 2020</a><div data-db-target-of="bib113-d13420944e4814" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Satpathy S. </li><li class="loa__item">Jaehnig E.J. </li><li class="loa__item">Krug K. </li><li class="loa__item">Kim B.-J. </li><li class="loa__item">Saltzman A.B. </li><li class="loa__item">Chan D.W. </li><li class="loa__item">Holloway K.R. </li><li class="loa__item">Anurag M. </li><li class="loa__item">Huang C. </li><li class="loa__item">Singh P. </li><li>et al.</li></ul><div class="reference-citations__detials">Microscaled proteogenomic methods for precision oncology.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">532</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41467-020-14381-2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31988290&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85078365239&amp;cf=" onclick="newWindow(this.href);return false">Scopus (25)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=532&journal=Nat.+Commun.&author=Satpathy+S.&author=Jaehnig+E.J.&author=Krug+K.&author=Kim+B.-J.&author=Saltzman+A.B.&author=Chan+D.W.&author=Holloway+K.R.&author=Anurag+M.&author=Huang+C.&author=Singh+P.&title=Microscaled+proteogenomic+methods+for+precision+oncology">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), in which the Z-score was the number of non-amplified set standard deviations above the mean of the non-amplified samples that the protein expression represented in a given amplified sample. Z-scores above 2 were considered to show elevated protein expression. For ERBB2 PG+ samples, we also required PG amplification of either the <em>STARD3</em> or <em>GRB7</em> gene flanking <em>ERBB2</em> in the amplicon. The same procedure was applied to log2 iTraq protein data and GISTIC data downloaded from LinkedOmics (<a class="inter-ref-link" href="http://linkedomics.org" target="_blank">http://linkedomics.org</a>) for the retrospective cohort (<div class="dropBlock reference-citations"><a href="#bib85" id="back-bib85" data-db-target-for="bib85-d13420944e4827" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib85">Mertins et al., 2016</a><div data-db-target-of="bib85-d13420944e4827" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Wang P. </li><li class="loa__item">Wang X. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Cao S. </li><li class="loa__item">Petralia F. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomics connects somatic mutations to signalling in breast cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (780)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=NCI+CPTAC&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div></section><section id="sec4.5.3.15" data-locator="sec4.5.3.15" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Immune profiling and downstream analysis</span></h4><div class="section-paragraph">To calculate RNA-based tumor immune scores and estimate immune-cell-specific contributions to each tumor, FPKM data was analyzed using ESTIMATE (R package) (<div class="dropBlock reference-citations"><a href="#bib146" id="back-bib146" data-db-target-for="bib146-d13420944e4835" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib146">Yoshihara et al., 2013</a><div data-db-target-of="bib146-d13420944e4835" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Yoshihara K. </li><li class="loa__item">Shahmoradgoli M. </li><li class="loa__item">Martínez E. </li><li class="loa__item">Vegesna R. </li><li class="loa__item">Kim H. </li><li class="loa__item">Torres-Garcia W. </li><li class="loa__item">Treviño V. </li><li class="loa__item">Shen H. </li><li class="loa__item">Laird P.W. </li><li class="loa__item">Levine D.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Inferring tumour purity and stromal and immune cell admixture from expression data.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">2612</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fncomms3612&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=24113773&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84885673911&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1675)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2612&journal=Nat.+Commun.&author=Yoshihara+K.&author=Shahmoradgoli+M.&author=Mart%C3%ADnez+E.&author=Vegesna+R.&author=Kim+H.&author=Torres-Garcia+W.&author=Trevi%C3%B1o+V.&author=Shen+H.&author=Laird+P.W.&author=Levine+D.A.&title=Inferring+tumour+purity+and+stromal+and+immune+cell+admixture+from+expression+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), CIBERSORT in absolute mode (<div class="dropBlock reference-citations"><a href="#bib92" id="back-bib92" data-db-target-for="bib92-d13420944e4838" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib92">Newman et al., 2015</a><div data-db-target-of="bib92-d13420944e4838" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Newman A.M. </li><li class="loa__item">Liu C.L. </li><li class="loa__item">Green M.R. </li><li class="loa__item">Gentles A.J. </li><li class="loa__item">Feng W. </li><li class="loa__item">Xu Y. </li><li class="loa__item">Hoang C.D. </li><li class="loa__item">Diehn M. </li><li class="loa__item">Alizadeh A.A. </li></ul><div class="reference-citations__detials">Robust enumeration of cell subsets from tissue expression profiles.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">453-457</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnmeth.3337&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=25822800&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84928927858&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2633)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=453-457&journal=Nat.+Methods&author=Newman+A.M.&author=Liu+C.L.&author=Green+M.R.&author=Gentles+A.J.&author=Feng+W.&author=Xu+Y.&author=Hoang+C.D.&author=Diehn+M.&author=Alizadeh+A.A.&title=Robust+enumeration+of+cell+subsets+from+tissue+expression+profiles">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), xCell (<div class="dropBlock reference-citations"><a href="#bib7" id="back-bib7" data-db-target-for="bib7-d13420944e4841" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib7">Aran et al., 2017</a><div data-db-target-of="bib7-d13420944e4841" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Aran D. </li><li class="loa__item">Hu Z. </li><li class="loa__item">Butte A.J. </li></ul><div class="reference-citations__detials">xCell: digitally portraying the tissue cellular heterogeneity landscape.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">220</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2Fs13059-017-1349-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29141660&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85034614124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (627)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=220&journal=Genome+Biol.&author=Aran+D.&author=Hu+Z.&author=Butte+A.J.&title=xCell%3A+digitally+portraying+the+tissue+cellular+heterogeneity+landscape">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and MCPcounter (R package) (<div class="dropBlock reference-citations"><a href="#bib13" id="back-bib13" data-db-target-for="bib13-d13420944e4844" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib13">Becht et al., 2016b</a><div data-db-target-of="bib13-d13420944e4844" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Becht E. </li><li class="loa__item">Giraldo N.A. </li><li class="loa__item">Lacroix L. </li><li class="loa__item">Buttard B. </li><li class="loa__item">Elarouci N. </li><li class="loa__item">Petitprez F. </li><li class="loa__item">Selves J. </li><li class="loa__item">Laurent-Puig P. </li><li class="loa__item">Sautès-Fridman C. </li><li class="loa__item">Fridman W.H. </li><li class="loa__item">de Reyniès A. </li></ul><div class="reference-citations__detials">Estimating the population abundance of tissue-infiltrating immune and stromal cell populations using gene expression.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">218</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2Fs13059-016-1070-5&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27765066&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84992053166&amp;cf=" onclick="newWindow(this.href);return false">Scopus (497)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=218&journal=Genome+Biol.&author=Becht+E.&author=Giraldo+N.A.&author=Lacroix+L.&author=Buttard+B.&author=Elarouci+N.&author=Petitprez+F.&author=Selves+J.&author=Laurent-Puig+P.&author=Saut%C3%A8s-Fridman+C.&author=Fridman+W.H.&author=de+Reyni%C3%A8s+A.&title=Estimating+the+population+abundance+of+tissue-infiltrating+immune+and+stromal+cell+populations+using+gene+expression">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). We also inferred the immune cell infiltration by ssGSEA using a recently published immune gene signature (<div class="dropBlock reference-citations"><a href="#bib3" id="back-bib3" data-db-target-for="bib3-d13420944e4847" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib3">Angelova et al., 2015</a><div data-db-target-of="bib3-d13420944e4847" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Angelova M. </li><li class="loa__item">Charoentong P. </li><li class="loa__item">Hackl H. </li><li class="loa__item">Fischer M.L. </li><li class="loa__item">Snajder R. </li><li class="loa__item">Krogsdam A.M. </li><li class="loa__item">Waldner M.J. </li><li class="loa__item">Bindea G. </li><li class="loa__item">Mlecnik B. </li><li class="loa__item">Galon J. </li><li class="loa__item">Trajanoski Z. </li></ul><div class="reference-citations__detials">Characterization of the immunophenotypes and antigenomes of colorectal cancers reveals distinct tumor escape mechanisms and novel targets for immunotherapy.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">64</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2Fs13059-015-0620-6&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=25853550&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84931426001&amp;cf=" onclick="newWindow(this.href);return false">Scopus (283)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=64&journal=Genome+Biol.&author=Angelova+M.&author=Charoentong+P.&author=Hackl+H.&author=Fischer+M.L.&author=Snajder+R.&author=Krogsdam+A.M.&author=Waldner+M.J.&author=Bindea+G.&author=Mlecnik+B.&author=Galon+J.&author=Trajanoski+Z.&title=Characterization+of+the+immunophenotypes+and+antigenomes+of+colorectal+cancers+reveals+distinct+tumor+escape+mechanisms+and+novel+targets+for+immunotherapy">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Protein-based immune scores for stimulatory and inhibitory immune modulators and the set of HLA proteins were calculated as the mean of the protein log ratios in each set defined in <div class="dropBlock reference-citations"><a href="#bib128" id="back-bib128" data-db-target-for="bib128-d13420944e4851" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib128">Thorsson et al., 2019</a><div data-db-target-of="bib128-d13420944e4851" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Thorsson V. </li><li class="loa__item">Gibbs D.L. </li><li class="loa__item">Brown S.D. </li><li class="loa__item">Wolf D. </li><li class="loa__item">Bortone D.S. </li><li class="loa__item">Ou Yang T.-H. </li><li class="loa__item">Porta-Pardo E. </li><li class="loa__item">Gao G.F. </li><li class="loa__item">Plaisier C.L. </li><li class="loa__item">Eddy J.A. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />The Immune Landscape of Cancer.<div class="popup-ref"><span class="ref__series"><i>Immunity.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">51</span></span>: <span class="ref__seriesPages">411-412</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.immuni.2019.08.004&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.immuni.2019.08.004&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.immuni.2019.08.004&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31433971&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85070213901&amp;cf=" onclick="newWindow(this.href);return false">Scopus (61)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2019&pages=411-412&journal=Immunity&author=Cancer+Genome+Atlas+Research+Network&title=The+Immune+Landscape+of+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>. Immune protein eigenvectors and signatures were calculated using protein data with the protocol and gene sets described in <div class="dropBlock reference-citations"><a href="#bib128" id="back-bib128" data-db-target-for="bib128-d13420944e4854" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib128">Thorsson et al., 2019</a><div data-db-target-of="bib128-d13420944e4854" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Thorsson V. </li><li class="loa__item">Gibbs D.L. </li><li class="loa__item">Brown S.D. </li><li class="loa__item">Wolf D. </li><li class="loa__item">Bortone D.S. </li><li class="loa__item">Ou Yang T.-H. </li><li class="loa__item">Porta-Pardo E. </li><li class="loa__item">Gao G.F. </li><li class="loa__item">Plaisier C.L. </li><li class="loa__item">Eddy J.A. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />The Immune Landscape of Cancer.<div class="popup-ref"><span class="ref__series"><i>Immunity.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">51</span></span>: <span class="ref__seriesPages">411-412</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.immuni.2019.08.004&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.immuni.2019.08.004&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.immuni.2019.08.004&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31433971&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85070213901&amp;cf=" onclick="newWindow(this.href);return false">Scopus (61)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2019&pages=411-412&journal=Immunity&author=Cancer+Genome+Atlas+Research+Network&title=The+Immune+Landscape+of+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>. Then the two top protein signatures closest by Euclidean distance to each of the five eigenvectors were shown in <a data-link="modal" data-target="#image-S0092-8674(20)31400-8figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">Figure S7</a>A.</div></section><section id="sec4.5.3.16" data-locator="sec4.5.3.16" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">PD-L1 correlation analysis</span></h4><div class="section-paragraph">LinkedOmics (<a class="inter-ref-link" href="http://linkedomics.org" target="_blank">http://linkedomics.org</a>; <div class="dropBlock reference-citations"><a href="#bib134" id="back-bib134" data-db-target-for="bib134-d13420944e4868" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib134">Vasaikar et al., 2018</a><div data-db-target-of="bib134-d13420944e4868" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Vasaikar S.V. </li><li class="loa__item">Straub P. </li><li class="loa__item">Wang J. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">LinkedOmics: analyzing multi-omics data within and across 32 cancer types.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">46</span></span>: <span class="ref__seriesPages">D956-D963</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgkx1090&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29136207&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85040936922&amp;cf=" onclick="newWindow(this.href);return false">Scopus (521)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D956-D963&journal=Nucleic+Acids+Res.&author=Vasaikar+S.V.&author=Straub+P.&author=Wang+J.&author=Zhang+B.&title=LinkedOmics%3A+analyzing+multi-omics+data+within+and+across+32+cancer+types">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) was used to identify proteins correlated with PD-L1 mRNA levels within the PAM50 luminal samples (luminal A + luminal B) and the PAM50 basal samples separately. Benjamini-Hochberg corrected p values for Spearman rank correlations between PD-L1 and each protein are shown in <a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>A. WebGestalt (<div class="dropBlock reference-citations"><a href="#bib71" id="back-bib71" data-db-target-for="bib71-d13420944e4874" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib71">Liao et al., 2019</a><div data-db-target-of="bib71-d13420944e4874" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liao Y. </li><li class="loa__item">Wang J. </li><li class="loa__item">Jaehnig E.J. </li><li class="loa__item">Shi Z. </li></ul><div class="reference-citations__detials">WebGestalt 2019: gene set analysis toolkit with revamped UIs and APIs.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">47</span></span>: <span class="ref__seriesPages">W199-W205</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgkz401&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31114916&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85069234777&amp;cf=" onclick="newWindow(this.href);return false">Scopus (567)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=W199-W205&journal=Nucleic+Acids+Res.&author=Liao+Y.&author=Wang+J.&author=Jaehnig+E.J.&author=Shi+Z.&title=WebGestalt+2019%3A+gene+set+analysis+toolkit+with+revamped+UIs+and+APIs">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <a class="inter-ref-link" href="http://webgestalt.org" target="_blank">http://webgestalt.org</a>) was used to perform GSEA for GO biological process sets (<div class="dropBlock reference-citations"><a href="#bib9" id="back-bib9" data-db-target-for="bib9-d13420944e4881" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib9">Ashburner et al., 2000</a><div data-db-target-of="bib9-d13420944e4881" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ashburner M. </li><li class="loa__item">Ball C.A. </li><li class="loa__item">Blake J.A. </li><li class="loa__item">Botstein D. </li><li class="loa__item">Butler H. </li><li class="loa__item">Cherry J.M. </li><li class="loa__item">Davis A.P. </li><li class="loa__item">Dolinski K. </li><li class="loa__item">Dwight S.S. </li><li class="loa__item">Eppig J.T. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>The Gene Ontology Consortium</span><br />Gene ontology: tool for the unification of biology.<div class="popup-ref"><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2000; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">25-29</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2F75556&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10802651&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-0034069495&amp;cf=" onclick="newWindow(this.href);return false">Scopus (23857)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2000&pages=25-29&journal=Nat.+Genet.&author=The+Gene+Ontology+Consortium&title=Gene+ontology%3A+tool+for+the+unification+of+biology">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) using the signed log P values (uncorrected) from the Spearman rank correlations of protein TMT log ratios with PD-L1 for each set of samples. For pairwise Spearman-rank correlation analysis within the luminal and basal PAM50 subsets, pathway scores were calculated as the mean of all TMT log ratios for proteins in each set for a given sample (<a href="/cms/10.1016/j.cell.2020.10.036/attachment/eb237fc7-15fb-4b8d-bf02-cae5f285b239/mmc6" class="scroll-into-link">Table S6</a>). Gene sets for this analysis included the aforementioned immune modulator sets (<div class="dropBlock reference-citations"><a href="#bib128" id="back-bib128" data-db-target-for="bib128-d13420944e4887" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib128">Thorsson et al., 2019</a><div data-db-target-of="bib128-d13420944e4887" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Thorsson V. </li><li class="loa__item">Gibbs D.L. </li><li class="loa__item">Brown S.D. </li><li class="loa__item">Wolf D. </li><li class="loa__item">Bortone D.S. </li><li class="loa__item">Ou Yang T.-H. </li><li class="loa__item">Porta-Pardo E. </li><li class="loa__item">Gao G.F. </li><li class="loa__item">Plaisier C.L. </li><li class="loa__item">Eddy J.A. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />The Immune Landscape of Cancer.<div class="popup-ref"><span class="ref__series"><i>Immunity.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">51</span></span>: <span class="ref__seriesPages">411-412</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.immuni.2019.08.004&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.immuni.2019.08.004&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.immuni.2019.08.004&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31433971&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85070213901&amp;cf=" onclick="newWindow(this.href);return false">Scopus (61)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2019&pages=411-412&journal=Immunity&author=Cancer+Genome+Atlas+Research+Network&title=The+Immune+Landscape+of+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), GO biological process sets for nucleotide excision repair (GO:0006289), mRNA processing (GO:0006397), and RNA splicing (GO:0006397) and unique proteins from DNA repair pathway sets (also used for analysis in <a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>) for base excision repair, direct repair, DNA damage checkpoint signaling, Fanconi anemia pathway, homologous recombination, mismatch repair, non-homologous end joining, nucleotide excision repair, and translesion synthesis defined by <div class="dropBlock reference-citations"><a href="#bib5" id="back-bib5" data-db-target-for="bib5-d13420944e4893" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib5">Anurag et al., 2018b</a><div data-db-target-of="bib5-d13420944e4893" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Anurag M. </li><li class="loa__item">Ellis M.J. </li><li class="loa__item">Haricharan S. </li></ul><div class="reference-citations__detials">DNA damage repair defects as a new class of endocrine treatment resistance driver.<div class="popup-ref"><span class="ref__series"><i>Oncotarget.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">36252-36253</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.18632%2Foncotarget.26363&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30555626&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85057030841&amp;cf=" onclick="newWindow(this.href);return false">Scopus (14)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=36252-36253&journal=Oncotarget&author=Anurag+M.&author=Ellis+M.J.&author=Haricharan+S.&title=DNA+damage+repair+defects+as+a+new+class+of+endocrine+treatment+resistance+driver">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>. Pairwise Spearman-rank correlation analysis was repeated for the retrospective cohort (<div class="dropBlock reference-citations"><a href="#bib85" id="back-bib85" data-db-target-for="bib85-d13420944e4896" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib85">Mertins et al., 2016</a><div data-db-target-of="bib85-d13420944e4896" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Wang P. </li><li class="loa__item">Wang X. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Cao S. </li><li class="loa__item">Petralia F. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomics connects somatic mutations to signalling in breast cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (780)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=NCI+CPTAC&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) using scores generated by averaging protein data for the same protein sets and from running CIBERSORT (<div class="dropBlock reference-citations"><a href="#bib92" id="back-bib92" data-db-target-for="bib92-d13420944e4900" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib92">Newman et al., 2015</a><div data-db-target-of="bib92-d13420944e4900" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Newman A.M. </li><li class="loa__item">Liu C.L. </li><li class="loa__item">Green M.R. </li><li class="loa__item">Gentles A.J. </li><li class="loa__item">Feng W. </li><li class="loa__item">Xu Y. </li><li class="loa__item">Hoang C.D. </li><li class="loa__item">Diehn M. </li><li class="loa__item">Alizadeh A.A. </li></ul><div class="reference-citations__detials">Robust enumeration of cell subsets from tissue expression profiles.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">453-457</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnmeth.3337&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=25822800&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84928927858&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2633)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=453-457&journal=Nat.+Methods&author=Newman+A.M.&author=Liu+C.L.&author=Green+M.R.&author=Gentles+A.J.&author=Feng+W.&author=Xu+Y.&author=Hoang+C.D.&author=Diehn+M.&author=Alizadeh+A.A.&title=Robust+enumeration+of+cell+subsets+from+tissue+expression+profiles">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) on RPKM RNA-seq data downloaded from LinkedOmics.</div></section><section id="sec4.5.3.17" data-locator="sec4.5.3.17" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">DNA damage response score</span></h4><div class="section-paragraph">To estimate the activity of the DNA double-stranded break response (DDR) pathway, we focused on phosphopeptide abundance of SQ/TQ sites that have been previously shown to increase in abundance following irradiation-induced double stranded breaks (<div class="dropBlock reference-citations"><a href="#bib79" id="back-bib79" data-db-target-for="bib79-d13420944e4908" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib79">Matsuoka et al., 2007</a><div data-db-target-of="bib79-d13420944e4908" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Matsuoka S. </li><li class="loa__item">Ballif B.A. </li><li class="loa__item">Smogorzewska A. </li><li class="loa__item">McDonald 3rd, E.R. </li><li class="loa__item">Hurov K.E. </li><li class="loa__item">Luo J. </li><li class="loa__item">Bakalarski C.E. </li><li class="loa__item">Zhao Z. </li><li class="loa__item">Solimini N. </li><li class="loa__item">Lerenthal Y. </li><li>et al.</li></ul><div class="reference-citations__detials">ATM and ATR substrate analysis reveals extensive protein networks responsive to DNA damage.<div class="popup-ref"><span class="ref__series"><i>Science.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">316</span></span>: <span class="ref__seriesPages">1160-1166</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1126%2Fscience.1140321&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=17525332&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-34249947699&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2181)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=316&publication_year=2007&pages=1160-1166&journal=Science&author=Matsuoka+S.&author=Ballif+B.A.&author=Smogorzewska+A.&author=McDonald+E.R.&author=Hurov+K.E.&author=Luo+J.&author=Bakalarski+C.E.&author=Zhao+Z.&author=Solimini+N.&author=Lerenthal+Y.&title=ATM+and+ATR+substrate+analysis+reveals+extensive+protein+networks+responsive+to+DNA+damage">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). We found phosphopeptides from <div class="dropBlock reference-citations"><a href="#bib79" id="back-bib79" data-db-target-for="bib79-d13420944e4911" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib79">Matsuoka et al., 2007</a><div data-db-target-of="bib79-d13420944e4911" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Matsuoka S. </li><li class="loa__item">Ballif B.A. </li><li class="loa__item">Smogorzewska A. </li><li class="loa__item">McDonald 3rd, E.R. </li><li class="loa__item">Hurov K.E. </li><li class="loa__item">Luo J. </li><li class="loa__item">Bakalarski C.E. </li><li class="loa__item">Zhao Z. </li><li class="loa__item">Solimini N. </li><li class="loa__item">Lerenthal Y. </li><li>et al.</li></ul><div class="reference-citations__detials">ATM and ATR substrate analysis reveals extensive protein networks responsive to DNA damage.<div class="popup-ref"><span class="ref__series"><i>Science.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">316</span></span>: <span class="ref__seriesPages">1160-1166</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1126%2Fscience.1140321&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=17525332&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-34249947699&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2181)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=316&publication_year=2007&pages=1160-1166&journal=Science&author=Matsuoka+S.&author=Ballif+B.A.&author=Smogorzewska+A.&author=McDonald+E.R.&author=Hurov+K.E.&author=Luo+J.&author=Bakalarski+C.E.&author=Zhao+Z.&author=Solimini+N.&author=Lerenthal+Y.&title=ATM+and+ATR+substrate+analysis+reveals+extensive+protein+networks+responsive+to+DNA+damage">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> that were also detected in our study by matching peptide sequences (N = 297). Since DDR often increases target peptide phosphorylation from undetectable to highly abundant, we converted values into up and down outliers using BlackSheep (described above). DDR score was the mean of outlier values for the DDR peptides per sample.</div></section><section id="sec4.5.3.18" data-locator="sec4.5.3.18" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Chromosome instability score</span></h4><div class="section-paragraph">The Chromosome instability (CIN) score was used to summarize the genome-wide SCNA intensity. From the SCNA segmentation results, we used a straightforward weighted-sum approach to derive the CIN score for each sample as described in <div class="dropBlock reference-citations"><a href="#bib135" id="back-bib135" data-db-target-for="bib135-d13420944e4919" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib135">Vasaikar et al., 2019</a><div data-db-target-of="bib135-d13420944e4919" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Vasaikar S. </li><li class="loa__item">Huang C. </li><li class="loa__item">Wang X. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Savage S.R. </li><li class="loa__item">Wen B. </li><li class="loa__item">Dou Y. </li><li class="loa__item">Zhang Y. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Arshad O.A. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Clinical Proteomic Tumor Analysis Consortium</span><br />Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">177</span></span>: <span class="ref__seriesPages">1035-1049.e19</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31031003&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85064886726&amp;cf=" onclick="newWindow(this.href);return false">Scopus (175)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=177&publication_year=2019&pages=1035-1049.e19&journal=Cell&author=Clinical+Proteomic+Tumor+Analysis+Consortium&title=Proteogenomic+Analysis+of+Human+Colon+Cancer+Reveals+New+Therapeutic+Opportunities">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>. Specifically, the absolute log2 ratios of all segments (indicating the copy number alteration of these segments) within a chromosome were summed, while being weighted by the segment length to derive the instability score for the chromosome. The genome-wide chromosome instability index was further derived by summing the instability score of all 22 autosomes.</div></section><section id="sec4.5.3.19" data-locator="sec4.5.3.19" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Determination of stemness score</span></h4><div class="section-paragraph">Stemness scores were calculated as previously described (<div class="dropBlock reference-citations"><a href="#bib77" id="back-bib77" data-db-target-for="bib77-d13420944e4927" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib77">Malta et al., 2018</a><div data-db-target-of="bib77-d13420944e4927" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Malta T.M. </li><li class="loa__item">Sokolov A. </li><li class="loa__item">Gentles A.J. </li><li class="loa__item">Burzykowski T. </li><li class="loa__item">Poisson L. </li><li class="loa__item">Weinstein J.N. </li><li class="loa__item">Kamińska B. </li><li class="loa__item">Huelsken J. </li><li class="loa__item">Omberg L. </li><li class="loa__item">Gevaert O. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />Machine Learning Identifies Stemness Features Associated with Oncogenic Dedifferentiation.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">173</span></span>: <span class="ref__seriesPages">338-354.e15</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2018.03.034&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2018.03.034&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2018.03.034&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29625051&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85044967234&amp;cf=" onclick="newWindow(this.href);return false">Scopus (363)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=338-354.e15&journal=Cell&author=Cancer+Genome+Atlas+Research+Network&title=Machine+Learning+Identifies+Stemness+Features+Associated+with+Oncogenic+Dedifferentiation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). First we used MoonlightR (<div class="dropBlock reference-citations"><a href="#bib31" id="back-bib31" data-db-target-for="bib31-d13420944e4930" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib31">Colaprico et al., 2020</a><div data-db-target-of="bib31-d13420944e4930" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Colaprico A. </li><li class="loa__item">Olsen C. </li><li class="loa__item">Bailey M.H. </li><li class="loa__item">Odom G.J. </li><li class="loa__item">Terkelsen T. </li><li class="loa__item">Silva T.C. </li><li class="loa__item">Olsen A.V. </li><li class="loa__item">Cantini L. </li><li class="loa__item">Zinovyev A. </li><li class="loa__item">Barillot E. </li><li>et al.</li></ul><div class="reference-citations__detials">Interpreting pathways to discover cancer driver genes with Moonlight.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">69</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41467-019-13803-0&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31900418&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85077479923&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=69&journal=Nat.+Commun.&author=Colaprico+A.&author=Olsen+C.&author=Bailey+M.H.&author=Odom+G.J.&author=Terkelsen+T.&author=Silva+T.C.&author=Olsen+A.V.&author=Cantini+L.&author=Zinovyev+A.&author=Barillot+E.&title=Interpreting+pathways+to+discover+cancer+driver+genes+with+Moonlight">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to query, download, and preprocess the pluripotent stem cell samples (ESC and iPSC) from the Progenitor Cell Biology Consortium (PCBC) dataset (<div class="dropBlock reference-citations"><a href="#bib35" id="back-bib35" data-db-target-for="bib35-d13420944e4933" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib35">Daily et al., 2017</a><div data-db-target-of="bib35-d13420944e4933" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Daily K. </li><li class="loa__item">Ho Sui S.J. </li><li class="loa__item">Schriml L.M. </li><li class="loa__item">Dexheimer P.J. </li><li class="loa__item">Salomonis N. </li><li class="loa__item">Schroll R. </li><li class="loa__item">Bush S. </li><li class="loa__item">Keddache M. </li><li class="loa__item">Mayhew C. </li><li class="loa__item">Lotia S. </li><li>et al.</li></ul><div class="reference-citations__detials">Molecular, phenotypic, and sample-associated data to describe pluripotent stem cell lines and derivatives.<div class="popup-ref"><span class="ref__series"><i>Sci. Data.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">170030</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fsdata.2017.30&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=28350385&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85016445988&amp;cf=" onclick="newWindow(this.href);return false">Scopus (24)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2017&pages=170030&journal=Sci.+Data&author=Daily+K.&author=Ho+Sui+S.J.&author=Schriml+L.M.&author=Dexheimer+P.J.&author=Salomonis+N.&author=Schroll+R.&author=Bush+S.&author=Keddache+M.&author=Mayhew+C.&author=Lotia+S.&title=Molecular%2C+phenotypic%2C+and+sample-associated+data+to+describe+pluripotent+stem+cell+lines+and+derivatives">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib112" id="back-bib112" data-db-target-for="bib112-d13420944e4936" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib112">Salomonis et al., 2016</a><div data-db-target-of="bib112-d13420944e4936" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Salomonis N. </li><li class="loa__item">Dexheimer P.J. </li><li class="loa__item">Omberg L. </li><li class="loa__item">Schroll R. </li><li class="loa__item">Bush S. </li><li class="loa__item">Huo J. </li><li class="loa__item">Schriml L. </li><li class="loa__item">Ho Sui S. </li><li class="loa__item">Keddache M. </li><li class="loa__item">Mayhew C. </li><li>et al.</li></ul><div class="reference-citations__detials">Integrated Genomic Analysis of Diverse Induced Pluripotent Stem Cells from the Progenitor Cell Biology Consortium.<div class="popup-ref"><span class="ref__series"><i>Stem Cell Reports.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">110-125</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.stemcr.2016.05.006&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.stemcr.2016.05.006&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.stemcr.2016.05.006&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27293150&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=110-125&journal=Stem+Cell+Reports&author=Salomonis+N.&author=Dexheimer+P.J.&author=Omberg+L.&author=Schroll+R.&author=Bush+S.&author=Huo+J.&author=Schriml+L.&author=Ho+Sui+S.&author=Keddache+M.&author=Mayhew+C.&title=Integrated+Genomic+Analysis+of+Diverse+Induced+Pluripotent+Stem+Cells+from+the+Progenitor+Cell+Biology+Consortium">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Second, to calculate the stemness scores based on mRNA expression, we built a predictive model using one-class logistic regression (OCLR) (<div class="dropBlock reference-citations"><a href="#bib120" id="back-bib120" data-db-target-for="bib120-d13420944e4939" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib120">Sokolov et al., 2016</a><div data-db-target-of="bib120-d13420944e4939" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Sokolov A. </li><li class="loa__item">Paull E.O. </li><li class="loa__item">Stuart J.M. </li></ul><div class="reference-citations__detials">ONE-CLASS DETECTION OF CELL STATES IN TUMOR SUBTYPES.<div class="popup-ref"><span class="ref__series"><i>Pac. Symp. Biocomput.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">21</span></span>: <span class="ref__seriesPages">405-416</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26776204&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=405-416&journal=Pac.+Symp.+Biocomput.&author=Sokolov+A.&author=Paull+E.O.&author=Stuart+J.M.&title=ONE-CLASS+DETECTION+OF+CELL+STATES+IN+TUMOR+SUBTYPES">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) on PCBC dataset.</div><div class="section-paragraph">For mRNA expression-based signatures, to ensure compatibility with the CPTAC BRCA cohort, we first mapped the gene names from Ensembl IDs to Human Genome Organization (HUGO), dropping any genes that had no such mapping. The resulting training matrix contained 12,954 mRNA expression values measured across all available PCBC samples. We used gene-centered FPKM mRNA expression values for all CPTAC BRCA tumors to generate the mRNASi (mRNA stemness index) for each sample. We used the function TCGAanalyze_Stemness from the package TCGAbiolinks (<div class="dropBlock reference-citations"><a href="#bib30" id="back-bib30" data-db-target-for="bib30-d13420944e4944" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib30">Colaprico et al., 2016</a><div data-db-target-of="bib30-d13420944e4944" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Colaprico A. </li><li class="loa__item">Silva T.C. </li><li class="loa__item">Olsen C. </li><li class="loa__item">Garofano L. </li><li class="loa__item">Cava C. </li><li class="loa__item">Garolini D. </li><li class="loa__item">Sabedot T.S. </li><li class="loa__item">Malta T.M. </li><li class="loa__item">Pagnotta S.M. </li><li class="loa__item">Castiglioni I. </li><li>et al.</li></ul><div class="reference-citations__detials">TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">e71</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgkv1507&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26704973&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84966269471&amp;cf=" onclick="newWindow(this.href);return false">Scopus (839)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=e71&journal=Nucleic+Acids+Res.&author=Colaprico+A.&author=Silva+T.C.&author=Olsen+C.&author=Garofano+L.&author=Cava+C.&author=Garolini+D.&author=Sabedot+T.S.&author=Malta+T.M.&author=Pagnotta+S.M.&author=Castiglioni+I.&title=TCGAbiolinks%3A+an+R%2FBioconductor+package+for+integrative+analysis+of+TCGA+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) following our previously-described workflow (<div class="dropBlock reference-citations"><a href="#bib89" id="back-bib89" data-db-target-for="bib89-d13420944e4947" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib89">Mounir et al., 2019</a><div data-db-target-of="bib89-d13420944e4947" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mounir M. </li><li class="loa__item">Lucchetta M. </li><li class="loa__item">Silva T.C. </li><li class="loa__item">Olsen C. </li><li class="loa__item">Bontempi G. </li><li class="loa__item">Chen X. </li><li class="loa__item">Noushmehr H. </li><li class="loa__item">Colaprico A. </li><li class="loa__item">Papaleo E. </li></ul><div class="reference-citations__detials">New functionalities in the TCGAbiolinks package for the study and integration of cancer data from GDC and GTEx.<div class="popup-ref"><span class="ref__series"><i>PLoS Comput. Biol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">e1006701</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1371%2Fjournal.pcbi.1006701&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30835723&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85062948513&amp;cf=" onclick="newWindow(this.href);return false">Scopus (88)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=e1006701&journal=PLoS+Comput.+Biol.&author=Mounir+M.&author=Lucchetta+M.&author=Silva+T.C.&author=Olsen+C.&author=Bontempi+G.&author=Chen+X.&author=Noushmehr+H.&author=Colaprico+A.&author=Papaleo+E.&title=New+functionalities+in+the+TCGAbiolinks+package+for+the+study+and+integration+of+cancer+data+from+GDC+and+GTEx">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), with “stemSig” argument set to PCBC_stemSig.</div></section><section id="sec4.5.3.20" data-locator="sec4.5.3.20" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">CDK4/6-related cell cycle analysis</span></h4><div class="section-paragraph">Multi-Gene Proliferation Scores (MGPS) were calculated from the median-MAD normalized RNA-seq data as described previously (<div class="dropBlock reference-citations"><a href="#bib39" id="back-bib39" data-db-target-for="bib39-d13420944e4955" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib39">Ellis et al., 2017</a><div data-db-target-of="bib39-d13420944e4955" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ellis M.J. </li><li class="loa__item">Suman V.J. </li><li class="loa__item">Hoog J. </li><li class="loa__item">Goncalves R. </li><li class="loa__item">Sanati S. </li><li class="loa__item">Creighton C.J. </li><li class="loa__item">DeSchryver K. </li><li class="loa__item">Crouch E. </li><li class="loa__item">Brink A. </li><li class="loa__item">Watson M. </li><li>et al.</li></ul><div class="reference-citations__detials">Ki67 Proliferation Index as a Tool for Chemotherapy Decisions During and After Neoadjuvant Aromatase Inhibitor Treatment of Breast Cancer: Results From the American College of Surgeons Oncology Group Z1031 Trial (Alliance).<div class="popup-ref"><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">1061-1069</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1200%2FJCO.2016.69.4406&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=28045625&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85016622416&amp;cf=" onclick="newWindow(this.href);return false">Scopus (127)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2017&pages=1061-1069&journal=J.%C2%A0Clin.+Oncol.&author=Ellis+M.J.&author=Suman+V.J.&author=Hoog+J.&author=Goncalves+R.&author=Sanati+S.&author=Creighton+C.J.&author=DeSchryver+K.&author=Crouch+E.&author=Brink+A.&author=Watson+M.&title=Ki67+Proliferation+Index+as+a+Tool+for+Chemotherapy+Decisions+During+and+After+Neoadjuvant+Aromatase+Inhibitor+Treatment+of+Breast+Cancer%3A+Results+From+the+American+College+of+Surgeons+Oncology+Group+Z1031+Trial+%28Alliance%29">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Briefly, MGPS was calculated as the mean expression level of all cell cycle-regulated genes identified by <div class="dropBlock reference-citations"><a href="#bib142" id="back-bib142" data-db-target-for="bib142-d13420944e4958" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib142">Whitfield et al., 2002</a><div data-db-target-of="bib142-d13420944e4958" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Whitfield M.L. </li><li class="loa__item">Sherlock G. </li><li class="loa__item">Saldanha A.J. </li><li class="loa__item">Murray J.I. </li><li class="loa__item">Ball C.A. </li><li class="loa__item">Alexander K.E. </li><li class="loa__item">Matese J.C. </li><li class="loa__item">Perou C.M. </li><li class="loa__item">Hurt M.M. </li><li class="loa__item">Brown P.O. </li><li class="loa__item">Botstein D. </li></ul><div class="reference-citations__detials">Identification of genes periodically expressed in the human cell cycle and their expression in tumors.<div class="popup-ref"><span class="ref__series"><i>Mol. Biol. Cell.</i> <span class="ref__seriesDate">2002; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">1977-2000</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1091%2Fmbc.02-02-0030.&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=12058064&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-0035985177&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1127)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2002&pages=1977-2000&journal=Mol.+Biol.+Cell&author=Whitfield+M.L.&author=Sherlock+G.&author=Saldanha+A.J.&author=Murray+J.I.&author=Ball+C.A.&author=Alexander+K.E.&author=Matese+J.C.&author=Perou+C.M.&author=Hurt+M.M.&author=Brown+P.O.&author=Botstein+D.&title=Identification+of+genes+periodically+expressed+in+the+human+cell+cycle+and+their+expression+in+tumors">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> in each sample. Apoptosis and E2F target gene scores were the ssGSEA normalized enrichment scores from the corresponding MSigDB Hallmark gene sets calculated above (Pathway projection using ssGSEA). Likewise, CDK1-7 and CDK9 target site/activity scores were the PTM-SEA scores calculated above for ssGSEA enrichment of PhosphositePlus (<div class="dropBlock reference-citations"><a href="#bib50" id="back-bib50" data-db-target-for="bib50-d13420944e4961" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib50">Hornbeck et al., 2015</a><div data-db-target-of="bib50-d13420944e4961" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hornbeck P.V. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Murray B. </li><li class="loa__item">Kornhauser J.M. </li><li class="loa__item">Latham V. </li><li class="loa__item">Skrzypek E. </li></ul><div class="reference-citations__detials">PhosphoSitePlus, 2014: mutations, PTMs and recalibrations.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">43</span></span>: <span class="ref__seriesPages">D512-D520</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgku1267&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=25514926&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84941120469&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1289)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=D512-D520&journal=Nucleic+Acids+Res.&author=Hornbeck+P.V.&author=Zhang+B.&author=Murray+B.&author=Kornhauser+J.M.&author=Latham+V.&author=Skrzypek+E.&title=PhosphoSitePlus%2C+2014%3A+mutations%2C+PTMs+and+recalibrations">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) target sites for each of these kinases (Kinase activity prediction via PTM-SEA). TNBCtype (<div class="dropBlock reference-citations"><a href="#bib22" id="back-bib22" data-db-target-for="bib22-d13420944e4964" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib22">Chen et al., 2012</a><div data-db-target-of="bib22-d13420944e4964" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Chen X. </li><li class="loa__item">Li J. </li><li class="loa__item">Gray W.H. </li><li class="loa__item">Lehmann B.D. </li><li class="loa__item">Bauer J.A. </li><li class="loa__item">Shyr Y. </li><li class="loa__item">Pietenpol J.A. </li></ul><div class="reference-citations__detials">TNBCtype: A Subtyping Tool for Triple-Negative Breast Cancer.<div class="popup-ref"><span class="ref__series"><i>Cancer Inform.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">147-156</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.4137%2FCIN.S9983&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=22872785&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84864579879&amp;cf=" onclick="newWindow(this.href);return false">Scopus (144)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=147-156&journal=Cancer+Inform.&author=Chen+X.&author=Li+J.&author=Gray+W.H.&author=Lehmann+B.D.&author=Bauer+J.A.&author=Shyr+Y.&author=Pietenpol+J.A.&title=TNBCtype%3A+A+Subtyping+Tool+for+Triple-Negative+Breast+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) was applied to assign triple-negative breast cancer samples to the four TNBC subtypes (BL1, BL2, M and LAR) based on RNaseq FPKM data (<div class="dropBlock reference-citations"><a href="#bib68" id="back-bib68" data-db-target-for="bib68-d13420944e4967" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib68">Lehmann et al., 2011</a><div data-db-target-of="bib68-d13420944e4967" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lehmann B.D. </li><li class="loa__item">Bauer J.A. </li><li class="loa__item">Chen X. </li><li class="loa__item">Sanders M.E. </li><li class="loa__item">Chakravarthy A.B. </li><li class="loa__item">Shyr Y. </li><li class="loa__item">Pietenpol J.A. </li></ul><div class="reference-citations__detials">Identification of human triple-negative breast cancer subtypes and preclinical models for selection of targeted therapies.<div class="popup-ref"><span class="ref__series"><i>J. Clin. Invest.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">121</span></span>: <span class="ref__seriesPages">2750-2767</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1172%2FJCI45014&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=21633166&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-79960015997&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2925)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=121&publication_year=2011&pages=2750-2767&journal=J.%C2%A0Clin.+Invest.&author=Lehmann+B.D.&author=Bauer+J.A.&author=Chen+X.&author=Sanders+M.E.&author=Chakravarthy+A.B.&author=Shyr+Y.&author=Pietenpol+J.A.&title=Identification+of+human+triple-negative+breast+cancer+subtypes+and+preclinical+models+for+selection+of+targeted+therapies">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#bib70" id="back-bib70" data-db-target-for="bib70-d13420944e4971" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib70">Lehmann et al., 2016</a><div data-db-target-of="bib70-d13420944e4971" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lehmann B.D. </li><li class="loa__item">Jovanović B. </li><li class="loa__item">Chen X. </li><li class="loa__item">Estrada M.V. </li><li class="loa__item">Johnson K.N. </li><li class="loa__item">Shyr Y. </li><li class="loa__item">Moses H.L. </li><li class="loa__item">Sanders M.E. </li><li class="loa__item">Pietenpol J.A. </li></ul><div class="reference-citations__detials">Refinement of Triple-Negative Breast Cancer Molecular Subtypes: Implications for Neoadjuvant Chemotherapy Selection.<div class="popup-ref"><span class="ref__series"><i>PLoS ONE.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">e0157368</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1371%2Fjournal.pone.0157368&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27310713&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=e0157368&journal=PLoS+ONE&author=Lehmann+B.D.&author=Jovanovi%C4%87+B.&author=Chen+X.&author=Estrada+M.V.&author=Johnson+K.N.&author=Shyr+Y.&author=Moses+H.L.&author=Sanders+M.E.&author=Pietenpol+J.A.&title=Refinement+of+Triple-Negative+Breast+Cancer+Molecular+Subtypes%3A+Implications+for+Neoadjuvant+Chemotherapy+Selection">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div></section><section id="sec4.5.3.21" data-locator="sec4.5.3.21" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">RB1 analysis in Cell lines</span></h4><div class="section-paragraph"><em>RB1</em> mutation status, copy number, and protein abundance for Cancer Cell Line Encyclopedia (CCLE) breast cancer cell lines along with ER and HER2 annotations were downloaded from DepMap (DepMap, Broad (2020): DepMap 20Q2 Public. figshare. Dataset. <a class="inter-ref-link" href="https://figshare.com/articles/DepMap_20Q2_Public/12280541/4" target="_blank">https://figshare.com/articles/DepMap_20Q2_Public/12280541/4</a>; <div class="dropBlock reference-citations"><a href="#bib43" id="back-bib43" data-db-target-for="bib43-d13420944e4983" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib43">Ghandi et al., 2019</a><div data-db-target-of="bib43-d13420944e4983" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ghandi M. </li><li class="loa__item">Huang F.W. </li><li class="loa__item">Jané-Valbuena J. </li><li class="loa__item">Kryukov G.V. </li><li class="loa__item">Lo C.C. </li><li class="loa__item">McDonald 3rd, E.R. </li><li class="loa__item">Barretina J. </li><li class="loa__item">Gelfand E.T. </li><li class="loa__item">Bielski C.M. </li><li class="loa__item">Li H. </li><li>et al.</li></ul><div class="reference-citations__detials">Next-generation characterization of the Cancer Cell Line Encyclopedia.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">569</span></span>: <span class="ref__seriesPages">503-508</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41586-019-1186-3&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31068700&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85065525804&amp;cf=" onclick="newWindow(this.href);return false">Scopus (624)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=569&publication_year=2019&pages=503-508&journal=Nature&author=Ghandi+M.&author=Huang+F.W.&author=Jan%C3%A9-Valbuena+J.&author=Kryukov+G.V.&author=Lo+C.C.&author=McDonald+E.R.&author=Barretina+J.&author=Gelfand+E.T.&author=Bielski+C.M.&author=Li+H.&title=Next-generation+characterization+of+the+Cancer+Cell+Line+Encyclopedia">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib94" id="back-bib94" data-db-target-for="bib94-d13420944e4986" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib94">Nusinow et al., 2020</a><div data-db-target-of="bib94-d13420944e4986" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Nusinow D.P. </li><li class="loa__item">Szpyt J. </li><li class="loa__item">Ghandi M. </li><li class="loa__item">Rose C.M. </li><li class="loa__item">McDonald 3rd, E.R. </li><li class="loa__item">Kalocsay M. </li><li class="loa__item">Jané-Valbuena J. </li><li class="loa__item">Gelfand E. </li><li class="loa__item">Schweppe D.K. </li><li class="loa__item">Jedrychowski M. </li><li>et al.</li></ul><div class="reference-citations__detials">Quantitative Proteomics of the Cancer Cell Line Encyclopedia.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">180</span></span>: <span class="ref__seriesPages">387-402.e16</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2019.12.023&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2019.12.023&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2019.12.023&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31978347&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85077916308&amp;cf=" onclick="newWindow(this.href);return false">Scopus (109)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=180&publication_year=2020&pages=387-402.e16&journal=Cell&author=Nusinow+D.P.&author=Szpyt+J.&author=Ghandi+M.&author=Rose+C.M.&author=McDonald+E.R.&author=Kalocsay+M.&author=Jan%C3%A9-Valbuena+J.&author=Gelfand+E.&author=Schweppe+D.K.&author=Jedrychowski+M.&title=Quantitative+Proteomics+of+the+Cancer+Cell+Line+Encyclopedia">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Area Under the Curve (AUC) drug responses to a CDK4/6 inhibitor, palbociclib, were retrieved from the Sanger/Massachusetts General Hospital Genomics of Drug Sensitivity Dataset 1 (<div class="dropBlock reference-citations"><a href="#bib53" id="back-bib53" data-db-target-for="bib53-d13420944e4989" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib53">Iorio et al., 2016</a><div data-db-target-of="bib53-d13420944e4989" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Iorio F. </li><li class="loa__item">Knijnenburg T.A. </li><li class="loa__item">Vis D.J. </li><li class="loa__item">Bignell G.R. </li><li class="loa__item">Menden M.P. </li><li class="loa__item">Schubert M. </li><li class="loa__item">Aben N. </li><li class="loa__item">Gonçalves E. </li><li class="loa__item">Barthorpe S. </li><li class="loa__item">Lightfoot H. </li><li>et al.</li></ul><div class="reference-citations__detials">A Landscape of Pharmacogenomic Interactions in Cancer.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">166</span></span>: <span class="ref__seriesPages">740-754</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2016.06.017&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2016.06.017&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2016.06.017&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27397505&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84979649916&amp;cf=" onclick="newWindow(this.href);return false">Scopus (702)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2016&pages=740-754&journal=Cell&author=Iorio+F.&author=Knijnenburg+T.A.&author=Vis+D.J.&author=Bignell+G.R.&author=Menden+M.P.&author=Schubert+M.&author=Aben+N.&author=Gon%C3%A7alves+E.&author=Barthorpe+S.&author=Lightfoot+H.&title=A+Landscape+of+Pharmacogenomic+Interactions+in+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib145" id="back-bib145" data-db-target-for="bib145-d13420944e4992" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib145">Yang et al., 2013</a><div data-db-target-of="bib145-d13420944e4992" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Yang W. </li><li class="loa__item">Soares J. </li><li class="loa__item">Greninger P. </li><li class="loa__item">Edelman E.J. </li><li class="loa__item">Lightfoot H. </li><li class="loa__item">Forbes S. </li><li class="loa__item">Bindal N. </li><li class="loa__item">Beare D. </li><li class="loa__item">Smith J.A. </li><li class="loa__item">Thompson I.R. </li><li>et al.</li></ul><div class="reference-citations__detials">Genomics of Drug Sensitivity in Cancer (GDSC): a resource for therapeutic biomarker discovery in cancer cells.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">41</span></span>: <span class="ref__seriesPages">D955-D961</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgks1111&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=23180760&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84876563391&amp;cf=" onclick="newWindow(this.href);return false">Scopus (893)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=D955-D961&journal=Nucleic+Acids+Res.&author=Yang+W.&author=Soares+J.&author=Greninger+P.&author=Edelman+E.J.&author=Lightfoot+H.&author=Forbes+S.&author=Bindal+N.&author=Beare+D.&author=Smith+J.A.&author=Thompson+I.R.&title=Genomics+of+Drug+Sensitivity+in+Cancer+%28GDSC%29%3A+a+resource+for+therapeutic+biomarker+discovery+in+cancer+cells">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). High AUC values indicate low sensitivity to the drug while low AUC values indicate high sensitivity. Cell lines with <em>RB1</em> gene level copy number &lt; –1 or having a deletion-causing frameshift mutation were categorized as <em>RB1</em> deleted/frameshift. Cell lines with an in-frame deletion or missense mutations in <em>RB1</em> were categorized as <em>RB1</em> missense. All other cell lines were <em>RB1</em> WT. For <a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>D, a Kruskal-Wallis test was performed to test for differences in palbociclib response among cell lines stratified by <em>RB1</em> status and ER/HER2 subtypes. For <a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>E, Spearman’s correlation coefficient was calculated using cell lines with RB1 protein measurements from <a data-link="modal" data-target="#image-S0092-8674(20)31400-8gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>D to test the association between RB1 protein abundance and palbociclib response.</div></section></section></section><section id="sec4.6" data-locator="sec4.6" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Additional Resources</span></h3><div class="section-paragraph">CPTAC program website, detailing program initiatives, investigators, and datasets, is available at <a class="inter-ref-link" href="https://proteomics.cancer.gov/programs/cptac" target="_blank">https://proteomics.cancer.gov/programs/cptac</a>.</div></section></section><section id="sec5" data-locator="sec5" tabindex="0"><h2 class="top" id="secsectitle0395" data-left-hand-nav="Consortia" tabindex="0"><span class="top__sub"></span><span class="top__text">Consortia</span></h2><div class="section-paragraph">The members of the Clinical Proteomic Tumor Analysis Consortium are Meenakshi Anurag, Shayan C. Avanessian, Erik Bergstrom, Chet Birger, Lili Blumenberg, Emily Boja, Shuang Cai, Song Cao, Steven A. Carr, Daniel Chan, Xian Chen, Karl R. Clauser, Antonio Colaprico, Li Ding, Yongchao Dou, Nathan J. Edwards, Matthew J. Ellis, David Fenyö, Yifat Geffen, Gad Getz, Michael A. Gillette, David I. Heiman, Tara Hiltke, Andrew N. Hoofnagle, Chen Huang, Eric J. Jaehnig, M. Harry Kane, Alla Karpova, Karen A. Ketchum, Christopher R. Kinsinger, Ramani B. Kothadia, Karsten Krug, Eric Kuhn, Jonathan T. Lei, Douglas A. Levine, Shunqiang Li, Yuxing Liao, Daniel C. Liebler, Tao Liu, Jingqin Luo, Subha Madhavan, Chris Maher, D. R. Mani, Yosef Maruvka, Jason E. McDermott, Peter B. McGarvey, Philipp Mertins, Mehdi Mesri, George Miles, Mauricio Oberti, Akhilesh Pandey, Samuel H. Payne, David F. Ransohoff, Robert C. Rivers, Ana I. Robles, Karin D. Rodland, Henry Rodriguez, Paul Rudnick, Kelly V. Ruggles, Melinda E. Sanders, Shankha Satpathy, Kenna M. Shaw, Zhiao Shi, Ie-Ming Shih, Robbert J. C. Slebos, Richard D. Smith, Michael Snyder, Stephen E. Stein, David L. Tabb, Lauren C. Tang, Ratna R. Thangudu, Mathangi Thiagarajan, Stefani Thomas, Jarey Wang, Yue Wang, Bo Wen, Thomas F. Westbrook, Forest M. White, Jeffrey R. Whiteaker, Gordon A. Whiteley, Maciej Wiznerowicz, Matthew A. Wyczalkowski, Bing Zhang, Hui Zhang, Zhen Zhang, Yingming Zhao, Heng Zhu, and Lisa J. Zimmerman.</div></section><section><h2 class="top" id="secsectitle0400" data-left-hand-nav="Acknowledgments" tabindex="0"><span class="top__sub"></span><span class="top__text">Acknowledgments</span></h2><div class="section-paragraph">This work was supported by the National Cancer Institute (NCI) Clinical Proteomic Tumor Analysis Consortium through grants U24CA160034 (to S.A.C.), U24CA210986 (to S.A.C. and M.A.G.), U01CA214125 (to M.J.E. and S.A.C.), U24CA210979 (to D.R.M. and G.G.), U24CA210954 (to B.Z.), and U24CA210972 (to L.D. and D.F.) and by NIH grant T32CA203690 (to J.T.L.). M.J.E. is a Susan G. Komen Foundation Scholar, a McNair Scholar supported by the McNair Medical Institute at The Robert and Janice McNair Foundation, and a recipient of a CPRIT (Cancer Prevention and Research Institute of Texas) Established Investigator Award (RR140027).</div><h3>Author Contributions</h3><div class="section-paragraph">Conception and Design, K.K., S.S., P.M., L.C.T., K.R.C., D.R.M., S.A.C., M.J.E., and M.A.G.; Experiment or Data Collection, S.S., G.M., P.M., and L.C.T.; Computation and Statistical Analysis, K.K., E.J.J., S.S., L.B., A.K., M.A., Y.G., D.I.H., S.C., Y.E.M., J.T.L., C.H., R.B.K., A.C., M.A.W., K.R.C., K.V.R., and D.R.M.; Writing – Original Draft, K.K., E.J.J., S.S., L.B., A.K., M.A., S.A.C., M.J.E., and M.A.G.; Supervision, S.S., C.B., T.F.W., L.D., G.G., D.F., K.V.R., B.Z., D.R.M., S.A.C., M.J.E., and M.A.G.; Administration, C.R.K., T.H., E.S.B., M.M., A.I.R., and H.R. All authors contributed to data interpretation and review and editing of the manuscript.</div><h3>Declaration of Interests</h3><div class="section-paragraph">M.J.E reports ownership and royalties associated with Bioclassifier LLC through sales by Nanostring LLC and Veracyte for the “Prosigna” breast cancer prognostic test. He also reports ad hoc consulting for AstraZeneca, Foundation Medicine, G1 Therapeutics, Novartis, Sermonix, Abbvie, Lilly and Pfizer. B.Z. has received research funding from Bristol-Myers Squibb. S.A.C. is a scientific advisory board member of Kymera, PTM BioLabs, and Seer and ad hoc consultant to Pfizer and Biogen.</div></section><section id="app2" data-locator="app2" tabindex="0"><h2 class="top" id="supplementaryMaterial" data-left-hand-nav="Supplemental Information" tabindex="0"><span class="top__sub"></span><span class="top__text">Supplemental Information</span></h2><div class="section-paragraph"><ul class="rlist"><li class="supplemental-information__item" id="mmc1"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.cell.2020.10.036/attachment/3dbb5a07-b92f-4ae5-9c88-f84c3dca95db/mmc1.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (2.38
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S1. Metadata, DNA QC, and RNA QC for 122 Tumors Analyzed in This Study, Related to Figure 1</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc2"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.cell.2020.10.036/attachment/a1bc3f3e-eeac-4605-a9ef-839abd334dcd/mmc2.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (85.02
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S2. Data Tables with Proteome, Phosphosite, Acetylsite, mRNA, and SNCA Abundances, Related to Figures 1, 2, 3, 4, 5, 6, and S1–S7</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc3"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.cell.2020.10.036/attachment/3870b80d-1b59-4995-a804-71c49e83c458/mmc3.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (4.36
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S3. Results of Several PG Analyses, Related to Figure S2 and Table 1</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc4"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.cell.2020.10.036/attachment/d7a1a20c-1c01-4f40-98d9-81f918417a6d/mmc4.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (.46
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S4. Characterization of NMF Subtypes, Related to Figures 1 and S3</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc5"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.cell.2020.10.036/attachment/48d23c21-8705-4760-9513-c5982cc559ee/mmc5.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (.1
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S5. Overrepresentation Analysis of the HER2-Inclusive Subtype, Related to Figures 1 and S3</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc6"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.cell.2020.10.036/attachment/eb237fc7-15fb-4b8d-bf02-cae5f285b239/mmc6.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (.16
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S6. Immune Signature, DNA Repair Pathway, and MGPSs for Each Sample in the Prospective and Retrospective Datasets, Related to Figures 4 and 5</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc7"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.cell.2020.10.036/attachment/ce88682c-a2ae-458a-a550-73ad6faa7636/mmc7.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (.06
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S7. Spearman-Rank Correlation Rho and p Values for Pairwise Correlations within the Luminal and Basal Samples from the Prospective and Retrospective Datasets and for Correlations with MGPS within HR+/ERBB2 PG− and TNBC Tumors from the Prospective Dataset, Related to Figures 5, 6 and S6</div></li></ul></div></section><div data-pb-dropzone="extra-dropzone" class="extra-dropzone pb-dropzone"><!-- Empty dropzone --></div><section class="refs"><h2 class="top" id="secsectitle0425" data-left-hand-nav="References" tabindex="0"><span class="top__sub"></span><span class="top__text">References</span></h2><ol class="ref__wrapper"><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Alhazzazi T.Y. </li><li class="ref__authors__name">Kamarajan P. </li><li class="ref__authors__name">Xu Y. </li><li class="ref__authors__name">Ai T. </li><li class="ref__authors__name">Chen L. </li><li class="ref__authors__name">Verdin E. </li><li class="ref__authors__name">Kapila Y.L. </li></ul><div class="ref__title">A Novel Sirtuin-3 Inhibitor, LC-0296, Inhibits Cell Survival and Proliferation, and Promotes Apoptosis of Head and Neck Cancer Cells.</div><span class="ref__series"><i>Anticancer Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">36</span></span>: <span class="ref__seriesPages">49-60</span><div class="ref__controls"><a href="#back-bib1" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26722027&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2016&pages=49-60&journal=Anticancer+Res.&author=Alhazzazi+T.Y.&author=Kamarajan+P.&author=Xu+Y.&author=Ai+T.&author=Chen+L.&author=Verdin+E.&author=Kapila+Y.L.&title=A+Novel+Sirtuin-3+Inhibitor%2C+LC-0296%2C+Inhibits+Cell+Survival+and+Proliferation%2C+and+Promotes+Apoptosis+of+Head+and+Neck+Cancer+Cells">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ali I. </li><li class="ref__authors__name">Conrad R.J. </li><li class="ref__authors__name">Verdin E. </li><li class="ref__authors__name">Ott M. </li></ul><div class="ref__title">Lysine Acetylation Goes Global: From Epigenetics to Metabolism and Therapeutics.</div><span class="ref__series"><i>Chem. Rev.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">118</span></span>: <span class="ref__seriesPages">1216-1252</span><div class="ref__controls"><a href="#back-bib2" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85042009680&amp;cf=" onclick="newWindow(this.href);return false">Scopus (112)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29405707&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1021%2Facs.chemrev.7b00181&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=1216-1252&journal=Chem.+Rev.&author=Ali+I.&author=Conrad+R.J.&author=Verdin+E.&author=Ott+M.&title=Lysine+Acetylation+Goes+Global%3A+From+Epigenetics+to+Metabolism+and+Therapeutics">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Angelova M. </li><li class="ref__authors__name">Charoentong P. </li><li class="ref__authors__name">Hackl H. </li><li class="ref__authors__name">Fischer M.L. </li><li class="ref__authors__name">Snajder R. </li><li class="ref__authors__name">Krogsdam A.M. </li><li class="ref__authors__name">Waldner M.J. </li><li class="ref__authors__name">Bindea G. </li><li class="ref__authors__name">Mlecnik B. </li><li class="ref__authors__name">Galon J. </li><li class="ref__authors__name">Trajanoski Z. </li></ul><div class="ref__title">Characterization of the immunophenotypes and antigenomes of colorectal cancers reveals distinct tumor escape mechanisms and novel targets for immunotherapy.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">64</span><div class="ref__controls"><a href="#back-bib3" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84931426001&amp;cf=" onclick="newWindow(this.href);return false">Scopus (283)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=25853550&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2Fs13059-015-0620-6&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=64&journal=Genome+Biol.&author=Angelova+M.&author=Charoentong+P.&author=Hackl+H.&author=Fischer+M.L.&author=Snajder+R.&author=Krogsdam+A.M.&author=Waldner+M.J.&author=Bindea+G.&author=Mlecnik+B.&author=Galon+J.&author=Trajanoski+Z.&title=Characterization+of+the+immunophenotypes+and+antigenomes+of+colorectal+cancers+reveals+distinct+tumor+escape+mechanisms+and+novel+targets+for+immunotherapy">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Anurag M. </li><li class="ref__authors__name">Punturi N. </li><li class="ref__authors__name">Hoog J. </li><li class="ref__authors__name">Bainbridge M.N. </li><li class="ref__authors__name">Ellis M.J. </li><li class="ref__authors__name">Haricharan S. </li></ul><div class="ref__title">Comprehensive Profiling of DNA Repair Defects in Breast Cancer Identifies a Novel Class of Endocrine Therapy Resistance Drivers.</div><span class="ref__series"><i>Clin. Cancer Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">4887-4899</span><div class="ref__controls"><a href="#back-bib4" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85054053323&amp;cf=" onclick="newWindow(this.href);return false">Scopus (33)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29793947&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1158%2F1078-0432.CCR-17-3702&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=4887-4899&journal=Clin.+Cancer+Res.&author=Anurag+M.&author=Punturi+N.&author=Hoog+J.&author=Bainbridge+M.N.&author=Ellis+M.J.&author=Haricharan+S.&title=Comprehensive+Profiling+of+DNA+Repair+Defects+in+Breast+Cancer+Identifies+a+Novel+Class+of+Endocrine+Therapy+Resistance+Drivers">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Anurag M. </li><li class="ref__authors__name">Ellis M.J. </li><li class="ref__authors__name">Haricharan S. </li></ul><div class="ref__title">DNA damage repair defects as a new class of endocrine treatment resistance driver.</div><span class="ref__series"><i>Oncotarget.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">36252-36253</span><div class="ref__controls"><a href="#back-bib5" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85057030841&amp;cf=" onclick="newWindow(this.href);return false">Scopus (14)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30555626&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.18632%2Foncotarget.26363&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=36252-36253&journal=Oncotarget&author=Anurag+M.&author=Ellis+M.J.&author=Haricharan+S.&title=DNA+damage+repair+defects+as+a+new+class+of+endocrine+treatment+resistance+driver">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Anurag M. </li><li class="ref__authors__name">Zhu M. </li><li class="ref__authors__name">Huang C. </li><li class="ref__authors__name">Vasaikar S. </li><li class="ref__authors__name">Wang J. </li><li class="ref__authors__name">Hoog J. </li><li class="ref__authors__name">Burugu S. </li><li class="ref__authors__name">Gao D. </li><li class="ref__authors__name">Suman V. </li><li class="ref__authors__name">Zhang X.H. </li><li>et al.</li></ul><div class="ref__title">Immune Checkpoint Profiles in Luminal B Breast Cancer (Alliance).</div><span class="ref__series"><i>J. Natl. Cancer Inst.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">112</span></span>: <span class="ref__seriesPages">737-746</span><div class="ref__controls"><a href="#back-bib6" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85079588411&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31665365&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fjnci%2Fdjz213&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2020&pages=737-746&journal=J.%C2%A0Natl.+Cancer+Inst&author=Anurag+M.&author=Zhu+M.&author=Huang+C.&author=Vasaikar+S.&author=Wang+J.&author=Hoog+J.&author=Burugu+S.&author=Gao+D.&author=Suman+V.&author=Zhang+X.H.&title=Immune+Checkpoint+Profiles+in+Luminal+B+Breast+Cancer+%28Alliance%29">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Aran D. </li><li class="ref__authors__name">Hu Z. </li><li class="ref__authors__name">Butte A.J. </li></ul><div class="ref__title">xCell: digitally portraying the tissue cellular heterogeneity landscape.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">220</span><div class="ref__controls"><a href="#back-bib7" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85034614124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (627)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29141660&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2Fs13059-017-1349-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=220&journal=Genome+Biol.&author=Aran+D.&author=Hu+Z.&author=Butte+A.J.&title=xCell%3A+digitally+portraying+the+tissue+cellular+heterogeneity+landscape">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Asghar U.S. </li><li class="ref__authors__name">Barr A.R. </li><li class="ref__authors__name">Cutts R. </li><li class="ref__authors__name">Beaney M. </li><li class="ref__authors__name">Babina I. </li><li class="ref__authors__name">Sampath D. </li><li class="ref__authors__name">Giltnane J. </li><li class="ref__authors__name">Lacap J.A. </li><li class="ref__authors__name">Crocker L. </li><li class="ref__authors__name">Young A. </li><li>et al.</li></ul><div class="ref__title">Single-Cell Dynamics Determines Response to CDK4/6 Inhibition in Triple-Negative Breast Cancer.</div><span class="ref__series"><i>Clin. Cancer Res.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">5561-5572</span><div class="ref__controls"><a href="#back-bib8" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85029515087&amp;cf=" onclick="newWindow(this.href);return false">Scopus (111)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=28606920&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1158%2F1078-0432.CCR-17-0369&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=5561-5572&journal=Clin.+Cancer+Res.&author=Asghar+U.S.&author=Barr+A.R.&author=Cutts+R.&author=Beaney+M.&author=Babina+I.&author=Sampath+D.&author=Giltnane+J.&author=Lacap+J.A.&author=Crocker+L.&author=Young+A.&title=Single-Cell+Dynamics+Determines+Response+to+CDK4%2F6+Inhibition+in+Triple-Negative+Breast+Cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ashburner M. </li><li class="ref__authors__name">Ball C.A. </li><li class="ref__authors__name">Blake J.A. </li><li class="ref__authors__name">Botstein D. </li><li class="ref__authors__name">Butler H. </li><li class="ref__authors__name">Cherry J.M. </li><li class="ref__authors__name">Davis A.P. </li><li class="ref__authors__name">Dolinski K. </li><li class="ref__authors__name">Dwight S.S. </li><li class="ref__authors__name">Eppig J.T. </li><li>et al.</li><li>The Gene Ontology Consortium</li></ul><div class="ref__title">Gene ontology: tool for the unification of biology.</div><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2000; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">25-29</span><div class="ref__controls"><a href="#back-bib9" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-0034069495&amp;cf=" onclick="newWindow(this.href);return false">Scopus (23857)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10802651&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2F75556&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2000&pages=25-29&journal=Nat.+Genet.&author=The+Gene+Ontology+Consortium&title=Gene+ontology%3A+tool+for+the+unification+of+biology">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Barbie D.A. </li><li class="ref__authors__name">Tamayo P. </li><li class="ref__authors__name">Boehm J.S. </li><li class="ref__authors__name">Kim S.Y. </li><li class="ref__authors__name">Moody S.E. </li><li class="ref__authors__name">Dunn I.F. </li><li class="ref__authors__name">Schinzel A.C. </li><li class="ref__authors__name">Sandy P. </li><li class="ref__authors__name">Meylan E. </li><li class="ref__authors__name">Scholl C. </li><li>et al.</li></ul><div class="ref__title">Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">462</span></span>: <span class="ref__seriesPages">108-112</span><div class="ref__controls"><a href="#back-bib10" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-70449091786&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1353)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=19847166&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature08460&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=462&publication_year=2009&pages=108-112&journal=Nature&author=Barbie+D.A.&author=Tamayo+P.&author=Boehm+J.S.&author=Kim+S.Y.&author=Moody+S.E.&author=Dunn+I.F.&author=Schinzel+A.C.&author=Sandy+P.&author=Meylan+E.&author=Scholl+C.&title=Systematic+RNA+interference+reveals+that+oncogenic+KRAS-driven+cancers+require+TBK1">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Barker L. </li></ul><div class="ref__title">A comparison of nine confidence intervals for a Poisson parameter when the expected number of events is ≤ 5.</div><span class="ref__series"><i>Am. Stat.</i> <span class="ref__seriesDate">2002; </span><span class="ref__seriesVolume">56</span></span>: <span class="ref__seriesPages">85-89</span><div class="ref__controls"><a href="#back-bib11" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-0035995688&amp;cf=" onclick="newWindow(this.href);return false">Scopus (35)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1198%2F000313002317572736&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2002&pages=85-89&journal=Am.+Stat.&author=Barker+L.&title=A+comparison+of+nine+confidence+intervals+for+a+Poisson+parameter+when+the+expected+number+of+events+is+%E2%89%A4+5">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Becht E. </li><li class="ref__authors__name">Giraldo N.A. </li><li class="ref__authors__name">Lacroix L. </li><li class="ref__authors__name">Buttard B. </li><li class="ref__authors__name">Elarouci N. </li><li class="ref__authors__name">Petitprez F. </li><li class="ref__authors__name">Selves J. </li><li class="ref__authors__name">Laurent-Puig P. </li><li class="ref__authors__name">Sautès-Fridman C. </li><li class="ref__authors__name">Fridman W.H. </li><li class="ref__authors__name">de Reyniès A. </li></ul><div class="ref__title">Erratum to: Estimating the population abundance of tissue-infiltrating immune and stromal cell populations using gene expression.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">249</span><div class="ref__controls"><a href="#back-bib12" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85002746810&amp;cf=" onclick="newWindow(this.href);return false">Scopus (13)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27908289&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2Fs13059-016-1113-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=249&journal=Genome+Biol.&author=Becht+E.&author=Giraldo+N.A.&author=Lacroix+L.&author=Buttard+B.&author=Elarouci+N.&author=Petitprez+F.&author=Selves+J.&author=Laurent-Puig+P.&author=Saut%C3%A8s-Fridman+C.&author=Fridman+W.H.&author=de+Reyni%C3%A8s+A.&title=Erratum+to%3A+Estimating+the+population+abundance+of+tissue-infiltrating+immune+and+stromal+cell+populations+using+gene+expression">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Becht E. </li><li class="ref__authors__name">Giraldo N.A. </li><li class="ref__authors__name">Lacroix L. </li><li class="ref__authors__name">Buttard B. </li><li class="ref__authors__name">Elarouci N. </li><li class="ref__authors__name">Petitprez F. </li><li class="ref__authors__name">Selves J. </li><li class="ref__authors__name">Laurent-Puig P. </li><li class="ref__authors__name">Sautès-Fridman C. </li><li class="ref__authors__name">Fridman W.H. </li><li class="ref__authors__name">de Reyniès A. </li></ul><div class="ref__title">Estimating the population abundance of tissue-infiltrating immune and stromal cell populations using gene expression.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">218</span><div class="ref__controls"><a href="#back-bib13" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84992053166&amp;cf=" onclick="newWindow(this.href);return false">Scopus (497)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27765066&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2Fs13059-016-1070-5&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=218&journal=Genome+Biol.&author=Becht+E.&author=Giraldo+N.A.&author=Lacroix+L.&author=Buttard+B.&author=Elarouci+N.&author=Petitprez+F.&author=Selves+J.&author=Laurent-Puig+P.&author=Saut%C3%A8s-Fridman+C.&author=Fridman+W.H.&author=de+Reyni%C3%A8s+A.&title=Estimating+the+population+abundance+of+tissue-infiltrating+immune+and+stromal+cell+populations+using+gene+expression">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Benjamini Y. </li><li class="ref__authors__name">Hochberg Y. </li></ul><div class="ref__title">Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing.</div><span class="ref__series"><i>J. R. Stat. Soc. Series B Methodol.</i> <span class="ref__seriesDate">1995; </span><span class="ref__seriesVolume">57</span></span>: <span class="ref__seriesPages">289-300</span><div class="ref__controls"><a href="#back-bib14" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1111%2Fj.2517-6161.1995.tb02031.x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1995&pages=289-300&journal=J.%C2%A0R.+Stat.+Soc.+Series+B+Methodol.&author=Benjamini+Y.&author=Hochberg+Y.&title=Controlling+the+False+Discovery+Rate%3A+A+Practical+and+Powerful+Approach+to+Multiple+Testing">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Binder J.X. </li><li class="ref__authors__name">Pletscher-Frankild S. </li><li class="ref__authors__name">Tsafou K. </li><li class="ref__authors__name">Stolte C. </li><li class="ref__authors__name">O’Donoghue S.I. </li><li class="ref__authors__name">Schneider R. </li><li class="ref__authors__name">Jensen L.J. </li></ul><div class="ref__title">COMPARTMENTS: unification and visualization of protein subcellular localization evidence.</div><span class="ref__series"><i>Database (Oxford).</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">2014</span></span>: <span class="ref__seriesPages">bau012</span><div class="ref__controls"><a href="#back-bib15" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84901788541&amp;cf=" onclick="newWindow(this.href);return false">Scopus (250)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=24573882&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fdatabase%2Fbau012&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2014&publication_year=2014&pages=bau012&journal=Database+%28Oxford%29&author=Binder+J.X.&author=Pletscher-Frankild+S.&author=Tsafou+K.&author=Stolte+C.&author=O%E2%80%99Donoghue+S.I.&author=Schneider+R.&author=Jensen+L.J.&title=COMPARTMENTS%3A+unification+and+visualization+of+protein+subcellular+localization+evidence">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Blumenberg L. </li><li class="ref__authors__name">Kawaler E. </li><li class="ref__authors__name">Cornwell M. </li><li class="ref__authors__name">Smith S. </li><li class="ref__authors__name">Ruggles K. </li><li class="ref__authors__name">Fenyö D. </li></ul><div class="ref__title">BlackSheep: A Bioconductor and Bioconda package for differential extreme value analysis.</div><span class="ref__series"><i>bioRxiv.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1101/825067">https://doi.org/10.1101/825067</a></span><div class="ref__controls"><a href="#back-bib16" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85095064917&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1101%2F825067&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&journal=bioRxiv&author=Blumenberg+L.&author=Kawaler+E.&author=Cornwell+M.&author=Smith+S.&author=Ruggles+K.&author=Feny%C3%B6+D.&title=BlackSheep%3A+A+Bioconductor+and+Bioconda+package+for+differential+extreme+value+analysis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Bouchal P. </li><li class="ref__authors__name">Schubert O.T. </li><li class="ref__authors__name">Faktor J. </li><li class="ref__authors__name">Capkova L. </li><li class="ref__authors__name">Imrichova H. </li><li class="ref__authors__name">Zoufalova K. </li><li class="ref__authors__name">Paralova V. </li><li class="ref__authors__name">Hrstka R. </li><li class="ref__authors__name">Liu Y. </li><li class="ref__authors__name">Ebhardt H.A. </li><li>et al.</li></ul><div class="ref__title">Breast Cancer Classification Based on Proteotypes Obtained by SWATH Mass Spectrometry.</div><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">832-843.e7</span><div class="ref__controls"><a href="#back-bib17" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85068466554&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31315058&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.celrep.2019.06.046&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.celrep.2019.06.046&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.celrep.2019.06.046&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2019&pages=832-843.e7&journal=Cell+Rep.&author=Bouchal+P.&author=Schubert+O.T.&author=Faktor+J.&author=Capkova+L.&author=Imrichova+H.&author=Zoufalova+K.&author=Paralova+V.&author=Hrstka+R.&author=Liu+Y.&author=Ebhardt+H.A.&title=Breast+Cancer+Classification+Based+on+Proteotypes+Obtained+by+SWATH+Mass+Spectrometry">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Brunet J.-P. </li><li class="ref__authors__name">Tamayo P. </li><li class="ref__authors__name">Golub T.R. </li><li class="ref__authors__name">Mesirov J.P. </li></ul><div class="ref__title">Metagenes and molecular pattern discovery using matrix factorization.</div><span class="ref__series"><i>Proc. Natl. Acad. Sci. USA.</i> <span class="ref__seriesDate">2004; </span><span class="ref__seriesVolume">101</span></span>: <span class="ref__seriesPages">4164-4169</span><div class="ref__controls"><a href="#back-bib18" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-1642529511&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1117)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=15016911&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1073%2Fpnas.0308531101&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=4164-4169&journal=Proc.+Natl.+Acad.+Sci.+USA&author=Brunet+J.-P.&author=Tamayo+P.&author=Golub+T.R.&author=Mesirov+J.P.&title=Metagenes+and+molecular+pattern+discovery+using+matrix+factorization">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Caldon C.E. </li></ul><div class="ref__title">Estrogen signaling and the DNA damage response in hormone dependent breast cancers.</div><span class="ref__series"><i>Front. Oncol.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">106</span><div class="ref__controls"><a href="#back-bib19" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84904599047&amp;cf=" onclick="newWindow(this.href);return false">Scopus (91)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=24860786&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.3389%2Ffonc.2014.00106&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2014&pages=106&journal=Front.+Oncol.&author=Caldon+C.E.&title=Estrogen+signaling+and+the+DNA+damage+response+in+hormone+dependent+breast+cancers">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li>Cancer Genome Atlas Network</li></ul><div class="ref__title">Comprehensive molecular portraits of human breast tumours.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">490</span></span>: <span class="ref__seriesPages">61-70</span><div class="ref__controls"><a href="#back-bib20" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84877028141&amp;cf=" onclick="newWindow(this.href);return false">Scopus (7217)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=23000897&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature11412&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=490&publication_year=2012&pages=61-70&journal=Nature&author=Cancer+Genome+Atlas+Network&title=Comprehensive+molecular+portraits+of+human+breast+tumours">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Carter S.L. </li><li class="ref__authors__name">Cibulskis K. </li><li class="ref__authors__name">Helman E. </li><li class="ref__authors__name">McKenna A. </li><li class="ref__authors__name">Shen H. </li><li class="ref__authors__name">Zack T. </li><li class="ref__authors__name">Laird P.W. </li><li class="ref__authors__name">Onofrio R.C. </li><li class="ref__authors__name">Winckler W. </li><li class="ref__authors__name">Weir B.A. </li><li>et al.</li></ul><div class="ref__title">Absolute quantification of somatic DNA alterations in human cancer.</div><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">413-421</span><div class="ref__controls"><a href="#back-bib21" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84860782006&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1036)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=22544022&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnbt.2203&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2012&pages=413-421&journal=Nat.+Biotechnol.&author=Carter+S.L.&author=Cibulskis+K.&author=Helman+E.&author=McKenna+A.&author=Shen+H.&author=Zack+T.&author=Laird+P.W.&author=Onofrio+R.C.&author=Winckler+W.&author=Weir+B.A.&title=Absolute+quantification+of+somatic+DNA+alterations+in+human+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Chen X. </li><li class="ref__authors__name">Li J. </li><li class="ref__authors__name">Gray W.H. </li><li class="ref__authors__name">Lehmann B.D. </li><li class="ref__authors__name">Bauer J.A. </li><li class="ref__authors__name">Shyr Y. </li><li class="ref__authors__name">Pietenpol J.A. </li></ul><div class="ref__title">TNBCtype: A Subtyping Tool for Triple-Negative Breast Cancer.</div><span class="ref__series"><i>Cancer Inform.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">147-156</span><div class="ref__controls"><a href="#back-bib22" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84864579879&amp;cf=" onclick="newWindow(this.href);return false">Scopus (144)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=22872785&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.4137%2FCIN.S9983&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=147-156&journal=Cancer+Inform.&author=Chen+X.&author=Li+J.&author=Gray+W.H.&author=Lehmann+B.D.&author=Bauer+J.A.&author=Shyr+Y.&author=Pietenpol+J.A.&title=TNBCtype%3A+A+Subtyping+Tool+for+Triple-Negative+Breast+Cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Chen Y. </li><li class="ref__authors__name">Fu L.L. </li><li class="ref__authors__name">Wen X. </li><li class="ref__authors__name">Wang X.Y. </li><li class="ref__authors__name">Liu J. </li><li class="ref__authors__name">Cheng Y. </li><li class="ref__authors__name">Huang J. </li></ul><div class="ref__title">Sirtuin-3 (SIRT3), a therapeutic target with oncogenic and tumor-suppressive function in cancer.</div><span class="ref__series"><i>Cell Death Dis.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">e1047</span><div class="ref__controls"><a href="#back-bib23" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84896756082&amp;cf=" onclick="newWindow(this.href);return false">Scopus (119)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=24503539&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fcddis.2014.14&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=e1047&journal=Cell+Death+Dis.&author=Chen+Y.&author=Fu+L.L.&author=Wen+X.&author=Wang+X.Y.&author=Liu+J.&author=Cheng+Y.&author=Huang+J.&title=Sirtuin-3+%28SIRT3%29%2C+a+therapeutic+target+with+oncogenic+and+tumor-suppressive+function+in+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Chiang J. </li><li class="ref__authors__name">Martinez-Agosto J.A. </li></ul><div class="ref__title">Effects of mTOR Inhibitors on Components of the Salvador-Warts-Hippo Pathway.</div><span class="ref__series"><i>Cells.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">886-904</span><div class="ref__controls"><a href="#back-bib24" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=24710534&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.3390%2Fcells1040886&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2012&pages=886-904&journal=Cells&author=Chiang+J.&author=Martinez-Agosto+J.A.&title=Effects+of+mTOR+Inhibitors+on+Components+of+the+Salvador-Warts-Hippo+Pathway">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Choudhary C. </li><li class="ref__authors__name">Kumar C. </li><li class="ref__authors__name">Gnad F. </li><li class="ref__authors__name">Nielsen M.L. </li><li class="ref__authors__name">Rehman M. </li><li class="ref__authors__name">Walther T.C. </li><li class="ref__authors__name">Olsen J.V. </li><li class="ref__authors__name">Mann M. </li></ul><div class="ref__title">Lysine acetylation targets protein complexes and co-regulates major cellular functions.</div><span class="ref__series"><i>Science.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">325</span></span>: <span class="ref__seriesPages">834-840</span><div class="ref__controls"><a href="#back-bib25" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-68949212379&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2881)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=19608861&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1126%2Fscience.1175371&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=325&publication_year=2009&pages=834-840&journal=Science&author=Choudhary+C.&author=Kumar+C.&author=Gnad+F.&author=Nielsen+M.L.&author=Rehman+M.&author=Walther+T.C.&author=Olsen+J.V.&author=Mann+M.&title=Lysine+acetylation+targets+protein+complexes+and+co-regulates+major+cellular+functions">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Chow K.N. </li><li class="ref__authors__name">Dean D.C. </li></ul><div class="ref__title">Domains A and B in the Rb pocket interact to form a transcriptional repressor motif.</div><span class="ref__series"><i>Mol. Cell. Biol.</i> <span class="ref__seriesDate">1996; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">4862-4868</span><div class="ref__controls"><a href="#back-bib26" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=8756645&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1128%2FMCB.16.9.4862&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1996&pages=4862-4868&journal=Mol.+Cell.+Biol.&author=Chow+K.N.&author=Dean+D.C.&title=Domains+A+and+B+in+the+Rb+pocket+interact+to+form+a+transcriptional+repressor+motif">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Cibulskis K. </li><li class="ref__authors__name">McKenna A. </li><li class="ref__authors__name">Fennell T. </li><li class="ref__authors__name">Banks E. </li><li class="ref__authors__name">DePristo M. </li><li class="ref__authors__name">Getz G. </li></ul><div class="ref__title">ContEst: estimating cross-contamination of human samples in next-generation sequencing data.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">27</span></span>: <span class="ref__seriesPages">2601-2602</span><div class="ref__controls"><a href="#back-bib27" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-80052707308&amp;cf=" onclick="newWindow(this.href);return false">Scopus (152)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=21803805&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fbioinformatics%2Fbtr446&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2011&pages=2601-2602&journal=Bioinformatics&author=Cibulskis+K.&author=McKenna+A.&author=Fennell+T.&author=Banks+E.&author=DePristo+M.&author=Getz+G.&title=ContEst%3A+estimating+cross-contamination+of+human+samples+in+next-generation+sequencing+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Cibulskis K. </li><li class="ref__authors__name">Lawrence M.S. </li><li class="ref__authors__name">Carter S.L. </li><li class="ref__authors__name">Sivachenko A. </li><li class="ref__authors__name">Jaffe D. </li><li class="ref__authors__name">Sougnez C. </li><li class="ref__authors__name">Gabriel S. </li><li class="ref__authors__name">Meyerson M. </li><li class="ref__authors__name">Lander E.S. </li><li class="ref__authors__name">Getz G. </li></ul><div class="ref__title">Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples.</div><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">213-219</span><div class="ref__controls"><a href="#back-bib28" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84874025843&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2431)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=23396013&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnbt.2514&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2013&pages=213-219&journal=Nat.+Biotechnol.&author=Cibulskis+K.&author=Lawrence+M.S.&author=Carter+S.L.&author=Sivachenko+A.&author=Jaffe+D.&author=Sougnez+C.&author=Gabriel+S.&author=Meyerson+M.&author=Lander+E.S.&author=Getz+G.&title=Sensitive+detection+of+somatic+point+mutations+in+impure+and+heterogeneous+cancer+samples">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Coates A.S. </li><li class="ref__authors__name">Winer E.P. </li><li class="ref__authors__name">Goldhirsch A. </li><li class="ref__authors__name">Gelber R.D. </li><li class="ref__authors__name">Gnant M. </li><li class="ref__authors__name">Piccart-Gebhart M. </li><li class="ref__authors__name">Thürlimann B. </li><li class="ref__authors__name">Senn H.-J. </li><li>Panel Members</li></ul><div class="ref__title">Tailoring therapies--improving the management of early breast cancer: St Gallen International Expert Consensus on the Primary Therapy of Early Breast Cancer 2015.</div><span class="ref__series"><i>Ann. Oncol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">1533-1546</span><div class="ref__controls"><a href="#back-bib29" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84939423024&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1118)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=25939896&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fannonc%2Fmdv221&amp;cf=abstract&amp;site=annonc-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fannonc%2Fmdv221&amp;cf=fulltext&amp;site=annonc-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fannonc%2Fmdv221&amp;cf=pdf&amp;site=annonc-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2015&pages=1533-1546&journal=Ann.+Oncol.&author=Panel+Members&title=Tailoring+therapies%2D%2Dimproving+the+management+of+early+breast+cancer%3A+St+Gallen+International+Expert+Consensus+on+the+Primary+Therapy+of+Early+Breast+Cancer+2015">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Colaprico A. </li><li class="ref__authors__name">Silva T.C. </li><li class="ref__authors__name">Olsen C. </li><li class="ref__authors__name">Garofano L. </li><li class="ref__authors__name">Cava C. </li><li class="ref__authors__name">Garolini D. </li><li class="ref__authors__name">Sabedot T.S. </li><li class="ref__authors__name">Malta T.M. </li><li class="ref__authors__name">Pagnotta S.M. </li><li class="ref__authors__name">Castiglioni I. </li><li>et al.</li></ul><div class="ref__title">TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">e71</span><div class="ref__controls"><a href="#back-bib30" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84966269471&amp;cf=" onclick="newWindow(this.href);return false">Scopus (839)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26704973&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgkv1507&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=e71&journal=Nucleic+Acids+Res.&author=Colaprico+A.&author=Silva+T.C.&author=Olsen+C.&author=Garofano+L.&author=Cava+C.&author=Garolini+D.&author=Sabedot+T.S.&author=Malta+T.M.&author=Pagnotta+S.M.&author=Castiglioni+I.&title=TCGAbiolinks%3A+an+R%2FBioconductor+package+for+integrative+analysis+of+TCGA+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Colaprico A. </li><li class="ref__authors__name">Olsen C. </li><li class="ref__authors__name">Bailey M.H. </li><li class="ref__authors__name">Odom G.J. </li><li class="ref__authors__name">Terkelsen T. </li><li class="ref__authors__name">Silva T.C. </li><li class="ref__authors__name">Olsen A.V. </li><li class="ref__authors__name">Cantini L. </li><li class="ref__authors__name">Zinovyev A. </li><li class="ref__authors__name">Barillot E. </li><li>et al.</li></ul><div class="ref__title">Interpreting pathways to discover cancer driver genes with Moonlight.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">69</span><div class="ref__controls"><a href="#back-bib31" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85077479923&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31900418&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41467-019-13803-0&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=69&journal=Nat.+Commun.&author=Colaprico+A.&author=Olsen+C.&author=Bailey+M.H.&author=Odom+G.J.&author=Terkelsen+T.&author=Silva+T.C.&author=Olsen+A.V.&author=Cantini+L.&author=Zinovyev+A.&author=Barillot+E.&title=Interpreting+pathways+to+discover+cancer+driver+genes+with+Moonlight">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Condorelli R. </li><li class="ref__authors__name">Mosele F. </li><li class="ref__authors__name">Verret B. </li><li class="ref__authors__name">Bachelot T. </li><li class="ref__authors__name">Bedard P.L. </li><li class="ref__authors__name">Cortes J. </li><li class="ref__authors__name">Hyman D.M. </li><li class="ref__authors__name">Juric D. </li><li class="ref__authors__name">Krop I. </li><li class="ref__authors__name">Bieche I. </li><li>et al.</li></ul><div class="ref__title">Genomic alterations in breast cancer: level of evidence for actionability according to ESMO Scale for Clinical Actionability of molecular Targets (ESCAT).</div><span class="ref__series"><i>Ann. Oncol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">365-373</span><div class="ref__controls"><a href="#back-bib32" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85064114526&amp;cf=" onclick="newWindow(this.href);return false">Scopus (40)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30715161&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fannonc%2Fmdz036&amp;cf=abstract&amp;site=annonc-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fannonc%2Fmdz036&amp;cf=fulltext&amp;site=annonc-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fannonc%2Fmdz036&amp;cf=pdf&amp;site=annonc-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2019&pages=365-373&journal=Ann.+Oncol.&author=Condorelli+R.&author=Mosele+F.&author=Verret+B.&author=Bachelot+T.&author=Bedard+P.L.&author=Cortes+J.&author=Hyman+D.M.&author=Juric+D.&author=Krop+I.&author=Bieche+I.&title=Genomic+alterations+in+breast+cancer%3A+level+of+evidence+for+actionability+according+to+ESMO+Scale+for+Clinical+Actionability+of+molecular+Targets+%28ESCAT%29">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Cotto K.C. </li><li class="ref__authors__name">Wagner A.H. </li><li class="ref__authors__name">Feng Y.-Y. </li><li class="ref__authors__name">Kiwala S. </li><li class="ref__authors__name">Coffman A.C. </li><li class="ref__authors__name">Spies G. </li><li class="ref__authors__name">Wollam A. </li><li class="ref__authors__name">Spies N.C. </li><li class="ref__authors__name">Griffith O.L. </li><li class="ref__authors__name">Griffith M. </li></ul><div class="ref__title">DGIdb 3.0: a redesign and expansion of the drug-gene interaction database.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">46</span></span>: <span class="ref__seriesPages">D1068-D1073</span><div class="ref__controls"><a href="#back-bib33" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85070670869&amp;cf=" onclick="newWindow(this.href);return false">Scopus (303)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29156001&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgkx1143&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D1068-D1073&journal=Nucleic+Acids+Res.&author=Cotto+K.C.&author=Wagner+A.H.&author=Feng+Y.-Y.&author=Kiwala+S.&author=Coffman+A.C.&author=Spies+G.&author=Wollam+A.&author=Spies+N.C.&author=Griffith+O.L.&author=Griffith+M.&title=DGIdb+3.0%3A+a+redesign+and+expansion+of+the+drug-gene+interaction+database">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Curtis C. </li><li class="ref__authors__name">Shah S.P. </li><li class="ref__authors__name">Chin S.-F. </li><li class="ref__authors__name">Turashvili G. </li><li class="ref__authors__name">Rueda O.M. </li><li class="ref__authors__name">Dunning M.J. </li><li class="ref__authors__name">Speed D. </li><li class="ref__authors__name">Lynch A.G. </li><li class="ref__authors__name">Samarajiwa S. </li><li class="ref__authors__name">Yuan Y. </li><li>et al.</li><li>METABRIC Group</li></ul><div class="ref__title">The genomic and transcriptomic architecture of 2,000 breast tumours reveals novel subgroups.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">486</span></span>: <span class="ref__seriesPages">346-352</span><div class="ref__controls"><a href="#back-bib34" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84861527388&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3169)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=22522925&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature10983&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=486&publication_year=2012&pages=346-352&journal=Nature&author=METABRIC+Group&title=The+genomic+and+transcriptomic+architecture+of+2%2C000+breast+tumours+reveals+novel+subgroups">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Daily K. </li><li class="ref__authors__name">Ho Sui S.J. </li><li class="ref__authors__name">Schriml L.M. </li><li class="ref__authors__name">Dexheimer P.J. </li><li class="ref__authors__name">Salomonis N. </li><li class="ref__authors__name">Schroll R. </li><li class="ref__authors__name">Bush S. </li><li class="ref__authors__name">Keddache M. </li><li class="ref__authors__name">Mayhew C. </li><li class="ref__authors__name">Lotia S. </li><li>et al.</li></ul><div class="ref__title">Molecular, phenotypic, and sample-associated data to describe pluripotent stem cell lines and derivatives.</div><span class="ref__series"><i>Sci. Data.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">170030</span><div class="ref__controls"><a href="#back-bib35" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85016445988&amp;cf=" onclick="newWindow(this.href);return false">Scopus (24)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=28350385&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fsdata.2017.30&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2017&pages=170030&journal=Sci.+Data&author=Daily+K.&author=Ho+Sui+S.J.&author=Schriml+L.M.&author=Dexheimer+P.J.&author=Salomonis+N.&author=Schroll+R.&author=Bush+S.&author=Keddache+M.&author=Mayhew+C.&author=Lotia+S.&title=Molecular%2C+phenotypic%2C+and+sample-associated+data+to+describe+pluripotent+stem+cell+lines+and+derivatives">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Dietze E.C. </li><li class="ref__authors__name">Sistrunk C. </li><li class="ref__authors__name">Miranda-Carboni G. </li><li class="ref__authors__name">O’Regan R. </li><li class="ref__authors__name">Seewaldt V.L. </li></ul><div class="ref__title">Triple-negative breast cancer in African-American women: disparities versus biology.</div><span class="ref__series"><i>Nat. Rev. Cancer.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">248-254</span><div class="ref__controls"><a href="#back-bib36" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84925819377&amp;cf=" onclick="newWindow(this.href);return false">Scopus (194)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=25673085&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnrc3896&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=248-254&journal=Nat.+Rev.+Cancer&author=Dietze+E.C.&author=Sistrunk+C.&author=Miranda-Carboni+G.&author=O%E2%80%99Regan+R.&author=Seewaldt+V.L.&title=Triple-negative+breast+cancer+in+African-American+women%3A+disparities+versus+biology">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Dong H. </li><li class="ref__authors__name">Adams N.M. </li><li class="ref__authors__name">Xu Y. </li><li class="ref__authors__name">Cao J. </li><li class="ref__authors__name">Allan D.S.J. </li><li class="ref__authors__name">Carlyle J.R. </li><li class="ref__authors__name">Chen X. </li><li class="ref__authors__name">Sun J.C. </li><li class="ref__authors__name">Glimcher L.H. </li></ul><div class="ref__title">The IRE1 endoplasmic reticulum stress sensor activates natural killer cell immunity in part by regulating c-Myc.</div><span class="ref__series"><i>Nat. Immunol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">865-878</span><div class="ref__controls"><a href="#back-bib37" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85065791702&amp;cf=" onclick="newWindow(this.href);return false">Scopus (47)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31086333&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41590-019-0388-z&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=865-878&journal=Nat.+Immunol.&author=Dong+H.&author=Adams+N.M.&author=Xu+Y.&author=Cao+J.&author=Allan+D.S.J.&author=Carlyle+J.R.&author=Chen+X.&author=Sun+J.C.&author=Glimcher+L.H.&title=The+IRE1+endoplasmic+reticulum+stress+sensor+activates+natural+killer+cell+immunity+in+part+by+regulating+c-Myc">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Dou Y. </li><li class="ref__authors__name">Kawaler E.A. </li><li class="ref__authors__name">Cui Zhou D. </li><li class="ref__authors__name">Gritsenko M.A. </li><li class="ref__authors__name">Huang C. </li><li class="ref__authors__name">Blumenberg L. </li><li class="ref__authors__name">Karpova A. </li><li class="ref__authors__name">Petyuk V.A. </li><li class="ref__authors__name">Savage S.R. </li><li class="ref__authors__name">Satpathy S. </li><li>et al.</li><li>Clinical Proteomic Tumor Analysis Consortium</li></ul><div class="ref__title">Proteogenomic Characterization of Endometrial Carcinoma.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">180</span></span>: <span class="ref__seriesPages">729-748.e26</span><div class="ref__controls"><a href="#back-bib38" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85079388976&amp;cf=" onclick="newWindow(this.href);return false">Scopus (80)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=32059776&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2020.01.026&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2020.01.026&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2020.01.026&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=180&publication_year=2020&pages=729-748.e26&journal=Cell&author=Clinical+Proteomic+Tumor+Analysis+Consortium&title=Proteogenomic+Characterization+of+Endometrial+Carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ellis M.J. </li><li class="ref__authors__name">Suman V.J. </li><li class="ref__authors__name">Hoog J. </li><li class="ref__authors__name">Goncalves R. </li><li class="ref__authors__name">Sanati S. </li><li class="ref__authors__name">Creighton C.J. </li><li class="ref__authors__name">DeSchryver K. </li><li class="ref__authors__name">Crouch E. </li><li class="ref__authors__name">Brink A. </li><li class="ref__authors__name">Watson M. </li><li>et al.</li></ul><div class="ref__title">Ki67 Proliferation Index as a Tool for Chemotherapy Decisions During and After Neoadjuvant Aromatase Inhibitor Treatment of Breast Cancer: Results From the American College of Surgeons Oncology Group Z1031 Trial (Alliance).</div><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">1061-1069</span><div class="ref__controls"><a href="#back-bib39" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85016622416&amp;cf=" onclick="newWindow(this.href);return false">Scopus (127)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=28045625&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1200%2FJCO.2016.69.4406&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2017&pages=1061-1069&journal=J.%C2%A0Clin.+Oncol.&author=Ellis+M.J.&author=Suman+V.J.&author=Hoog+J.&author=Goncalves+R.&author=Sanati+S.&author=Creighton+C.J.&author=DeSchryver+K.&author=Crouch+E.&author=Brink+A.&author=Watson+M.&title=Ki67+Proliferation+Index+as+a+Tool+for+Chemotherapy+Decisions+During+and+After+Neoadjuvant+Aromatase+Inhibitor+Treatment+of+Breast+Cancer%3A+Results+From+the+American+College+of+Surgeons+Oncology+Group+Z1031+Trial+%28Alliance%29">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Fabregat A. </li><li class="ref__authors__name">Jupe S. </li><li class="ref__authors__name">Matthews L. </li><li class="ref__authors__name">Sidiropoulos K. </li><li class="ref__authors__name">Gillespie M. </li><li class="ref__authors__name">Garapati P. </li><li class="ref__authors__name">Haw R. </li><li class="ref__authors__name">Jassal B. </li><li class="ref__authors__name">Korninger F. </li><li class="ref__authors__name">May B. </li><li>et al.</li></ul><div class="ref__title">The Reactome Pathway Knowledgebase.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">46</span></span>: <span class="ref__seriesPages">D649-D655</span><div class="ref__controls"><a href="#back-bib40" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85040929878&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1166)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29145629&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgkx1132&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D649-D655&journal=Nucleic+Acids+Res.&author=Fabregat+A.&author=Jupe+S.&author=Matthews+L.&author=Sidiropoulos+K.&author=Gillespie+M.&author=Garapati+P.&author=Haw+R.&author=Jassal+B.&author=Korninger+F.&author=May+B.&title=The+Reactome+Pathway+Knowledgebase">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Flockhart D.A. </li><li class="ref__authors__name">Corbin J.D. </li></ul><div class="ref__title">Regulatory mechanisms in the control of protein kinases.</div><span class="ref__series"><i>CRC Crit. Rev. Biochem.</i> <span class="ref__seriesDate">1982; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">133-186</span><div class="ref__controls"><a href="#back-bib41" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=7039969&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.3109%2F10409238209108705&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1982&pages=133-186&journal=CRC+Crit.+Rev.+Biochem.&author=Flockhart+D.A.&author=Corbin+J.D.&title=Regulatory+mechanisms+in+the+control+of+protein+kinases">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gaujoux R. </li><li class="ref__authors__name">Seoighe C. </li></ul><div class="ref__title">A flexible R package for nonnegative matrix factorization.</div><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">367</span><div class="ref__controls"><a href="#back-bib42" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-77954069195&amp;cf=" onclick="newWindow(this.href);return false">Scopus (528)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=20598126&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2F1471-2105-11-367&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=367&journal=BMC+Bioinformatics&author=Gaujoux+R.&author=Seoighe+C.&title=A+flexible+R+package+for+nonnegative+matrix+factorization">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ghandi M. </li><li class="ref__authors__name">Huang F.W. </li><li class="ref__authors__name">Jané-Valbuena J. </li><li class="ref__authors__name">Kryukov G.V. </li><li class="ref__authors__name">Lo C.C. </li><li class="ref__authors__name">McDonald 3rd, E.R. </li><li class="ref__authors__name">Barretina J. </li><li class="ref__authors__name">Gelfand E.T. </li><li class="ref__authors__name">Bielski C.M. </li><li class="ref__authors__name">Li H. </li><li>et al.</li></ul><div class="ref__title">Next-generation characterization of the Cancer Cell Line Encyclopedia.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">569</span></span>: <span class="ref__seriesPages">503-508</span><div class="ref__controls"><a href="#back-bib43" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85065525804&amp;cf=" onclick="newWindow(this.href);return false">Scopus (624)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31068700&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41586-019-1186-3&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=569&publication_year=2019&pages=503-508&journal=Nature&author=Ghandi+M.&author=Huang+F.W.&author=Jan%C3%A9-Valbuena+J.&author=Kryukov+G.V.&author=Lo+C.C.&author=McDonald+E.R.&author=Barretina+J.&author=Gelfand+E.T.&author=Bielski+C.M.&author=Li+H.&title=Next-generation+characterization+of+the+Cancer+Cell+Line+Encyclopedia">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gillette M.A. </li><li class="ref__authors__name">Carr S.A. </li></ul><div class="ref__title">Quantitative analysis of peptides and proteins in biomedicine by targeted mass spectrometry.</div><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">28-34</span><div class="ref__controls"><a href="#back-bib44" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84871983086&amp;cf=" onclick="newWindow(this.href);return false">Scopus (326)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=23269374&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnmeth.2309&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=28-34&journal=Nat.+Methods&author=Gillette+M.A.&author=Carr+S.A.&title=Quantitative+analysis+of+peptides+and+proteins+in+biomedicine+by+targeted+mass+spectrometry">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gillette M.A. </li><li class="ref__authors__name">Satpathy S. </li><li class="ref__authors__name">Cao S. </li><li class="ref__authors__name">Dhanasekaran S.M. </li><li class="ref__authors__name">Vasaikar S.V. </li><li class="ref__authors__name">Krug K. </li><li class="ref__authors__name">Petralia F. </li><li class="ref__authors__name">Li Y. </li><li class="ref__authors__name">Liang W.-W. </li><li class="ref__authors__name">Reva B. </li><li>et al.</li><li>Clinical Proteomic Tumor Analysis Consortium</li></ul><div class="ref__title">Proteogenomic Characterization Reveals Therapeutic Vulnerabilities in Lung Adenocarcinoma.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">182</span></span>: <span class="ref__seriesPages">200-225.e35</span><div class="ref__controls"><a href="#back-bib45" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85087400742&amp;cf=" onclick="newWindow(this.href);return false">Scopus (76)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=32649874&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2020.06.013&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2020.06.013&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2020.06.013&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=182&publication_year=2020&pages=200-225.e35&journal=Cell&author=Clinical+Proteomic+Tumor+Analysis+Consortium&title=Proteogenomic+Characterization+Reveals+Therapeutic+Vulnerabilities+in+Lung+Adenocarcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Goel S. </li><li class="ref__authors__name">DeCristo M.J. </li><li class="ref__authors__name">McAllister S.S. </li><li class="ref__authors__name">Zhao J.J. </li></ul><div class="ref__title">CDK4/6 Inhibition in Cancer: Beyond Cell Cycle Arrest.</div><span class="ref__series"><i>Trends Cell Biol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">911-925</span><div class="ref__controls"><a href="#back-bib46" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85050508022&amp;cf=" onclick="newWindow(this.href);return false">Scopus (100)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30061045&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.tcb.2018.07.002&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.tcb.2018.07.002&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.tcb.2018.07.002&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=911-925&journal=Trends+Cell+Biol.&author=Goel+S.&author=DeCristo+M.J.&author=McAllister+S.S.&author=Zhao+J.J.&title=CDK4%2F6+Inhibition+in+Cancer%3A+Beyond+Cell+Cycle+Arrest">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Harbour J.W. </li></ul><div class="ref__title">Molecular basis of low-penetrance retinoblastoma.</div><span class="ref__series"><i>Arch. Ophthalmol.</i> <span class="ref__seriesDate">2001; </span><span class="ref__seriesVolume">119</span></span>: <span class="ref__seriesPages">1699-1704</span><div class="ref__controls"><a href="#back-bib47" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=11709023&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1001%2Farchopht.119.11.1699&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2001&pages=1699-1704&journal=Arch.+Ophthalmol.&author=Harbour+J.W.&title=Molecular+basis+of+low-penetrance+retinoblastoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Haricharan S. </li><li class="ref__authors__name">Punturi N. </li><li class="ref__authors__name">Singh P. </li><li class="ref__authors__name">Holloway K.R. </li><li class="ref__authors__name">Anurag M. </li><li class="ref__authors__name">Schmelz J. </li><li class="ref__authors__name">Schmidt C. </li><li class="ref__authors__name">Lei J.T. </li><li class="ref__authors__name">Suman V. </li><li class="ref__authors__name">Hunt K. </li><li>et al.</li></ul><div class="ref__title">Loss of MutL Disrupts CHK2-Dependent Cell-Cycle Control through CDK4/6 to Promote Intrinsic Endocrine Therapy Resistance in Primary Breast Cancer.</div><span class="ref__series"><i>Cancer Discov.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">1168-1183</span><div class="ref__controls"><a href="#back-bib48" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85030681935&amp;cf=" onclick="newWindow(this.href);return false">Scopus (12)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=28801307&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1158%2F2159-8290.CD-16-1179&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=1168-1183&journal=Cancer+Discov.&author=Haricharan+S.&author=Punturi+N.&author=Singh+P.&author=Holloway+K.R.&author=Anurag+M.&author=Schmelz+J.&author=Schmidt+C.&author=Lei+J.T.&author=Suman+V.&author=Hunt+K.&title=Loss+of+MutL+Disrupts+CHK2-Dependent+Cell-Cycle+Control+through+CDK4%2F6+to+Promote+Intrinsic+Endocrine+Therapy+Resistance+in+Primary+Breast+Cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Harris L.N. </li><li class="ref__authors__name">Broadwater G. </li><li class="ref__authors__name">Abu-Khalaf M. </li><li class="ref__authors__name">Cowan D. </li><li class="ref__authors__name">Thor A.D. </li><li class="ref__authors__name">Budman D. </li><li class="ref__authors__name">Cirrincione C.T. </li><li class="ref__authors__name">Berry D.A. </li><li class="ref__authors__name">Winer E.P. </li><li class="ref__authors__name">Hudis C.A. </li><li>et al.</li></ul><div class="ref__title">Topoisomerase IIalpha amplification does not predict benefit from dose-intense cyclophosphamide, doxorubicin, and fluorouracil therapy in HER2-amplified early breast cancer: results of CALGB 8541/150013.</div><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">27</span></span>: <span class="ref__seriesPages">3430-3436</span><div class="ref__controls"><a href="#back-bib49" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-70249085601&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=19470942&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1200%2FJCO.2008.18.4085&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2009&pages=3430-3436&journal=J.%C2%A0Clin.+Oncol.&author=Harris+L.N.&author=Broadwater+G.&author=Abu-Khalaf+M.&author=Cowan+D.&author=Thor+A.D.&author=Budman+D.&author=Cirrincione+C.T.&author=Berry+D.A.&author=Winer+E.P.&author=Hudis+C.A.&title=Topoisomerase+IIalpha+amplification+does+not+predict+benefit+from+dose-intense+cyclophosphamide%2C+doxorubicin%2C+and+fluorouracil+therapy+in+HER2-amplified+early+breast+cancer%3A+results+of+CALGB+8541%2F150013">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hornbeck P.V. </li><li class="ref__authors__name">Zhang B. </li><li class="ref__authors__name">Murray B. </li><li class="ref__authors__name">Kornhauser J.M. </li><li class="ref__authors__name">Latham V. </li><li class="ref__authors__name">Skrzypek E. </li></ul><div class="ref__title">PhosphoSitePlus, 2014: mutations, PTMs and recalibrations.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">43</span></span>: <span class="ref__seriesPages">D512-D520</span><div class="ref__controls"><a href="#back-bib50" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84941120469&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1289)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=25514926&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgku1267&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=D512-D520&journal=Nucleic+Acids+Res.&author=Hornbeck+P.V.&author=Zhang+B.&author=Murray+B.&author=Kornhauser+J.M.&author=Latham+V.&author=Skrzypek+E.&title=PhosphoSitePlus%2C+2014%3A+mutations%2C+PTMs+and+recalibrations">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hunt A.L. </li><li class="ref__authors__name">Bateman N.W. </li><li class="ref__authors__name">Hood B.L. </li><li class="ref__authors__name">Conrads K.A. </li><li class="ref__authors__name">Zhou M. </li><li class="ref__authors__name">Litzi T.J. </li><li class="ref__authors__name">Oliver J. </li><li class="ref__authors__name">Mitchell D. </li><li class="ref__authors__name">Gist G. </li><li class="ref__authors__name">Blanton B. </li><li>et al.</li></ul><div class="ref__title">Extensive Intratumor Proteogenomic Heterogeneity Revealed by Multiregion Sampling in a High-Grade Serous Ovarian Tumor Specimen.</div><span class="ref__series"><i>bioRxiv.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1101/761155">https://doi.org/10.1101/761155</a></span><div class="ref__controls"><a href="#back-bib51" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85095660210&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1101%2F761155&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&journal=bioRxiv&author=Hunt+A.L.&author=Bateman+N.W.&author=Hood+B.L.&author=Conrads+K.A.&author=Zhou+M.&author=Litzi+T.J.&author=Oliver+J.&author=Mitchell+D.&author=Gist+G.&author=Blanton+B.&title=Extensive+Intratumor+Proteogenomic+Heterogeneity+Revealed+by+Multiregion+Sampling+in+a+High-Grade+Serous+Ovarian+Tumor+Specimen">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hyman D.M. </li><li class="ref__authors__name">Taylor B.S. </li><li class="ref__authors__name">Baselga J. </li></ul><div class="ref__title">Implementing Genome-Driven Oncology.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">168</span></span>: <span class="ref__seriesPages">584-599</span><div class="ref__controls"><a href="#back-bib52" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85012024906&amp;cf=" onclick="newWindow(this.href);return false">Scopus (244)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=28187282&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2016.12.015&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2016.12.015&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2016.12.015&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=168&publication_year=2017&pages=584-599&journal=Cell&author=Hyman+D.M.&author=Taylor+B.S.&author=Baselga+J.&title=Implementing+Genome-Driven+Oncology">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Iorio F. </li><li class="ref__authors__name">Knijnenburg T.A. </li><li class="ref__authors__name">Vis D.J. </li><li class="ref__authors__name">Bignell G.R. </li><li class="ref__authors__name">Menden M.P. </li><li class="ref__authors__name">Schubert M. </li><li class="ref__authors__name">Aben N. </li><li class="ref__authors__name">Gonçalves E. </li><li class="ref__authors__name">Barthorpe S. </li><li class="ref__authors__name">Lightfoot H. </li><li>et al.</li></ul><div class="ref__title">A Landscape of Pharmacogenomic Interactions in Cancer.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">166</span></span>: <span class="ref__seriesPages">740-754</span><div class="ref__controls"><a href="#back-bib53" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84979649916&amp;cf=" onclick="newWindow(this.href);return false">Scopus (702)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27397505&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2016.06.017&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2016.06.017&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2016.06.017&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2016&pages=740-754&journal=Cell&author=Iorio+F.&author=Knijnenburg+T.A.&author=Vis+D.J.&author=Bignell+G.R.&author=Menden+M.P.&author=Schubert+M.&author=Aben+N.&author=Gon%C3%A7alves+E.&author=Barthorpe+S.&author=Lightfoot+H.&title=A+Landscape+of+Pharmacogenomic+Interactions+in+Cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Jackson H.W. </li><li class="ref__authors__name">Fischer J.R. </li><li class="ref__authors__name">Zanotelli V.R.T. </li><li class="ref__authors__name">Ali H.R. </li><li class="ref__authors__name">Mechera R. </li><li class="ref__authors__name">Soysal S.D. </li><li class="ref__authors__name">Moch H. </li><li class="ref__authors__name">Muenst S. </li><li class="ref__authors__name">Varga Z. </li><li class="ref__authors__name">Weber W.P. </li><li class="ref__authors__name">Bodenmiller B. </li></ul><div class="ref__title">The single-cell pathology landscape of breast cancer.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">578</span></span>: <span class="ref__seriesPages">615-620</span><div class="ref__controls"><a href="#back-bib54" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85078202432&amp;cf=" onclick="newWindow(this.href);return false">Scopus (155)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31959985&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41586-019-1876-x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=578&publication_year=2020&pages=615-620&journal=Nature&author=Jackson+H.W.&author=Fischer+J.R.&author=Zanotelli+V.R.T.&author=Ali+H.R.&author=Mechera+R.&author=Soysal+S.D.&author=Moch+H.&author=Muenst+S.&author=Varga+Z.&author=Weber+W.P.&author=Bodenmiller+B.&title=The+single-cell+pathology+landscape+of+breast+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Johansson H.J. </li><li class="ref__authors__name">Socciarelli F. </li><li class="ref__authors__name">Vacanti N.M. </li><li class="ref__authors__name">Haugen M.H. </li><li class="ref__authors__name">Zhu Y. </li><li class="ref__authors__name">Siavelis I. </li><li class="ref__authors__name">Fernandez-Woodbridge A. </li><li class="ref__authors__name">Aure M.R. </li><li class="ref__authors__name">Sennblad B. </li><li class="ref__authors__name">Vesterlund M. </li><li>et al.</li><li>Consortia Oslo Breast Cancer Research Consortium (OSBREAC)</li></ul><div class="ref__title">Breast cancer quantitative proteome and proteogenomic landscape.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">1600</span><div class="ref__controls"><a href="#back-bib55" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85064079271&amp;cf=" onclick="newWindow(this.href);return false">Scopus (66)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30962452&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41467-019-09018-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1600&journal=Nat.+Commun.&author=Consortia+Oslo+Breast+Cancer+Research+Consortium+%28OSBREAC%29&title=Breast+cancer+quantitative+proteome+and+proteogenomic+landscape">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kanehisa M. </li><li class="ref__authors__name">Goto S. </li></ul><div class="ref__title">KEGG: kyoto encyclopedia of genes and genomes.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2000; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">27-30</span><div class="ref__controls"><a href="#back-bib56" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10592173&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2F28.1.27&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=27-30&journal=Nucleic+Acids+Res.&author=Kanehisa+M.&author=Goto+S.&title=KEGG%3A+kyoto+encyclopedia+of+genes+and+genomes">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kasar S. </li><li class="ref__authors__name">Kim J. </li><li class="ref__authors__name">Improgo R. </li><li class="ref__authors__name">Tiao G. </li><li class="ref__authors__name">Polak P. </li><li class="ref__authors__name">Haradhvala N. </li><li class="ref__authors__name">Lawrence M.S. </li><li class="ref__authors__name">Kiezun A. </li><li class="ref__authors__name">Fernandes S.M. </li><li class="ref__authors__name">Bahl S. </li><li>et al.</li></ul><div class="ref__title">Whole-genome sequencing reveals activation-induced cytidine deaminase signatures during indolent chronic lymphocytic leukaemia evolution.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">8866</span><div class="ref__controls"><a href="#back-bib57" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26638776&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fncomms9866&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=8866&journal=Nat.+Commun.&author=Kasar+S.&author=Kim+J.&author=Improgo+R.&author=Tiao+G.&author=Polak+P.&author=Haradhvala+N.&author=Lawrence+M.S.&author=Kiezun+A.&author=Fernandes+S.M.&author=Bahl+S.&title=Whole-genome+sequencing+reveals+activation-induced+cytidine+deaminase+signatures+during+indolent+chronic+lymphocytic+leukaemia+evolution">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kawamura D. </li><li class="ref__authors__name">Takemoto Y. </li><li class="ref__authors__name">Nishimoto A. </li><li class="ref__authors__name">Ueno K. </li><li class="ref__authors__name">Hosoyama T. </li><li class="ref__authors__name">Shirasawa B. </li><li class="ref__authors__name">Tanaka T. </li><li class="ref__authors__name">Kugimiya N. </li><li class="ref__authors__name">Harada E. </li><li class="ref__authors__name">Hamano K. </li></ul><div class="ref__title">Enhancement of cytotoxic effects of gemcitabine by Dclk1 inhibition through suppression of Chk1 phosphorylation in human pancreatic cancer cells.</div><span class="ref__series"><i>Oncol. Rep.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">38</span></span>: <span class="ref__seriesPages">3238-3244</span><div class="ref__controls"><a href="#back-bib58" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85031027382&amp;cf=" onclick="newWindow(this.href);return false">Scopus (9)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29048622&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.3892%2For.2017.5974&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=3238-3244&journal=Oncol.+Rep.&author=Kawamura+D.&author=Takemoto+Y.&author=Nishimoto+A.&author=Ueno+K.&author=Hosoyama+T.&author=Shirasawa+B.&author=Tanaka+T.&author=Kugimiya+N.&author=Harada+E.&author=Hamano+K.&title=Enhancement+of+cytotoxic+effects+of+gemcitabine+by+Dclk1+inhibition+through+suppression+of+Chk1+phosphorylation+in+human+pancreatic+cancer+cells">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kim H. </li><li class="ref__authors__name">Park H. </li></ul><div class="ref__title">Sparse non-negative matrix factorizations via alternating non-negativity-constrained least squares for microarray data analysis.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">1495-1502</span><div class="ref__controls"><a href="#back-bib59" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-34547844077&amp;cf=" onclick="newWindow(this.href);return false">Scopus (531)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=17483501&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fbioinformatics%2Fbtm134&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2007&pages=1495-1502&journal=Bioinformatics&author=Kim+H.&author=Park+H.&title=Sparse+non-negative+matrix+factorizations+via+alternating+non-negativity-constrained+least+squares+for+microarray+data+analysis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kim J. </li><li class="ref__authors__name">Mouw K.W. </li><li class="ref__authors__name">Polak P. </li><li class="ref__authors__name">Braunstein L.Z. </li><li class="ref__authors__name">Kamburov A. </li><li class="ref__authors__name">Kwiatkowski D.J. </li><li class="ref__authors__name">Rosenberg J.E. </li><li class="ref__authors__name">Van Allen E.M. </li><li class="ref__authors__name">D’Andrea A. </li><li class="ref__authors__name">Getz G. </li></ul><div class="ref__title">Somatic ERCC2 mutations are associated with a distinct genomic signature in urothelial tumors.</div><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">48</span></span>: <span class="ref__seriesPages">600-606</span><div class="ref__controls"><a href="#back-bib60" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84964308124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (189)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27111033&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fng.3557&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2016&pages=600-606&journal=Nat.+Genet.&author=Kim+J.&author=Mouw+K.W.&author=Polak+P.&author=Braunstein+L.Z.&author=Kamburov+A.&author=Kwiatkowski+D.J.&author=Rosenberg+J.E.&author=Van+Allen+E.M.&author=D%E2%80%99Andrea+A.&author=Getz+G.&title=Somatic+ERCC2+mutations+are+associated+with+a+distinct+genomic+signature+in+urothelial+tumors">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kim S. </li><li class="ref__authors__name">Scheffler K. </li><li class="ref__authors__name">Halpern A.L. </li><li class="ref__authors__name">Bekritsky M.A. </li><li class="ref__authors__name">Noh E. </li><li class="ref__authors__name">Källberg M. </li><li class="ref__authors__name">Chen X. </li><li class="ref__authors__name">Kim Y. </li><li class="ref__authors__name">Beyter D. </li><li class="ref__authors__name">Krusche P. </li><li class="ref__authors__name">Saunders C.T. </li></ul><div class="ref__title">Strelka2: fast and accurate calling of germline and somatic variants.</div><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">591-594</span><div class="ref__controls"><a href="#back-bib61" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85049953163&amp;cf=" onclick="newWindow(this.href);return false">Scopus (263)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30013048&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41592-018-0051-x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=591-594&journal=Nat.+Methods&author=Kim+S.&author=Scheffler+K.&author=Halpern+A.L.&author=Bekritsky+M.A.&author=Noh+E.&author=K%C3%A4llberg+M.&author=Chen+X.&author=Kim+Y.&author=Beyter+D.&author=Krusche+P.&author=Saunders+C.T.&title=Strelka2%3A+fast+and+accurate+calling+of+germline+and+somatic+variants">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Krug K. </li><li class="ref__authors__name">Mertins P. </li><li class="ref__authors__name">Zhang B. </li><li class="ref__authors__name">Hornbeck P. </li><li class="ref__authors__name">Raju R. </li><li class="ref__authors__name">Ahmad R. </li><li class="ref__authors__name">Szucs M. </li><li class="ref__authors__name">Mundt F. </li><li class="ref__authors__name">Forestier D. </li><li class="ref__authors__name">Jane-Valbuena J. </li><li>et al.</li></ul><div class="ref__title">A Curated Resource for Phosphosite-specific Signature Analysis.</div><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">576-593</span><div class="ref__controls"><a href="#back-bib62" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85062992827&amp;cf=" onclick="newWindow(this.href);return false">Scopus (42)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30563849&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=576-593&journal=Mol.+Cell.+Proteomics&author=Krug+K.&author=Mertins+P.&author=Zhang+B.&author=Hornbeck+P.&author=Raju+R.&author=Ahmad+R.&author=Szucs+M.&author=Mundt+F.&author=Forestier+D.&author=Jane-Valbuena+J.&title=A+Curated+Resource+for+Phosphosite-specific+Signature+Analysis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lamb J. </li><li class="ref__authors__name">Crawford E.D. </li><li class="ref__authors__name">Peck D. </li><li class="ref__authors__name">Modell J.W. </li><li class="ref__authors__name">Blat I.C. </li><li class="ref__authors__name">Wrobel M.J. </li><li class="ref__authors__name">Lerner J. </li><li class="ref__authors__name">Brunet J.-P. </li><li class="ref__authors__name">Subramanian A. </li><li class="ref__authors__name">Ross K.N. </li><li>et al.</li></ul><div class="ref__title">The Connectivity Map: using gene-expression signatures to connect small molecules, genes, and disease.</div><span class="ref__series"><i>Science.</i> <span class="ref__seriesDate">2006; </span><span class="ref__seriesVolume">313</span></span>: <span class="ref__seriesPages">1929-1935</span><div class="ref__controls"><a href="#back-bib63" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-33749335282&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3071)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=17008526&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1126%2Fscience.1132939&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=313&publication_year=2006&pages=1929-1935&journal=Science&author=Lamb+J.&author=Crawford+E.D.&author=Peck+D.&author=Modell+J.W.&author=Blat+I.C.&author=Wrobel+M.J.&author=Lerner+J.&author=Brunet+J.-P.&author=Subramanian+A.&author=Ross+K.N.&title=The+Connectivity+Map%3A+using+gene-expression+signatures+to+connect+small+molecules%2C+genes%2C+and+disease">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lawrence M.S. </li><li class="ref__authors__name">Stojanov P. </li><li class="ref__authors__name">Mermel C.H. </li><li class="ref__authors__name">Robinson J.T. </li><li class="ref__authors__name">Garraway L.A. </li><li class="ref__authors__name">Golub T.R. </li><li class="ref__authors__name">Meyerson M. </li><li class="ref__authors__name">Gabriel S.B. </li><li class="ref__authors__name">Lander E.S. </li><li class="ref__authors__name">Getz G. </li></ul><div class="ref__title">Discovery and saturation analysis of cancer genes across 21 tumour types.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">505</span></span>: <span class="ref__seriesPages">495-501</span><div class="ref__controls"><a href="#back-bib64" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84892833777&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1826)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=24390350&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature12912&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=505&publication_year=2014&pages=495-501&journal=Nature&author=Lawrence+M.S.&author=Stojanov+P.&author=Mermel+C.H.&author=Robinson+J.T.&author=Garraway+L.A.&author=Golub+T.R.&author=Meyerson+M.&author=Gabriel+S.B.&author=Lander+E.S.&author=Getz+G.&title=Discovery+and+saturation+analysis+of+cancer+genes+across+21+tumour+types">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lee D.D. </li><li class="ref__authors__name">Seung H.S. </li></ul><div class="ref__title">Learning the parts of objects by non-negative matrix factorization.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">1999; </span><span class="ref__seriesVolume">401</span></span>: <span class="ref__seriesPages">788-791</span><div class="ref__controls"><a href="#back-bib65" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-0033592606&amp;cf=" onclick="newWindow(this.href);return false">Scopus (8387)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10548103&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2F44565&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=401&publication_year=1999&pages=788-791&journal=Nature&author=Lee+D.D.&author=Seung+H.S.&title=Learning+the+parts+of+objects+by+non-negative+matrix+factorization">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lee D.D. </li><li class="ref__authors__name">Seung H.S. </li></ul><div class="ref__title">Algorithms for Non-negative Matrix Factorization.</div>in: <span class="editors">Leen T.K. Dietterich T.G. Tresp V. </span>Advances in Neural Information Processing Systems 13.  <span class="refPublisherName">MIT Press</span>,
        <span class="refPublisherLoc"></span><span class="refDate">2001</span>: <span class="ref__seriesPages">556-562</span><div class="ref__controls"><a href="#back-bib66" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Lee+D.D.+Seung+H.S.+Algorithms+for+Non-negative+Matrix+Factorization.in%3A+Leen+T.K.+Dietterich+T.G.+Tresp+V.+Advances+in+Neural+Information+Processing+Systems+13.+MIT+Press%2C%0A+2001%3A+556-562">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lee J.O. </li><li class="ref__authors__name">Russo A.A. </li><li class="ref__authors__name">Pavletich N.P. </li></ul><div class="ref__title">Structure of the retinoblastoma tumour-suppressor pocket domain bound to a peptide from HPV E7.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">1998; </span><span class="ref__seriesVolume">391</span></span>: <span class="ref__seriesPages">859-865</span><div class="ref__controls"><a href="#back-bib67" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-2642601106&amp;cf=" onclick="newWindow(this.href);return false">Scopus (353)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=9495340&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2F36038&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=391&publication_year=1998&pages=859-865&journal=Nature&author=Lee+J.O.&author=Russo+A.A.&author=Pavletich+N.P.&title=Structure+of+the+retinoblastoma+tumour-suppressor+pocket+domain+bound+to+a+peptide+from+HPV+E7">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lehmann B.D. </li><li class="ref__authors__name">Bauer J.A. </li><li class="ref__authors__name">Chen X. </li><li class="ref__authors__name">Sanders M.E. </li><li class="ref__authors__name">Chakravarthy A.B. </li><li class="ref__authors__name">Shyr Y. </li><li class="ref__authors__name">Pietenpol J.A. </li></ul><div class="ref__title">Identification of human triple-negative breast cancer subtypes and preclinical models for selection of targeted therapies.</div><span class="ref__series"><i>J. Clin. Invest.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">121</span></span>: <span class="ref__seriesPages">2750-2767</span><div class="ref__controls"><a href="#back-bib68" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-79960015997&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2925)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=21633166&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1172%2FJCI45014&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=121&publication_year=2011&pages=2750-2767&journal=J.%C2%A0Clin.+Invest.&author=Lehmann+B.D.&author=Bauer+J.A.&author=Chen+X.&author=Sanders+M.E.&author=Chakravarthy+A.B.&author=Shyr+Y.&author=Pietenpol+J.A.&title=Identification+of+human+triple-negative+breast+cancer+subtypes+and+preclinical+models+for+selection+of+targeted+therapies">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lehmann B.D. </li><li class="ref__authors__name">Bauer J.A. </li><li class="ref__authors__name">Schafer J.M. </li><li class="ref__authors__name">Pendleton C.S. </li><li class="ref__authors__name">Tang L. </li><li class="ref__authors__name">Johnson K.C. </li><li class="ref__authors__name">Chen X. </li><li class="ref__authors__name">Balko J.M. </li><li class="ref__authors__name">Gómez H. </li><li class="ref__authors__name">Arteaga C.L. </li><li>et al.</li></ul><div class="ref__title">PIK3CA mutations in androgen receptor-positive triple negative breast cancer confer sensitivity to the combination of PI3K and androgen receptor inhibitors.</div><span class="ref__series"><i>Breast Cancer Res.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">406</span><div class="ref__controls"><a href="#back-bib69" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84919875495&amp;cf=" onclick="newWindow(this.href);return false">Scopus (181)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=25103565&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2Fs13058-014-0406-x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2014&pages=406&journal=Breast+Cancer+Res.&author=Lehmann+B.D.&author=Bauer+J.A.&author=Schafer+J.M.&author=Pendleton+C.S.&author=Tang+L.&author=Johnson+K.C.&author=Chen+X.&author=Balko+J.M.&author=G%C3%B3mez+H.&author=Arteaga+C.L.&title=PIK3CA+mutations+in+androgen+receptor-positive+triple+negative+breast+cancer+confer+sensitivity+to+the+combination+of+PI3K+and+androgen+receptor+inhibitors">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lehmann B.D. </li><li class="ref__authors__name">Jovanović B. </li><li class="ref__authors__name">Chen X. </li><li class="ref__authors__name">Estrada M.V. </li><li class="ref__authors__name">Johnson K.N. </li><li class="ref__authors__name">Shyr Y. </li><li class="ref__authors__name">Moses H.L. </li><li class="ref__authors__name">Sanders M.E. </li><li class="ref__authors__name">Pietenpol J.A. </li></ul><div class="ref__title">Refinement of Triple-Negative Breast Cancer Molecular Subtypes: Implications for Neoadjuvant Chemotherapy Selection.</div><span class="ref__series"><i>PLoS ONE.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">e0157368</span><div class="ref__controls"><a href="#back-bib70" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27310713&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1371%2Fjournal.pone.0157368&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=e0157368&journal=PLoS+ONE&author=Lehmann+B.D.&author=Jovanovi%C4%87+B.&author=Chen+X.&author=Estrada+M.V.&author=Johnson+K.N.&author=Shyr+Y.&author=Moses+H.L.&author=Sanders+M.E.&author=Pietenpol+J.A.&title=Refinement+of+Triple-Negative+Breast+Cancer+Molecular+Subtypes%3A+Implications+for+Neoadjuvant+Chemotherapy+Selection">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Liao Y. </li><li class="ref__authors__name">Wang J. </li><li class="ref__authors__name">Jaehnig E.J. </li><li class="ref__authors__name">Shi Z. </li></ul><div class="ref__title">WebGestalt 2019: gene set analysis toolkit with revamped UIs and APIs.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">47</span></span>: <span class="ref__seriesPages">W199-W205</span><div class="ref__controls"><a href="#back-bib71" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85069234777&amp;cf=" onclick="newWindow(this.href);return false">Scopus (567)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31114916&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgkz401&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=W199-W205&journal=Nucleic+Acids+Res.&author=Liao+Y.&author=Wang+J.&author=Jaehnig+E.J.&author=Shi+Z.&title=WebGestalt+2019%3A+gene+set+analysis+toolkit+with+revamped+UIs+and+APIs">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Liberzon A. </li><li class="ref__authors__name">Birger C. </li><li class="ref__authors__name">Thorvaldsdóttir H. </li><li class="ref__authors__name">Ghandi M. </li><li class="ref__authors__name">Mesirov J.P. </li><li class="ref__authors__name">Tamayo P. </li></ul><div class="ref__title">The Molecular Signatures Database (MSigDB) hallmark gene set collection.</div><span class="ref__series"><i>Cell Syst.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">417-425</span><div class="ref__controls"><a href="#back-bib72" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84955328286&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2256)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26771021&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=417-425&journal=Cell+Syst.&author=Liberzon+A.&author=Birger+C.&author=Thorvaldsd%C3%B3ttir+H.&author=Ghandi+M.&author=Mesirov+J.P.&author=Tamayo+P.&title=The+Molecular+Signatures+Database+%28MSigDB%29+hallmark+gene+set+collection">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lin C.-C. </li><li class="ref__authors__name">Mabe N.W. </li><li class="ref__authors__name">Lin Y.-T. </li><li class="ref__authors__name">Yang W.-H. </li><li class="ref__authors__name">Tang X. </li><li class="ref__authors__name">Hong L. </li><li class="ref__authors__name">Sun T. </li><li class="ref__authors__name">Force J. </li><li class="ref__authors__name">Marks J.R. </li><li class="ref__authors__name">Yao T.-P. </li><li>et al.</li></ul><div class="ref__title">RIPK3 upregulation confers robust proliferation and collateral cystine-dependence on breast cancer recurrence.</div><span class="ref__series"><i>Cell Death Differ.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">27</span></span>: <span class="ref__seriesPages">2234-2247</span><div class="ref__controls"><a href="#back-bib73" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85078494025&amp;cf=" onclick="newWindow(this.href);return false">Scopus (12)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31988496&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41418-020-0499-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2020&pages=2234-2247&journal=Cell+Death+Differ.&author=Lin+C.-C.&author=Mabe+N.W.&author=Lin+Y.-T.&author=Yang+W.-H.&author=Tang+X.&author=Hong+L.&author=Sun+T.&author=Force+J.&author=Marks+J.R.&author=Yao+T.-P.&title=RIPK3+upregulation+confers+robust+proliferation+and+collateral+cystine-dependence+on+breast+cancer+recurrence">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Liu Y.-H. </li><li class="ref__authors__name">Tsang J.Y.S. </li><li class="ref__authors__name">Ni Y.-B. </li><li class="ref__authors__name">Hlaing T. </li><li class="ref__authors__name">Chan S.-K. </li><li class="ref__authors__name">Chan K.-F. </li><li class="ref__authors__name">Ko C.-W. </li><li class="ref__authors__name">Mujtaba S.S. </li><li class="ref__authors__name">Tse G.M. </li></ul><div class="ref__title">Doublecortin-like kinase 1 expression associates with breast cancer with neuroendocrine differentiation.</div><span class="ref__series"><i>Oncotarget.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">1464-1476</span><div class="ref__controls"><a href="#back-bib74" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84957692009&amp;cf=" onclick="newWindow(this.href);return false">Scopus (25)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26621833&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.18632%2Foncotarget.6386&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=1464-1476&journal=Oncotarget&author=Liu+Y.-H.&author=Tsang+J.Y.S.&author=Ni+Y.-B.&author=Hlaing+T.&author=Chan+S.-K.&author=Chan+K.-F.&author=Ko+C.-W.&author=Mujtaba+S.S.&author=Tse+G.M.&title=Doublecortin-like+kinase+1+expression+associates+with+breast+cancer+with+neuroendocrine+differentiation">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Liu C.-Y. </li><li class="ref__authors__name">Lau K.-Y. </li><li class="ref__authors__name">Hsu C.-C. </li><li class="ref__authors__name">Chen J.-L. </li><li class="ref__authors__name">Lee C.-H. </li><li class="ref__authors__name">Huang T.-T. </li><li class="ref__authors__name">Chen Y.-T. </li><li class="ref__authors__name">Huang C.-T. </li><li class="ref__authors__name">Lin P.-H. </li><li class="ref__authors__name">Tseng L.-M. </li></ul><div class="ref__title">Combination of palbociclib with enzalutamide shows in vitro activity in RB proficient and androgen receptor positive triple negative breast cancer cells.</div><span class="ref__series"><i>PLoS ONE.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">e0189007</span><div class="ref__controls"><a href="#back-bib75" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85038890108&amp;cf=" onclick="newWindow(this.href);return false">Scopus (9)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29261702&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1371%2Fjournal.pone.0189007&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=e0189007&journal=PLoS+ONE&author=Liu+C.-Y.&author=Lau+K.-Y.&author=Hsu+C.-C.&author=Chen+J.-L.&author=Lee+C.-H.&author=Huang+T.-T.&author=Chen+Y.-T.&author=Huang+C.-T.&author=Lin+P.-H.&author=Tseng+L.-M.&title=Combination+of+palbociclib+with+enzalutamide+shows+in%C2%A0vitro+activity+in+RB+proficient+and+androgen+receptor+positive+triple+negative+breast+cancer+cells">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Luengo A. </li><li class="ref__authors__name">Gui D.Y. </li><li class="ref__authors__name">Vander Heiden M.G. </li></ul><div class="ref__title">Targeting Metabolism for Cancer Therapy.</div><span class="ref__series"><i>Cell Chem. Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">1161-1180</span><div class="ref__controls"><a href="#back-bib76" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=28938091&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.chembiol.2017.08.028&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.chembiol.2017.08.028&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.chembiol.2017.08.028&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=1161-1180&journal=Cell+Chem.+Biol.&author=Luengo+A.&author=Gui+D.Y.&author=Vander+Heiden+M.G.&title=Targeting+Metabolism+for+Cancer+Therapy">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Malta T.M. </li><li class="ref__authors__name">Sokolov A. </li><li class="ref__authors__name">Gentles A.J. </li><li class="ref__authors__name">Burzykowski T. </li><li class="ref__authors__name">Poisson L. </li><li class="ref__authors__name">Weinstein J.N. </li><li class="ref__authors__name">Kamińska B. </li><li class="ref__authors__name">Huelsken J. </li><li class="ref__authors__name">Omberg L. </li><li class="ref__authors__name">Gevaert O. </li><li>et al.</li><li>Cancer Genome Atlas Research Network</li></ul><div class="ref__title">Machine Learning Identifies Stemness Features Associated with Oncogenic Dedifferentiation.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">173</span></span>: <span class="ref__seriesPages">338-354.e15</span><div class="ref__controls"><a href="#back-bib77" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85044967234&amp;cf=" onclick="newWindow(this.href);return false">Scopus (363)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29625051&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2018.03.034&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2018.03.034&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2018.03.034&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=338-354.e15&journal=Cell&author=Cancer+Genome+Atlas+Research+Network&title=Machine+Learning+Identifies+Stemness+Features+Associated+with+Oncogenic+Dedifferentiation">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Masuda M. </li><li class="ref__authors__name">Yamada T. </li></ul><div class="ref__title">The emergence of TNIK as a therapeutic target for colorectal cancer.</div><span class="ref__series"><i>Expert Opin. Ther. Targets.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">21</span></span>: <span class="ref__seriesPages">353-355</span><div class="ref__controls"><a href="#back-bib78" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85014502350&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=28281900&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1080%2F14728222.2017.1297799&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2017&pages=353-355&journal=Expert+Opin.+Ther.+Targets&author=Masuda+M.&author=Yamada+T.&title=The+emergence+of+TNIK+as+a+therapeutic+target+for+colorectal+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Matsuoka S. </li><li class="ref__authors__name">Ballif B.A. </li><li class="ref__authors__name">Smogorzewska A. </li><li class="ref__authors__name">McDonald 3rd, E.R. </li><li class="ref__authors__name">Hurov K.E. </li><li class="ref__authors__name">Luo J. </li><li class="ref__authors__name">Bakalarski C.E. </li><li class="ref__authors__name">Zhao Z. </li><li class="ref__authors__name">Solimini N. </li><li class="ref__authors__name">Lerenthal Y. </li><li>et al.</li></ul><div class="ref__title">ATM and ATR substrate analysis reveals extensive protein networks responsive to DNA damage.</div><span class="ref__series"><i>Science.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">316</span></span>: <span class="ref__seriesPages">1160-1166</span><div class="ref__controls"><a href="#back-bib79" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-34249947699&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2181)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=17525332&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1126%2Fscience.1140321&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=316&publication_year=2007&pages=1160-1166&journal=Science&author=Matsuoka+S.&author=Ballif+B.A.&author=Smogorzewska+A.&author=McDonald+E.R.&author=Hurov+K.E.&author=Luo+J.&author=Bakalarski+C.E.&author=Zhao+Z.&author=Solimini+N.&author=Lerenthal+Y.&title=ATM+and+ATR+substrate+analysis+reveals+extensive+protein+networks+responsive+to+DNA+damage">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mayakonda A. </li><li class="ref__authors__name">Lin D.-C. </li><li class="ref__authors__name">Assenov Y. </li><li class="ref__authors__name">Plass C. </li><li class="ref__authors__name">Koeffler H.P. </li></ul><div class="ref__title">Maftools: efficient and comprehensive analysis of somatic variants in cancer.</div><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">1747-1756</span><div class="ref__controls"><a href="#back-bib80" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85055908064&amp;cf=" onclick="newWindow(this.href);return false">Scopus (519)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30341162&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1101%2Fgr.239244.118&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=1747-1756&journal=Genome+Res.&author=Mayakonda+A.&author=Lin+D.-C.&author=Assenov+Y.&author=Plass+C.&author=Koeffler+H.P.&title=Maftools%3A+efficient+and+comprehensive+analysis+of+somatic+variants+in+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">McKenna A. </li><li class="ref__authors__name">Hanna M. </li><li class="ref__authors__name">Banks E. </li><li class="ref__authors__name">Sivachenko A. </li><li class="ref__authors__name">Cibulskis K. </li><li class="ref__authors__name">Kernytsky A. </li><li class="ref__authors__name">Garimella K. </li><li class="ref__authors__name">Altshuler D. </li><li class="ref__authors__name">Gabriel S. </li><li class="ref__authors__name">Daly M. </li><li class="ref__authors__name">DePristo M.A. </li></ul><div class="ref__title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</div><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">1297-1303</span><div class="ref__controls"><a href="#back-bib81" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-77956295988&amp;cf=" onclick="newWindow(this.href);return false">Scopus (12104)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=20644199&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1101%2Fgr.107524.110&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=1297-1303&journal=Genome+Res.&author=McKenna+A.&author=Hanna+M.&author=Banks+E.&author=Sivachenko+A.&author=Cibulskis+K.&author=Kernytsky+A.&author=Garimella+K.&author=Altshuler+D.&author=Gabriel+S.&author=Daly+M.&author=DePristo+M.A.&title=The+Genome+Analysis+Toolkit%3A+a+MapReduce+framework+for+analyzing+next-generation+DNA+sequencing+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">McLaren W. </li><li class="ref__authors__name">Gil L. </li><li class="ref__authors__name">Hunt S.E. </li><li class="ref__authors__name">Riat H.S. </li><li class="ref__authors__name">Ritchie G.R.S. </li><li class="ref__authors__name">Thormann A. </li><li class="ref__authors__name">Flicek P. </li><li class="ref__authors__name">Cunningham F. </li></ul><div class="ref__title">The Ensembl Variant Effect Predictor.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">122</span><div class="ref__controls"><a href="#back-bib82" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84973338712&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1927)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27268795&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2Fs13059-016-0974-4&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=122&journal=Genome+Biol.&author=McLaren+W.&author=Gil+L.&author=Hunt+S.E.&author=Riat+H.S.&author=Ritchie+G.R.S.&author=Thormann+A.&author=Flicek+P.&author=Cunningham+F.&title=The+Ensembl+Variant+Effect+Predictor">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mermel C.H. </li><li class="ref__authors__name">Schumacher S.E. </li><li class="ref__authors__name">Hill B. </li><li class="ref__authors__name">Meyerson M.L. </li><li class="ref__authors__name">Beroukhim R. </li><li class="ref__authors__name">Getz G. </li></ul><div class="ref__title">GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R41</span><div class="ref__controls"><a href="#back-bib83" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-79955166265&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1334)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=21527027&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2Fgb-2011-12-4-r41&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R41&journal=Genome+Biol.&author=Mermel+C.H.&author=Schumacher+S.E.&author=Hill+B.&author=Meyerson+M.L.&author=Beroukhim+R.&author=Getz+G.&title=GISTIC2.0+facilitates+sensitive+and+confident+localization+of+the+targets+of+focal+somatic+copy-number+alteration+in+human+cancers">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mertins P. </li><li class="ref__authors__name">Yang F. </li><li class="ref__authors__name">Liu T. </li><li class="ref__authors__name">Mani D.R. </li><li class="ref__authors__name">Petyuk V.A. </li><li class="ref__authors__name">Gillette M.A. </li><li class="ref__authors__name">Clauser K.R. </li><li class="ref__authors__name">Qiao J.W. </li><li class="ref__authors__name">Gritsenko M.A. </li><li class="ref__authors__name">Moore R.J. </li><li>et al.</li></ul><div class="ref__title">Ischemia in tumors induces early and sustained phosphorylation changes in stress kinase pathways but does not affect global protein levels.</div><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">1690-1704</span><div class="ref__controls"><a href="#back-bib84" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84904111890&amp;cf=" onclick="newWindow(this.href);return false">Scopus (222)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=24719451&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.M113.036392&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.M113.036392&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.M113.036392&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=1690-1704&journal=Mol.+Cell.+Proteomics&author=Mertins+P.&author=Yang+F.&author=Liu+T.&author=Mani+D.R.&author=Petyuk+V.A.&author=Gillette+M.A.&author=Clauser+K.R.&author=Qiao+J.W.&author=Gritsenko+M.A.&author=Moore+R.J.&title=Ischemia+in+tumors+induces+early+and+sustained+phosphorylation+changes+in+stress+kinase+pathways+but+does+not+affect+global+protein+levels">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mertins P. </li><li class="ref__authors__name">Mani D.R. </li><li class="ref__authors__name">Ruggles K.V. </li><li class="ref__authors__name">Gillette M.A. </li><li class="ref__authors__name">Clauser K.R. </li><li class="ref__authors__name">Wang P. </li><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Qiao J.W. </li><li class="ref__authors__name">Cao S. </li><li class="ref__authors__name">Petralia F. </li><li>et al.</li><li>NCI CPTAC</li></ul><div class="ref__title">Proteogenomics connects somatic mutations to signalling in breast cancer.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span><div class="ref__controls"><a href="#back-bib85" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (780)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=NCI+CPTAC&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mertins P. </li><li class="ref__authors__name">Tang L.C. </li><li class="ref__authors__name">Krug K. </li><li class="ref__authors__name">Clark D.J. </li><li class="ref__authors__name">Gritsenko M.A. </li><li class="ref__authors__name">Chen L. </li><li class="ref__authors__name">Clauser K.R. </li><li class="ref__authors__name">Clauss T.R. </li><li class="ref__authors__name">Shah P. </li><li class="ref__authors__name">Gillette M.A. </li><li>et al.</li></ul><div class="ref__title">Reproducible workflow for multiplexed deep-scale proteome and phosphoproteome analysis of tumor tissues by liquid chromatography-mass spectrometry.</div><span class="ref__series"><i>Nat. Protoc.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">1632-1661</span><div class="ref__controls"><a href="#back-bib86" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85049660247&amp;cf=" onclick="newWindow(this.href);return false">Scopus (121)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29988108&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41596-018-0006-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=1632-1661&journal=Nat.+Protoc.&author=Mertins+P.&author=Tang+L.C.&author=Krug+K.&author=Clark+D.J.&author=Gritsenko+M.A.&author=Chen+L.&author=Clauser+K.R.&author=Clauss+T.R.&author=Shah+P.&author=Gillette+M.A.&title=Reproducible+workflow+for+multiplexed+deep-scale+proteome+and+phosphoproteome+analysis+of+tumor+tissues+by+liquid+chromatography-mass+spectrometry">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Miotto B. </li><li class="ref__authors__name">Struhl K. </li></ul><div class="ref__title">HBO1 histone acetylase activity is essential for DNA replication licensing and inhibited by Geminin.</div><span class="ref__series"><i>Mol. Cell.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">37</span></span>: <span class="ref__seriesPages">57-66</span><div class="ref__controls"><a href="#back-bib87" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-73649089696&amp;cf=" onclick="newWindow(this.href);return false">Scopus (174)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=20129055&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.molcel.2009.12.012&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.molcel.2009.12.012&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.molcel.2009.12.012&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2010&pages=57-66&journal=Mol.+Cell&author=Miotto+B.&author=Struhl+K.&title=HBO1+histone+acetylase+activity+is+essential+for+DNA+replication+licensing+and+inhibited+by+Geminin">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mishima Y. </li><li class="ref__authors__name">Miyagi S. </li><li class="ref__authors__name">Saraya A. </li><li class="ref__authors__name">Negishi M. </li><li class="ref__authors__name">Endoh M. </li><li class="ref__authors__name">Endo T.A. </li><li class="ref__authors__name">Toyoda T. </li><li class="ref__authors__name">Shinga J. </li><li class="ref__authors__name">Katsumoto T. </li><li class="ref__authors__name">Chiba T. </li><li>et al.</li></ul><div class="ref__title">The Hbo1-Brd1/Brpf2 complex is responsible for global acetylation of H3K14 and required for fetal liver erythropoiesis.</div><span class="ref__series"><i>Blood.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">118</span></span>: <span class="ref__seriesPages">2443-2453</span><div class="ref__controls"><a href="#back-bib88" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-80052408223&amp;cf=" onclick="newWindow(this.href);return false">Scopus (114)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=21753189&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1182%2Fblood-2011-01-331892&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2011&pages=2443-2453&journal=Blood&author=Mishima+Y.&author=Miyagi+S.&author=Saraya+A.&author=Negishi+M.&author=Endoh+M.&author=Endo+T.A.&author=Toyoda+T.&author=Shinga+J.&author=Katsumoto+T.&author=Chiba+T.&title=The+Hbo1-Brd1%2FBrpf2+complex+is+responsible+for+global+acetylation+of+H3K14+and+required+for+fetal+liver+erythropoiesis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mounir M. </li><li class="ref__authors__name">Lucchetta M. </li><li class="ref__authors__name">Silva T.C. </li><li class="ref__authors__name">Olsen C. </li><li class="ref__authors__name">Bontempi G. </li><li class="ref__authors__name">Chen X. </li><li class="ref__authors__name">Noushmehr H. </li><li class="ref__authors__name">Colaprico A. </li><li class="ref__authors__name">Papaleo E. </li></ul><div class="ref__title">New functionalities in the TCGAbiolinks package for the study and integration of cancer data from GDC and GTEx.</div><span class="ref__series"><i>PLoS Comput. Biol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">e1006701</span><div class="ref__controls"><a href="#back-bib89" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85062948513&amp;cf=" onclick="newWindow(this.href);return false">Scopus (88)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30835723&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1371%2Fjournal.pcbi.1006701&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=e1006701&journal=PLoS+Comput.+Biol.&author=Mounir+M.&author=Lucchetta+M.&author=Silva+T.C.&author=Olsen+C.&author=Bontempi+G.&author=Chen+X.&author=Noushmehr+H.&author=Colaprico+A.&author=Papaleo+E.&title=New+functionalities+in+the+TCGAbiolinks+package+for+the+study+and+integration+of+cancer+data+from+GDC+and+GTEx">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mullarky E. </li><li class="ref__authors__name">Xu J. </li><li class="ref__authors__name">Robin A.D. </li><li class="ref__authors__name">Huggins D.J. </li><li class="ref__authors__name">Jennings A. </li><li class="ref__authors__name">Noguchi N. </li><li class="ref__authors__name">Olland A. </li><li class="ref__authors__name">Lakshminarasimhan D. </li><li class="ref__authors__name">Miller M. </li><li class="ref__authors__name">Tomita D. </li><li>et al.</li></ul><div class="ref__title">Inhibition of 3-phosphoglycerate dehydrogenase (PHGDH) by indole amides abrogates de novo serine synthesis in cancer cells.</div><span class="ref__series"><i>Bioorg. Med. Chem. Lett.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">2503-2510</span><div class="ref__controls"><a href="#back-bib90" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85069538867&amp;cf=" onclick="newWindow(this.href);return false">Scopus (21)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31327531&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.bmcl.2019.07.011&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=2503-2510&journal=Bioorg.+Med.+Chem.+Lett.&author=Mullarky+E.&author=Xu+J.&author=Robin+A.D.&author=Huggins+D.J.&author=Jennings+A.&author=Noguchi+N.&author=Olland+A.&author=Lakshminarasimhan+D.&author=Miller+M.&author=Tomita+D.&title=Inhibition+of+3-phosphoglycerate+dehydrogenase+%28PHGDH%29+by+indole+amides+abrogates+de+novo+serine+synthesis+in+cancer+cells">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Murphy J.P. </li><li class="ref__authors__name">Giacomantonio M.A. </li><li class="ref__authors__name">Paulo J.A. </li><li class="ref__authors__name">Everley R.A. </li><li class="ref__authors__name">Kennedy B.E. </li><li class="ref__authors__name">Pathak G.P. </li><li class="ref__authors__name">Clements D.R. </li><li class="ref__authors__name">Kim Y. </li><li class="ref__authors__name">Dai C. </li><li class="ref__authors__name">Sharif T. </li><li>et al.</li></ul><div class="ref__title">The NAD<sup>+</sup> Salvage Pathway Supports PHGDH-Driven Serine Biosynthesis.</div><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">2381-2391.e5</span><div class="ref__controls"><a href="#back-bib91" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85051748917&amp;cf=" onclick="newWindow(this.href);return false">Scopus (23)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30157431&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.celrep.2018.07.086&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.celrep.2018.07.086&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.celrep.2018.07.086&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=2381-2391.e5&journal=Cell+Rep.&author=Murphy+J.P.&author=Giacomantonio+M.A.&author=Paulo+J.A.&author=Everley+R.A.&author=Kennedy+B.E.&author=Pathak+G.P.&author=Clements+D.R.&author=Kim+Y.&author=Dai+C.&author=Sharif+T.&title=The+NAD%2B+Salvage+Pathway+Supports+PHGDH-Driven+Serine+Biosynthesis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Newman A.M. </li><li class="ref__authors__name">Liu C.L. </li><li class="ref__authors__name">Green M.R. </li><li class="ref__authors__name">Gentles A.J. </li><li class="ref__authors__name">Feng W. </li><li class="ref__authors__name">Xu Y. </li><li class="ref__authors__name">Hoang C.D. </li><li class="ref__authors__name">Diehn M. </li><li class="ref__authors__name">Alizadeh A.A. </li></ul><div class="ref__title">Robust enumeration of cell subsets from tissue expression profiles.</div><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">453-457</span><div class="ref__controls"><a href="#back-bib92" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84928927858&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2633)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=25822800&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnmeth.3337&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=453-457&journal=Nat.+Methods&author=Newman+A.M.&author=Liu+C.L.&author=Green+M.R.&author=Gentles+A.J.&author=Feng+W.&author=Xu+Y.&author=Hoang+C.D.&author=Diehn+M.&author=Alizadeh+A.A.&title=Robust+enumeration+of+cell+subsets+from+tissue+expression+profiles">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Nik-Zainal S. </li><li class="ref__authors__name">Davies H. </li><li class="ref__authors__name">Staaf J. </li><li class="ref__authors__name">Ramakrishna M. </li><li class="ref__authors__name">Glodzik D. </li><li class="ref__authors__name">Zou X. </li><li class="ref__authors__name">Martincorena I. </li><li class="ref__authors__name">Alexandrov L.B. </li><li class="ref__authors__name">Martin S. </li><li class="ref__authors__name">Wedge D.C. </li><li>et al.</li></ul><div class="ref__title">Landscape of somatic mutations in 560 breast cancer whole-genome sequences.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">47-54</span><div class="ref__controls"><a href="#back-bib93" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84973594792&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1028)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27135926&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature17676&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=47-54&journal=Nature&author=Nik-Zainal+S.&author=Davies+H.&author=Staaf+J.&author=Ramakrishna+M.&author=Glodzik+D.&author=Zou+X.&author=Martincorena+I.&author=Alexandrov+L.B.&author=Martin+S.&author=Wedge+D.C.&title=Landscape+of+somatic+mutations+in+560+breast+cancer+whole-genome+sequences">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Nusinow D.P. </li><li class="ref__authors__name">Szpyt J. </li><li class="ref__authors__name">Ghandi M. </li><li class="ref__authors__name">Rose C.M. </li><li class="ref__authors__name">McDonald 3rd, E.R. </li><li class="ref__authors__name">Kalocsay M. </li><li class="ref__authors__name">Jané-Valbuena J. </li><li class="ref__authors__name">Gelfand E. </li><li class="ref__authors__name">Schweppe D.K. </li><li class="ref__authors__name">Jedrychowski M. </li><li>et al.</li></ul><div class="ref__title">Quantitative Proteomics of the Cancer Cell Line Encyclopedia.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">180</span></span>: <span class="ref__seriesPages">387-402.e16</span><div class="ref__controls"><a href="#back-bib94" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85077916308&amp;cf=" onclick="newWindow(this.href);return false">Scopus (109)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31978347&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2019.12.023&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2019.12.023&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2019.12.023&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=180&publication_year=2020&pages=387-402.e16&journal=Cell&author=Nusinow+D.P.&author=Szpyt+J.&author=Ghandi+M.&author=Rose+C.M.&author=McDonald+E.R.&author=Kalocsay+M.&author=Jan%C3%A9-Valbuena+J.&author=Gelfand+E.&author=Schweppe+D.K.&author=Jedrychowski+M.&title=Quantitative+Proteomics+of+the+Cancer+Cell+Line+Encyclopedia">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">O’Leary B. </li><li class="ref__authors__name">Finn R.S. </li><li class="ref__authors__name">Turner N.C. </li></ul><div class="ref__title">Treating cancer with selective CDK4/6 inhibitors.</div><span class="ref__series"><i>Nat. Rev. Clin. Oncol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">417-430</span><div class="ref__controls"><a href="#back-bib95" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27030077&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnrclinonc.2016.26&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2016&pages=417-430&journal=Nat.+Rev.+Clin.+Oncol.&author=O%E2%80%99Leary+B.&author=Finn+R.S.&author=Turner+N.C.&title=Treating+cancer+with+selective+CDK4%2F6+inhibitors">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Parker J.S. </li><li class="ref__authors__name">Mullins M. </li><li class="ref__authors__name">Cheang M.C.U. </li><li class="ref__authors__name">Leung S. </li><li class="ref__authors__name">Voduc D. </li><li class="ref__authors__name">Vickery T. </li><li class="ref__authors__name">Davies S. </li><li class="ref__authors__name">Fauron C. </li><li class="ref__authors__name">He X. </li><li class="ref__authors__name">Hu Z. </li><li>et al.</li></ul><div class="ref__title">Supervised risk predictor of breast cancer based on intrinsic subtypes.</div><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">27</span></span>: <span class="ref__seriesPages">1160-1167</span><div class="ref__controls"><a href="#back-bib96" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-62449131093&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2602)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=19204204&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1200%2FJCO.2008.18.1370&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2009&pages=1160-1167&journal=J.%C2%A0Clin.+Oncol.&author=Parker+J.S.&author=Mullins+M.&author=Cheang+M.C.U.&author=Leung+S.&author=Voduc+D.&author=Vickery+T.&author=Davies+S.&author=Fauron+C.&author=He+X.&author=Hu+Z.&title=Supervised+risk+predictor+of+breast+cancer+based+on+intrinsic+subtypes">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Parkes E.E. </li><li class="ref__authors__name">Walker S.M. </li><li class="ref__authors__name">Taggart L.E. </li><li class="ref__authors__name">McCabe N. </li><li class="ref__authors__name">Knight L.A. </li><li class="ref__authors__name">Wilkinson R. </li><li class="ref__authors__name">McCloskey K.D. </li><li class="ref__authors__name">Buckley N.E. </li><li class="ref__authors__name">Savage K.I. </li><li class="ref__authors__name">Salto-Tellez M. </li><li>et al.</li></ul><div class="ref__title">Activation of STING-Dependent Innate Immune Signaling By S-Phase-Specific DNA Damage in Breast Cancer.</div><span class="ref__series"><i>J. Natl. Cancer Inst.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">109</span></span>: <span class="ref__seriesPages">109</span><div class="ref__controls"><a href="#back-bib97" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2016&pages=109&journal=J.%C2%A0Natl.+Cancer+Inst.&author=Parkes+E.E.&author=Walker+S.M.&author=Taggart+L.E.&author=McCabe+N.&author=Knight+L.A.&author=Wilkinson+R.&author=McCloskey+K.D.&author=Buckley+N.E.&author=Savage+K.I.&author=Salto-Tellez+M.&title=Activation+of+STING-Dependent+Innate+Immune+Signaling+By+S-Phase-Specific+DNA+Damage+in+Breast+Cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Patel J.M. </li><li class="ref__authors__name">Goss A. </li><li class="ref__authors__name">Garber J.E. </li><li class="ref__authors__name">Torous V. </li><li class="ref__authors__name">Richardson E.T. </li><li class="ref__authors__name">Haviland M.J. </li><li class="ref__authors__name">Hacker M.R. </li><li class="ref__authors__name">Freeman G.J. </li><li class="ref__authors__name">Nalven T. </li><li class="ref__authors__name">Alexander B. </li><li>et al.</li></ul><div class="ref__title">Retinoblastoma protein expression and its predictors in triple-negative breast cancer.</div><span class="ref__series"><i>NPJ Breast Cancer.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">19</span><div class="ref__controls"><a href="#back-bib98" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85085969298&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=32550264&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41523-020-0160-4&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2020&pages=19&journal=NPJ+Breast+Cancer&author=Patel+J.M.&author=Goss+A.&author=Garber+J.E.&author=Torous+V.&author=Richardson+E.T.&author=Haviland+M.J.&author=Hacker+M.R.&author=Freeman+G.J.&author=Nalven+T.&author=Alexander+B.&title=Retinoblastoma+protein+expression+and+its+predictors+in+triple-negative+breast+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Pavlova N.N. </li><li class="ref__authors__name">Thompson C.B. </li></ul><div class="ref__title">The Emerging Hallmarks of Cancer Metabolism.</div><span class="ref__series"><i>Cell Metab.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">27-47</span><div class="ref__controls"><a href="#back-bib99" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84955326448&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2067)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26771115&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cmet.2015.12.006&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cmet.2015.12.006&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cmet.2015.12.006&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=27-47&journal=Cell+Metab.&author=Pavlova+N.N.&author=Thompson+C.B.&title=The+Emerging+Hallmarks+of+Cancer+Metabolism">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Pernas S. </li><li class="ref__authors__name">Tolaney S.M. </li><li class="ref__authors__name">Winer E.P. </li><li class="ref__authors__name">Goel S. </li></ul><div class="ref__title">CDK4/6 inhibition in breast cancer: current practice and future directions.</div><span class="ref__series"><i>Ther. Adv. Med. Oncol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">10</span> (<span class="refComment">1758835918786451</span>)</span><div class="ref__controls"><a href="#back-bib100" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85054681352&amp;cf=" onclick="newWindow(this.href);return false">Scopus (129)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30038670&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1177%2F1758835918786451&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&journal=Ther.+Adv.+Med.+Oncol.&author=Pernas+S.&author=Tolaney+S.M.&author=Winer+E.P.&author=Goel+S.&title=CDK4%2F6+inhibition+in+breast+cancer%3A+current+practice+and+future+directions">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Phan L.M. </li><li class="ref__authors__name">Yeung S.-C.J. </li><li class="ref__authors__name">Lee M.-H. </li></ul><div class="ref__title">Cancer metabolic reprogramming: importance, main features, and potentials for precise targeted anti-cancer therapies.</div><span class="ref__series"><i>Cancer Biol. Med.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">1-19</span><div class="ref__controls"><a href="#back-bib101" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=24738035&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2014&pages=1-19&journal=Cancer+Biol.+Med.&author=Phan+L.M.&author=Yeung+S.-C.J.&author=Lee+M.-H.&title=Cancer+metabolic+reprogramming%3A+importance%2C+main+features%2C+and+potentials+for+precise+targeted+anti-cancer+therapies">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Poplin R. </li><li class="ref__authors__name">Ruano-Rubio V. </li><li class="ref__authors__name">DePristo M.A. </li><li class="ref__authors__name">Fennell T.J. </li><li class="ref__authors__name">Carneiro M.O. </li><li class="ref__authors__name">Van der Auwera G.A. </li><li class="ref__authors__name">Kling D.E. </li><li class="ref__authors__name">Gauthier L.D. </li><li class="ref__authors__name">Levy-Moonshine A. </li><li class="ref__authors__name">Roazen D. </li><li>et al.</li></ul><div class="ref__title">Scaling accurate genetic variant discovery to tens of thousands of samples.</div><span class="ref__series"><i>bioRxiv.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1101/201178">https://doi.org/10.1101/201178</a></span><div class="ref__controls"><a href="#back-bib102" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85094275866&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1101%2F201178&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&journal=bioRxiv&author=Poplin+R.&author=Ruano-Rubio+V.&author=DePristo+M.A.&author=Fennell+T.J.&author=Carneiro+M.O.&author=Van+der+Auwera+G.A.&author=Kling+D.E.&author=Gauthier+L.D.&author=Levy-Moonshine+A.&author=Roazen+D.&title=Scaling+accurate+genetic+variant+discovery+to+tens+of+thousands+of+samples">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Possemato R. </li><li class="ref__authors__name">Marks K.M. </li><li class="ref__authors__name">Shaul Y.D. </li><li class="ref__authors__name">Pacold M.E. </li><li class="ref__authors__name">Kim D. </li><li class="ref__authors__name">Birsoy K. </li><li class="ref__authors__name">Sethumadhavan S. </li><li class="ref__authors__name">Woo H.-K. </li><li class="ref__authors__name">Jang H.G. </li><li class="ref__authors__name">Jha A.K. </li><li>et al.</li></ul><div class="ref__title">Functional genomics reveal that the serine synthesis pathway is essential in breast cancer.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">476</span></span>: <span class="ref__seriesPages">346-350</span><div class="ref__controls"><a href="#back-bib103" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-80051923932&amp;cf=" onclick="newWindow(this.href);return false">Scopus (953)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=21760589&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature10350&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=476&publication_year=2011&pages=346-350&journal=Nature&author=Possemato+R.&author=Marks+K.M.&author=Shaul+Y.D.&author=Pacold+M.E.&author=Kim+D.&author=Birsoy+K.&author=Sethumadhavan+S.&author=Woo+H.-K.&author=Jang+H.G.&author=Jha+A.K.&title=Functional+genomics+reveal+that+the+serine+synthesis+pathway+is+essential+in+breast+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ramos A.H. </li><li class="ref__authors__name">Lichtenstein L. </li><li class="ref__authors__name">Gupta M. </li><li class="ref__authors__name">Lawrence M.S. </li><li class="ref__authors__name">Pugh T.J. </li><li class="ref__authors__name">Saksena G. </li><li class="ref__authors__name">Meyerson M. </li><li class="ref__authors__name">Getz G. </li></ul><div class="ref__title">Oncotator: cancer variant annotation tool.</div><span class="ref__series"><i>Hum. Mutat.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">36</span></span>: <span class="ref__seriesPages">E2423-E2429</span><div class="ref__controls"><a href="#back-bib104" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84925965197&amp;cf=" onclick="newWindow(this.href);return false">Scopus (283)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=25703262&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1002%2Fhumu.22771&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=E2423-E2429&journal=Hum.+Mutat.&author=Ramos+A.H.&author=Lichtenstein+L.&author=Gupta+M.&author=Lawrence+M.S.&author=Pugh+T.J.&author=Saksena+G.&author=Meyerson+M.&author=Getz+G.&title=Oncotator%3A+cancer+variant+annotation+tool">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Reimand J. </li><li class="ref__authors__name">Arak T. </li><li class="ref__authors__name">Adler P. </li><li class="ref__authors__name">Kolberg L. </li><li class="ref__authors__name">Reisberg S. </li><li class="ref__authors__name">Peterson H. </li><li class="ref__authors__name">Vilo J. </li></ul><div class="ref__title">g:Profiler-a web server for functional interpretation of gene lists (2016 update).</div><span class="ref__series"><i>Nucleic. Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">W83-W89</span><div class="ref__controls"><a href="#back-bib105" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27098042&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgkw199&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=W83-W89&journal=Nucleic.+Acids+Res.&author=Reimand+J.&author=Arak+T.&author=Adler+P.&author=Kolberg+L.&author=Reisberg+S.&author=Peterson+H.&author=Vilo+J.&title=g%3AProfiler-a+web+server+for+functional+interpretation+of+gene+lists+%282016+update%29">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ritchie M.E. </li><li class="ref__authors__name">Phipson B. </li><li class="ref__authors__name">Wu D. </li><li class="ref__authors__name">Hu Y. </li><li class="ref__authors__name">Law C.W. </li><li class="ref__authors__name">Shi W. </li><li class="ref__authors__name">Smyth G.K. </li></ul><div class="ref__title">limma powers differential expression analyses for RNA-sequencing and microarray studies.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">43</span></span>: <span class="ref__seriesPages">e47</span><div class="ref__controls"><a href="#back-bib106" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84926507971&amp;cf=" onclick="newWindow(this.href);return false">Scopus (10188)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=25605792&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgkv007&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=e47&journal=Nucleic+Acids+Res.&author=Ritchie+M.E.&author=Phipson+B.&author=Wu+D.&author=Hu+Y.&author=Law+C.W.&author=Shi+W.&author=Smyth+G.K.&title=limma+powers+differential+expression+analyses+for+RNA-sequencing+and+microarray+studies">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Roberts S.A. </li><li class="ref__authors__name">Gordenin D.A. </li></ul><div class="ref__title">Hypermutation in human cancer genomes: footprints and mechanisms.</div><span class="ref__series"><i>Nat. Rev. Cancer.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">786-800</span><div class="ref__controls"><a href="#back-bib107" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84925497848&amp;cf=" onclick="newWindow(this.href);return false">Scopus (243)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=25568919&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnrc3816&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=786-800&journal=Nat.+Rev.+Cancer&author=Roberts+S.A.&author=Gordenin+D.A.&title=Hypermutation+in+human+cancer+genomes%3A+footprints+and+mechanisms">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Rosenthal R. </li><li class="ref__authors__name">McGranahan N. </li><li class="ref__authors__name">Herrero J. </li><li class="ref__authors__name">Taylor B.S. </li><li class="ref__authors__name">Swanton C. </li></ul><div class="ref__title">DeconstructSigs: delineating mutational processes in single tumors distinguishes DNA repair deficiencies and patterns of carcinoma evolution.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">31</span><div class="ref__controls"><a href="#back-bib108" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84959467752&amp;cf=" onclick="newWindow(this.href);return false">Scopus (462)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26899170&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2Fs13059-016-0893-4&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=31&journal=Genome+Biol.&author=Rosenthal+R.&author=McGranahan+N.&author=Herrero+J.&author=Taylor+B.S.&author=Swanton+C.&title=DeconstructSigs%3A+delineating+mutational+processes+in+single+tumors+distinguishes+DNA+repair+deficiencies+and+patterns+of+carcinoma+evolution">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ross J.S. </li><li class="ref__authors__name">Symmans W.F. </li><li class="ref__authors__name">Pusztai L. </li><li class="ref__authors__name">Hortobagyi G.N. </li></ul><div class="ref__title">Standardizing slide-based assays in breast cancer: hormone receptors, HER2, and sentinel lymph nodes.</div><span class="ref__series"><i>Clin. Cancer Res.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">2831-2835</span><div class="ref__controls"><a href="#back-bib109" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-34249822213&amp;cf=" onclick="newWindow(this.href);return false">Scopus (52)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=17504980&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1158%2F1078-0432.CCR-06-2522&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2007&pages=2831-2835&journal=Clin.+Cancer+Res.&author=Ross+J.S.&author=Symmans+W.F.&author=Pusztai+L.&author=Hortobagyi+G.N.&title=Standardizing+slide-based+assays+in+breast+cancer%3A+hormone+receptors%2C+HER2%2C+and+sentinel+lymph+nodes">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ruggles K.V. </li><li class="ref__authors__name">Tang Z. </li><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Grover H. </li><li class="ref__authors__name">Askenazi M. </li><li class="ref__authors__name">Teubl J. </li><li class="ref__authors__name">Cao S. </li><li class="ref__authors__name">McLellan M.D. </li><li class="ref__authors__name">Clauser K.R. </li><li class="ref__authors__name">Tabb D.L. </li><li>et al.</li></ul><div class="ref__title">An analysis of the sensitivity of proteogenomic mapping of somatic mutations and novel splicing events in cancer.</div><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">1060-1071</span><div class="ref__controls"><a href="#back-bib110" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84962585103&amp;cf=" onclick="newWindow(this.href);return false">Scopus (56)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26631509&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.M115.056226&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.M115.056226&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.M115.056226&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=1060-1071&journal=Mol.+Cell.+Proteomics&author=Ruggles+K.V.&author=Tang+Z.&author=Wang+X.&author=Grover+H.&author=Askenazi+M.&author=Teubl+J.&author=Cao+S.&author=McLellan+M.D.&author=Clauser+K.R.&author=Tabb+D.L.&title=An+analysis+of+the+sensitivity+of+proteogenomic+mapping+of+somatic+mutations+and+novel+splicing+events+in+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ruggles K.V. </li><li class="ref__authors__name">Krug K. </li><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Clauser K.R. </li><li class="ref__authors__name">Wang J. </li><li class="ref__authors__name">Payne S.H. </li><li class="ref__authors__name">Fenyö D. </li><li class="ref__authors__name">Zhang B. </li><li class="ref__authors__name">Mani D.R. </li></ul><div class="ref__title">Methods, tools and current perspectives in proteogenomics.</div><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">959-981</span><div class="ref__controls"><a href="#back-bib111" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85020476458&amp;cf=" onclick="newWindow(this.href);return false">Scopus (68)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=28456751&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.MR117.000024&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.MR117.000024&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1074%2Fmcp.MR117.000024&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=959-981&journal=Mol.+Cell.+Proteomics&author=Ruggles+K.V.&author=Krug+K.&author=Wang+X.&author=Clauser+K.R.&author=Wang+J.&author=Payne+S.H.&author=Feny%C3%B6+D.&author=Zhang+B.&author=Mani+D.R.&title=Methods%2C+tools+and+current+perspectives+in+proteogenomics">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Salomonis N. </li><li class="ref__authors__name">Dexheimer P.J. </li><li class="ref__authors__name">Omberg L. </li><li class="ref__authors__name">Schroll R. </li><li class="ref__authors__name">Bush S. </li><li class="ref__authors__name">Huo J. </li><li class="ref__authors__name">Schriml L. </li><li class="ref__authors__name">Ho Sui S. </li><li class="ref__authors__name">Keddache M. </li><li class="ref__authors__name">Mayhew C. </li><li>et al.</li></ul><div class="ref__title">Integrated Genomic Analysis of Diverse Induced Pluripotent Stem Cells from the Progenitor Cell Biology Consortium.</div><span class="ref__series"><i>Stem Cell Reports.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">110-125</span><div class="ref__controls"><a href="#back-bib112" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27293150&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.stemcr.2016.05.006&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.stemcr.2016.05.006&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.stemcr.2016.05.006&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=110-125&journal=Stem+Cell+Reports&author=Salomonis+N.&author=Dexheimer+P.J.&author=Omberg+L.&author=Schroll+R.&author=Bush+S.&author=Huo+J.&author=Schriml+L.&author=Ho+Sui+S.&author=Keddache+M.&author=Mayhew+C.&title=Integrated+Genomic+Analysis+of+Diverse+Induced+Pluripotent+Stem+Cells+from+the+Progenitor+Cell+Biology+Consortium">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Satpathy S. </li><li class="ref__authors__name">Jaehnig E.J. </li><li class="ref__authors__name">Krug K. </li><li class="ref__authors__name">Kim B.-J. </li><li class="ref__authors__name">Saltzman A.B. </li><li class="ref__authors__name">Chan D.W. </li><li class="ref__authors__name">Holloway K.R. </li><li class="ref__authors__name">Anurag M. </li><li class="ref__authors__name">Huang C. </li><li class="ref__authors__name">Singh P. </li><li>et al.</li></ul><div class="ref__title">Microscaled proteogenomic methods for precision oncology.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">532</span><div class="ref__controls"><a href="#back-bib113" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85078365239&amp;cf=" onclick="newWindow(this.href);return false">Scopus (25)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31988290&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41467-020-14381-2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=532&journal=Nat.+Commun.&author=Satpathy+S.&author=Jaehnig+E.J.&author=Krug+K.&author=Kim+B.-J.&author=Saltzman+A.B.&author=Chan+D.W.&author=Holloway+K.R.&author=Anurag+M.&author=Huang+C.&author=Singh+P.&title=Microscaled+proteogenomic+methods+for+precision+oncology">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Schmidlin T. </li><li class="ref__authors__name">Debets D.O. </li><li class="ref__authors__name">van Gelder C.A.G.H. </li><li class="ref__authors__name">Stecker K.E. </li><li class="ref__authors__name">Rontogianni S. </li><li class="ref__authors__name">van den Eshof B.L. </li><li class="ref__authors__name">Kemper K. </li><li class="ref__authors__name">Lips E.H. </li><li class="ref__authors__name">van den Biggelaar M. </li><li class="ref__authors__name">Peeper D.S. </li><li>et al.</li></ul><div class="ref__title">High-Throughput Assessment of Kinome-wide Activation States.</div><span class="ref__series"><i>Cell Syst.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">366-374.e5</span><div class="ref__controls"><a href="#back-bib114" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85073437904&amp;cf=" onclick="newWindow(this.href);return false">Scopus (12)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31521607&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cels.2019.08.005&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cels.2019.08.005&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cels.2019.08.005&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=366-374.e5&journal=Cell+Syst.&author=Schmidlin+T.&author=Debets+D.O.&author=van+Gelder+C.A.G.H.&author=Stecker+K.E.&author=Rontogianni+S.&author=van+den+Eshof+B.L.&author=Kemper+K.&author=Lips+E.H.&author=van+den+Biggelaar+M.&author=Peeper+D.S.&title=High-Throughput+Assessment+of+Kinome-wide+Activation+States">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Shadforth I.P. </li><li class="ref__authors__name">Dunkley T.P.J. </li><li class="ref__authors__name">Lilley K.S. </li><li class="ref__authors__name">Bessant C. </li></ul><div class="ref__title">i-Tracker: for quantitative proteomics using iTRAQ.</div><span class="ref__series"><i>BMC Genomics.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">145</span><div class="ref__controls"><a href="#back-bib115" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-27744450286&amp;cf=" onclick="newWindow(this.href);return false">Scopus (220)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=16242023&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2F1471-2164-6-145&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=145&journal=BMC+Genomics&author=Shadforth+I.P.&author=Dunkley+T.P.J.&author=Lilley+K.S.&author=Bessant+C.&title=i-Tracker%3A+for+quantitative+proteomics+using+iTRAQ">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Shin S.-Y. </li><li class="ref__authors__name">Nguyen L.K. </li></ul><div class="ref__title">Unveiling Hidden Dynamics of Hippo Signalling: A Systems Analysis.</div><span class="ref__series"><i>Genes (Basel).</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">44</span><div class="ref__controls"><a href="#back-bib116" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84982292513&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.3390%2Fgenes7080044&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=44&journal=Genes+%28Basel%29&author=Shin+S.-Y.&author=Nguyen+L.K.&title=Unveiling+Hidden+Dynamics+of+Hippo+Signalling%3A+A+Systems+Analysis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Shindo Y. </li><li class="ref__authors__name">Hazama S. </li><li class="ref__authors__name">Tsunedomi R. </li><li class="ref__authors__name">Suzuki N. </li><li class="ref__authors__name">Nagano H. </li></ul><div class="ref__title">Novel Biomarkers for Personalized Cancer Immunotherapy.</div><span class="ref__series"><i>Cancers (Basel).</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">1223</span><div class="ref__controls"><a href="#back-bib117" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85071874767&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.3390%2Fcancers11091223&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=1223&journal=Cancers+%28Basel%29&author=Shindo+Y.&author=Hazama+S.&author=Tsunedomi+R.&author=Suzuki+N.&author=Nagano+H.&title=Novel+Biomarkers+for+Personalized+Cancer+Immunotherapy">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Smid M. </li><li class="ref__authors__name">Rodríguez-González F.G. </li><li class="ref__authors__name">Sieuwerts A.M. </li><li class="ref__authors__name">Salgado R. </li><li class="ref__authors__name">Prager-Van der Smissen W.J.C. </li><li class="ref__authors__name">Vlugt-Daane M.V. </li><li class="ref__authors__name">van Galen A. </li><li class="ref__authors__name">Nik-Zainal S. </li><li class="ref__authors__name">Staaf J. </li><li class="ref__authors__name">Brinkman A.B. </li><li>et al.</li></ul><div class="ref__title">Breast cancer genome and transcriptome integration implicates specific mutational signatures with immune cell infiltration.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">12910</span><div class="ref__controls"><a href="#back-bib118" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27666519&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fncomms12910&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=12910&journal=Nat.+Commun.&author=Smid+M.&author=Rodr%C3%ADguez-Gonz%C3%A1lez+F.G.&author=Sieuwerts+A.M.&author=Salgado+R.&author=Prager-Van+der+Smissen+W.J.C.&author=Vlugt-Daane+M.V.&author=van+Galen+A.&author=Nik-Zainal+S.&author=Staaf+J.&author=Brinkman+A.B.&title=Breast+cancer+genome+and+transcriptome+integration+implicates+specific+mutational+signatures+with+immune+cell+infiltration">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Smith J.A. </li><li class="ref__authors__name">Francis S.H. </li><li class="ref__authors__name">Corbin J.D. </li></ul><div class="ref__title">Autophosphorylation: a salient feature of protein kinases.</div><span class="ref__series"><i>Mol. Cell. Biochem.</i> <span class="ref__seriesDate">1993; </span><span class="ref__seriesVolume">127-128</span></span>: <span class="ref__seriesPages">51-70</span><div class="ref__controls"><a href="#back-bib119" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=7935362&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1007%2FBF01076757&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127-128&publication_year=1993&pages=51-70&journal=Mol.+Cell.+Biochem.&author=Smith+J.A.&author=Francis+S.H.&author=Corbin+J.D.&title=Autophosphorylation%3A+a+salient+feature+of+protein+kinases">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Sokolov A. </li><li class="ref__authors__name">Paull E.O. </li><li class="ref__authors__name">Stuart J.M. </li></ul><div class="ref__title">ONE-CLASS DETECTION OF CELL STATES IN TUMOR SUBTYPES.</div><span class="ref__series"><i>Pac. Symp. Biocomput.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">21</span></span>: <span class="ref__seriesPages">405-416</span><div class="ref__controls"><a href="#back-bib120" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26776204&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=405-416&journal=Pac.+Symp.+Biocomput.&author=Sokolov+A.&author=Paull+E.O.&author=Stuart+J.M.&title=ONE-CLASS+DETECTION+OF+CELL+STATES+IN+TUMOR+SUBTYPES">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Subramanian A. </li><li class="ref__authors__name">Tamayo P. </li><li class="ref__authors__name">Mootha V.K. </li><li class="ref__authors__name">Mukherjee S. </li><li class="ref__authors__name">Ebert B.L. </li><li class="ref__authors__name">Gillette M.A. </li><li class="ref__authors__name">Paulovich A. </li><li class="ref__authors__name">Pomeroy S.L. </li><li class="ref__authors__name">Golub T.R. </li><li class="ref__authors__name">Lander E.S. </li><li class="ref__authors__name">Mesirov J.P. </li></ul><div class="ref__title">Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles.</div><span class="ref__series"><i>Proc. Natl. Acad. Sci. USA.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">102</span></span>: <span class="ref__seriesPages">15545-15550</span><div class="ref__controls"><a href="#back-bib121" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-27344435774&amp;cf=" onclick="newWindow(this.href);return false">Scopus (20710)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=16199517&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1073%2Fpnas.0506580102&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=15545-15550&journal=Proc.+Natl.+Acad.+Sci.+USA&author=Subramanian+A.&author=Tamayo+P.&author=Mootha+V.K.&author=Mukherjee+S.&author=Ebert+B.L.&author=Gillette+M.A.&author=Paulovich+A.&author=Pomeroy+S.L.&author=Golub+T.R.&author=Lander+E.S.&author=Mesirov+J.P.&title=Gene+set+enrichment+analysis%3A+a+knowledge-based+approach+for+interpreting+genome-wide+expression+profiles">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Subramanian A. </li><li class="ref__authors__name">Narayan R. </li><li class="ref__authors__name">Corsello S.M. </li><li class="ref__authors__name">Peck D.D. </li><li class="ref__authors__name">Natoli T.E. </li><li class="ref__authors__name">Lu X. </li><li class="ref__authors__name">Gould J. </li><li class="ref__authors__name">Davis J.F. </li><li class="ref__authors__name">Tubelli A.A. </li><li class="ref__authors__name">Asiedu J.K. </li><li>et al.</li></ul><div class="ref__title">A Next Generation Connectivity Map: L1000 Platform and the First 1,000,000 Profiles.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">171</span></span>: <span class="ref__seriesPages">1437-1452.e17</span><div class="ref__controls"><a href="#back-bib122" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85036572148&amp;cf=" onclick="newWindow(this.href);return false">Scopus (772)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29195078&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2017&pages=1437-1452.e17&journal=Cell&author=Subramanian+A.&author=Narayan+R.&author=Corsello+S.M.&author=Peck+D.D.&author=Natoli+T.E.&author=Lu+X.&author=Gould+J.&author=Davis+J.F.&author=Tubelli+A.A.&author=Asiedu+J.K.&title=A+Next+Generation+Connectivity+Map%3A+L1000+Platform+and+the+First+1%2C000%2C000+Profiles">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Suehiro Y. </li><li class="ref__authors__name">Takemoto Y. </li><li class="ref__authors__name">Nishimoto A. </li><li class="ref__authors__name">Ueno K. </li><li class="ref__authors__name">Shirasawa B. </li><li class="ref__authors__name">Tanaka T. </li><li class="ref__authors__name">Kugimiya N. </li><li class="ref__authors__name">Suga A. </li><li class="ref__authors__name">Harada E. </li><li class="ref__authors__name">Hamano K. </li></ul><div class="ref__title">Dclk1 Inhibition Cancels 5-FU-induced Cell-cycle Arrest and Decreases Cell Survival in Colorectal Cancer.</div><span class="ref__series"><i>Anticancer Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">38</span></span>: <span class="ref__seriesPages">6225-6230</span><div class="ref__controls"><a href="#back-bib123" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85056107950&amp;cf=" onclick="newWindow(this.href);return false">Scopus (12)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30396941&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.21873%2Fanticanres.12977&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2018&pages=6225-6230&journal=Anticancer+Res.&author=Suehiro+Y.&author=Takemoto+Y.&author=Nishimoto+A.&author=Ueno+K.&author=Shirasawa+B.&author=Tanaka+T.&author=Kugimiya+N.&author=Suga+A.&author=Harada+E.&author=Hamano+K.&title=Dclk1+Inhibition+Cancels+5-FU-induced+Cell-cycle+Arrest+and+Decreases+Cell+Survival+in+Colorectal+Cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Tan V.Y.F. </li><li class="ref__authors__name">Févotte C. </li></ul><div class="ref__title">Automatic relevance determination in nonnegative matrix factorization with the β-divergence.</div><span class="ref__series"><i>IEEE Trans. Pattern Anal. Mach. Intell.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">1592-1605</span><div class="ref__controls"><a href="#back-bib124" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84878158173&amp;cf=" onclick="newWindow(this.href);return false">Scopus (150)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=23681989&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1109%2FTPAMI.2012.240&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2013&pages=1592-1605&journal=IEEE+Trans.+Pattern+Anal.+Mach.+Intell.&author=Tan+V.Y.F.&author=F%C3%A9votte+C.&title=Automatic+relevance+determination+in+nonnegative+matrix+factorization+with+the+%CE%B2-divergence">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Tate J.G. </li><li class="ref__authors__name">Bamford S. </li><li class="ref__authors__name">Jubb H.C. </li><li class="ref__authors__name">Sondka Z. </li><li class="ref__authors__name">Beare D.M. </li><li class="ref__authors__name">Bindal N. </li><li class="ref__authors__name">Boutselakis H. </li><li class="ref__authors__name">Cole C.G. </li><li class="ref__authors__name">Creatore C. </li><li class="ref__authors__name">Dawson E. </li><li>et al.</li></ul><div class="ref__title">COSMIC: the Catalogue Of Somatic Mutations In Cancer.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">47</span></span>: <span class="ref__seriesPages">D941-D947</span><div class="ref__controls"><a href="#back-bib125" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85057093998&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1040)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30371878&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgky1015&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D941-D947&journal=Nucleic+Acids+Res.&author=Tate+J.G.&author=Bamford+S.&author=Jubb+H.C.&author=Sondka+Z.&author=Beare+D.M.&author=Bindal+N.&author=Boutselakis+H.&author=Cole+C.G.&author=Creatore+C.&author=Dawson+E.&title=COSMIC%3A+the+Catalogue+Of+Somatic+Mutations+In+Cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Taylor-Weiner A. </li><li class="ref__authors__name">Stewart C. </li><li class="ref__authors__name">Giordano T. </li><li class="ref__authors__name">Miller M. </li><li class="ref__authors__name">Rosenberg M. </li><li class="ref__authors__name">Macbeth A. </li><li class="ref__authors__name">Lennon N. </li><li class="ref__authors__name">Rheinbay E. </li><li class="ref__authors__name">Landau D.-A. </li><li class="ref__authors__name">Wu C.J. </li><li class="ref__authors__name">Getz G. </li></ul><div class="ref__title">DeTiN: overcoming tumor-in-normal contamination.</div><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">531-534</span><div class="ref__controls"><a href="#back-bib126" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85048968644&amp;cf=" onclick="newWindow(this.href);return false">Scopus (20)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29941871&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41592-018-0036-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=531-534&journal=Nat.+Methods&author=Taylor-Weiner+A.&author=Stewart+C.&author=Giordano+T.&author=Miller+M.&author=Rosenberg+M.&author=Macbeth+A.&author=Lennon+N.&author=Rheinbay+E.&author=Landau+D.-A.&author=Wu+C.J.&author=Getz+G.&title=DeTiN%3A+overcoming+tumor-in-normal+contamination">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Telli M.L. </li><li class="ref__authors__name">Gradishar W.J. </li><li class="ref__authors__name">Ward J.H. </li></ul><div class="ref__title">NCCN Guidelines Updates: Breast Cancer.</div><span class="ref__series"><i>J. Natl. Compr. Canc. Netw.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">552-555</span><div class="ref__controls"><a href="#back-bib127" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31117035&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2019&pages=552-555&journal=J.%C2%A0Natl.+Compr.+Canc.+Netw.&author=Telli+M.L.&author=Gradishar+W.J.&author=Ward+J.H.&title=NCCN+Guidelines+Updates%3A+Breast+Cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Thorsson V. </li><li class="ref__authors__name">Gibbs D.L. </li><li class="ref__authors__name">Brown S.D. </li><li class="ref__authors__name">Wolf D. </li><li class="ref__authors__name">Bortone D.S. </li><li class="ref__authors__name">Ou Yang T.-H. </li><li class="ref__authors__name">Porta-Pardo E. </li><li class="ref__authors__name">Gao G.F. </li><li class="ref__authors__name">Plaisier C.L. </li><li class="ref__authors__name">Eddy J.A. </li><li>et al.</li><li>Cancer Genome Atlas Research Network</li></ul><div class="ref__title">The Immune Landscape of Cancer.</div><span class="ref__series"><i>Immunity.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">51</span></span>: <span class="ref__seriesPages">411-412</span><div class="ref__controls"><a href="#back-bib128" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85070213901&amp;cf=" onclick="newWindow(this.href);return false">Scopus (61)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31433971&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.immuni.2019.08.004&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.immuni.2019.08.004&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.immuni.2019.08.004&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2019&pages=411-412&journal=Immunity&author=Cancer+Genome+Atlas+Research+Network&title=The+Immune+Landscape+of+Cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Trapnell C. </li><li class="ref__authors__name">Williams B.A. </li><li class="ref__authors__name">Pertea G. </li><li class="ref__authors__name">Mortazavi A. </li><li class="ref__authors__name">Kwan G. </li><li class="ref__authors__name">van Baren M.J. </li><li class="ref__authors__name">Salzberg S.L. </li><li class="ref__authors__name">Wold B.J. </li><li class="ref__authors__name">Pachter L. </li></ul><div class="ref__title">Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation.</div><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">511-515</span><div class="ref__controls"><a href="#back-bib129" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-77952123055&amp;cf=" onclick="newWindow(this.href);return false">Scopus (9185)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=20436464&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnbt.1621&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2010&pages=511-515&journal=Nat.+Biotechnol.&author=Trapnell+C.&author=Williams+B.A.&author=Pertea+G.&author=Mortazavi+A.&author=Kwan+G.&author=van+Baren+M.J.&author=Salzberg+S.L.&author=Wold+B.J.&author=Pachter+L.&title=Transcript+assembly+and+quantification+by+RNA-Seq+reveals+unannotated+transcripts+and+isoform+switching+during+cell+differentiation">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Tyanova S. </li><li class="ref__authors__name">Albrechtsen R. </li><li class="ref__authors__name">Kronqvist P. </li><li class="ref__authors__name">Cox J. </li><li class="ref__authors__name">Mann M. </li><li class="ref__authors__name">Geiger T. </li></ul><div class="ref__title">Proteomic maps of breast cancer subtypes.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">10259</span><div class="ref__controls"><a href="#back-bib130" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26725330&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fncomms10259&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=10259&journal=Nat.+Commun.&author=Tyanova+S.&author=Albrechtsen+R.&author=Kronqvist+P.&author=Cox+J.&author=Mann+M.&author=Geiger+T.&title=Proteomic+maps+of+breast+cancer+subtypes">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Udeshi N.D. </li><li class="ref__authors__name">Mani D.C. </li><li class="ref__authors__name">Satpathy S. </li><li class="ref__authors__name">Fereshetian S. </li><li class="ref__authors__name">Gasser J.A. </li><li class="ref__authors__name">Svinkina T. </li><li class="ref__authors__name">Olive M.E. </li><li class="ref__authors__name">Ebert B.L. </li><li class="ref__authors__name">Mertins P. </li><li class="ref__authors__name">Carr S.A. </li></ul><div class="ref__title">Rapid and deep-scale ubiquitylation profiling for biology and translational research.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">359</span><div class="ref__controls"><a href="#back-bib131" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85078023220&amp;cf=" onclick="newWindow(this.href);return false">Scopus (26)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31953384&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41467-019-14175-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=359&journal=Nat.+Commun.&author=Udeshi+N.D.&author=Mani+D.C.&author=Satpathy+S.&author=Fereshetian+S.&author=Gasser+J.A.&author=Svinkina+T.&author=Olive+M.E.&author=Ebert+B.L.&author=Mertins+P.&author=Carr+S.A.&title=Rapid+and+deep-scale+ubiquitylation+profiling+for+biology+and+translational+research">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Vander Heiden M.G. </li><li class="ref__authors__name">DeBerardinis R.J. </li></ul><div class="ref__title">Understanding the Intersections between Metabolism and Cancer Biology.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">168</span></span>: <span class="ref__seriesPages">657-669</span><div class="ref__controls"><a href="#back-bib132" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85012011397&amp;cf=" onclick="newWindow(this.href);return false">Scopus (791)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=28187287&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2016.12.039&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2016.12.039&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2016.12.039&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=168&publication_year=2017&pages=657-669&journal=Cell&author=Vander+Heiden+M.G.&author=DeBerardinis+R.J.&title=Understanding+the+Intersections+between+Metabolism+and+Cancer+Biology">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Varn F.S. </li><li class="ref__authors__name">Andrews E.H. </li><li class="ref__authors__name">Mullins D.W. </li><li class="ref__authors__name">Cheng C. </li></ul><div class="ref__title">Integrative analysis of breast cancer reveals prognostic haematopoietic activity and patient-specific immune response profiles.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">10248</span><div class="ref__controls"><a href="#back-bib133" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84953306200&amp;cf=" onclick="newWindow(this.href);return false">Scopus (29)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26725977&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fncomms10248&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=10248&journal=Nat.+Commun.&author=Varn+F.S.&author=Andrews+E.H.&author=Mullins+D.W.&author=Cheng+C.&title=Integrative+analysis+of+breast+cancer+reveals+prognostic+haematopoietic+activity+and+patient-specific+immune+response+profiles">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Vasaikar S.V. </li><li class="ref__authors__name">Straub P. </li><li class="ref__authors__name">Wang J. </li><li class="ref__authors__name">Zhang B. </li></ul><div class="ref__title">LinkedOmics: analyzing multi-omics data within and across 32 cancer types.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">46</span></span>: <span class="ref__seriesPages">D956-D963</span><div class="ref__controls"><a href="#back-bib134" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85040936922&amp;cf=" onclick="newWindow(this.href);return false">Scopus (521)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29136207&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgkx1090&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D956-D963&journal=Nucleic+Acids+Res.&author=Vasaikar+S.V.&author=Straub+P.&author=Wang+J.&author=Zhang+B.&title=LinkedOmics%3A+analyzing+multi-omics+data+within+and+across+32+cancer+types">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Vasaikar S. </li><li class="ref__authors__name">Huang C. </li><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Petyuk V.A. </li><li class="ref__authors__name">Savage S.R. </li><li class="ref__authors__name">Wen B. </li><li class="ref__authors__name">Dou Y. </li><li class="ref__authors__name">Zhang Y. </li><li class="ref__authors__name">Shi Z. </li><li class="ref__authors__name">Arshad O.A. </li><li>et al.</li><li>Clinical Proteomic Tumor Analysis Consortium</li></ul><div class="ref__title">Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">177</span></span>: <span class="ref__seriesPages">1035-1049.e19</span><div class="ref__controls"><a href="#back-bib135" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85064886726&amp;cf=" onclick="newWindow(this.href);return false">Scopus (175)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31031003&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=177&publication_year=2019&pages=1035-1049.e19&journal=Cell&author=Clinical+Proteomic+Tumor+Analysis+Consortium&title=Proteogenomic+Analysis+of+Human+Colon+Cancer+Reveals+New+Therapeutic+Opportunities">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Verdin E. </li><li class="ref__authors__name">Ott M. </li></ul><div class="ref__title">50 years of protein acetylation: from gene regulation to epigenetics, metabolism and beyond.</div><span class="ref__series"><i>Nat. Rev. Mol. Cell Biol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">258-264</span><div class="ref__controls"><a href="#back-bib136" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84926163974&amp;cf=" onclick="newWindow(this.href);return false">Scopus (398)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=25549891&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnrm3931&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=258-264&journal=Nat.+Rev.+Mol.+Cell+Biol.&author=Verdin+E.&author=Ott+M.&title=50+years+of+protein+acetylation%3A+from+gene+regulation+to+epigenetics%2C+metabolism+and+beyond">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wang Z.-X. </li><li class="ref__authors__name">Wu J.-W. </li></ul><div class="ref__title">Autophosphorylation kinetics of protein kinases.</div><span class="ref__series"><i>Biochem. J.</i> <span class="ref__seriesDate">2002; </span><span class="ref__seriesVolume">368</span></span>: <span class="ref__seriesPages">947-952</span><div class="ref__controls"><a href="#back-bib137" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=12190618&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1042%2Fbj20020557&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2002&pages=947-952&journal=Biochem.+J.&author=Wang+Z.-X.&author=Wu+J.-W.&title=Autophosphorylation+kinetics+of+protein+kinases">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Weinert B.T. </li><li class="ref__authors__name">Moustafa T. </li><li class="ref__authors__name">Iesmantavicius V. </li><li class="ref__authors__name">Zechner R. </li><li class="ref__authors__name">Choudhary C. </li></ul><div class="ref__title">Analysis of acetylation stoichiometry suggests that SIRT3 repairs nonenzymatic acetylation lesions.</div><span class="ref__series"><i>EMBO J.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">34</span></span>: <span class="ref__seriesPages">2620-2632</span><div class="ref__controls"><a href="#back-bib138" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84946488466&amp;cf=" onclick="newWindow(this.href);return false">Scopus (80)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=26358839&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.15252%2Fembj.201591271&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2015&pages=2620-2632&journal=EMBO+J.&author=Weinert+B.T.&author=Moustafa+T.&author=Iesmantavicius+V.&author=Zechner+R.&author=Choudhary+C.&title=Analysis+of+acetylation+stoichiometry+suggests+that+SIRT3+repairs+nonenzymatic+acetylation+lesions">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Weinert B.T. </li><li class="ref__authors__name">Narita T. </li><li class="ref__authors__name">Satpathy S. </li><li class="ref__authors__name">Srinivasan B. </li><li class="ref__authors__name">Hansen B.K. </li><li class="ref__authors__name">Schölz C. </li><li class="ref__authors__name">Hamilton W.B. </li><li class="ref__authors__name">Zucconi B.E. </li><li class="ref__authors__name">Wang W.W. </li><li class="ref__authors__name">Liu W.R. </li><li>et al.</li></ul><div class="ref__title">Time-Resolved Analysis Reveals Rapid Dynamics and Broad Scope of the CBP/p300 Acetylome.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">174</span></span>: <span class="ref__seriesPages">231-244.e12</span><div class="ref__controls"><a href="#back-bib139" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85047227031&amp;cf=" onclick="newWindow(this.href);return false">Scopus (108)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=29804834&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2018.04.033&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2018.04.033&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=4&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1016%2Fj.cell.2018.04.033&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=174&publication_year=2018&pages=231-244.e12&journal=Cell&author=Weinert+B.T.&author=Narita+T.&author=Satpathy+S.&author=Srinivasan+B.&author=Hansen+B.K.&author=Sch%C3%B6lz+C.&author=Hamilton+W.B.&author=Zucconi+B.E.&author=Wang+W.W.&author=Liu+W.R.&title=Time-Resolved+Analysis+Reveals+Rapid+Dynamics+and+Broad+Scope+of+the+CBP%2Fp300+Acetylome">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Weinstabl H. </li><li class="ref__authors__name">Treu M. </li><li class="ref__authors__name">Rinnenthal J. </li><li class="ref__authors__name">Zahn S.K. </li><li class="ref__authors__name">Ettmayer P. </li><li class="ref__authors__name">Bader G. </li><li class="ref__authors__name">Dahmann G. </li><li class="ref__authors__name">Kessler D. </li><li class="ref__authors__name">Rumpel K. </li><li class="ref__authors__name">Mischerikow N. </li><li>et al.</li></ul><div class="ref__title">Intracellular Trapping of the Selective Phosphoglycerate Dehydrogenase (PHGDH) Inhibitor BI-4924 Disrupts Serine Biosynthesis.</div><span class="ref__series"><i>J. Med. Chem.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">62</span></span>: <span class="ref__seriesPages">7976-7997</span><div class="ref__controls"><a href="#back-bib140" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85071640342&amp;cf=" onclick="newWindow(this.href);return false">Scopus (23)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=31365252&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1021%2Facs.jmedchem.9b00718&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=7976-7997&journal=J.%C2%A0Med.+Chem.&author=Weinstabl+H.&author=Treu+M.&author=Rinnenthal+J.&author=Zahn+S.K.&author=Ettmayer+P.&author=Bader+G.&author=Dahmann+G.&author=Kessler+D.&author=Rumpel+K.&author=Mischerikow+N.&title=Intracellular+Trapping+of+the+Selective+Phosphoglycerate+Dehydrogenase+%28PHGDH%29+Inhibitor+BI-4924+Disrupts+Serine+Biosynthesis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Weygant N. </li><li class="ref__authors__name">Qu D. </li><li class="ref__authors__name">Berry W.L. </li><li class="ref__authors__name">May R. </li><li class="ref__authors__name">Chandrakesan P. </li><li class="ref__authors__name">Owen D.B. </li><li class="ref__authors__name">Sureban S.M. </li><li class="ref__authors__name">Ali N. </li><li class="ref__authors__name">Janknecht R. </li><li class="ref__authors__name">Houchen C.W. </li></ul><div class="ref__title">Small molecule kinase inhibitor LRRK2-IN-1 demonstrates potent activity against colorectal and pancreatic cancer through inhibition of doublecortin-like kinase 1.</div><span class="ref__series"><i>Mol. Cancer.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">103</span><div class="ref__controls"><a href="#back-bib141" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84901279099&amp;cf=" onclick="newWindow(this.href);return false">Scopus (62)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=24885928&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1186%2F1476-4598-13-103&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=103&journal=Mol.+Cancer&author=Weygant+N.&author=Qu+D.&author=Berry+W.L.&author=May+R.&author=Chandrakesan+P.&author=Owen+D.B.&author=Sureban+S.M.&author=Ali+N.&author=Janknecht+R.&author=Houchen+C.W.&title=Small+molecule+kinase+inhibitor+LRRK2-IN-1+demonstrates+potent+activity+against+colorectal+and+pancreatic+cancer+through+inhibition+of+doublecortin-like+kinase+1">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Whitfield M.L. </li><li class="ref__authors__name">Sherlock G. </li><li class="ref__authors__name">Saldanha A.J. </li><li class="ref__authors__name">Murray J.I. </li><li class="ref__authors__name">Ball C.A. </li><li class="ref__authors__name">Alexander K.E. </li><li class="ref__authors__name">Matese J.C. </li><li class="ref__authors__name">Perou C.M. </li><li class="ref__authors__name">Hurt M.M. </li><li class="ref__authors__name">Brown P.O. </li><li class="ref__authors__name">Botstein D. </li></ul><div class="ref__title">Identification of genes periodically expressed in the human cell cycle and their expression in tumors.</div><span class="ref__series"><i>Mol. Biol. Cell.</i> <span class="ref__seriesDate">2002; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">1977-2000</span><div class="ref__controls"><a href="#back-bib142" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-0035985177&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1127)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=12058064&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1091%2Fmbc.02-02-0030.&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2002&pages=1977-2000&journal=Mol.+Biol.+Cell&author=Whitfield+M.L.&author=Sherlock+G.&author=Saldanha+A.J.&author=Murray+J.I.&author=Ball+C.A.&author=Alexander+K.E.&author=Matese+J.C.&author=Perou+C.M.&author=Hurt+M.M.&author=Brown+P.O.&author=Botstein+D.&title=Identification+of+genes+periodically+expressed+in+the+human+cell+cycle+and+their+expression+in+tumors">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Xiong Y. </li><li class="ref__authors__name">Wang M. </li><li class="ref__authors__name">Zhao J. </li><li class="ref__authors__name">Han Y. </li><li class="ref__authors__name">Jia L. </li></ul><div class="ref__title">Sirtuin 3: A Janus face in cancer (Review).</div><span class="ref__series"><i>Int. J. Oncol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">49</span></span>: <span class="ref__seriesPages">2227-2235</span><div class="ref__controls"><a href="#back-bib143" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84996557477&amp;cf=" onclick="newWindow(this.href);return false">Scopus (15)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=27840909&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.3892%2Fijo.2016.3767&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2016&pages=2227-2235&journal=Int.+J.+Oncol.&author=Xiong+Y.&author=Wang+M.&author=Zhao+J.&author=Han+Y.&author=Jia+L.&title=Sirtuin+3%3A+A+Janus+face+in+cancer+%28Review%29">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Yamamoto T. </li><li class="ref__authors__name">Kanaya N. </li><li class="ref__authors__name">Somlo G. </li><li class="ref__authors__name">Chen S. </li></ul><div class="ref__title">Synergistic anti-cancer activity of CDK4/6 inhibitor palbociclib and dual mTOR kinase inhibitor MLN0128 in pRb-expressing ER-negative breast cancer.</div><span class="ref__series"><i>Breast Cancer Res. Treat.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">174</span></span>: <span class="ref__seriesPages">615-625</span><div class="ref__controls"><a href="#back-bib144" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85059592784&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30607633&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1007%2Fs10549-018-05104-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=174&publication_year=2019&pages=615-625&journal=Breast+Cancer+Res.+Treat.&author=Yamamoto+T.&author=Kanaya+N.&author=Somlo+G.&author=Chen+S.&title=Synergistic+anti-cancer+activity+of+CDK4%2F6+inhibitor+palbociclib+and+dual+mTOR+kinase+inhibitor+MLN0128+in+pRb-expressing+ER-negative+breast+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Yang W. </li><li class="ref__authors__name">Soares J. </li><li class="ref__authors__name">Greninger P. </li><li class="ref__authors__name">Edelman E.J. </li><li class="ref__authors__name">Lightfoot H. </li><li class="ref__authors__name">Forbes S. </li><li class="ref__authors__name">Bindal N. </li><li class="ref__authors__name">Beare D. </li><li class="ref__authors__name">Smith J.A. </li><li class="ref__authors__name">Thompson I.R. </li><li>et al.</li></ul><div class="ref__title">Genomics of Drug Sensitivity in Cancer (GDSC): a resource for therapeutic biomarker discovery in cancer cells.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">41</span></span>: <span class="ref__seriesPages">D955-D961</span><div class="ref__controls"><a href="#back-bib145" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84876563391&amp;cf=" onclick="newWindow(this.href);return false">Scopus (893)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=23180760&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1093%2Fnar%2Fgks1111&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=D955-D961&journal=Nucleic+Acids+Res.&author=Yang+W.&author=Soares+J.&author=Greninger+P.&author=Edelman+E.J.&author=Lightfoot+H.&author=Forbes+S.&author=Bindal+N.&author=Beare+D.&author=Smith+J.A.&author=Thompson+I.R.&title=Genomics+of+Drug+Sensitivity+in+Cancer+%28GDSC%29%3A+a+resource+for+therapeutic+biomarker+discovery+in+cancer+cells">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Yoshihara K. </li><li class="ref__authors__name">Shahmoradgoli M. </li><li class="ref__authors__name">Martínez E. </li><li class="ref__authors__name">Vegesna R. </li><li class="ref__authors__name">Kim H. </li><li class="ref__authors__name">Torres-Garcia W. </li><li class="ref__authors__name">Treviño V. </li><li class="ref__authors__name">Shen H. </li><li class="ref__authors__name">Laird P.W. </li><li class="ref__authors__name">Levine D.A. </li><li>et al.</li></ul><div class="ref__title">Inferring tumour purity and stromal and immune cell admixture from expression data.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">2612</span><div class="ref__controls"><a href="#back-bib146" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84885673911&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1675)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=24113773&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fncomms3612&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2612&journal=Nat.+Commun.&author=Yoshihara+K.&author=Shahmoradgoli+M.&author=Mart%C3%ADnez+E.&author=Vegesna+R.&author=Kim+H.&author=Torres-Garcia+W.&author=Trevi%C3%B1o+V.&author=Shen+H.&author=Laird+P.W.&author=Levine+D.A.&title=Inferring+tumour+purity+and+stromal+and+immune+cell+admixture+from+expression+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Zhang B. </li><li class="ref__authors__name">Wang J. </li><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Zhu J. </li><li class="ref__authors__name">Liu Q. </li><li class="ref__authors__name">Shi Z. </li><li class="ref__authors__name">Chambers M.C. </li><li class="ref__authors__name">Zimmerman L.J. </li><li class="ref__authors__name">Shaddox K.F. </li><li class="ref__authors__name">Kim S. </li><li>et al.</li><li>NCI CPTAC</li></ul><div class="ref__title">Proteogenomic characterization of human colon and rectal cancer.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">513</span></span>: <span class="ref__seriesPages">382-387</span><div class="ref__controls"><a href="#back-bib147" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-84907269780&amp;cf=" onclick="newWindow(this.href);return false">Scopus (797)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=25043054&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fnature13438&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2014&pages=382-387&journal=Nature&author=NCI+CPTAC&title=Proteogenomic+characterization+of+human+colon+and+rectal+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Zhang B. </li><li class="ref__authors__name">Whiteaker J.R. </li><li class="ref__authors__name">Hoofnagle A.N. </li><li class="ref__authors__name">Baird G.S. </li><li class="ref__authors__name">Rodland K.D. </li><li class="ref__authors__name">Paulovich A.G. </li></ul><div class="ref__title">Clinical potential of mass spectrometry-based proteogenomics.</div><span class="ref__series"><i>Nat. Rev. Clin. Oncol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">256-268</span><div class="ref__controls"><a href="#back-bib148" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2020.10.036&amp;key=2-s2.0-85057891078&amp;cf=" onclick="newWindow(this.href);return false">Scopus (66)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=8&amp;doi=10.1016/j.cell.2020.10.036&amp;key=30487530&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=16&amp;doi=10.1016/j.cell.2020.10.036&amp;key=10.1038%2Fs41571-018-0135-7&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2019&pages=256-268&journal=Nat.+Rev.+Clin.+Oncol.&author=Zhang+B.&author=Whiteaker+J.R.&author=Hoofnagle+A.N.&author=Baird+G.S.&author=Rodland+K.D.&author=Paulovich+A.G.&title=Clinical+potential+of+mass+spectrometry-based+proteogenomics">Google Scholar</a></li></ul></div></li></ol></section><section class="article-info"><h2 class="top" id="articleInformation">Article info</h2><section><h3 class="publicationHistory__label">Publication history</h3><div class="article-info__dates"><div class="article-info__date">Published: November 18, 2020</div><div class="article-info__date">Accepted:
            October 21,
            2020</div><div class="article-info__date">Received in revised form:
            July 14,
            2020</div><div class="article-info__date">Received:
            March 3,
            2020</div></div></section><section><h3>Identification</h3><p>DOI: <a href="https://doi.org/10.1016/j.cell.2020.10.036">https://doi.org/10.1016/j.cell.2020.10.036</a></p></section><section><h3>Copyright</h3><div class="copyright">© 2020 The Authors. Published by Elsevier Inc.</div></section><section class="userLicenseRpl">                                              <h3>User license</h3>                                               <a href="http://creativecommons.org/licenses/by-nc-nd/4.0/" class="userLicLnk" target="blank">Creative Commons Attribution – NonCommercial – NoDerivs (CC BY-NC-ND 4.0)</a> |                                                      <div class="dropBlock reference-citations">                                                     <a data-db-target-for="modal-user-license" href="#" class="reference-citations__ctrl" >How you can reuse</a>                                                     <img src="/templates/jsp/_style2/_marlin/images/information.png" alt="Information Icon" class="infoIcon"/>                                                     <div data-db-target-of="modal-user-license" class="dropBlock__holder reference-citations__holder">                                                      <div class="reference-citations__border"></div>                                             		 <div class="modal-header"><span class="userLicTitle">Creative Commons Attribution – NonCommercial – NoDerivs (CC BY-NC-ND 4.0)</span></div>                                                       <div class="modal-body"><p><h2>Permitted</h2>
<h3>For non-commercial purposes:</h3>
<ul>
<li>Read, print &amp; download</li>
<li>Redistribute or republish the final article</li>

<li>Text &amp; data mine</li>
<li>Translate the article (private use only, not for distribution)</li>
<li>Reuse portions or extracts from the article in other works</li></ul>
<h2>Not Permitted</h2><ul>

<li>Sell or re-use for commercial purposes</li>



<li>Distribute translations or adaptations of the article</li></ul>
<br><a href="https://www.elsevier.com/about/open-science/open-access" target= "blank">Elsevier's open access license policy</a></p></div>                                                       </div>                                                      </div>                                              </section><section><h3>ScienceDirect</h3><a href="https://www.sciencedirect.com/user/institution/login?targetUrl=/science/article/pii/S0092867420314008">Access this article on ScienceDirect</a></section></section><div id="figure-viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title"></div></nav><div class="frames"><ul class="frames__list"></ul><a href="#" title="Previous figure" class="frameButton previousFrameButton frames__prev"><i class="icon-arrow_l"></i></a><a href="#" title="Next figure" class="frameButton nextFrameButton frames__next"><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in article</span></a></div><div class="thumbstrip"><a class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li><a target="_blank" href="#" title="View Large Image in a new tab" class="viewLargeImage"><i class="icon icon-gizmo-image_figure figure-viewer__download__viewer-size-icon"></i><span class="figure-viewer__download__text">View Large Image</span></a></li><li><a target="_blank" href="#" title="View high resolution image in a new tab" class="downloadThisButton" download=""><i class="icon icon-gizmo-image_figure figure-viewer__download__viewer-size-icon"></i><span class="figure-viewer__download__text">Download Hi-res image </span></a></li><li class="hidden-xs hidden-sm"><a href="#" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><div class="figures-placeholder"></div><div class="tables-placeholder"></div>
    

        


    

        


    

        


    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        


    

        


    

        


    

            

            

            <section><h2 id="relatedArticles" class="top">Related Articles</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="f03b945a-ad7e-4ca2-9d54-9b913bce3420" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867420314008" class="ux3DeferredOnScrollWidget"></div></section>

            

        
<div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title"></div></nav><div class="frames"><ul class="frames__list"></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li><a target="_blank" href="/specs/products/marlin/components/figure-viewer/images/facets-2015-0010-f10-standard.jpg" title="View high resolution image in a new tab" class="downloadThisButton"><i class="icon icon-gizmo-image_figure figure-viewer__download__viewer-size-icon"></i><span class="figure-viewer__download__text">Download Hi-res image</span></a></li><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?articleNumber=e00489" target="_blank" title="Doanload .PPT" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><div data-toggle="transplant" data-direction="from" data-transplant="self" data-target=".adplaceholder--slot2, .adplaceholder--slot3 " data-remove="false" class="adplaceholder__transplant hidden-md hidden-lg"></div></div><div class="col-md-3 col-lg-2"></div></div></div></div></article>

            

        

        </main>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id='ad for nucleus-single-molecule-technologies'>
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(20)31400-8?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867420314008%3Fshowall%3Dtrue';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(20)31400-8?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867420314008%3Fshowall%3Dtrue';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async defer src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement"/>
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"/>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"/>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"/>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"/>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"/>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"/>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"/>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"/>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"/>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"/>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"/>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"/>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<div class="group__title">Life &amp; medical sciences journals</div>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		<li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<div class="group__title">Physical sciences &amp; engineering journals</div>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<div class="group__title">Multidisciplinary journals</div>
	<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<div class="group__title">Authors</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<div class="group__title">Reviewers</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<div class="group__title">News &amp; events</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
				<li>
					<a alt="Lablinks" href="/lablinks">LabLinks</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<div class="group__title">Multimedia</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<div class="group__title">About</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<div class="group__title">Contact</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<div class="group__title">Careers</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
				<li>
					<a alt="Scientific job board" href="https://careers.cell.com/" target="_blank">Scientific job board</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<div class="group__title">Access</div>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<div class="group__title">Collections</div>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<div class="group__title">Information</div>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr/>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr/>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie settings</button></b> for this site.<br />

All content on this site: Copyright © 2024 Elsevier Inc., its licensors, and contributors. <br/>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br/>For all open access content, the Creative Commons licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>

#brandingRow {
   background-color: #fff;
   padding: 7px 32px;
   color: #007DBC;
   font-size: 14px;
   clear: both;
   width: 100%
}

#brandingRow a {
   text-decoration: none;
   color: #007DBC;
}

#brandingRow a:hover {
   text-decoration: underline;
}

#brandingRowRight {
   display: inline;
   float: right;
   width: auto;
   padding: 0;
   margin: 0;
}

div#brandingRowInnerWrapper {
    max-width: 1440px;
    margin-left: auto;
    margin-right: auto;
    padding: 0 16px;
}

a#cpIconLnk {
   display: flex;
   margin-right: 12px;
}

@media screen and (max-width: 600px) {
    #brandingRow{
        font-size: 12px;
    }
}

@media screen and (max-width: 750px) {
    XXXsvg#cp_logo{
        margin-right: 10px;
    }
}

@media screen and (max-width: 800px) {
    a#cpIconLnk {
       display: none;
    }
}

.header__wrapper.clearfix .col-xs-6:first-of-type {
    width: 55%;
	padding-left: 32px;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
    width: 45%;
}
@media screen and (min-width: 800px) and (max-width: 1000px) {
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width: 75%;
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width: 25%;
    }
}

@media (max-width: 1139px) {
	.header__wrapper.clearfix .col-xs-6:first-of-type {
		padding-left: 16px;
	}
	
	#brandingRow {
		padding: 7px 16px 7px 16px;
	}
	
}

/* code for global header */

.header__wrapper {
   /* max-width: 1440px; */
   margin-left: auto;
   margin-right: auto;
}

@media (min-width: 769px) {
    .ux3 .quick-search .dropBlock [data-db-target-of], .quick-search .dropBlock [data-db-target-of] {
       width: 100vw !important;
       max-width: 1440px;
    }
} 

/* adjust for CP infinity icon */

svg#cp_logo {
    float: none;
    height: 32px;
    width: auto;
    margin-left: 0px;
    margin-top: 0px;
}

header .journal-logos {
    margin-right: 30px;
}

header .journal-logos:last-of-type {
    margin-right: 0px;
}

@media (min-width: 768px) {
    header .journal-logos {
       margin-left: 0 !important;
    }

/* adjust for logos with descender characters */
header .journal-logos.descender {
    margin-top: 8px;
}
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "ynexus" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");

	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
	$("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press'><svg id='cp_logo' width='324' height='204' viewBox='0 0 324 204' fill='none' xmlns='http://www.w3.org/2000/svg'><path d='M242.8 59.3C235.9 60.3 230.4 65.3 228.7 72.1L223.2 95C222.5 98.1 219.8 100.4 216.7 100.7L197.9 102.9C187.5 104 179.4 113 180.2 123.4C181 134.2 190.6 142 201.2 140.9C208.1 139.9 213.6 134.9 215.3 128.1L220.8 105.2C221.5 102.1 224.2 99.8 227.3 99.5L246.1 97.3C256.5 96.2 264.6 87.2 263.8 76.8C263 66 253.4 58.1 242.8 59.3Z' fill='#007DBC'/><path d='M221.7 0C165.5 0 120 45.5 120 101.7C120 157.9 165.5 203.4 221.7 203.4C277.9 203.4 323.4 157.9 323.4 101.7C323.4 45.5 277.8 0 221.7 0ZM251 135L221.3 138.4C216.1 139 211.8 142.8 210.6 147.9L205.6 168.8C204.1 175.1 198 179.2 191.6 178.2C167 174.5 146.5 155.6 142.8 130.1C138.2 97.9 161.1 68.6 193.2 65.1L222.9 61.7C228.1 61.1 232.4 57.3 233.6 52.2L238.6 31.3C240.1 25 246.2 20.9 252.6 21.9C277.2 25.6 297.7 44.5 301.4 70C306 102.2 283 131.5 251 135Z' fill='#007DBC'/><path d='M54.9 33.8L46.9 79L47.5 79.6C53 74.1 58.9 70.2 65.1 67.8C71.4 65.4 78.4 64.2 86.4 64.2C96.2 64.2 105 66 112.5 69.6C120.1 73.2 126.5 78 131.7 84.1C136.9 90.2 140.9 97.3 143.6 105.5C146.3 113.7 147.7 122.5 147.7 131.8C147.7 142 145.7 151.5 141.7 160.3C137.7 169.1 132.4 176.7 125.6 183.2C118.9 189.6 111.1 194.6 102.2 198.1C93.3 201.6 83.8 203.3 73.8 203.1C64.2 203.1 54.9 201.8 46.1 199.3C37.3 196.7 29.5 192.8 22.7 187.5C15.9 182.2 10.4 175.6 6.4 167.8C2.3 159.9 0.2 150.8 0 140.4H40.3C41.2 149.5 44.7 156.7 50.5 162.1C56.4 167.5 63.8 170.2 72.9 170.2C78.2 170.2 83 169.1 87.2 166.9C91.5 164.7 95.1 161.9 98 158.4C100.9 154.9 103.2 150.8 104.8 146.2C106.4 141.6 107.2 136.8 107.2 131.8C107.2 126.7 106.4 121.9 104.9 117.3C103.4 112.7 101.1 108.8 98.1 105.4C95.1 102 91.5 99.3 87.3 97.4C83.1 95.5 78.3 94.6 72.8 94.6C65.6 94.6 59.7 95.9 55.2 98.4C50.6 100.9 46.4 105 42.4 110.5H6.4L26 0.5H137V33.7H54.9V33.8Z' fill='#007DBC'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");

}

</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-9fa7ff1ab4c041fb83f6.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1723231651114","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"ae:Journal Branded Solutions - Guests|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:375335|ae:228598","accountName":"ae:Journal Branded Solutions - Guests|ae:ScienceDirect Guests","departmentId":"ae:388406|ae:291352","ipAddress":"73.149.180.196","loginStatus":"anonymous","userId":"ae:19319093|ae:12975512"},"content":[{"accessType":"article:free:openaccess","volumeNumber":"183","issueNumber":"5","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867420314008","coverDate":"202011","type":"XOCS-JOURNAL:SCOPE-FULL","title":"Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
</body>
</html>
