{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Note: include in the tests folder a .env file that contains the API keys for the services used in the tests    \n",
    "import os\n",
    "\n",
    "import pandas as pd  # type: ignore\n",
    "\n",
    "from palimpzest.constants import Cardinality\n",
    "from palimpzest.core.lib.fields import Field\n",
    "from palimpzest.core.lib.schemas import Schema, Table, XLSFile\n",
    "from palimpzest.datamanager.datamanager import DataDirectory\n",
    "from palimpzest.policy import MaxQuality, MinCost\n",
    "from palimpzest.query.processor.config import QueryProcessorConfig\n",
    "from palimpzest.sets import Dataset\n",
    "\n",
    "if not os.environ.get('OPENAI_API_KEY'):\n",
    "    from palimpzest.utils import load_env\n",
    "    load_env()\n",
    "\n",
    "DataDirectory().clear_cache(keep_registry=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Schema definition\n",
    "In the following cell we target schema of the case study we are interested in. Note how we are not specifying target attributes with fine grained metadata but rather with natural language, possibly ambiguous, specifications."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "class CaseData(Schema):\n",
    "    \"\"\"An individual row extracted from a table containing medical study data.\"\"\"\n",
    "    case_submitter_id = Field(desc=\"The ID of the case\")\n",
    "    age_at_diagnosis = Field(desc=\"The age of the patient at the time of diagnosis\")\n",
    "    race = Field(desc=\"An arbitrary classification of a taxonomic group that is a division of a species.\")\n",
    "    ethnicity = Field(desc=\"Whether an individual describes themselves as Hispanic or Latino or not.\")\n",
    "    gender = Field(desc=\"Text designations that identify gender.\")\n",
    "    vital_status = Field(desc=\"The vital status of the patient\")\n",
    "    ajcc_pathologic_t = Field(desc=\"Code of pathological T (primary tumor) to define the size or contiguous extension of the primary tumor (T), using staging criteria from the American Joint Committee on Cancer (AJCC).\")\n",
    "    ajcc_pathologic_n = Field(desc=\"The codes that represent the stage of cancer based on the nodes present (N stage) according to criteria based on multiple editions of the AJCC's Cancer Staging Manual.\")\n",
    "    ajcc_pathologic_stage = Field(desc=\"The extent of a cancer, especially whether the disease has spread from the original site to other parts of the body based on AJCC staging criteria.\")\n",
    "    tumor_grade = Field(desc=\"Numeric value to express the degree of abnormality of cancer cells, a measure of differentiation and aggressiveness.\")\n",
    "    tumor_focality = Field(desc=\"The text term used to describe whether the patient's disease originated in a single location or multiple locations.\")\n",
    "    tumor_largest_dimension_diameter = Field(desc=\"The tumor largest dimension diameter.\")\n",
    "    primary_diagnosis = Field(desc=\"Text term used to describe the patient's histologic diagnosis, as described by the World Health Organization's (WHO) International Classification of Diseases for Oncology (ICD-O).\")\n",
    "    morphology = Field(desc=\"The Morphological code of the tumor, as described by the World Health Organization's (WHO) International Classification of Diseases for Oncology (ICD-O).\")\n",
    "    tissue_or_organ_of_origin = Field(desc=\"The text term used to describe the anatomic site of origin, of the patient's malignant disease, as described by the World Health Organization's (WHO) International Classification of Diseases for Oncology (ICD-O).\")\n",
    "    # tumor_code = Field(desc=\"The tumor code\")\n",
    "    study = Field(desc=\"The last name of the author of the study, from the table name\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Printing base dataset\n",
    "In the following cell we print the base dataset we are interested in. Note how we are not specifying target attributes with fine grained metadata but rather with natural language, possibly ambiguous, specifications."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Available models:  [GPT_3_5, GPT_4]\n",
      "LOGICAL PLANS: 1\n",
      "INITIAL PLANS: 1\n",
      "DEDUP PLANS: 1\n",
      "PARETO PLANS: 1\n",
      "Table name: dou_mmc1.xlsx_UCEC_CPTAC3_meta_table_V2.1\n",
      "idx | Proteomics_Participant_ID | Case_excluded | Proteomics_TMT_batch | Proteomics_TMT_plex | Prote ...\n",
      "S001 | C3L-00006 | No | 2 | 5 | 128N | C3L-00006-01 | CPT0001460012 | Tumor | No | United States | F ...\n",
      "S002 | C3L-00008 | No | 4 | 16 | 130N | C3L-00008-01 | CPT0001300009 | Tumor | No | United States |  ...\n",
      "S003 | C3L-00032 | No | 1 | 2 | 131 | C3L-00032-01 | CPT0001420009 | Tumor | No | United States | FI ...\n",
      "\n",
      "Table name: vasaikar_mmc1.xlsx_Description\n",
      "Sheet | Description ...\n",
      "A-Annotation | Column annotations ...\n",
      "B-ClinicalData | Clinical Information for 110 Colon cancer patients ...\n",
      "\n",
      "Table name: vasaikar_mmc1.xlsx_A-Annotation\n",
      "Column name | Column definition | Unnamed: 2 | Unnamed: 3 | Data type | Definition ...\n",
      "SampleID | ID of the prospective participant | nan | nan | Bin | Binary ...\n",
      "Clinical | Clinical data availability | nan | nan | Cat | Categorical ...\n",
      "Mutation | Mutation data availability | nan | nan | Con | Continuous ...\n",
      "\n",
      "Table name: vasaikar_mmc1.xlsx_B-ClinicalData\n",
      "SampleID | Clinical | Mutation | CNV | miRNA-seq | mRNA-seq | Label-free Proteome | TMT Proteome | T ...\n",
      "Type | BIN | BIN | BIN | BIN | BIN | BIN | BIN | BIN | BIN | BIN | CON | BIN | BIN | CAT | ORD | ORD ...\n",
      "01CO001 | 1 | 1 | 1 | 1 | 1 | 1 | nan | 1 | nan | 1 | 729 | Male | Mucinous | Sigmoid Colon | T4a |  ...\n",
      "01CO005 | 1 | 1 | 1 | 1 | 1 | 1 | 1 | 1 | 1 | 1 | 838 | Female | Not Mucinous | Sigmoid Colon | T3 | ...\n",
      "\n",
      "Table name: wang_mmc2.xlsx_README\n",
      "README for Table S1 | Unnamed: 1 | Unnamed: 2 ...\n",
      "nan | nan | nan ...\n",
      "Tab1 | clinical_data | Clinical data of the CPTAC GBM discovery cohort (99 tumors and 10 GTEx normal ...\n",
      "Tab2 | additional_annotations | Additional subtype and annotation for each case (see STAR Methods) ...\n",
      "\n",
      "Table name: wang_mmc2.xlsx_clinical_data\n",
      "case_id | age | gender | height | weight | bmi | country_of_origin | race | ethnicity | ethnicity_se ...\n",
      "C3L-00104 | 58 | Male | 188.0 | 115.0 | 32.54 | United States | White | Not-Hispanic or Latino | Cau ...\n",
      "C3L-00365 | 59 | Female | 162.0 | 54.0 | 20.61 | United States | White | Not-Hispanic or Latino | Ca ...\n",
      "C3L-00674 | 45 | Male | 193.0 | 102.0 | 27.44 | nan | White | Not-Hispanic or Latino | Caucasian | n ...\n",
      "\n",
      "Table name: wang_mmc2.xlsx_additional_annotations\n",
      "case | sample_type | multiomic | nmf_consensus | nmf_cluster_membership | rna_wang_cancer_cell_2017  ...\n",
      "C3L-00104 | tumor | nmf1 | nmf1 | 0.743 | Proneural | 0.678244020987608 | dm2 | 0.314080871350208 |  ...\n",
      "C3L-00365 | tumor | nmf3 | nmf3 | 0.614 | Classical | 0.681122006453892 | dm4 | 0.891148938751837 |  ...\n",
      "C3L-00674 | tumor | nmf1 | nmf1 | 0.507 | Mesenchymal | 0.744635304438083 | dm5 | 0.625633701322078  ...\n",
      "\n",
      "Table name: wang_mmc2.xlsx_cbttc_sample_info\n",
      "aliquot_id | sample_id | original_aliquot_id | batch | plex | tmt_channel | research_id | kids_first ...\n",
      "7316-288-TISS [894482] | 7316-288 | 7316-288-TISS [894482] | 1 | 1 | 127N | C54735 | PT_84Q11Z5X | A ...\n",
      "7316-89-TISS [894435] | 7316-89 | 7316-89-TISS [894435] | 1 | 1 | 128N | C25461 | PT_N32BYY8A | Aliv ...\n",
      "7316-2176-TISS [894450] | 7316-2176 | 7316-2176-TISS [894450] | 1 | 1 | 129N | C624225 | PT_VAJN5QP8 ...\n",
      "\n",
      "Table name: wang_mmc2.xlsx_survival_analysis\n",
      "Stratification group | Group value | n | events | median | 0.95LCL | 0.95UCL | log-rank p-value ...\n",
      "DNA methylation subtype\n",
      "(dna_methyl) | dm1 | 15 | 12 | 1.04 | 0.805 | nan | 0.55 ...\n",
      "nan | dm2 | 18 | 10 | 1.1 | 0.797 | nan | nan ...\n",
      "nan | dm3 | 21 | 14 | 1.07 | 0.931 | nan | nan ...\n",
      "\n"
     ]
    }
   ],
   "source": [
    "def print_tables(output):\n",
    "    for table in output:\n",
    "        header = table.header\n",
    "        subset_rows = table.rows[:3]\n",
    "\n",
    "        print(\"Table name:\", table.name)\n",
    "        print(\" | \".join(header)[:100], \"...\")\n",
    "        for row in subset_rows:\n",
    "            print(\" | \".join(row)[:100], \"...\")\n",
    "        print()\n",
    "\n",
    "\n",
    "xls = Dataset('biofabric-tiny', schema=XLSFile)\n",
    "patient_tables = xls.convert(Table, desc=\"All tables in the file\", cardinality=Cardinality.ONE_TO_MANY)\n",
    "output = patient_tables\n",
    "\n",
    "policy = MinCost()\n",
    "config = QueryProcessorConfig(\n",
    "    policy=policy,\n",
    "    nocache=True,\n",
    "    processing_strategy=\"no_sentinel\",\n",
    ")\n",
    "data_record_collection = output.run(config)\n",
    "\n",
    "print_tables(data_record_collection.data_records)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Filtering stage\n",
    "In the following cell we define the first part of the workload, that comprises a filtering stage responsible for selecting the tables from all the spreadsheets that contain relevant biometric information about the patient.\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Available models:  [GPT_3_5, GPT_4]\n",
      "LOGICAL PLANS: 1\n",
      "INITIAL PLANS: 2\n",
      "DEDUP PLANS: 2\n",
      "PARETO PLANS: 1\n",
      "Table name: dou_mmc1.xlsx_UCEC_CPTAC3_meta_table_V2.1\n",
      "idx | Proteomics_Participant_ID | Case_excluded | Proteomics_TMT_batch | Proteomics_TMT_plex | Prote ...\n",
      "S001 | C3L-00006 | No | 2 | 5 | 128N | C3L-00006-01 | CPT0001460012 | Tumor | No | United States | F ...\n",
      "S002 | C3L-00008 | No | 4 | 16 | 130N | C3L-00008-01 | CPT0001300009 | Tumor | No | United States |  ...\n",
      "S003 | C3L-00032 | No | 1 | 2 | 131 | C3L-00032-01 | CPT0001420009 | Tumor | No | United States | FI ...\n",
      "\n",
      "Table name: wang_mmc2.xlsx_clinical_data\n",
      "case_id | age | gender | height | weight | bmi | country_of_origin | race | ethnicity | ethnicity_se ...\n",
      "C3L-00104 | 58 | Male | 188.0 | 115.0 | 32.54 | United States | White | Not-Hispanic or Latino | Cau ...\n",
      "C3L-00365 | 59 | Female | 162.0 | 54.0 | 20.61 | United States | White | Not-Hispanic or Latino | Ca ...\n",
      "C3L-00674 | 45 | Male | 193.0 | 102.0 | 27.44 | nan | White | Not-Hispanic or Latino | Caucasian | n ...\n",
      "\n",
      "Total_plan_time=9.442001581192017 \n",
      "Total_plan_cost=0.0021945 \n",
      "0. MarshalAndScanDataOp time=0.04193472862243652 cost=0.0 \n",
      "1. ConvertXLSToTable time=0.41283321380615234 cost=0.0 \n",
      "2. LLMFilter time=8.986163854598999 cost=0.0021945 \n",
      "\n"
     ]
    }
   ],
   "source": [
    "\n",
    "# Make sure to run\n",
    "# pz reg --name biofabric-tiny --path testdata/biofabric-tiny\n",
    "DataDirectory().clear_cache(keep_registry=True)\n",
    "\n",
    "xls = Dataset('biofabric-tiny', schema=XLSFile)\n",
    "patient_tables = xls.convert(Table, desc=\"All tables in the file\", cardinality=Cardinality.ONE_TO_MANY)\n",
    "patient_tables = patient_tables.filter(\"The table contains biometric information about the patient\")\n",
    "\n",
    "output = patient_tables\n",
    "\n",
    "policy = MinCost()\n",
    "config = QueryProcessorConfig(\n",
    "    policy=policy,\n",
    "    nocache=True,\n",
    "    processing_strategy=\"no_sentinel\",\n",
    ")\n",
    "data_record_collection = patient_tables.run(config)\n",
    "\n",
    "for table in data_record_collection:\n",
    "    header = table.header\n",
    "    subset_rows = table.rows[:3]\n",
    "\n",
    "    print(\"Table name:\", table.name)\n",
    "    print(\" | \".join(header)[:100], \"...\")\n",
    "    for row in subset_rows:\n",
    "        print(\" | \".join(row)[:100], \"...\")\n",
    "    print()\n",
    "\n",
    "print(data_record_collection.execution_stats)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Available models:  [GPT_3_5, GPT_4]\n",
      "LOGICAL PLANS: 1\n",
      "INITIAL PLANS: 2\n",
      "DEDUP PLANS: 2\n",
      "PARETO PLANS: 1\n",
      "Table name: dou_mmc1.xlsx_UCEC_CPTAC3_meta_table_V2.1\n",
      "idx | Proteomics_Participant_ID | Case_excluded | Proteomics_TMT_batch | Proteomics_TMT_plex | Prote ...\n",
      "S001 | C3L-00006 | No | 2 | 5 | 128N | C3L-00006-01 | CPT0001460012 | Tumor | No | United States | F ...\n",
      "S002 | C3L-00008 | No | 4 | 16 | 130N | C3L-00008-01 | CPT0001300009 | Tumor | No | United States |  ...\n",
      "S003 | C3L-00032 | No | 1 | 2 | 131 | C3L-00032-01 | CPT0001420009 | Tumor | No | United States | FI ...\n",
      "\n",
      "Table name: wang_mmc2.xlsx_clinical_data\n",
      "case_id | age | gender | height | weight | bmi | country_of_origin | race | ethnicity | ethnicity_se ...\n",
      "C3L-00104 | 58 | Male | 188.0 | 115.0 | 32.54 | United States | White | Not-Hispanic or Latino | Cau ...\n",
      "C3L-00365 | 59 | Female | 162.0 | 54.0 | 20.61 | United States | White | Not-Hispanic or Latino | Ca ...\n",
      "C3L-00674 | 45 | Male | 193.0 | 102.0 | 27.44 | nan | White | Not-Hispanic or Latino | Caucasian | n ...\n",
      "\n",
      "Total_plan_time=0.5001308917999268 \n",
      "Total_plan_cost=0.0021945 \n",
      "0. MarshalAndScanDataOp time=0.052803993225097656 cost=0.0 \n",
      "1. ConvertXLSToTable time=0.43227100372314453 cost=0.0 \n",
      "2. LLMFilter time=0.012064456939697266 cost=0.0021945 \n",
      "\n"
     ]
    }
   ],
   "source": [
    "policy = MinCost()\n",
    "config = QueryProcessorConfig(\n",
    "    policy=policy,\n",
    "    nocache=True,\n",
    "    processing_strategy=\"no_sentinel\",\n",
    "    execution_strategy=\"pipelined_parallel\",\n",
    ")\n",
    "data_record_collection = patient_tables.run(config)\n",
    "\n",
    "for table in data_record_collection:\n",
    "    header = table.header\n",
    "    subset_rows = table.rows[:3]\n",
    "\n",
    "    print(\"Table name:\", table.name)\n",
    "    print(\" | \".join(header)[:100], \"...\")\n",
    "    for row in subset_rows:\n",
    "        print(\" | \".join(row)[:100], \"...\")\n",
    "    print()\n",
    "\n",
    "print(data_record_collection.execution_stats)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Chosen plan:\n",
      "PhysicalPlan:\n",
      "0. dir -> MarshalAndScanDataOp -> XLSFile(contents, filename, number_sheets, sheet_names)\n",
      "(contents,filename,number_sheet) \n",
      "\n",
      "1. XLSFile -> ConvertXLSToTable -> Table\n",
      "(contents,filename,number_sheet) -> (filename,header,name,rows) \n",
      " \n",
      "\n",
      "Stats: Total_plan_time=0.5795912742614746 \n",
      "Total_plan_cost=0.0 \n",
      "0. MarshalAndScanDataOp time=0.06175112724304199 cost=0.0 \n",
      "1. ConvertXLSToTable time=0.516960620880127 cost=0.0 \n",
      "\n"
     ]
    }
   ],
   "source": [
    "print(\"Chosen plan:\")\n",
    "print(data_record_collection.executed_plans, \"\\n\")\n",
    "print(\"Stats:\", data_record_collection.execution_stats)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Matching stage\n",
    "Once filtered, we can define the second part of the workload, that matches the patient records in the different tables and merging the information into a single table."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Minimum cost: GPT 3.5"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Available models:  [GPT_3_5, GPT_4]\n",
      "LOGICAL PLANS: 1\n",
      "INITIAL PLANS: 14\n",
      "DEDUP PLANS: 2\n",
      "PARETO PLANS: 1\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>rows</th>\n",
       "      <th>filename</th>\n",
       "      <th>header</th>\n",
       "      <th>name</th>\n",
       "      <th>age_at_diagnosis</th>\n",
       "      <th>ajcc_pathologic_n</th>\n",
       "      <th>ajcc_pathologic_stage</th>\n",
       "      <th>ajcc_pathologic_t</th>\n",
       "      <th>case_submitter_id</th>\n",
       "      <th>ethnicity</th>\n",
       "      <th>gender</th>\n",
       "      <th>morphology</th>\n",
       "      <th>primary_diagnosis</th>\n",
       "      <th>race</th>\n",
       "      <th>study</th>\n",
       "      <th>tissue_or_organ_of_origin</th>\n",
       "      <th>tumor_focality</th>\n",
       "      <th>tumor_grade</th>\n",
       "      <th>tumor_largest_dimension_diameter</th>\n",
       "      <th>vital_status</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>[[A-Annotation, Column annotations], [B-Clinic...</td>\n",
       "      <td>vasaikar_mmc1.xlsx</td>\n",
       "      <td>[Sheet, Description]</td>\n",
       "      <td>vasaikar_mmc1.xlsx_Description</td>\n",
       "      <td>Value based on data in the rows</td>\n",
       "      <td>The AJCC pathologic N</td>\n",
       "      <td>The AJCC pathologic stage</td>\n",
       "      <td>The AJCC pathologic T</td>\n",
       "      <td>ID123</td>\n",
       "      <td>Whether an individual describes themselves as ...</td>\n",
       "      <td>Text designations that identify gender</td>\n",
       "      <td>The morphology</td>\n",
       "      <td></td>\n",
       "      <td>An arbitrary classification of a taxonomic gro...</td>\n",
       "      <td>vasaikar</td>\n",
       "      <td>NA</td>\n",
       "      <td></td>\n",
       "      <td>The tumor grade</td>\n",
       "      <td>The tumor largest dimension diameter</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>[[Type, BIN, BIN, BIN, BIN, BIN, BIN, BIN, BIN...</td>\n",
       "      <td>vasaikar_mmc1.xlsx</td>\n",
       "      <td>[SampleID, Clinical, Mutation, CNV, miRNA-seq,...</td>\n",
       "      <td>vasaikar_mmc1.xlsx_B-ClinicalData</td>\n",
       "      <td>Age</td>\n",
       "      <td>PN</td>\n",
       "      <td>Stage</td>\n",
       "      <td>PT</td>\n",
       "      <td>SampleID</td>\n",
       "      <td>Hispanic or Latino</td>\n",
       "      <td>Gender</td>\n",
       "      <td></td>\n",
       "      <td>B-ClinicalData</td>\n",
       "      <td>An arbitrary classification of a taxonomic gro...</td>\n",
       "      <td>vasaikar</td>\n",
       "      <td>Colon</td>\n",
       "      <td>TMT Proteome</td>\n",
       "      <td>Grade 2</td>\n",
       "      <td>TumorPurity</td>\n",
       "      <td>Vital.Status</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>[[S001, C3L-00006, No, 2, 5, 128N, C3L-00006-0...</td>\n",
       "      <td>dou_mmc1.xlsx</td>\n",
       "      <td>[idx, Proteomics_Participant_ID, Case_excluded...</td>\n",
       "      <td>dou_mmc1.xlsx_UCEC_CPTAC3_meta_table_V2.1</td>\n",
       "      <td>Age of the patient at the time of diagnosis</td>\n",
       "      <td>The AJCC pathologic N</td>\n",
       "      <td>The AJCC pathologic stage</td>\n",
       "      <td>Path_Stage_Primary_Tumor-pT</td>\n",
       "      <td>Proteomics_Participant_ID</td>\n",
       "      <td>Whether an individual describes themselves as ...</td>\n",
       "      <td>Text designations that identify gender</td>\n",
       "      <td>The morphology field value extracted from the ...</td>\n",
       "      <td>Endometrial Carcinoma</td>\n",
       "      <td>An arbitrary classification of a taxonomic gro...</td>\n",
       "      <td>CPTAC3</td>\n",
       "      <td>Country</td>\n",
       "      <td>Tumor focality information extracted from the ...</td>\n",
       "      <td>The tumor grade</td>\n",
       "      <td>tumor_largest_dimension_diameter</td>\n",
       "      <td>The vital status of the patient</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>[[C3L-00104, 58, Male, 188.0, 115.0, 32.54, Un...</td>\n",
       "      <td>wang_mmc2.xlsx</td>\n",
       "      <td>[case_id, age, gender, height, weight, bmi, co...</td>\n",
       "      <td>wang_mmc2.xlsx_clinical_data</td>\n",
       "      <td>age</td>\n",
       "      <td>AJCC pathologic N</td>\n",
       "      <td>AJCC pathologic stage value</td>\n",
       "      <td>AJCC pathologic T</td>\n",
       "      <td>case_id</td>\n",
       "      <td>ethnicity_self_identify</td>\n",
       "      <td>Text designations that identify gender</td>\n",
       "      <td>Additional information or clarification needed</td>\n",
       "      <td>The primary diagnosis</td>\n",
       "      <td>An arbitrary classification of a taxonomic gro...</td>\n",
       "      <td>wang</td>\n",
       "      <td>NA</td>\n",
       "      <td>tumor_focality</td>\n",
       "      <td>The tumor grade</td>\n",
       "      <td>tumor_largest_dimension_diameter</td>\n",
       "      <td>The vital status of the patient</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                rows            filename  \\\n",
       "0  [[A-Annotation, Column annotations], [B-Clinic...  vasaikar_mmc1.xlsx   \n",
       "1  [[Type, BIN, BIN, BIN, BIN, BIN, BIN, BIN, BIN...  vasaikar_mmc1.xlsx   \n",
       "2  [[S001, C3L-00006, No, 2, 5, 128N, C3L-00006-0...       dou_mmc1.xlsx   \n",
       "3  [[C3L-00104, 58, Male, 188.0, 115.0, 32.54, Un...      wang_mmc2.xlsx   \n",
       "\n",
       "                                              header  \\\n",
       "0                               [Sheet, Description]   \n",
       "1  [SampleID, Clinical, Mutation, CNV, miRNA-seq,...   \n",
       "2  [idx, Proteomics_Participant_ID, Case_excluded...   \n",
       "3  [case_id, age, gender, height, weight, bmi, co...   \n",
       "\n",
       "                                        name  \\\n",
       "0             vasaikar_mmc1.xlsx_Description   \n",
       "1          vasaikar_mmc1.xlsx_B-ClinicalData   \n",
       "2  dou_mmc1.xlsx_UCEC_CPTAC3_meta_table_V2.1   \n",
       "3               wang_mmc2.xlsx_clinical_data   \n",
       "\n",
       "                              age_at_diagnosis      ajcc_pathologic_n  \\\n",
       "0              Value based on data in the rows  The AJCC pathologic N   \n",
       "1                                          Age                     PN   \n",
       "2  Age of the patient at the time of diagnosis  The AJCC pathologic N   \n",
       "3                                          age      AJCC pathologic N   \n",
       "\n",
       "         ajcc_pathologic_stage            ajcc_pathologic_t  \\\n",
       "0    The AJCC pathologic stage        The AJCC pathologic T   \n",
       "1                        Stage                           PT   \n",
       "2    The AJCC pathologic stage  Path_Stage_Primary_Tumor-pT   \n",
       "3  AJCC pathologic stage value            AJCC pathologic T   \n",
       "\n",
       "           case_submitter_id  \\\n",
       "0                      ID123   \n",
       "1                   SampleID   \n",
       "2  Proteomics_Participant_ID   \n",
       "3                    case_id   \n",
       "\n",
       "                                           ethnicity  \\\n",
       "0  Whether an individual describes themselves as ...   \n",
       "1                                 Hispanic or Latino   \n",
       "2  Whether an individual describes themselves as ...   \n",
       "3                            ethnicity_self_identify   \n",
       "\n",
       "                                   gender  \\\n",
       "0  Text designations that identify gender   \n",
       "1                                  Gender   \n",
       "2  Text designations that identify gender   \n",
       "3  Text designations that identify gender   \n",
       "\n",
       "                                          morphology      primary_diagnosis  \\\n",
       "0                                     The morphology                          \n",
       "1                                                            B-ClinicalData   \n",
       "2  The morphology field value extracted from the ...  Endometrial Carcinoma   \n",
       "3     Additional information or clarification needed  The primary diagnosis   \n",
       "\n",
       "                                                race     study  \\\n",
       "0  An arbitrary classification of a taxonomic gro...  vasaikar   \n",
       "1  An arbitrary classification of a taxonomic gro...  vasaikar   \n",
       "2  An arbitrary classification of a taxonomic gro...    CPTAC3   \n",
       "3  An arbitrary classification of a taxonomic gro...      wang   \n",
       "\n",
       "  tissue_or_organ_of_origin  \\\n",
       "0                        NA   \n",
       "1                     Colon   \n",
       "2                   Country   \n",
       "3                        NA   \n",
       "\n",
       "                                      tumor_focality      tumor_grade  \\\n",
       "0                                                     The tumor grade   \n",
       "1                                       TMT Proteome          Grade 2   \n",
       "2  Tumor focality information extracted from the ...  The tumor grade   \n",
       "3                                     tumor_focality  The tumor grade   \n",
       "\n",
       "       tumor_largest_dimension_diameter                     vital_status  \n",
       "0  The tumor largest dimension diameter                                   \n",
       "1                           TumorPurity                     Vital.Status  \n",
       "2      tumor_largest_dimension_diameter  The vital status of the patient  \n",
       "3      tumor_largest_dimension_diameter  The vital status of the patient  "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# Make sure to run\n",
    "# pz reg --name biofabric-tiny-filtered --path testdata/biofabric-tiny-filtered\n",
    "DataDirectory().clear_cache(keep_registry=True)\n",
    "\n",
    "input_dataset = Dataset('biofabric-tiny-filtered', schema=XLSFile)\n",
    "patient_tables = input_dataset.convert(Table, desc=\"All tables in the file\", cardinality=Cardinality.ONE_TO_MANY)\n",
    "case_data = patient_tables.convert(CaseData, desc=\"The patient data in the table\", cardinality=Cardinality.ONE_TO_MANY)\n",
    "\n",
    "policy = MinCost()\n",
    "config = QueryProcessorConfig(\n",
    "    policy=policy,\n",
    "    nocache=True,\n",
    "    allow_code_synth=False,\n",
    "    processing_strategy=\"no_sentinel\",\n",
    "    execution_strategy=\"pipelined_parallel\",\n",
    ")\n",
    "data_record_collection = case_data.run(config) \n",
    "\n",
    "output_rows = []\n",
    "for output_table in data_record_collection:\n",
    "    output_rows.append(output_table.to_dict()) \n",
    "\n",
    "output_df = pd.DataFrame(output_rows)\n",
    "display(output_df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PhysicalPlan:\n",
      "0. dir -> MarshalAndScanDataOp -> XLSFile(contents, filename, number_sheets, sheet_names)\n",
      "(contents,filename,number_sheet) \n",
      "\n",
      "1. XLSFile -> ConvertXLSToTable -> Table\n",
      "(contents,filename,number_sheet) -> (filename,header,name,rows) \n",
      "\n",
      "2. Table -> LLMConvertConventional -> CaseData\n",
      "(filename,header,name,rows) -> (age_at_diagnosis,ajcc_patholog) \n",
      "Using model: gpt-3.5-turbo-0125\n",
      "Prompt strategy: dspy-chain-of-thought-question\n",
      "Query strategy: bonded-with-fallback\n",
      " \n",
      "\n",
      "Stats: Total_plan_time=84.75389361381531 \n",
      "Total_plan_cost=0.028860499999999994 \n",
      "0. MarshalAndScanDataOp time=0.04571676254272461 cost=0.0 \n",
      "1. ConvertXLSToTable time=0.46941590309143066 cost=0.0 \n",
      "2. LLMConvertConventional time=84.23336219787598 cost=0.028860499999999994 \n",
      "\n",
      "dict_keys(['plan_id', 'operator_stats', 'total_plan_time', 'total_plan_cost', 'plan_idx'])\n"
     ]
    }
   ],
   "source": [
    "print(data_record_collection.executed_plans, \"\\n\")\n",
    "print(\"Stats:\", data_record_collection.execution_stats)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Maximum Quality: GPT 4"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Available models:  [GPT_3_5, GPT_4]\n",
      "LOGICAL PLANS: 1\n",
      "INITIAL PLANS: 14\n",
      "DEDUP PLANS: 2\n",
      "PARETO PLANS: 1\n",
      "Error parsing answer: No output was found!\n",
      "Error parsing answer: No output was found!\n",
      "Error parsing answer: No output was found!\n",
      "Error parsing answer: No output was found!\n",
      "Error parsing answer: No output was found!\n",
      "Error parsing answer: No output was found!\n",
      "Error parsing answer: No output was found!\n",
      "Error parsing answer: No output was found!\n",
      "Error parsing answer: No output was found!\n",
      "Error parsing answer: No output was found!\n",
      "Error parsing answer: No output was found!\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>rows</th>\n",
       "      <th>filename</th>\n",
       "      <th>header</th>\n",
       "      <th>name</th>\n",
       "      <th>age_at_diagnosis</th>\n",
       "      <th>ajcc_pathologic_n</th>\n",
       "      <th>ajcc_pathologic_stage</th>\n",
       "      <th>ajcc_pathologic_t</th>\n",
       "      <th>case_submitter_id</th>\n",
       "      <th>ethnicity</th>\n",
       "      <th>gender</th>\n",
       "      <th>morphology</th>\n",
       "      <th>primary_diagnosis</th>\n",
       "      <th>race</th>\n",
       "      <th>study</th>\n",
       "      <th>tissue_or_organ_of_origin</th>\n",
       "      <th>tumor_focality</th>\n",
       "      <th>tumor_grade</th>\n",
       "      <th>tumor_largest_dimension_diameter</th>\n",
       "      <th>vital_status</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>[[S001, C3L-00006, No, 2, 5, 128N, C3L-00006-0...</td>\n",
       "      <td>dou_mmc1.xlsx</td>\n",
       "      <td>[idx, Proteomics_Participant_ID, Case_excluded...</td>\n",
       "      <td>dou_mmc1.xlsx_UCEC_CPTAC3_meta_table_V2.1</td>\n",
       "      <td>Age</td>\n",
       "      <td></td>\n",
       "      <td>[]</td>\n",
       "      <td>[]</td>\n",
       "      <td>Proteomics_Participant_ID</td>\n",
       "      <td>Description of whether an individual describes...</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td>dou</td>\n",
       "      <td></td>\n",
       "      <td>Tumor_Focality</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>[[A-Annotation, Column annotations], [B-Clinic...</td>\n",
       "      <td>vasaikar_mmc1.xlsx</td>\n",
       "      <td>[Sheet, Description]</td>\n",
       "      <td>vasaikar_mmc1.xlsx_Description</td>\n",
       "      <td>None</td>\n",
       "      <td></td>\n",
       "      <td>[]</td>\n",
       "      <td>[]</td>\n",
       "      <td></td>\n",
       "      <td>None</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td>None</td>\n",
       "      <td>vasaikar</td>\n",
       "      <td></td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>[]</td>\n",
       "      <td>[]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>[[Type, BIN, BIN, BIN, BIN, BIN, BIN, BIN, BIN...</td>\n",
       "      <td>vasaikar_mmc1.xlsx</td>\n",
       "      <td>[SampleID, Clinical, Mutation, CNV, miRNA-seq,...</td>\n",
       "      <td>vasaikar_mmc1.xlsx_B-ClinicalData</td>\n",
       "      <td>Derived from the 'Age' column in the table</td>\n",
       "      <td>None</td>\n",
       "      <td>Stage</td>\n",
       "      <td></td>\n",
       "      <td>SampleID</td>\n",
       "      <td>[]</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td>None</td>\n",
       "      <td>[]</td>\n",
       "      <td>vasaikar</td>\n",
       "      <td>None</td>\n",
       "      <td>[]</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>[[C3L-00104, 58, Male, 188.0, 115.0, 32.54, Un...</td>\n",
       "      <td>wang_mmc2.xlsx</td>\n",
       "      <td>[case_id, age, gender, height, weight, bmi, co...</td>\n",
       "      <td>wang_mmc2.xlsx_clinical_data</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>[]</td>\n",
       "      <td>[]</td>\n",
       "      <td>None</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>An arbitrary classification of a taxonomic gro...</td>\n",
       "      <td>wang</td>\n",
       "      <td></td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>unknown</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                rows            filename  \\\n",
       "0  [[S001, C3L-00006, No, 2, 5, 128N, C3L-00006-0...       dou_mmc1.xlsx   \n",
       "1  [[A-Annotation, Column annotations], [B-Clinic...  vasaikar_mmc1.xlsx   \n",
       "2  [[Type, BIN, BIN, BIN, BIN, BIN, BIN, BIN, BIN...  vasaikar_mmc1.xlsx   \n",
       "3  [[C3L-00104, 58, Male, 188.0, 115.0, 32.54, Un...      wang_mmc2.xlsx   \n",
       "\n",
       "                                              header  \\\n",
       "0  [idx, Proteomics_Participant_ID, Case_excluded...   \n",
       "1                               [Sheet, Description]   \n",
       "2  [SampleID, Clinical, Mutation, CNV, miRNA-seq,...   \n",
       "3  [case_id, age, gender, height, weight, bmi, co...   \n",
       "\n",
       "                                        name  \\\n",
       "0  dou_mmc1.xlsx_UCEC_CPTAC3_meta_table_V2.1   \n",
       "1             vasaikar_mmc1.xlsx_Description   \n",
       "2          vasaikar_mmc1.xlsx_B-ClinicalData   \n",
       "3               wang_mmc2.xlsx_clinical_data   \n",
       "\n",
       "                             age_at_diagnosis ajcc_pathologic_n  \\\n",
       "0                                         Age                     \n",
       "1                                        None                     \n",
       "2  Derived from the 'Age' column in the table              None   \n",
       "3                                        None              None   \n",
       "\n",
       "  ajcc_pathologic_stage ajcc_pathologic_t          case_submitter_id  \\\n",
       "0                    []                []  Proteomics_Participant_ID   \n",
       "1                    []                []                              \n",
       "2                 Stage                                     SampleID   \n",
       "3                    []                []                       None   \n",
       "\n",
       "                                           ethnicity gender morphology  \\\n",
       "0  Description of whether an individual describes...                     \n",
       "1                                               None                     \n",
       "2                                                 []                     \n",
       "3                                                                 None   \n",
       "\n",
       "  primary_diagnosis                                               race  \\\n",
       "0                                                                        \n",
       "1                                                                 None   \n",
       "2              None                                                 []   \n",
       "3              None  An arbitrary classification of a taxonomic gro...   \n",
       "\n",
       "      study tissue_or_organ_of_origin  tumor_focality tumor_grade  \\\n",
       "0       dou                            Tumor_Focality        None   \n",
       "1  vasaikar                                      None        None   \n",
       "2  vasaikar                      None              []        None   \n",
       "3      wang                                      None        None   \n",
       "\n",
       "  tumor_largest_dimension_diameter vital_status  \n",
       "0                             None         None  \n",
       "1                               []           []  \n",
       "2                             None               \n",
       "3                             None      unknown  "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "DataDirectory().clear_cache(keep_registry=True)\n",
    "\n",
    "input_dataset = Dataset('biofabric-tiny-filtered', schema=XLSFile)\n",
    "patient_tables = input_dataset.convert(Table, cardinality=Cardinality.ONE_TO_MANY)\n",
    "case_data = patient_tables.convert(CaseData, desc=\"The patient data in the table\", cardinality=Cardinality.ONE_TO_MANY)\n",
    "\n",
    "policy = MaxQuality()\n",
    "config = QueryProcessorConfig(\n",
    "    policy=policy,\n",
    "    nocache=True,\n",
    "    allow_code_synth=False,\n",
    "    processing_strategy=\"no_sentinel\",\n",
    "    execution_strategy=\"pipelined_parallel\",\n",
    ")\n",
    "data_record_collection = case_data.run(config)\n",
    "\n",
    "output_rows = []\n",
    "for output_table in data_record_collection:\n",
    "    output_rows.append(output_table.to_dict()) \n",
    "\n",
    "output_df = pd.DataFrame(output_rows)\n",
    "display(output_df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PhysicalPlan:\n",
      "0. dir -> MarshalAndScanDataOp -> XLSFile(contents, filename, number_sheets, sheet_names)\n",
      "(contents,filename,number_sheet) \n",
      "\n",
      "1. XLSFile -> ConvertXLSToTable -> Table\n",
      "(contents,filename,number_sheet) -> (filename,header,name,rows) \n",
      "\n",
      "2. Table -> LLMConvertConventional -> CaseData\n",
      "(filename,header,name,rows) -> (age_at_diagnosis,ajcc_patholog) \n",
      "Using model: gpt-3.5-turbo-0125\n",
      "Prompt strategy: dspy-chain-of-thought-question\n",
      "Query strategy: bonded-with-fallback\n",
      " \n",
      "\n",
      "Stats: Total_plan_time=381.9855651855469 \n",
      "Total_plan_cost=0.70077 \n",
      "0. MarshalAndScanDataOp time=0.13936471939086914 cost=0.0 \n",
      "1. ConvertXLSToTable time=0.8186140060424805 cost=0.0 \n",
      "2. LLMConvertConventional time=381.0207107067108 cost=0.70077 \n",
      "\n"
     ]
    }
   ],
   "source": [
    "print(data_record_collection.executed_plans, \"\\n\")\n",
    "print(\"Stats:\", data_record_collection.execution_stats)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## End to end"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Minimum cost : GPT 3.5"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Available models:  [GPT_3_5, GPT_4]\n",
      "LOGICAL PLANS: 2\n",
      "INITIAL PLANS: 6\n",
      "INITIAL PLANS: 4\n",
      "DEDUP PLANS: 8\n",
      "PARETO PLANS: 1\n",
      "Time for planning: 0.02268195152282715\n",
      "Iteration number:  1 Last record:  False\n",
      "dict_keys(['rows', 'filename', 'header', 'name', 'age_at_diagnosis', 'ajcc_pathologic_n', 'ajcc_pathologic_stage', 'ajcc_pathologic_t', 'case_submitter_id', 'ethnicity', 'gender', 'morphology', 'primary_diagnosis', 'race', 'study', 'tissue_or_organ_of_origin', 'tumor_focality', 'tumor_grade', 'tumor_largest_dimension_diameter', 'vital_status'])\n",
      "Iteration number:  2 Last record:  False\n",
      "Iteration number:  3 Last record:  True\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>rows</th>\n",
       "      <th>filename</th>\n",
       "      <th>header</th>\n",
       "      <th>name</th>\n",
       "      <th>age_at_diagnosis</th>\n",
       "      <th>ajcc_pathologic_n</th>\n",
       "      <th>ajcc_pathologic_stage</th>\n",
       "      <th>ajcc_pathologic_t</th>\n",
       "      <th>case_submitter_id</th>\n",
       "      <th>ethnicity</th>\n",
       "      <th>gender</th>\n",
       "      <th>morphology</th>\n",
       "      <th>primary_diagnosis</th>\n",
       "      <th>race</th>\n",
       "      <th>study</th>\n",
       "      <th>tissue_or_organ_of_origin</th>\n",
       "      <th>tumor_focality</th>\n",
       "      <th>tumor_grade</th>\n",
       "      <th>tumor_largest_dimension_diameter</th>\n",
       "      <th>vital_status</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>[[S001, C3L-00006, No, 2, 5, 128N, C3L-00006-0...</td>\n",
       "      <td>dou_mmc1.xlsx</td>\n",
       "      <td>[idx, Proteomics_Participant_ID, Case_excluded...</td>\n",
       "      <td>dou_mmc1.xlsx_UCEC_CPTAC3_meta_table_V2.1</td>\n",
       "      <td>Age of the patient at the time of diagnosis</td>\n",
       "      <td>The codes that represent the stage of cancer b...</td>\n",
       "      <td>The extent of a cancer, especially whether the...</td>\n",
       "      <td>Path_Stage_Primary_Tumor-pT</td>\n",
       "      <td>Proteomics_Participant_ID</td>\n",
       "      <td>Whether an individual describes themselves as ...</td>\n",
       "      <td>Text designations that identify gender</td>\n",
       "      <td>ICD-O Morphological code</td>\n",
       "      <td>Text term used to describe the patient's histo...</td>\n",
       "      <td>An arbitrary classification of a taxonomic gro...</td>\n",
       "      <td>CPTAC3</td>\n",
       "      <td>The text term used to describe the anatomic si...</td>\n",
       "      <td>The text term used to describe whether the pat...</td>\n",
       "      <td>Numeric value to express the degree of abnorma...</td>\n",
       "      <td>value</td>\n",
       "      <td>The vital status of the patient</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                rows       filename  \\\n",
       "0  [[S001, C3L-00006, No, 2, 5, 128N, C3L-00006-0...  dou_mmc1.xlsx   \n",
       "\n",
       "                                              header  \\\n",
       "0  [idx, Proteomics_Participant_ID, Case_excluded...   \n",
       "\n",
       "                                        name  \\\n",
       "0  dou_mmc1.xlsx_UCEC_CPTAC3_meta_table_V2.1   \n",
       "\n",
       "                              age_at_diagnosis  \\\n",
       "0  Age of the patient at the time of diagnosis   \n",
       "\n",
       "                                   ajcc_pathologic_n  \\\n",
       "0  The codes that represent the stage of cancer b...   \n",
       "\n",
       "                               ajcc_pathologic_stage  \\\n",
       "0  The extent of a cancer, especially whether the...   \n",
       "\n",
       "             ajcc_pathologic_t          case_submitter_id  \\\n",
       "0  Path_Stage_Primary_Tumor-pT  Proteomics_Participant_ID   \n",
       "\n",
       "                                           ethnicity  \\\n",
       "0  Whether an individual describes themselves as ...   \n",
       "\n",
       "                                   gender                morphology  \\\n",
       "0  Text designations that identify gender  ICD-O Morphological code   \n",
       "\n",
       "                                   primary_diagnosis  \\\n",
       "0  Text term used to describe the patient's histo...   \n",
       "\n",
       "                                                race   study  \\\n",
       "0  An arbitrary classification of a taxonomic gro...  CPTAC3   \n",
       "\n",
       "                           tissue_or_organ_of_origin  \\\n",
       "0  The text term used to describe the anatomic si...   \n",
       "\n",
       "                                      tumor_focality  \\\n",
       "0  The text term used to describe whether the pat...   \n",
       "\n",
       "                                         tumor_grade  \\\n",
       "0  Numeric value to express the degree of abnorma...   \n",
       "\n",
       "  tumor_largest_dimension_diameter                     vital_status  \n",
       "0                            value  The vital status of the patient  "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "DataDirectory().clear_cache(keep_registry=True)\n",
    "\n",
    "xls = Dataset('biofabric-tiny', schema=XLSFile)\n",
    "patient_tables = xls.convert(Table, desc=\"All tables in the file\", cardinality=Cardinality.ONE_TO_MANY)\n",
    "patient_tables = patient_tables.filter(\"The table contains biometric information about the patient\")\n",
    "case_data = patient_tables.convert(CaseData, desc=\"The patient data in the table\", cardinality=Cardinality.ONE_TO_MANY)\n",
    "\n",
    "policy = MinCost()\n",
    "config = QueryProcessorConfig(\n",
    "    policy=policy,\n",
    "    nocache=True,\n",
    "    allow_code_synth=False,\n",
    "    allow_token_reduction=False,\n",
    "    processing_strategy=\"streaming\",\n",
    "    execution_strategy=\"sequential\",\n",
    ")\n",
    "iterable = case_data.run(config)\n",
    "\n",
    "output_rows = []\n",
    "for data_record_collection in iterable:  # noqa: B007\n",
    "    for output_table in data_record_collection:\n",
    "        print(output_table.to_dict().keys())\n",
    "        output_rows.append(output_table.to_dict()) \n",
    "\n",
    "output_df = pd.DataFrame(output_rows)\n",
    "display(output_df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "output_table.__dict__.keys()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Maximum Cost: GPT 4"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "LOGICAL PLANS: 1\n",
      "INITIAL PLANS: 27\n",
      "DEDUP PLANS: 27\n",
      "PARETO PLANS: 17\n",
      "----------\n",
      "Policy is: Maximum Quality\n",
      "Chosen plan: Time est: 411.647 -- Cost est: 0.973 -- Quality est: 0.486\n",
      " 0. MarshalAndScanDataOp -> File \n",
      "\n",
      " 1. File -> InduceFromCandidateOp -> XLSFile \n",
      "    Using Model.GPT_3_5\n",
      "    (contents,filena...) -> (contents,filena...)\n",
      "\n",
      " 2. XLSFile -> InduceFromCandidateOp -> Table \n",
      "    Using Model.GPT_4\n",
      "    (contents,filena...) -> (filename,header...)\n",
      "\n",
      " 3. Table -> FilterCandidateOp -> Table \n",
      "    Using Model.GPT_4\n",
      "    Filter: \"The table contains biometric information about the patient\"\n",
      "    (filename,header...) -> (filename,header...)\n",
      "\n",
      " 4. Table -> InduceFromCandidateOp -> CaseData \n",
      "    Using Model.GPT_4\n",
      "    (filename,header...) -> (age_at_diagnosi...)\n",
      "\n",
      "Bonded query processing error: No output objects were generated with bonded query - trying with conventional query...\n",
      "BondedQuery Error: No output objects were generated with bonded query - trying with conventional query...\n",
      "Falling back to conventional query\n",
      "Could not find any items in the JSON response\n",
      "Could not find any items in the JSON response\n",
      "Bonded query processing error: No output objects were generated with bonded query - trying with conventional query...\n",
      "BondedQuery Error: No output objects were generated with bonded query - trying with conventional query...\n",
      "Falling back to conventional query\n",
      "Could not find any items in the JSON response\n",
      "Could not find any items in the JSON response\n",
      "Could not find any items in the JSON response\n",
      "Could not find any items in the JSON response\n",
      "Could not find any items in the JSON response\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>age_at_diagnosis</th>\n",
       "      <th>ajcc_pathologic_n</th>\n",
       "      <th>ajcc_pathologic_stage</th>\n",
       "      <th>ajcc_pathologic_t</th>\n",
       "      <th>case_submitter_id</th>\n",
       "      <th>ethnicity</th>\n",
       "      <th>gender</th>\n",
       "      <th>morphology</th>\n",
       "      <th>primary_diagnosis</th>\n",
       "      <th>race</th>\n",
       "      <th>study</th>\n",
       "      <th>tissue_or_organ_of_origin</th>\n",
       "      <th>tumor_focality</th>\n",
       "      <th>tumor_grade</th>\n",
       "      <th>tumor_largest_dimension_diameter</th>\n",
       "      <th>vital_status</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>64.0</td>\n",
       "      <td>pN0</td>\n",
       "      <td>Stage I</td>\n",
       "      <td>pT1a (FIGO IA)</td>\n",
       "      <td>C3L-00006</td>\n",
       "      <td>Not-Hispanic or Latino</td>\n",
       "      <td>Female</td>\n",
       "      <td>Endometrioid</td>\n",
       "      <td>FIGO grade 1</td>\n",
       "      <td>White</td>\n",
       "      <td>UCEC_CPTAC3</td>\n",
       "      <td>Anterior endometrium</td>\n",
       "      <td>Unifocal</td>\n",
       "      <td>Cannot be determined</td>\n",
       "      <td>2.9</td>\n",
       "      <td>No</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>58.0</td>\n",
       "      <td>pNX</td>\n",
       "      <td>Stage IV</td>\n",
       "      <td>pT1a (FIGO IA)</td>\n",
       "      <td>C3L-00008</td>\n",
       "      <td>Not-Hispanic or Latino</td>\n",
       "      <td>Female</td>\n",
       "      <td>Endometrioid</td>\n",
       "      <td>FIGO grade 1</td>\n",
       "      <td>White</td>\n",
       "      <td>UCEC_CPTAC3</td>\n",
       "      <td>Posterior endometrium</td>\n",
       "      <td>Unifocal</td>\n",
       "      <td>Cannot be determined</td>\n",
       "      <td>3.5</td>\n",
       "      <td>No</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>50.0</td>\n",
       "      <td>pN0</td>\n",
       "      <td>Stage I</td>\n",
       "      <td>pT1a (FIGO IA)</td>\n",
       "      <td>C3L-00032</td>\n",
       "      <td>Not-Hispanic or Latino</td>\n",
       "      <td>Female</td>\n",
       "      <td>Endometrioid</td>\n",
       "      <td>FIGO grade 2</td>\n",
       "      <td>White</td>\n",
       "      <td>UCEC_CPTAC3</td>\n",
       "      <td>Other, specify</td>\n",
       "      <td>Unifocal</td>\n",
       "      <td>Cannot be determined</td>\n",
       "      <td>4.5</td>\n",
       "      <td>Yes</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>C3L-00084</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>Carcinosarcoma</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>UCEC_CPTAC3</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>75.0</td>\n",
       "      <td>pNX</td>\n",
       "      <td>Stage I</td>\n",
       "      <td>pT1a (FIGO IA)</td>\n",
       "      <td>C3L-00090</td>\n",
       "      <td>Not-Hispanic or Latino</td>\n",
       "      <td>Female</td>\n",
       "      <td>Endometrioid</td>\n",
       "      <td>FIGO grade 2</td>\n",
       "      <td>White</td>\n",
       "      <td>UCEC_CPTAC3</td>\n",
       "      <td>Other, specify</td>\n",
       "      <td>Unifocal</td>\n",
       "      <td>Cannot be determined</td>\n",
       "      <td>3.5</td>\n",
       "      <td>No</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>729.0</td>\n",
       "      <td>N2b</td>\n",
       "      <td>Stage III</td>\n",
       "      <td>T4a</td>\n",
       "      <td>01CO001</td>\n",
       "      <td>None</td>\n",
       "      <td>Male</td>\n",
       "      <td>Mucinous</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>Vasaikar</td>\n",
       "      <td>Sigmoid Colon</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Living</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>838.0</td>\n",
       "      <td>N0</td>\n",
       "      <td>Stage II</td>\n",
       "      <td>T3</td>\n",
       "      <td>01CO005</td>\n",
       "      <td>None</td>\n",
       "      <td>Female</td>\n",
       "      <td>Not Mucinous</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>Vasaikar</td>\n",
       "      <td>Sigmoid Colon</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Deceased</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>904.0</td>\n",
       "      <td>N2b</td>\n",
       "      <td>Stage III</td>\n",
       "      <td>T4a</td>\n",
       "      <td>01CO006</td>\n",
       "      <td>None</td>\n",
       "      <td>Female</td>\n",
       "      <td>Mucinous</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>Vasaikar</td>\n",
       "      <td>Ascending Colon</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Living</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>652.0</td>\n",
       "      <td>N0</td>\n",
       "      <td>Stage II</td>\n",
       "      <td>T3</td>\n",
       "      <td>01CO008</td>\n",
       "      <td>None</td>\n",
       "      <td>Female</td>\n",
       "      <td>Mucinous</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>Vasaikar</td>\n",
       "      <td>Descending Colon</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Living</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>58.0</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>C3L-00104</td>\n",
       "      <td>Not-Hispanic or Latino</td>\n",
       "      <td>Male</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>White</td>\n",
       "      <td>wang</td>\n",
       "      <td>Frontal Lobe</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Deceased</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>59.0</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>C3L-00365</td>\n",
       "      <td>Not-Hispanic or Latino</td>\n",
       "      <td>Female</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>White</td>\n",
       "      <td>wang</td>\n",
       "      <td>Parietal Lobe</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Deceased</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>45.0</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>C3L-00674</td>\n",
       "      <td>Not-Hispanic or Latino</td>\n",
       "      <td>Male</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>White</td>\n",
       "      <td>wang</td>\n",
       "      <td>Frontal Lobe</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Deceased</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>69.0</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>C3L-00677</td>\n",
       "      <td>Not-Hispanic or Latino</td>\n",
       "      <td>Female</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>White</td>\n",
       "      <td>wang</td>\n",
       "      <td>Frontal Lobe</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Deceased</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>77.0</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>C3L-01040</td>\n",
       "      <td>nan</td>\n",
       "      <td>Male</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>nan</td>\n",
       "      <td>wang</td>\n",
       "      <td>Frontal Lobe</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Living</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    age_at_diagnosis ajcc_pathologic_n ajcc_pathologic_stage  \\\n",
       "0               64.0               pN0               Stage I   \n",
       "1               58.0               pNX              Stage IV   \n",
       "2               50.0               pN0               Stage I   \n",
       "3                NaN              None                  None   \n",
       "4               75.0               pNX               Stage I   \n",
       "5              729.0               N2b             Stage III   \n",
       "6              838.0                N0              Stage II   \n",
       "7              904.0               N2b             Stage III   \n",
       "8              652.0                N0              Stage II   \n",
       "9               58.0              None                  None   \n",
       "10              59.0              None                  None   \n",
       "11              45.0              None                  None   \n",
       "12              69.0              None                  None   \n",
       "13              77.0              None                  None   \n",
       "\n",
       "   ajcc_pathologic_t case_submitter_id               ethnicity  gender  \\\n",
       "0     pT1a (FIGO IA)         C3L-00006  Not-Hispanic or Latino  Female   \n",
       "1     pT1a (FIGO IA)         C3L-00008  Not-Hispanic or Latino  Female   \n",
       "2     pT1a (FIGO IA)         C3L-00032  Not-Hispanic or Latino  Female   \n",
       "3               None         C3L-00084                    None    None   \n",
       "4     pT1a (FIGO IA)         C3L-00090  Not-Hispanic or Latino  Female   \n",
       "5                T4a           01CO001                    None    Male   \n",
       "6                 T3           01CO005                    None  Female   \n",
       "7                T4a           01CO006                    None  Female   \n",
       "8                 T3           01CO008                    None  Female   \n",
       "9               None         C3L-00104  Not-Hispanic or Latino    Male   \n",
       "10              None         C3L-00365  Not-Hispanic or Latino  Female   \n",
       "11              None         C3L-00674  Not-Hispanic or Latino    Male   \n",
       "12              None         C3L-00677  Not-Hispanic or Latino  Female   \n",
       "13              None         C3L-01040                     nan    Male   \n",
       "\n",
       "        morphology primary_diagnosis   race        study  \\\n",
       "0     Endometrioid      FIGO grade 1  White  UCEC_CPTAC3   \n",
       "1     Endometrioid      FIGO grade 1  White  UCEC_CPTAC3   \n",
       "2     Endometrioid      FIGO grade 2  White  UCEC_CPTAC3   \n",
       "3   Carcinosarcoma              None   None  UCEC_CPTAC3   \n",
       "4     Endometrioid      FIGO grade 2  White  UCEC_CPTAC3   \n",
       "5         Mucinous              None   None     Vasaikar   \n",
       "6     Not Mucinous              None   None     Vasaikar   \n",
       "7         Mucinous              None   None     Vasaikar   \n",
       "8         Mucinous              None   None     Vasaikar   \n",
       "9             None              None  White         wang   \n",
       "10            None              None  White         wang   \n",
       "11            None              None  White         wang   \n",
       "12            None              None  White         wang   \n",
       "13            None              None    nan         wang   \n",
       "\n",
       "   tissue_or_organ_of_origin tumor_focality           tumor_grade  \\\n",
       "0       Anterior endometrium       Unifocal  Cannot be determined   \n",
       "1      Posterior endometrium       Unifocal  Cannot be determined   \n",
       "2             Other, specify       Unifocal  Cannot be determined   \n",
       "3                       None           None                  None   \n",
       "4             Other, specify       Unifocal  Cannot be determined   \n",
       "5              Sigmoid Colon           None                  None   \n",
       "6              Sigmoid Colon           None                  None   \n",
       "7            Ascending Colon           None                  None   \n",
       "8           Descending Colon           None                  None   \n",
       "9               Frontal Lobe           None                  None   \n",
       "10             Parietal Lobe           None                  None   \n",
       "11              Frontal Lobe           None                  None   \n",
       "12              Frontal Lobe           None                  None   \n",
       "13              Frontal Lobe           None                  None   \n",
       "\n",
       "    tumor_largest_dimension_diameter vital_status  \n",
       "0                                2.9           No  \n",
       "1                                3.5           No  \n",
       "2                                4.5          Yes  \n",
       "3                                NaN         None  \n",
       "4                                3.5           No  \n",
       "5                                NaN       Living  \n",
       "6                                NaN     Deceased  \n",
       "7                                NaN       Living  \n",
       "8                                NaN       Living  \n",
       "9                                NaN     Deceased  \n",
       "10                               NaN     Deceased  \n",
       "11                               NaN     Deceased  \n",
       "12                               NaN     Deceased  \n",
       "13                               NaN       Living  "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "DataDirectory().clear_cache(keep_registry=True)\n",
    "\n",
    "xls = Dataset('biofabric-tiny', schema=XLSFile)\n",
    "patient_tables = xls.convert(Table, desc=\"All tables in the file\", cardinality=Cardinality.ONE_TO_MANY)\n",
    "patient_tables = patient_tables.filter(\"The table contains biometric information about the patient\")\n",
    "case_data = patient_tables.convert(CaseData, desc=\"The patient data in the table\",cardinality=Cardinality.ONE_TO_MANY)\n",
    "\n",
    "policy = MaxQuality()\n",
    "config = QueryProcessorConfig(\n",
    "    policy=policy,\n",
    "    nocache=True,\n",
    "    allow_token_reduction=False,\n",
    "    allow_code_synth=False,\n",
    "    processing_strategy=\"streaming\",\n",
    "    execution_strategy=\"sequential\",\n",
    ")\n",
    "iterable = case_data.run(config)\n",
    "\n",
    "output_rows = []\n",
    "for data_record_collection in iterable:  # noqa: B007\n",
    "    for output_table in data_record_collection:\n",
    "        print(output_table.to_dict().keys())\n",
    "        output_rows.append(output_table.to_dict()) \n",
    "\n",
    "output_df = pd.DataFrame(output_rows)\n",
    "display(output_df)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.14"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
